<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 Division | Cell Atlas</title>
  <meta name="description" content="An atlas produced by the Jensen Lab." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 Division | Cell Atlas" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="An atlas produced by the Jensen Lab." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 Division | Cell Atlas" />
  
  <meta name="twitter:description" content="An atlas produced by the Jensen Lab." />
  

<meta name="author" content="Jensen Lab" />


<meta name="date" content="2020-02-25" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="growth.html"/>
<link rel="next" href="motility.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/CL-0.1.2/CL.js"></script>
<script src="libs/doivideo-binding-0.0.0.9000/doivideo.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Cell Atlas</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>1</b> Methods</a><ul>
<li class="chapter" data-level="1.1" data-path="methods.html"><a href="methods.html#light-microscopy"><i class="fa fa-check"></i><b>1.1</b> Light microscopy</a></li>
<li class="chapter" data-level="1.2" data-path="methods.html"><a href="methods.html#fluorescence-light-microscopy"><i class="fa fa-check"></i><b>1.2</b> Fluorescence light microscopy</a></li>
<li class="chapter" data-level="1.3" data-path="methods.html"><a href="methods.html#electron-microscopy"><i class="fa fa-check"></i><b>1.3</b> Electron microscopy</a></li>
<li class="chapter" data-level="1.4" data-path="methods.html"><a href="methods.html#thin-section-tem"><i class="fa fa-check"></i><b>1.4</b> Thin-section TEM</a></li>
<li class="chapter" data-level="1.5" data-path="methods.html"><a href="methods.html#cryo-em"><i class="fa fa-check"></i><b>1.5</b> Cryo-EM</a></li>
<li class="chapter" data-level="1.6" data-path="methods.html"><a href="methods.html#cryo-et"><i class="fa fa-check"></i><b>1.6</b> Cryo-ET</a></li>
<li class="chapter" data-level="1.7" data-path="methods.html"><a href="methods.html#subtomogram-averaging"><i class="fa fa-check"></i><b>1.7</b> Subtomogram averaging</a></li>
<li class="chapter" data-level="1.8" data-path="methods.html"><a href="methods.html#clem"><i class="fa fa-check"></i><b>1.8</b> CLEM</a></li>
<li class="chapter" data-level="1.9" data-path="methods.html"><a href="methods.html#single-particle-reconstruction"><i class="fa fa-check"></i><b>1.9</b> Single-particle reconstruction</a></li>
<li class="chapter" data-level="1.10" data-path="methods.html"><a href="methods.html#x-ray-crystallography"><i class="fa fa-check"></i><b>1.10</b> X-ray crystallography</a></li>
<li class="chapter" data-level="1.11" data-path="methods.html"><a href="methods.html#visual-proteomics"><i class="fa fa-check"></i><b>1.11</b> Visual proteomics</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="envelope.html"><a href="envelope.html"><i class="fa fa-check"></i><b>2</b> Envelope</a><ul>
<li class="chapter" data-level="2.1" data-path="envelope.html"><a href="envelope.html#mycoplasma-genitalium"><i class="fa fa-check"></i><b>2.1</b> Mycoplasma genitalium</a><ul>
<li class="chapter" data-level="2.1.1" data-path="envelope.html"><a href="envelope.html#Lipid_bilayer"><i class="fa fa-check"></i><b>2.1.1</b> Lipid bilayer</a></li>
<li class="chapter" data-level="2.1.2" data-path="envelope.html"><a href="envelope.html#ATP_synthase_and_energy_production"><i class="fa fa-check"></i><b>2.1.2</b> ATP synthase and energy production</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="envelope.html"><a href="envelope.html#listeria-monocytogenes"><i class="fa fa-check"></i><b>2.2</b> Listeria monocytogenes</a><ul>
<li class="chapter" data-level="2.2.1" data-path="envelope.html"><a href="envelope.html#Peptidoglycan_architecture"><i class="fa fa-check"></i><b>2.2.1</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="2.2.2" data-path="envelope.html"><a href="envelope.html#L-form_bacteria"><i class="fa fa-check"></i><b>2.2.2</b> L-form bacteria</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="envelope.html"><a href="envelope.html#cupriavidus-necator"><i class="fa fa-check"></i><b>2.3</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="2.3.1" data-path="envelope.html"><a href="envelope.html#Brauns_lipoprotein"><i class="fa fa-check"></i><b>2.3.1</b> Braun’s lipoprotein</a></li>
<li class="chapter" data-level="2.3.2" data-path="envelope.html"><a href="envelope.html#Diderm_sacculus_architecture"><i class="fa fa-check"></i><b>2.3.2</b> Escherichia coli</a></li>
<li class="chapter" data-level="2.3.3" data-path="envelope.html"><a href="envelope.html#Sacculus_remodeling"><i class="fa fa-check"></i><b>2.3.3</b> Sacculus remodeling</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="envelope.html"><a href="envelope.html#borrelia-burgdorferi"><i class="fa fa-check"></i><b>2.4</b> Borrelia burgdorferi</a><ul>
<li class="chapter" data-level="2.4.1" data-path="envelope.html"><a href="envelope.html#Vesicle_morphologies"><i class="fa fa-check"></i><b>2.4.1</b> Borrelia burgdorferi</a></li>
<li class="chapter" data-level="2.4.2" data-path="envelope.html"><a href="envelope.html#Inner_membrane_vesicles"><i class="fa fa-check"></i><b>2.4.2</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="2.4.3" data-path="envelope.html"><a href="envelope.html#Cytoplasmic_vesicle_variety"><i class="fa fa-check"></i><b>2.4.3</b> Prosthecobacter debontii</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="envelope.html"><a href="envelope.html#mycoplasma-marinum"><i class="fa fa-check"></i><b>2.5</b> Mycoplasma marinum</a><ul>
<li class="chapter" data-level="2.5.1" data-path="envelope.html"><a href="envelope.html#Mycobacterial_architecture"><i class="fa fa-check"></i><b>2.5.1</b> Mycobacterial architecture</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="envelope.html"><a href="envelope.html#caulobacter-crescentus"><i class="fa fa-check"></i><b>2.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="2.6.1" data-path="envelope.html"><a href="envelope.html#S-layer_architecture"><i class="fa fa-check"></i><b>2.6.1</b> S-layer architecture</a></li>
<li class="chapter" data-level="2.6.2" data-path="envelope.html"><a href="envelope.html#Nanopods"><i class="fa fa-check"></i><b>2.6.2</b> Delftia acidovorans</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="envelope.html"><a href="envelope.html#sulfolobus-solfataricus"><i class="fa fa-check"></i><b>2.7</b> Sulfolobus solfataricus</a><ul>
<li class="chapter" data-level="2.7.1" data-path="envelope.html"><a href="envelope.html#M._alcaliphilum"><i class="fa fa-check"></i><b>2.7.1</b> Methylomicrobium alcaliphilum</a></li>
<li class="chapter" data-level="2.7.2" data-path="envelope.html"><a href="envelope.html#N._maritimus"><i class="fa fa-check"></i><b>2.7.2</b> Nitrosopumilis maritimus</a></li>
<li class="chapter" data-level="2.7.3" data-path="envelope.html"><a href="envelope.html#Methanoregula_formicica"><i class="fa fa-check"></i><b>2.7.3</b> Methanoregula formicica</a></li>
</ul></li>
<li class="chapter" data-level="2.8" data-path="envelope.html"><a href="envelope.html#methanospirillum-hungatei"><i class="fa fa-check"></i><b>2.8</b> Methanospirillum hungatei</a></li>
<li class="chapter" data-level="2.9" data-path="envelope.html"><a href="envelope.html#haloarcula-argentinensis"><i class="fa fa-check"></i><b>2.9</b> Haloarcula argentinensis</a></li>
<li class="chapter" data-level="2.10" data-path="envelope.html"><a href="envelope.html#bdellovibrio-bacteriovorus"><i class="fa fa-check"></i><b>2.10</b> Bdellovibrio bacteriovorus</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="shape.html"><a href="shape.html"><i class="fa fa-check"></i><b>3</b> Shape</a><ul>
<li class="chapter" data-level="3.1" data-path="shape.html"><a href="shape.html#simkania-negevensis"><i class="fa fa-check"></i><b>3.1</b> Simkania negevensis</a><ul>
<li class="chapter" data-level="3.1.1" data-path="shape.html"><a href="shape.html#Cell_shape_diversity"><i class="fa fa-check"></i><b>3.1.1</b> Cell shape diversity</a></li>
<li class="chapter" data-level="3.1.2" data-path="shape.html"><a href="shape.html#Spherical_growth"><i class="fa fa-check"></i><b>3.1.2</b> Spherical growth</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="shape.html"><a href="shape.html#cupriavidus-necator-1"><i class="fa fa-check"></i><b>3.2</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="3.2.1" data-path="shape.html"><a href="shape.html#Rod-shaped_growth"><i class="fa fa-check"></i><b>3.2.1</b> Rod-shaped growth</a></li>
<li class="chapter" data-level="3.2.2" data-path="shape.html"><a href="shape.html#Rod_variety"><i class="fa fa-check"></i><b>3.2.2</b> Brucella abortus</a></li>
<li class="chapter" data-level="3.2.3" data-path="shape.html"><a href="shape.html#Archaeal_rods"><i class="fa fa-check"></i><b>3.2.3</b> Methanoregula formicica</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="shape.html"><a href="shape.html#hylemonella-gracilis"><i class="fa fa-check"></i><b>3.3</b> Hylemonella gracilis</a></li>
<li class="chapter" data-level="3.4" data-path="shape.html"><a href="shape.html#caulobacter-crescentus-1"><i class="fa fa-check"></i><b>3.4</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="3.4.1" data-path="shape.html"><a href="shape.html#Vibrioid_growth"><i class="fa fa-check"></i><b>3.4.1</b> Vibrioid growth</a></li>
<li class="chapter" data-level="3.4.2" data-path="shape.html"><a href="shape.html#CTP_synthase"><i class="fa fa-check"></i><b>3.4.2</b> CTP synthase</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="shape.html"><a href="shape.html#campylobacter-jejuni"><i class="fa fa-check"></i><b>3.5</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="3.5.1" data-path="shape.html"><a href="shape.html#Borrelia_burgdorferi"><i class="fa fa-check"></i><b>3.5.1</b> Borrelia burgdorferi</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="shape.html"><a href="shape.html#verrucomicrobium-spinosum"><i class="fa fa-check"></i><b>3.6</b> Verrucomicrobium spinosum</a><ul>
<li class="chapter" data-level="3.6.1" data-path="shape.html"><a href="shape.html#Bactofilin"><i class="fa fa-check"></i><b>3.6.1</b> Bactofilin</a></li>
<li class="chapter" data-level="3.6.2" data-path="shape.html"><a href="shape.html#Bacterial_microtubules"><i class="fa fa-check"></i><b>3.6.2</b> Prosthecobacter vanneervenii</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="shape.html"><a href="shape.html#haloquadratum-walsbyi"><i class="fa fa-check"></i><b>3.7</b> Haloquadratum walsbyi</a><ul>
<li class="chapter" data-level="3.7.1" data-path="shape.html"><a href="shape.html#Gas_vesicles"><i class="fa fa-check"></i><b>3.7.1</b> Halobacterium salinarum</a></li>
<li class="chapter" data-level="3.7.2" data-path="shape.html"><a href="shape.html#Gas_vesicles"><i class="fa fa-check"></i><b>3.7.2</b> Gas vesicles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="growth.html"><a href="growth.html"><i class="fa fa-check"></i><b>4</b> Growth</a><ul>
<li class="chapter" data-level="4.1" data-path="growth.html"><a href="growth.html#caulobacter-crescentus-2"><i class="fa fa-check"></i><b>4.1</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="4.2" data-path="growth.html"><a href="growth.html#rhodopseudomonas-palustris"><i class="fa fa-check"></i><b>4.2</b> Rhodopseudomonas palustris</a><ul>
<li class="chapter" data-level="4.2.1" data-path="growth.html"><a href="growth.html#ICM_variety"><i class="fa fa-check"></i><b>4.2.1</b> Methyloprofundus sedimenti</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="growth.html"><a href="growth.html#chloroflexi"><i class="fa fa-check"></i><b>4.3</b> Chloroflexi</a></li>
<li class="chapter" data-level="4.4" data-path="growth.html"><a href="growth.html#thiomonas-intermedia"><i class="fa fa-check"></i><b>4.4</b> Thiomonas intermedia</a><ul>
<li class="chapter" data-level="4.4.1" data-path="growth.html"><a href="growth.html#Carboxysome_variety"><i class="fa fa-check"></i><b>4.4.1</b> Halothiobacillus neapolitanus</a></li>
<li class="chapter" data-level="4.4.2" data-path="growth.html"><a href="growth.html#Long_carboxysomes"><i class="fa fa-check"></i><b>4.4.2</b> Hydrogenovibrio crunogenus</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="growth.html"><a href="growth.html#cupriavidus-necator-2"><i class="fa fa-check"></i><b>4.5</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="4.5.1" data-path="growth.html"><a href="growth.html#Storage_granule_growth"><i class="fa fa-check"></i><b>4.5.1</b> Lysobacter antibioticus</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="growth.html"><a href="growth.html#agrobacterium-tumefaciens"><i class="fa fa-check"></i><b>4.6</b> Agrobacterium tumefaciens</a></li>
<li class="chapter" data-level="4.7" data-path="growth.html"><a href="growth.html#hydrogenovibrio-crunogenus"><i class="fa fa-check"></i><b>4.7</b> Hydrogenovibrio crunogenus</a></li>
<li class="chapter" data-level="4.8" data-path="growth.html"><a href="growth.html#halothiobacillus-neapolitanus"><i class="fa fa-check"></i><b>4.8</b> Halothiobacillus neapolitanus</a></li>
<li class="chapter" data-level="4.9" data-path="growth.html"><a href="growth.html#halorubrum-litoreum"><i class="fa fa-check"></i><b>4.9</b> Halorubrum litoreum</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="division.html"><a href="division.html"><i class="fa fa-check"></i><b>5</b> Division</a><ul>
<li class="chapter" data-level="5.1" data-path="division.html"><a href="division.html#thiomonas-intermedia-1"><i class="fa fa-check"></i><b>5.1</b> Thiomonas intermedia</a></li>
<li class="chapter" data-level="5.2" data-path="division.html"><a href="division.html#caulobacter-crescentus-3"><i class="fa fa-check"></i><b>5.2</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="5.3" data-path="division.html"><a href="division.html#hyphomonas-neptunium"><i class="fa fa-check"></i><b>5.3</b> Hyphomonas neptunium</a><ul>
<li class="chapter" data-level="5.3.1" data-path="division.html"><a href="division.html#Hyphomonas_stalk"><i class="fa fa-check"></i><b>5.3.1</b> Hyphomonas neptunium</a></li>
<li class="chapter" data-level="5.3.2" data-path="division.html"><a href="division.html#Hyphomonas_life_cycle"><i class="fa fa-check"></i><b>5.3.2</b> Hyphomonas neptunium</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="division.html"><a href="division.html#tetrasphaera-remsis"><i class="fa fa-check"></i><b>5.4</b> Tetrasphaera remsis</a></li>
<li class="chapter" data-level="5.5" data-path="division.html"><a href="division.html#idiomarina-loihiensis"><i class="fa fa-check"></i><b>5.5</b> Idiomarina loihiensis</a><ul>
<li class="chapter" data-level="5.5.1" data-path="division.html"><a href="division.html#sphingopyxis-alaskensis-sphingopyxis_alaskensis_divisionthis-diderm-cell-is-about-halfway-through-division.-the-two-daughter-cells-are-still-connected-by-a-wide-bridge-of-cytoplasm-but-the-inner-membrane-and-cell-wall-are-drawing-in-all-around."><i class="fa fa-check"></i><b>5.5.1</b> Sphingopyxis alaskensis {#Sphingopyxis_alaskensis_division}This diderm cell is about halfway through division. The two daughter cells are still connected by a wide bridge of cytoplasm, but the inner membrane and cell wall are drawing in all around.</a></li>
<li class="chapter" data-level="5.5.2" data-path="division.html"><a href="division.html#Chloroflexi_division"><i class="fa fa-check"></i><b>5.5.2</b> Chloroflexi</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="division.html"><a href="division.html#caulobacter-crescentus-4"><i class="fa fa-check"></i><b>5.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="5.6.1" data-path="division.html"><a href="division.html#Caulobacter_division"><i class="fa fa-check"></i><b>5.6.1</b> Caulobacter crescentus</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="division.html"><a href="division.html#agrobacterium-tumefaciens-1"><i class="fa fa-check"></i><b>5.7</b> Agrobacterium tumefaciens</a></li>
<li class="chapter" data-level="5.8" data-path="division.html"><a href="division.html#cupriavidus-necator-3"><i class="fa fa-check"></i><b>5.8</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="5.8.1" data-path="division.html"><a href="division.html#Asymmetric_constriction"><i class="fa fa-check"></i><b>5.8.1</b> Cupriavidus necator</a></li>
</ul></li>
<li class="chapter" data-level="5.9" data-path="division.html"><a href="division.html#caulobacter-crescentus-5"><i class="fa fa-check"></i><b>5.9</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="5.10" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius"><i class="fa fa-check"></i><b>5.10</b> Sulfolobus acidocaldarius</a></li>
<li class="chapter" data-level="5.11" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius-1"><i class="fa fa-check"></i><b>5.11</b> Sulfolobus acidocaldarius</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="motility.html"><a href="motility.html"><i class="fa fa-check"></i><b>6</b> Motility</a><ul>
<li class="chapter" data-level="6.1" data-path="motility.html"><a href="motility.html#caulobacter-crescentus-6"><i class="fa fa-check"></i><b>6.1</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="6.1.1" data-path="motility.html"><a href="motility.html#Flagellum_morphology"><i class="fa fa-check"></i><b>6.1.1</b> Campylobacter jejuni</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="motility.html"><a href="motility.html#bdellovibrio-bacteriovorus-1"><i class="fa fa-check"></i><b>6.2</b> Bdellovibrio bacteriovorus</a><ul>
<li class="chapter" data-level="6.2.1" data-path="motility.html"><a href="motility.html#Flagellar_motor_structure"><i class="fa fa-check"></i><b>6.2.1</b> Bdellovibrio bacteriovorus</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="motility.html"><a href="motility.html#hylemonella-gracilis-1"><i class="fa fa-check"></i><b>6.3</b> Hylemonella gracilis</a><ul>
<li class="chapter" data-level="6.3.1" data-path="motility.html"><a href="motility.html#Flagellar_motor_disassembly"><i class="fa fa-check"></i><b>6.3.1</b> Pseudomonas aeruginosa</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="motility.html"><a href="motility.html#campylobacter-jejuni-1"><i class="fa fa-check"></i><b>6.4</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="6.4.1" data-path="motility.html"><a href="motility.html#Lophotrichous_bacteria"><i class="fa fa-check"></i><b>6.4.1</b> Helicobacter pylori</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="motility.html"><a href="motility.html#pseudomonas-flexibilis"><i class="fa fa-check"></i><b>6.5</b> Pseudomonas flexibilis</a></li>
<li class="chapter" data-level="6.6" data-path="motility.html"><a href="motility.html#helicobacter-hepaticus"><i class="fa fa-check"></i><b>6.6</b> Helicobacter hepaticus</a></li>
<li class="chapter" data-level="6.7" data-path="motility.html"><a href="motility.html#borrelia-burgdorferi-1"><i class="fa fa-check"></i><b>6.7</b> Borrelia burgdorferi</a><ul>
<li class="chapter" data-level="6.7.1" data-path="motility.html"><a href="motility.html#Bacterial_invasion"><i class="fa fa-check"></i><b>6.7.1</b> Macaca mulatta</a></li>
<li class="chapter" data-level="6.7.2" data-path="motility.html"><a href="motility.html#Treponema_primitia"><i class="fa fa-check"></i><b>6.7.2</b> Treponema primitia</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="motility.html"><a href="motility.html#methanoregula-formicica"><i class="fa fa-check"></i><b>6.8</b> Methanoregula formicica</a></li>
<li class="chapter" data-level="6.9" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis"><i class="fa fa-check"></i><b>6.9</b> Thermococcus kodakaraensis</a><ul>
<li class="chapter" data-level="6.9.1" data-path="motility.html"><a href="motility.html#Cone_structure"><i class="fa fa-check"></i><b>6.9.1</b> Thermococcus kodakaraensis</a></li>
<li class="chapter" data-level="6.9.2" data-path="motility.html"><a href="motility.html#Organizing_center"><i class="fa fa-check"></i><b>6.9.2</b> Thermococcus kodakaraensis</a></li>
</ul></li>
<li class="chapter" data-level="6.10" data-path="motility.html"><a href="motility.html#myxococcus-xanthus"><i class="fa fa-check"></i><b>6.10</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="6.11" data-path="motility.html"><a href="motility.html#flavobacterium-johnsoniae"><i class="fa fa-check"></i><b>6.11</b> Flavobacterium johnsoniae</a></li>
<li class="chapter" data-level="6.12" data-path="motility.html"><a href="motility.html#mycoplasma-pneumoniae"><i class="fa fa-check"></i><b>6.12</b> Mycoplasma pneumoniae</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="navigation.html"><a href="navigation.html"><i class="fa fa-check"></i><b>7</b> Navigation</a></li>
<li class="chapter" data-level="8" data-path="lifecycle.html"><a href="lifecycle.html"><i class="fa fa-check"></i><b>8</b> Lifecycle</a></li>
<li class="chapter" data-level="9" data-path="interaction.html"><a href="interaction.html"><i class="fa fa-check"></i><b>9</b> Interaction</a></li>
<li class="chapter" data-level="10" data-path="viruses.html"><a href="viruses.html"><i class="fa fa-check"></i><b>10</b> Viruses</a></li>
<li class="chapter" data-level="11" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>11</b> References</a></li>
<li class="divider"></li>
<li><a href="https://library.caltech.edu" target="blank">Published by Caltech Library</a><li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cell Atlas</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="division" class="section level1">
<h1><span class="header-section-number">Chapter 5</span> Division</h1>
<div id="thiomonas-intermedia-1" class="section level2">
<h2><span class="header-section-number">5.1</span> Thiomonas intermedia</h2>
<p>Growth only gets you so far. If your cell is coccoid, growth increases volume more rapidly than surface area, which is a problem if your cell relies on nutrients imported from the environment. Even if your cell is rod-shaped (so its surface area to volume ratio remains relatively constant), increased volume increases diffusion times, making metabolism less efficient. So what can you do to keep your cell thriving?</p>
<p>It may be time to divide. In essence, division simply splits a cell, the “mother,” into two “daughters.” Each daughter will be roughly half the size of the mother. A fair split, though, particularly of critical components like the genome, requires careful coordination. Think about the things in your cell. Some of them are present in many copies, like most of the proteins. Others are present in very few copies, like the chromosome(s). Conceptually, we can sort components into two broad categories: high copy-number and low copy-number items. How would you split the high copy-number items? Easy, right? Just split the cell in the middle and each half will have plenty. This is true, for instance, for the ribosomes and carboxysomes you see in this Thiomonas intermedia cell.</p>
<p>What about a low copy-number component? Look at the poly-phosphate storage granules in the same cell. To ensure each daughter cell gets the same number, they’re evenly spaced along the length of the cell, ready for division in the middle.</p>
</div>
<div id="caulobacter-crescentus-3" class="section level2">
<h2><span class="header-section-number">5.2</span> Caulobacter crescentus</h2>
<p>The most important low-copy number component in your cell is its genome; without the instructions, nothing gets built. Reflecting its importance, cells have evolved complex mechanisms to coordinate DNA replication and segregation in time as well as space. The details are beyond our scope here, but let’s examine some general structural principles. If your cell divides by splitting in the middle, what’s the easiest way to get one complete genome copy to each daughter? Assume your cell has a single chromosome (like most bacteria and archaea). Why not just tether each chromosome copy to an opposite pole of the cell? Nearly all bacteria, and many archaea, have a protein called ParB (for Partitioning) that recognizes a specific sequence (ParS) on the chromosome, creating a molecular handle. In Caulobacter crescentus like this cell, ParB also binds a scaffolding protein at the pole called PopZ, hooking the handle to the pole. The PopZ scaffold isn’t highly ordered, so we see it as a diffuse blob of DNA and protein, noticeable because it excludes other large protein complexes like ribosomes. Several species of bacteria use PopZ or other hub-organizing proteins to tether a genome copy, as well as other things like chemosensory machinery, to the cell pole. Other species use a different mechanism involving many copies of a dynamic protein called ParA that bind and release ParB, ratcheting the ParS handle of the chromosome across the cell. Other components, like the storage granules you just saw, piggyback on this machinery for segregation.</p>
<p>Remember that your cell’s chromosome is colossal, so getting the ParS handle to one side is only part of the battle. An army of other proteins work to condense the chromosome to a more manageable volume and wrangle with the division machinery to make sure no straggling loops get caught off-sides.</p>
</div>
<div id="hyphomonas-neptunium" class="section level2">
<h2><span class="header-section-number">5.3</span> Hyphomonas neptunium</h2>
<p>What if your cell divides a different way? Some bacteria divide not by fission, but by budding, like this Hyphomonas neptunium cell. These cells have a stalk [More – Hyphomonas stalk] and concentrate their growth at the end of the stalk, producing a daughter cell like blowing a bubble. When the bud becomes big enough, they divide at the end of the stalk to release it [More – Hyphomonas life cycle]. First, though, they have to make sure all the necessary components make it into the bud. The process is most dramatic for the genome; here you can see a copy being transferred through the stalk. The chromosome here resembles a single double-stranded DNA helix, but it’s actually a higher-order structure of supercoiled DNA. (The crossbands aren’t hydrogen-bonded bases, but rather proteins that help pack the DNA.)</p>
<p>Several other bacterial species divide by budding, although not all have stalks; some simply bud from the main cell body, as you’ll see later in this chapter.</p>
<div id="Hyphomonas_stalk" class="section level3">
<h3><span class="header-section-number">5.3.1</span> Hyphomonas neptunium</h3>
<p>Hyphomonas neptunium grow a single stalk from one end of their cell body, similar to the Caulobacter crescentus you saw in Chapter 3. The function of this stalk, though, is very different, as you’ll see.</p>
</div>
<div id="Hyphomonas_life_cycle" class="section level3">
<h3><span class="header-section-number">5.3.2</span> Hyphomonas neptunium</h3>
<p>Hyphomonas neptunium have evolved a program of stages they pass through in the course of their life. When a newborn cell like this one is released, it is in the “swarmer” stage, using its flagellum (discussed in the next chapter) to swim away to a new location. After a while, it settles down, jettisoning its flagellum and growing a stalk to ultimately make its own bud–the next generation. Once a cell settles down into the stalked stage, it spends the rest of its life sending off buds as long as conditions are good. Only new cells are equipped with flagella to go adventuring. We’ll discuss how lifecycles like this can be a beneficial strategy in Chapter 8.</p>
<p>[4_Staphylococcus aureus]</p>
<p>Once your cell has gotten everything where it needs to go, how can it actually divide? Fences separate neighbors, so why not use the cell wall to build a septum (or “fence”) between the two daughters? That’s what this Staphylococcus aureus cell is doing. In monoderm bacteria like this with a thick cell wall, it’s easy to see the septum.</p>
</div>
</div>
<div id="tetrasphaera-remsis" class="section level2">
<h2><span class="header-section-number">5.4</span> Tetrasphaera remsis</h2>
<p>The septum grows in from the periphery of the dividing cell toward the middle, drawing the membrane with it. When it reaches the middle, the membrane seals off on each side and the septum separates to release the two cells. The Tetrasphaera remsis cell at the bottom is at a fairly late stage of this process. This cell also illustrates another point about division: the division plane isn’t always exactly in the middle of the cell. Depending on the species, the two daughter cells may be different sizes.</p>
<p>There can also be more than two daughter cells. Some species undergo simultaneous divisions, with two or more septa forming more or less at the same time. The genus of Tetrasphaera remsis gets its name from this property–cells can either divide into two (as in this case) or four, roughly spherical cells.</p>
</div>
<div id="idiomarina-loihiensis" class="section level2">
<h2><span class="header-section-number">5.5</span> Idiomarina loihiensis</h2>
<p>The division process is similar for diderm bacteria, as you can see in this Idiomarina loihiensis cell, with the cell wall and inner membrane building inward. The process is almost complete here, with just a thin channel of cytoplasm connecting the daughter cells in the very middle. You can see the cell wall zippering apart at the edges of the septum. For another example slightly earlier in the process, check out [More–Sphingopyxis alaskensis division]. For a later example, check out [More–Chloroflexi]. In some species, the outer membrane constricts at the same time as the inner membrane, as you saw, for instance, in the T. intermedia cell at the beginning of this chapter. In others, including this one and S. alaskensis, the outer membrane is mainly remodeled at the end of the process, as the cells separate. Other species are intermediate [More–Caulobacter crescentus division].</p>
<div id="sphingopyxis-alaskensis-sphingopyxis_alaskensis_divisionthis-diderm-cell-is-about-halfway-through-division.-the-two-daughter-cells-are-still-connected-by-a-wide-bridge-of-cytoplasm-but-the-inner-membrane-and-cell-wall-are-drawing-in-all-around." class="section level3">
<h3><span class="header-section-number">5.5.1</span> Sphingopyxis alaskensis {#Sphingopyxis_alaskensis_division}This diderm cell is about halfway through division. The two daughter cells are still connected by a wide bridge of cytoplasm, but the inner membrane and cell wall are drawing in all around.</h3>
</div>
<div id="Chloroflexi_division" class="section level3">
<h3><span class="header-section-number">5.5.2</span> Chloroflexi</h3>
<p>This diderm cell is in the final stage of division. The cytoplasm of the two daughter cells is completely separated by inner membrane and cell wall. The outer membrane, though, has not yet divided.</p>
</div>
</div>
<div id="caulobacter-crescentus-4" class="section level2">
<h2><span class="header-section-number">5.6</span> Caulobacter crescentus</h2>
<p>In some species, the outer membrane constricts at the same time as the inner membrane, as you saw, for instance, in the T. intermedia cell at the beginning of this chapter. In others, including the cells you saw on the last page, the outer membrane is mainly remodeled at the end of the process, as the cells separate. Other species are intermediate. In Caulobacter crescentus, the outer membrane constricts along with the inner membrane and cell wall. At the end of division, though, the inner membrane seals off before the outer membrane, as you can see in this cell, which is at the very end stage of division. For an even later stage, check out [More–Caulobacter division].</p>
<div id="Caulobacter_division" class="section level3">
<h3><span class="header-section-number">5.6.1</span> Caulobacter crescentus</h3>
<p>These daughter cells have completely separated their mother’s inner membrane and cell wall, but not yet the outer membrane and surface layer.</p>
</div>
</div>
<div id="agrobacterium-tumefaciens-1" class="section level2">
<h2><span class="header-section-number">5.7</span> Agrobacterium tumefaciens</h2>
<p>Like monoderms, most diderms divide into roughly equally-sized daughter cells, but not all do. Just like the Tetrasphaera remsis you saw earlier, some diderm species produce one larger, and one smaller, daughter, like this Agrobacterium tumefaciens cell. In this case, it’s because the smaller daughter is actually a bud. A. tumefaciens concentrate their growth on one pole, pushing out a bud that, once it’s large enough, is pinched off and released. The process is very similar to what you saw in Hyphomonas neptunium, just without the stalk.</p>
</div>
<div id="cupriavidus-necator-3" class="section level2">
<h2><span class="header-section-number">5.8</span> Cupriavidus necator</h2>
<p>Division can also be asymmetric in a different way: how constricted one side of the cell is relative to the other. You can see that in this Cupriavidus necator cell. Asymmetric constriction is most common early in division, but sometimes it persists for a while [More–Asymmetric constriction]. To understand how this happens, let’s take a closer look at how your cell divides.</p>
<p>To divide, your cell’s membrane and cell wall constrict. But remember that the cell wall is resisting a lot of force coming from turgor pressure pushing outward. So how can it overcome this pressure and grow inward? For almost all bacteria and many archaea, the answer is a protein contractor called FtsZ (Filamenting Temperature-Sensitive mutant Z, named for a genetic screen for cells that didn’t divide, and therefore grew very long, or filamentous). FtsZ is another piece of your cell’s cytoskeleton. Homologous to eukaryotic tubulin, FtsZ polymerizes into filaments that are linked to the cell membrane at the division plane [Schematic–FtsZ structure]. The filaments are highly dynamic, forming, disassembling and reassembling within seconds.</p>
<p>How does FtsZ find the right spot? There are multiple mechanisms, but an elegant one in cells like this one involves an inhibitor that localizes to the cell poles, making a repressive gradient that’s strongest at the ends of the cell and weakest in the middle. As the rod-shaped cell grows, the inhibitor concentration at the middle eventually drops low enough that FtsZ can polymerize. Most cells also inhibit FtsZ polymerization when the genome is still in the middle, a mechanism called “nucleoid occlusion.” Some species do this with an FtsZ-inhibiting protein called MipZ that grabs onto the ParB chromosomal handle, ensuring that FtsZ filaments don’t form until the chromosome is clear.</p>
<p>Once everything’s ready (the cell is long enough and the nucleoid’s out of the way), FtsZ filaments form at the division site. Initially, a single, short filament (or a few) appears. Already, this is enough to begin constricting the cell which explains why many bacterial cells, like this one, start to constrict only on one side. FtsZ filaments run parallel to the cell membrane, so they appear as dots in cross-section; but if we rotate the cell around, we can see them. Note: we can’t see the membrane (and any associated FtsZ filaments) all the way around the cell due to the “missing wedge” of information in this imaging technique – see Chapter 1 for an explanation.</p>
<div id="Asymmetric_constriction" class="section level3">
<h3><span class="header-section-number">5.8.1</span> Cupriavidus necator</h3>
<p>This C. necator cell is at a later stage of division. In most cells, as you’ll see on the next page, constriction is more or less uniform around the circumference by now, but in some cells, including this one, the asymmetry persists.</p>
<p>5_9_1 Schematic: FtsZ structure</p>
<p>While the structure of FtsZ is known, its mechanism is not. It is an enzyme, so it’s possible that the filament changes curvature as its subunits flip from one conformation to another as a result of the chemical reaction (hydrolyzing GTP). This different filament conformation may pull on the membrane, allowing the cell wall to build inward.</p>
</div>
</div>
<div id="caulobacter-crescentus-5" class="section level2">
<h2><span class="header-section-number">5.9</span> Caulobacter crescentus</h2>
<p>As division progresses, FtsZ filaments keep assembling and growing, and soon they form a complete ring of long, overlapping filaments all around the circumference of the cell. This drives further constriction, more or less uniformly. In this Caulobacter crescentus cell in a later stage of division, you can see many FtsZ filaments lined up in cross-section – the dots on both sides of the division plane; if we rotate the cell around, you can see filaments extending on both sides as far as the membrane is visible (see note on the previous page about why we can’t see the membrane all the way around). They will continue to pull the membrane inward, and direct cell wall to be built behind it, until division is complete.</p>
</div>
<div id="sulfolobus-acidocaldarius" class="section level2">
<h2><span class="header-section-number">5.10</span> Sulfolobus acidocaldarius</h2>
<p>Nearly all bacteria, and many archaea use FtsZ to divide. Other species of archaea, belonging to the Crenarchaeota phylum, use a different cytoskeletal system called Cdv (for Cell division). Cdv proteins are homologous to proteins of the Endosomal Sorting Complexes Required for Transport (ESCRT). ESCRT proteins were discovered in eukaryotes, where they are involved in many processes that involve cinching off membranes, from the final stage of cell division, to virus budding, to endocytosis (hence the name), where cells take things in from the environment. Again, despite its fundamental importance, we still don’t know exactly how the ESCRT machinery works to constrict membranes. In archaeal cells like this Sulfolobus acidocaldarius, Cdv proteins form a belt of parallel filaments around the middle of the cell, defining the division plane <a href="#ESCRT_structure">Schematic – ESCRT structure</a>. Notice that the rigid surface layer is dismantled outside the belt so that the membrane can be pulled inward.</p>
<p>5_11_1 Schematic: ESCRT structure</p>
</div>
<div id="sulfolobus-acidocaldarius-1" class="section level2">
<h2><span class="header-section-number">5.11</span> Sulfolobus acidocaldarius</h2>
<p>The Cdv filaments then pull the membrane inward, constricting the cell, as you can see in this Sulfolobus acidocaldarius cell in a later stage of division. Again, the mechanism isn’t known, but it may be that the filaments slide against each other into a tightening spiral on either side <a href="#Hourglass_model">Schematic – Hourglass model</a>.</p>
<p>The majority of cells divide by one of these two mechanisms–FtsZ and Cdv–but not all do. Some bacterial and archaeal species have neither FtsZ nor Cdv, and we’re still figuring out how they divide.</p>
<p>5_12_1 Schematic: Hourglass model</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="growth.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="motility.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": null,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/caltechlibrary/cell-atlas/edit/master/05_Division.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["cell-atlas.pdf", "cell-atlas.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
