<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Envelope | Cell Atlas</title>
  <meta name="description" content="An atlas produced by the Jensen Lab." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Envelope | Cell Atlas" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="splash/HeaderImage.jpg" />
  <meta property="og:description" content="An atlas produced by the Jensen Lab." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Envelope | Cell Atlas" />
  
  <meta name="twitter:description" content="An atlas produced by the Jensen Lab." />
  <meta name="twitter:image" content="splash/HeaderImage.jpg" />

<meta name="author" content="Jensen Lab" />


<meta name="date" content="2020-08-03" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="methods.html"/>
<link rel="next" href="shape.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/CL-0.1.2/CL.js"></script>
<script src="libs/doivideo-binding-0.0.0.9000/doivideo.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Cell Atlas</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>1</b> Methods</a><ul>
<li class="chapter" data-level="1.1" data-path="methods.html"><a href="methods.html#light-microscopy"><i class="fa fa-check"></i><b>1.1</b> Light Microscopy</a><ul>
<li class="chapter" data-level="1.1.1" data-path="methods.html"><a href="methods.html#Structural_biology_toolkit"><i class="fa fa-check"></i><b>1.1.1</b> Structural biology toolkit</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="methods.html"><a href="methods.html#fluorescence-light-microscopy"><i class="fa fa-check"></i><b>1.2</b> Fluorescence Light Microscopy</a></li>
<li class="chapter" data-level="1.3" data-path="methods.html"><a href="methods.html#scanning-electron-microscopy"><i class="fa fa-check"></i><b>1.3</b> Scanning Electron Microscopy</a><ul>
<li class="chapter" data-level="1.3.1" data-path="methods.html"><a href="methods.html#Electron_microscopy_modes"><i class="fa fa-check"></i><b>1.3.1</b> Electron microscopy modes</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="methods.html"><a href="methods.html#transmission-electron-microscopy"><i class="fa fa-check"></i><b>1.4</b> Transmission Electron Microscopy</a></li>
<li class="chapter" data-level="1.5" data-path="methods.html"><a href="methods.html#cryogenic-electron-microscopy"><i class="fa fa-check"></i><b>1.5</b> Cryogenic Electron Microscopy</a></li>
<li class="chapter" data-level="1.6" data-path="methods.html"><a href="methods.html#electron-cryotomography"><i class="fa fa-check"></i><b>1.6</b> Electron Cryotomography</a><ul>
<li class="chapter" data-level="1.6.1" data-path="methods.html"><a href="methods.html#Missing_wedge"><i class="fa fa-check"></i><b>1.6.1</b> Missing wedge</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="methods.html"><a href="methods.html#sub-tomogram-averaging"><i class="fa fa-check"></i><b>1.7</b> Sub-Tomogram Averaging</a></li>
<li class="chapter" data-level="1.8" data-path="methods.html"><a href="methods.html#single-particle-reconstruction"><i class="fa fa-check"></i><b>1.8</b> Single Particle Reconstruction</a></li>
<li class="chapter" data-level="1.9" data-path="methods.html"><a href="methods.html#x-ray-crystallography"><i class="fa fa-check"></i><b>1.9</b> X-Ray Crystallography</a><ul>
<li class="chapter" data-level="1.9.1" data-path="methods.html"><a href="methods.html#Structural_biology_timeline"><i class="fa fa-check"></i><b>1.9.1</b> Structural biology timeline</a></li>
</ul></li>
<li class="chapter" data-level="1.10" data-path="methods.html"><a href="methods.html#putting-it-all-together"><i class="fa fa-check"></i><b>1.10</b> Putting It All Together</a></li>
<li class="chapter" data-level="1.11" data-path="methods.html"><a href="methods.html#moredifference-mapping"><i class="fa fa-check"></i><b>1.11</b> More–Difference mapping</a></li>
<li class="chapter" data-level="1.12" data-path="methods.html"><a href="methods.html#further-reading"><i class="fa fa-check"></i><b>1.12</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="envelope.html"><a href="envelope.html"><i class="fa fa-check"></i><b>2</b> Envelope</a><ul>
<li class="chapter" data-level="2.1" data-path="envelope.html"><a href="envelope.html#membrane"><i class="fa fa-check"></i><b>2.1</b> Membrane</a><ul>
<li class="chapter" data-level="2.1.1" data-path="envelope.html"><a href="envelope.html#Lipid_bilayer"><i class="fa fa-check"></i><b>2.1.1</b> Lipid bilayer</a></li>
<li class="chapter" data-level="2.1.2" data-path="envelope.html"><a href="envelope.html#PDB:_5T4O_ATP_synthase_and_energy_production"><i class="fa fa-check"></i><b>2.1.2</b> PDB: 5T4O ATP synthase and energy production</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="envelope.html"><a href="envelope.html#cell-wall"><i class="fa fa-check"></i><b>2.2</b> Cell Wall</a></li>
<li class="chapter" data-level="2.3" data-path="envelope.html"><a href="envelope.html#morepeptidoglycan-architecture"><i class="fa fa-check"></i><b>2.3</b> More–Peptidoglycan architecture</a></li>
<li class="chapter" data-level="2.4" data-path="envelope.html"><a href="envelope.html#moremethanobacterium-formicicum"><i class="fa fa-check"></i><b>2.4</b> More–Methanobacterium formicicum</a></li>
<li class="chapter" data-level="2.5" data-path="envelope.html"><a href="envelope.html#outer-membrane"><i class="fa fa-check"></i><b>2.5</b> Outer Membrane</a><ul>
<li class="chapter" data-level="2.5.1" data-path="envelope.html"><a href="envelope.html#pdb-1eq7-brauns-lipoprotein-pdb1eq7brauns_lipoprotein"><i class="fa fa-check"></i><b>2.5.1</b> PDB: 1EQ7 –Braun’s lipoprotein {#PDB:<em>1EQ7</em>–Brauns_lipoprotein}</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="envelope.html"><a href="envelope.html#morediderm-sacculus-architecture"><i class="fa fa-check"></i><b>2.6</b> More–Diderm sacculus architecture</a><ul>
<li class="chapter" data-level="2.6.1" data-path="envelope.html"><a href="envelope.html#Sacculus_remodeling"><i class="fa fa-check"></i><b>2.6.1</b> Sacculus remodeling</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="envelope.html"><a href="envelope.html#vesicles"><i class="fa fa-check"></i><b>2.7</b> Vesicles</a></li>
<li class="chapter" data-level="2.8" data-path="envelope.html"><a href="envelope.html#morepearled-vesicles"><i class="fa fa-check"></i><b>2.8</b> More–Pearled vesicles</a></li>
<li class="chapter" data-level="2.9" data-path="envelope.html"><a href="envelope.html#moretubular-vesicles"><i class="fa fa-check"></i><b>2.9</b> More–Tubular vesicles</a></li>
<li class="chapter" data-level="2.10" data-path="envelope.html"><a href="envelope.html#morecytoplasmic-vesicles"><i class="fa fa-check"></i><b>2.10</b> More–Cytoplasmic vesicles</a></li>
<li class="chapter" data-level="2.11" data-path="envelope.html"><a href="envelope.html#more-cytoplasmic-vesicle-variety"><i class="fa fa-check"></i><b>2.11</b> More: Cytoplasmic vesicle variety</a></li>
<li class="chapter" data-level="2.12" data-path="envelope.html"><a href="envelope.html#morearchaeal-vesicles"><i class="fa fa-check"></i><b>2.12</b> More–Archaeal vesicles</a></li>
<li class="chapter" data-level="2.13" data-path="envelope.html"><a href="envelope.html#variation"><i class="fa fa-check"></i><b>2.13</b> Variation</a></li>
<li class="chapter" data-level="2.14" data-path="envelope.html"><a href="envelope.html#surface-layer"><i class="fa fa-check"></i><b>2.14</b> Surface Layer</a><ul>
<li class="chapter" data-level="2.14.1" data-path="envelope.html"><a href="envelope.html#PDB:_5N97_Surface_layer_architecture"><i class="fa fa-check"></i><b>2.14.1</b> PDB: 5N97 Surface layer architecture</a></li>
</ul></li>
<li class="chapter" data-level="2.15" data-path="envelope.html"><a href="envelope.html#morenanopods"><i class="fa fa-check"></i><b>2.15</b> More–Nanopods</a></li>
<li class="chapter" data-level="2.16" data-path="envelope.html"><a href="envelope.html#surface-layer-variety"><i class="fa fa-check"></i><b>2.16</b> Surface Layer Variety</a></li>
<li class="chapter" data-level="2.17" data-path="envelope.html"><a href="envelope.html#morem.-alcaliphilum"><i class="fa fa-check"></i><b>2.17</b> More–M. alcaliphilum</a></li>
<li class="chapter" data-level="2.18" data-path="envelope.html"><a href="envelope.html#moreclostridium-thermocellum"><i class="fa fa-check"></i><b>2.18</b> More–Clostridium thermocellum</a></li>
<li class="chapter" data-level="2.19" data-path="envelope.html"><a href="envelope.html#moren.-maritimus"><i class="fa fa-check"></i><b>2.19</b> More–N. maritimus</a></li>
<li class="chapter" data-level="2.20" data-path="envelope.html"><a href="envelope.html#moremethanoregula-formicica"><i class="fa fa-check"></i><b>2.20</b> More–Methanoregula formicica</a></li>
<li class="chapter" data-level="2.21" data-path="envelope.html"><a href="envelope.html#sheath"><i class="fa fa-check"></i><b>2.21</b> Sheath</a></li>
<li class="chapter" data-level="2.22" data-path="envelope.html"><a href="envelope.html#dna"><i class="fa fa-check"></i><b>2.22</b> DNA</a></li>
<li class="chapter" data-level="2.23" data-path="envelope.html"><a href="envelope.html#nucleoid"><i class="fa fa-check"></i><b>2.23</b> Nucleoid</a></li>
<li class="chapter" data-level="2.24" data-path="envelope.html"><a href="envelope.html#further-reading-1"><i class="fa fa-check"></i><b>2.24</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="shape.html"><a href="shape.html"><i class="fa fa-check"></i><b>3</b> Shape</a><ul>
<li class="chapter" data-level="3.1" data-path="shape.html"><a href="shape.html#coccoid"><i class="fa fa-check"></i><b>3.1</b> Coccoid</a></li>
<li class="chapter" data-level="3.2" data-path="shape.html"><a href="shape.html#rod"><i class="fa fa-check"></i><b>3.2</b> Rod</a></li>
<li class="chapter" data-level="3.3" data-path="shape.html"><a href="shape.html#morerod-variety"><i class="fa fa-check"></i><b>3.3</b> More–Rod variety</a></li>
<li class="chapter" data-level="3.4" data-path="shape.html"><a href="shape.html#morearchaeal-rods"><i class="fa fa-check"></i><b>3.4</b> More–Archaeal rods</a></li>
<li class="chapter" data-level="3.5" data-path="shape.html"><a href="shape.html#length"><i class="fa fa-check"></i><b>3.5</b> Length</a></li>
<li class="chapter" data-level="3.6" data-path="shape.html"><a href="shape.html#vibrioid"><i class="fa fa-check"></i><b>3.6</b> Vibrioid</a><ul>
<li class="chapter" data-level="3.6.1" data-path="shape.html"><a href="shape.html#PDB:_5U3C_CTP_synthase"><i class="fa fa-check"></i><b>3.6.1</b> PDB: 5U3C CTP synthase</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="shape.html"><a href="shape.html#helical"><i class="fa fa-check"></i><b>3.7</b> Helical</a></li>
<li class="chapter" data-level="3.8" data-path="shape.html"><a href="shape.html#moreborrelia-burgdorferi"><i class="fa fa-check"></i><b>3.8</b> More–Borrelia burgdorferi</a></li>
<li class="chapter" data-level="3.9" data-path="shape.html"><a href="shape.html#prosthecate"><i class="fa fa-check"></i><b>3.9</b> Prosthecate</a><ul>
<li class="chapter" data-level="3.9.1" data-path="shape.html"><a href="shape.html#PDB:_6RIB_Bactofilin"><i class="fa fa-check"></i><b>3.9.1</b> PDB: 6RIB Bactofilin</a></li>
</ul></li>
<li class="chapter" data-level="3.10" data-path="shape.html"><a href="shape.html#morebacterial-microtubules"><i class="fa fa-check"></i><b>3.10</b> More–Bacterial microtubules</a></li>
<li class="chapter" data-level="3.11" data-path="shape.html"><a href="shape.html#square"><i class="fa fa-check"></i><b>3.11</b> Square</a></li>
<li class="chapter" data-level="3.12" data-path="shape.html"><a href="shape.html#moregas-vesicles"><i class="fa fa-check"></i><b>3.12</b> More–Gas vesicles</a></li>
<li class="chapter" data-level="3.13" data-path="shape.html"><a href="shape.html#further-reading-2"><i class="fa fa-check"></i><b>3.13</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="growth.html"><a href="growth.html"><i class="fa fa-check"></i><b>4</b> Growth</a><ul>
<li class="chapter" data-level="4.1" data-path="growth.html"><a href="growth.html#diffusion"><i class="fa fa-check"></i><b>4.1</b> Diffusion</a></li>
<li class="chapter" data-level="4.2" data-path="growth.html"><a href="growth.html#anaerobic-respiration"><i class="fa fa-check"></i><b>4.2</b> Anaerobic Respiration</a></li>
<li class="chapter" data-level="4.3" data-path="growth.html"><a href="growth.html#photosynthesis"><i class="fa fa-check"></i><b>4.3</b> Photosynthesis</a><ul>
<li class="chapter" data-level="4.3.1" data-path="growth.html"><a href="growth.html#pdb-6kgx-photocomplexes-pdb6kgxphotocomplexes"><i class="fa fa-check"></i><b>4.3.1</b> PDB: 6KGX –Photocomplexes {#PDB:<em>6KGX</em>–Photocomplexes}</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="growth.html"><a href="growth.html#moreicm-variety"><i class="fa fa-check"></i><b>4.4</b> More–ICM variety</a></li>
<li class="chapter" data-level="4.5" data-path="growth.html"><a href="growth.html#photosynthesis-contd."><i class="fa fa-check"></i><b>4.5</b> Photosynthesis (cont’d.)</a></li>
<li class="chapter" data-level="4.6" data-path="growth.html"><a href="growth.html#enzyme-regulation"><i class="fa fa-check"></i><b>4.6</b> Enzyme Regulation</a><ul>
<li class="chapter" data-level="4.6.1" data-path="growth.html"><a href="growth.html#pdb-6tqhadhe-spirosome-structure-pdb_6tqhadhe_spirosome_structure"><i class="fa fa-check"></i><b>4.6.1</b> PDB: 6TQH–AdhE spirosome structure {#PDB:_6TQH–AdhE_spirosome_structure} <img src="img/schematics/4_5_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="growth.html"><a href="growth.html#microcompartments"><i class="fa fa-check"></i><b>4.7</b> Microcompartments</a><ul>
<li class="chapter" data-level="4.7.1" data-path="growth.html"><a href="growth.html#pdb-3dktmicrocompartment-shells-pdb_3dktmicrocompartment_shells"><i class="fa fa-check"></i><b>4.7.1</b> PDB: 3DKT–Microcompartment shells {#PDB:_3DKT–Microcompartment_shells} <img src="img/schematics/4_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="growth.html"><a href="growth.html#carboxysomes"><i class="fa fa-check"></i><b>4.8</b> Carboxysomes</a></li>
<li class="chapter" data-level="4.9" data-path="growth.html"><a href="growth.html#morecarboxysome-variety"><i class="fa fa-check"></i><b>4.9</b> More–Carboxysome variety</a></li>
<li class="chapter" data-level="4.10" data-path="growth.html"><a href="growth.html#morelong-carboxysomes"><i class="fa fa-check"></i><b>4.10</b> More–Long carboxysomes</a></li>
<li class="chapter" data-level="4.11" data-path="growth.html"><a href="growth.html#storage"><i class="fa fa-check"></i><b>4.11</b> Storage</a></li>
<li class="chapter" data-level="4.12" data-path="growth.html"><a href="growth.html#morestorage-granule-growth"><i class="fa fa-check"></i><b>4.12</b> More–Storage granule growth</a></li>
<li class="chapter" data-level="4.13" data-path="growth.html"><a href="growth.html#storage-contd."><i class="fa fa-check"></i><b>4.13</b> Storage (cont’d.)</a></li>
<li class="chapter" data-level="4.14" data-path="growth.html"><a href="growth.html#moregranule-containing-carboxysomes"><i class="fa fa-check"></i><b>4.14</b> More–Granule-containing carboxysomes</a></li>
<li class="chapter" data-level="4.15" data-path="growth.html"><a href="growth.html#morecarboxysomes-granule-connections"><i class="fa fa-check"></i><b>4.15</b> More–Carboxysomes-granule connections</a></li>
<li class="chapter" data-level="4.16" data-path="growth.html"><a href="growth.html#archaeal-storage-granules"><i class="fa fa-check"></i><b>4.16</b> Archaeal Storage Granules</a></li>
<li class="chapter" data-level="4.17" data-path="growth.html"><a href="growth.html#morehaloferax-gibbonsii-granules"><i class="fa fa-check"></i><b>4.17</b> More–Haloferax gibbonsii granules</a></li>
<li class="chapter" data-level="4.18" data-path="growth.html"><a href="growth.html#morehalohasta-litchfieldiae-granules"><i class="fa fa-check"></i><b>4.18</b> More–Halohasta litchfieldiae granules</a></li>
<li class="chapter" data-level="4.19" data-path="growth.html"><a href="growth.html#further-reading-3"><i class="fa fa-check"></i><b>4.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="division.html"><a href="division.html"><i class="fa fa-check"></i><b>5</b> Division</a><ul>
<li class="chapter" data-level="5.1" data-path="division.html"><a href="division.html#segregation"><i class="fa fa-check"></i><b>5.1</b> Segregation</a></li>
<li class="chapter" data-level="5.2" data-path="division.html"><a href="division.html#chromosome-segregation"><i class="fa fa-check"></i><b>5.2</b> Chromosome Segregation</a></li>
<li class="chapter" data-level="5.3" data-path="division.html"><a href="division.html#budding"><i class="fa fa-check"></i><b>5.3</b> Budding</a></li>
<li class="chapter" data-level="5.4" data-path="division.html"><a href="division.html#morehyphomonas-stalk"><i class="fa fa-check"></i><b>5.4</b> More–Hyphomonas stalk</a></li>
<li class="chapter" data-level="5.5" data-path="division.html"><a href="division.html#morehyphomonas-lifecycle"><i class="fa fa-check"></i><b>5.5</b> More–Hyphomonas lifecycle</a></li>
<li class="chapter" data-level="5.6" data-path="division.html"><a href="division.html#monoderm-cytokinesis"><i class="fa fa-check"></i><b>5.6</b> Monoderm Cytokinesis</a></li>
<li class="chapter" data-level="5.7" data-path="division.html"><a href="division.html#monoderm-cytokinesis-contd."><i class="fa fa-check"></i><b>5.7</b> Monoderm Cytokinesis (cont’d.)</a></li>
<li class="chapter" data-level="5.8" data-path="division.html"><a href="division.html#diderm-cytokinesis"><i class="fa fa-check"></i><b>5.8</b> Diderm Cytokinesis</a></li>
<li class="chapter" data-level="5.9" data-path="division.html"><a href="division.html#moresphingopyxis-alaskensis-division"><i class="fa fa-check"></i><b>5.9</b> More–Sphingopyxis alaskensis division</a></li>
<li class="chapter" data-level="5.10" data-path="division.html"><a href="division.html#morechloroflexi-division"><i class="fa fa-check"></i><b>5.10</b> More–Chloroflexi division</a></li>
<li class="chapter" data-level="5.11" data-path="division.html"><a href="division.html#diderm-cytokinesis-contd."><i class="fa fa-check"></i><b>5.11</b> Diderm Cytokinesis (cont’d.)</a></li>
<li class="chapter" data-level="5.12" data-path="division.html"><a href="division.html#morecaulobacter-division"><i class="fa fa-check"></i><b>5.12</b> More–Caulobacter division</a></li>
<li class="chapter" data-level="5.13" data-path="division.html"><a href="division.html#budding-contd."><i class="fa fa-check"></i><b>5.13</b> Budding (cont’d.)</a></li>
<li class="chapter" data-level="5.14" data-path="division.html"><a href="division.html#ftsz"><i class="fa fa-check"></i><b>5.14</b> FtsZ</a></li>
<li class="chapter" data-level="5.15" data-path="division.html"><a href="division.html#moreasymmetric-constriction"><i class="fa fa-check"></i><b>5.15</b> More–Asymmetric constriction</a><ul>
<li class="chapter" data-level="5.15.1" data-path="division.html"><a href="division.html#pdb-5mn4-ftsz-structure-pdb5mn4ftsz_structure"><i class="fa fa-check"></i><b>5.15.1</b> PDB: 5MN4 –FtsZ structure {#PDB:<em>5MN4</em>–FtsZ_structure}</a></li>
</ul></li>
<li class="chapter" data-level="5.16" data-path="division.html"><a href="division.html#ftsz-contd."><i class="fa fa-check"></i><b>5.16</b> FtsZ (cont’d.)</a></li>
<li class="chapter" data-level="5.17" data-path="division.html"><a href="division.html#escrt"><i class="fa fa-check"></i><b>5.17</b> ESCRT</a></li>
<li class="chapter" data-level="5.18" data-path="division.html"><a href="division.html#escrt-contd."><i class="fa fa-check"></i><b>5.18</b> ESCRT (cont’d.)</a></li>
<li class="chapter" data-level="5.19" data-path="division.html"><a href="division.html#further-reading-4"><i class="fa fa-check"></i><b>5.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="motility.html"><a href="motility.html"><i class="fa fa-check"></i><b>6</b> Motility</a><ul>
<li class="chapter" data-level="6.1" data-path="motility.html"><a href="motility.html#flagellum"><i class="fa fa-check"></i><b>6.1</b> Flagellum</a><ul>
<li class="chapter" data-level="6.1.1" data-path="motility.html"><a href="motility.html#pdb-5wjtflagellum-structure-pdb_5wjtflagellum_structure"><i class="fa fa-check"></i><b>6.1.1</b> PDB: 5WJT–Flagellum structure {#PDB:_5WJT–Flagellum_structure} <img src="img/schematics/6_1_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="motility.html"><a href="motility.html#flagellar-motor"><i class="fa fa-check"></i><b>6.2</b> Flagellar Motor</a></li>
<li class="chapter" data-level="6.3" data-path="motility.html"><a href="motility.html#moreflagellar-motor-structure"><i class="fa fa-check"></i><b>6.3</b> More–Flagellar motor structure</a><ul>
<li class="chapter" data-level="6.3.1" data-path="motility.html"><a href="motility.html#Flagellar_motor_diversity"><i class="fa fa-check"></i><b>6.3.1</b> Flagellar motor diversity</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="motility.html"><a href="motility.html#flagellar-assembly"><i class="fa fa-check"></i><b>6.4</b> Flagellar Assembly</a></li>
<li class="chapter" data-level="6.5" data-path="motility.html"><a href="motility.html#moreflagellar-motor-disassembly"><i class="fa fa-check"></i><b>6.5</b> More–Flagellar motor disassembly</a></li>
<li class="chapter" data-level="6.6" data-path="motility.html"><a href="motility.html#flagella-patterns"><i class="fa fa-check"></i><b>6.6</b> Flagella Patterns</a></li>
<li class="chapter" data-level="6.7" data-path="motility.html"><a href="motility.html#morelophotrichous-bacteria"><i class="fa fa-check"></i><b>6.7</b> More–Lophotrichous bacteria</a></li>
<li class="chapter" data-level="6.8" data-path="motility.html"><a href="motility.html#flagella-patterns-contd."><i class="fa fa-check"></i><b>6.8</b> Flagella Patterns (cont’d.)</a></li>
<li class="chapter" data-level="6.9" data-path="motility.html"><a href="motility.html#moreproteus-mirabilis-flagella"><i class="fa fa-check"></i><b>6.9</b> More–Proteus mirabilis flagella</a></li>
<li class="chapter" data-level="6.10" data-path="motility.html"><a href="motility.html#flagellar-sheath"><i class="fa fa-check"></i><b>6.10</b> Flagellar Sheath</a></li>
<li class="chapter" data-level="6.11" data-path="motility.html"><a href="motility.html#periplasmic-flagella"><i class="fa fa-check"></i><b>6.11</b> Periplasmic Flagella</a></li>
<li class="chapter" data-level="6.12" data-path="motility.html"><a href="motility.html#moretreponema-primitia"><i class="fa fa-check"></i><b>6.12</b> More–Treponema primitia</a></li>
<li class="chapter" data-level="6.13" data-path="motility.html"><a href="motility.html#archaellum"><i class="fa fa-check"></i><b>6.13</b> Archaellum</a><ul>
<li class="chapter" data-level="6.13.1" data-path="motility.html"><a href="motility.html#pdb-5tfyarchaellum-structure-pdb_5tfyarchaellum_structure"><i class="fa fa-check"></i><b>6.13.1</b> PDB: 5TFY–Archaellum structure {#PDB:_5TFY–Archaellum_structure} <img src="img/schematics/6_8_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="6.14" data-path="motility.html"><a href="motility.html#archaella-patterns"><i class="fa fa-check"></i><b>6.14</b> Archaella Patterns</a></li>
<li class="chapter" data-level="6.15" data-path="motility.html"><a href="motility.html#morecone-structure"><i class="fa fa-check"></i><b>6.15</b> More–Cone structure</a></li>
<li class="chapter" data-level="6.16" data-path="motility.html"><a href="motility.html#moreorganizing-center"><i class="fa fa-check"></i><b>6.16</b> More–Organizing center</a></li>
<li class="chapter" data-level="6.17" data-path="motility.html"><a href="motility.html#type-iv-pilus"><i class="fa fa-check"></i><b>6.17</b> Type IV Pilus</a><ul>
<li class="chapter" data-level="6.17.1" data-path="motility.html"><a href="motility.html#Type_IV_pilus_structure"><i class="fa fa-check"></i><b>6.17.1</b> Type IV pilus structure</a></li>
</ul></li>
<li class="chapter" data-level="6.18" data-path="motility.html"><a href="motility.html#type-ix-secretion-system"><i class="fa fa-check"></i><b>6.18</b> Type IX Secretion System</a></li>
<li class="chapter" data-level="6.19" data-path="motility.html"><a href="motility.html#terminal-organelle"><i class="fa fa-check"></i><b>6.19</b> Terminal Organelle</a></li>
<li class="chapter" data-level="6.20" data-path="motility.html"><a href="motility.html#further-reading-5"><i class="fa fa-check"></i><b>6.20</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="navigation.html"><a href="navigation.html"><i class="fa fa-check"></i><b>7</b> Navigation</a><ul>
<li class="chapter" data-level="7.1" data-path="navigation.html"><a href="navigation.html#chemotaxis"><i class="fa fa-check"></i><b>7.1</b> Chemotaxis</a><ul>
<li class="chapter" data-level="7.1.1" data-path="navigation.html"><a href="navigation.html#pdb-2ch7chemoreceptor-structure-pdb_2ch7chemoreceptor_structure"><i class="fa fa-check"></i><b>7.1.1</b> PDB: 2CH7–Chemoreceptor structure {#PDB:_2CH7–Chemoreceptor_structure} <img src="img/schematics/7_1_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="navigation.html"><a href="navigation.html#chemosensory-array"><i class="fa fa-check"></i><b>7.2</b> Chemosensory Array</a><ul>
<li class="chapter" data-level="7.2.1" data-path="navigation.html"><a href="navigation.html#PDB:_6S1K_Chemosensory_array_architecture"><i class="fa fa-check"></i><b>7.2.1</b> PDB: 6S1K Chemosensory array architecture</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="navigation.html"><a href="navigation.html#chemosensory-array-conservation"><i class="fa fa-check"></i><b>7.3</b> Chemosensory Array Conservation</a></li>
<li class="chapter" data-level="7.4" data-path="navigation.html"><a href="navigation.html#chemoreceptors"><i class="fa fa-check"></i><b>7.4</b> Chemoreceptors</a></li>
<li class="chapter" data-level="7.5" data-path="navigation.html"><a href="navigation.html#moreazospirillum-brasilense-aerotaxis"><i class="fa fa-check"></i><b>7.5</b> More–Azospirillum brasilense aerotaxis</a></li>
<li class="chapter" data-level="7.6" data-path="navigation.html"><a href="navigation.html#cytoplasmic-chemosensory-array"><i class="fa fa-check"></i><b>7.6</b> Cytoplasmic Chemosensory Array</a></li>
<li class="chapter" data-level="7.7" data-path="navigation.html"><a href="navigation.html#morevibrio-cholerae-chemosensory-arrays"><i class="fa fa-check"></i><b>7.7</b> More–Vibrio cholerae chemosensory arrays</a></li>
<li class="chapter" data-level="7.8" data-path="navigation.html"><a href="navigation.html#magnetotaxis"><i class="fa fa-check"></i><b>7.8</b> Magnetotaxis</a><ul>
<li class="chapter" data-level="7.8.1" data-path="navigation.html"><a href="navigation.html#pdb-5jygmamk-structure-pdb_5jygmamk_structure"><i class="fa fa-check"></i><b>7.8.1</b> PDB: 5JYG–MamK structure {#PDB:_5JYG–MamK_structure} <img src="img/schematics/7_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="navigation.html"><a href="navigation.html#further-reading-6"><i class="fa fa-check"></i><b>7.9</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="lifecycle.html"><a href="lifecycle.html"><i class="fa fa-check"></i><b>8</b> Lifecycle</a><ul>
<li class="chapter" data-level="8.1" data-path="lifecycle.html"><a href="lifecycle.html#stationary-phase"><i class="fa fa-check"></i><b>8.1</b> Stationary Phase</a></li>
<li class="chapter" data-level="8.2" data-path="lifecycle.html"><a href="lifecycle.html#genome-protection"><i class="fa fa-check"></i><b>8.2</b> Genome Protection</a></li>
<li class="chapter" data-level="8.3" data-path="lifecycle.html"><a href="lifecycle.html#stationary-phase-contd."><i class="fa fa-check"></i><b>8.3</b> Stationary Phase (cont’d.)</a></li>
<li class="chapter" data-level="8.4" data-path="lifecycle.html"><a href="lifecycle.html#differentiation"><i class="fa fa-check"></i><b>8.4</b> Differentiation</a></li>
<li class="chapter" data-level="8.5" data-path="lifecycle.html"><a href="lifecycle.html#moreholdfast"><i class="fa fa-check"></i><b>8.5</b> More–Holdfast</a></li>
<li class="chapter" data-level="8.6" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-sporulation"><i class="fa fa-check"></i><b>8.6</b> Monoderm Sporulation</a></li>
<li class="chapter" data-level="8.7" data-path="lifecycle.html"><a href="lifecycle.html#moresporulation-septum-formation"><i class="fa fa-check"></i><b>8.7</b> More–Sporulation septum formation</a></li>
<li class="chapter" data-level="8.8" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-sporulation-contd."><i class="fa fa-check"></i><b>8.8</b> Monoderm Sporulation (cont’d.)</a></li>
<li class="chapter" data-level="8.9" data-path="lifecycle.html"><a href="lifecycle.html#morespore-engulfment"><i class="fa fa-check"></i><b>8.9</b> More–Spore engulfment</a></li>
<li class="chapter" data-level="8.10" data-path="lifecycle.html"><a href="lifecycle.html#morespore-coating"><i class="fa fa-check"></i><b>8.10</b> More–Spore coating</a></li>
<li class="chapter" data-level="8.11" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-spores"><i class="fa fa-check"></i><b>8.11</b> Monoderm Spores</a></li>
<li class="chapter" data-level="8.12" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-germination"><i class="fa fa-check"></i><b>8.12</b> Monoderm Germination</a></li>
<li class="chapter" data-level="8.13" data-path="lifecycle.html"><a href="lifecycle.html#diderm-sporulation"><i class="fa fa-check"></i><b>8.13</b> Diderm Sporulation</a></li>
<li class="chapter" data-level="8.14" data-path="lifecycle.html"><a href="lifecycle.html#diderm-sporulation-contd."><i class="fa fa-check"></i><b>8.14</b> Diderm Sporulation (cont’d.)</a></li>
<li class="chapter" data-level="8.15" data-path="lifecycle.html"><a href="lifecycle.html#morecomplete-engulfment"><i class="fa fa-check"></i><b>8.15</b> More–Complete engulfment</a></li>
<li class="chapter" data-level="8.16" data-path="lifecycle.html"><a href="lifecycle.html#moremature-spore"><i class="fa fa-check"></i><b>8.16</b> More–Mature spore</a></li>
<li class="chapter" data-level="8.17" data-path="lifecycle.html"><a href="lifecycle.html#moresporomusa-acidovorans-sporulation"><i class="fa fa-check"></i><b>8.17</b> More–Sporomusa acidovorans sporulation</a></li>
<li class="chapter" data-level="8.18" data-path="lifecycle.html"><a href="lifecycle.html#diderm-germination"><i class="fa fa-check"></i><b>8.18</b> Diderm Germination</a></li>
<li class="chapter" data-level="8.19" data-path="lifecycle.html"><a href="lifecycle.html#further-reading-7"><i class="fa fa-check"></i><b>8.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="interaction.html"><a href="interaction.html"><i class="fa fa-check"></i><b>9</b> Interaction</a><ul>
<li class="chapter" data-level="9.1" data-path="interaction.html"><a href="interaction.html#biofilm"><i class="fa fa-check"></i><b>9.1</b> Biofilm</a></li>
<li class="chapter" data-level="9.2" data-path="interaction.html"><a href="interaction.html#biofilm-contd."><i class="fa fa-check"></i><b>9.2</b> Biofilm (cont’d.)</a></li>
<li class="chapter" data-level="9.3" data-path="interaction.html"><a href="interaction.html#type-ii-and-type-iv-secretion"><i class="fa fa-check"></i><b>9.3</b> Type II and Type IV Secretion</a><ul>
<li class="chapter" data-level="9.3.1" data-path="interaction.html"><a href="interaction.html#emd-20712-emd-20713legionella-t2ss-structure-emd-20712_emd-20713legionella_t2ss_structure"><i class="fa fa-check"></i><b>9.3.1</b> EMD-20712 EMD-20713–Legionella T2SS structure {#EMD-20712_EMD-20713–Legionella_T2SS_structure}</a></li>
<li class="chapter" data-level="9.3.2" data-path="interaction.html"><a href="interaction.html#emd-0566-legionella-t4ss-structure-emd-0566_legionella_t4ss_structure"><i class="fa fa-check"></i><b>9.3.2</b> EMD-0566 –Legionella T4SS structure {#EMD-0566_–Legionella_T4SS_structure}</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="interaction.html"><a href="interaction.html#morehelicobacter-pylori-tubes"><i class="fa fa-check"></i><b>9.4</b> More–Helicobacter pylori tubes</a></li>
<li class="chapter" data-level="9.5" data-path="interaction.html"><a href="interaction.html#type-iii-secretion"><i class="fa fa-check"></i><b>9.5</b> Type III Secretion</a><ul>
<li class="chapter" data-level="9.5.1" data-path="interaction.html"><a href="interaction.html#EMD-20611_Injectisome_structure"><i class="fa fa-check"></i><b>9.5.1</b> EMD-20611 Injectisome structure</a></li>
<li class="chapter" data-level="9.5.2" data-path="interaction.html"><a href="interaction.html#PDB:_5WQ8_Secretin_structure"><i class="fa fa-check"></i><b>9.5.2</b> PDB: 5WQ8 Secretin structure</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="interaction.html"><a href="interaction.html#type-v-secretion"><i class="fa fa-check"></i><b>9.6</b> Type V Secretion</a></li>
<li class="chapter" data-level="9.7" data-path="interaction.html"><a href="interaction.html#type-vi-secretion"><i class="fa fa-check"></i><b>9.7</b> Type VI Secretion</a><ul>
<li class="chapter" data-level="9.7.1" data-path="interaction.html"><a href="interaction.html#pdb-5urwprimed-t6ss-structure-pdb_5urwprimed_t6ss_structure"><i class="fa fa-check"></i><b>9.7.1</b> PDB: 5URW–Primed T6SS structure {#PDB:_5URW–Primed_T6SS_structure} <img src="img/schematics/9_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="9.8" data-path="interaction.html"><a href="interaction.html#type-vi-secretion-contd."><i class="fa fa-check"></i><b>9.8</b> Type VI Secretion (cont’d.)</a><ul>
<li class="chapter" data-level="9.8.1" data-path="interaction.html"><a href="interaction.html#pdb-5urxfired-t6ss-structure-pdb_5urxfired_t6ss_structure"><i class="fa fa-check"></i><b>9.8.1</b> PDB: 5URX–Fired T6SS structure {#PDB:_5URX–Fired_T6SS_structure} <img src="img/schematics/9_7_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="9.9" data-path="interaction.html"><a href="interaction.html#moret6ss-battery"><i class="fa fa-check"></i><b>9.9</b> More–T6SS battery</a></li>
<li class="chapter" data-level="9.10" data-path="interaction.html"><a href="interaction.html#moremacs"><i class="fa fa-check"></i><b>9.10</b> More–MACs</a><ul>
<li class="chapter" data-level="9.10.1" data-path="interaction.html"><a href="interaction.html#MAC_array_structure"><i class="fa fa-check"></i><b>9.10.1</b> MAC array structure</a></li>
</ul></li>
<li class="chapter" data-level="9.11" data-path="interaction.html"><a href="interaction.html#type-ix-secretion"><i class="fa fa-check"></i><b>9.11</b> Type IX Secretion</a></li>
<li class="chapter" data-level="9.12" data-path="interaction.html"><a href="interaction.html#predation"><i class="fa fa-check"></i><b>9.12</b> Predation</a></li>
<li class="chapter" data-level="9.13" data-path="interaction.html"><a href="interaction.html#predation-contd."><i class="fa fa-check"></i><b>9.13</b> Predation (cont’d.)</a></li>
<li class="chapter" data-level="9.14" data-path="interaction.html"><a href="interaction.html#morebdellovibrio-bacteriovorus-entry"><i class="fa fa-check"></i><b>9.14</b> More–Bdellovibrio bacteriovorus entry</a></li>
<li class="chapter" data-level="9.15" data-path="interaction.html"><a href="interaction.html#predation-contd.-1"><i class="fa fa-check"></i><b>9.15</b> Predation (cont’d.)</a></li>
<li class="chapter" data-level="9.16" data-path="interaction.html"><a href="interaction.html#further-reading-8"><i class="fa fa-check"></i><b>9.16</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="viruses.html"><a href="viruses.html"><i class="fa fa-check"></i><b>10</b> Viruses</a><ul>
<li class="chapter" data-level="10.1" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis"><i class="fa fa-check"></i><b>10.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.2" data-path="viruses.html"><a href="viruses.html#bacteroidetes-strain-jt5"><i class="fa fa-check"></i><b>10.2</b> Bacteroidetes strain JT5</a></li>
<li class="chapter" data-level="10.3" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-1"><i class="fa fa-check"></i><b>10.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.4" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae"><i class="fa fa-check"></i><b>10.4</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="10.5" data-path="viruses.html"><a href="viruses.html#escherichia-coli"><i class="fa fa-check"></i><b>10.5</b> Escherichia coli</a></li>
<li class="chapter" data-level="10.6" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus"><i class="fa fa-check"></i><b>10.6</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="10.7" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.7</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="10.8" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus"><i class="fa fa-check"></i><b>10.8</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="10.9" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-1"><i class="fa fa-check"></i><b>10.9</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="10.10" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-2"><i class="fa fa-check"></i><b>10.10</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.11" data-path="viruses.html"><a href="viruses.html#further-reading-9"><i class="fa fa-check"></i><b>10.11</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>11</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.library.caltech.edu" target="blank">Published by Caltech Library</a><li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cell Atlas</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="envelope" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Envelope</h1>
<p>“It is not a simple life to be a single cell, although I have no right to say so, having been a single cell so long ago myself that I have no memory at all of that stage of my life.” - Lewis Thomas <span class="citation">Thomas (<a href="#ref-thomas1990">1990</a>)</span></p>
<div id="membrane" class="section level2">
<h2><span class="header-section-number">2.1</span> Membrane</h2>
<p>The fundamental unit of life is the cell– a contained self-replicating assembly. For many species, including all bacteria and archaea, the organism consists of a single cell. And for nearly all species, no matter how many cells an organism eventually contains (probably around 10 trillion in your case), it started life as a single cell. The details vary, but every cell on Earth is the same at heart–a DNA-based replicating machine built from just four macromolecules: nucleic acids, proteins, lipids and carbohydrates.</p>
<p>Imagine for a moment that you are a structural engineer tasked with building one of these cells. Where would you start? How about the container? No matter what the first self-replicating molecules were (likely _r_ibo_n_ucleic _a_cid, or RNA), they were not a cell until they acquired a container. You would probably want a flexible container that allowed you to sort certain kinds of molecules from the environment. Evolution agrees. All cells are enclosed by a selectively permeable membrane, made of lipids and proteins [Schematic–Lipid bilayer], that allows them to differentiate their contents from the environment. The selectivity of this membrane is a critical feature for the life of the cell [Schematic–ATP synthase].</p>
<p>Now your cell has a clearly delineated exterior and interior. The interior is called the cytoplasm (“cell mold,” the membrane being the container that shapes the mold). Almost all archaea and many bacteria, like these <em>Mycoplasma genitalium</em> cells, are monoderms (“single skin”). This means that their cytoplasm is enclosed by a single membrane. At this resolution, the membrane looks like a single dark line, but remember that it is really a bilayer, as you will be able to see in some later examples.</p>
<div class="figure"><span id="fig:unnamed-chunk-17"></span>
<div id="htmlwidget-50b5218581c90a9ad842" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-50b5218581c90a9ad842">{"x":{"doi":"10.22002/D1.1350","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.1: Mycoplasma genitalium Collected by: Gregory Henderson <a href="https://doi.org/10.22002/D1.1350">10.22002/D1.1350</a>
</p>
</div>
<div id="Lipid_bilayer" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Lipid bilayer</h3>
<p><img src="img/schematics/2_1_1.png" /><!-- --></p>
<p>Lipids have a hydrophilic head (red/orange here) and hydrophobic tails (grey); in water they spontaneously pack side-by-side and tail-to-tail to shield their tails from unfavorable interactions with water. This results in closed double-layered bags–membranes. One key difference between archaea and bacteria (and with them, eukaryotes) is the kind of lipid in their membranes. Hybrid membranes containing both these lipid types can be made artificially, and it is possible that the last universal common ancestor of all cells on Earth contained both types, with specialization occurring later.</p>
<p>Membranes also contain many proteins. Some have hydrophobic regions that embed them into the lipid bilayer. Other proteins are fused, or tethered, to the lipids. In fact, cells’ “lipid” membranes are made up of roughly equal parts lipids and proteins.</p>
</div>
<div id="PDB:_5T4O_ATP_synthase_and_energy_production" class="section level3">
<h3><span class="header-section-number">2.1.2</span> PDB: 5T4O ATP synthase and energy production</h3>
<p><img src="img/schematics/2_1_2.gif" /><!-- --></p>
<p>The chemical properties of lipids make membranes impermeable to ions and large or hydrophilic molecules (but not to water). Cells take advantage of this property to establish an ion gradient across the membrane, using a chain of electron-carrying proteins in the membrane to pump protons out of the cell. Protein complexes in the membrane called ATP synthases (like this one from <em>Escherichia coli</em> <span class="citation">Sobti et al. (<a href="#ref-sobti2016">2016</a>)</span>) use the resulting ion potential to generate energy. The machine provides a conduit for protons to flow down their potential, producing a “proton-motive force” that spins the machine’s rotor, generating energy that is chemically stored in ATP, the energetic currency of the cell. For this reason, we say that the membrane is “energized.” Holes in the membrane allow the ion gradient to equilibrate, destroying the cell’s means of generating energy, and thus its life.</p>
</div>
</div>
<div id="cell-wall" class="section level2">
<h2><span class="header-section-number">2.2</span> Cell Wall</h2>
<p>Being able to selectively move things into your cell enables it to do some powerful things. It also poses a structural problem. Remember that water can pass freely through the membrane, which means that increasing the solute concentration inside relative to the environment outside will cause water to rush in as well, introducing a pressure (known as turgor pressure) on the membrane. Lipid bilayers, unfortunately, are unable to withstand much pressure. If your cell lives exclusively in a constant, and fairly high-osmolarity, environment (like our bodies, in the case of the pathogenic <em>Mycoplasma genitalium</em> you just saw), it can balance internal and external osmolarity to minimize turgor pressure on its membrane. But most cells experience more variable, and dilute, environments. How could you keep your cell from bursting in such conditions?</p>
<p>You might want to add some rigid scaffolding outside the membrane to buttress it against turgor pressure. Nearly all bacteria do this with a material called peptidoglycan: long stiff polymers of glycan sugars crosslinked by short peptides into a chain-mail-like mesh. The full scaffold of this material surrounding the cell is called its sacculus, or cell wall. In monoderm bacteria like this <em>Listeria monocytogenes</em>, the cell wall is significantly thicker than the membrane. It comprises several layers of peptidoglycan, which are indistinguishable at this resolution, so the cell wall appears as a uniformly textured layer <a href="envelope.html#morepeptidoglycan-architecture">More–Peptidoglycan architecture</a>. It remains a mystery how large molecules can pass through this dense layer on their way to and from the cell.</p>
<p>While most bacteria build their walls from peptidoglycan, some use slightly different materials <a href="envelope.html#moremethanobacterium-formicicum">More–Methanobacterium formicicum</a>. Some archaea also have cell walls, also made of a molecule similar to peptidoglycan, but chemically distinct. Most archaea, though, rely on a different structure for support, which you will see in a few pages.</p>
<p>A wall can be a boon to your cell, but it can also prove a liability in certain conditions. For instance, many antibiotics target the cell wall. Some bacteria have developed the ability to jettison their walls in such conditions, taking on a so-called “L-form” (named for the Lister Institute where it was discovered). L-form bacteria give us a fascinating window into how early cells–prior to the evolution of the cell wall–might have looked and behaved.</p>
<div class="figure"><span id="fig:unnamed-chunk-20"></span>
<div id="htmlwidget-f13ac48c5549fbc4f2df" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-f13ac48c5549fbc4f2df">{"x":{"doi":"10.22002/D1.1351","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.2: Listeria monocytogenes Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1351">10.22002/D1.1351</a>
</p>
</div>
</div>
<div id="morepeptidoglycan-architecture" class="section level2">
<h2><span class="header-section-number">2.3</span> More–Peptidoglycan architecture</h2>
<p>The sacculus is so robust that it persists even after cells are lysed and their other components digested. This sacculus isolated from a <em>Bacillus subtilis</em> cell has retained its shape, simply flattening with the release of contents and pressure from inside. The long glycan strands are oriented in hoops running parallel to the short axis of the cylindrical cell and the short peptide crosslinks are more or less aligned with the long axis.</p>
<div class="figure"><span id="fig:unnamed-chunk-21"></span>
<div id="htmlwidget-3955fa155a0d08b0c628" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-3955fa155a0d08b0c628">{"x":{"doi":"10.22002/D1.1360","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.3: Bacillus subtilis Collected by: Morgan Beeby <a href="https://doi.org/10.22002/D1.1360">10.22002/D1.1360</a>
</p>
</div>
</div>
<div id="moremethanobacterium-formicicum" class="section level2">
<h2><span class="header-section-number">2.4</span> More–Methanobacterium formicicum</h2>
<p>These methane-producing bacteria, found in the rumen of animals like cows, build their walls with a polymer that has a slightly different chemical composition than peptidoglycan, but which forms a similar structure, as you can see.</p>
<div class="figure"><span id="fig:unnamed-chunk-22"></span>
<div id="htmlwidget-4d08bb2ff082ab54a89b" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-4d08bb2ff082ab54a89b">{"x":{"doi":"10.22002/D1.1361","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.4: Methanobacterium formicicum Collected by: Elitza Tocheva <a href="https://doi.org/10.22002/D1.1361">10.22002/D1.1361</a>
</p>
</div>
</div>
<div id="outer-membrane" class="section level2">
<h2><span class="header-section-number">2.5</span> Outer Membrane</h2>
<p>Why stop at one membrane? Eukaryotic cells use internal membranes to form specialized compartments like the nucleus and mitochondria. While bacteria lack such organelles, many create an additional compartment <em>outside</em> the cell with a second, outer, membrane. Such bacteria, like the <em>Cupriavidus necator</em> cell you see here, are called diderms (“double skin”). The extra compartment between their membranes is known as the periplasm (“mold between”). This antechamber contains a unique subset of proteins, many of which function in escorting things into and out of the main cell.</p>
<p>Compared to the inner membrane, the outer membrane has some unique properties. It is more permeable and not proton-tight (so it cannot be used to generate ATP). It is often asymmetric, with a different composition of lipids and proteins in each of the two leaflets. A few species of bacteria, particularly pathogens, have labile outer membranes (for an example, check out the <em>Helicobacter pylori</em> in Chapter 8–RecA). Most, however, firmly anchor their outer membrane to the cell wall [Schematic–Braun’s lipoprotein]. Rather than containing many layers of peptidoglycan like in the <em>Listeria monocytogenes</em> cell you just saw, the cell wall of diderms is usually composed of a single layer of peptidoglycan mesh <a href="envelope.html#morediderm-sacculus-architecture">More–Diderm sacculus architecture</a>, which is visible here as a thin line in the periplasm. This minimal sacculus presents a considerable challenge for growth: how can you remodel a single-layered wall that is under pressure? The answer, as we are beginning to figure out, is very carefully [Schematic–Sacculus remodeling].</p>
<p>The difference in thickness of monoderm and diderm cell walls enables a well-known classification system: the Gram stain, which binds peptidoglycan. Gram-positive bacteria, typically monoderm, contain much more peptidoglycan than Gram-negative bacteria, which are typically diderm.</p>
<div class="figure"><span id="fig:unnamed-chunk-23"></span>
<div id="htmlwidget-2ed8c3608178a0e62483" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-2ed8c3608178a0e62483">{"x":{"doi":"10.22002/D1.1352","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.5: Hydrogenovibrio crunogenus Collected by: Cristina Iancu <a href="https://doi.org/10.22002/D1.1352">10.22002/D1.1352</a>
</p>
</div>
<div id="pdb-1eq7-brauns-lipoprotein-pdb1eq7brauns_lipoprotein" class="section level3">
<h3><span class="header-section-number">2.5.1</span> PDB: 1EQ7 –Braun’s lipoprotein {#PDB:<em>1EQ7</em>–Brauns_lipoprotein}</h3>
<p><img src="img/schematics/2_3_1.gif" /><!-- --></p>
<p>Lipoproteins are hybrid molecules, formed by covalently linked lipid and protein pieces. The lipid allows them to embed into a membrane, tethering the attached protein to function nearby. Braun’s lipoprotein, or Lpp <span class="citation">Shu et al. (<a href="#ref-shu2000">2000</a>)</span>, is one of the most abundant molecules in the outer membrane of cells like <em>Escherichia coli</em>. The top of the trimeric protein “coiled-coil” contains the lipid tether. The bottom binds the peptidoglycan cell wall, creating a link between the outer membrane and the cell wall (adding up, for a typical <em>E. coli</em> cell, to about 100,000 links). These links determine the distance between the two layers.</p>
</div>
</div>
<div id="morediderm-sacculus-architecture" class="section level2">
<h2><span class="header-section-number">2.6</span> More–Diderm sacculus architecture</h2>
<p>Compare this diderm sacculus purified from <em>Escherichia coli</em> to the monoderm sacculus on the last page. Since this one is thinner, we can make out more details, like the glycan strands running around the circumference of the cell. The main difference between the two types of sacculi seems to be whether they have largely a single layer of peptidoglycan (diderm) or many layers (monoderm). So even though the two types of wall look different, their architecture is fundamentally the same. In some circumstances, as you will see in Chapter 8, cells can even switch between the two forms.</p>
<div class="figure"><span id="fig:unnamed-chunk-25"></span>
<div id="htmlwidget-bd3b9077a22c6b716a25" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-bd3b9077a22c6b716a25">{"x":{"doi":"10.22002/D1.1362","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.6: Escherichia coli Collected by: Lu Gan <a href="https://doi.org/10.22002/D1.1362">10.22002/D1.1362</a>
</p>
</div>
<div id="Sacculus_remodeling" class="section level3">
<h3><span class="header-section-number">2.6.1</span> Sacculus remodeling</h3>
<p><img src="img/schematics/2_3_2.gif" /><!-- --></p>
<p>Encasing your cell in a rigid scaffold presents a problem: how can it grow? It is easy to make membranes larger simply by adding more lipids. But to add more peptidoglycan strands, they must be linked into the existing network, which means breaking existing links to accommodate them. To do this, cells use three tools: an enzyme that links glycan sugars into strands, an enzyme that links glycan strands together with peptide bonds, and an enzyme that cuts these peptide links to allow new strands to be linked. Remember, though, that your cell, with its solute-rich interior, has a turgor pressure pushing outward with a force of perhaps 3 atmospheres, equivalent to what we would feel at a depth of 20 meters in the ocean. This is more than enough pressure to lyse an exposed membrane, so the tools must be wielded with care or the cell would burst. We are still figuring out how this works, with help from computer simulations like this one. Here you see a model of an <em>Escherichia coli</em> sacculus being enlarged using the three enzyme tools we just described (the colored balls). This simulation was run to test whether just having the tools function in a complex rather than separately might provide enough coordination for smooth, safe growth <span class="citation">Nguyen et al. (<a href="#ref-nguyen2015">2015</a>)</span>. (The answer was yes.)</p>
</div>
</div>
<div id="vesicles" class="section level2">
<h2><span class="header-section-number">2.7</span> Vesicles</h2>
<p>What else can your cell do with an extra membrane? Since membranes make such excellent containers for molecules, why not get into the shipping business? In the coming chapters (especially Chapter 9), you will see some of the ways that cells interact with each other and their environment. For diderm bacteria, many of these interactions are made possible by outer membrane vesicles (“little bladders”)–self-contained pockets budded off the membrane. The vesicles may carry cargo of antibiotics to inhibit competitors’ growth, or toxins to lyse neighboring cells. Or enzymes to digest those lysed remains into nutrients that your cell can easily take up as food. Alternatively, they may carry emergency kits, first aid and survival factors for other members of a community biofilm. The appearance of these vesicles can vary as much as their contents <a href="envelope.html#moretubular-vesicles">More–Pearled vesicles</a>. They are usually spherical, though, of a fairly consistent size, and often come off the cell at one or a few sites, forming chains, as you can see in this <em>Myxococcus xanthus</em> cell.</p>
<p>Not all diderms produce outer membrane vesicles, and even for those that do, we still do not know exactly how they do it. Maybe it happens spontaneously due to the physics of lipids and proteins in a certain configuration. Or maybe there is a dedicated protein machine in the membrane, blowing bubbles. Vesicles can also bud from the cytoplasmic or (for monoderms) inner membrane into the cytoplasm or periplasm <a href="envelope.html#morecytoplasmic-vesicles">More–Cytoplasmic vesicles</a>. This seems to be a less regulated process than outer membrane vesicle formation, and we see it in many species when they are stressed by low nutrients or high cell density, suggesting that it is a general phenomenon. Cells shrink in harsh conditions (more on that in Chapter 8), so cytoplasmic or periplasmic vesicles may simply offer a place to put the extra membrane or, more optimistically, to store it until the time comes to grow again. Just as with outer membrane vesicles, the appearance of cytoplasmic vesicles varies widely [More–Cytoplasmic vesicle variety]. Archaea also produce membrane vesicles, both extracellular and cytoplasmic <a href="envelope.html#morearchaeal-vesicles">More–Archaeal vesicles</a>. They have been studied less than their bacterial counterparts, but likely serve similar roles in metabolism and community interactions.</p>
<div class="figure"><span id="fig:unnamed-chunk-27"></span>
<div id="htmlwidget-13359e86acaeed85caa1" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-13359e86acaeed85caa1">{"x":{"doi":"10.22002/D1.1474","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.7: Myxococcus xanthus Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1474">10.22002/D1.1474</a>
</p>
</div>
</div>
<div id="morepearled-vesicles" class="section level2">
<h2><span class="header-section-number">2.8</span> More–Pearled vesicles</h2>
<p>Different species can produce outer membrane vesicles that look very different. The same species can even produce vesicles that look very different. Sometimes they come off the cell as a chain of spheres; sometimes the spheres remain connected, like a string of pearls, as in this <em>Borrelia burgdorferi</em> cell. Sometimes vesicles form long tubes instead <a href="envelope.html#moretubular-vesicles">More–Tubular vesicles</a>. Sometimes the same chain can be tubular in one section (usually at the base, connected to the cell), and a string of spheres in another.</p>
<div class="figure"><span id="fig:unnamed-chunk-28"></span>
<div id="htmlwidget-33c14053ffd20cff3e65" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-33c14053ffd20cff3e65">{"x":{"doi":"10.22002/D1.1363","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.8: Borrelia burgdorferi 2 Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1363">10.22002/D1.1363</a>
</p>
</div>
</div>
<div id="moretubular-vesicles" class="section level2">
<h2><span class="header-section-number">2.9</span> More–Tubular vesicles</h2>
<p>Here, again from a <em>Borrelia burgdorferi</em> cell, you can see extended, tubular outer membrane vesicles.</p>
<div class="figure"><span id="fig:unnamed-chunk-29"></span>
<div id="htmlwidget-d8c1e67ae4ca80991089" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-d8c1e67ae4ca80991089">{"x":{"doi":"10.22002/D1.1353","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.9: Borrelia burgdorferi Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1353">10.22002/D1.1353</a>
</p>
</div>
</div>
<div id="morecytoplasmic-vesicles" class="section level2">
<h2><span class="header-section-number">2.10</span> More–Cytoplasmic vesicles</h2>
<p>Not all vesicles come from the outer membrane. The cytoplasmic or inner membrane can also form vesicles that are released into the cytoplasm, as in this <em>Myxococcus xanthus</em> cell, or into the periplasm.</p>
<div class="figure"><span id="fig:unnamed-chunk-30"></span>
<div id="htmlwidget-47bee23f4c0cc62d1aaf" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-47bee23f4c0cc62d1aaf">{"x":{"doi":"10.22002/D1.1364","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.10: Myxococcus xanthus Collected by: Matthew Swulius <a href="https://doi.org/10.22002/D1.1364">10.22002/D1.1364</a>
</p>
</div>
</div>
<div id="more-cytoplasmic-vesicle-variety" class="section level2">
<h2><span class="header-section-number">2.11</span> More: Cytoplasmic vesicle variety</h2>
<p>Cytoplasmic vesicles exhibit a variety of sizes and shapes. Some are nested, with vesicles inside vesicles. In this <em>Prosthecobacter debontii</em> cell, you can see two other morphologies. One is a large, flattened horseshoe-shaped vesicle. Another is a more typical spherical shape, but is decorated with what look like protein complexes.</p>
<p>This cell also has unusual structures on its surface that have yet to be identified.</p>
<div class="figure"><span id="fig:unnamed-chunk-31"></span>
<div id="htmlwidget-0ae004f521998a8dfa41" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-0ae004f521998a8dfa41">{"x":{"doi":"10.22002/D1.1365","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.11: Prosthecobacter debontii Collected by: Martin Pilhofer <a href="https://doi.org/10.22002/D1.1365">10.22002/D1.1365</a>
</p>
</div>
</div>
<div id="morearchaeal-vesicles" class="section level2">
<h2><span class="header-section-number">2.12</span> More–Archaeal vesicles</h2>
<p>Just like bacteria, archaea also produce both internal and external vesicles, as you can see in this <em>Halomicrobium mukohataei</em> cell.</p>
<div class="figure"><span id="fig:unnamed-chunk-32"></span>
<div id="htmlwidget-59151f295aca6d52b8d2" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-59151f295aca6d52b8d2">{"x":{"doi":"10.22002/D1.1475","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.12: Halomicrobium mukohataei Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1475">10.22002/D1.1475</a>
</p>
</div>
</div>
<div id="variation" class="section level2">
<h2><span class="header-section-number">2.13</span> Variation</h2>
<p>Evolution is endlessly creative, providing exceptions to nearly every classification rule. We have just described a neat breakdown of bacteria into monoderms (one membrane, thick sacculus, positive Gram stain) and diderms (two membranes, thin sacculus, negative Gram stain). But some cells, like this <em>Mycobacterium marinum</em>, defy such easy classification. <em>Mycobacteria</em> are diderm, with an inner and an outer membrane, and a cell wall. But they have unique molecules (named mycolic acids in their honor) in the outer membrane. These acids interfere with Gram staining, yielding an intermediate result between positive and negative. And their sacculus consists of three layers of sugars, each with a unique molecular composition. The middle layer is the familiar peptidoglycan.</p>
<p>This cell illustrates another important point: we cannot always see everything that is there. In this case, we cannot see an additional layer outside the outer membrane called the capsule. The capsule, present in many bacteria (mostly diderm, but also some monoderm), is made up of an “_e_xtracellular _p_olymeric _s_ubstance,&quot; or EPS: long chains of sugars, sometimes linked to the outer membrane and sometimes free-floating. These sugars help the cell attach to surfaces and offer an extra layer of protection, trapping water to prevent desiccation and making it more difficult for hydrophobic molecules like detergents to get through to disrupt the membrane(s). It also makes it more difficult for viruses to reach the cell, and for eukaryotic predators like macrophages to eat it. The EPS does not interact strongly with electrons, though, so the capsule is usually invisible by electron microscopy.</p>
<div class="figure"><span id="fig:unnamed-chunk-33"></span>
<div id="htmlwidget-22058d3402b0f01be58d" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-22058d3402b0f01be58d">{"x":{"doi":"10.22002/D1.1354","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.13: Mycobacterium marinum Collected by: Elitza Tocheva <a href="https://doi.org/10.22002/D1.1354">10.22002/D1.1354</a>
</p>
</div>
</div>
<div id="surface-layer" class="section level2">
<h2><span class="header-section-number">2.14</span> Surface Layer</h2>
<p>How else can you protect your cell from the rigors of a harsh world? What about encasing it in an armored shell a la the armadillo? Many bacteria (both monoderms and diderms) and archaea use modular proteins for this purpose, interlocking Lego-block-like pieces into a shell called the Surface Layer, or S-layer. This must offer a significant evolutionary advantage since up to 15% of the total protein in the cell can be found in the structure. In fact, surface layers play many roles for cells, but one of their main functions for bacteria is as a gatekeeper, preventing large things like viruses from reaching the membrane.</p>
<p>Surface layers are crystalline lattices, and they can be striking in appearance, as on this <em>Caulobacter crescentus</em> cell. Amazingly, the lattice is made from (many copies of) a single protein [Schematic–Surface layer architecture]. The pinwheel-like subunits interact laterally, leaving pores large enough for nutrients to pass through, but too small for viruses. The modular nature of the lattice means that units can be popped in as the cell grows, or popped out to allow a cell appendage to poke through. They even accommodate vesicles <a href="envelope.html#morenanopods">More–Nanopods</a>. Surface layer proteins are often chemically modified to alter their properties; for instance, a sugar may be added to stick the cell to a surface.</p>
<div class="figure"><span id="fig:unnamed-chunk-34"></span>
<div id="htmlwidget-65a3907220d6e0b4053d" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-65a3907220d6e0b4053d">{"x":{"doi":"10.22002/D1.1355","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.14: Caulobacter crescentus Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1355">10.22002/D1.1355</a>
</p>
</div>
<div id="PDB:_5N97_Surface_layer_architecture" class="section level3">
<h3><span class="header-section-number">2.14.1</span> PDB: 5N97 Surface layer architecture</h3>
<p><img src="img/schematics/2_6_1.gif" /><!-- --></p>
<p>A single protein forms the surface layer you just saw in <em>Caulobacter crescentus</em>. The protein has two domains. The bottom domain anchors it to lipoproteins attached to the outer membrane. The top domain forms the canopy of the S-layer, organizing hierarchically into hexameric rosettes like this that in turn pack into a larger hexameric lattice <span class="citation">Bharat et al. (<a href="#ref-bharat2017">2017</a>)</span>. This lattice is flexible enough to curve around even narrow regions of the cell like the stalk discussed in Chapter 3.</p>
</div>
</div>
<div id="morenanopods" class="section level2">
<h2><span class="header-section-number">2.15</span> More–Nanopods</h2>
<p>Archaea and bacteria with surface layers produce characteristic outer membrane vesicles: they bud off with the surface layer attached. You will see plenty of examples in the coming pages. <em>Delftia acidovorans</em> like this produce so-called nanopods: chains of outer membrane vesicles ensheathed in surface layer.</p>
<div class="figure"><span id="fig:unnamed-chunk-36"></span>
<div id="htmlwidget-9412ce7465a537844172" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-9412ce7465a537844172">{"x":{"doi":"10.22002/D1.1366","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.15: Delftia acidovorans Collected by: Elitza Tocheva <a href="https://doi.org/10.22002/D1.1366">10.22002/D1.1366</a>
</p>
</div>
</div>
<div id="surface-layer-variety" class="section level2">
<h2><span class="header-section-number">2.16</span> Surface Layer Variety</h2>
<p>One of the most striking features of the surface layer is how different it can look in different species. For instance, compare this archaeal <em>Sulfolobus solfataricus</em> cell to the bacterium on the last page, or to other diderm [More–Methylomicrobium alcaliphilum] or monoderm <a href="envelope.html#moreclostridium-thermocellum">More–Clostridium thermocellum</a> bacteria, or archaea <a href="envelope.html#moremethanoregula-formicica">More–Nitrosopumilis maritimus</a>. All surface layers are crystalline lattices of a single–or in a few cases, two–proteins, but the particular pattern of the lattice depends on the shape of this building block and how it multimerizes into a higher-order structure. The shape of the building block varies considerably; there is almost no sequence homology between S-layer proteins from different species. And shapes come together in different ways, forming repeating units of one, two, three (as on this cell), four, or six blocks.</p>
<p>S-layers are very common in archaea like this <em>S. solfataricus</em>. Most archaea lack cell walls, so the surface layer functions as external scaffolding. Remember, too, that nearly all archaea are monoderms, lacking the extra periplasmic compartment that diderms have. Here again the surface layer serves a similar function, enclosing a space around the cell’s membrane called the pseudo-periplasmic space. Just as with the bacterial periplasm, this space serves as an antechamber for the cell, restricting access by large molecules. In some cases, the pseudo-periplasmic space also contains specific proteins that function in metabolism.</p>
<div class="figure"><span id="fig:unnamed-chunk-37"></span>
<div id="htmlwidget-e798ab4d59df0830dba7" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-e798ab4d59df0830dba7">{"x":{"doi":"10.22002/D1.1356","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.16: Sulfolobus solfataricus Collected by: Lu Gan <a href="https://doi.org/10.22002/D1.1356">10.22002/D1.1356</a>
</p>
</div>
</div>
<div id="morem.-alcaliphilum" class="section level2">
<h2><span class="header-section-number">2.17</span> More–M. alcaliphilum</h2>
<p>In <em>Methylomicrobium alcaliphilum</em>, V-shaped surface layer proteins come together to form cups that pack into a hexagonal pattern.</p>
<div class="figure"><span id="fig:unnamed-chunk-38"></span>
<div id="htmlwidget-edef3c10baa6e4f2a0bd" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-edef3c10baa6e4f2a0bd">{"x":{"doi":"10.22002/D1.1367","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.17: Methylomicrobium alcaliphilum Collected by: Songye Chen <a href="https://doi.org/10.22002/D1.1367">10.22002/D1.1367</a>
</p>
</div>
</div>
<div id="moreclostridium-thermocellum" class="section level2">
<h2><span class="header-section-number">2.18</span> More–Clostridium thermocellum</h2>
<p>Not only diderm bacteria have surface layers, as you can see on this monoderm <em>Clostridium thermocellum</em> cell. The main difference is that the proteins are anchored to the cell wall, rather than to the outer membrane (or associated lipoproteins).</p>
<div class="figure"><span id="fig:unnamed-chunk-39"></span>
<div id="htmlwidget-469f0834b61fec82397d" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-469f0834b61fec82397d">{"x":{"doi":"10.22002/D1.1476","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.18: Clostridium thermocellum Collected by: William Nicolas <a href="https://doi.org/10.22002/D1.1476">10.22002/D1.1476</a>
</p>
</div>
</div>
<div id="moren.-maritimus" class="section level2">
<h2><span class="header-section-number">2.19</span> More–N. maritimus</h2>
<p>In <em>Nitrosopumilis maritimus</em>, surface layer proteins form hexagonal rosettes that in turn pack into a hexagonal lattice.</p>
<div class="figure"><span id="fig:unnamed-chunk-40"></span>
<div id="htmlwidget-768b91a56649815c9057" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-768b91a56649815c9057">{"x":{"doi":"10.22002/D1.1368","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.19: Nitrosopumilis maritimus Collected by: Rasika Ramdasi <a href="https://doi.org/10.22002/D1.1368">10.22002/D1.1368</a>
</p>
</div>
</div>
<div id="moremethanoregula-formicica" class="section level2">
<h2><span class="header-section-number">2.20</span> More–Methanoregula formicica</h2>
<p>The surface layer of <em>Methanoregula formicica</em> is a hexagonal lattice of small, nearly circular subunits.</p>
<div class="figure"><span id="fig:unnamed-chunk-41"></span>
<div id="htmlwidget-d9b1607af770d87109c8" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-d9b1607af770d87109c8">{"x":{"doi":"10.22002/D1.1369","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.20: Methanoregula formicica Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1369">10.22002/D1.1369</a>
</p>
</div>
</div>
<div id="sheath" class="section level2">
<h2><span class="header-section-number">2.21</span> Sheath</h2>
<p>Why stop at a single layer of protein? For proof that Nature is endlessly inventive, consider this <em>Methanospirillum hungatei</em> cell. These archaea encase themselves in a surface layer, as well as an additional protein layer that forms a highly impermeable sheath. The sheath is also very resistant to pressure, which could be important in the cells’ line of work. <em>M. hungatei</em> were discovered in sewage, where they break down organic waste, producing methane. One theory is that the sheath acts as a pressure regulator; when enough methane builds up inside the cell, the pressure expands the sheath, opening its pores wide enough to allow the methane to dissipate and new metabolic substrates like hydrogen and carbon dioxide to enter.</p>
<p>The rules of architecture remain the same, though. Just as in the bacterial cell wall, sheath polymers are arranged as hoops perpendicular to the long axis of the rod-shaped cell. The ends of the sheath, as you can see, are special, with multiple protein layers stacking into a thick plug. Cells divide within the sheath, and long chains of cells in a continuous sheath are often observed.</p>
<div class="figure"><span id="fig:unnamed-chunk-42"></span>
<div id="htmlwidget-795f6c32c9636b41e277" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-795f6c32c9636b41e277">{"x":{"doi":"10.22002/D1.1357","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.21: Methanospirillum hungatei Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1357">10.22002/D1.1357</a>
</p>
</div>
</div>
<div id="dna" class="section level2">
<h2><span class="header-section-number">2.22</span> DNA</h2>
<p>All the layers we just discussed collectively make up the container, or envelope, of a cell. As you have seen, different species use different combinations of these components to form their envelopes; the only constant is the cytoplasmic (or inner, for diderms) membrane.</p>
<p>Now consider what these envelopes contain. In addition to water and small molecules, you have already seen some large protein complexes like motility machines. You have also seen many ribosomes–the protein/RNA complexes responsible for translation. But you might have been surprised <em>not</em> to see something else: deoxyribonucleic acid, or DNA. The molecule containing the instructions for the life of the cell is of paramount importance yet often invisible by microscopy. But not always. Thin filaments of DNA, only about 2 nm wide, blend in with the dense cytoplasm of the cell. When a cell lyses, though, its cytoplasm diffuses into the environment and the DNA filaments stand out against the now-much-reduced background. You can get an idea of the sheer abundance of DNA inside a cell from this <em>Haloarcula argentinensis</em> whose envelope has ruptured and contents are spilling out.</p>
<div class="figure"><span id="fig:unnamed-chunk-43"></span>
<div id="htmlwidget-6cde7da89b53c3d33c96" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-6cde7da89b53c3d33c96">{"x":{"doi":"10.22002/D1.1358","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.22: Haloarcula argentinensis Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1358">10.22002/D1.1358</a>
</p>
</div>
</div>
<div id="nucleoid" class="section level2">
<h2><span class="header-section-number">2.23</span> Nucleoid</h2>
<p>Cells contain enormous amounts of DNA. The single, circular chromosome of this <em>Bdellovibrio bacteriovorus</em> cell contains 3,782,950 individual nucleotide pairs, which means that if the circle were cut and laid out as a long piece, it would be about <em>one thousand times</em> longer than the cell itself. To fit and function inside the cell, the chromosome has to be extraordinarily organized and packed, a feat we still do not understand. Some of this packing is evident in nearly every cell: the center of the cell tends to have very few large macromolecular complexes like ribosomes, because they are excluded by the densely-packed chromosome(s). Look for these ribosome-excluding zones in the rest of the book; they indicate the location of the bulk of the cell’s DNA. Since this region is not enclosed by an internal membrane, it is not called a nucleus (the “karyon” that defines eukaryotes). Instead, we use the term nucleoid to describe the cytoplasmic region where most of the DNA is concentrated.</p>
<p>At times, the nucleoid becomes easier to see. Imagine that your cell wanted to decrease its gene expression (we will discuss why in Chapters 8 and 9). One approach is simply to pack the chromosome so tightly that the transcriptional machinery cannot access the genes. This cell is doing just that, condensing its nucleoid into a dense twisted braid we can easily visualize.</p>
<div class="figure"><span id="fig:unnamed-chunk-44"></span>
<div id="htmlwidget-ed76a81d35dcc17dcab3" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-ed76a81d35dcc17dcab3">{"x":{"doi":"10.22002/D1.1359","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.23: Bdellovibrio bacteriovorus Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1359">10.22002/D1.1359</a>
</p>
</div>
</div>
<div id="further-reading-1" class="section level2">
<h2><span class="header-section-number">2.24</span> Further Reading</h2>
<p><span class="citation">Errington (<a href="#ref-errington2013">2013</a>)</span> <span class="citation">Ptacin and Shapiro (<a href="#ref-ptacin2013">2013</a>)</span> <span class="citation">Sleytr and Beveridge (<a href="#ref-sleytr1999">1999</a>)</span> <span class="citation">Strahl and Errington (<a href="#ref-strahl2017">2017</a>)</span></p>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-bharat2017">
<p>Bharat, T. A. M., D. Kureisaite-Ciziene, G. G. Hardy, E. W. Yu, J. M. Devant, W. J. H. Hagen, Y. V. Brun, J. A. G. Briggs, and J. Lowe. 2017. “Structure of the Hexagonal Surface Layer on Caulobacter Crescentus Cells.” <em>Nat Microbiol</em> 2 (April): 17059. doi:<a href="https://doi.org/10.1038/nmicrobiol.2017.59">10.1038/nmicrobiol.2017.59</a>.</p>
</div>
<div id="ref-errington2013">
<p>Errington, J. 2013. “L-Form Bacteria, Cell Walls and the Origins of Life.” <em>Open Biol</em> 3 (1): 120143. doi:<a href="https://doi.org/10.1098/rsob.120143">10.1098/rsob.120143</a>.</p>
</div>
<div id="ref-nguyen2015">
<p>Nguyen, L. T., J. C. Gumbart, M. Beeby, and G. J. Jensen. 2015. “Coarse-Grained Simulations of Bacterial Cell Wall Growth Reveal That Local Coordination Alone Can Be Sufficient to Maintain Rod Shape.” <em>Proc Natl Acad Sci U S A</em> 112 (28): E3689–98. doi:<a href="https://doi.org/10.1073/pnas.1504281112">10.1073/pnas.1504281112</a>.</p>
</div>
<div id="ref-ptacin2013">
<p>Ptacin, J. L., and L. Shapiro. 2013. “Chromosome Architecture Is a Key Element of Bacterial Cellular Organization.” <em>Cell Microbiol</em> 15 (1): 45–52. doi:<a href="https://doi.org/10.1111/cmi.12049">10.1111/cmi.12049</a>.</p>
</div>
<div id="ref-shu2000">
<p>Shu, Wei, Jie Liu, Hong Ji, and Min Lu. 2000. “Core Structure of the Outer Membrane Lipoprotein from Escherichia Coli at 1.9 å resolution11Edited by D. Rees.” <em>Journal of Molecular Biology</em> 299 (4): 1101–12. doi:<a href="https://doi.org/10.1006/jmbi.2000.3776">10.1006/jmbi.2000.3776</a>.</p>
</div>
<div id="ref-sleytr1999">
<p>Sleytr, U. B., and T. J. Beveridge. 1999. “Bacterial S-Layers.” <em>Trends Microbiol</em> 7 (6): 253–60.</p>
</div>
<div id="ref-sobti2016">
<p>Sobti, Meghna, Callum Smits, Andrew SW Wong, Robert Ishmukhametov, Daniela Stock, Sara Sandin, and Alastair G Stewart. 2016. “Cryo-EM Structures of the Autoinhibited E. Coli ATP Synthase in Three Rotational States.” Edited by Werner Kühlbrandt. <em>eLife</em> 5 (December). eLife Sciences Publications, Ltd: e21598. doi:<a href="https://doi.org/10.7554/eLife.21598">10.7554/eLife.21598</a>.</p>
</div>
<div id="ref-strahl2017">
<p>Strahl, H., and J. Errington. 2017. “Bacterial Membranes: Structure, Domains, and Function.” <em>Annu Rev Microbiol</em>, July. doi:<a href="https://doi.org/10.1146/annurev-micro-102215-095630">10.1146/annurev-micro-102215-095630</a>.</p>
</div>
<div id="ref-thomas1990">
<p>Thomas, Lewis. 1990. <em>A Long Line of Cells: Collected Essays</em>. New York: Book-of-the-Month-Club.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="methods.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="shape.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": null,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/caltechlibrary/cell-atlas/edit/master/02_Envelope.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["cell-atlas.pdf", "cell-atlas.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
