<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Envelope | Cell Atlas</title>
  <meta name="description" content="An atlas produced by the Jensen Lab." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Envelope | Cell Atlas" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="splash/HeaderImage.jpg" />
  <meta property="og:description" content="An atlas produced by the Jensen Lab." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Envelope | Cell Atlas" />
  
  <meta name="twitter:description" content="An atlas produced by the Jensen Lab." />
  <meta name="twitter:image" content="splash/HeaderImage.jpg" />

<meta name="author" content="Jensen Lab" />


<meta name="date" content="2020-07-29" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="methods.html"/>
<link rel="next" href="shape.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/CL-0.1.2/CL.js"></script>
<script src="libs/doivideo-binding-0.0.0.9000/doivideo.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Cell Atlas</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#section"><i class="fa fa-check"></i><b>1.1</b> 1</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#schematicstructural-biology-toolkit"><i class="fa fa-check"></i><b>2.1</b> 1[Schematic–Structural biology toolkit]</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#escherichia-coli"><i class="fa fa-check"></i><b>2.2</b> Escherichia coli</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#shewanella-oneidensis"><i class="fa fa-check"></i><b>2.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#schematicelectron-microscopy-modes"><i class="fa fa-check"></i><b>2.4</b> 3[Schematic–Electron microscopy modes]</a></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#shewanella-oneidensis-1"><i class="fa fa-check"></i><b>2.5</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="2.6" data-path="methods.html"><a href="methods.html#caulobacter-crescentus"><i class="fa fa-check"></i><b>2.6</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="2.7" data-path="methods.html"><a href="methods.html#caulobacter-crescentus-1"><i class="fa fa-check"></i><b>2.7</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="2.8" data-path="methods.html"><a href="methods.html#schematicmissing-wedge"><i class="fa fa-check"></i><b>2.8</b> 6[Schematic–Missing wedge]</a></li>
<li class="chapter" data-level="2.9" data-path="methods.html"><a href="methods.html#campylobacter-jejuni"><i class="fa fa-check"></i><b>2.9</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.10" data-path="methods.html"><a href="methods.html#campylobacter-jejuni-1"><i class="fa fa-check"></i><b>2.10</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.11" data-path="methods.html"><a href="methods.html#campylobacter-jejuni-2"><i class="fa fa-check"></i><b>2.11</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.12" data-path="methods.html"><a href="methods.html#schematicstructural-biology-timeline"><i class="fa fa-check"></i><b>2.12</b> 9[Schematic–Structural biology timeline]</a></li>
<li class="chapter" data-level="2.13" data-path="methods.html"><a href="methods.html#bdellovibrio-bacteriovorus"><i class="fa fa-check"></i><b>2.13</b> Bdellovibrio bacteriovorus</a><ul>
<li class="chapter" data-level="2.13.1" data-path="methods.html"><a href="methods.html#Difference_mapping"><i class="fa fa-check"></i><b>2.13.1</b> Campylobacter jejuni</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="envelope.html"><a href="envelope.html"><i class="fa fa-check"></i><b>3</b> Envelope</a><ul>
<li class="chapter" data-level="3.1" data-path="envelope.html"><a href="envelope.html#schematic-lipid-bilayer"><i class="fa fa-check"></i><b>3.1</b> 1[Schematic: Lipid bilayer]</a></li>
<li class="chapter" data-level="3.2" data-path="envelope.html"><a href="envelope.html#schematic-atp-synthase-and-energy-production"><i class="fa fa-check"></i><b>3.2</b> 1[Schematic: ATP synthase and energy production]</a></li>
<li class="chapter" data-level="3.3" data-path="envelope.html"><a href="envelope.html#listeria-monocytogenes"><i class="fa fa-check"></i><b>3.3</b> Listeria monocytogenes</a><ul>
<li class="chapter" data-level="3.3.1" data-path="envelope.html"><a href="envelope.html#Peptidoglycan_architecture"><i class="fa fa-check"></i><b>3.3.1</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="3.3.2" data-path="envelope.html"><a href="envelope.html#Methanobacterium_formicicum"><i class="fa fa-check"></i><b>3.3.2</b> Methanobacterium formicicum</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="envelope.html"><a href="envelope.html#cupriavidus-necator"><i class="fa fa-check"></i><b>3.4</b> Cupriavidus necator</a></li>
<li class="chapter" data-level="3.5" data-path="envelope.html"><a href="envelope.html#schematicbrauns-lipoprotein"><i class="fa fa-check"></i><b>3.5</b> 3[Schematic–Braun’s lipoprotein]</a><ul>
<li class="chapter" data-level="3.5.1" data-path="envelope.html"><a href="envelope.html#escherichia-coli-diderm_sacculus_architecturecompare-this-diderm-sacculus-purified-from-escherichia-coli-to-the-monoderm-sacculus-on-the-last-page.-since-this-one-is-thinner-we-can-make-out-more-details-like-the-glycan-strands-running-around-the-circumference-of-the-cell.-the-main-difference-between-the-two-types-of-sacculi-seems-to-be-whether-they-have-largely-a-single-layer-of-peptidoglycan-diderm-or-many-layers-monoderm.-so-even-though-the-two-types-of-wall-look-different-their-architecture-is-fundamentally-the-same.-in-some-circumstances-as-you-will-see-in-chapter-8-cells-can-even-switch-between-the-two-forms."><i class="fa fa-check"></i><b>3.5.1</b> Escherichia coli {#Diderm_sacculus_architecture}Compare this diderm sacculus purified from <em>Escherichia coli</em> to the monoderm sacculus on the last page. Since this one is thinner, we can make out more details, like the glycan strands running around the circumference of the cell. The main difference between the two types of sacculi seems to be whether they have largely a single layer of peptidoglycan (diderm) or many layers (monoderm). So even though the two types of wall look different, their architecture is fundamentally the same. In some circumstances, as you will see in Chapter 8, cells can even switch between the two forms.</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="envelope.html"><a href="envelope.html#schematicsacculus-remodeling"><i class="fa fa-check"></i><b>3.6</b> 3[Schematic–Sacculus remodeling]</a></li>
<li class="chapter" data-level="3.7" data-path="envelope.html"><a href="envelope.html#myxococcus-xanthus"><i class="fa fa-check"></i><b>3.7</b> Myxococcus xanthus</a><ul>
<li class="chapter" data-level="3.7.1" data-path="envelope.html"><a href="envelope.html#borrelia-burgdorferi-pearled_vesiclesdifferent-species-can-produce-outer-membrane-vesicles-that-look-very-different.-the-same-species-can-even-produce-vesicles-that-look-very-different.-sometimes-they-come-off-the-cell-as-a-chain-of-spheres-sometimes-the-spheres-remain-connected-like-a-string-of-pearls-as-in-this-borrelia-burgdorferi-cell.-sometimes-vesicles-form-long-tubes-instead-moretubular-vesicles.-sometimes-the-same-chain-can-be-tubular-in-one-section-usually-at-the-base-connected-to-the-cell-and-a-string-of-spheres-in-another."><i class="fa fa-check"></i><b>3.7.1</b> Borrelia burgdorferi {#Pearled_vesicles}Different species can produce outer membrane vesicles that look very different. The same species can even produce vesicles that look very different. Sometimes they come off the cell as a chain of spheres; sometimes the spheres remain connected, like a string of pearls, as in this <em>Borrelia burgdorferi</em> cell. Sometimes vesicles form long tubes instead [More–Tubular vesicles]. Sometimes the same chain can be tubular in one section (usually at the base, connected to the cell), and a string of spheres in another.</a></li>
<li class="chapter" data-level="3.7.2" data-path="envelope.html"><a href="envelope.html#borrelia-burgdorferi-tubular_vesicleshere-again-from-a-borrelia-burgdorferi-cell-you-can-see-extended-tubular-outer-membrane-vesicles."><i class="fa fa-check"></i><b>3.7.2</b> Borrelia burgdorferi {#Tubular_vesicles}Here, again from a <em>Borrelia burgdorferi</em> cell, you can see extended, tubular outer membrane vesicles.</a></li>
<li class="chapter" data-level="3.7.3" data-path="envelope.html"><a href="envelope.html#myxococcus-xanthus-cytoplasmic_vesiclesnot-all-vesicles-come-from-the-outer-membrane.-the-cytoplasmic-or-inner-membrane-can-also-form-vesicles-that-are-released-into-the-cytoplasm-as-in-this-myxococcus-xanthus-cell-or-into-the-periplasm."><i class="fa fa-check"></i><b>3.7.3</b> Myxococcus xanthus {#Cytoplasmic_vesicles}Not all vesicles come from the outer membrane. The cytoplasmic or inner membrane can also form vesicles that are released into the cytoplasm, as in this <em>Myxococcus xanthus</em> cell, or into the periplasm.</a></li>
<li class="chapter" data-level="3.7.4" data-path="envelope.html"><a href="envelope.html#prosthecobacter-debontii-cytoplasmic_vesicle_varietycytoplasmic-vesicles-exhibit-a-variety-of-sizes-and-shapes.-some-are-nested-with-vesicles-inside-vesicles.-in-this-prosthecobacter-debontii-cell-you-can-see-two-other-morphologies.-one-is-a-large-flattened-horseshoe-shaped-vesicle.-another-is-a-more-typical-spherical-shape-but-is-decorated-with-what-look-like-protein-complexes."><i class="fa fa-check"></i><b>3.7.4</b> Prosthecobacter debontii {#Cytoplasmic_vesicle_variety}Cytoplasmic vesicles exhibit a variety of sizes and shapes. Some are nested, with vesicles inside vesicles. In this <em>Prosthecobacter debontii</em> cell, you can see two other morphologies. One is a large, flattened horseshoe-shaped vesicle. Another is a more typical spherical shape, but is decorated with what look like protein complexes.</a></li>
<li class="chapter" data-level="3.7.5" data-path="envelope.html"><a href="envelope.html#halomicrobium-mukohataei-archaeal_vesiclesjust-like-bacteria-archaea-also-produce-both-internal-and-external-vesicles-as-you-can-see-in-this-halomicrobium-mukohataei-cell."><i class="fa fa-check"></i><b>3.7.5</b> Halomicrobium mukohataei {#Archaeal_vesicles}Just like bacteria, archaea also produce both internal and external vesicles, as you can see in this <em>Halomicrobium mukohataei</em> cell.</a></li>
</ul></li>
<li class="chapter" data-level="3.8" data-path="envelope.html"><a href="envelope.html#mycobacterium-marinum"><i class="fa fa-check"></i><b>3.8</b> Mycobacterium marinum</a></li>
<li class="chapter" data-level="3.9" data-path="envelope.html"><a href="envelope.html#caulobacter-crescentus-2"><i class="fa fa-check"></i><b>3.9</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="3.10" data-path="envelope.html"><a href="envelope.html#schematicsurface-layer-architecture"><i class="fa fa-check"></i><b>3.10</b> 6[Schematic–Surface layer architecture]</a></li>
<li class="chapter" data-level="3.11" data-path="envelope.html"><a href="envelope.html#aarchaea-and-bacteria-with-surface-layers-produce-characteristic-outer-membrane-vesicles-they-bud-off-with-the-surface-layer-attached.-you-will-see-plenty-of-examples-in-the-coming-pages.-delftia-acidovorans-like-this-produce-so-called-nanopods-chains-of-outer-membrane-vesicles-ensheathed-in-surface-layer."><i class="fa fa-check"></i><b>3.11</b> 6aArchaea and bacteria with surface layers produce characteristic outer membrane vesicles: they bud off with the surface layer attached. You will see plenty of examples in the coming pages. <em>Delftia acidovorans</em> like this produce so-called nanopods: chains of outer membrane vesicles ensheathed in surface layer.</a></li>
<li class="chapter" data-level="3.12" data-path="envelope.html"><a href="envelope.html#sulfolobus-solfataricus"><i class="fa fa-check"></i><b>3.12</b> Sulfolobus solfataricus</a><ul>
<li class="chapter" data-level="3.12.1" data-path="envelope.html"><a href="envelope.html#methylomicrobium-alcaliphilum-m._alcaliphilumin-methylomicrobium-alcaliphilum-v-shaped-surface-layer-proteins-come-together-to-form-cups-that-pack-into-a-hexagonal-pattern."><i class="fa fa-check"></i><b>3.12.1</b> Methylomicrobium alcaliphilum {#M._alcaliphilum}In <em>Methylomicrobium alcaliphilum</em>, V-shaped surface layer proteins come together to form cups that pack into a hexagonal pattern.</a></li>
<li class="chapter" data-level="3.12.2" data-path="envelope.html"><a href="envelope.html#clostridium-thermocellum-clostridium_thermocellumnot-only-diderm-bacteria-have-surface-layers-as-you-can-see-on-this-monoderm-clostridium-thermocellum-cell.-the-main-difference-is-that-the-proteins-are-anchored-to-the-cell-wall-rather-than-to-the-outer-membrane-or-associated-lipoproteins."><i class="fa fa-check"></i><b>3.12.2</b> Clostridium thermocellum {#Clostridium_thermocellum}Not only diderm bacteria have surface layers, as you can see on this monoderm <em>Clostridium thermocellum</em> cell. The main difference is that the proteins are anchored to the cell wall, rather than to the outer membrane (or associated lipoproteins).</a></li>
<li class="chapter" data-level="3.12.3" data-path="envelope.html"><a href="envelope.html#nitrosopumilis-maritimus-n._maritimusin-nitrosopumilis-maritimus-surface-layer-proteins-form-hexagonal-rosettes-that-in-turn-pack-into-a-hexagonal-lattice."><i class="fa fa-check"></i><b>3.12.3</b> Nitrosopumilis maritimus {#N._maritimus}In <em>Nitrosopumilis maritimus</em>, surface layer proteins form hexagonal rosettes that in turn pack into a hexagonal lattice.</a></li>
<li class="chapter" data-level="3.12.4" data-path="envelope.html"><a href="envelope.html#methanoregula-formicica-methanoregula_formicicathe-surface-layer-of-methanoregula-formicica-is-a-hexagonal-lattice-of-small-nearly-circular-subunits."><i class="fa fa-check"></i><b>3.12.4</b> Methanoregula formicica {#Methanoregula_formicica}The surface layer of <em>Methanoregula formicica</em> is a hexagonal lattice of small, nearly circular subunits.</a></li>
</ul></li>
<li class="chapter" data-level="3.13" data-path="envelope.html"><a href="envelope.html#methanospirillum-hungatei"><i class="fa fa-check"></i><b>3.13</b> Methanospirillum hungatei</a></li>
<li class="chapter" data-level="3.14" data-path="envelope.html"><a href="envelope.html#haloarcula-argentinensis"><i class="fa fa-check"></i><b>3.14</b> Haloarcula argentinensis</a></li>
<li class="chapter" data-level="3.15" data-path="envelope.html"><a href="envelope.html#bdellovibrio-bacteriovorus-1"><i class="fa fa-check"></i><b>3.15</b> Bdellovibrio bacteriovorus</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="shape.html"><a href="shape.html"><i class="fa fa-check"></i><b>4</b> Shape</a><ul>
<li class="chapter" data-level="4.1" data-path="shape.html"><a href="shape.html#cupriavidus-necator-1"><i class="fa fa-check"></i><b>4.1</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="4.1.1" data-path="shape.html"><a href="shape.html#brucella-abortus-rod_varietynot-all-rod-shaped-cells-are-perfectly-cylindrical.-for-instance-brucella-abortus-like-this-one-are-pear-shaped.-this-is-one-of-the-rod-shaped-bacterial-species-that-do-not-use-mreb."><i class="fa fa-check"></i><b>4.1.1</b> Brucella abortus {#Rod_variety}Not all rod-shaped cells are perfectly cylindrical. For instance, <em>Brucella abortus</em> like this one are pear-shaped. This is one of the rod-shaped bacterial species that do not use MreB.</a></li>
<li class="chapter" data-level="4.1.2" data-path="shape.html"><a href="shape.html#methanoregula-formicica-archaeal_rodssome-archaeal-species-like-this-methanoregula-formicica-are-rod-shaped.-the-rigid-surface-layer-seems-to-help-define-the-shape-but-the-details-are-still-unknown."><i class="fa fa-check"></i><b>4.1.2</b> Methanoregula formicica {#Archaeal_rods}Some archaeal species, like this <em>Methanoregula formicica</em>, are rod-shaped. The rigid surface layer seems to help define the shape, but the details are still unknown.</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="shape.html"><a href="shape.html#hylemonella-gracilis"><i class="fa fa-check"></i><b>4.2</b> Hylemonella gracilis</a></li>
<li class="chapter" data-level="4.3" data-path="shape.html"><a href="shape.html#caulobacter-crescentus-3"><i class="fa fa-check"></i><b>4.3</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="4.4" data-path="shape.html"><a href="shape.html#schematic-ctp-synthase"><i class="fa fa-check"></i><b>4.4</b> 4[Schematic: CTP synthase]</a></li>
<li class="chapter" data-level="4.5" data-path="shape.html"><a href="shape.html#campylobacter-jejuni-3"><i class="fa fa-check"></i><b>4.5</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="4.5.1" data-path="shape.html"><a href="shape.html#borrelia-burgdorferi-borrelia_burgdorferiborrelia-burgdorferi-like-this-cell-use-the-long-filaments-of-their-motility-machinery-flagella-discussed-in-chapter-6-as-a-kind-of-cytoskeleton.-a-bundle-of-flagella-wraps-around-the-cell-in-the-periplasm-between-the-cell-wall-and-the-outer-membrane.-spun-by-motors-at-their-base-more-on-that-in-chapter-6-the-filaments-impart-a-wave-pattern-to-the-growing-sacculus.-without-the-motors-rotation-the-cells-develop-a-rod-shape.-note-that-b.-burgdorferi-are-not-helical-but-rather-adopt-a-two-dimensional-waveform."><i class="fa fa-check"></i><b>4.5.1</b> Borrelia burgdorferi {#Borrelia_burgdorferi}<em>Borrelia burgdorferi</em> like this cell use the long filaments of their motility machinery (flagella, discussed in Chapter 6) as a kind of cytoskeleton. A bundle of flagella wraps around the cell in the periplasm, between the cell wall and the outer membrane. Spun by motors at their base (more on that in Chapter 6), the filaments impart a wave pattern to the growing sacculus. Without the motors’ rotation, the cells develop a rod shape. Note that <em>B. burgdorferi</em> are not helical, but rather adopt a two-dimensional waveform.</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="shape.html"><a href="shape.html#verrucomicrobium-spinosum"><i class="fa fa-check"></i><b>4.6</b> Verrucomicrobium spinosum</a><ul>
<li class="chapter" data-level="4.6.1" data-path="shape.html"><a href="shape.html#Bactofilin"><i class="fa fa-check"></i><b>4.6.1</b> Bactofilin</a></li>
<li class="chapter" data-level="4.6.2" data-path="shape.html"><a href="shape.html#prosthecobacter-vanneervenii-bacterial_microtubulessome-bacterial-species-with-prosthecae-express-structures-similar-to-eukaryotic-microtubules-made-from-two-proteins-called-btuba-and-btubb-to-reflect-their-homology-to-eukaryotic-tubulins.-eukaryotic-microtubules-are-hollow-tubes-formed-by-13-protofilaments-bacterial-microtubules-are-smaller-with-only-5-protofilaments.-cells-commonly-contain-a-bundle-of-microtubules-in-their-prostheca-like-this-prosthecobacter-vanneervenii-cell-which-has-a-bundle-of-four."><i class="fa fa-check"></i><b>4.6.2</b> Prosthecobacter vanneervenii {#Bacterial_microtubules}Some bacterial species with prosthecae express structures similar to eukaryotic microtubules, made from two proteins called BtubA and BtubB to reflect their homology to eukaryotic tubulins. Eukaryotic microtubules are hollow tubes formed by 13 protofilaments; bacterial microtubules are smaller, with only ~5 protofilaments. Cells commonly contain a bundle of microtubules in their prostheca, like this <em>Prosthecobacter vanneervenii</em> cell, which has a bundle of four.</a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="shape.html"><a href="shape.html#haloquadratum-walsbyi"><i class="fa fa-check"></i><b>4.7</b> Haloquadratum walsbyi</a><ul>
<li class="chapter" data-level="4.7.1" data-path="shape.html"><a href="shape.html#halobacterium-salinarum-gas_vesiclessome-species-of-archaea-and-bacteria-use-gas-vesicles-to-control-their-buoyancy.-this-can-allow-them-to-rise-or-fall-in-a-water-column-which-can-be-a-great-advantage.-halobacterium-salinarum-like-this-one-produce-gas-vesicles-in-response-to-cues-from-the-environment-lifting-themselves-out-of-the-sediment-and-into-more-favorable-conditions-of-oxygen-or-sunlight-for-photosynthesis.-this-cell-has-just-started-producing-gas-vesicles-so-they-are-small-and-isolated.-later-the-vesicles-will-elongate-into-cylinders-with-conical-ends-as-you-saw-in-haloquadratum-walsbyi.-each-cell-might-contain-dozens-of-vesicles-and-they-often-cluster-together."><i class="fa fa-check"></i><b>4.7.1</b> Halobacterium salinarum {#Gas_vesicles}Some species of archaea and bacteria use gas vesicles to control their buoyancy. This can allow them to rise or fall in a water column, which can be a great advantage. <em>Halobacterium salinarum</em> like this one produce gas vesicles in response to cues from the environment, lifting themselves out of the sediment and into more favorable conditions of oxygen or sunlight for photosynthesis. This cell has just started producing gas vesicles, so they are small and isolated. Later, the vesicles will elongate into cylinders with conical ends, as you saw in <em>Haloquadratum walsbyi</em>. Each cell might contain dozens of vesicles, and they often cluster together.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="growth.html"><a href="growth.html"><i class="fa fa-check"></i><b>5</b> Growth</a><ul>
<li class="chapter" data-level="5.1" data-path="growth.html"><a href="growth.html#shewanella-oneidensis-2"><i class="fa fa-check"></i><b>5.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="5.2" data-path="growth.html"><a href="growth.html#rhodopseudomonas-palustris"><i class="fa fa-check"></i><b>5.2</b> Rhodopseudomonas palustris</a></li>
<li class="chapter" data-level="5.3" data-path="growth.html"><a href="growth.html#schematic-photocomplexes"><i class="fa fa-check"></i><b>5.3</b> 3[Schematic: Photocomplexes]</a><ul>
<li class="chapter" data-level="5.3.1" data-path="growth.html"><a href="growth.html#methyloprofundus-sedimenti-icm_varietyin-some-species-like-this-methyloprofundus-sedimenti-the-intracytoplasmic-membrane-is-very-extensive-and-appears-to-be-fully-separated-from-the-cell-membrane.-as-you-can-see-it-occupies-much-of-the-cytoplasmic-space."><i class="fa fa-check"></i><b>5.3.1</b> Methyloprofundus sedimenti {#ICM_variety}In some species, like this <em>Methyloprofundus sedimenti</em>, the intracytoplasmic membrane is very extensive and appears to be fully separated from the cell membrane. As you can see, it occupies much of the cytoplasmic space.</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="growth.html"><a href="growth.html#chloroflexi"><i class="fa fa-check"></i><b>5.4</b> Chloroflexi</a></li>
<li class="chapter" data-level="5.5" data-path="growth.html"><a href="growth.html#clostridium-thermocellum"><i class="fa fa-check"></i><b>5.5</b> Clostridium thermocellum</a></li>
<li class="chapter" data-level="5.6" data-path="growth.html"><a href="growth.html#schematicadhe-spirosome-structure"><i class="fa fa-check"></i><b>5.6</b> 5[Schematic–AdhE spirosome structure]</a></li>
<li class="chapter" data-level="5.7" data-path="growth.html"><a href="growth.html#acetonema-longum"><i class="fa fa-check"></i><b>5.7</b> Acetonema longum</a></li>
<li class="chapter" data-level="5.8" data-path="growth.html"><a href="growth.html#schematicmicrocompartment-shells"><i class="fa fa-check"></i><b>5.8</b> 6[Schematic–Microcompartment shells]</a></li>
<li class="chapter" data-level="5.9" data-path="growth.html"><a href="growth.html#thiomonas-intermedia"><i class="fa fa-check"></i><b>5.9</b> Thiomonas intermedia</a><ul>
<li class="chapter" data-level="5.9.1" data-path="growth.html"><a href="growth.html#halothiobacillus-neapolitanus-carboxysome_varietymost-carboxysomes-look-like-the-ones-you-saw-on-the-last-page-but-not-all.-some-are-less-regular-polyhedra-as-in-this-halothiobacillus-neapolitanus-cell.-and-some-are-stranger-still-morelong-carboxysomes."><i class="fa fa-check"></i><b>5.9.1</b> Halothiobacillus neapolitanus {#Carboxysome_variety}Most carboxysomes look like the ones you saw on the last page, but not all. Some are less regular polyhedra, as in this <em>Halothiobacillus neapolitanus</em> cell. And some are stranger still [More–Long carboxysomes].</a></li>
<li class="chapter" data-level="5.9.2" data-path="growth.html"><a href="growth.html#hydrogenovibrio-crunogenus-long_carboxysomesthe-carboxysomes-in-this-hydrogenovibrio-crunogenus-have-grown-abnormally-long-extending-across-the-full-width-of-the-cell.-these-are-unusual-but-less-dramatic-elongations-are-common."><i class="fa fa-check"></i><b>5.9.2</b> Hydrogenovibrio crunogenus {#Long_carboxysomes}The carboxysomes in this <em>Hydrogenovibrio crunogenus</em> have grown abnormally long, extending across the full width of the cell. These are unusual, but less dramatic elongations are common.</a></li>
</ul></li>
<li class="chapter" data-level="5.10" data-path="growth.html"><a href="growth.html#cupriavidus-necator-2"><i class="fa fa-check"></i><b>5.10</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="5.10.1" data-path="growth.html"><a href="growth.html#lysobacter-antibioticus-storage_granule_growthstorage-granules-expand-and-contract-to-accommodate-their-contents.-this-lysobacter-antibioticus-cell-exhibits-a-typical-range-of-sizes-of-poly-phosphate-granules."><i class="fa fa-check"></i><b>5.10.1</b> Lysobacter antibioticus {#Storage_granule_growth}Storage granules expand and contract to accommodate their contents. This <em>Lysobacter antibioticus</em> cell exhibits a typical range of sizes of poly-phosphate granules.</a></li>
</ul></li>
<li class="chapter" data-level="5.11" data-path="growth.html"><a href="growth.html#agrobacterium-tumefaciens"><i class="fa fa-check"></i><b>5.11</b> Agrobacterium tumefaciens</a><ul>
<li class="chapter" data-level="5.11.1" data-path="growth.html"><a href="growth.html#hydrogenovibrio-crunogenus-granule-containing_carboxysomessmall-poly-phosphate-storage-granules-are-sometimes-seen-inside-carboxysomes-as-in-this-hydrogenovibrio-crunogenus-cell.-their-purpose-is-unknown."><i class="fa fa-check"></i><b>5.11.1</b> Hydrogenovibrio crunogenus {#Granule-containing_carboxysomes}Small poly-phosphate storage granules are sometimes seen <em>inside</em> carboxysomes, as in this <em>Hydrogenovibrio crunogenus</em> cell. Their purpose is unknown.</a></li>
<li class="chapter" data-level="5.11.2" data-path="growth.html"><a href="growth.html#halothiobacillus-neapolitanus-carboxysomes-granule_connectionswe-sometimes-see-comb-like-structures-connecting-external-poly-phosphate-storage-granules-with-carboxysomes-as-in-this-halothiobacillus-neapolitanus-cell.-the-nature-of-the-relationship-between-the-structures-and-the-identity-and-function-of-the-combs-remains-a-mystery."><i class="fa fa-check"></i><b>5.11.2</b> Halothiobacillus neapolitanus {#Carboxysomes-granule_connections}We sometimes see comb-like structures connecting external poly-phosphate storage granules with carboxysomes, as in this <em>Halothiobacillus neapolitanus</em> cell. The nature of the relationship between the structures, and the identity and function of the combs, remains a mystery.</a></li>
</ul></li>
<li class="chapter" data-level="5.12" data-path="growth.html"><a href="growth.html#halorubrum-litoreum"><i class="fa fa-check"></i><b>5.12</b> Halorubrum litoreum</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="division.html"><a href="division.html"><i class="fa fa-check"></i><b>6</b> Division</a><ul>
<li class="chapter" data-level="6.1" data-path="division.html"><a href="division.html#caulobacter-crescentus-4"><i class="fa fa-check"></i><b>6.1</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="6.2" data-path="division.html"><a href="division.html#hyphomonas-neptunium"><i class="fa fa-check"></i><b>6.2</b> Hyphomonas neptunium</a><ul>
<li class="chapter" data-level="6.2.1" data-path="division.html"><a href="division.html#hyphomonas-neptunium-hyphomonas_stalkhyphomonas-neptunium-grow-a-single-stalk-from-one-end-of-their-cell-body-similar-to-the-caulobacter-crescentus-you-saw-in-chapter-3.-the-function-of-the-stalk-though-is-different-in-this-budding-bacterium-than-in-c.-crescentus-which-divide-by-more-conventional-fission."><i class="fa fa-check"></i><b>6.2.1</b> Hyphomonas neptunium {#Hyphomonas_stalk}<em>Hyphomonas neptunium</em> grow a single stalk from one end of their cell body, similar to the <em>Caulobacter crescentus</em> you saw in Chapter 3. The function of the stalk, though, is different in this budding bacterium than in <em>C. crescentus</em>, which divide by more conventional fission.</a></li>
<li class="chapter" data-level="6.2.2" data-path="division.html"><a href="division.html#hyphomonas-neptunium-hyphomonas_lifecyclehyphomonas-neptunium-have-evolved-a-program-of-stages-they-pass-through-in-the-course-of-their-life.-when-a-newborn-cell-like-this-one-is-released-it-is-in-the-swarmer-stage-using-its-flagellum-discussed-in-the-next-chapter-to-swim-away-to-a-new-location.-after-a-while-it-settles-down-jettisoning-its-flagellum-and-growing-a-stalk-with-which-to-make-its-own-budthe-next-generation.-once-a-cell-settles-down-into-the-stalked-stage-it-spends-the-rest-of-its-life-sending-off-buds-as-long-as-conditions-are-good.-only-newborn-cells-are-equipped-with-flagella-to-go-adventuring.-we-will-discuss-more-lifecycles-in-chapter-8."><i class="fa fa-check"></i><b>6.2.2</b> Hyphomonas neptunium {#Hyphomonas_lifecycle}<em>Hyphomonas neptunium</em> have evolved a program of stages they pass through in the course of their life. When a newborn cell like this one is released, it is in the “swarmer” stage, using its flagellum (discussed in the next chapter) to swim away to a new location. After a while, it settles down, jettisoning its flagellum and growing a stalk with which to make its own bud–the next generation. Once a cell settles down into the stalked stage, it spends the rest of its life sending off buds as long as conditions are good. Only newborn cells are equipped with flagella to go adventuring. We will discuss more lifecycles in Chapter 8.</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="division.html"><a href="division.html#staphylococcus-aureus"><i class="fa fa-check"></i><b>6.3</b> Staphylococcus aureus</a></li>
<li class="chapter" data-level="6.4" data-path="division.html"><a href="division.html#tetrasphaera-remsis"><i class="fa fa-check"></i><b>6.4</b> Tetrasphaera remsis</a></li>
<li class="chapter" data-level="6.5" data-path="division.html"><a href="division.html#idiomarina-loihiensis"><i class="fa fa-check"></i><b>6.5</b> Idiomarina loihiensis</a><ul>
<li class="chapter" data-level="6.5.1" data-path="division.html"><a href="division.html#sphingopyxis-alaskensis-sphingopyxis_alaskensis_divisionthis-diderm-cell-is-about-halfway-through-cytokinesis.-the-two-daughter-cells-are-still-connected-by-a-wide-bridge-of-cytoplasm-but-the-inner-membrane-and-cell-wall-are-drawing-in-all-around."><i class="fa fa-check"></i><b>6.5.1</b> Sphingopyxis alaskensis {#Sphingopyxis_alaskensis_division}This diderm cell is about halfway through cytokinesis. The two daughter cells are still connected by a wide bridge of cytoplasm, but the inner membrane and cell wall are drawing in all around.</a></li>
<li class="chapter" data-level="6.5.2" data-path="division.html"><a href="division.html#chloroflexi-chloroflexi_divisionthis-diderm-cell-is-in-the-final-stage-of-cytokinesis.-the-cytoplasm-of-the-two-daughter-cells-is-completely-separated-by-inner-membrane-and-cell-wall.-the-outer-membrane-though-has-not-yet-divided."><i class="fa fa-check"></i><b>6.5.2</b> Chloroflexi {#Chloroflexi_division}This diderm cell is in the final stage of cytokinesis. The cytoplasm of the two daughter cells is completely separated by inner membrane and cell wall. The outer membrane, though, has not yet divided.</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="division.html"><a href="division.html#caulobacter-crescentus-5"><i class="fa fa-check"></i><b>6.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="6.6.1" data-path="division.html"><a href="division.html#caulobacter-crescentus-caulobacter_divisionthese-daughter-cells-have-completely-separated-their-mothers-inner-membrane-and-cell-wall-but-not-yet-the-outer-membrane-and-surface-layer."><i class="fa fa-check"></i><b>6.6.1</b> Caulobacter crescentus {#Caulobacter_division}These daughter cells have completely separated their mother’s inner membrane and cell wall, but not yet the outer membrane and surface layer.</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="division.html"><a href="division.html#agrobacterium-tumefaciens-1"><i class="fa fa-check"></i><b>6.7</b> Agrobacterium tumefaciens</a></li>
<li class="chapter" data-level="6.8" data-path="division.html"><a href="division.html#cupriavidus-necator-3"><i class="fa fa-check"></i><b>6.8</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="6.8.1" data-path="division.html"><a href="division.html#cupriavidus-necator-asymmetric_constrictionthis-c.-necator-cell-is-at-a-late-stage-of-cytokinesis.-in-most-cells-constriction-would-be-more-or-less-uniform-around-the-circumference-by-now-but-in-some-cells-including-this-one-the-asymmetry-persists."><i class="fa fa-check"></i><b>6.8.1</b> Cupriavidus necator {#Asymmetric_constriction}This <em>C. necator</em> cell is at a late stage of cytokinesis. In most cells, constriction would be more or less uniform around the circumference by now, but in some cells, including this one, the asymmetry persists.</a></li>
</ul></li>
<li class="chapter" data-level="6.9" data-path="division.html"><a href="division.html#schematicftsz-structure"><i class="fa fa-check"></i><b>6.9</b> 9[Schematic–FtsZ structure]</a></li>
<li class="chapter" data-level="6.10" data-path="division.html"><a href="division.html#caulobacter-crescentus-6"><i class="fa fa-check"></i><b>6.10</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="6.11" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius"><i class="fa fa-check"></i><b>6.11</b> Sulfolobus acidocaldarius</a></li>
<li class="chapter" data-level="6.12" data-path="division.html"><a href="division.html#schematicescrt-structure"><i class="fa fa-check"></i><b>6.12</b> 11[Schematic–ESCRT structure]</a></li>
<li class="chapter" data-level="6.13" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius-1"><i class="fa fa-check"></i><b>6.13</b> Sulfolobus acidocaldarius</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="motility.html"><a href="motility.html"><i class="fa fa-check"></i><b>7</b> Motility</a><ul>
<li class="chapter" data-level="7.1" data-path="motility.html"><a href="motility.html#schematicflagellum-structure"><i class="fa fa-check"></i><b>7.1</b> 1[Schematic–Flagellum structure]</a></li>
<li class="chapter" data-level="7.2" data-path="motility.html"><a href="motility.html#bdellovibrio-bacteriovorus-2"><i class="fa fa-check"></i><b>7.2</b> Bdellovibrio bacteriovorus</a><ul>
<li class="chapter" data-level="7.2.1" data-path="motility.html"><a href="motility.html#bdellovibrio-bacteriovorus-flagellar_motor_structurethis-is-an-average-of-the-flagellar-motors-from-more-than-one-thousand-bdellovibrio-bacteriovorus-cells.-working-upward-from-the-base-the-major-parts-of-the-rotor-are-the-c-ring-for-_c_ytoplasmic-the-ms-ring-for-_m_embrane-and-_s_upramembrane-the-rod-the-hook-and-finally-the-filament.-they-are-surrounded-by-stationary-parts-the-stator-ring-which-is-dynamic-so-we-cannot-resolve-the-stators-as-they-cross-the-inner-membrane-and-connect-to-the-c-ring-and-a-series-of-bushings-that-allow-rotation-within-the-cell-wall-the-_p_eriplasmic-or-p-ring-and-outer-membrane-_l_ipopolysaccharide-or-l-ring.-additional-cytoplasmic-components-form-the-export-apparatus-which-is-involved-in-assembly-discussed-on-the-next-page."><i class="fa fa-check"></i><b>7.2.1</b> Bdellovibrio bacteriovorus {#Flagellar_motor_structure}This is an average of the flagellar motors from more than one thousand <em>Bdellovibrio bacteriovorus</em> cells. Working upward from the base, the major parts of the rotor are the C-ring (for _C_ytoplasmic), the MS-ring (for _M_embrane and _S_upramembrane), the rod, the hook, and finally the filament. They are surrounded by stationary parts: the stator ring (which is dynamic, so we cannot resolve the stators as they cross the inner membrane and connect to the C-ring) and a series of bushings that allow rotation within the cell wall (the _P_eriplasmic or P-ring) and outer membrane (_L_ipopolysaccharide or L-ring). Additional cytoplasmic components form the export apparatus, which is involved in assembly (discussed on the next page).</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="motility.html"><a href="motility.html#schematicflagellar-motor-diversity"><i class="fa fa-check"></i><b>7.3</b> 2[Schematic–Flagellar motor diversity]</a></li>
<li class="chapter" data-level="7.4" data-path="motility.html"><a href="motility.html#hylemonella-gracilis-1"><i class="fa fa-check"></i><b>7.4</b> Hylemonella gracilis</a><ul>
<li class="chapter" data-level="7.4.1" data-path="motility.html"><a href="motility.html#pseudomonas-aeruginosa-flagellar_motor_disassemblywhen-flagella-break-or-in-some-species-like-caulobacter-crescentus-are-ejected-so-the-cell-can-attach-to-a-surface-the-motor-is-disassembled-usually-beginning-with-the-export-apparatus.-not-everything-is-dismantled-though-the-p--and-l-rings-remain-in-place-in-the-cell-wall-and-outer-membrane-respectively-as-you-can-see-in-this-pseudomonas-aeruginosa-cell.-we-do-not-know-if-the-cell-has-a-reason-for-leaving-them-there-but-one-possibility-is-that-they-may-function-as-a-plug-for-the-outer-membrane-when-the-hook-is-no-longer-present."><i class="fa fa-check"></i><b>7.4.1</b> Pseudomonas aeruginosa {#Flagellar_motor_disassembly}When flagella break (or in some species, like <em>Caulobacter crescentus</em>, are ejected so the cell can attach to a surface), the motor is disassembled, usually beginning with the export apparatus. Not everything is dismantled, though; the P- and L-rings remain in place in the cell wall and outer membrane, respectively, as you can see in this <em>Pseudomonas aeruginosa</em> cell. We do not know if the cell has a reason for leaving them there, but one possibility is that they may function as a plug for the outer membrane when the hook is no longer present.</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="motility.html"><a href="motility.html#campylobacter-jejuni-4"><i class="fa fa-check"></i><b>7.5</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="7.5.1" data-path="motility.html"><a href="motility.html#helicobacter-pylori-lophotrichous_bacteriamany-lophotrichous-species-like-the-hylemonella-gracilis-you-saw-in-chapter-3length-or-this-helicobacter-pylori-have-a-tuft-of-flagella-at-their-cell-pole.-in-some-species-though-the-tuft-is-located-elsewhere-for-example-a-clump-of-flagella-on-the-concave-side-of-banana-shaped-selenomonas-artemidis-pushes-the-cells-sideways-in-a-seesawing-swimming-pattern."><i class="fa fa-check"></i><b>7.5.1</b> Helicobacter pylori {#Lophotrichous_bacteria}Many lophotrichous species, like the <em>Hylemonella gracilis</em> you saw in Chapter 3–Length or this <em>Helicobacter pylori</em>, have a tuft of flagella at their cell pole. In some species, though, the tuft is located elsewhere; for example, a clump of flagella on the concave side of banana-shaped <em>Selenomonas artemidis</em> pushes the cells sideways in a seesawing swimming pattern.</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="motility.html"><a href="motility.html#pseudomonas-flexibilis"><i class="fa fa-check"></i><b>7.6</b> Pseudomonas flexibilis</a><ul>
<li class="chapter" data-level="7.6.1" data-path="motility.html"><a href="motility.html#proteus-mirabilis-proteus_mirabilis_flagellaproteus-mirabilis-adapt-their-motility-machinery-to-their-environment.-in-liquid-the-short-rod-shaped-cells-swim-with-the-help-of-a-handful-of-flagella-distributed-peritrichously-around-their-cell-body.-when-they-encounter-a-solid-surface-the-cells-elongate-and-build-many-more-flagella-as-you-can-see-on-this-cell.-instead-of-swimming-they-now-use-their-flagella-to-propel-themselves-in-groups-across-the-surface-a-motility-mode-known-as-swarming.-this-is-an-example-of-a-differentiated-lifecycle-which-will-come-up-again-in-chapter-8."><i class="fa fa-check"></i><b>7.6.1</b> Proteus mirabilis {#Proteus_mirabilis_flagella}<em>Proteus mirabilis</em> adapt their motility machinery to their environment. In liquid, the short rod-shaped cells swim with the help of a handful of flagella distributed peritrichously around their cell body. When they encounter a solid surface, the cells elongate and build many more flagella, as you can see on this cell. Instead of swimming, they now use their flagella to propel themselves in groups across the surface, a motility mode known as “swarming.” This is an example of a differentiated lifecycle, which will come up again in Chapter 8.</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="motility.html"><a href="motility.html#helicobacter-hepaticus"><i class="fa fa-check"></i><b>7.7</b> Helicobacter hepaticus</a></li>
<li class="chapter" data-level="7.8" data-path="motility.html"><a href="motility.html#borrelia-burgdorferi"><i class="fa fa-check"></i><b>7.8</b> Borrelia burgdorferi</a><ul>
<li class="chapter" data-level="7.8.1" data-path="motility.html"><a href="motility.html#treponema-primitia-treponema_primitiatreponema-primitia-like-this-are-commensal-residents-of-the-termite-gut-helping-break-down-cellulose.-in-addition-to-two-periplasmic-flagella-the-cells-have-arrays-of-bowl--and-hook-like-structures-on-their-surface-the-function-of-which-likely-related-to-motility-remains-mysterious."><i class="fa fa-check"></i><b>7.8.1</b> Treponema primitia {#Treponema_primitia}<em>Treponema primitia</em> like this are commensal residents of the termite gut, helping break down cellulose. In addition to two periplasmic flagella, the cells have arrays of bowl- and hook-like structures on their surface, the function of which, likely related to motility, remains mysterious.</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="motility.html"><a href="motility.html#methanoregula-formicica"><i class="fa fa-check"></i><b>7.9</b> Methanoregula formicica</a></li>
<li class="chapter" data-level="7.10" data-path="motility.html"><a href="motility.html#schematic-archaellum-structure"><i class="fa fa-check"></i><b>7.10</b> 8[Schematic: Archaellum structure]</a></li>
<li class="chapter" data-level="7.11" data-path="motility.html"><a href="motility.html#schematic-archaellar-motor-structure"><i class="fa fa-check"></i><b>7.11</b> 8[Schematic: Archaellar motor structure]</a></li>
<li class="chapter" data-level="7.12" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis"><i class="fa fa-check"></i><b>7.12</b> Thermococcus kodakaraensis</a><ul>
<li class="chapter" data-level="7.12.1" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis-cone_structurethe-archaellar-plate-does-not-come-to-a-point-at-the-tip-but-rather-is-a-conical-frustum-open-at-the-top-resembling-a-lampshade.-in-the-center-of-the-tip-is-a-small-ring-as-you-can-see-in-a-top-view-in-this-lysed-flattened-thermococcus-kodakaraensis-cell.-the-function-of-the-ring-remains-unknown-perhaps-it-helps-nucleate-the-rest-of-the-structure"><i class="fa fa-check"></i><b>7.12.1</b> Thermococcus kodakaraensis {#Cone_structure}The archaellar plate does not come to a point at the tip, but rather is a conical frustum (open at the top), resembling a lampshade. In the center of the tip is a small ring, as you can see in a top view in this lysed, flattened <em>Thermococcus kodakaraensis</em> cell. The function of the ring remains unknown; perhaps it helps nucleate the rest of the structure?</a></li>
<li class="chapter" data-level="7.12.2" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis-organizing_centeras-you-can-see-more-clearly-in-this-partially-lysed-flattened-thermococcus-kodakaraensis-cell-the-conical-plate-in-the-cytoplasm-is-attached-to-more-than-just-the-archaella.-it-is-also-associated-with-chemosensory-arrays-discussed-in-the-next-chapter-and-dna-as-you-can-see-from-the-ribosome-excluding-zone.-this-structure-may-therefore-perform-an-analogous-function-to-bacterial-organizing-proteins-such-as-popz-tethering-cellular-components-into-a-de-facto-pole-for-the-in-this-case-round-cell."><i class="fa fa-check"></i><b>7.12.2</b> Thermococcus kodakaraensis {#Organizing_center}As you can see more clearly in this partially-lysed, flattened <em>Thermococcus kodakaraensis</em> cell, the conical plate in the cytoplasm is attached to more than just the archaella. It is also associated with chemosensory arrays (discussed in the next chapter) and DNA, as you can see from the ribosome-excluding zone. This structure may therefore perform an analogous function to bacterial organizing proteins such as PopZ, tethering cellular components into a <em>de facto</em> pole for the (in this case round) cell.</a></li>
</ul></li>
<li class="chapter" data-level="7.13" data-path="motility.html"><a href="motility.html#myxococcus-xanthus-1"><i class="fa fa-check"></i><b>7.13</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="7.14" data-path="motility.html"><a href="motility.html#schematictype-iv-pilus-structure"><i class="fa fa-check"></i><b>7.14</b> 10[Schematic–Type IV pilus structure]</a></li>
<li class="chapter" data-level="7.15" data-path="motility.html"><a href="motility.html#flavobacterium-johnsoniae"><i class="fa fa-check"></i><b>7.15</b> Flavobacterium johnsoniae</a></li>
<li class="chapter" data-level="7.16" data-path="motility.html"><a href="motility.html#mycoplasma-pneumoniae"><i class="fa fa-check"></i><b>7.16</b> Mycoplasma pneumoniae</a></li>
<li class="chapter" data-level="7.17" data-path="motility.html"><a href="motility.html#schematicterminal-organelle-structure"><i class="fa fa-check"></i><b>7.17</b> 12[ Schematic–Terminal organelle structure]</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="navigation.html"><a href="navigation.html"><i class="fa fa-check"></i><b>8</b> Navigation</a><ul>
<li class="chapter" data-level="8.1" data-path="navigation.html"><a href="navigation.html#schematicchemoreceptor-structure"><i class="fa fa-check"></i><b>8.1</b> 1[Schematic–Chemoreceptor structure]</a></li>
<li class="chapter" data-level="8.2" data-path="navigation.html"><a href="navigation.html#styphimurium"><i class="fa fa-check"></i><b>8.2</b> Styphimurium</a></li>
<li class="chapter" data-level="8.3" data-path="navigation.html"><a href="navigation.html#schematic-chemosensory-array-architecture"><i class="fa fa-check"></i><b>8.3</b> 2[Schematic: Chemosensory array architecture]</a></li>
<li class="chapter" data-level="8.4" data-path="navigation.html"><a href="navigation.html#methanospirillum-hungatei-1"><i class="fa fa-check"></i><b>8.4</b> Methanospirillum hungatei</a></li>
<li class="chapter" data-level="8.5" data-path="navigation.html"><a href="navigation.html#schematicconservation-of-chemosensory-array-architecture"><i class="fa fa-check"></i><b>8.5</b> 3[Schematic–Conservation of chemosensory array architecture]</a></li>
<li class="chapter" data-level="8.6" data-path="navigation.html"><a href="navigation.html#vibrio-cholerae"><i class="fa fa-check"></i><b>8.6</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="8.7" data-path="navigation.html"><a href="navigation.html#aa-length-difference-of-as-little-as-2-nm-is-enough-to-separate-receptors-like-these-in-azospirillum-brasilense-that-sense-oxygen-28-nm-array-and-sources-of-energy-like-malate-30-nm-array.-both-receptors-send-signals-to-the-single-flagellar-motor-promoting-runs-or-direction-switches-to-guide-the-bacterium-through-a-combination-of-chemotaxis-and-aerotaxis-ordered-movement-in-an-oxygen-gradient-to-its-target-plant-roots.-a.-brasilense-fixes-nitrogen-boosting-the-growth-of-plants-it-colonizes."><i class="fa fa-check"></i><b>8.7</b> 4aA length difference of as little as 2 nm is enough to separate receptors, like these in <em>Azospirillum brasilense</em> that sense oxygen (28 nm array) and sources of energy like malate (30 nm array). Both receptors send signals to the single flagellar motor, promoting runs or direction switches to guide the bacterium through a combination of chemotaxis and aerotaxis (ordered movement in an oxygen gradient) to its target: plant roots. <em>A. brasilense</em> fixes nitrogen, boosting the growth of plants it colonizes.</a></li>
<li class="chapter" data-level="8.8" data-path="navigation.html"><a href="navigation.html#methanoregula-formicica-1"><i class="fa fa-check"></i><b>8.8</b> Methanoregula formicica</a></li>
<li class="chapter" data-level="8.9" data-path="navigation.html"><a href="navigation.html#schematiccytoplasmic-chemosensory-array-structure"><i class="fa fa-check"></i><b>8.9</b> 5[Schematic–Cytoplasmic chemosensory array structure]</a></li>
<li class="chapter" data-level="8.10" data-path="navigation.html"><a href="navigation.html#athis-vibrio-cholerae-cell-showcases-a-full-repertoire-of-chemosensory-systems-two-membrane-embedded-arrays-the-shorter-one-signaling-to-the-flagellar-motor-and-the-longer-one-signaling-to-a-still-unknown-target-and-a-cytoplasmic-array-also-with-unknown-function.-in-all-cases-the-basic-architecture-is-identical-hexagonally-arranged-trimers-of-chemoreceptor-dimers."><i class="fa fa-check"></i><b>8.10</b> 5aThis <em>Vibrio cholerae</em> cell showcases a full repertoire of chemosensory systems: two membrane-embedded arrays (the shorter one signaling to the flagellar motor and the longer one signaling to a still-unknown target) and a cytoplasmic array (also with unknown function). In all cases, the basic architecture is identical: hexagonally-arranged trimers of chemoreceptor dimers.</a></li>
<li class="chapter" data-level="8.11" data-path="navigation.html"><a href="navigation.html#magnetospirillum-magneticum"><i class="fa fa-check"></i><b>8.11</b> Magnetospirillum magneticum</a></li>
<li class="chapter" data-level="8.12" data-path="navigation.html"><a href="navigation.html#schematicmamk-structure"><i class="fa fa-check"></i><b>8.12</b> 6[Schematic–MamK structure]</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="lifecycle.html"><a href="lifecycle.html"><i class="fa fa-check"></i><b>9</b> Lifecycle</a><ul>
<li class="chapter" data-level="9.1" data-path="lifecycle.html"><a href="lifecycle.html#helicobacter-pylori"><i class="fa fa-check"></i><b>9.1</b> Helicobacter pylori</a></li>
<li class="chapter" data-level="9.2" data-path="lifecycle.html"><a href="lifecycle.html#thermococcus-kodakaraensis-1"><i class="fa fa-check"></i><b>9.2</b> Thermococcus kodakaraensis</a></li>
<li class="chapter" data-level="9.3" data-path="lifecycle.html"><a href="lifecycle.html#caulobacter-crescentus-7"><i class="fa fa-check"></i><b>9.3</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="9.3.1" data-path="lifecycle.html"><a href="lifecycle.html#prosthecobacter-vanneervenii-holdfastprosthecobacter-vanneervenii-is-a-close-relative-of-caulobacter-crescentus-and-attaches-to-surfaces-by-the-same-mechanism-a-sticky-ball-of-holdfast-at-the-tip-of-its-stalk.-here-you-see-four-cells-whose-stalks-have-attached-to-one-another.-unlike-its-cousin-c.-crescentus-however-p.-vanneervenii-is-non-motile-no-flagellum-and-does-not-have-a-dimorphic-life-cycle.-it-divides-symmetrically-to-produce-two-daughters-each-with-their-own-stalk."><i class="fa fa-check"></i><b>9.3.1</b> Prosthecobacter vanneervenii {#Holdfast}<em>Prosthecobacter vanneervenii</em> is a close relative of Caulobacter crescentus, and attaches to surfaces by the same mechanism: a sticky ball of holdfast at the tip of its stalk. Here you see four cells whose stalks have attached to one another. Unlike its cousin <em>C. crescentus</em>, however, <em>P. vanneervenii</em> is non-motile (no flagellum) and does not have a dimorphic life cycle. It divides symmetrically to produce two daughters, each with their own stalk.</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis"><i class="fa fa-check"></i><b>9.4</b> Bacillus subtilis</a><ul>
<li class="chapter" data-level="9.4.1" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-sporulation_septum_formationfrom-hundreds-of-sporulating-bacillus-subtilis-we-imaged-by-cryoet-this-is-the-only-cell-captured-in-the-process-of-forming-a-septum.-as-you-can-see-it-is-at-a-very-early-stage-in-the-process.-the-fact-that-we-do-not-see-more-intermediates-suggests-that-once-the-septum-starts-to-form-it-quickly-proceeds-to-completion."><i class="fa fa-check"></i><b>9.4.1</b> Bacillus subtilis {#Sporulation_septum_formation}From hundreds of sporulating <em>Bacillus subtilis</em> we imaged by cryoET, this is the only cell captured in the process of forming a septum. As you can see, it is at a very early stage in the process. The fact that we do not see more intermediates suggests that once the septum starts to form, it quickly proceeds to completion.</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-1"><i class="fa fa-check"></i><b>9.5</b> Bacillus subtilis</a><ul>
<li class="chapter" data-level="9.5.1" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-spore_engulfmentthe-process-of-spore-engulfment-by-the-mother-cell-membrane-culminates-in-closure-of-the-membranes-creating-an-outer-spore-membrane-and-an-inner-spore-membrane-on-either-side-of-the-sporulation-septum.-the-final-point-of-separation-is-usually-at-the-center-of-the-cell-pole-but-not-always-as-you-can-see-in-this-bacillus-subtilis-spore-being-engulfed-from-the-side-of-the-mother-cell.-the-layer-to-the-right-of-the-developing-spore-may-be-a-precursor-of-the-coat-morespore-coating."><i class="fa fa-check"></i><b>9.5.1</b> Bacillus subtilis {#Spore_engulfment}The process of spore engulfment by the mother cell membrane culminates in closure of the membranes, creating an outer spore membrane and an inner spore membrane on either side of the sporulation septum. The final point of separation is usually at the center of the cell pole, but not always, as you can see in this <em>Bacillus subtilis</em> spore being engulfed from the side of the mother cell. The layer to the right of the developing spore may be a precursor of the coat [More–Spore coating].</a></li>
<li class="chapter" data-level="9.5.2" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-spore_coatingin-this-later-stage-of-sporulation-you-can-see-that-the-bacillus-subtilis-spore-has-now-separated-completely-from-the-mother-cell-envelope.-it-has-also-moved-away-from-the-cell-pole-this-sometimes-occurs-even-as-the-envelopes-are-still-attached-morespore-engulfment.-the-layer-flanking-the-spore-on-the-right-may-be-the-assembling-spore-coat."><i class="fa fa-check"></i><b>9.5.2</b> Bacillus subtilis {#Spore_coating}In this later stage of sporulation, you can see that the <em>Bacillus subtilis</em> spore has now separated completely from the mother cell envelope. It has also moved away from the cell pole (this sometimes occurs even as the envelopes are still attached [More–Spore engulfment]). The layer flanking the spore on the right may be the assembling spore coat.</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-2"><i class="fa fa-check"></i><b>9.6</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="9.7" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-3"><i class="fa fa-check"></i><b>9.7</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="9.8" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-1"><i class="fa fa-check"></i><b>9.8</b> Acetonema longum</a></li>
<li class="chapter" data-level="9.9" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-2"><i class="fa fa-check"></i><b>9.9</b> Acetonema longum</a><ul>
<li class="chapter" data-level="9.9.1" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-complete_engulfmentthis-acetonema-longum-cell-has-completed-engulfment-creating-a-forespore.-acetonema-longum-forespores-and-mature-spores-typically-contain-several-granules-likely-storing-poly-phosphate.-these-may-help-power-outgrowth-when-the-spore-germinates.-spores-from-other-species-though-like-bacillus-subtilis-do-not-contain-obvious-storage-granules-so-they-may-serve-another-species-specific-role."><i class="fa fa-check"></i><b>9.9.1</b> Acetonema longum {#Complete_engulfment}This <em>Acetonema longum</em> cell has completed engulfment, creating a “forespore.” <em>Acetonema longum</em> forespores (and mature spores) typically contain several granules, likely storing poly-phosphate. These may help power outgrowth when the spore germinates. Spores from other species, though, like <em>Bacillus subtilis</em>, do not contain obvious storage granules so they may serve another, species-specific role.</a></li>
<li class="chapter" data-level="9.9.2" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-mature_sporethis-mature-acetonema-longum-spore-has-been-released-by-lysis-of-its-mother-cell.-note-its-extensive-armor-a-thick-cortex-between-the-membranes-and-a-multilayered-protein-coat."><i class="fa fa-check"></i><b>9.9.2</b> Acetonema longum {#Mature_spore}This mature <em>Acetonema longum</em> spore has been released by lysis of its mother cell. Note its extensive armor: a thick cortex between the membranes, and a multilayered protein coat.</a></li>
<li class="chapter" data-level="9.9.3" data-path="lifecycle.html"><a href="lifecycle.html#sporomusa-acidovorans-sporomusa_acidovorans_sporulationthis-diderm-sporomusa-acidovorans-cell-is-in-the-process-of-forming-an-ovoid-endospore.-note-also-what-may-be-an-early-stage-of-cortex-formation-at-the-end-of-the-spore-as-well-as-novel-structures-at-the-points-of-cell-wall-remodeling.-we-do-not-know-what-they-are-but-they-may-be-involved-in-engulfment."><i class="fa fa-check"></i><b>9.9.3</b> Sporomusa acidovorans {#Sporomusa_acidovorans_sporulation}This diderm <em>Sporomusa acidovorans</em> cell is in the process of forming an ovoid endospore. Note also what may be an early stage of cortex formation at the end of the spore, as well as novel structures at the points of cell wall remodeling. We do not know what they are, but they may be involved in engulfment.</a></li>
</ul></li>
<li class="chapter" data-level="9.10" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-3"><i class="fa fa-check"></i><b>9.10</b> Acetonema longum</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="interaction.html"><a href="interaction.html"><i class="fa fa-check"></i><b>10</b> Interaction</a><ul>
<li class="chapter" data-level="10.1" data-path="interaction.html"><a href="interaction.html#gluconacetobacter-hansenii"><i class="fa fa-check"></i><b>10.1</b> Gluconacetobacter hansenii</a></li>
<li class="chapter" data-level="10.2" data-path="interaction.html"><a href="interaction.html#legionella-pneumophila"><i class="fa fa-check"></i><b>10.2</b> Legionella pneumophila</a></li>
<li class="chapter" data-level="10.3" data-path="interaction.html"><a href="interaction.html#schematiclegionella-t2ss-structure"><i class="fa fa-check"></i><b>10.3</b> 3[Schematic–Legionella T2SS structure]</a></li>
<li class="chapter" data-level="10.4" data-path="interaction.html"><a href="interaction.html#schematiclegionella-t4ss-structure"><i class="fa fa-check"></i><b>10.4</b> 3[Schematic–Legionella T4SS structure]</a></li>
<li class="chapter" data-level="10.5" data-path="interaction.html"><a href="interaction.html#athis-helicobacter-pylori-cell-was-cultured-together-with-human-gastric-epithelial-cells-its-target-for-infection.-it-contains-inactive-type-iv-secretion-systems-as-well-as-several-mysterious-tubes.-the-tubes-are-formed-from-the-outer-membrane-but-we-do-not-know-what-scaffolds-them-they-have-a-constant-diameter-of-37-nm-or-what-makes-the-portals-along-their-length.-the-fact-that-we-sometimes-see-tubes-near-primed-t4sss-and-the-fact-that-the-tubes-are-missing-from-mutant-cells-lacking-t4ss-components-suggests-a-relationship-but-the-details-are-unclear.-much-remains-mysterious-about-this-structure-and-its-potential-role-in-infection."><i class="fa fa-check"></i><b>10.5</b> 3aThis <em>Helicobacter pylori</em> cell was cultured together with human gastric epithelial cells, its target for infection. It contains inactive type IV secretion systems, as well as several mysterious tubes. The tubes are formed from the outer membrane, but we do not know what scaffolds them (they have a constant diameter of 37 nm), or what makes the portals along their length. The fact that we sometimes see tubes near primed T4SSs, and the fact that the tubes are missing from mutant cells lacking T4SS components suggests a relationship, but the details are unclear. Much remains mysterious about this structure and its potential role in infection.</a></li>
<li class="chapter" data-level="10.6" data-path="interaction.html"><a href="interaction.html#pseudomonas-aeruginosa-δexsd"><i class="fa fa-check"></i><b>10.6</b> Pseudomonas aeruginosa ΔexsD</a></li>
<li class="chapter" data-level="10.7" data-path="interaction.html"><a href="interaction.html#a-emd-2667"><i class="fa fa-check"></i><b>10.7</b> 4a EMD-2667</a></li>
<li class="chapter" data-level="10.8" data-path="interaction.html"><a href="interaction.html#b"><i class="fa fa-check"></i><b>10.8</b> 4b</a></li>
<li class="chapter" data-level="10.9" data-path="interaction.html"><a href="interaction.html#escherichia-coli-cdia"><i class="fa fa-check"></i><b>10.9</b> Escherichia coli +CdiA</a></li>
<li class="chapter" data-level="10.10" data-path="interaction.html"><a href="interaction.html#schematiccdi-mechanism"><i class="fa fa-check"></i><b>10.10</b> 5[Schematic–CDI mechanism]</a></li>
<li class="chapter" data-level="10.11" data-path="interaction.html"><a href="interaction.html#myxococcus-xanthus-2"><i class="fa fa-check"></i><b>10.11</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="10.12" data-path="interaction.html"><a href="interaction.html#schematicprimed-t6ss-structure"><i class="fa fa-check"></i><b>10.12</b> 6[Schematic–Primed T6SS structure]</a></li>
<li class="chapter" data-level="10.13" data-path="interaction.html"><a href="interaction.html#vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.13</b> Vibrio cholerae</a><ul>
<li class="chapter" data-level="10.13.1" data-path="interaction.html"><a href="interaction.html#amoebophilus-asiaticus-t6ss_batterysome-cells-assemble-a-few-scattered-type-vi-secretion-systems.-other-species-like-this-amoebophilus-asiaticus-build-ordered-arrays-of-parallel-machines.-these-diderm-bacteria-live-symbiotically-inside-amoebae.-their-t6ss-batteries-help-them-colonize-their-host-perhaps-by-rupturing-the-membrane-of-the-host-phagosome-that-encloses-them-after-they-are-engulfed-releasing-them-into-the-host-cytoplasm.-the-number-of-t6sss-in-each-array-varies-but-can-reach-a-few-dozen."><i class="fa fa-check"></i><b>10.13.1</b> Amoebophilus asiaticus {#T6SS_battery}Some cells assemble a few, scattered type VI secretion systems. Other species, like this <em>Amoebophilus asiaticus</em> build ordered arrays of parallel machines. These diderm bacteria live symbiotically inside amoebae. Their T6SS batteries help them colonize their host, perhaps by rupturing the membrane of the host phagosome that encloses them after they are engulfed, releasing them into the host cytoplasm. The number of T6SSs in each array varies, but can reach a few dozen.</a></li>
<li class="chapter" data-level="10.13.2" data-path="interaction.html"><a href="interaction.html#pseudoalteromonas-luteoviolacea-δt6ssbacteriocin-macspseudoalteromonas-luteoviolacea-produce-massive-arrays-of-macs-contractile-machines-related-to-pyocins.-cells-lyse-to-release-the-arrays-like-the-one-you-see-here-which-can-contain-100-individual-macs-and-extend-more-than-1-µm-across.-the-arrays-are-highly-ordered-with-the-tail-fibers-of-the-macs-networked-into-a-hexagonal-array-that-may-help-synchronize-firing-schematicmac-array-structure.-mac-arrays-serve-an-important-function-for-larvae-of-the-marine-tubeworm-hydroides-elegans-and-likely-the-larvae-of-other-invertebrates-such-as-sea-urchins-and-corals-as-well.-they-signal-to-the-larva-that-it-is-the-right-place-to-settle-down-and-differentiate-into-a-sessile-surface-attached-adult.-the-function-of-mac-arrays-for-the-bacterium-is-less-clear.-it-may-be-that-the-invertebrates-benefit-the-lysed-cells-surviving-relatives-in-the-biofilm.-or-perhaps-the-macs-play-another-completely-unrelated-role."><i class="fa fa-check"></i><b>10.13.2</b> Pseudoalteromonas luteoviolacea ΔT6SS/bacteriocin {#MACs}<em>Pseudoalteromonas luteoviolacea</em> produce massive arrays of MACs, contractile machines related to pyocins. Cells lyse to release the arrays, like the one you see here, which can contain 100 individual MACs and extend more than 1 µm across. The arrays are highly ordered, with the tail fibers of the MACs networked into a hexagonal array that may help synchronize firing [Schematic–MAC array structure]. MAC arrays serve an important function for larvae of the marine tubeworm <em>Hydroides elegans</em>, and likely the larvae of other invertebrates such as sea urchins and corals as well. They signal to the larva that it is the right place to settle down and differentiate into a sessile (surface-attached) adult. The function of MAC arrays for the bacterium is less clear. It may be that the invertebrates benefit the lysed cell’s surviving relatives in the biofilm. Or perhaps the MACs play another, completely unrelated role.</a></li>
</ul></li>
<li class="chapter" data-level="10.14" data-path="interaction.html"><a href="interaction.html#schematicmac-array-structure"><i class="fa fa-check"></i><b>10.14</b> 7[Schematic–MAC array structure]</a></li>
<li class="chapter" data-level="10.15" data-path="interaction.html"><a href="interaction.html#porphyromonas-gingivalis"><i class="fa fa-check"></i><b>10.15</b> Porphyromonas gingivalis</a></li>
<li class="chapter" data-level="10.16" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-vibrio-cholerae"><i class="fa fa-check"></i><b>10.16</b> Bdellovibrio bacteriovorus + Vibrio cholerae</a></li>
<li class="chapter" data-level="10.17" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-escherichia-coli-minicell"><i class="fa fa-check"></i><b>10.17</b> Bdellovibrio bacteriovorus + Escherichia coli minicell</a><ul>
<li class="chapter" data-level="10.17.1" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-escherichia-coli-minicell-bdellovibrio_bacteriovorus_entrywe-still-do-not-know-exactly-how-bdellovibrio-bacteriovorus-enters-its-host-but-the-process-is-extremely-rapid.-in-order-to-catch-a-snapshot-we-mixed-b.-bacteriovorus-with-miniaturized-prey-see-details-on-the-main-page.-as-the-b.-bacteriovorus-cell-cannot-fit-inside-the-prey-it-remains-stuck-half-in-and-half-out.-note-the-tight-collar-of-host-outer-membrane-around-the-invader.-how-this-seal-is-maintained-throughout-invasion-remains-a-mystery."><i class="fa fa-check"></i><b>10.17.1</b> Bdellovibrio bacteriovorus + Escherichia coli minicell {#Bdellovibrio_bacteriovorus_entry}We still do not know exactly how <em>Bdellovibrio bacteriovorus</em> enters its host, but the process is extremely rapid. In order to catch a snapshot, we mixed <em>B. bacteriovorus</em> with miniaturized prey (see details on the main page). As the <em>B. bacteriovorus</em> cell cannot fit inside the prey, it remains stuck half-in and half-out. Note the tight collar of host outer membrane around the invader. How this seal is maintained throughout invasion remains a mystery.</a></li>
</ul></li>
<li class="chapter" data-level="10.18" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.18</b> Bdellovibrio bacteriovorus + Vibrio cholerae</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="viruses.html"><a href="viruses.html"><i class="fa fa-check"></i><b>11</b> Viruses</a><ul>
<li class="chapter" data-level="11.1" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-3"><i class="fa fa-check"></i><b>11.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="11.2" data-path="viruses.html"><a href="viruses.html#bacteroidetes-strain-jt5"><i class="fa fa-check"></i><b>11.2</b> Bacteroidetes strain JT5</a></li>
<li class="chapter" data-level="11.3" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-4"><i class="fa fa-check"></i><b>11.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="11.4" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-2"><i class="fa fa-check"></i><b>11.4</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="11.5" data-path="viruses.html"><a href="viruses.html#escherichia-coli-1"><i class="fa fa-check"></i><b>11.5</b> Escherichia coli</a></li>
<li class="chapter" data-level="11.6" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus-8"><i class="fa fa-check"></i><b>11.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="11.6.1" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus-φcb13_infectionhere-you-see-several-φcb13-phage-that-have-successfully-reached-their-target-receptors-at-the-pole-of-a-caulobacter-crescentus-and-are-in-the-process-of-infecting-the-cell-note-the-mix-of-filled-capsids-that-have-not-yet-released-their-genome-and-empty-capsids-that-have."><i class="fa fa-check"></i><b>11.6.1</b> Caulobacter crescentus {#φCb13_infection}Here you see several φCb13 phage that have successfully reached their target receptors at the pole of a <em>Caulobacter crescentus</em> and are in the process of infecting the cell (note the mix of filled capsids that have not yet released their genome and empty capsids that have).</a></li>
</ul></li>
<li class="chapter" data-level="11.7" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-3"><i class="fa fa-check"></i><b>11.7</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="11.8" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-1"><i class="fa fa-check"></i><b>11.8</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="11.9" data-path="viruses.html"><a href="viruses.html#schematicvap-structure"><i class="fa fa-check"></i><b>11.9</b> 8[Schematic–VAP structure]</a></li>
<li class="chapter" data-level="11.10" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-2"><i class="fa fa-check"></i><b>11.10</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="11.11" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-5"><i class="fa fa-check"></i><b>11.11</b> Shewanella oneidensis</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>12</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.library.caltech.edu" target="blank">Published by Caltech Library</a><li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cell Atlas</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="envelope" class="section level1">
<h1><span class="header-section-number">Chapter 3</span> Envelope</h1>
<p>“It is not a simple life to be a single cell, although I have no right to say so, having been a single cell so long ago myself that I have no memory at all of that stage of my life.” - Lewis Thomas <span class="citation">Lewis Thomas (<a href="#ref-thomas1990">1990</a>)</span></p>
<p>Membrane ## Mycoplasma genitalium</p>
<p>The fundamental unit of life is the cell– a contained self-replicating assembly. For many species, including all bacteria and archaea, the organism consists of a single cell. And for nearly all species, no matter how many cells an organism eventually contains (probably around 10 trillion in your case), it started life as a single cell. The details vary, but every cell on Earth is the same at heart–a DNA-based replicating machine built from just four macromolecules: nucleic acids, proteins, lipids and carbohydrates.</p>
<p>Imagine for a moment that you are a structural engineer tasked with building one of these cells. Where would you start? How about the container? No matter what the first self-replicating molecules were (likely _r_ibo_n_ucleic _a_cid, or RNA), they were not a cell until they acquired a container. You would probably want a flexible container that allowed you to sort certain kinds of molecules from the environment. Evolution agrees. All cells are enclosed by a selectively permeable membrane, made of lipids and proteins [Schematic–Lipid bilayer], that allows them to differentiate their contents from the environment. The selectivity of this membrane is a critical feature for the life of the cell [Schematic–ATP synthase].</p>
<p>Now your cell has a clearly delineated exterior and interior. The interior is called the cytoplasm (“cell mold,” the membrane being the container that shapes the mold). Almost all archaea and many bacteria, like these <em>Mycoplasma genitalium</em> cells, are monoderms (“single skin”). This means that their cytoplasm is enclosed by a single membrane. At this resolution, the membrane looks like a single dark line, but remember that it is really a bilayer, as you will be able to see in some later examples.</p>
<div class="figure"><span id="fig:unnamed-chunk-16"></span>
<div id="htmlwidget-eeda8abbceeaf8b494e1" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-eeda8abbceeaf8b494e1">{"x":{"doi":"10.22002/D1.1350","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.1: Mycoplasma genitalium Collected by: Gregory Henderson <a href="https://doi.org/10.22002/D1.1350">10.22002/D1.1350</a>
</p>
</div>
<div id="schematic-lipid-bilayer" class="section level2">
<h2><span class="header-section-number">3.1</span> 1[Schematic: Lipid bilayer]</h2>
<p>Lipids have a hydrophilic head (red/orange here) and hydrophobic tails (grey); in water they spontaneously pack side-by-side and tail-to-tail to shield their tails from unfavorable interactions with water. This results in closed double-layered bags–membranes. One key difference between archaea and bacteria (and with them, eukaryotes) is the kind of lipid in their membranes. Hybrid membranes containing both these lipid types can be made artificially, and it is possible that the last universal common ancestor of all cells on Earth contained both types, with specialization occurring later.</p>
<p>Membranes also contain many proteins. Some have hydrophobic regions that embed them into the lipid bilayer. Other proteins are fused, or tethered, to the lipids. In fact, cells’ “lipid” membranes are made up of roughly equal parts lipids and proteins.</p>
<div class="figure"><span id="fig:unnamed-chunk-17"></span>
<div id="htmlwidget-0ef3a32b334c60388e5a" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-0ef3a32b334c60388e5a">{"x":{"doi":"10.22002/D1.1351","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.2: Listeria monocytogenes Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1351">10.22002/D1.1351</a>
</p>
</div>
</div>
<div id="schematic-atp-synthase-and-energy-production" class="section level2">
<h2><span class="header-section-number">3.2</span> 1[Schematic: ATP synthase and energy production]</h2>
<p>The chemical properties of lipids make membranes impermeable to ions and large or hydrophilic molecules (but not to water). Cells take advantage of this property to establish an ion gradient across the membrane, using a chain of electron-carrying proteins in the membrane to pump protons out of the cell. Protein complexes in the membrane called ATP synthases (like this one from <em>Escherichia coli</em> <span class="citation">Sobti et al. (<a href="#ref-sobti2016">2016</a>)</span>) use the resulting ion potential to generate energy. The machine provides a conduit for protons to flow down their potential, producing a “proton-motive force” that spins the machine’s rotor, generating energy that is chemically stored in ATP, the energetic currency of the cell. For this reason, we say that the membrane is “energized.” Holes in the membrane allow the ion gradient to equilibrate, destroying the cell’s means of generating energy, and thus its life.</p>
Cell Wall
<div class="figure"><span id="fig:unnamed-chunk-18"></span>
<div id="htmlwidget-ec02f20f3fcdf2fe768f" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-ec02f20f3fcdf2fe768f">{"x":{"doi":"10.22002/D1.1351","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.3: Listeria monocytogenes Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1351">10.22002/D1.1351</a>
</p>
</div>
</div>
<div id="listeria-monocytogenes" class="section level2">
<h2><span class="header-section-number">3.3</span> Listeria monocytogenes</h2>
<p>Being able to selectively move things into your cell enables it to do some powerful things. It also poses a structural problem. Remember that water can pass freely through the membrane, which means that increasing the solute concentration inside relative to the environment outside will cause water to rush in as well, introducing a pressure (known as turgor pressure) on the membrane. Lipid bilayers, unfortunately, are unable to withstand much pressure. If your cell lives exclusively in a constant, and fairly high-osmolarity, environment (like our bodies, in the case of the pathogenic <em>Mycoplasma genitalium</em> you just saw), it can balance internal and external osmolarity to minimize turgor pressure on its membrane. But most cells experience more variable, and dilute, environments. How could you keep your cell from bursting in such conditions?</p>
<p>You might want to add some rigid scaffolding outside the membrane to buttress it against turgor pressure. Nearly all bacteria do this with a material called peptidoglycan: long stiff polymers of glycan sugars crosslinked by short peptides into a chain-mail-like mesh. The full scaffold of this material surrounding the cell is called its sacculus, or cell wall. In monoderm bacteria like this <em>Listeria monocytogenes</em>, the cell wall is significantly thicker than the membrane. It comprises several layers of peptidoglycan, which are indistinguishable at this resolution, so the cell wall appears as a uniformly textured layer [More–Peptidoglycan architecture]. It remains a mystery how large molecules can pass through this dense layer on their way to and from the cell.</p>
<p>While most bacteria build their walls from peptidoglycan, some use slightly different materials [More–Methanobacterium formicicum]. Some archaea also have cell walls, also made of a molecule similar to peptidoglycan, but chemically distinct. Most archaea, though, rely on a different structure for support, which you will see in a few pages.</p>
<p>A wall can be a boon to your cell, but it can also prove a liability in certain conditions. For instance, many antibiotics target the cell wall. Some bacteria have developed the ability to jettison their walls in such conditions, taking on a so-called “L-form” (named for the Lister Institute where it was discovered). L-form bacteria give us a fascinating window into how early cells–prior to the evolution of the cell wall–might have looked and behaved.</p>
<div class="figure"><span id="fig:unnamed-chunk-19"></span>
<div id="htmlwidget-02e6341efb6a06695165" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-02e6341efb6a06695165">{"x":{"doi":"10.22002/D1.1351","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.4: Listeria monocytogenes Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1351">10.22002/D1.1351</a>
</p>
</div>
<div id="Peptidoglycan_architecture" class="section level3">
<h3><span class="header-section-number">3.3.1</span> Bacillus subtilis</h3>
<p>The sacculus is so robust that it persists even after cells are lysed and their other components digested. This sacculus isolated from a <em>Bacillus subtilis</em> cell has retained its shape, simply flattening with the release of contents and pressure from inside. The long glycan strands are oriented in hoops running parallel to the short axis of the cylindrical cell and the short peptide crosslinks are more or less aligned with the long axis.</p>
<div class="figure"><span id="fig:unnamed-chunk-20"></span>
<div id="htmlwidget-7ac4e505ea4f6fd983d3" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-7ac4e505ea4f6fd983d3">{"x":{"doi":"10.22002/D1.1360","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.5: Bacillus subtilis Collected by: Morgan Beeby <a href="https://doi.org/10.22002/D1.1360">10.22002/D1.1360</a>
</p>
</div>
</div>
<div id="Methanobacterium_formicicum" class="section level3">
<h3><span class="header-section-number">3.3.2</span> Methanobacterium formicicum</h3>
<p>These methane-producing bacteria, found in the rumen of animals like cows, build their walls with a polymer that has a slightly different chemical composition than peptidoglycan, but which forms a similar structure, as you can see.</p>
Outer Membrane
<div class="figure"><span id="fig:unnamed-chunk-21"></span>
<div id="htmlwidget-9f9da185fe4cf103bf69" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-9f9da185fe4cf103bf69">{"x":{"doi":"10.22002/D1.1361","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.6: Methanobacterium formicicum Collected by: Elitza Tocheva <a href="https://doi.org/10.22002/D1.1361">10.22002/D1.1361</a>
</p>
</div>
</div>
</div>
<div id="cupriavidus-necator" class="section level2">
<h2><span class="header-section-number">3.4</span> Cupriavidus necator</h2>
<p>Why stop at one membrane? Eukaryotic cells use internal membranes to form specialized compartments like the nucleus and mitochondria. While bacteria lack such organelles, many create an additional compartment <em>outside</em> the cell with a second, outer, membrane. Such bacteria, like the <em>Cupriavidus necator</em> cell you see here, are called diderms (“double skin”). The extra compartment between their membranes is known as the periplasm (“mold between”). This antechamber contains a unique subset of proteins, many of which function in escorting things into and out of the main cell.</p>
<p>Compared to the inner membrane, the outer membrane has some unique properties. It is more permeable and not proton-tight (so it cannot be used to generate ATP). It is often asymmetric, with a different composition of lipids and proteins in each of the two leaflets. A few species of bacteria, particularly pathogens, have labile outer membranes (for an example, check out the <em>Helicobacter pylori</em> in Chapter 8–RecA). Most, however, firmly anchor their outer membrane to the cell wall [Schematic–Braun’s lipoprotein]. Rather than containing many layers of peptidoglycan like in the <em>Listeria monocytogenes</em> cell you just saw, the cell wall of diderms is usually composed of a single layer of peptidoglycan mesh [More–Diderm sacculus architecture], which is visible here as a thin line in the periplasm. This minimal sacculus presents a considerable challenge for growth: how can you remodel a single-layered wall that is under pressure? The answer, as we are beginning to figure out, is very carefully [Schematic–Sacculus remodeling].</p>
<p>The difference in thickness of monoderm and diderm cell walls enables a well-known classification system: the Gram stain, which binds peptidoglycan. Gram-positive bacteria, typically monoderm, contain much more peptidoglycan than Gram-negative bacteria, which are typically diderm.</p>
<div class="figure"><span id="fig:unnamed-chunk-22"></span>
<div id="htmlwidget-cd5ef8c9486b3581a956" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-cd5ef8c9486b3581a956">{"x":{"doi":"10.22002/D1.1352","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.7: Hydrogenovibrio crunogenus Collected by: Cristina Iancu <a href="https://doi.org/10.22002/D1.1352">10.22002/D1.1352</a>
</p>
</div>
</div>
<div id="schematicbrauns-lipoprotein" class="section level2">
<h2><span class="header-section-number">3.5</span> 3[Schematic–Braun’s lipoprotein]</h2>
<p>Lipoproteins are hybrid molecules, formed by covalently linked lipid and protein pieces. The lipid allows them to embed into a membrane, tethering the attached protein to function nearby. Braun’s lipoprotein, or Lpp <span class="citation">Shu et al. (<a href="#ref-shu2000">2000</a>)</span>, is one of the most abundant molecules in the outer membrane of cells like <em>Escherichia coli</em>. The top of the trimeric protein “coiled-coil” contains the lipid tether. The bottom binds the peptidoglycan cell wall, creating a link between the outer membrane and the cell wall (adding up, for a typical <em>E. coli</em> cell, to about 100,000 links). These links determine the distance between the two layers.</p>
<div class="figure"><span id="fig:unnamed-chunk-23"></span>
<div id="htmlwidget-fd86ddd5c08c79316f58" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-fd86ddd5c08c79316f58">{"x":{"doi":"10.22002/D1.1351","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.8: Listeria monocytogenes Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1351">10.22002/D1.1351</a>
</p>
</div>
<div id="escherichia-coli-diderm_sacculus_architecturecompare-this-diderm-sacculus-purified-from-escherichia-coli-to-the-monoderm-sacculus-on-the-last-page.-since-this-one-is-thinner-we-can-make-out-more-details-like-the-glycan-strands-running-around-the-circumference-of-the-cell.-the-main-difference-between-the-two-types-of-sacculi-seems-to-be-whether-they-have-largely-a-single-layer-of-peptidoglycan-diderm-or-many-layers-monoderm.-so-even-though-the-two-types-of-wall-look-different-their-architecture-is-fundamentally-the-same.-in-some-circumstances-as-you-will-see-in-chapter-8-cells-can-even-switch-between-the-two-forms." class="section level3">
<h3><span class="header-section-number">3.5.1</span> Escherichia coli {#Diderm_sacculus_architecture}Compare this diderm sacculus purified from <em>Escherichia coli</em> to the monoderm sacculus on the last page. Since this one is thinner, we can make out more details, like the glycan strands running around the circumference of the cell. The main difference between the two types of sacculi seems to be whether they have largely a single layer of peptidoglycan (diderm) or many layers (monoderm). So even though the two types of wall look different, their architecture is fundamentally the same. In some circumstances, as you will see in Chapter 8, cells can even switch between the two forms.</h3>
<div class="figure"><span id="fig:unnamed-chunk-24"></span>
<div id="htmlwidget-320f879fa67f94236475" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-320f879fa67f94236475">{"x":{"doi":"10.22002/D1.1362","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.9: Escherichia coli Collected by: Lu Gan <a href="https://doi.org/10.22002/D1.1362">10.22002/D1.1362</a>
</p>
</div>
</div>
</div>
<div id="schematicsacculus-remodeling" class="section level2">
<h2><span class="header-section-number">3.6</span> 3[Schematic–Sacculus remodeling]</h2>
<p>Encasing your cell in a rigid scaffold presents a problem: how can it grow? It is easy to make membranes larger simply by adding more lipids. But to add more peptidoglycan strands, they must be linked into the existing network, which means breaking existing links to accommodate them. To do this, cells use three tools: an enzyme that links glycan sugars into strands, an enzyme that links glycan strands together with peptide bonds, and an enzyme that cuts these peptide links to allow new strands to be linked. Remember, though, that your cell, with its solute-rich interior, has a turgor pressure pushing outward with a force of perhaps 3 atmospheres, equivalent to what we would feel at a depth of 20 meters in the ocean. This is more than enough pressure to lyse an exposed membrane, so the tools must be wielded with care or the cell would burst. We are still figuring out how this works, with help from computer simulations like this one. Here you see a model of an <em>Escherichia coli</em> sacculus being enlarged using the three enzyme tools we just described (the colored balls). This simulation was run to test whether just having the tools function in a complex rather than separately might provide enough coordination for smooth, safe growth <span class="citation">Nguyen et al. (<a href="#ref-nguyen2015">2015</a>)</span>. (The answer was yes.)</p>
Vesicles
<div class="figure"><span id="fig:unnamed-chunk-25"></span>
<div id="htmlwidget-203dacc94cc7007e59db" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-203dacc94cc7007e59db">{"x":{"doi":"10.22002/D1.1351","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.10: Listeria monocytogenes Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1351">10.22002/D1.1351</a>
</p>
</div>
</div>
<div id="myxococcus-xanthus" class="section level2">
<h2><span class="header-section-number">3.7</span> Myxococcus xanthus</h2>
<p>What else can your cell do with an extra membrane? Since membranes make such excellent containers for molecules, why not get into the shipping business? In the coming chapters (especially Chapter 9), you will see some of the ways that cells interact with each other and their environment. For diderm bacteria, many of these interactions are made possible by outer membrane vesicles (“little bladders”)–self-contained pockets budded off the membrane. The vesicles may carry cargo of antibiotics to inhibit competitors’ growth, or toxins to lyse neighboring cells. Or enzymes to digest those lysed remains into nutrients that your cell can easily take up as food. Alternatively, they may carry emergency kits, first aid and survival factors for other members of a community biofilm. The appearance of these vesicles can vary as much as their contents [More–Pearled vesicles] [More–Tubular vesicles]. They are usually spherical, though, of a fairly consistent size, and often come off the cell at one or a few sites, forming chains, as you can see in this <em>Myxococcus xanthus</em> cell.</p>
<p>Not all diderms produce outer membrane vesicles, and even for those that do, we still do not know exactly how they do it. Maybe it happens spontaneously due to the physics of lipids and proteins in a certain configuration. Or maybe there is a dedicated protein machine in the membrane, blowing bubbles. Vesicles can also bud from the cytoplasmic or (for monoderms) inner membrane into the cytoplasm or periplasm [More–Cytoplasmic vesicles]. This seems to be a less regulated process than outer membrane vesicle formation, and we see it in many species when they are stressed by low nutrients or high cell density, suggesting that it is a general phenomenon. Cells shrink in harsh conditions (more on that in Chapter 8), so cytoplasmic or periplasmic vesicles may simply offer a place to put the extra membrane or, more optimistically, to store it until the time comes to grow again. Just as with outer membrane vesicles, the appearance of cytoplasmic vesicles varies widely [More–Cytoplasmic vesicle variety]. Archaea also produce membrane vesicles, both extracellular and cytoplasmic [More–Archaeal vesicles]. They have been studied less than their bacterial counterparts, but likely serve similar roles in metabolism and community interactions.</p>
<div class="figure"><span id="fig:unnamed-chunk-26"></span>
<div id="htmlwidget-35fce3ce2f013622584e" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-35fce3ce2f013622584e">{"x":{"doi":"10.22002/D1.1474","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.11: Myxococcus xanthus Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1474">10.22002/D1.1474</a>
</p>
</div>
<div id="borrelia-burgdorferi-pearled_vesiclesdifferent-species-can-produce-outer-membrane-vesicles-that-look-very-different.-the-same-species-can-even-produce-vesicles-that-look-very-different.-sometimes-they-come-off-the-cell-as-a-chain-of-spheres-sometimes-the-spheres-remain-connected-like-a-string-of-pearls-as-in-this-borrelia-burgdorferi-cell.-sometimes-vesicles-form-long-tubes-instead-moretubular-vesicles.-sometimes-the-same-chain-can-be-tubular-in-one-section-usually-at-the-base-connected-to-the-cell-and-a-string-of-spheres-in-another." class="section level3">
<h3><span class="header-section-number">3.7.1</span> Borrelia burgdorferi {#Pearled_vesicles}Different species can produce outer membrane vesicles that look very different. The same species can even produce vesicles that look very different. Sometimes they come off the cell as a chain of spheres; sometimes the spheres remain connected, like a string of pearls, as in this <em>Borrelia burgdorferi</em> cell. Sometimes vesicles form long tubes instead [More–Tubular vesicles]. Sometimes the same chain can be tubular in one section (usually at the base, connected to the cell), and a string of spheres in another.</h3>
<div class="figure"><span id="fig:unnamed-chunk-27"></span>
<div id="htmlwidget-3d1672bdd95c1ceaec1b" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-3d1672bdd95c1ceaec1b">{"x":{"doi":"10.22002/D1.1363","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.12: Borrelia burgdorferi 2 Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1363">10.22002/D1.1363</a>
</p>
</div>
</div>
<div id="borrelia-burgdorferi-tubular_vesicleshere-again-from-a-borrelia-burgdorferi-cell-you-can-see-extended-tubular-outer-membrane-vesicles." class="section level3">
<h3><span class="header-section-number">3.7.2</span> Borrelia burgdorferi {#Tubular_vesicles}Here, again from a <em>Borrelia burgdorferi</em> cell, you can see extended, tubular outer membrane vesicles.</h3>
<div class="figure"><span id="fig:unnamed-chunk-28"></span>
<div id="htmlwidget-f36c51359e3a54671eab" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-f36c51359e3a54671eab">{"x":{"doi":"10.22002/D1.1353","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.13: Borrelia burgdorferi Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1353">10.22002/D1.1353</a>
</p>
</div>
</div>
<div id="myxococcus-xanthus-cytoplasmic_vesiclesnot-all-vesicles-come-from-the-outer-membrane.-the-cytoplasmic-or-inner-membrane-can-also-form-vesicles-that-are-released-into-the-cytoplasm-as-in-this-myxococcus-xanthus-cell-or-into-the-periplasm." class="section level3">
<h3><span class="header-section-number">3.7.3</span> Myxococcus xanthus {#Cytoplasmic_vesicles}Not all vesicles come from the outer membrane. The cytoplasmic or inner membrane can also form vesicles that are released into the cytoplasm, as in this <em>Myxococcus xanthus</em> cell, or into the periplasm.</h3>
<div class="figure"><span id="fig:unnamed-chunk-29"></span>
<div id="htmlwidget-c7ff81c668259935159a" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-c7ff81c668259935159a">{"x":{"doi":"10.22002/D1.1364","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.14: Myxococcus xanthus Collected by: Matthew Swulius <a href="https://doi.org/10.22002/D1.1364">10.22002/D1.1364</a>
</p>
</div>
</div>
<div id="prosthecobacter-debontii-cytoplasmic_vesicle_varietycytoplasmic-vesicles-exhibit-a-variety-of-sizes-and-shapes.-some-are-nested-with-vesicles-inside-vesicles.-in-this-prosthecobacter-debontii-cell-you-can-see-two-other-morphologies.-one-is-a-large-flattened-horseshoe-shaped-vesicle.-another-is-a-more-typical-spherical-shape-but-is-decorated-with-what-look-like-protein-complexes." class="section level3">
<h3><span class="header-section-number">3.7.4</span> Prosthecobacter debontii {#Cytoplasmic_vesicle_variety}Cytoplasmic vesicles exhibit a variety of sizes and shapes. Some are nested, with vesicles inside vesicles. In this <em>Prosthecobacter debontii</em> cell, you can see two other morphologies. One is a large, flattened horseshoe-shaped vesicle. Another is a more typical spherical shape, but is decorated with what look like protein complexes.</h3>
<p>This cell also has unusual structures on its surface that have yet to be identified.</p>
<div class="figure"><span id="fig:unnamed-chunk-30"></span>
<div id="htmlwidget-85620352abce974c8e4a" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-85620352abce974c8e4a">{"x":{"doi":"10.22002/D1.1365","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.15: Prosthecobacter debontii Collected by: Martin Pilhofer <a href="https://doi.org/10.22002/D1.1365">10.22002/D1.1365</a>
</p>
</div>
</div>
<div id="halomicrobium-mukohataei-archaeal_vesiclesjust-like-bacteria-archaea-also-produce-both-internal-and-external-vesicles-as-you-can-see-in-this-halomicrobium-mukohataei-cell." class="section level3">
<h3><span class="header-section-number">3.7.5</span> Halomicrobium mukohataei {#Archaeal_vesicles}Just like bacteria, archaea also produce both internal and external vesicles, as you can see in this <em>Halomicrobium mukohataei</em> cell.</h3>
Variation
<div class="figure"><span id="fig:unnamed-chunk-31"></span>
<div id="htmlwidget-ae701b4ef5df5d4ee813" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-ae701b4ef5df5d4ee813">{"x":{"doi":"10.22002/D1.1475","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.16: Halomicrobium mukohataei Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1475">10.22002/D1.1475</a>
</p>
</div>
</div>
</div>
<div id="mycobacterium-marinum" class="section level2">
<h2><span class="header-section-number">3.8</span> Mycobacterium marinum</h2>
<p>Evolution is endlessly creative, providing exceptions to nearly every classification rule. We have just described a neat breakdown of bacteria into monoderms (one membrane, thick sacculus, positive Gram stain) and diderms (two membranes, thin sacculus, negative Gram stain). But some cells, like this <em>Mycobacterium marinum</em>, defy such easy classification. <em>Mycobacteria</em> are diderm, with an inner and an outer membrane, and a cell wall. But they have unique molecules (named mycolic acids in their honor) in the outer membrane. These acids interfere with Gram staining, yielding an intermediate result between positive and negative. And their sacculus consists of three layers of sugars, each with a unique molecular composition. The middle layer is the familiar peptidoglycan.</p>
<p>This cell illustrates another important point: we cannot always see everything that is there. In this case, we cannot see an additional layer outside the outer membrane called the capsule. The capsule, present in many bacteria (mostly diderm, but also some monoderm), is made up of an “_e_xtracellular _p_olymeric _s_ubstance,&quot; or EPS: long chains of sugars, sometimes linked to the outer membrane and sometimes free-floating. These sugars help the cell attach to surfaces and offer an extra layer of protection, trapping water to prevent desiccation and making it more difficult for hydrophobic molecules like detergents to get through to disrupt the membrane(s). It also makes it more difficult for viruses to reach the cell, and for eukaryotic predators like macrophages to eat it. The EPS does not interact strongly with electrons, though, so the capsule is usually invisible by electron microscopy.</p>
Surface Layer
<div class="figure"><span id="fig:unnamed-chunk-32"></span>
<div id="htmlwidget-2142838a8d82b5e925db" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-2142838a8d82b5e925db">{"x":{"doi":"10.22002/D1.1354","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.17: Mycobacterium marinum Collected by: Elitza Tocheva <a href="https://doi.org/10.22002/D1.1354">10.22002/D1.1354</a>
</p>
</div>
</div>
<div id="caulobacter-crescentus-2" class="section level2">
<h2><span class="header-section-number">3.9</span> Caulobacter crescentus</h2>
<p>How else can you protect your cell from the rigors of a harsh world? What about encasing it in an armored shell a la the armadillo? Many bacteria (both monoderms and diderms) and archaea use modular proteins for this purpose, interlocking Lego-block-like pieces into a shell called the Surface Layer, or S-layer. This must offer a significant evolutionary advantage since up to 15% of the total protein in the cell can be found in the structure. In fact, surface layers play many roles for cells, but one of their main functions for bacteria is as a gatekeeper, preventing large things like viruses from reaching the membrane.</p>
<p>Surface layers are crystalline lattices, and they can be striking in appearance, as on this <em>Caulobacter crescentus</em> cell. Amazingly, the lattice is made from (many copies of) a single protein [Schematic–Surface layer architecture]. The pinwheel-like subunits interact laterally, leaving pores large enough for nutrients to pass through, but too small for viruses. The modular nature of the lattice means that units can be popped in as the cell grows, or popped out to allow a cell appendage to poke through. They even accommodate vesicles [More–Nanopods]. Surface layer proteins are often chemically modified to alter their properties; for instance, a sugar may be added to stick the cell to a surface.</p>
<div class="figure"><span id="fig:unnamed-chunk-33"></span>
<div id="htmlwidget-622acc56a17fd1ffd04e" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-622acc56a17fd1ffd04e">{"x":{"doi":"10.22002/D1.1355","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.18: Caulobacter crescentus Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1355">10.22002/D1.1355</a>
</p>
</div>
</div>
<div id="schematicsurface-layer-architecture" class="section level2">
<h2><span class="header-section-number">3.10</span> 6[Schematic–Surface layer architecture]</h2>
<p>A single protein forms the surface layer you just saw in <em>Caulobacter crescentus</em>. The protein has two domains. The bottom domain anchors it to lipoproteins attached to the outer membrane. The top domain forms the canopy of the S-layer, organizing hierarchically into hexameric rosettes like this that in turn pack into a larger hexameric lattice <span class="citation">Bharat et al. (<a href="#ref-bharat2017">2017</a>)</span>. This lattice is flexible enough to curve around even narrow regions of the cell like the stalk discussed in Chapter 3.</p>
<div class="figure"><span id="fig:unnamed-chunk-34"></span>
<div id="htmlwidget-89a518726e58bf2fe1aa" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-89a518726e58bf2fe1aa">{"x":{"doi":"10.22002/D1.1351","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.19: Listeria monocytogenes Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1351">10.22002/D1.1351</a>
</p>
</div>
</div>
<div id="aarchaea-and-bacteria-with-surface-layers-produce-characteristic-outer-membrane-vesicles-they-bud-off-with-the-surface-layer-attached.-you-will-see-plenty-of-examples-in-the-coming-pages.-delftia-acidovorans-like-this-produce-so-called-nanopods-chains-of-outer-membrane-vesicles-ensheathed-in-surface-layer." class="section level2">
<h2><span class="header-section-number">3.11</span> 6aArchaea and bacteria with surface layers produce characteristic outer membrane vesicles: they bud off with the surface layer attached. You will see plenty of examples in the coming pages. <em>Delftia acidovorans</em> like this produce so-called nanopods: chains of outer membrane vesicles ensheathed in surface layer.</h2>
Surface Layer Variety
<div class="figure"><span id="fig:unnamed-chunk-35"></span>
<div id="htmlwidget-7f865acf95a3e46c3504" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-7f865acf95a3e46c3504">{"x":{"doi":"10.22002/D1.1351","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.20: Listeria monocytogenes Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1351">10.22002/D1.1351</a>
</p>
</div>
</div>
<div id="sulfolobus-solfataricus" class="section level2">
<h2><span class="header-section-number">3.12</span> Sulfolobus solfataricus</h2>
<p>One of the most striking features of the surface layer is how different it can look in different species. For instance, compare this archaeal <em>Sulfolobus solfataricus</em> cell to the bacterium on the last page, or to other diderm [More–Methylomicrobium alcaliphilum] or monoderm [More–Clostridium thermocellum] bacteria, or archaea [More–Nitrosopumilis maritimus] [More–Methanoregula formicica]. All surface layers are crystalline lattices of a single–or in a few cases, two–proteins, but the particular pattern of the lattice depends on the shape of this building block and how it multimerizes into a higher-order structure. The shape of the building block varies considerably; there is almost no sequence homology between S-layer proteins from different species. And shapes come together in different ways, forming repeating units of one, two, three (as on this cell), four, or six blocks.</p>
<p>S-layers are very common in archaea like this <em>S. solfataricus</em>. Most archaea lack cell walls, so the surface layer functions as external scaffolding. Remember, too, that nearly all archaea are monoderms, lacking the extra periplasmic compartment that diderms have. Here again the surface layer serves a similar function, enclosing a space around the cell’s membrane called the pseudo-periplasmic space. Just as with the bacterial periplasm, this space serves as an antechamber for the cell, restricting access by large molecules. In some cases, the pseudo-periplasmic space also contains specific proteins that function in metabolism.</p>
<div class="figure"><span id="fig:unnamed-chunk-36"></span>
<div id="htmlwidget-a7033d4e86804e461a25" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-a7033d4e86804e461a25">{"x":{"doi":"10.22002/D1.1356","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.21: Sulfolobus solfataricus Collected by: Lu Gan <a href="https://doi.org/10.22002/D1.1356">10.22002/D1.1356</a>
</p>
</div>
<div id="methylomicrobium-alcaliphilum-m._alcaliphilumin-methylomicrobium-alcaliphilum-v-shaped-surface-layer-proteins-come-together-to-form-cups-that-pack-into-a-hexagonal-pattern." class="section level3">
<h3><span class="header-section-number">3.12.1</span> Methylomicrobium alcaliphilum {#M._alcaliphilum}In <em>Methylomicrobium alcaliphilum</em>, V-shaped surface layer proteins come together to form cups that pack into a hexagonal pattern.</h3>
<div class="figure"><span id="fig:unnamed-chunk-37"></span>
<div id="htmlwidget-805bb0ea31e4b4de2465" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-805bb0ea31e4b4de2465">{"x":{"doi":"10.22002/D1.1367","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.22: Methylomicrobium alcaliphilum Collected by: Songye Chen <a href="https://doi.org/10.22002/D1.1367">10.22002/D1.1367</a>
</p>
</div>
</div>
<div id="clostridium-thermocellum-clostridium_thermocellumnot-only-diderm-bacteria-have-surface-layers-as-you-can-see-on-this-monoderm-clostridium-thermocellum-cell.-the-main-difference-is-that-the-proteins-are-anchored-to-the-cell-wall-rather-than-to-the-outer-membrane-or-associated-lipoproteins." class="section level3">
<h3><span class="header-section-number">3.12.2</span> Clostridium thermocellum {#Clostridium_thermocellum}Not only diderm bacteria have surface layers, as you can see on this monoderm <em>Clostridium thermocellum</em> cell. The main difference is that the proteins are anchored to the cell wall, rather than to the outer membrane (or associated lipoproteins).</h3>
<div class="figure"><span id="fig:unnamed-chunk-38"></span>
<div id="htmlwidget-40b8cd6f9d5b63b8a3a5" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-40b8cd6f9d5b63b8a3a5">{"x":{"doi":"10.22002/D1.1476","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.23: Clostridium thermocellum Collected by: William Nicolas <a href="https://doi.org/10.22002/D1.1476">10.22002/D1.1476</a>
</p>
</div>
</div>
<div id="nitrosopumilis-maritimus-n._maritimusin-nitrosopumilis-maritimus-surface-layer-proteins-form-hexagonal-rosettes-that-in-turn-pack-into-a-hexagonal-lattice." class="section level3">
<h3><span class="header-section-number">3.12.3</span> Nitrosopumilis maritimus {#N._maritimus}In <em>Nitrosopumilis maritimus</em>, surface layer proteins form hexagonal rosettes that in turn pack into a hexagonal lattice.</h3>
<div class="figure"><span id="fig:unnamed-chunk-39"></span>
<div id="htmlwidget-cb5632fd3f41334a1e19" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-cb5632fd3f41334a1e19">{"x":{"doi":"10.22002/D1.1368","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.24: Nitrosopumilis maritimus Collected by: Rasika Ramdasi <a href="https://doi.org/10.22002/D1.1368">10.22002/D1.1368</a>
</p>
</div>
</div>
<div id="methanoregula-formicica-methanoregula_formicicathe-surface-layer-of-methanoregula-formicica-is-a-hexagonal-lattice-of-small-nearly-circular-subunits." class="section level3">
<h3><span class="header-section-number">3.12.4</span> Methanoregula formicica {#Methanoregula_formicica}The surface layer of <em>Methanoregula formicica</em> is a hexagonal lattice of small, nearly circular subunits.</h3>
Sheath
<div class="figure"><span id="fig:unnamed-chunk-40"></span>
<div id="htmlwidget-f2599b2cb1eb90edb375" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-f2599b2cb1eb90edb375">{"x":{"doi":"10.22002/D1.1369","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.25: Methanoregula formicica Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1369">10.22002/D1.1369</a>
</p>
</div>
</div>
</div>
<div id="methanospirillum-hungatei" class="section level2">
<h2><span class="header-section-number">3.13</span> Methanospirillum hungatei</h2>
<p>Why stop at a single layer of protein? For proof that Nature is endlessly inventive, consider this <em>Methanospirillum hungatei</em> cell. These archaea encase themselves in a surface layer, as well as an additional protein layer that forms a highly impermeable sheath. The sheath is also very resistant to pressure, which could be important in the cells’ line of work. <em>M. hungatei</em> were discovered in sewage, where they break down organic waste, producing methane. One theory is that the sheath acts as a pressure regulator; when enough methane builds up inside the cell, the pressure expands the sheath, opening its pores wide enough to allow the methane to dissipate and new metabolic substrates like hydrogen and carbon dioxide to enter.</p>
<p>The rules of architecture remain the same, though. Just as in the bacterial cell wall, sheath polymers are arranged as hoops perpendicular to the long axis of the rod-shaped cell. The ends of the sheath, as you can see, are special, with multiple protein layers stacking into a thick plug. Cells divide within the sheath, and long chains of cells in a continuous sheath are often observed.</p>
DNA
<div class="figure"><span id="fig:unnamed-chunk-41"></span>
<div id="htmlwidget-02ad0cc9e2e4543c52aa" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-02ad0cc9e2e4543c52aa">{"x":{"doi":"10.22002/D1.1357","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.26: Methanospirillum hungatei Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1357">10.22002/D1.1357</a>
</p>
</div>
</div>
<div id="haloarcula-argentinensis" class="section level2">
<h2><span class="header-section-number">3.14</span> Haloarcula argentinensis</h2>
<p>All the layers we just discussed collectively make up the container, or envelope, of a cell. As you have seen, different species use different combinations of these components to form their envelopes; the only constant is the cytoplasmic (or inner, for diderms) membrane.</p>
<p>Now consider what these envelopes contain. In addition to water and small molecules, you have already seen some large protein complexes like motility machines. You have also seen many ribosomes–the protein/RNA complexes responsible for translation. But you might have been surprised <em>not</em> to see something else: deoxyribonucleic acid, or DNA. The molecule containing the instructions for the life of the cell is of paramount importance yet often invisible by microscopy. But not always. Thin filaments of DNA, only about 2 nm wide, blend in with the dense cytoplasm of the cell. When a cell lyses, though, its cytoplasm diffuses into the environment and the DNA filaments stand out against the now-much-reduced background. You can get an idea of the sheer abundance of DNA inside a cell from this <em>Haloarcula argentinensis</em> whose envelope has ruptured and contents are spilling out.</p>
Nucleoid
<div class="figure"><span id="fig:unnamed-chunk-42"></span>
<div id="htmlwidget-f3dbffecdcd5fa6b679e" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-f3dbffecdcd5fa6b679e">{"x":{"doi":"10.22002/D1.1358","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.27: Haloarcula argentinensis Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1358">10.22002/D1.1358</a>
</p>
</div>
</div>
<div id="bdellovibrio-bacteriovorus-1" class="section level2">
<h2><span class="header-section-number">3.15</span> Bdellovibrio bacteriovorus</h2>
<p>Cells contain enormous amounts of DNA. The single, circular chromosome of this <em>Bdellovibrio bacteriovorus</em> cell contains 3,782,950 individual nucleotide pairs, which means that if the circle were cut and laid out as a long piece, it would be about <em>one thousand times</em> longer than the cell itself. To fit and function inside the cell, the chromosome has to be extraordinarily organized and packed, a feat we still do not understand. Some of this packing is evident in nearly every cell: the center of the cell tends to have very few large macromolecular complexes like ribosomes, because they are excluded by the densely-packed chromosome(s). Look for these ribosome-excluding zones in the rest of the book; they indicate the location of the bulk of the cell’s DNA. Since this region is not enclosed by an internal membrane, it is not called a nucleus (the “karyon” that defines eukaryotes). Instead, we use the term nucleoid to describe the cytoplasmic region where most of the DNA is concentrated.</p>
<p>At times, the nucleoid becomes easier to see. Imagine that your cell wanted to decrease its gene expression (we will discuss why in Chapters 8 and 9). One approach is simply to pack the chromosome so tightly that the transcriptional machinery cannot access the genes. This cell is doing just that, condensing its nucleoid into a dense twisted braid we can easily visualize.</p>
<p>References</p>
<p>Further Reading</p>
<span class="citation">Errington (<a href="#ref-errington2013">2013</a>)</span> <span class="citation">Ptacin and Shapiro (<a href="#ref-ptacin2013">2013</a>)</span> <span class="citation">Sleytr and Beveridge (<a href="#ref-sleytr1999">1999</a>)</span> <span class="citation">Strahl and Errington (<a href="#ref-strahl2017">2017</a>)</span>
<div class="figure"><span id="fig:unnamed-chunk-43"></span>
<div id="htmlwidget-86d3fd24be48a8ed683d" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-86d3fd24be48a8ed683d">{"x":{"doi":"10.22002/D1.1359","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 3.28: Bdellovibrio bacteriovorus Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1359">10.22002/D1.1359</a>
</p>
</div>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-bharat2017">
<p>Bharat, T. A. M., D. Kureisaite-Ciziene, G. G. Hardy, E. W. Yu, J. M. Devant, W. J. H. Hagen, Y. V. Brun, J. A. G. Briggs, and J. Lowe. 2017. “Structure of the Hexagonal Surface Layer on Caulobacter Crescentus Cells.” <em>Nat Microbiol</em> 2 (April): 17059. doi:<a href="https://doi.org/10.1038/nmicrobiol.2017.59">10.1038/nmicrobiol.2017.59</a>.</p>
</div>
<div id="ref-errington2013">
<p>Errington, J. 2013. “L-Form Bacteria, Cell Walls and the Origins of Life.” <em>Open Biol</em> 3 (1): 120143. doi:<a href="https://doi.org/10.1098/rsob.120143">10.1098/rsob.120143</a>.</p>
</div>
<div id="ref-nguyen2015">
<p>Nguyen, L. T., J. C. Gumbart, M. Beeby, and G. J. Jensen. 2015. “Coarse-Grained Simulations of Bacterial Cell Wall Growth Reveal That Local Coordination Alone Can Be Sufficient to Maintain Rod Shape.” <em>Proc Natl Acad Sci U S A</em> 112 (28): E3689–98. doi:<a href="https://doi.org/10.1073/pnas.1504281112">10.1073/pnas.1504281112</a>.</p>
</div>
<div id="ref-ptacin2013">
<p>Ptacin, J. L., and L. Shapiro. 2013. “Chromosome Architecture Is a Key Element of Bacterial Cellular Organization.” <em>Cell Microbiol</em> 15 (1): 45–52. doi:<a href="https://doi.org/10.1111/cmi.12049">10.1111/cmi.12049</a>.</p>
</div>
<div id="ref-shu2000">
<p>Shu, Wei, Jie Liu, Hong Ji, and Min Lu. 2000. “Core Structure of the Outer Membrane Lipoprotein from Escherichia Coli at 1.9 å resolution11Edited by D. Rees.” <em>Journal of Molecular Biology</em> 299 (4): 1101–12. doi:<a href="https://doi.org/10.1006/jmbi.2000.3776">10.1006/jmbi.2000.3776</a>.</p>
</div>
<div id="ref-sleytr1999">
<p>Sleytr, U. B., and T. J. Beveridge. 1999. “Bacterial S-Layers.” <em>Trends Microbiol</em> 7 (6): 253–60.</p>
</div>
<div id="ref-sobti2016">
<p>Sobti, Meghna, Callum Smits, Andrew SW Wong, Robert Ishmukhametov, Daniela Stock, Sara Sandin, and Alastair G Stewart. 2016. “Cryo-EM Structures of the Autoinhibited E. Coli ATP Synthase in Three Rotational States.” Edited by Werner Kühlbrandt. <em>eLife</em> 5 (December). eLife Sciences Publications, Ltd: e21598. doi:<a href="https://doi.org/10.7554/eLife.21598">10.7554/eLife.21598</a>.</p>
</div>
<div id="ref-strahl2017">
<p>Strahl, H., and J. Errington. 2017. “Bacterial Membranes: Structure, Domains, and Function.” <em>Annu Rev Microbiol</em>, July. doi:<a href="https://doi.org/10.1146/annurev-micro-102215-095630">10.1146/annurev-micro-102215-095630</a>.</p>
</div>
<div id="ref-thomas1990">
<p>Thomas, Lewis. 1990. <em>A Long Line of Cells: Collected Essays</em>. New York: Book-of-the-Month-Club.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="methods.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="shape.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": null,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/caltechlibrary/cell-atlas/edit/master/02_Envelope.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["cell-atlas.pdf", "cell-atlas.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
