<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Envelope | Cell Atlas</title>
  <meta name="description" content="An atlas produced by the Jensen Lab." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Envelope | Cell Atlas" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="splash/HeaderImage.jpg" />
  <meta property="og:description" content="An atlas produced by the Jensen Lab." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Envelope | Cell Atlas" />
  
  <meta name="twitter:description" content="An atlas produced by the Jensen Lab." />
  <meta name="twitter:image" content="splash/HeaderImage.jpg" />

<meta name="author" content="Jensen Lab" />


<meta name="date" content="2020-02-25" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="methods.html"/>
<link rel="next" href="shape.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/CL-0.1.2/CL.js"></script>
<script src="libs/doivideo-binding-0.0.0.9000/doivideo.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Cell Atlas</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>1</b> Methods</a><ul>
<li class="chapter" data-level="1.1" data-path="methods.html"><a href="methods.html#light-microscopy"><i class="fa fa-check"></i><b>1.1</b> Light microscopy</a></li>
<li class="chapter" data-level="1.2" data-path="methods.html"><a href="methods.html#fluorescence-light-microscopy"><i class="fa fa-check"></i><b>1.2</b> Fluorescence light microscopy</a></li>
<li class="chapter" data-level="1.3" data-path="methods.html"><a href="methods.html#electron-microscopy"><i class="fa fa-check"></i><b>1.3</b> Electron microscopy</a></li>
<li class="chapter" data-level="1.4" data-path="methods.html"><a href="methods.html#thin-section-tem"><i class="fa fa-check"></i><b>1.4</b> Thin-section TEM</a></li>
<li class="chapter" data-level="1.5" data-path="methods.html"><a href="methods.html#cryo-em"><i class="fa fa-check"></i><b>1.5</b> Cryo-EM</a></li>
<li class="chapter" data-level="1.6" data-path="methods.html"><a href="methods.html#cryo-et"><i class="fa fa-check"></i><b>1.6</b> Cryo-ET</a></li>
<li class="chapter" data-level="1.7" data-path="methods.html"><a href="methods.html#subtomogram-averaging"><i class="fa fa-check"></i><b>1.7</b> Subtomogram averaging</a></li>
<li class="chapter" data-level="1.8" data-path="methods.html"><a href="methods.html#clem"><i class="fa fa-check"></i><b>1.8</b> CLEM</a></li>
<li class="chapter" data-level="1.9" data-path="methods.html"><a href="methods.html#single-particle-reconstruction"><i class="fa fa-check"></i><b>1.9</b> Single-particle reconstruction</a></li>
<li class="chapter" data-level="1.10" data-path="methods.html"><a href="methods.html#x-ray-crystallography"><i class="fa fa-check"></i><b>1.10</b> X-ray crystallography</a></li>
<li class="chapter" data-level="1.11" data-path="methods.html"><a href="methods.html#visual-proteomics"><i class="fa fa-check"></i><b>1.11</b> Visual proteomics</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="envelope.html"><a href="envelope.html"><i class="fa fa-check"></i><b>2</b> Envelope</a><ul>
<li class="chapter" data-level="2.1" data-path="envelope.html"><a href="envelope.html#mycoplasma-genitalium"><i class="fa fa-check"></i><b>2.1</b> Mycoplasma genitalium</a><ul>
<li class="chapter" data-level="2.1.1" data-path="envelope.html"><a href="envelope.html#Lipid_bilayer"><i class="fa fa-check"></i><b>2.1.1</b> Lipid bilayer</a></li>
<li class="chapter" data-level="2.1.2" data-path="envelope.html"><a href="envelope.html#ATP_synthase_and_energy_production"><i class="fa fa-check"></i><b>2.1.2</b> ATP synthase and energy production</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="envelope.html"><a href="envelope.html#listeria-monocytogenes"><i class="fa fa-check"></i><b>2.2</b> Listeria monocytogenes</a><ul>
<li class="chapter" data-level="2.2.1" data-path="envelope.html"><a href="envelope.html#Peptidoglycan_architecture"><i class="fa fa-check"></i><b>2.2.1</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="2.2.2" data-path="envelope.html"><a href="envelope.html#L-form_bacteria"><i class="fa fa-check"></i><b>2.2.2</b> L-form bacteria</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="envelope.html"><a href="envelope.html#cupriavidus-necator"><i class="fa fa-check"></i><b>2.3</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="2.3.1" data-path="envelope.html"><a href="envelope.html#Brauns_lipoprotein"><i class="fa fa-check"></i><b>2.3.1</b> Braun’s lipoprotein</a></li>
<li class="chapter" data-level="2.3.2" data-path="envelope.html"><a href="envelope.html#Diderm_sacculus_architecture"><i class="fa fa-check"></i><b>2.3.2</b> Escherichia coli</a></li>
<li class="chapter" data-level="2.3.3" data-path="envelope.html"><a href="envelope.html#Sacculus_remodeling"><i class="fa fa-check"></i><b>2.3.3</b> Sacculus remodeling</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="envelope.html"><a href="envelope.html#borrelia-burgdorferi"><i class="fa fa-check"></i><b>2.4</b> Borrelia burgdorferi</a><ul>
<li class="chapter" data-level="2.4.1" data-path="envelope.html"><a href="envelope.html#Vesicle_morphologies"><i class="fa fa-check"></i><b>2.4.1</b> Borrelia burgdorferi</a></li>
<li class="chapter" data-level="2.4.2" data-path="envelope.html"><a href="envelope.html#Inner_membrane_vesicles"><i class="fa fa-check"></i><b>2.4.2</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="2.4.3" data-path="envelope.html"><a href="envelope.html#Cytoplasmic_vesicle_variety"><i class="fa fa-check"></i><b>2.4.3</b> Prosthecobacter debontii</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="envelope.html"><a href="envelope.html#mycoplasma-marinum"><i class="fa fa-check"></i><b>2.5</b> Mycoplasma marinum</a><ul>
<li class="chapter" data-level="2.5.1" data-path="envelope.html"><a href="envelope.html#Mycobacterial_architecture"><i class="fa fa-check"></i><b>2.5.1</b> Mycobacterial architecture</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="envelope.html"><a href="envelope.html#caulobacter-crescentus"><i class="fa fa-check"></i><b>2.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="2.6.1" data-path="envelope.html"><a href="envelope.html#S-layer_architecture"><i class="fa fa-check"></i><b>2.6.1</b> S-layer architecture</a></li>
<li class="chapter" data-level="2.6.2" data-path="envelope.html"><a href="envelope.html#Nanopods"><i class="fa fa-check"></i><b>2.6.2</b> Delftia acidovorans</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="envelope.html"><a href="envelope.html#sulfolobus-solfataricus"><i class="fa fa-check"></i><b>2.7</b> Sulfolobus solfataricus</a><ul>
<li class="chapter" data-level="2.7.1" data-path="envelope.html"><a href="envelope.html#M._alcaliphilum"><i class="fa fa-check"></i><b>2.7.1</b> Methylomicrobium alcaliphilum</a></li>
<li class="chapter" data-level="2.7.2" data-path="envelope.html"><a href="envelope.html#N._maritimus"><i class="fa fa-check"></i><b>2.7.2</b> Nitrosopumilis maritimus</a></li>
<li class="chapter" data-level="2.7.3" data-path="envelope.html"><a href="envelope.html#Methanoregula_formicica"><i class="fa fa-check"></i><b>2.7.3</b> Methanoregula formicica</a></li>
</ul></li>
<li class="chapter" data-level="2.8" data-path="envelope.html"><a href="envelope.html#methanospirillum-hungatei"><i class="fa fa-check"></i><b>2.8</b> Methanospirillum hungatei</a></li>
<li class="chapter" data-level="2.9" data-path="envelope.html"><a href="envelope.html#haloarcula-argentinensis"><i class="fa fa-check"></i><b>2.9</b> Haloarcula argentinensis</a></li>
<li class="chapter" data-level="2.10" data-path="envelope.html"><a href="envelope.html#bdellovibrio-bacteriovorus"><i class="fa fa-check"></i><b>2.10</b> Bdellovibrio bacteriovorus</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="shape.html"><a href="shape.html"><i class="fa fa-check"></i><b>3</b> Shape</a><ul>
<li class="chapter" data-level="3.1" data-path="shape.html"><a href="shape.html#simkania-negevensis"><i class="fa fa-check"></i><b>3.1</b> Simkania negevensis</a><ul>
<li class="chapter" data-level="3.1.1" data-path="shape.html"><a href="shape.html#Cell_shape_diversity"><i class="fa fa-check"></i><b>3.1.1</b> Cell shape diversity</a></li>
<li class="chapter" data-level="3.1.2" data-path="shape.html"><a href="shape.html#Spherical_growth"><i class="fa fa-check"></i><b>3.1.2</b> Spherical growth</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="shape.html"><a href="shape.html#cupriavidus-necator-1"><i class="fa fa-check"></i><b>3.2</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="3.2.1" data-path="shape.html"><a href="shape.html#Rod-shaped_growth"><i class="fa fa-check"></i><b>3.2.1</b> Rod-shaped growth</a></li>
<li class="chapter" data-level="3.2.2" data-path="shape.html"><a href="shape.html#Rod_variety"><i class="fa fa-check"></i><b>3.2.2</b> Brucella abortus</a></li>
<li class="chapter" data-level="3.2.3" data-path="shape.html"><a href="shape.html#Archaeal_rods"><i class="fa fa-check"></i><b>3.2.3</b> Methanoregula formicica</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="shape.html"><a href="shape.html#hylemonella-gracilis"><i class="fa fa-check"></i><b>3.3</b> Hylemonella gracilis</a></li>
<li class="chapter" data-level="3.4" data-path="shape.html"><a href="shape.html#caulobacter-crescentus-1"><i class="fa fa-check"></i><b>3.4</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="3.4.1" data-path="shape.html"><a href="shape.html#Vibrioid_growth"><i class="fa fa-check"></i><b>3.4.1</b> Vibrioid growth</a></li>
<li class="chapter" data-level="3.4.2" data-path="shape.html"><a href="shape.html#CTP_synthase"><i class="fa fa-check"></i><b>3.4.2</b> CTP synthase</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="shape.html"><a href="shape.html#campylobacter-jejuni"><i class="fa fa-check"></i><b>3.5</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="3.5.1" data-path="shape.html"><a href="shape.html#Borrelia_burgdorferi"><i class="fa fa-check"></i><b>3.5.1</b> Borrelia burgdorferi</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="shape.html"><a href="shape.html#verrucomicrobium-spinosum"><i class="fa fa-check"></i><b>3.6</b> Verrucomicrobium spinosum</a><ul>
<li class="chapter" data-level="3.6.1" data-path="shape.html"><a href="shape.html#Bactofilin"><i class="fa fa-check"></i><b>3.6.1</b> Bactofilin</a></li>
<li class="chapter" data-level="3.6.2" data-path="shape.html"><a href="shape.html#Bacterial_microtubules"><i class="fa fa-check"></i><b>3.6.2</b> Prosthecobacter vanneervenii</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="shape.html"><a href="shape.html#haloquadratum-walsbyi"><i class="fa fa-check"></i><b>3.7</b> Haloquadratum walsbyi</a><ul>
<li class="chapter" data-level="3.7.1" data-path="shape.html"><a href="shape.html#Gas_vesicles"><i class="fa fa-check"></i><b>3.7.1</b> Halobacterium salinarum</a></li>
<li class="chapter" data-level="3.7.2" data-path="shape.html"><a href="shape.html#Gas_vesicles"><i class="fa fa-check"></i><b>3.7.2</b> Gas vesicles</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="4" data-path="growth.html"><a href="growth.html"><i class="fa fa-check"></i><b>4</b> Growth</a><ul>
<li class="chapter" data-level="4.1" data-path="growth.html"><a href="growth.html#caulobacter-crescentus-2"><i class="fa fa-check"></i><b>4.1</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="4.2" data-path="growth.html"><a href="growth.html#rhodopseudomonas-palustris"><i class="fa fa-check"></i><b>4.2</b> Rhodopseudomonas palustris</a><ul>
<li class="chapter" data-level="4.2.1" data-path="growth.html"><a href="growth.html#ICM_variety"><i class="fa fa-check"></i><b>4.2.1</b> Methyloprofundus sedimenti</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="growth.html"><a href="growth.html#chloroflexi"><i class="fa fa-check"></i><b>4.3</b> Chloroflexi</a></li>
<li class="chapter" data-level="4.4" data-path="growth.html"><a href="growth.html#thiomonas-intermedia"><i class="fa fa-check"></i><b>4.4</b> Thiomonas intermedia</a><ul>
<li class="chapter" data-level="4.4.1" data-path="growth.html"><a href="growth.html#Carboxysome_variety"><i class="fa fa-check"></i><b>4.4.1</b> Halothiobacillus neapolitanus</a></li>
<li class="chapter" data-level="4.4.2" data-path="growth.html"><a href="growth.html#Long_carboxysomes"><i class="fa fa-check"></i><b>4.4.2</b> Hydrogenovibrio crunogenus</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="growth.html"><a href="growth.html#cupriavidus-necator-2"><i class="fa fa-check"></i><b>4.5</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="4.5.1" data-path="growth.html"><a href="growth.html#Storage_granule_growth"><i class="fa fa-check"></i><b>4.5.1</b> Lysobacter antibioticus</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="growth.html"><a href="growth.html#agrobacterium-tumefaciens"><i class="fa fa-check"></i><b>4.6</b> Agrobacterium tumefaciens</a></li>
<li class="chapter" data-level="4.7" data-path="growth.html"><a href="growth.html#hydrogenovibrio-crunogenus"><i class="fa fa-check"></i><b>4.7</b> Hydrogenovibrio crunogenus</a></li>
<li class="chapter" data-level="4.8" data-path="growth.html"><a href="growth.html#halothiobacillus-neapolitanus"><i class="fa fa-check"></i><b>4.8</b> Halothiobacillus neapolitanus</a></li>
<li class="chapter" data-level="4.9" data-path="growth.html"><a href="growth.html#halorubrum-litoreum"><i class="fa fa-check"></i><b>4.9</b> Halorubrum litoreum</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="division.html"><a href="division.html"><i class="fa fa-check"></i><b>5</b> Division</a><ul>
<li class="chapter" data-level="5.1" data-path="division.html"><a href="division.html#thiomonas-intermedia-1"><i class="fa fa-check"></i><b>5.1</b> Thiomonas intermedia</a></li>
<li class="chapter" data-level="5.2" data-path="division.html"><a href="division.html#caulobacter-crescentus-3"><i class="fa fa-check"></i><b>5.2</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="5.3" data-path="division.html"><a href="division.html#hyphomonas-neptunium"><i class="fa fa-check"></i><b>5.3</b> Hyphomonas neptunium</a><ul>
<li class="chapter" data-level="5.3.1" data-path="division.html"><a href="division.html#Hyphomonas_stalk"><i class="fa fa-check"></i><b>5.3.1</b> Hyphomonas neptunium</a></li>
<li class="chapter" data-level="5.3.2" data-path="division.html"><a href="division.html#Hyphomonas_life_cycle"><i class="fa fa-check"></i><b>5.3.2</b> Hyphomonas neptunium</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="division.html"><a href="division.html#tetrasphaera-remsis"><i class="fa fa-check"></i><b>5.4</b> Tetrasphaera remsis</a></li>
<li class="chapter" data-level="5.5" data-path="division.html"><a href="division.html#idiomarina-loihiensis"><i class="fa fa-check"></i><b>5.5</b> Idiomarina loihiensis</a><ul>
<li class="chapter" data-level="5.5.1" data-path="division.html"><a href="division.html#sphingopyxis-alaskensis-sphingopyxis_alaskensis_divisionthis-diderm-cell-is-about-halfway-through-division.-the-two-daughter-cells-are-still-connected-by-a-wide-bridge-of-cytoplasm-but-the-inner-membrane-and-cell-wall-are-drawing-in-all-around."><i class="fa fa-check"></i><b>5.5.1</b> Sphingopyxis alaskensis {#Sphingopyxis_alaskensis_division}This diderm cell is about halfway through division. The two daughter cells are still connected by a wide bridge of cytoplasm, but the inner membrane and cell wall are drawing in all around.</a></li>
<li class="chapter" data-level="5.5.2" data-path="division.html"><a href="division.html#Chloroflexi_division"><i class="fa fa-check"></i><b>5.5.2</b> Chloroflexi</a></li>
</ul></li>
<li class="chapter" data-level="5.6" data-path="division.html"><a href="division.html#caulobacter-crescentus-4"><i class="fa fa-check"></i><b>5.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="5.6.1" data-path="division.html"><a href="division.html#Caulobacter_division"><i class="fa fa-check"></i><b>5.6.1</b> Caulobacter crescentus</a></li>
</ul></li>
<li class="chapter" data-level="5.7" data-path="division.html"><a href="division.html#agrobacterium-tumefaciens-1"><i class="fa fa-check"></i><b>5.7</b> Agrobacterium tumefaciens</a></li>
<li class="chapter" data-level="5.8" data-path="division.html"><a href="division.html#cupriavidus-necator-3"><i class="fa fa-check"></i><b>5.8</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="5.8.1" data-path="division.html"><a href="division.html#Asymmetric_constriction"><i class="fa fa-check"></i><b>5.8.1</b> Cupriavidus necator</a></li>
</ul></li>
<li class="chapter" data-level="5.9" data-path="division.html"><a href="division.html#caulobacter-crescentus-5"><i class="fa fa-check"></i><b>5.9</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="5.10" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius"><i class="fa fa-check"></i><b>5.10</b> Sulfolobus acidocaldarius</a></li>
<li class="chapter" data-level="5.11" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius-1"><i class="fa fa-check"></i><b>5.11</b> Sulfolobus acidocaldarius</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="motility.html"><a href="motility.html"><i class="fa fa-check"></i><b>6</b> Motility</a><ul>
<li class="chapter" data-level="6.1" data-path="motility.html"><a href="motility.html#caulobacter-crescentus-6"><i class="fa fa-check"></i><b>6.1</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="6.1.1" data-path="motility.html"><a href="motility.html#Flagellum_morphology"><i class="fa fa-check"></i><b>6.1.1</b> Campylobacter jejuni</a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="motility.html"><a href="motility.html#bdellovibrio-bacteriovorus-1"><i class="fa fa-check"></i><b>6.2</b> Bdellovibrio bacteriovorus</a><ul>
<li class="chapter" data-level="6.2.1" data-path="motility.html"><a href="motility.html#Flagellar_motor_structure"><i class="fa fa-check"></i><b>6.2.1</b> Bdellovibrio bacteriovorus</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="motility.html"><a href="motility.html#hylemonella-gracilis-1"><i class="fa fa-check"></i><b>6.3</b> Hylemonella gracilis</a><ul>
<li class="chapter" data-level="6.3.1" data-path="motility.html"><a href="motility.html#Flagellar_motor_disassembly"><i class="fa fa-check"></i><b>6.3.1</b> Pseudomonas aeruginosa</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="motility.html"><a href="motility.html#campylobacter-jejuni-1"><i class="fa fa-check"></i><b>6.4</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="6.4.1" data-path="motility.html"><a href="motility.html#Lophotrichous_bacteria"><i class="fa fa-check"></i><b>6.4.1</b> Helicobacter pylori</a></li>
</ul></li>
<li class="chapter" data-level="6.5" data-path="motility.html"><a href="motility.html#pseudomonas-flexibilis"><i class="fa fa-check"></i><b>6.5</b> Pseudomonas flexibilis</a></li>
<li class="chapter" data-level="6.6" data-path="motility.html"><a href="motility.html#helicobacter-hepaticus"><i class="fa fa-check"></i><b>6.6</b> Helicobacter hepaticus</a></li>
<li class="chapter" data-level="6.7" data-path="motility.html"><a href="motility.html#borrelia-burgdorferi-1"><i class="fa fa-check"></i><b>6.7</b> Borrelia burgdorferi</a><ul>
<li class="chapter" data-level="6.7.1" data-path="motility.html"><a href="motility.html#Bacterial_invasion"><i class="fa fa-check"></i><b>6.7.1</b> Macaca mulatta</a></li>
<li class="chapter" data-level="6.7.2" data-path="motility.html"><a href="motility.html#Treponema_primitia"><i class="fa fa-check"></i><b>6.7.2</b> Treponema primitia</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="motility.html"><a href="motility.html#methanoregula-formicica"><i class="fa fa-check"></i><b>6.8</b> Methanoregula formicica</a></li>
<li class="chapter" data-level="6.9" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis"><i class="fa fa-check"></i><b>6.9</b> Thermococcus kodakaraensis</a><ul>
<li class="chapter" data-level="6.9.1" data-path="motility.html"><a href="motility.html#Cone_structure"><i class="fa fa-check"></i><b>6.9.1</b> Thermococcus kodakaraensis</a></li>
<li class="chapter" data-level="6.9.2" data-path="motility.html"><a href="motility.html#Organizing_center"><i class="fa fa-check"></i><b>6.9.2</b> Thermococcus kodakaraensis</a></li>
</ul></li>
<li class="chapter" data-level="6.10" data-path="motility.html"><a href="motility.html#myxococcus-xanthus"><i class="fa fa-check"></i><b>6.10</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="6.11" data-path="motility.html"><a href="motility.html#flavobacterium-johnsoniae"><i class="fa fa-check"></i><b>6.11</b> Flavobacterium johnsoniae</a></li>
<li class="chapter" data-level="6.12" data-path="motility.html"><a href="motility.html#mycoplasma-pneumoniae"><i class="fa fa-check"></i><b>6.12</b> Mycoplasma pneumoniae</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="navigation.html"><a href="navigation.html"><i class="fa fa-check"></i><b>7</b> Navigation</a></li>
<li class="chapter" data-level="8" data-path="lifecycle.html"><a href="lifecycle.html"><i class="fa fa-check"></i><b>8</b> Lifecycle</a></li>
<li class="chapter" data-level="9" data-path="interaction.html"><a href="interaction.html"><i class="fa fa-check"></i><b>9</b> Interaction</a></li>
<li class="chapter" data-level="10" data-path="viruses.html"><a href="viruses.html"><i class="fa fa-check"></i><b>10</b> Viruses</a></li>
<li class="chapter" data-level="11" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>11</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.library.caltech.edu" target="blank">Published by Caltech Library</a><li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cell Atlas</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="envelope" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Envelope</h1>
<div id="mycoplasma-genitalium" class="section level2">
<h2><span class="header-section-number">2.1</span> Mycoplasma genitalium</h2>
<p>The fundamental unit of life is the cell – a contained self-replicating assembly. For many species, including all <em>Bacteria</em> and Archaea, the organism consists of a single cell. And for nearly all species, no matter how many cells an organism eventually contains (probably around 10 trillion in your case), it started life as a single cell. As you’ll see, the details of these cells vary, but every cell on Earth is the same at heart – a DNA-based replicator machine built from just four macromolecules: nucleic acids, proteins, lipids and carbohydrates.</p>
<p>Imagine that you’re a structural engineer tasked with building one of these cells. What’s the first step? Let’s start with the container. No matter what the first self-replicating molecules were (likely RNA), they didn’t constitute a cell until they became packaged in a container. You’d probably want a flexible container that allowed you to sort a subset of molecules from the environment. Evolution agrees. All cells are enclosed by a selectively permeable membrane, made of lipids and proteins <a href="envelope.html#Lipid_bilayer">Schematic – Lipid bilayer</a>, that allows them to differentiate their contents from the environment. This selectivity is a critical feature for the life of the cell <a href="envelope.html#ATP_synthase_and_energy_production">Schematic – ATP synthase</a>. The compartment enclosed by a cell’s membrane is called the cytoplasm (“cell mold” [the membrane being the container that shapes the mold]).</p>
<p>Almost all archaea and many bacteria, like these Mycoplasma genitalium cells, are monoderms (“single skin”). This means that their cytoplasm is enclosed by a single membrane. At the resolution of this image, the membrane looks like a single dark line, but remember that it’s really a bilayer, as you’ll see in some later images.</p>
<div class="figure"><span id="fig:unnamed-chunk-1"></span>
<div id="htmlwidget-f57e428476b534421cd7" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-f57e428476b534421cd7">{"x":{"doi":"10.22002/D1.1350","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.1: Mycoplasma genitalium Collected by: Gregory Henderson <a href="https://doi.org/10.22002/D1.1350">10.22002/D1.1350</a>
</p>
</div>
<div id="Lipid_bilayer" class="section level3">
<h3><span class="header-section-number">2.1.1</span> Lipid bilayer</h3>
<p><img src="img/02_schematic/2_1_1_LipidBilayer.png" /><!-- --></p>
<p>Lipids have a hydrophilic head (red/orange in the schematic) and hydrophobic tails (gray); in water they spontaneously pack together side-by-side to shield their tails from unfavorable interactions with water, forming closed double-layered bags. Proteins with regions of amino acids containing hydrophobic side groups embed these regions in lipid bilayers. Other proteins are fused to lipids, tethering them to the membrane. In fact, cells’ “lipid” membranes actually constitute roughly equal fractions of lipids and proteins. One key difference between archaea and bacteria (and with them, eukaryotes) is the lipid that makes up their membranes. Hybrid membranes containing both these lipid types can be made artificially, and it’s possible that the last universal common ancestor of all cells on Earth contained both types, with specialization later occurring in different lineages.</p>
</div>
<div id="ATP_synthase_and_energy_production" class="section level3">
<h3><span class="header-section-number">2.1.2</span> ATP synthase and energy production</h3>
<p><img src="img/02_schematic/2_1_2_ATPsynthase.png" /><!-- --></p>
<p>The chemical properties of lipids make membranes impermeable to ions and large or hydrophilic molecules (but not to water). Cells take advantage of this property to establish an ion gradient across the membrane, using a chain of electron-carrying proteins in the membrane to pump protons out of the cell. Protein complexes in the membrane called ATP synthases (like this one from Escherichia coli) use the resulting ion potential to generate energy. The machine provides a conduit for protons to flow down their potential, producing a “proton-motive force” that spins the machine’s rotor, generating energy that is chemically stored in ATP, the energetic currency of the cell.</p>
</div>
</div>
<div id="listeria-monocytogenes" class="section level2">
<h2><span class="header-section-number">2.2</span> Listeria monocytogenes</h2>
<p>Being able to selectively move things into your cell enables it to do some powerful things. It also poses a structural problem, though. Remember that water can pass freely through the membrane, which means that increasing the solute concentration inside relative to the environment will cause water to rush in as well, introducing a pressure (known as turgor pressure) on the membrane. Lipid bilayers, though, are unable to withstand much pressure. If your cell lives exclusively in a consistent, and fairly high-osmolarity, environment (like our bodies, in the case of the Mycoplasma genitalium you just saw), it can balance internal and external osmolarity to minimize turgor pressure on the membrane. But most cells experience much more variable environments. How can you keep your cell from bursting in such conditions?</p>
<p>You’ll probably want to add some rigid scaffolding outside the membrane to buttress it against turgor pressure. Nearly all bacteria do this using a material called peptidoglycan: long stiff polymers of glycan sugars crosslinked by short peptides into a chain-mail-like mesh. The full scaffold of this material surrounding the cell is called its sacculus, or cell wall. Some archaea also have cell walls, made of a molecule similar to peptidoglycan, but chemically distinct. Most archaea, though, rely on a different structure for support, as you’ll see later in this chapter.</p>
<p>In monoderm bacteria like this Listeria monocytogenes, the cell wall is significantly thicker than the membrane. It comprises several layers of peptidoglycan, which can’t be seen individually at the resolution of this image, so the cell wall appears as a uniformly textured layer <a href="envelope.html#Peptidoglycan_architecture">More: Peptidoglycan architecture</a>. It’s still a mystery how large molecules can pass through this dense layer on their way to and from the cell. Not all bacterial cell walls are chemically identical <a href="#">More: Methanobacterium formicicum</a>. And in some conditions, cells can lose their walls <a href="envelope.html#L-form_bacteria">Schematic – L-form bacteria</a>.</p>
<div class="figure"><span id="fig:unnamed-chunk-4"></span>
<div id="htmlwidget-e61ba16dccabe99b3593" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-e61ba16dccabe99b3593">{"x":{"doi":"10.22002/D1.1351","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.2: Listeria monocytogenes Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1351">10.22002/D1.1351</a>
</p>
</div>
<div id="Peptidoglycan_architecture" class="section level3">
<h3><span class="header-section-number">2.2.1</span> Bacillus subtilis</h3>
<p>The sacculus is so robust that it persists even after cells are lysed and their other components digested. This sacculus isolated from a Bacillus subtilis cell has retained its shape, simply flattening with the release of contents and pressure from inside. By observing how these purified sacculi rip and curl, we can infer something about the architecture of the cell wall; we think that the long glycan strands are oriented in hoops circling the short axis of the cylindrical cell and the short peptide crosslinks are more or less aligned with the long axis.</p>
<div class="figure"><span id="fig:unnamed-chunk-5"></span>
<div id="htmlwidget-96ba34a9fe3e6b6004d1" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-96ba34a9fe3e6b6004d1">{"x":{"doi":"10.22002/D1.1360","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.3: Bacillus subtilis Collected by: Morgan Beeby <a href="https://doi.org/10.22002/D1.1360">10.22002/D1.1360</a>
</p>
</div>
</div>
<div id="L-form_bacteria" class="section level3">
<h3><span class="header-section-number">2.2.2</span> L-form bacteria</h3>
<p><img src="img/02_schematic/2_2_1_L_form_bacteria.jpeg" /><!-- --></p>
<p>There is no universal adaptation in Nature; advantages in one environment can become liabilities in another. This concept is exemplified by an adaptation of some bacteria which lose their cell walls in certain conditions, such as in the presence of antibiotics (the cell wall is a common target of antibiotics). This state is called the L-form (named for the Lister Institute where it were discovered). As you can see in these Bacillus subtilis, cells in this state are pleomorphic, exhibiting a variety of sizes and shapes. As you would expect, L-form cells are more sensitive to environmental conditions. In the lab, they’re protected from lysis by increasing the osmotic pressure of the environment, for instance by adding sucrose. The environment in your body, though, would have the same effect, as we discussed for Mycoplasma genitalium. L-form bacteria are interesting for many reasons (including human health), one of which is that they give us a fascinating window into how early cells – prior to the evolution of the cell wall – might have looked and behaved.</p>
</div>
</div>
<div id="cupriavidus-necator" class="section level2">
<h2><span class="header-section-number">2.3</span> Cupriavidus necator</h2>
<p>Why stop at one membrane, though? Think about how the bacteria you’ve just seen compare with eukaryotic cells. The eukaryotic cells are much larger (maybe 100-1,000 times larger in volume), and they contain many internal membranes that form specialized subcompartments, like the nucleus and mitochondria. Bacteria and archaea don’t have membrane-bound organelles inside, but in fact many bacteria do create an additional compartment outside the cell with another, outer, membrane. These bacteria, like the Cupriavidus necator cell you see here, are called diderms (“double skin”). The extra compartment between their membranes is known as the periplasm (“mold between”). This antechamber contains a unique subset of proteins, many of which function in escorting things into and out of the main cell, as you’ll see in later chapters.</p>
<p>Compared to the inner membrane, the outer membrane has some unique properties! It is more permeable and not proton-tight (so it can’t be used to generate ATP). It is often asymmetric, with a different composition of lipids and proteins in each of the two leaflets. The outer membrane is anchored to the sacculus <a href="envelope.html#Brauns_lipoprotein">Schematic – Braun’s lipoprotein</a>. And the sacculus itself is different. Rather than containing many layers of peptidoglycan like in the Listeria monocytogenes cell you just saw, the diderm sacculus consists largely of a single-layered peptidoglycan mesh <a href="envelope.html#Diderm_sacculus_architecture">More: Diderm sacculus architecture</a>, which you can see here as a thin line in the periplasm. This difference in the sacculus enables a well-known bacterial classification system: the Gram stain, which binds peptidoglycan. Gram-positive cells, typically monoderm, contain much more peptidoglycan than Gram-negative cells, which are typically diderm. Thinking about this thin defense against turgor pressure underscores a major challenge for cell growth. To insert new material, existing bonds in the sacculus must be broken, without bursting the cell in the process <a href="envelope.html#Sacculus_remodeling">Schematic – Sacculus remodeling</a>.</p>
<div class="figure"><span id="fig:unnamed-chunk-7"></span>
<div id="htmlwidget-14fc454f09b5aa201f9f" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-14fc454f09b5aa201f9f">{"x":{"doi":"10.22002/D1.1352","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.4: Hydrogenovibrio crunogenus Collected by: Cristina Iancu <a href="https://doi.org/10.22002/D1.1352">10.22002/D1.1352</a>
</p>
</div>
<div id="Brauns_lipoprotein" class="section level3">
<h3><span class="header-section-number">2.3.1</span> Braun’s lipoprotein</h3>
<p><img src="img/02_schematic/2_3_1_BLP.png" /><!-- --></p>
<p>Lipoproteins are hybrid molecules, formed by covalently linked lipid and protein pieces. The lipid allows them to embed into a membrane, tethering the attached protein to function nearby. Braun’s lipoprotein, which is one of the most abundant molecules in the outer membrane of cells like Escherichia coli, uses its tethered protein portion to bind the peptidoglycan cell wall, creating a link between the outer membrane and the cell wall (adding up, for a typical E. coli cell, to about 100,000 links). These links determine the distance between these two components. Not all diderms use Braun’s lipoprotein, though, and some bacteria have notably labile outer membranes <a href="#">More: Outer membrane lability</a>.</p>
</div>
<div id="Diderm_sacculus_architecture" class="section level3">
<h3><span class="header-section-number">2.3.2</span> Escherichia coli</h3>
<p>Compare this diderm sacculus purified from Escherichia coli to the monoderm sacculus on the last page. Since this one is thinner; we can make out more details. Instead of inferring how the glycan strands are oriented, we can now see them running around the circumference of the cell. The main difference between the two types of sacculi seems to be whether they have largely a single layer of peptidoglycan (diderm) or many layers (monoderm). So even though the two cell walls look different, their architecture is fundamentally the same. In some circumstances, cells can even switch between the two forms, as you’ll see in Chapter 7.</p>
<div class="figure"><span id="fig:unnamed-chunk-9"></span>
<div id="htmlwidget-9f1e44684233f18d449e" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-9f1e44684233f18d449e">{"x":{"doi":"10.22002/D1.1362","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.5: Escherichia coli Collected by: Lu Gan <a href="https://doi.org/10.22002/D1.1362">10.22002/D1.1362</a>
</p>
</div>
</div>
<div id="Sacculus_remodeling" class="section level3">
<h3><span class="header-section-number">2.3.3</span> Sacculus remodeling</h3>
<p><img src="img/02_schematic/2_3_2_SacculusRemodeling.jpg" /><!-- --></p>
<p>Encasing your cell in a rigid scaffold presents a problem: how can it grow? It’s easy to make membranes larger simply by adding more lipids. But to add more peptidoglycan strands, they have to be linked into the existing network, which means breaking existing links to accommodate them. In fact, cells remodel their sacculi with the tools you’d expect: an enzyme that links glycan sugars into strands, an enzyme that links strands together with peptide bonds, and an enzyme that cuts these peptide links. Remember, though, that your cell, with its solute-rich interior, has a turgor pressure pushing outward with a force of maybe 3 atmospheres, equivalent to what we would feel at a depth of 20 meters in the ocean. This is more than enough to lyse an exposed bacterial membrane. So these tools must be used carefully to ensure that the cell doesn’t burst. We’re still figuring out exactly how this works, with help from computer simulations like this one. Here you see a model of an Escherichia coli sacculus being enlarged using the three enzyme tools we just described. This simulation was run to test whether just having the tools function in a complex rather than separately might give enough coordination for smooth, safe growth. (The answer was yes.)</p>
<p>More: Diderm archaea Nearly all diderm cells are bacteria. Not all, though. As you see here, Ignicoccus hospitalis, an archaeon, has an outer membrane that appears very loosely associated with the cell, forming an extra large periplasm. You can see membrane-bound vesicles shuttling cargo across this vast space (more about vesicles on the next page). Interestingly, this is also an exception to the rule that the inner membrane of diderms is “energized” (proton-tight). In I. hospitalis, it is the outer membrane that contains the ATP synthases. These unusual characteristics suggest a symbiotic origin for this species; perhaps its ancestor used to live inside a host cell, which was eventually reduced to a mere membrane.</p>
</div>
</div>
<div id="borrelia-burgdorferi" class="section level2">
<h2><span class="header-section-number">2.4</span> Borrelia burgdorferi</h2>
<p>What else can you do with an extra membrane? Since membranes make excellent containers for molecules, why not get into the shipping business? In the coming chapters (especially Chapter 8), you’ll see some of the ways that cells interact with each other and their environment. For diderm bacteria, many of these interactions are made possible by outer membrane vesicles (“little bladders”)–self-contained pockets budded off the membrane. The vesicles may carry cargo of antibiotics to inhibit competitors’ growth, or toxins to lyse neighboring cells. Or enzymes to digest those lysed remains into nutrients that your cell can easily take up as food. Alternatively, they may carry emergency kits (first aid and survival factors) for other members of a community biofilm. The appearance of these vesicles varies as much as their contents <a href="envelope.html#Vesicle_morphologies">More: Vesicle morphologies</a>. They are usually spherical, of a consistent size, and often come off the cell at one or a few sites, forming chains, as you can see in this Borrelia burgdorferi cell.</p>
<p>Not all diderms produce outer membrane vesicles, and even for the ones that do, we still don’t know exactly how they do it. Maybe it happens spontaneously due to the physics of lipids and proteins in a certain configuration. Or maybe there’s a dedicated protein machine in the membrane, blowing bubbles. Some archaea (monoderms) also produce membrane vesicles. They’re less studied than their bacterial counterparts, but likely serve similar roles in metabolism and community interactions.</p>
<div class="figure"><span id="fig:unnamed-chunk-11"></span>
<div id="htmlwidget-2c1e881c6a41cc6703cd" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-2c1e881c6a41cc6703cd">{"x":{"doi":"10.22002/D1.1353","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.6: Borrelia burgdorferi Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1353">10.22002/D1.1353</a>
</p>
</div>
<div id="Vesicle_morphologies" class="section level3">
<h3><span class="header-section-number">2.4.1</span> Borrelia burgdorferi</h3>
<p>Different species can produce outer membrane vesicles that look very different. The same species can also produce vesicles that look very different. Sometimes they come off the cell as a chain of spheres; sometimes the spheres remain connected, like a string of pearls; sometimes vesicles form long tubes instead, like from this Borrelia burgdorferi cell. Sometimes the same chain can be tubular in one section (usually at the base, connected to the cell), and a string of spheres in another.</p>
<div class="figure"><span id="fig:unnamed-chunk-12"></span>
<div id="htmlwidget-5d8dcca3d32e76b9445f" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-5d8dcca3d32e76b9445f">{"x":{"doi":"10.22002/D1.1363","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.7: Borrelia burgdorferi 2 Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1363">10.22002/D1.1363</a>
</p>
</div>
</div>
<div id="Inner_membrane_vesicles" class="section level3">
<h3><span class="header-section-number">2.4.2</span> Myxococcus xanthus</h3>
<p>Not all vesicles come from the outer membrane. The cytoplasmic or inner membrane can also form vesicles that are released into the cytoplasm, as in this Myxococcus xanthus cell, or into the periplasm. This seems to be a less regulated process than outer membrane vesicle formation, and we see it in many species when they are stressed by low nutrients or high cell density, suggesting that it is a general phenomenon. Cells shrink in harsh conditions (more on that in Chapter 7), so cytoplasmic or periplasmic vesicles may simply offer a place to put the extra membrane or, more optimistically, to store it until the time comes to grow again. Just as with outer membrane vesicles, the appearance of cytoplasmic vesicles can vary widely <a href="envelope.html#Cytoplasmic_vesicle_variety">More: Cytoplasmic vesicle variety</a>.</p>
<div class="figure"><span id="fig:unnamed-chunk-13"></span>
<div id="htmlwidget-17d7357e8eb37dc86c75" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-17d7357e8eb37dc86c75">{"x":{"doi":"10.22002/D1.1364","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.8: Myxococcus xanthus Collected by: Matthew Swulius <a href="https://doi.org/10.22002/D1.1364">10.22002/D1.1364</a>
</p>
</div>
</div>
<div id="Cytoplasmic_vesicle_variety" class="section level3">
<h3><span class="header-section-number">2.4.3</span> Prosthecobacter debontii</h3>
<p>Cytoplasmic vesicles exhibit a variety of sizes and shapes. Some are nested, with vesicles inside vesicles. In this Prosthecobacter debontii cell, you can see two other morphologies. One is a large, flattened horseshoe-shaped vesicle. Another is a more typical spherical shape, but is decorated with what looks like protein complexes.</p>
<p>This cell also has unusual structures on its surface that have yet to be identified.</p>
<div class="figure"><span id="fig:unnamed-chunk-14"></span>
<div id="htmlwidget-6cd3986d34b899102bbe" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-6cd3986d34b899102bbe">{"x":{"doi":"10.22002/D1.1365","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.9: Prosthecobacter debontii Collected by: Martin Pilhofer <a href="https://doi.org/10.22002/D1.1365">10.22002/D1.1365</a>
</p>
</div>
</div>
</div>
<div id="mycoplasma-marinum" class="section level2">
<h2><span class="header-section-number">2.5</span> Mycoplasma marinum</h2>
<p>Evolution is endlessly creative, providing exceptions to nearly every classification rule. We’ve just described a neat breakdown of bacteria into monoderms (one membrane, thick sacculus, positive Gram stain) and diderms (two membranes, thin sacculus, negative Gram stain). But some cells, like this Mycobacterium marinum, defy classification. Mycobacteria are diderm, with an inner and an outer membrane, and a cell wall. But they have unique molecules (named mycolic acids in their honor) in the outer membrane. These acids interfere with Gram staining, yielding an intermediate result between positive and negative. And their sacculus consists of three layers, each with a unique molecular composition <a href="envelope.html#Mycobacterial_architecture">Schematic – Mycobacterial architecture</a>. In this case, the middle layer is the familiar peptidoglycan.</p>
<p>When you look at the schematic, you’ll see another layer outside the outer membrane. This layer, the capsule, isn’t visible in the movie of the cell, though. It is made up of “Extracellular Polymeric Substance,” abbreviated EPS, or long chains of sugars, sometimes linked to the outer membrane and sometimes free-floating. These sugars don’t interact strongly with electrons, so the capsule is usually invisible by electron microscopy. But in fact, many bacteria (mostly diderm, but also some monoderm) have a capsule. The capsule can help bacteria attach onto surfaces and offers an extra layer of protection, trapping water to prevent desiccation and making it more difficult for hydrophobic molecules like detergents to get through to disrupt the membranes. It also makes it more difficult for viruses to reach the cell, and for eukaryotic “predators” like macrophages to eat it.</p>
<div class="figure"><span id="fig:unnamed-chunk-15"></span>
<div id="htmlwidget-2a37dffd7f4a46af77e0" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-2a37dffd7f4a46af77e0">{"x":{"doi":"10.22002/D1.1354","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.10: Mycobacterium marinum Collected by: Elitza Tocheva <a href="https://doi.org/10.22002/D1.1354">10.22002/D1.1354</a>
</p>
</div>
<div id="Mycobacterial_architecture" class="section level3">
<h3><span class="header-section-number">2.5.1</span> Mycobacterial architecture</h3>
<p><img src="img/02_schematic/2_5_1_Mycobacteria.png" /><!-- --></p>
<p>A unique architecture encloses Mycobacterial cells. Note the outer membrane, with lipids only in the outer leaflet, and mycolic acids forming the inner. Note also the multiple strata of sugars in the cell wall. Also note the capsule depicted outside the outer membrane.</p>
</div>
</div>
<div id="caulobacter-crescentus" class="section level2">
<h2><span class="header-section-number">2.6</span> Caulobacter crescentus</h2>
<p>How else can you protect your cell from the rigors of a harsh world? What about encasing it in an armored shell a la the armadillo? Many bacteria (both monoderms and diderms) and archaea use modular proteins for this purpose, interlocking Lego-block-like pieces into a shell called the Surface Layer, or S-layer. You can guess that this must offer a significant evolutionary advantage since up to 15% of the total protein in the cell can be found in the structure. In fact, S-layers play many roles for cells, some of which you’ll see on the next page, but one of their main functions is as a gatekeeper, preventing large things like viruses from reaching the membrane.</p>
<p>S-layers are crystalline lattices, and they can be striking in appearance, as on this Caulobacter crescentus cell. Amazingly, the lattice is made from (many copies of) a single protein <a href="envelope.html#S-layer_architecture">Schematic – S-layer architecture</a>. The pinwheel-like subunits interact laterally, leaving pores large enough for nutrients to pass through, but not large enough for viruses. The modular nature of the lattice means that units can be popped in as the cell grows, or popped out to allow a cell appendage to poke through. In general, S-layers are quite accommodating; they don’t even interfere with the production of outer membrane vesicles <a href="envelope.html#Nanopods">More: Nanopods</a>. S-layer proteins can also be modified to alter their properties; for instance attachment of a sugar can enable them to stick the cell to a surface.</p>
<div class="figure"><span id="fig:unnamed-chunk-17"></span>
<div id="htmlwidget-39387eeac925a04996a5" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-39387eeac925a04996a5">{"x":{"doi":"10.22002/D1.1355","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.11: Caulobacter crescentus Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1355">10.22002/D1.1355</a>
</p>
</div>
<div id="S-layer_architecture" class="section level3">
<h3><span class="header-section-number">2.6.1</span> S-layer architecture</h3>
<p><img src="img/02_schematic/2_6_1_SLayerTop.png" /><!-- --></p>
<p>A single protein forms the S-layer you just saw in Caulobacter crescentus. The protein has two domains. The bottom domain anchors to lipoproteins attached to the outer membrane. The top domain forms the canopy of the S-layer, organizing hierarchically into hexameric rosettes like this that in turn pack into a larger hexameric lattice. This lattice is flexible enough to curve around even narrow regions of the cell (more on this stalk in Chapter 3).</p>
</div>
<div id="Nanopods" class="section level3">
<h3><span class="header-section-number">2.6.2</span> Delftia acidovorans</h3>
<p>Archaea and diderm bacteria with S-layers produce characteristic outer membrane vesicles: they bud off with the S-layer attached. Delftia acidovorans, like this produce so-called nanopods: chains of outer membrane vesicles ensheathed in S-layer.</p>
<div class="figure"><span id="fig:unnamed-chunk-19"></span>
<div id="htmlwidget-0a0cd4e416db4f014df1" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-0a0cd4e416db4f014df1">{"x":{"doi":"10.22002/D1.1366","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.12: Delftia acidovorans Collected by: Elitza Tocheva <a href="https://doi.org/10.22002/D1.1366">10.22002/D1.1366</a>
</p>
</div>
</div>
</div>
<div id="sulfolobus-solfataricus" class="section level2">
<h2><span class="header-section-number">2.7</span> Sulfolobus solfataricus</h2>
<p>One of the most striking features of the S-layer is how different it can look in different species. For instance, compare this archaeal Sulfolobus solfataricus cell to the bacterium on the last page, or to other diderm <a href="envelope.html#M._alcaliphilum">More: M. alcaliphilum</a> or monoderm <a href="#">More: C. thermocellum</a> bacteria, or archaea <a href="envelope.html#N._maritimus">More: N. maritimus</a> <a href="envelope.html#Methanoregula_formicica">More: Methanoregula formicica</a>. All S-layers are crystalline lattices of a single–or in a few cases, two–proteins, but the particular pattern of the lattice depends on the shape of this building block and how it multimerizes into a higher-order structure. The shape of the building block varies considerably; there is almost no sequence homology between S-layer proteins from different species. And shapes come together in different ways, forming repeating units of one, two, three (as on this cell), four, or six blocks.</p>
<p>S-layers are very common in archaea like this cell. Most archaea don’t have cell walls, but the S-layer provides the same function of external scaffolding. Remember, too, that nearly all archaea are monoderms, lacking the extra periplasmic compartment that diderms have. Here again the S-layer serves a similar function, enclosing a space around the cell’s membrane called the pseudo-periplasmic space. Just as with the bacterial periplasm, this space serves as an antechamber for the cell, restricting access by large molecules. In some cases, the pseudo-periplasmic space also contains specific proteins that function in metabolism.</p>
<div class="figure"><span id="fig:unnamed-chunk-20"></span>
<div id="htmlwidget-9899f117343f132f5cc3" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-9899f117343f132f5cc3">{"x":{"doi":"10.22002/D1.1356","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.13: Sulfolobus solfataricus Collected by: Lu Gan <a href="https://doi.org/10.22002/D1.1356">10.22002/D1.1356</a>
</p>
</div>
<div id="M._alcaliphilum" class="section level3">
<h3><span class="header-section-number">2.7.1</span> Methylomicrobium alcaliphilum</h3>
<p>In Methylomicrobium alcaliphilum, V-shaped S-layer proteins come together to form cups that pack into a hexagonal pattern.</p>
<div class="figure"><span id="fig:unnamed-chunk-21"></span>
<div id="htmlwidget-81c70fb8657ce92a45b5" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-81c70fb8657ce92a45b5">{"x":{"doi":"10.22002/D1.1367","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.14: Methylomicrobium alcaliphilum Collected by: Songye Chen <a href="https://doi.org/10.22002/D1.1367">10.22002/D1.1367</a>
</p>
</div>
</div>
<div id="N._maritimus" class="section level3">
<h3><span class="header-section-number">2.7.2</span> Nitrosopumilis maritimus</h3>
<p>In Nitrosopumilis maritimus, S-layer proteins form hexagonal rosettes that in turn pack into a hexagonal lattice.</p>
<div class="figure"><span id="fig:unnamed-chunk-22"></span>
<div id="htmlwidget-a0604e1856d4a2b1066d" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-a0604e1856d4a2b1066d">{"x":{"doi":"10.22002/D1.1368","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.15: Nitrosopumilis maritimus Collected by: Rasika Ramdasi <a href="https://doi.org/10.22002/D1.1368">10.22002/D1.1368</a>
</p>
</div>
</div>
<div id="Methanoregula_formicica" class="section level3">
<h3><span class="header-section-number">2.7.3</span> Methanoregula formicica</h3>
<div class="figure"><span id="fig:unnamed-chunk-23"></span>
<div id="htmlwidget-44ba4c6313818242a4fa" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-44ba4c6313818242a4fa">{"x":{"doi":"10.22002/D1.1369","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.16: Methanoregula formicica Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1369">10.22002/D1.1369</a>
</p>
</div>
</div>
</div>
<div id="methanospirillum-hungatei" class="section level2">
<h2><span class="header-section-number">2.8</span> Methanospirillum hungatei</h2>
<p>Why stop with a single layer of protein? For proof that Nature is endlessly inventive, consider this Methanospirillum hungatei cell. These archaea encase themselves in an S-layer, and then an additional protein layer that forms a highly impermeable sheath. The sheath is also very resistant to pressure, which could be important in the cells’ line of work. M. hungatei were discovered in sewage, where they break down organic waste, producing methane. One theory is that the sheath acts as a pressure regulator; when enough methane builds up inside the cell, the pressure expands the sheath, opening its pores wide enough to allow the methane to dissipate and new metabolic substrates like hydrogen and carbon dioxide to enter.</p>
<p>The rules of architecture remain the same, though. Just as in the bacterial cell wall, sheath polymers are arranged as hoops perpendicular to the long axis of the rod-shaped cell. The ends of the sheath, as you can see, are special, with multiple protein layers stacking into a thick plug. Cells divide within the sheath, and long chains of cells in a continuous sheath are often observed.</p>
<div class="figure"><span id="fig:unnamed-chunk-24"></span>
<div id="htmlwidget-3546ad056934a25cf535" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-3546ad056934a25cf535">{"x":{"doi":"10.22002/D1.1357","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.17: Methanospirillum hungatei Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1357">10.22002/D1.1357</a>
</p>
</div>
</div>
<div id="haloarcula-argentinensis" class="section level2">
<h2><span class="header-section-number">2.9</span> Haloarcula argentinensis</h2>
<p>All the layers we’ve just discussed collectively make up the container, or envelope, of a cell. As you’ve seen, different species use different combinations of these components to form their envelopes; the only constant is the cytoplasmic (or inner, for diderms) membrane. Before we move on, let’s talk briefly about what these envelopes contain. In addition to water and small molecules, you’ve already seen some large protein complexes like motility machineries. You’ve also seen the ribosomes – the protein/RNA complexes responsible for translation. But you might have been surprised not to see something else: DNA. The replicating molecule containing the instructions for the life of the cell is of paramount importance but often invisible by microscopy. But not always.</p>
<p>Thin filaments of DNA, only about 2 nm wide, blend in with the dense cytoplasm of the cell. When a cell lyses, though, its cytoplasm diffuses into the environment and the DNA filaments stand out against the now-much-reduced background. You can get an idea of the sheer abundance of DNA in a cell from this Haloarcula argentinensis where the envelope has ruptured and the contents are spilling out.</p>
<div class="figure"><span id="fig:unnamed-chunk-25"></span>
<div id="htmlwidget-405226122a212250f287" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-405226122a212250f287">{"x":{"doi":"10.22002/D1.1358","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.18: Haloarcula argentinensis Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1358">10.22002/D1.1358</a>
</p>
</div>
</div>
<div id="bdellovibrio-bacteriovorus" class="section level2">
<h2><span class="header-section-number">2.10</span> Bdellovibrio bacteriovorus</h2>
<p>Cells contain enormous amounts of DNA. The single, circular chromosome of this Bdellovibrio bacteriovorus cell contains 3,782,950 individual base pairs, which means that if the circle were cut and laid out as a long piece, it would be about one thousand times longer than the cell itself. To fit and function inside the cell, the chromosome has to be extraordinarily organized and packed, a feat we still don’t understand. You can see some of this packing in nearly every cell: the center of the cell tends to have very few large macromolecular complexes like ribosomes, because they’re excluded by the densely-packed chromosome(s). Watch out for these ribosome-excluding zones in the rest of the book; they indicate the location of the bulk of the cell’s DNA. Since bacteria and archaea don’t enclose their DNA in a subcellular membrane, we don’t call this region a nucleus (the “karyon” that defines eukaryotes). Instead, we use the term nucleoid to describe the cytoplasmic region where most of the DNA is concentrated.</p>
<p>At times, the nucleoid becomes easier to see. Imagine that you want to decrease gene expression in your cell. Don’t worry yet about why – we’ll discuss that in Chapter 8 – for now, just think about how. One approach is simply to pack the chromosome so tightly that the transcriptional machinery can’t access the genes. That’s what this cell is doing, condensing its nucleoid into a dense twisted braid we can easily visualize.</p>
<div class="figure"><span id="fig:unnamed-chunk-26"></span>
<div id="htmlwidget-daf899eee48cf8b03316" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-daf899eee48cf8b03316">{"x":{"doi":"10.22002/D1.1359","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 2.19: Bdellovibrio bacteriovorus Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1359">10.22002/D1.1359</a>
</p>
</div>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="methods.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="shape.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": null,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/caltechlibrary/cell-atlas/edit/master/02_Envelope.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["cell-atlas.pdf", "cell-atlas.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
