<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 9 Interaction | Cell Atlas</title>
  <meta name="description" content="An atlas produced by the Jensen Lab." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 9 Interaction | Cell Atlas" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="splash/HeaderImage.jpg" />
  <meta property="og:description" content="An atlas produced by the Jensen Lab." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 9 Interaction | Cell Atlas" />
  
  <meta name="twitter:description" content="An atlas produced by the Jensen Lab." />
  <meta name="twitter:image" content="splash/HeaderImage.jpg" />

<meta name="author" content="Jensen Lab" />


<meta name="date" content="2020-08-03" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="lifecycle.html"/>
<link rel="next" href="viruses.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/CL-0.1.2/CL.js"></script>
<script src="libs/doivideo-binding-0.0.0.9000/doivideo.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Cell Atlas</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>1</b> Methods</a><ul>
<li class="chapter" data-level="1.1" data-path="methods.html"><a href="methods.html#light-microscopy"><i class="fa fa-check"></i><b>1.1</b> Light Microscopy</a><ul>
<li class="chapter" data-level="1.1.1" data-path="methods.html"><a href="methods.html#Structural_biology_toolkit"><i class="fa fa-check"></i><b>1.1.1</b> Structural biology toolkit</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="methods.html"><a href="methods.html#fluorescence-light-microscopy"><i class="fa fa-check"></i><b>1.2</b> Fluorescence Light Microscopy</a></li>
<li class="chapter" data-level="1.3" data-path="methods.html"><a href="methods.html#scanning-electron-microscopy"><i class="fa fa-check"></i><b>1.3</b> Scanning Electron Microscopy</a><ul>
<li class="chapter" data-level="1.3.1" data-path="methods.html"><a href="methods.html#Electron_microscopy_modes"><i class="fa fa-check"></i><b>1.3.1</b> Electron microscopy modes</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="methods.html"><a href="methods.html#transmission-electron-microscopy"><i class="fa fa-check"></i><b>1.4</b> Transmission Electron Microscopy</a></li>
<li class="chapter" data-level="1.5" data-path="methods.html"><a href="methods.html#cryogenic-electron-microscopy"><i class="fa fa-check"></i><b>1.5</b> Cryogenic Electron Microscopy</a></li>
<li class="chapter" data-level="1.6" data-path="methods.html"><a href="methods.html#electron-cryotomography"><i class="fa fa-check"></i><b>1.6</b> Electron Cryotomography</a><ul>
<li class="chapter" data-level="1.6.1" data-path="methods.html"><a href="methods.html#Missing_wedge"><i class="fa fa-check"></i><b>1.6.1</b> Missing wedge</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="methods.html"><a href="methods.html#sub-tomogram-averaging"><i class="fa fa-check"></i><b>1.7</b> Sub-Tomogram Averaging</a></li>
<li class="chapter" data-level="1.8" data-path="methods.html"><a href="methods.html#single-particle-reconstruction"><i class="fa fa-check"></i><b>1.8</b> Single Particle Reconstruction</a></li>
<li class="chapter" data-level="1.9" data-path="methods.html"><a href="methods.html#x-ray-crystallography"><i class="fa fa-check"></i><b>1.9</b> X-Ray Crystallography</a><ul>
<li class="chapter" data-level="1.9.1" data-path="methods.html"><a href="methods.html#Structural_biology_timeline"><i class="fa fa-check"></i><b>1.9.1</b> Structural biology timeline</a></li>
</ul></li>
<li class="chapter" data-level="1.10" data-path="methods.html"><a href="methods.html#putting-it-all-together"><i class="fa fa-check"></i><b>1.10</b> Putting It All Together</a></li>
<li class="chapter" data-level="1.11" data-path="methods.html"><a href="methods.html#moredifference-mapping"><i class="fa fa-check"></i><b>1.11</b> More–Difference mapping</a></li>
<li class="chapter" data-level="1.12" data-path="methods.html"><a href="methods.html#further-reading"><i class="fa fa-check"></i><b>1.12</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="envelope.html"><a href="envelope.html"><i class="fa fa-check"></i><b>2</b> Envelope</a><ul>
<li class="chapter" data-level="2.1" data-path="envelope.html"><a href="envelope.html#membrane"><i class="fa fa-check"></i><b>2.1</b> Membrane</a><ul>
<li class="chapter" data-level="2.1.1" data-path="envelope.html"><a href="envelope.html#Lipid_bilayer"><i class="fa fa-check"></i><b>2.1.1</b> Lipid bilayer</a></li>
<li class="chapter" data-level="2.1.2" data-path="envelope.html"><a href="envelope.html#PDB:_5T4O_ATP_synthase_and_energy_production"><i class="fa fa-check"></i><b>2.1.2</b> PDB: 5T4O ATP synthase and energy production</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="envelope.html"><a href="envelope.html#cell-wall"><i class="fa fa-check"></i><b>2.2</b> Cell Wall</a></li>
<li class="chapter" data-level="2.3" data-path="envelope.html"><a href="envelope.html#morepeptidoglycan-architecture"><i class="fa fa-check"></i><b>2.3</b> More–Peptidoglycan architecture</a></li>
<li class="chapter" data-level="2.4" data-path="envelope.html"><a href="envelope.html#moremethanobacterium-formicicum"><i class="fa fa-check"></i><b>2.4</b> More–Methanobacterium formicicum</a></li>
<li class="chapter" data-level="2.5" data-path="envelope.html"><a href="envelope.html#outer-membrane"><i class="fa fa-check"></i><b>2.5</b> Outer Membrane</a><ul>
<li class="chapter" data-level="2.5.1" data-path="envelope.html"><a href="envelope.html#pdb-1eq7-brauns-lipoprotein-pdb1eq7brauns_lipoprotein"><i class="fa fa-check"></i><b>2.5.1</b> PDB: 1EQ7 –Braun’s lipoprotein {#PDB:<em>1EQ7</em>–Brauns_lipoprotein}</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="envelope.html"><a href="envelope.html#morediderm-sacculus-architecture"><i class="fa fa-check"></i><b>2.6</b> More–Diderm sacculus architecture</a><ul>
<li class="chapter" data-level="2.6.1" data-path="envelope.html"><a href="envelope.html#Sacculus_remodeling"><i class="fa fa-check"></i><b>2.6.1</b> Sacculus remodeling</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="envelope.html"><a href="envelope.html#vesicles"><i class="fa fa-check"></i><b>2.7</b> Vesicles</a></li>
<li class="chapter" data-level="2.8" data-path="envelope.html"><a href="envelope.html#morepearled-vesicles"><i class="fa fa-check"></i><b>2.8</b> More–Pearled vesicles</a></li>
<li class="chapter" data-level="2.9" data-path="envelope.html"><a href="envelope.html#moretubular-vesicles"><i class="fa fa-check"></i><b>2.9</b> More–Tubular vesicles</a></li>
<li class="chapter" data-level="2.10" data-path="envelope.html"><a href="envelope.html#morecytoplasmic-vesicles"><i class="fa fa-check"></i><b>2.10</b> More–Cytoplasmic vesicles</a></li>
<li class="chapter" data-level="2.11" data-path="envelope.html"><a href="envelope.html#more-cytoplasmic-vesicle-variety"><i class="fa fa-check"></i><b>2.11</b> More: Cytoplasmic vesicle variety</a></li>
<li class="chapter" data-level="2.12" data-path="envelope.html"><a href="envelope.html#morearchaeal-vesicles"><i class="fa fa-check"></i><b>2.12</b> More–Archaeal vesicles</a></li>
<li class="chapter" data-level="2.13" data-path="envelope.html"><a href="envelope.html#variation"><i class="fa fa-check"></i><b>2.13</b> Variation</a></li>
<li class="chapter" data-level="2.14" data-path="envelope.html"><a href="envelope.html#surface-layer"><i class="fa fa-check"></i><b>2.14</b> Surface Layer</a><ul>
<li class="chapter" data-level="2.14.1" data-path="envelope.html"><a href="envelope.html#PDB:_5N97_Surface_layer_architecture"><i class="fa fa-check"></i><b>2.14.1</b> PDB: 5N97 Surface layer architecture</a></li>
</ul></li>
<li class="chapter" data-level="2.15" data-path="envelope.html"><a href="envelope.html#morenanopods"><i class="fa fa-check"></i><b>2.15</b> More–Nanopods</a></li>
<li class="chapter" data-level="2.16" data-path="envelope.html"><a href="envelope.html#surface-layer-variety"><i class="fa fa-check"></i><b>2.16</b> Surface Layer Variety</a></li>
<li class="chapter" data-level="2.17" data-path="envelope.html"><a href="envelope.html#morem.-alcaliphilum"><i class="fa fa-check"></i><b>2.17</b> More–M. alcaliphilum</a></li>
<li class="chapter" data-level="2.18" data-path="envelope.html"><a href="envelope.html#moreclostridium-thermocellum"><i class="fa fa-check"></i><b>2.18</b> More–Clostridium thermocellum</a></li>
<li class="chapter" data-level="2.19" data-path="envelope.html"><a href="envelope.html#moren.-maritimus"><i class="fa fa-check"></i><b>2.19</b> More–N. maritimus</a></li>
<li class="chapter" data-level="2.20" data-path="envelope.html"><a href="envelope.html#moremethanoregula-formicica"><i class="fa fa-check"></i><b>2.20</b> More–Methanoregula formicica</a></li>
<li class="chapter" data-level="2.21" data-path="envelope.html"><a href="envelope.html#sheath"><i class="fa fa-check"></i><b>2.21</b> Sheath</a></li>
<li class="chapter" data-level="2.22" data-path="envelope.html"><a href="envelope.html#dna"><i class="fa fa-check"></i><b>2.22</b> DNA</a></li>
<li class="chapter" data-level="2.23" data-path="envelope.html"><a href="envelope.html#nucleoid"><i class="fa fa-check"></i><b>2.23</b> Nucleoid</a></li>
<li class="chapter" data-level="2.24" data-path="envelope.html"><a href="envelope.html#further-reading-1"><i class="fa fa-check"></i><b>2.24</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="shape.html"><a href="shape.html"><i class="fa fa-check"></i><b>3</b> Shape</a><ul>
<li class="chapter" data-level="3.1" data-path="shape.html"><a href="shape.html#coccoid"><i class="fa fa-check"></i><b>3.1</b> Coccoid</a></li>
<li class="chapter" data-level="3.2" data-path="shape.html"><a href="shape.html#rod"><i class="fa fa-check"></i><b>3.2</b> Rod</a></li>
<li class="chapter" data-level="3.3" data-path="shape.html"><a href="shape.html#morerod-variety"><i class="fa fa-check"></i><b>3.3</b> More–Rod variety</a></li>
<li class="chapter" data-level="3.4" data-path="shape.html"><a href="shape.html#morearchaeal-rods"><i class="fa fa-check"></i><b>3.4</b> More–Archaeal rods</a></li>
<li class="chapter" data-level="3.5" data-path="shape.html"><a href="shape.html#length"><i class="fa fa-check"></i><b>3.5</b> Length</a></li>
<li class="chapter" data-level="3.6" data-path="shape.html"><a href="shape.html#vibrioid"><i class="fa fa-check"></i><b>3.6</b> Vibrioid</a><ul>
<li class="chapter" data-level="3.6.1" data-path="shape.html"><a href="shape.html#PDB:_5U3C_CTP_synthase"><i class="fa fa-check"></i><b>3.6.1</b> PDB: 5U3C CTP synthase</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="shape.html"><a href="shape.html#helical"><i class="fa fa-check"></i><b>3.7</b> Helical</a></li>
<li class="chapter" data-level="3.8" data-path="shape.html"><a href="shape.html#moreborrelia-burgdorferi"><i class="fa fa-check"></i><b>3.8</b> More–Borrelia burgdorferi</a></li>
<li class="chapter" data-level="3.9" data-path="shape.html"><a href="shape.html#prosthecate"><i class="fa fa-check"></i><b>3.9</b> Prosthecate</a><ul>
<li class="chapter" data-level="3.9.1" data-path="shape.html"><a href="shape.html#PDB:_6RIB_Bactofilin"><i class="fa fa-check"></i><b>3.9.1</b> PDB: 6RIB Bactofilin</a></li>
</ul></li>
<li class="chapter" data-level="3.10" data-path="shape.html"><a href="shape.html#morebacterial-microtubules"><i class="fa fa-check"></i><b>3.10</b> More–Bacterial microtubules</a></li>
<li class="chapter" data-level="3.11" data-path="shape.html"><a href="shape.html#square"><i class="fa fa-check"></i><b>3.11</b> Square</a></li>
<li class="chapter" data-level="3.12" data-path="shape.html"><a href="shape.html#moregas-vesicles"><i class="fa fa-check"></i><b>3.12</b> More–Gas vesicles</a></li>
<li class="chapter" data-level="3.13" data-path="shape.html"><a href="shape.html#further-reading-2"><i class="fa fa-check"></i><b>3.13</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="growth.html"><a href="growth.html"><i class="fa fa-check"></i><b>4</b> Growth</a><ul>
<li class="chapter" data-level="4.1" data-path="growth.html"><a href="growth.html#diffusion"><i class="fa fa-check"></i><b>4.1</b> Diffusion</a></li>
<li class="chapter" data-level="4.2" data-path="growth.html"><a href="growth.html#anaerobic-respiration"><i class="fa fa-check"></i><b>4.2</b> Anaerobic Respiration</a></li>
<li class="chapter" data-level="4.3" data-path="growth.html"><a href="growth.html#photosynthesis"><i class="fa fa-check"></i><b>4.3</b> Photosynthesis</a><ul>
<li class="chapter" data-level="4.3.1" data-path="growth.html"><a href="growth.html#pdb-6kgx-photocomplexes-pdb6kgxphotocomplexes"><i class="fa fa-check"></i><b>4.3.1</b> PDB: 6KGX –Photocomplexes {#PDB:<em>6KGX</em>–Photocomplexes}</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="growth.html"><a href="growth.html#moreicm-variety"><i class="fa fa-check"></i><b>4.4</b> More–ICM variety</a></li>
<li class="chapter" data-level="4.5" data-path="growth.html"><a href="growth.html#photosynthesis-contd."><i class="fa fa-check"></i><b>4.5</b> Photosynthesis (cont’d.)</a></li>
<li class="chapter" data-level="4.6" data-path="growth.html"><a href="growth.html#enzyme-regulation"><i class="fa fa-check"></i><b>4.6</b> Enzyme Regulation</a><ul>
<li class="chapter" data-level="4.6.1" data-path="growth.html"><a href="growth.html#pdb-6tqhadhe-spirosome-structure-pdb_6tqhadhe_spirosome_structure"><i class="fa fa-check"></i><b>4.6.1</b> PDB: 6TQH–AdhE spirosome structure {#PDB:_6TQH–AdhE_spirosome_structure} <img src="img/schematics/4_5_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="growth.html"><a href="growth.html#microcompartments"><i class="fa fa-check"></i><b>4.7</b> Microcompartments</a><ul>
<li class="chapter" data-level="4.7.1" data-path="growth.html"><a href="growth.html#pdb-3dktmicrocompartment-shells-pdb_3dktmicrocompartment_shells"><i class="fa fa-check"></i><b>4.7.1</b> PDB: 3DKT–Microcompartment shells {#PDB:_3DKT–Microcompartment_shells} <img src="img/schematics/4_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="growth.html"><a href="growth.html#carboxysomes"><i class="fa fa-check"></i><b>4.8</b> Carboxysomes</a></li>
<li class="chapter" data-level="4.9" data-path="growth.html"><a href="growth.html#morecarboxysome-variety"><i class="fa fa-check"></i><b>4.9</b> More–Carboxysome variety</a></li>
<li class="chapter" data-level="4.10" data-path="growth.html"><a href="growth.html#morelong-carboxysomes"><i class="fa fa-check"></i><b>4.10</b> More–Long carboxysomes</a></li>
<li class="chapter" data-level="4.11" data-path="growth.html"><a href="growth.html#storage"><i class="fa fa-check"></i><b>4.11</b> Storage</a></li>
<li class="chapter" data-level="4.12" data-path="growth.html"><a href="growth.html#morestorage-granule-growth"><i class="fa fa-check"></i><b>4.12</b> More–Storage granule growth</a></li>
<li class="chapter" data-level="4.13" data-path="growth.html"><a href="growth.html#storage-contd."><i class="fa fa-check"></i><b>4.13</b> Storage (cont’d.)</a></li>
<li class="chapter" data-level="4.14" data-path="growth.html"><a href="growth.html#moregranule-containing-carboxysomes"><i class="fa fa-check"></i><b>4.14</b> More–Granule-containing carboxysomes</a></li>
<li class="chapter" data-level="4.15" data-path="growth.html"><a href="growth.html#morecarboxysomes-granule-connections"><i class="fa fa-check"></i><b>4.15</b> More–Carboxysomes-granule connections</a></li>
<li class="chapter" data-level="4.16" data-path="growth.html"><a href="growth.html#archaeal-storage-granules"><i class="fa fa-check"></i><b>4.16</b> Archaeal Storage Granules</a></li>
<li class="chapter" data-level="4.17" data-path="growth.html"><a href="growth.html#morehaloferax-gibbonsii-granules"><i class="fa fa-check"></i><b>4.17</b> More–Haloferax gibbonsii granules</a></li>
<li class="chapter" data-level="4.18" data-path="growth.html"><a href="growth.html#morehalohasta-litchfieldiae-granules"><i class="fa fa-check"></i><b>4.18</b> More–Halohasta litchfieldiae granules</a></li>
<li class="chapter" data-level="4.19" data-path="growth.html"><a href="growth.html#further-reading-3"><i class="fa fa-check"></i><b>4.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="division.html"><a href="division.html"><i class="fa fa-check"></i><b>5</b> Division</a><ul>
<li class="chapter" data-level="5.1" data-path="division.html"><a href="division.html#segregation"><i class="fa fa-check"></i><b>5.1</b> Segregation</a></li>
<li class="chapter" data-level="5.2" data-path="division.html"><a href="division.html#chromosome-segregation"><i class="fa fa-check"></i><b>5.2</b> Chromosome Segregation</a></li>
<li class="chapter" data-level="5.3" data-path="division.html"><a href="division.html#budding"><i class="fa fa-check"></i><b>5.3</b> Budding</a></li>
<li class="chapter" data-level="5.4" data-path="division.html"><a href="division.html#morehyphomonas-stalk"><i class="fa fa-check"></i><b>5.4</b> More–Hyphomonas stalk</a></li>
<li class="chapter" data-level="5.5" data-path="division.html"><a href="division.html#morehyphomonas-lifecycle"><i class="fa fa-check"></i><b>5.5</b> More–Hyphomonas lifecycle</a></li>
<li class="chapter" data-level="5.6" data-path="division.html"><a href="division.html#monoderm-cytokinesis"><i class="fa fa-check"></i><b>5.6</b> Monoderm Cytokinesis</a></li>
<li class="chapter" data-level="5.7" data-path="division.html"><a href="division.html#monoderm-cytokinesis-contd."><i class="fa fa-check"></i><b>5.7</b> Monoderm Cytokinesis (cont’d.)</a></li>
<li class="chapter" data-level="5.8" data-path="division.html"><a href="division.html#diderm-cytokinesis"><i class="fa fa-check"></i><b>5.8</b> Diderm Cytokinesis</a></li>
<li class="chapter" data-level="5.9" data-path="division.html"><a href="division.html#moresphingopyxis-alaskensis-division"><i class="fa fa-check"></i><b>5.9</b> More–Sphingopyxis alaskensis division</a></li>
<li class="chapter" data-level="5.10" data-path="division.html"><a href="division.html#morechloroflexi-division"><i class="fa fa-check"></i><b>5.10</b> More–Chloroflexi division</a></li>
<li class="chapter" data-level="5.11" data-path="division.html"><a href="division.html#diderm-cytokinesis-contd."><i class="fa fa-check"></i><b>5.11</b> Diderm Cytokinesis (cont’d.)</a></li>
<li class="chapter" data-level="5.12" data-path="division.html"><a href="division.html#morecaulobacter-division"><i class="fa fa-check"></i><b>5.12</b> More–Caulobacter division</a></li>
<li class="chapter" data-level="5.13" data-path="division.html"><a href="division.html#budding-contd."><i class="fa fa-check"></i><b>5.13</b> Budding (cont’d.)</a></li>
<li class="chapter" data-level="5.14" data-path="division.html"><a href="division.html#ftsz"><i class="fa fa-check"></i><b>5.14</b> FtsZ</a></li>
<li class="chapter" data-level="5.15" data-path="division.html"><a href="division.html#moreasymmetric-constriction"><i class="fa fa-check"></i><b>5.15</b> More–Asymmetric constriction</a><ul>
<li class="chapter" data-level="5.15.1" data-path="division.html"><a href="division.html#pdb-5mn4-ftsz-structure-pdb5mn4ftsz_structure"><i class="fa fa-check"></i><b>5.15.1</b> PDB: 5MN4 –FtsZ structure {#PDB:<em>5MN4</em>–FtsZ_structure}</a></li>
</ul></li>
<li class="chapter" data-level="5.16" data-path="division.html"><a href="division.html#ftsz-contd."><i class="fa fa-check"></i><b>5.16</b> FtsZ (cont’d.)</a></li>
<li class="chapter" data-level="5.17" data-path="division.html"><a href="division.html#escrt"><i class="fa fa-check"></i><b>5.17</b> ESCRT</a></li>
<li class="chapter" data-level="5.18" data-path="division.html"><a href="division.html#escrt-contd."><i class="fa fa-check"></i><b>5.18</b> ESCRT (cont’d.)</a></li>
<li class="chapter" data-level="5.19" data-path="division.html"><a href="division.html#further-reading-4"><i class="fa fa-check"></i><b>5.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="motility.html"><a href="motility.html"><i class="fa fa-check"></i><b>6</b> Motility</a><ul>
<li class="chapter" data-level="6.1" data-path="motility.html"><a href="motility.html#flagellum"><i class="fa fa-check"></i><b>6.1</b> Flagellum</a><ul>
<li class="chapter" data-level="6.1.1" data-path="motility.html"><a href="motility.html#pdb-5wjtflagellum-structure-pdb_5wjtflagellum_structure"><i class="fa fa-check"></i><b>6.1.1</b> PDB: 5WJT–Flagellum structure {#PDB:_5WJT–Flagellum_structure} <img src="img/schematics/6_1_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="motility.html"><a href="motility.html#flagellar-motor"><i class="fa fa-check"></i><b>6.2</b> Flagellar Motor</a></li>
<li class="chapter" data-level="6.3" data-path="motility.html"><a href="motility.html#moreflagellar-motor-structure"><i class="fa fa-check"></i><b>6.3</b> More–Flagellar motor structure</a><ul>
<li class="chapter" data-level="6.3.1" data-path="motility.html"><a href="motility.html#Flagellar_motor_diversity"><i class="fa fa-check"></i><b>6.3.1</b> Flagellar motor diversity</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="motility.html"><a href="motility.html#flagellar-assembly"><i class="fa fa-check"></i><b>6.4</b> Flagellar Assembly</a></li>
<li class="chapter" data-level="6.5" data-path="motility.html"><a href="motility.html#moreflagellar-motor-disassembly"><i class="fa fa-check"></i><b>6.5</b> More–Flagellar motor disassembly</a></li>
<li class="chapter" data-level="6.6" data-path="motility.html"><a href="motility.html#flagella-patterns"><i class="fa fa-check"></i><b>6.6</b> Flagella Patterns</a></li>
<li class="chapter" data-level="6.7" data-path="motility.html"><a href="motility.html#morelophotrichous-bacteria"><i class="fa fa-check"></i><b>6.7</b> More–Lophotrichous bacteria</a></li>
<li class="chapter" data-level="6.8" data-path="motility.html"><a href="motility.html#flagella-patterns-contd."><i class="fa fa-check"></i><b>6.8</b> Flagella Patterns (cont’d.)</a></li>
<li class="chapter" data-level="6.9" data-path="motility.html"><a href="motility.html#moreproteus-mirabilis-flagella"><i class="fa fa-check"></i><b>6.9</b> More–Proteus mirabilis flagella</a></li>
<li class="chapter" data-level="6.10" data-path="motility.html"><a href="motility.html#flagellar-sheath"><i class="fa fa-check"></i><b>6.10</b> Flagellar Sheath</a></li>
<li class="chapter" data-level="6.11" data-path="motility.html"><a href="motility.html#periplasmic-flagella"><i class="fa fa-check"></i><b>6.11</b> Periplasmic Flagella</a></li>
<li class="chapter" data-level="6.12" data-path="motility.html"><a href="motility.html#moretreponema-primitia"><i class="fa fa-check"></i><b>6.12</b> More–Treponema primitia</a></li>
<li class="chapter" data-level="6.13" data-path="motility.html"><a href="motility.html#archaellum"><i class="fa fa-check"></i><b>6.13</b> Archaellum</a><ul>
<li class="chapter" data-level="6.13.1" data-path="motility.html"><a href="motility.html#pdb-5tfyarchaellum-structure-pdb_5tfyarchaellum_structure"><i class="fa fa-check"></i><b>6.13.1</b> PDB: 5TFY–Archaellum structure {#PDB:_5TFY–Archaellum_structure} <img src="img/schematics/6_8_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="6.14" data-path="motility.html"><a href="motility.html#archaella-patterns"><i class="fa fa-check"></i><b>6.14</b> Archaella Patterns</a></li>
<li class="chapter" data-level="6.15" data-path="motility.html"><a href="motility.html#morecone-structure"><i class="fa fa-check"></i><b>6.15</b> More–Cone structure</a></li>
<li class="chapter" data-level="6.16" data-path="motility.html"><a href="motility.html#moreorganizing-center"><i class="fa fa-check"></i><b>6.16</b> More–Organizing center</a></li>
<li class="chapter" data-level="6.17" data-path="motility.html"><a href="motility.html#type-iv-pilus"><i class="fa fa-check"></i><b>6.17</b> Type IV Pilus</a><ul>
<li class="chapter" data-level="6.17.1" data-path="motility.html"><a href="motility.html#Type_IV_pilus_structure"><i class="fa fa-check"></i><b>6.17.1</b> Type IV pilus structure</a></li>
</ul></li>
<li class="chapter" data-level="6.18" data-path="motility.html"><a href="motility.html#type-ix-secretion-system"><i class="fa fa-check"></i><b>6.18</b> Type IX Secretion System</a></li>
<li class="chapter" data-level="6.19" data-path="motility.html"><a href="motility.html#terminal-organelle"><i class="fa fa-check"></i><b>6.19</b> Terminal Organelle</a></li>
<li class="chapter" data-level="6.20" data-path="motility.html"><a href="motility.html#further-reading-5"><i class="fa fa-check"></i><b>6.20</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="navigation.html"><a href="navigation.html"><i class="fa fa-check"></i><b>7</b> Navigation</a><ul>
<li class="chapter" data-level="7.1" data-path="navigation.html"><a href="navigation.html#chemotaxis"><i class="fa fa-check"></i><b>7.1</b> Chemotaxis</a><ul>
<li class="chapter" data-level="7.1.1" data-path="navigation.html"><a href="navigation.html#pdb-2ch7chemoreceptor-structure-pdb_2ch7chemoreceptor_structure"><i class="fa fa-check"></i><b>7.1.1</b> PDB: 2CH7–Chemoreceptor structure {#PDB:_2CH7–Chemoreceptor_structure} <img src="img/schematics/7_1_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="navigation.html"><a href="navigation.html#chemosensory-array"><i class="fa fa-check"></i><b>7.2</b> Chemosensory Array</a><ul>
<li class="chapter" data-level="7.2.1" data-path="navigation.html"><a href="navigation.html#PDB:_6S1K_Chemosensory_array_architecture"><i class="fa fa-check"></i><b>7.2.1</b> PDB: 6S1K Chemosensory array architecture</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="navigation.html"><a href="navigation.html#chemosensory-array-conservation"><i class="fa fa-check"></i><b>7.3</b> Chemosensory Array Conservation</a></li>
<li class="chapter" data-level="7.4" data-path="navigation.html"><a href="navigation.html#chemoreceptors"><i class="fa fa-check"></i><b>7.4</b> Chemoreceptors</a></li>
<li class="chapter" data-level="7.5" data-path="navigation.html"><a href="navigation.html#moreazospirillum-brasilense-aerotaxis"><i class="fa fa-check"></i><b>7.5</b> More–Azospirillum brasilense aerotaxis</a></li>
<li class="chapter" data-level="7.6" data-path="navigation.html"><a href="navigation.html#cytoplasmic-chemosensory-array"><i class="fa fa-check"></i><b>7.6</b> Cytoplasmic Chemosensory Array</a></li>
<li class="chapter" data-level="7.7" data-path="navigation.html"><a href="navigation.html#morevibrio-cholerae-chemosensory-arrays"><i class="fa fa-check"></i><b>7.7</b> More–Vibrio cholerae chemosensory arrays</a></li>
<li class="chapter" data-level="7.8" data-path="navigation.html"><a href="navigation.html#magnetotaxis"><i class="fa fa-check"></i><b>7.8</b> Magnetotaxis</a><ul>
<li class="chapter" data-level="7.8.1" data-path="navigation.html"><a href="navigation.html#pdb-5jygmamk-structure-pdb_5jygmamk_structure"><i class="fa fa-check"></i><b>7.8.1</b> PDB: 5JYG–MamK structure {#PDB:_5JYG–MamK_structure} <img src="img/schematics/7_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="navigation.html"><a href="navigation.html#further-reading-6"><i class="fa fa-check"></i><b>7.9</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="lifecycle.html"><a href="lifecycle.html"><i class="fa fa-check"></i><b>8</b> Lifecycle</a><ul>
<li class="chapter" data-level="8.1" data-path="lifecycle.html"><a href="lifecycle.html#stationary-phase"><i class="fa fa-check"></i><b>8.1</b> Stationary Phase</a></li>
<li class="chapter" data-level="8.2" data-path="lifecycle.html"><a href="lifecycle.html#genome-protection"><i class="fa fa-check"></i><b>8.2</b> Genome Protection</a></li>
<li class="chapter" data-level="8.3" data-path="lifecycle.html"><a href="lifecycle.html#stationary-phase-contd."><i class="fa fa-check"></i><b>8.3</b> Stationary Phase (cont’d.)</a></li>
<li class="chapter" data-level="8.4" data-path="lifecycle.html"><a href="lifecycle.html#differentiation"><i class="fa fa-check"></i><b>8.4</b> Differentiation</a></li>
<li class="chapter" data-level="8.5" data-path="lifecycle.html"><a href="lifecycle.html#moreholdfast"><i class="fa fa-check"></i><b>8.5</b> More–Holdfast</a></li>
<li class="chapter" data-level="8.6" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-sporulation"><i class="fa fa-check"></i><b>8.6</b> Monoderm Sporulation</a></li>
<li class="chapter" data-level="8.7" data-path="lifecycle.html"><a href="lifecycle.html#moresporulation-septum-formation"><i class="fa fa-check"></i><b>8.7</b> More–Sporulation septum formation</a></li>
<li class="chapter" data-level="8.8" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-sporulation-contd."><i class="fa fa-check"></i><b>8.8</b> Monoderm Sporulation (cont’d.)</a></li>
<li class="chapter" data-level="8.9" data-path="lifecycle.html"><a href="lifecycle.html#morespore-engulfment"><i class="fa fa-check"></i><b>8.9</b> More–Spore engulfment</a></li>
<li class="chapter" data-level="8.10" data-path="lifecycle.html"><a href="lifecycle.html#morespore-coating"><i class="fa fa-check"></i><b>8.10</b> More–Spore coating</a></li>
<li class="chapter" data-level="8.11" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-spores"><i class="fa fa-check"></i><b>8.11</b> Monoderm Spores</a></li>
<li class="chapter" data-level="8.12" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-germination"><i class="fa fa-check"></i><b>8.12</b> Monoderm Germination</a></li>
<li class="chapter" data-level="8.13" data-path="lifecycle.html"><a href="lifecycle.html#diderm-sporulation"><i class="fa fa-check"></i><b>8.13</b> Diderm Sporulation</a></li>
<li class="chapter" data-level="8.14" data-path="lifecycle.html"><a href="lifecycle.html#diderm-sporulation-contd."><i class="fa fa-check"></i><b>8.14</b> Diderm Sporulation (cont’d.)</a></li>
<li class="chapter" data-level="8.15" data-path="lifecycle.html"><a href="lifecycle.html#morecomplete-engulfment"><i class="fa fa-check"></i><b>8.15</b> More–Complete engulfment</a></li>
<li class="chapter" data-level="8.16" data-path="lifecycle.html"><a href="lifecycle.html#moremature-spore"><i class="fa fa-check"></i><b>8.16</b> More–Mature spore</a></li>
<li class="chapter" data-level="8.17" data-path="lifecycle.html"><a href="lifecycle.html#moresporomusa-acidovorans-sporulation"><i class="fa fa-check"></i><b>8.17</b> More–Sporomusa acidovorans sporulation</a></li>
<li class="chapter" data-level="8.18" data-path="lifecycle.html"><a href="lifecycle.html#diderm-germination"><i class="fa fa-check"></i><b>8.18</b> Diderm Germination</a></li>
<li class="chapter" data-level="8.19" data-path="lifecycle.html"><a href="lifecycle.html#further-reading-7"><i class="fa fa-check"></i><b>8.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="interaction.html"><a href="interaction.html"><i class="fa fa-check"></i><b>9</b> Interaction</a><ul>
<li class="chapter" data-level="9.1" data-path="interaction.html"><a href="interaction.html#biofilm"><i class="fa fa-check"></i><b>9.1</b> Biofilm</a></li>
<li class="chapter" data-level="9.2" data-path="interaction.html"><a href="interaction.html#biofilm-contd."><i class="fa fa-check"></i><b>9.2</b> Biofilm (cont’d.)</a></li>
<li class="chapter" data-level="9.3" data-path="interaction.html"><a href="interaction.html#type-ii-and-type-iv-secretion"><i class="fa fa-check"></i><b>9.3</b> Type II and Type IV Secretion</a><ul>
<li class="chapter" data-level="9.3.1" data-path="interaction.html"><a href="interaction.html#emd-20712-emd-20713legionella-t2ss-structure-emd-20712_emd-20713legionella_t2ss_structure"><i class="fa fa-check"></i><b>9.3.1</b> EMD-20712 EMD-20713–Legionella T2SS structure {#EMD-20712_EMD-20713–Legionella_T2SS_structure}</a></li>
<li class="chapter" data-level="9.3.2" data-path="interaction.html"><a href="interaction.html#emd-0566-legionella-t4ss-structure-emd-0566_legionella_t4ss_structure"><i class="fa fa-check"></i><b>9.3.2</b> EMD-0566 –Legionella T4SS structure {#EMD-0566_–Legionella_T4SS_structure}</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="interaction.html"><a href="interaction.html#morehelicobacter-pylori-tubes"><i class="fa fa-check"></i><b>9.4</b> More–Helicobacter pylori tubes</a></li>
<li class="chapter" data-level="9.5" data-path="interaction.html"><a href="interaction.html#type-iii-secretion"><i class="fa fa-check"></i><b>9.5</b> Type III Secretion</a><ul>
<li class="chapter" data-level="9.5.1" data-path="interaction.html"><a href="interaction.html#EMD-20611_Injectisome_structure"><i class="fa fa-check"></i><b>9.5.1</b> EMD-20611 Injectisome structure</a></li>
<li class="chapter" data-level="9.5.2" data-path="interaction.html"><a href="interaction.html#PDB:_5WQ8_Secretin_structure"><i class="fa fa-check"></i><b>9.5.2</b> PDB: 5WQ8 Secretin structure</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="interaction.html"><a href="interaction.html#type-v-secretion"><i class="fa fa-check"></i><b>9.6</b> Type V Secretion</a></li>
<li class="chapter" data-level="9.7" data-path="interaction.html"><a href="interaction.html#type-vi-secretion"><i class="fa fa-check"></i><b>9.7</b> Type VI Secretion</a><ul>
<li class="chapter" data-level="9.7.1" data-path="interaction.html"><a href="interaction.html#pdb-5urwprimed-t6ss-structure-pdb_5urwprimed_t6ss_structure"><i class="fa fa-check"></i><b>9.7.1</b> PDB: 5URW–Primed T6SS structure {#PDB:_5URW–Primed_T6SS_structure} <img src="img/schematics/9_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="9.8" data-path="interaction.html"><a href="interaction.html#type-vi-secretion-contd."><i class="fa fa-check"></i><b>9.8</b> Type VI Secretion (cont’d.)</a><ul>
<li class="chapter" data-level="9.8.1" data-path="interaction.html"><a href="interaction.html#pdb-5urxfired-t6ss-structure-pdb_5urxfired_t6ss_structure"><i class="fa fa-check"></i><b>9.8.1</b> PDB: 5URX–Fired T6SS structure {#PDB:_5URX–Fired_T6SS_structure} <img src="img/schematics/9_7_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="9.9" data-path="interaction.html"><a href="interaction.html#moret6ss-battery"><i class="fa fa-check"></i><b>9.9</b> More–T6SS battery</a></li>
<li class="chapter" data-level="9.10" data-path="interaction.html"><a href="interaction.html#moremacs"><i class="fa fa-check"></i><b>9.10</b> More–MACs</a><ul>
<li class="chapter" data-level="9.10.1" data-path="interaction.html"><a href="interaction.html#MAC_array_structure"><i class="fa fa-check"></i><b>9.10.1</b> MAC array structure</a></li>
</ul></li>
<li class="chapter" data-level="9.11" data-path="interaction.html"><a href="interaction.html#type-ix-secretion"><i class="fa fa-check"></i><b>9.11</b> Type IX Secretion</a></li>
<li class="chapter" data-level="9.12" data-path="interaction.html"><a href="interaction.html#predation"><i class="fa fa-check"></i><b>9.12</b> Predation</a></li>
<li class="chapter" data-level="9.13" data-path="interaction.html"><a href="interaction.html#predation-contd."><i class="fa fa-check"></i><b>9.13</b> Predation (cont’d.)</a></li>
<li class="chapter" data-level="9.14" data-path="interaction.html"><a href="interaction.html#morebdellovibrio-bacteriovorus-entry"><i class="fa fa-check"></i><b>9.14</b> More–Bdellovibrio bacteriovorus entry</a></li>
<li class="chapter" data-level="9.15" data-path="interaction.html"><a href="interaction.html#predation-contd.-1"><i class="fa fa-check"></i><b>9.15</b> Predation (cont’d.)</a></li>
<li class="chapter" data-level="9.16" data-path="interaction.html"><a href="interaction.html#further-reading-8"><i class="fa fa-check"></i><b>9.16</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="viruses.html"><a href="viruses.html"><i class="fa fa-check"></i><b>10</b> Viruses</a><ul>
<li class="chapter" data-level="10.1" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis"><i class="fa fa-check"></i><b>10.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.2" data-path="viruses.html"><a href="viruses.html#bacteroidetes-strain-jt5"><i class="fa fa-check"></i><b>10.2</b> Bacteroidetes strain JT5</a></li>
<li class="chapter" data-level="10.3" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-1"><i class="fa fa-check"></i><b>10.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.4" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae"><i class="fa fa-check"></i><b>10.4</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="10.5" data-path="viruses.html"><a href="viruses.html#escherichia-coli"><i class="fa fa-check"></i><b>10.5</b> Escherichia coli</a></li>
<li class="chapter" data-level="10.6" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus"><i class="fa fa-check"></i><b>10.6</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="10.7" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.7</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="10.8" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus"><i class="fa fa-check"></i><b>10.8</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="10.9" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-1"><i class="fa fa-check"></i><b>10.9</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="10.10" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-2"><i class="fa fa-check"></i><b>10.10</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.11" data-path="viruses.html"><a href="viruses.html#further-reading-9"><i class="fa fa-check"></i><b>10.11</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>11</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.library.caltech.edu" target="blank">Published by Caltech Library</a><li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cell Atlas</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="interaction" class="section level1">
<h1><span class="header-section-number">Chapter 9</span> Interaction</h1>
<p>“We do not have solitary beings. Every creature is, in some sense, connected to and dependent on the rest.” - Lewis Thomas <span class="citation">Thomas (<a href="#ref-thomas1974">1974</a>)</span></p>
<div id="biofilm" class="section level2">
<h2><span class="header-section-number">9.1</span> Biofilm</h2>
<p>So far we have largely ignored a major part of a cell’s environment: other organisms. Your cell is far from alone out there. It shares space and resources with cells of the same species (some its own mothers/sisters/daughters), other species, and even other domains. Think about strategies that could help your cell thrive in such a crowded world. For starters, why can’t we all just get along? Cooperation is common in Nature, often in the form of biofilms–communal groups of microorganisms. Biofilms offer advantages to their members, shielding cells on the interior from harsh conditions or antibiotics, and ensuring nutrients produced by cells’ metabolism or released by their death are readily available to others. Biofilms should be familiar to you; think of the scum on your teeth, or the nearest pond. In fact, the earliest physical evidence we have of life on earth is in the form of stromatolites (“layered rocks”)–meter-scale fossilized biofilms of cyanobacteria (you can see still-growing stromatolites in Shark Bay, Australia).</p>
<p>To attach to a biofilm, cells use strategies we have already discussed, including Tad pili and holdfast. <em>Agrobacterium tumefaciens</em> like this one use a holdfast-like polymer called _U_ni_p_olar _p_olysaccharide, or UPP. Some bacteria, including this one, also contribute to the superstructure of the biofilm by secreting cellulose fibers that help form the non-living matrix of the community.</p>
<div class="figure"><span id="fig:unnamed-chunk-151"></span>
<div id="htmlwidget-037391eb41f7fd1fcb3b" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-037391eb41f7fd1fcb3b">{"x":{"doi":"10.22002/D1.1571","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.1: Agrobacterium tumefaciens Collected by: Elitza Tocheva <a href="https://doi.org/10.22002/D1.1571">10.22002/D1.1571</a>
</p>
</div>
</div>
<div id="biofilm-contd." class="section level2">
<h2><span class="header-section-number">9.2</span> Biofilm (cont’d.)</h2>
<p>Individual cellulose fibers associate into bundles to form a 3D matrix for the cells, as you can see in this thin slice through a <em>Gluconacetobacter hansenii</em> biofilm. Not all biofilms contain cellulose, but all have an extracellular matrix, typically made of proteins secreted by the component cells. These proteins have properties that help them aggregate, like the unipolar polysaccharide you just saw. They may also have other properties useful to the community; for instance, <em>Bacillus subtilis</em> secretes a hydrophobic protein that forms a resistant coat around the biofilm. Matrices also contain lipids and extracellular DNA, and dead cells contribute material to the superstructure. Keep in mind that the lab-grown biofilm you see here consists of a single species, but in Nature a biofilm often contains multiple species.</p>
<div class="figure"><span id="fig:unnamed-chunk-152"></span>
<div id="htmlwidget-968d036f5c5ad1ba3e29" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-968d036f5c5ad1ba3e29">{"x":{"doi":"10.22002/D1.1572","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.2: Gluconacetobacter hansenii Collected by: William Nicolas <a href="https://doi.org/10.22002/D1.1572">10.22002/D1.1572</a>
</p>
</div>
</div>
<div id="type-ii-and-type-iv-secretion" class="section level2">
<h2><span class="header-section-number">9.3</span> Type II and Type IV Secretion</h2>
<p>How else might your cell deal with the crowd? Well, if you can’t join them, maybe you should try to beat (and eat) them. Bacteria have evolved an impressive arsenal of molecular weaponry. In fact, most of the antibiotics we use were invented by bacteria. In many cases, cells simply release these antibiotics to the environment, either directly or in membrane vesicles, which may travel further. For more specific targeting, though, cells employ a varied array of secretion systems. You already saw some of these nanomachines in Chapter 6: a type III secretion system assembles the flagellum and a type II secretion system makes (and unmakes) type IV pili. Other family members have evolved more militaristic functions.</p>
<p>Type II secretion systems (T2SSs) in <em>Legionella pneumophila</em> like this pump out effector proteins that enable a complex pathogenic lifecycle. <em>L. pneumophila</em> live in several environments: inside amoebae, outside cells (but sometimes with others in biofilms), and inside <em>our</em> cells, where they cause the severe pneumonia known as Legionnaires disease. In each environment, the T2SS pumps out proteins that facilitate growth there. For instance, once we inhale <em>L. pneumophila</em> with water droplets, they are internalized by macrophages. They then use their T2SSs to secrete proteins that prevent their pockets in the cell (called <em>Legionella</em>-containing vacuoles) from being degraded, and suppress our innate immune response. This gives the bacteria time to replicate and establish a persistent colony. The T2SSs also secrete a toxin protein that breaks down lung tissue.</p>
<p>Unlike the secretion systems in Chapter 6, the T2SS does not build an extracellular appendage. Instead, it is thought to work by pumping proteins through the envelope channel using a short piston-like “pseudo-pilus” [Schematic–Legionella T2SS structure], pushing the molecules into the extracellular space. (Keep in mind that for these intracellular pathogens, their extracellular space is the interior of the host cell.)</p>
<p><em>L. pneumophila</em> also use another, unrelated machine, a type IV secretion system (T4SS), to deliver more than 400(!) different kinds of effector proteins into host cells to promote infection. You can see multiple T4SSs primed for action in this cell [Schematic–Legionella T4SS structure]. We do not yet know what they look like during active secretion, so the mechanism remains unclear. In another human pathogen, <em>Helicobacter pylori</em>, inactive T4SSs are sometimes seen next to mysterious extracellular tubes, raising the intriguing possibility that the structures are somehow related <a href="interaction.html#morehelicobacter-pylori-tubes">More–Helicobacter pylori tubes</a>.</p>
<p>This <em>L. pneumophila</em> cell is not unusual; it is common for pathogenic bacteria to employ multiple types of secretion systems. The bristling arsenal of bacteria is a fearsome thing.</p>
<div class="figure"><span id="fig:unnamed-chunk-153"></span>
<div id="htmlwidget-52909a30d55f1fd3395a" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-52909a30d55f1fd3395a">{"x":{"doi":"10.22002/D1.1573","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.3: Legionella pneumophila Collected by: Debnath Ghosal <a href="https://doi.org/10.22002/D1.1573">10.22002/D1.1573</a>
</p>
</div>
<div id="emd-20712-emd-20713legionella-t2ss-structure-emd-20712_emd-20713legionella_t2ss_structure" class="section level3">
<h3><span class="header-section-number">9.3.1</span> EMD-20712 EMD-20713–Legionella T2SS structure {#EMD-20712_EMD-20713–Legionella_T2SS_structure}</h3>
<p><img src="img/schematics/9_3_1.jpg" /><!-- --></p>
<p>This average of type II secretion systems from many <em>Legionella pneumophila</em> cells shows the overall structure: a channel through the cell envelope, gated near the outer membrane <span class="citation">Ghosal, Kim, et al. (<a href="#ref-ghosal2019">2019</a>)</span>. The pseudo-pilus can be seen in individual T2SSs, extending from the inner membrane to the bottom of the main channel. It gets washed out in the average, though, indicating that it is relatively flexible (i.e. not always in the same position). We still need to figure out exactly how the system works, including whether the pseudo-pilus does act like a piston, and how the gate opens.</p>
</div>
<div id="emd-0566-legionella-t4ss-structure-emd-0566_legionella_t4ss_structure" class="section level3">
<h3><span class="header-section-number">9.3.2</span> EMD-0566 –Legionella T4SS structure {#EMD-0566_–Legionella_T4SS_structure}</h3>
<p><img src="img/schematics/9_3_2.jpg" /><!-- --></p>
<p>This average of type IV secretion systems from many <em>Legionella pneumophila</em> cells shows the overall structure <span class="citation">Ghosal, Jeong, et al. (<a href="#ref-ghosal2019a">2019</a>)</span>. We do not yet know how secretion works, but one possibility is that effectors are channeled either straight through the channel from the cytoplasm and/or first into the periplasm and from there into the windowed chamber just inside the outer membrane before finally being escorted out of the cell.</p>
</div>
</div>
<div id="morehelicobacter-pylori-tubes" class="section level2">
<h2><span class="header-section-number">9.4</span> More–Helicobacter pylori tubes</h2>
<p>This <em>Helicobacter pylori</em> cell was cultured together with human gastric epithelial cells, its target for infection. It contains inactive type IV secretion systems, as well as several mysterious tubes. The tubes are formed from the outer membrane, but we do not know what scaffolds them (they have a constant diameter of 37 nm), or what makes the portals along their length. The fact that we sometimes see tubes near primed T4SSs, and the fact that the tubes are missing from mutant cells lacking T4SS components suggests a relationship, but the details are unclear. Much remains mysterious about this structure and its potential role in infection.</p>
<div class="figure"><span id="fig:unnamed-chunk-156"></span>
<div id="htmlwidget-dbfc7f65f0005168cd38" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-dbfc7f65f0005168cd38">{"x":{"doi":"10.22002/D1.1582","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.4: Helicobacter pylori Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1582">10.22002/D1.1582</a>
</p>
</div>
</div>
<div id="type-iii-secretion" class="section level2">
<h2><span class="header-section-number">9.5</span> Type III Secretion</h2>
<p>What if your cell wanted to go a step further and deliver toxins directly into its target? Perhaps it needs a needle. One of the most common, and best-studied, pathogenic secretion systems is the type III secretion system (T3SS) that assembles a needle-like toxin delivery system called the injectisome. T3SS injectisomes and flagellar motors are closely related, and the basal machines have similar structures [Schematic–Injectisome structure]. Rather than assembling a long helical filament for motility, though, injectisomes form a shorter needle, which you can see on this <em>Pseudomonas aeruginosa</em>. The needle can penetrate a eukaryotic cell, delivering effectors directly to its cytoplasm that block the immune response of the host, and lead to cell lysis.</p>
<p><em>It would be wasteful, or detrimental, for your cell to secrete toxins all the time. Pathogens typically have multi-stage lifecycles, often passing through an intermediate host and/or the environment. Pathogenic machinery like secretion systems are typically only produced when conditions signal that the cell has reached its target host (e.g. increased temperature or salt concentration). Until then, the genes encoding the protein components are turned off. This cell is missing a protein (ExsD) that turns off expression of these injectisome components. As a result, it makes injectisomes even outside the host, enabling us to image their structure.</em></p>
<p>T3SSs are closely related to type II secretion systems (T2SSs) and share a common element: the channel embedded in the outer membrane, formed by many copies of a protein called a secretin [Schematic–Secretin structure]. As you can see, though, the rest of the machine looks very different.</p>
<div class="figure"><span id="fig:unnamed-chunk-157"></span>
<div id="htmlwidget-6a5f6f3368d821fd73b9" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-6a5f6f3368d821fd73b9">{"x":{"doi":"10.22002/D1.1574","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.5: Pseudomonas aeruginosa Collected by: Steven Wang <a href="https://doi.org/10.22002/D1.1574">10.22002/D1.1574</a>
</p>
</div>
<div id="EMD-20611_Injectisome_structure" class="section level3">
<h3><span class="header-section-number">9.5.1</span> EMD-20611 Injectisome structure</h3>
<p><img src="img/schematics/9_4_1.jpg" /><!-- --></p>
<p>This average of T3SS injectisomes from many <em>Shigella flexneri</em> cells shows the overall structure, including the large “sorting platform” in the cytoplasm which selects proteins for secretion through the channel of the needle <span class="citation">Tachiyama et al. (<a href="#ref-tachiyama2019">2019</a>)</span>.</p>
</div>
<div id="PDB:_5WQ8_Secretin_structure" class="section level3">
<h3><span class="header-section-number">9.5.2</span> PDB: 5WQ8 Secretin structure</h3>
<p><img src="img/schematics/9_4_2.gif" /><!-- --></p>
<p>Many copies of a Secretin protein come together to form a channel, as in this one from <em>Vibrio cholerae</em> <span class="citation">Yan et al. (<a href="#ref-yan2017a">2017</a>)</span>. In addition to the main channel situated in the outer membrane, different domains of the protein form a series of rings in the periplasm that interact with other components of the system.</p>
</div>
</div>
<div id="type-v-secretion" class="section level2">
<h2><span class="header-section-number">9.6</span> Type V Secretion</h2>
<p>Other secretion systems independently evolved similar mechanisms, though using very different structures. Some strains of <em>Escherichia coli</em> have an ability, called “_C_ontact-_D_ependent _I_nhibition” or CDI, to inhibit the growth of neighboring diderm bacterial cells. They do this by coopting a pore in the outer membrane of the target cell to introduce a toxin that halts its growth. To safeguard themselves, they make an antitoxin that binds to the toxin and renders it harmless. Such antidote system are common for microbe-produced toxins.</p>
<p>To deliver the toxin to the target, the cells use type V secretion systems (T5SSs) like the ones on this <em>Escherichia coli</em>. In contrast to the nanomachines you have been seeing, which contain dozens of unique components, this system is elegantly spare: a channel formed by several copies of one protein (CdiB) in the outer membrane secretes another protein, CdiA. Or rather, it secretes half of it. The first half of CdiA forms the stick you see here, with the second half keeping the toxin domain at the end of the protein sequestered in the periplasm. The tip of the stick contains the domain that recognizes and docks to the receptor pore on the target. When it bumps into a neighboring cell and locks on, the second half of CdiA is secreted, whipping the toxin like a tetherball into the other cell.</p>
<p><em>This cell belongs to a strain that is normally incapable of CDI, but has been genetically engineered to produce the T5SS components. Once again, this lets us image the machines more easily; in this case, the CDI-practicing strain has a thicker layer of extracellular proteins, which obscure the needles in a visual haystack.</em></p>
<div class="figure"><span id="fig:unnamed-chunk-160"></span>
<div id="htmlwidget-900285758635ea5d315d" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-900285758635ea5d315d">{"x":{"doi":"10.22002/D1.1575","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.6: Escherichia coli Collected by: Poorna Subramanian <a href="https://doi.org/10.22002/D1.1575">10.22002/D1.1575</a>
</p>
</div>
</div>
<div id="type-vi-secretion" class="section level2">
<h2><span class="header-section-number">9.7</span> Type VI Secretion</h2>
<p>How else could your cell deliver a toxin to a nearby competitor? As usual, Nature has provided another, even more entertaining, option: the poison dart gun. Many diderm bacteria, including <em>Myxococcus xanthus</em> like this, use type VI secretion systems (T6SSs) to launch effector proteins into neighboring cells. The effectors vary–from toxins that cause lysis to factors that promote biofilm formation. The target range of the T6SS is similarly broad, including monoderm and diderm bacteria and eukaryotic cells.</p>
<p>This primed T6SS consists of a hollow outer tube, with a narrower dart inside [Schematic–Primed T6SS structure]. The effector protein is loaded at the tip of the dart, where the machine is anchored in the cell envelope. In a few cells, including this one, we see additional filaments flanking the primed T6SS. They are lacking in other cells even of the same species, and their identity and function remain unknown.</p>
<div class="figure"><span id="fig:unnamed-chunk-161"></span>
<div id="htmlwidget-5c7e43e3bf8fc1f78436" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-5c7e43e3bf8fc1f78436">{"x":{"doi":"10.22002/D1.1576","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.7: Myxococcus xanthus Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1576">10.22002/D1.1576</a>
</p>
</div>
<div id="pdb-5urwprimed-t6ss-structure-pdb_5urwprimed_t6ss_structure" class="section level3">
<h3><span class="header-section-number">9.7.1</span> PDB: 5URW–Primed T6SS structure {#PDB:_5URW–Primed_T6SS_structure} <img src="img/schematics/9_6_1.gif" /><!-- --></h3>
<p>In a primed T6SS, the projectile rod (orange) nests inside the contractile sheath (blue), as in this section of a T6SS from <em>Myxococcus xanthus</em> <span class="citation">Chang et al. (<a href="#ref-chang2017">2017</a>)</span>.</p>
</div>
</div>
<div id="type-vi-secretion-contd." class="section level2">
<h2><span class="header-section-number">9.8</span> Type VI Secretion (cont’d.)</h2>
<p>When the T6SS fires, the helical outer sheath twists into a shorter, wider form, a conformational change that provides the energy to explosively propel the inner dart, cargo-first, out and into its target [Schematic–Fired T6SS structure]. In some species, like this <em>Vibrio cholerae</em>, the outer sheath will then be recycled, broken down into building blocks that can be used to rapidly assemble a new machine. Other species build again from scratch, translating new protein building blocks. Warfare is dynamic, with cells rapidly assembling and firing T6SSs. Cells have even been observed to engage in T6SS duels, with an attacked cell building and firing a T6SS back in the direction from which it was hit. Some species even assemble batteries of parallel T6SSs <a href="interaction.html#moret6ss-battery">More–T6SS battery</a>.</p>
<p>These contractile weapons evolved from the “tail” structure viruses called phage use to inject their genetic material into bacteria and archaea (more on that in the next chapter). Some related bacterial weapons are even closer to the phage tail, in that they are released to the environment in a primed state, waiting to bump into a target cell to fire. The target is recognized by filamentous proteins called “tail fibers,” the same identification mechanism used by phage. Other members of this evolutionary family are even stranger, like the_M_orphogenesis-_A_ssociated _C_ontractile structures (MACs) released by a marine bacterium which play a key role in the lifecycle of a (eukaryotic) worm [More–MACs] [Schematic-MAC array structure].</p>
<div class="figure"><span id="fig:unnamed-chunk-163"></span>
<div id="htmlwidget-fa48d0db40b7516e894c" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-fa48d0db40b7516e894c">{"x":{"doi":"10.22002/D1.1577","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.8: Vibrio cholerae Collected by: Martin Pilhofer <a href="https://doi.org/10.22002/D1.1577">10.22002/D1.1577</a>
</p>
</div>
<div id="pdb-5urxfired-t6ss-structure-pdb_5urxfired_t6ss_structure" class="section level3">
<h3><span class="header-section-number">9.8.1</span> PDB: 5URX–Fired T6SS structure {#PDB:_5URX–Fired_T6SS_structure} <img src="img/schematics/9_7_1.gif" /><!-- --></h3>
<p>A conformational change in the T6SS sheath provides the energy to fire the inner rod toward its target. The remaining sheath is shorter and wider than in its primed state, as in this section of a fired T6SS from <em>Myxococcus xanthus</em> <span class="citation">Chang et al. (<a href="#ref-chang2017">2017</a>)</span>.</p>
</div>
</div>
<div id="moret6ss-battery" class="section level2">
<h2><span class="header-section-number">9.9</span> More–T6SS battery</h2>
<p>Some cells assemble a few, scattered type VI secretion systems. Other species, like this <em>Amoebophilus asiaticus</em> build ordered arrays of parallel machines. These diderm bacteria live symbiotically inside amoebae. Their T6SS batteries help them colonize their host, perhaps by rupturing the membrane of the host phagosome that encloses them after they are engulfed, releasing them into the host cytoplasm. The number of T6SSs in each array varies, but can reach a few dozen.</p>
<div class="figure"><span id="fig:unnamed-chunk-165"></span>
<div id="htmlwidget-2b6df4a10b54fedd6f85" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-2b6df4a10b54fedd6f85">{"x":{"doi":"10.22002/D1.1583","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.9: Amoebophilus asiaticus Collected by: Martin Pilhofer <a href="https://doi.org/10.22002/D1.1583">10.22002/D1.1583</a>
</p>
</div>
</div>
<div id="moremacs" class="section level2">
<h2><span class="header-section-number">9.10</span> More–MACs</h2>
<p><em>Pseudoalteromonas luteoviolacea</em> produce massive arrays of MACs, contractile machines related to pyocins. Cells lyse to release the arrays, like the one you see here, which can contain 100 individual MACs and extend more than 1 µm across. The arrays are highly ordered, with the tail fibers of the MACs networked into a hexagonal array that may help synchronize firing [Schematic–MAC array structure]. MAC arrays serve an important function for larvae of the marine tubeworm <em>Hydroides elegans</em>, and likely the larvae of other invertebrates such as sea urchins and corals as well. They signal to the larva that it is the right place to settle down and differentiate into a sessile (surface-attached) adult. The function of MAC arrays for the bacterium is less clear. It may be that the invertebrates benefit the lysed cell’s surviving relatives in the biofilm. Or perhaps the MACs play another, completely unrelated role.</p>
<p><em>Note that this MAC array was produced by a cell lacking genes encoding other contractile weaponry (type VI secretion systems and bacteriocins). This was done to make it easier to identify the MACs.</em></p>
<div class="figure"><span id="fig:unnamed-chunk-166"></span>
<div id="htmlwidget-6657f00857c090585d69" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-6657f00857c090585d69">{"x":{"doi":"10.22002/D1.1584","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.10: Pseudoalteromonas luteoviolacea Collected by: Martin Pilhofer <a href="https://doi.org/10.22002/D1.1584">10.22002/D1.1584</a>
</p>
</div>
<div id="MAC_array_structure" class="section level3">
<h3><span class="header-section-number">9.10.1</span> MAC array structure</h3>
<p><img src="img/schematics/9_7_1.gif" /><!-- --></p>
<p>This digital segmentation of the MAC array you just saw <a href="interaction.html#moremacs">More–MACs</a> shows the overall order of the structure <span class="citation">Shikuma et al. (<a href="#ref-shikuma2014">2014</a>)</span>. Outer sheaths are in blue, inner rods in green, tail fibers in orange, and networking fibers in white.</p>
</div>
</div>
<div id="type-ix-secretion" class="section level2">
<h2><span class="header-section-number">9.11</span> Type IX Secretion</h2>
<p>Instead of going to all the trouble of launching a weapon, why not just turn yourself into the weapon? <em>Porphyromonas gingivalis</em> like this one use yet another secretion system, the type IX secretion system (T9SS), to secrete proteins called gingipains to the cell surface, where they are anchored to the outer membrane. Gingipains are peptidases, enzymes that chop up other proteins. In their lifestyle as human oral pathogens, the cells use gingipains to block host immune system signals and digest host biomolecules for food, causing the tissue damage of periodontitis.</p>
<p>In Chapter 6 you saw another T9SS with a very different function, in the gliding motility of <em>Flavobacterium johnsoniae</em>. In that case, instead of gingipains, the machine secretes adhesive filaments that are also anchored to the cell surface.</p>
<div class="figure"><span id="fig:unnamed-chunk-168"></span>
<div id="htmlwidget-caa8079f1ec75a5fc0ff" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-caa8079f1ec75a5fc0ff">{"x":{"doi":"10.22002/D1.1578","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.11: Porphyromonas gingivalis Collected by: Poorna Subramanian <a href="https://doi.org/10.22002/D1.1578">10.22002/D1.1578</a>
</p>
</div>
</div>
<div id="predation" class="section level2">
<h2><span class="header-section-number">9.12</span> Predation</h2>
<p>Some bacteria take the strategy of eating the neighbors to a whole new level. Remember the pack-based hunting of <em>Myxococcus xanthus</em> in Chapter 7 (chasing their prey with type IV pilus (T4P)-based twitching motility)? There are also solitary hunters, like <em>Bdellovibrio bacteriovorus</em>, which has evolved an elaborate vampiric lifestyle centered around its prey. You have already seen examples of <em>B. bacteriovorus</em> cells on the prowl, in what is called “attack phase”. Their bodies are ideally shaped and sized for rapid flagellum-propelled swimming and all superfluous functions like growth and division are halted (remember the highly condensed nucleoid you saw at the end of Chapter 2). When a cell finds a diderm bacterium–<em>Vibrio cholerae</em> in this case–it uses its type IV pili to lock on and pull itself closer, as you can see here.</p>
<div class="figure"><span id="fig:unnamed-chunk-169"></span>
<div id="htmlwidget-647f75b67af6c6839bc4" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-647f75b67af6c6839bc4">{"x":{"doi":"10.22002/D1.1579","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.12: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1579">10.22002/D1.1579</a>
</p>
</div>
</div>
<div id="predation-contd." class="section level2">
<h2><span class="header-section-number">9.13</span> Predation (cont’d.)</h2>
<p>Once a <em>Bdellovibrio bacteriovorus</em> like this one has latched onto its unfortunate host, it opens a portal in the host’s outer membrane, secreting a cocktail of enzymes that alter the host cell wall, causing the cell to round up and enlarge the periplasmic space. The <em>B. bacteriovorus</em> quickly slides into the host periplasm through the portal <a href="interaction.html#morebdellovibrio-bacteriovorus-entry">More–Bdellovibrio bacteriovorus entry</a> and seals the door behind it. In the process of preparing for entry, the cell dismantles equipment that is no longer necessary, including its chemosensory array and flagellum. As you can see here, the sheathed flagellum dislocates from the motor and is pulled into the periplasm, where the filament is degraded.</p>
<p><em>Note that the prey here–the small, round cell–comes from a mutant strain of </em>Escherichia coli* that has an altered form of the shape-regulating protein MreB, which results in thinner cells. It is also lacking the Min system that positions the division plane at midcell. As a result, division occurs randomly along the length of the cell. The cells that divide near the pole produce “minicells” like this one. Minicells do not contain a genome and are not viable cells, but they can be useful for experiments. In this case, the reduced thickness of the sample improves the quality of cryoET imaging. It also lets us capture intermediates in the <em>B. bacteriovorus</em> entry process <a href="interaction.html#morebdellovibrio-bacteriovorus-entry">More–Bdellovibrio bacteriovorus entry</a>.*</p>
<div class="figure"><span id="fig:unnamed-chunk-170"></span>
<div id="htmlwidget-5c286e5c5d3f6adb5aa9" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-5c286e5c5d3f6adb5aa9">{"x":{"doi":"10.22002/D1.1580","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.13: Bdellovibrio bacteriovorus / Escherichia coli Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1580">10.22002/D1.1580</a>
</p>
</div>
</div>
<div id="morebdellovibrio-bacteriovorus-entry" class="section level2">
<h2><span class="header-section-number">9.14</span> More–Bdellovibrio bacteriovorus entry</h2>
<p>We still do not know exactly how <em>Bdellovibrio bacteriovorus</em> enters its host, but the process is extremely rapid. In order to catch a snapshot, we mixed <em>B. bacteriovorus</em> with miniaturized prey (see details on the main page). As the <em>B. bacteriovorus</em> cell cannot fit inside the prey, it remains stuck half-in and half-out. Note the tight collar of host outer membrane around the invader. How this seal is maintained throughout invasion remains a mystery.</p>
<div class="figure"><span id="fig:unnamed-chunk-171"></span>
<div id="htmlwidget-282966bb7790f4d2fa18" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-282966bb7790f4d2fa18">{"x":{"doi":"10.22002/D1.1585","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.14: Bdellovibrio bacteriovorus / Escherichia coli Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1585">10.22002/D1.1585</a>
</p>
</div>
</div>
<div id="predation-contd.-1" class="section level2">
<h2><span class="header-section-number">9.15</span> Predation (cont’d.)</h2>
<p>Once inside its host, <em>Bdellovibrio bacteriovorus</em> has the leisure to grow and replicate. From its position in the periplasm, the <em>B. bacteriovorus</em> cell proceeds to digest the contents of the host cytoplasm, which shrinks down to a dense ball of (presumably undigestible) material. This meal provides the fuel necessary for the <em>B. bacteriovorus</em> to grow to several times its original length and undergo a synchronous division to produce several (usually between 2 and 7) daughters. The number of progeny depends on the size of the host cell: two in the case of this relatively small <em>Vibrio cholerae</em>. The progeny, reset to the attack phase (note the condensed nucleoids), finally lyse what is left of the host to escape and swim off in search of their next victims [Schematic–Bdellovibrio bacteriovorus lifecycle].</p>
<div class="figure"><span id="fig:unnamed-chunk-172"></span>
<div id="htmlwidget-fca7942dc48c239dc78d" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-fca7942dc48c239dc78d">{"x":{"doi":"10.22002/D1.1581","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 9.15: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1581">10.22002/D1.1581</a>
</p>
</div>
</div>
<div id="further-reading-8" class="section level2">
<h2><span class="header-section-number">9.16</span> Further Reading</h2>
<p><span class="citation">(<span class="citeproc-not-found" data-reference-id="christie2019"><strong>???</strong></span>)</span> <span class="citation">Flemming et al. (<a href="#ref-flemming2016">2016</a>)</span> <span class="citation">(<span class="citeproc-not-found" data-reference-id="patz2019"><strong>???</strong></span>)</span> <span class="citation">Sockett (<a href="#ref-sockett2009">2009</a>)</span></p>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-chang2017">
<p>Chang, Y. W., L. A. Rettberg, D. R. Ortega, and G. J. Jensen. 2017. “In Vivo Structures of an Intact Type VI Secretion System Revealed by Electron Cryotomography.” <em>EMBO Rep</em> 18 (7): 1090–9. doi:<a href="https://doi.org/10.15252/embr.201744072">10.15252/embr.201744072</a>.</p>
</div>
<div id="ref-flemming2016">
<p>Flemming, H. C., J. Wingender, U. Szewzyk, P. Steinberg, S. A. Rice, and S. Kjelleberg. 2016. “Biofilms: An Emergent Form of Bacterial Life.” <em>Nat Rev Microbiol</em> 14 (9): 563–75. doi:<a href="https://doi.org/10.1038/nrmicro.2016.94">10.1038/nrmicro.2016.94</a>.</p>
</div>
<div id="ref-ghosal2019a">
<p>Ghosal, Debnath, Kwangcheol C. Jeong, Yi-Wei Chang, Jacob Gyore, Lin Teng, Adam Gardner, Joseph P. Vogel, and Grant J. Jensen. 2019. “Molecular Architecture, Polar Targeting and Biogenesis of the Legionella Dot/Icm T4ss.” <em>Nat Microbiol</em> 4 (7): 1173–82. doi:<a href="https://doi.org/10.1038/s41564-019-0427-4">10.1038/s41564-019-0427-4</a>.</p>
</div>
<div id="ref-ghosal2019">
<p>Ghosal, Debnath, Ki Woo Kim, Huaixin Zheng, Mohammed Kaplan, Hilary K. Truchan, Alberto E. Lopez, Ian E. McIntire, Joseph P. Vogel, Nicholas P. Cianciotto, and Grant J. Jensen. 2019. “In Vivo Structure of the Legionella Type II Secretion System by Electron Cryotomography.” <em>Nature Microbiology</em> 4 (12): 2101–8. doi:<a href="https://doi.org/10.1038/s41564-019-0603-6">10.1038/s41564-019-0603-6</a>.</p>
</div>
<div id="ref-shikuma2014">
<p>Shikuma, N. J., M. Pilhofer, G. L. Weiss, M. G. Hadfield, G. J. Jensen, and D. K. Newman. 2014. “Marine Tubeworm Metamorphosis Induced by Arrays of Bacterial Phage Tail-Like Structures.” <em>Science</em> 343 (6170): 529–33. doi:<a href="https://doi.org/10.1126/science.1246794">10.1126/science.1246794</a>.</p>
</div>
<div id="ref-sockett2009">
<p>Sockett, R. E. 2009. “Predatory Lifestyle of Bdellovibrio Bacteriovorus.” <em>Annu Rev Microbiol</em> 63: 523–39. doi:<a href="https://doi.org/10.1146/annurev.micro.091208.073346">10.1146/annurev.micro.091208.073346</a>.</p>
</div>
<div id="ref-tachiyama2019">
<p>Tachiyama, Shoichi, Yunjie Chang, Meenakumari Muthuramalingam, Bo Hu, Michael L. Barta, Wendy L. Picking, Jun Liu, and William D. Picking. 2019. “The Cytoplasmic Domain of MxiG Interacts with MxiK and Directs Assembly of the Sorting Platform in the Shigella Type III Secretion System.” <em>J. Biol. Chem.</em> 294 (50). American Society for Biochemistry and Molecular Biology: 19184–96. doi:<a href="https://doi.org/10.1074/jbc.RA119.009125">10.1074/jbc.RA119.009125</a>.</p>
</div>
<div id="ref-thomas1974">
<p>Thomas, L. 1974. <em>The Lives of a Cell</em>. New York: The Viking Press.</p>
</div>
<div id="ref-yan2017a">
<p>Yan, Zhaofeng, Meng Yin, Dandan Xu, Yongqun Zhu, and Xueming Li. 2017. “Structural Insights into the Secretin Translocation Channel in the Type II Secretion System.” <em>Nature Structural &amp; Molecular Biology</em> 24 (2). Nature Publishing Group: 177–83. doi:<a href="https://doi.org/10.1038/nsmb.3350">10.1038/nsmb.3350</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="lifecycle.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="viruses.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": null,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/caltechlibrary/cell-atlas/edit/master/09_Interaction.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["cell-atlas.pdf", "cell-atlas.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
