<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 10 Interaction | Cell Atlas</title>
  <meta name="description" content="An atlas produced by the Jensen Lab." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 10 Interaction | Cell Atlas" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="splash/HeaderImage.jpg" />
  <meta property="og:description" content="An atlas produced by the Jensen Lab." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 10 Interaction | Cell Atlas" />
  
  <meta name="twitter:description" content="An atlas produced by the Jensen Lab." />
  <meta name="twitter:image" content="splash/HeaderImage.jpg" />

<meta name="author" content="Jensen Lab" />


<meta name="date" content="2020-07-29" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="lifecycle.html"/>
<link rel="next" href="viruses.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/CL-0.1.2/CL.js"></script>
<script src="libs/doivideo-binding-0.0.0.9000/doivideo.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Cell Atlas</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#section"><i class="fa fa-check"></i><b>1.1</b> 1</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#schematicstructural-biology-toolkit"><i class="fa fa-check"></i><b>2.1</b> 1[Schematic–Structural biology toolkit]</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#escherichia-coli"><i class="fa fa-check"></i><b>2.2</b> Escherichia coli</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#shewanella-oneidensis"><i class="fa fa-check"></i><b>2.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#schematicelectron-microscopy-modes"><i class="fa fa-check"></i><b>2.4</b> 3[Schematic–Electron microscopy modes]</a></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#shewanella-oneidensis-1"><i class="fa fa-check"></i><b>2.5</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="2.6" data-path="methods.html"><a href="methods.html#caulobacter-crescentus"><i class="fa fa-check"></i><b>2.6</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="2.7" data-path="methods.html"><a href="methods.html#caulobacter-crescentus-1"><i class="fa fa-check"></i><b>2.7</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="2.8" data-path="methods.html"><a href="methods.html#schematicmissing-wedge"><i class="fa fa-check"></i><b>2.8</b> 6[Schematic–Missing wedge]</a></li>
<li class="chapter" data-level="2.9" data-path="methods.html"><a href="methods.html#campylobacter-jejuni"><i class="fa fa-check"></i><b>2.9</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.10" data-path="methods.html"><a href="methods.html#campylobacter-jejuni-1"><i class="fa fa-check"></i><b>2.10</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.11" data-path="methods.html"><a href="methods.html#campylobacter-jejuni-2"><i class="fa fa-check"></i><b>2.11</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.12" data-path="methods.html"><a href="methods.html#schematicstructural-biology-timeline"><i class="fa fa-check"></i><b>2.12</b> 9[Schematic–Structural biology timeline]</a></li>
<li class="chapter" data-level="2.13" data-path="methods.html"><a href="methods.html#bdellovibrio-bacteriovorus"><i class="fa fa-check"></i><b>2.13</b> Bdellovibrio bacteriovorus</a><ul>
<li class="chapter" data-level="2.13.1" data-path="methods.html"><a href="methods.html#Difference_mapping"><i class="fa fa-check"></i><b>2.13.1</b> Campylobacter jejuni</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="envelope.html"><a href="envelope.html"><i class="fa fa-check"></i><b>3</b> Envelope</a><ul>
<li class="chapter" data-level="3.1" data-path="envelope.html"><a href="envelope.html#schematic-lipid-bilayer"><i class="fa fa-check"></i><b>3.1</b> 1[Schematic: Lipid bilayer]</a></li>
<li class="chapter" data-level="3.2" data-path="envelope.html"><a href="envelope.html#schematic-atp-synthase-and-energy-production"><i class="fa fa-check"></i><b>3.2</b> 1[Schematic: ATP synthase and energy production]</a></li>
<li class="chapter" data-level="3.3" data-path="envelope.html"><a href="envelope.html#listeria-monocytogenes"><i class="fa fa-check"></i><b>3.3</b> Listeria monocytogenes</a><ul>
<li class="chapter" data-level="3.3.1" data-path="envelope.html"><a href="envelope.html#Peptidoglycan_architecture"><i class="fa fa-check"></i><b>3.3.1</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="3.3.2" data-path="envelope.html"><a href="envelope.html#Methanobacterium_formicicum"><i class="fa fa-check"></i><b>3.3.2</b> Methanobacterium formicicum</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="envelope.html"><a href="envelope.html#cupriavidus-necator"><i class="fa fa-check"></i><b>3.4</b> Cupriavidus necator</a></li>
<li class="chapter" data-level="3.5" data-path="envelope.html"><a href="envelope.html#schematicbrauns-lipoprotein"><i class="fa fa-check"></i><b>3.5</b> 3[Schematic–Braun’s lipoprotein]</a><ul>
<li class="chapter" data-level="3.5.1" data-path="envelope.html"><a href="envelope.html#escherichia-coli-diderm_sacculus_architecturecompare-this-diderm-sacculus-purified-from-escherichia-coli-to-the-monoderm-sacculus-on-the-last-page.-since-this-one-is-thinner-we-can-make-out-more-details-like-the-glycan-strands-running-around-the-circumference-of-the-cell.-the-main-difference-between-the-two-types-of-sacculi-seems-to-be-whether-they-have-largely-a-single-layer-of-peptidoglycan-diderm-or-many-layers-monoderm.-so-even-though-the-two-types-of-wall-look-different-their-architecture-is-fundamentally-the-same.-in-some-circumstances-as-you-will-see-in-chapter-8-cells-can-even-switch-between-the-two-forms."><i class="fa fa-check"></i><b>3.5.1</b> Escherichia coli {#Diderm_sacculus_architecture}Compare this diderm sacculus purified from <em>Escherichia coli</em> to the monoderm sacculus on the last page. Since this one is thinner, we can make out more details, like the glycan strands running around the circumference of the cell. The main difference between the two types of sacculi seems to be whether they have largely a single layer of peptidoglycan (diderm) or many layers (monoderm). So even though the two types of wall look different, their architecture is fundamentally the same. In some circumstances, as you will see in Chapter 8, cells can even switch between the two forms.</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="envelope.html"><a href="envelope.html#schematicsacculus-remodeling"><i class="fa fa-check"></i><b>3.6</b> 3[Schematic–Sacculus remodeling]</a></li>
<li class="chapter" data-level="3.7" data-path="envelope.html"><a href="envelope.html#myxococcus-xanthus"><i class="fa fa-check"></i><b>3.7</b> Myxococcus xanthus</a><ul>
<li class="chapter" data-level="3.7.1" data-path="envelope.html"><a href="envelope.html#borrelia-burgdorferi-pearled_vesiclesdifferent-species-can-produce-outer-membrane-vesicles-that-look-very-different.-the-same-species-can-even-produce-vesicles-that-look-very-different.-sometimes-they-come-off-the-cell-as-a-chain-of-spheres-sometimes-the-spheres-remain-connected-like-a-string-of-pearls-as-in-this-borrelia-burgdorferi-cell.-sometimes-vesicles-form-long-tubes-instead-moretubular-vesicles.-sometimes-the-same-chain-can-be-tubular-in-one-section-usually-at-the-base-connected-to-the-cell-and-a-string-of-spheres-in-another."><i class="fa fa-check"></i><b>3.7.1</b> Borrelia burgdorferi {#Pearled_vesicles}Different species can produce outer membrane vesicles that look very different. The same species can even produce vesicles that look very different. Sometimes they come off the cell as a chain of spheres; sometimes the spheres remain connected, like a string of pearls, as in this <em>Borrelia burgdorferi</em> cell. Sometimes vesicles form long tubes instead [More–Tubular vesicles]. Sometimes the same chain can be tubular in one section (usually at the base, connected to the cell), and a string of spheres in another.</a></li>
<li class="chapter" data-level="3.7.2" data-path="envelope.html"><a href="envelope.html#borrelia-burgdorferi-tubular_vesicleshere-again-from-a-borrelia-burgdorferi-cell-you-can-see-extended-tubular-outer-membrane-vesicles."><i class="fa fa-check"></i><b>3.7.2</b> Borrelia burgdorferi {#Tubular_vesicles}Here, again from a <em>Borrelia burgdorferi</em> cell, you can see extended, tubular outer membrane vesicles.</a></li>
<li class="chapter" data-level="3.7.3" data-path="envelope.html"><a href="envelope.html#myxococcus-xanthus-cytoplasmic_vesiclesnot-all-vesicles-come-from-the-outer-membrane.-the-cytoplasmic-or-inner-membrane-can-also-form-vesicles-that-are-released-into-the-cytoplasm-as-in-this-myxococcus-xanthus-cell-or-into-the-periplasm."><i class="fa fa-check"></i><b>3.7.3</b> Myxococcus xanthus {#Cytoplasmic_vesicles}Not all vesicles come from the outer membrane. The cytoplasmic or inner membrane can also form vesicles that are released into the cytoplasm, as in this <em>Myxococcus xanthus</em> cell, or into the periplasm.</a></li>
<li class="chapter" data-level="3.7.4" data-path="envelope.html"><a href="envelope.html#prosthecobacter-debontii-cytoplasmic_vesicle_varietycytoplasmic-vesicles-exhibit-a-variety-of-sizes-and-shapes.-some-are-nested-with-vesicles-inside-vesicles.-in-this-prosthecobacter-debontii-cell-you-can-see-two-other-morphologies.-one-is-a-large-flattened-horseshoe-shaped-vesicle.-another-is-a-more-typical-spherical-shape-but-is-decorated-with-what-look-like-protein-complexes."><i class="fa fa-check"></i><b>3.7.4</b> Prosthecobacter debontii {#Cytoplasmic_vesicle_variety}Cytoplasmic vesicles exhibit a variety of sizes and shapes. Some are nested, with vesicles inside vesicles. In this <em>Prosthecobacter debontii</em> cell, you can see two other morphologies. One is a large, flattened horseshoe-shaped vesicle. Another is a more typical spherical shape, but is decorated with what look like protein complexes.</a></li>
<li class="chapter" data-level="3.7.5" data-path="envelope.html"><a href="envelope.html#halomicrobium-mukohataei-archaeal_vesiclesjust-like-bacteria-archaea-also-produce-both-internal-and-external-vesicles-as-you-can-see-in-this-halomicrobium-mukohataei-cell."><i class="fa fa-check"></i><b>3.7.5</b> Halomicrobium mukohataei {#Archaeal_vesicles}Just like bacteria, archaea also produce both internal and external vesicles, as you can see in this <em>Halomicrobium mukohataei</em> cell.</a></li>
</ul></li>
<li class="chapter" data-level="3.8" data-path="envelope.html"><a href="envelope.html#mycobacterium-marinum"><i class="fa fa-check"></i><b>3.8</b> Mycobacterium marinum</a></li>
<li class="chapter" data-level="3.9" data-path="envelope.html"><a href="envelope.html#caulobacter-crescentus-2"><i class="fa fa-check"></i><b>3.9</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="3.10" data-path="envelope.html"><a href="envelope.html#schematicsurface-layer-architecture"><i class="fa fa-check"></i><b>3.10</b> 6[Schematic–Surface layer architecture]</a></li>
<li class="chapter" data-level="3.11" data-path="envelope.html"><a href="envelope.html#aarchaea-and-bacteria-with-surface-layers-produce-characteristic-outer-membrane-vesicles-they-bud-off-with-the-surface-layer-attached.-you-will-see-plenty-of-examples-in-the-coming-pages.-delftia-acidovorans-like-this-produce-so-called-nanopods-chains-of-outer-membrane-vesicles-ensheathed-in-surface-layer."><i class="fa fa-check"></i><b>3.11</b> 6aArchaea and bacteria with surface layers produce characteristic outer membrane vesicles: they bud off with the surface layer attached. You will see plenty of examples in the coming pages. <em>Delftia acidovorans</em> like this produce so-called nanopods: chains of outer membrane vesicles ensheathed in surface layer.</a></li>
<li class="chapter" data-level="3.12" data-path="envelope.html"><a href="envelope.html#sulfolobus-solfataricus"><i class="fa fa-check"></i><b>3.12</b> Sulfolobus solfataricus</a><ul>
<li class="chapter" data-level="3.12.1" data-path="envelope.html"><a href="envelope.html#methylomicrobium-alcaliphilum-m._alcaliphilumin-methylomicrobium-alcaliphilum-v-shaped-surface-layer-proteins-come-together-to-form-cups-that-pack-into-a-hexagonal-pattern."><i class="fa fa-check"></i><b>3.12.1</b> Methylomicrobium alcaliphilum {#M._alcaliphilum}In <em>Methylomicrobium alcaliphilum</em>, V-shaped surface layer proteins come together to form cups that pack into a hexagonal pattern.</a></li>
<li class="chapter" data-level="3.12.2" data-path="envelope.html"><a href="envelope.html#clostridium-thermocellum-clostridium_thermocellumnot-only-diderm-bacteria-have-surface-layers-as-you-can-see-on-this-monoderm-clostridium-thermocellum-cell.-the-main-difference-is-that-the-proteins-are-anchored-to-the-cell-wall-rather-than-to-the-outer-membrane-or-associated-lipoproteins."><i class="fa fa-check"></i><b>3.12.2</b> Clostridium thermocellum {#Clostridium_thermocellum}Not only diderm bacteria have surface layers, as you can see on this monoderm <em>Clostridium thermocellum</em> cell. The main difference is that the proteins are anchored to the cell wall, rather than to the outer membrane (or associated lipoproteins).</a></li>
<li class="chapter" data-level="3.12.3" data-path="envelope.html"><a href="envelope.html#nitrosopumilis-maritimus-n._maritimusin-nitrosopumilis-maritimus-surface-layer-proteins-form-hexagonal-rosettes-that-in-turn-pack-into-a-hexagonal-lattice."><i class="fa fa-check"></i><b>3.12.3</b> Nitrosopumilis maritimus {#N._maritimus}In <em>Nitrosopumilis maritimus</em>, surface layer proteins form hexagonal rosettes that in turn pack into a hexagonal lattice.</a></li>
<li class="chapter" data-level="3.12.4" data-path="envelope.html"><a href="envelope.html#methanoregula-formicica-methanoregula_formicicathe-surface-layer-of-methanoregula-formicica-is-a-hexagonal-lattice-of-small-nearly-circular-subunits."><i class="fa fa-check"></i><b>3.12.4</b> Methanoregula formicica {#Methanoregula_formicica}The surface layer of <em>Methanoregula formicica</em> is a hexagonal lattice of small, nearly circular subunits.</a></li>
</ul></li>
<li class="chapter" data-level="3.13" data-path="envelope.html"><a href="envelope.html#methanospirillum-hungatei"><i class="fa fa-check"></i><b>3.13</b> Methanospirillum hungatei</a></li>
<li class="chapter" data-level="3.14" data-path="envelope.html"><a href="envelope.html#haloarcula-argentinensis"><i class="fa fa-check"></i><b>3.14</b> Haloarcula argentinensis</a></li>
<li class="chapter" data-level="3.15" data-path="envelope.html"><a href="envelope.html#bdellovibrio-bacteriovorus-1"><i class="fa fa-check"></i><b>3.15</b> Bdellovibrio bacteriovorus</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="shape.html"><a href="shape.html"><i class="fa fa-check"></i><b>4</b> Shape</a><ul>
<li class="chapter" data-level="4.1" data-path="shape.html"><a href="shape.html#cupriavidus-necator-1"><i class="fa fa-check"></i><b>4.1</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="4.1.1" data-path="shape.html"><a href="shape.html#brucella-abortus-rod_varietynot-all-rod-shaped-cells-are-perfectly-cylindrical.-for-instance-brucella-abortus-like-this-one-are-pear-shaped.-this-is-one-of-the-rod-shaped-bacterial-species-that-do-not-use-mreb."><i class="fa fa-check"></i><b>4.1.1</b> Brucella abortus {#Rod_variety}Not all rod-shaped cells are perfectly cylindrical. For instance, <em>Brucella abortus</em> like this one are pear-shaped. This is one of the rod-shaped bacterial species that do not use MreB.</a></li>
<li class="chapter" data-level="4.1.2" data-path="shape.html"><a href="shape.html#methanoregula-formicica-archaeal_rodssome-archaeal-species-like-this-methanoregula-formicica-are-rod-shaped.-the-rigid-surface-layer-seems-to-help-define-the-shape-but-the-details-are-still-unknown."><i class="fa fa-check"></i><b>4.1.2</b> Methanoregula formicica {#Archaeal_rods}Some archaeal species, like this <em>Methanoregula formicica</em>, are rod-shaped. The rigid surface layer seems to help define the shape, but the details are still unknown.</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="shape.html"><a href="shape.html#hylemonella-gracilis"><i class="fa fa-check"></i><b>4.2</b> Hylemonella gracilis</a></li>
<li class="chapter" data-level="4.3" data-path="shape.html"><a href="shape.html#caulobacter-crescentus-3"><i class="fa fa-check"></i><b>4.3</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="4.4" data-path="shape.html"><a href="shape.html#schematic-ctp-synthase"><i class="fa fa-check"></i><b>4.4</b> 4[Schematic: CTP synthase]</a></li>
<li class="chapter" data-level="4.5" data-path="shape.html"><a href="shape.html#campylobacter-jejuni-3"><i class="fa fa-check"></i><b>4.5</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="4.5.1" data-path="shape.html"><a href="shape.html#borrelia-burgdorferi-borrelia_burgdorferiborrelia-burgdorferi-like-this-cell-use-the-long-filaments-of-their-motility-machinery-flagella-discussed-in-chapter-6-as-a-kind-of-cytoskeleton.-a-bundle-of-flagella-wraps-around-the-cell-in-the-periplasm-between-the-cell-wall-and-the-outer-membrane.-spun-by-motors-at-their-base-more-on-that-in-chapter-6-the-filaments-impart-a-wave-pattern-to-the-growing-sacculus.-without-the-motors-rotation-the-cells-develop-a-rod-shape.-note-that-b.-burgdorferi-are-not-helical-but-rather-adopt-a-two-dimensional-waveform."><i class="fa fa-check"></i><b>4.5.1</b> Borrelia burgdorferi {#Borrelia_burgdorferi}<em>Borrelia burgdorferi</em> like this cell use the long filaments of their motility machinery (flagella, discussed in Chapter 6) as a kind of cytoskeleton. A bundle of flagella wraps around the cell in the periplasm, between the cell wall and the outer membrane. Spun by motors at their base (more on that in Chapter 6), the filaments impart a wave pattern to the growing sacculus. Without the motors’ rotation, the cells develop a rod shape. Note that <em>B. burgdorferi</em> are not helical, but rather adopt a two-dimensional waveform.</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="shape.html"><a href="shape.html#verrucomicrobium-spinosum"><i class="fa fa-check"></i><b>4.6</b> Verrucomicrobium spinosum</a><ul>
<li class="chapter" data-level="4.6.1" data-path="shape.html"><a href="shape.html#Bactofilin"><i class="fa fa-check"></i><b>4.6.1</b> Bactofilin</a></li>
<li class="chapter" data-level="4.6.2" data-path="shape.html"><a href="shape.html#prosthecobacter-vanneervenii-bacterial_microtubulessome-bacterial-species-with-prosthecae-express-structures-similar-to-eukaryotic-microtubules-made-from-two-proteins-called-btuba-and-btubb-to-reflect-their-homology-to-eukaryotic-tubulins.-eukaryotic-microtubules-are-hollow-tubes-formed-by-13-protofilaments-bacterial-microtubules-are-smaller-with-only-5-protofilaments.-cells-commonly-contain-a-bundle-of-microtubules-in-their-prostheca-like-this-prosthecobacter-vanneervenii-cell-which-has-a-bundle-of-four."><i class="fa fa-check"></i><b>4.6.2</b> Prosthecobacter vanneervenii {#Bacterial_microtubules}Some bacterial species with prosthecae express structures similar to eukaryotic microtubules, made from two proteins called BtubA and BtubB to reflect their homology to eukaryotic tubulins. Eukaryotic microtubules are hollow tubes formed by 13 protofilaments; bacterial microtubules are smaller, with only ~5 protofilaments. Cells commonly contain a bundle of microtubules in their prostheca, like this <em>Prosthecobacter vanneervenii</em> cell, which has a bundle of four.</a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="shape.html"><a href="shape.html#haloquadratum-walsbyi"><i class="fa fa-check"></i><b>4.7</b> Haloquadratum walsbyi</a><ul>
<li class="chapter" data-level="4.7.1" data-path="shape.html"><a href="shape.html#halobacterium-salinarum-gas_vesiclessome-species-of-archaea-and-bacteria-use-gas-vesicles-to-control-their-buoyancy.-this-can-allow-them-to-rise-or-fall-in-a-water-column-which-can-be-a-great-advantage.-halobacterium-salinarum-like-this-one-produce-gas-vesicles-in-response-to-cues-from-the-environment-lifting-themselves-out-of-the-sediment-and-into-more-favorable-conditions-of-oxygen-or-sunlight-for-photosynthesis.-this-cell-has-just-started-producing-gas-vesicles-so-they-are-small-and-isolated.-later-the-vesicles-will-elongate-into-cylinders-with-conical-ends-as-you-saw-in-haloquadratum-walsbyi.-each-cell-might-contain-dozens-of-vesicles-and-they-often-cluster-together."><i class="fa fa-check"></i><b>4.7.1</b> Halobacterium salinarum {#Gas_vesicles}Some species of archaea and bacteria use gas vesicles to control their buoyancy. This can allow them to rise or fall in a water column, which can be a great advantage. <em>Halobacterium salinarum</em> like this one produce gas vesicles in response to cues from the environment, lifting themselves out of the sediment and into more favorable conditions of oxygen or sunlight for photosynthesis. This cell has just started producing gas vesicles, so they are small and isolated. Later, the vesicles will elongate into cylinders with conical ends, as you saw in <em>Haloquadratum walsbyi</em>. Each cell might contain dozens of vesicles, and they often cluster together.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="growth.html"><a href="growth.html"><i class="fa fa-check"></i><b>5</b> Growth</a><ul>
<li class="chapter" data-level="5.1" data-path="growth.html"><a href="growth.html#shewanella-oneidensis-2"><i class="fa fa-check"></i><b>5.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="5.2" data-path="growth.html"><a href="growth.html#rhodopseudomonas-palustris"><i class="fa fa-check"></i><b>5.2</b> Rhodopseudomonas palustris</a></li>
<li class="chapter" data-level="5.3" data-path="growth.html"><a href="growth.html#schematic-photocomplexes"><i class="fa fa-check"></i><b>5.3</b> 3[Schematic: Photocomplexes]</a><ul>
<li class="chapter" data-level="5.3.1" data-path="growth.html"><a href="growth.html#methyloprofundus-sedimenti-icm_varietyin-some-species-like-this-methyloprofundus-sedimenti-the-intracytoplasmic-membrane-is-very-extensive-and-appears-to-be-fully-separated-from-the-cell-membrane.-as-you-can-see-it-occupies-much-of-the-cytoplasmic-space."><i class="fa fa-check"></i><b>5.3.1</b> Methyloprofundus sedimenti {#ICM_variety}In some species, like this <em>Methyloprofundus sedimenti</em>, the intracytoplasmic membrane is very extensive and appears to be fully separated from the cell membrane. As you can see, it occupies much of the cytoplasmic space.</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="growth.html"><a href="growth.html#chloroflexi"><i class="fa fa-check"></i><b>5.4</b> Chloroflexi</a></li>
<li class="chapter" data-level="5.5" data-path="growth.html"><a href="growth.html#clostridium-thermocellum"><i class="fa fa-check"></i><b>5.5</b> Clostridium thermocellum</a></li>
<li class="chapter" data-level="5.6" data-path="growth.html"><a href="growth.html#schematicadhe-spirosome-structure"><i class="fa fa-check"></i><b>5.6</b> 5[Schematic–AdhE spirosome structure]</a></li>
<li class="chapter" data-level="5.7" data-path="growth.html"><a href="growth.html#acetonema-longum"><i class="fa fa-check"></i><b>5.7</b> Acetonema longum</a></li>
<li class="chapter" data-level="5.8" data-path="growth.html"><a href="growth.html#schematicmicrocompartment-shells"><i class="fa fa-check"></i><b>5.8</b> 6[Schematic–Microcompartment shells]</a></li>
<li class="chapter" data-level="5.9" data-path="growth.html"><a href="growth.html#thiomonas-intermedia"><i class="fa fa-check"></i><b>5.9</b> Thiomonas intermedia</a><ul>
<li class="chapter" data-level="5.9.1" data-path="growth.html"><a href="growth.html#halothiobacillus-neapolitanus-carboxysome_varietymost-carboxysomes-look-like-the-ones-you-saw-on-the-last-page-but-not-all.-some-are-less-regular-polyhedra-as-in-this-halothiobacillus-neapolitanus-cell.-and-some-are-stranger-still-morelong-carboxysomes."><i class="fa fa-check"></i><b>5.9.1</b> Halothiobacillus neapolitanus {#Carboxysome_variety}Most carboxysomes look like the ones you saw on the last page, but not all. Some are less regular polyhedra, as in this <em>Halothiobacillus neapolitanus</em> cell. And some are stranger still [More–Long carboxysomes].</a></li>
<li class="chapter" data-level="5.9.2" data-path="growth.html"><a href="growth.html#hydrogenovibrio-crunogenus-long_carboxysomesthe-carboxysomes-in-this-hydrogenovibrio-crunogenus-have-grown-abnormally-long-extending-across-the-full-width-of-the-cell.-these-are-unusual-but-less-dramatic-elongations-are-common."><i class="fa fa-check"></i><b>5.9.2</b> Hydrogenovibrio crunogenus {#Long_carboxysomes}The carboxysomes in this <em>Hydrogenovibrio crunogenus</em> have grown abnormally long, extending across the full width of the cell. These are unusual, but less dramatic elongations are common.</a></li>
</ul></li>
<li class="chapter" data-level="5.10" data-path="growth.html"><a href="growth.html#cupriavidus-necator-2"><i class="fa fa-check"></i><b>5.10</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="5.10.1" data-path="growth.html"><a href="growth.html#lysobacter-antibioticus-storage_granule_growthstorage-granules-expand-and-contract-to-accommodate-their-contents.-this-lysobacter-antibioticus-cell-exhibits-a-typical-range-of-sizes-of-poly-phosphate-granules."><i class="fa fa-check"></i><b>5.10.1</b> Lysobacter antibioticus {#Storage_granule_growth}Storage granules expand and contract to accommodate their contents. This <em>Lysobacter antibioticus</em> cell exhibits a typical range of sizes of poly-phosphate granules.</a></li>
</ul></li>
<li class="chapter" data-level="5.11" data-path="growth.html"><a href="growth.html#agrobacterium-tumefaciens"><i class="fa fa-check"></i><b>5.11</b> Agrobacterium tumefaciens</a><ul>
<li class="chapter" data-level="5.11.1" data-path="growth.html"><a href="growth.html#hydrogenovibrio-crunogenus-granule-containing_carboxysomessmall-poly-phosphate-storage-granules-are-sometimes-seen-inside-carboxysomes-as-in-this-hydrogenovibrio-crunogenus-cell.-their-purpose-is-unknown."><i class="fa fa-check"></i><b>5.11.1</b> Hydrogenovibrio crunogenus {#Granule-containing_carboxysomes}Small poly-phosphate storage granules are sometimes seen <em>inside</em> carboxysomes, as in this <em>Hydrogenovibrio crunogenus</em> cell. Their purpose is unknown.</a></li>
<li class="chapter" data-level="5.11.2" data-path="growth.html"><a href="growth.html#halothiobacillus-neapolitanus-carboxysomes-granule_connectionswe-sometimes-see-comb-like-structures-connecting-external-poly-phosphate-storage-granules-with-carboxysomes-as-in-this-halothiobacillus-neapolitanus-cell.-the-nature-of-the-relationship-between-the-structures-and-the-identity-and-function-of-the-combs-remains-a-mystery."><i class="fa fa-check"></i><b>5.11.2</b> Halothiobacillus neapolitanus {#Carboxysomes-granule_connections}We sometimes see comb-like structures connecting external poly-phosphate storage granules with carboxysomes, as in this <em>Halothiobacillus neapolitanus</em> cell. The nature of the relationship between the structures, and the identity and function of the combs, remains a mystery.</a></li>
</ul></li>
<li class="chapter" data-level="5.12" data-path="growth.html"><a href="growth.html#halorubrum-litoreum"><i class="fa fa-check"></i><b>5.12</b> Halorubrum litoreum</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="division.html"><a href="division.html"><i class="fa fa-check"></i><b>6</b> Division</a><ul>
<li class="chapter" data-level="6.1" data-path="division.html"><a href="division.html#caulobacter-crescentus-4"><i class="fa fa-check"></i><b>6.1</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="6.2" data-path="division.html"><a href="division.html#hyphomonas-neptunium"><i class="fa fa-check"></i><b>6.2</b> Hyphomonas neptunium</a><ul>
<li class="chapter" data-level="6.2.1" data-path="division.html"><a href="division.html#hyphomonas-neptunium-hyphomonas_stalkhyphomonas-neptunium-grow-a-single-stalk-from-one-end-of-their-cell-body-similar-to-the-caulobacter-crescentus-you-saw-in-chapter-3.-the-function-of-the-stalk-though-is-different-in-this-budding-bacterium-than-in-c.-crescentus-which-divide-by-more-conventional-fission."><i class="fa fa-check"></i><b>6.2.1</b> Hyphomonas neptunium {#Hyphomonas_stalk}<em>Hyphomonas neptunium</em> grow a single stalk from one end of their cell body, similar to the <em>Caulobacter crescentus</em> you saw in Chapter 3. The function of the stalk, though, is different in this budding bacterium than in <em>C. crescentus</em>, which divide by more conventional fission.</a></li>
<li class="chapter" data-level="6.2.2" data-path="division.html"><a href="division.html#hyphomonas-neptunium-hyphomonas_lifecyclehyphomonas-neptunium-have-evolved-a-program-of-stages-they-pass-through-in-the-course-of-their-life.-when-a-newborn-cell-like-this-one-is-released-it-is-in-the-swarmer-stage-using-its-flagellum-discussed-in-the-next-chapter-to-swim-away-to-a-new-location.-after-a-while-it-settles-down-jettisoning-its-flagellum-and-growing-a-stalk-with-which-to-make-its-own-budthe-next-generation.-once-a-cell-settles-down-into-the-stalked-stage-it-spends-the-rest-of-its-life-sending-off-buds-as-long-as-conditions-are-good.-only-newborn-cells-are-equipped-with-flagella-to-go-adventuring.-we-will-discuss-more-lifecycles-in-chapter-8."><i class="fa fa-check"></i><b>6.2.2</b> Hyphomonas neptunium {#Hyphomonas_lifecycle}<em>Hyphomonas neptunium</em> have evolved a program of stages they pass through in the course of their life. When a newborn cell like this one is released, it is in the “swarmer” stage, using its flagellum (discussed in the next chapter) to swim away to a new location. After a while, it settles down, jettisoning its flagellum and growing a stalk with which to make its own bud–the next generation. Once a cell settles down into the stalked stage, it spends the rest of its life sending off buds as long as conditions are good. Only newborn cells are equipped with flagella to go adventuring. We will discuss more lifecycles in Chapter 8.</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="division.html"><a href="division.html#staphylococcus-aureus"><i class="fa fa-check"></i><b>6.3</b> Staphylococcus aureus</a></li>
<li class="chapter" data-level="6.4" data-path="division.html"><a href="division.html#tetrasphaera-remsis"><i class="fa fa-check"></i><b>6.4</b> Tetrasphaera remsis</a></li>
<li class="chapter" data-level="6.5" data-path="division.html"><a href="division.html#idiomarina-loihiensis"><i class="fa fa-check"></i><b>6.5</b> Idiomarina loihiensis</a><ul>
<li class="chapter" data-level="6.5.1" data-path="division.html"><a href="division.html#sphingopyxis-alaskensis-sphingopyxis_alaskensis_divisionthis-diderm-cell-is-about-halfway-through-cytokinesis.-the-two-daughter-cells-are-still-connected-by-a-wide-bridge-of-cytoplasm-but-the-inner-membrane-and-cell-wall-are-drawing-in-all-around."><i class="fa fa-check"></i><b>6.5.1</b> Sphingopyxis alaskensis {#Sphingopyxis_alaskensis_division}This diderm cell is about halfway through cytokinesis. The two daughter cells are still connected by a wide bridge of cytoplasm, but the inner membrane and cell wall are drawing in all around.</a></li>
<li class="chapter" data-level="6.5.2" data-path="division.html"><a href="division.html#chloroflexi-chloroflexi_divisionthis-diderm-cell-is-in-the-final-stage-of-cytokinesis.-the-cytoplasm-of-the-two-daughter-cells-is-completely-separated-by-inner-membrane-and-cell-wall.-the-outer-membrane-though-has-not-yet-divided."><i class="fa fa-check"></i><b>6.5.2</b> Chloroflexi {#Chloroflexi_division}This diderm cell is in the final stage of cytokinesis. The cytoplasm of the two daughter cells is completely separated by inner membrane and cell wall. The outer membrane, though, has not yet divided.</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="division.html"><a href="division.html#caulobacter-crescentus-5"><i class="fa fa-check"></i><b>6.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="6.6.1" data-path="division.html"><a href="division.html#caulobacter-crescentus-caulobacter_divisionthese-daughter-cells-have-completely-separated-their-mothers-inner-membrane-and-cell-wall-but-not-yet-the-outer-membrane-and-surface-layer."><i class="fa fa-check"></i><b>6.6.1</b> Caulobacter crescentus {#Caulobacter_division}These daughter cells have completely separated their mother’s inner membrane and cell wall, but not yet the outer membrane and surface layer.</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="division.html"><a href="division.html#agrobacterium-tumefaciens-1"><i class="fa fa-check"></i><b>6.7</b> Agrobacterium tumefaciens</a></li>
<li class="chapter" data-level="6.8" data-path="division.html"><a href="division.html#cupriavidus-necator-3"><i class="fa fa-check"></i><b>6.8</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="6.8.1" data-path="division.html"><a href="division.html#cupriavidus-necator-asymmetric_constrictionthis-c.-necator-cell-is-at-a-late-stage-of-cytokinesis.-in-most-cells-constriction-would-be-more-or-less-uniform-around-the-circumference-by-now-but-in-some-cells-including-this-one-the-asymmetry-persists."><i class="fa fa-check"></i><b>6.8.1</b> Cupriavidus necator {#Asymmetric_constriction}This <em>C. necator</em> cell is at a late stage of cytokinesis. In most cells, constriction would be more or less uniform around the circumference by now, but in some cells, including this one, the asymmetry persists.</a></li>
</ul></li>
<li class="chapter" data-level="6.9" data-path="division.html"><a href="division.html#schematicftsz-structure"><i class="fa fa-check"></i><b>6.9</b> 9[Schematic–FtsZ structure]</a></li>
<li class="chapter" data-level="6.10" data-path="division.html"><a href="division.html#caulobacter-crescentus-6"><i class="fa fa-check"></i><b>6.10</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="6.11" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius"><i class="fa fa-check"></i><b>6.11</b> Sulfolobus acidocaldarius</a></li>
<li class="chapter" data-level="6.12" data-path="division.html"><a href="division.html#schematicescrt-structure"><i class="fa fa-check"></i><b>6.12</b> 11[Schematic–ESCRT structure]</a></li>
<li class="chapter" data-level="6.13" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius-1"><i class="fa fa-check"></i><b>6.13</b> Sulfolobus acidocaldarius</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="motility.html"><a href="motility.html"><i class="fa fa-check"></i><b>7</b> Motility</a><ul>
<li class="chapter" data-level="7.1" data-path="motility.html"><a href="motility.html#schematicflagellum-structure"><i class="fa fa-check"></i><b>7.1</b> 1[Schematic–Flagellum structure]</a></li>
<li class="chapter" data-level="7.2" data-path="motility.html"><a href="motility.html#bdellovibrio-bacteriovorus-2"><i class="fa fa-check"></i><b>7.2</b> Bdellovibrio bacteriovorus</a><ul>
<li class="chapter" data-level="7.2.1" data-path="motility.html"><a href="motility.html#bdellovibrio-bacteriovorus-flagellar_motor_structurethis-is-an-average-of-the-flagellar-motors-from-more-than-one-thousand-bdellovibrio-bacteriovorus-cells.-working-upward-from-the-base-the-major-parts-of-the-rotor-are-the-c-ring-for-_c_ytoplasmic-the-ms-ring-for-_m_embrane-and-_s_upramembrane-the-rod-the-hook-and-finally-the-filament.-they-are-surrounded-by-stationary-parts-the-stator-ring-which-is-dynamic-so-we-cannot-resolve-the-stators-as-they-cross-the-inner-membrane-and-connect-to-the-c-ring-and-a-series-of-bushings-that-allow-rotation-within-the-cell-wall-the-_p_eriplasmic-or-p-ring-and-outer-membrane-_l_ipopolysaccharide-or-l-ring.-additional-cytoplasmic-components-form-the-export-apparatus-which-is-involved-in-assembly-discussed-on-the-next-page."><i class="fa fa-check"></i><b>7.2.1</b> Bdellovibrio bacteriovorus {#Flagellar_motor_structure}This is an average of the flagellar motors from more than one thousand <em>Bdellovibrio bacteriovorus</em> cells. Working upward from the base, the major parts of the rotor are the C-ring (for _C_ytoplasmic), the MS-ring (for _M_embrane and _S_upramembrane), the rod, the hook, and finally the filament. They are surrounded by stationary parts: the stator ring (which is dynamic, so we cannot resolve the stators as they cross the inner membrane and connect to the C-ring) and a series of bushings that allow rotation within the cell wall (the _P_eriplasmic or P-ring) and outer membrane (_L_ipopolysaccharide or L-ring). Additional cytoplasmic components form the export apparatus, which is involved in assembly (discussed on the next page).</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="motility.html"><a href="motility.html#schematicflagellar-motor-diversity"><i class="fa fa-check"></i><b>7.3</b> 2[Schematic–Flagellar motor diversity]</a></li>
<li class="chapter" data-level="7.4" data-path="motility.html"><a href="motility.html#hylemonella-gracilis-1"><i class="fa fa-check"></i><b>7.4</b> Hylemonella gracilis</a><ul>
<li class="chapter" data-level="7.4.1" data-path="motility.html"><a href="motility.html#pseudomonas-aeruginosa-flagellar_motor_disassemblywhen-flagella-break-or-in-some-species-like-caulobacter-crescentus-are-ejected-so-the-cell-can-attach-to-a-surface-the-motor-is-disassembled-usually-beginning-with-the-export-apparatus.-not-everything-is-dismantled-though-the-p--and-l-rings-remain-in-place-in-the-cell-wall-and-outer-membrane-respectively-as-you-can-see-in-this-pseudomonas-aeruginosa-cell.-we-do-not-know-if-the-cell-has-a-reason-for-leaving-them-there-but-one-possibility-is-that-they-may-function-as-a-plug-for-the-outer-membrane-when-the-hook-is-no-longer-present."><i class="fa fa-check"></i><b>7.4.1</b> Pseudomonas aeruginosa {#Flagellar_motor_disassembly}When flagella break (or in some species, like <em>Caulobacter crescentus</em>, are ejected so the cell can attach to a surface), the motor is disassembled, usually beginning with the export apparatus. Not everything is dismantled, though; the P- and L-rings remain in place in the cell wall and outer membrane, respectively, as you can see in this <em>Pseudomonas aeruginosa</em> cell. We do not know if the cell has a reason for leaving them there, but one possibility is that they may function as a plug for the outer membrane when the hook is no longer present.</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="motility.html"><a href="motility.html#campylobacter-jejuni-4"><i class="fa fa-check"></i><b>7.5</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="7.5.1" data-path="motility.html"><a href="motility.html#helicobacter-pylori-lophotrichous_bacteriamany-lophotrichous-species-like-the-hylemonella-gracilis-you-saw-in-chapter-3length-or-this-helicobacter-pylori-have-a-tuft-of-flagella-at-their-cell-pole.-in-some-species-though-the-tuft-is-located-elsewhere-for-example-a-clump-of-flagella-on-the-concave-side-of-banana-shaped-selenomonas-artemidis-pushes-the-cells-sideways-in-a-seesawing-swimming-pattern."><i class="fa fa-check"></i><b>7.5.1</b> Helicobacter pylori {#Lophotrichous_bacteria}Many lophotrichous species, like the <em>Hylemonella gracilis</em> you saw in Chapter 3–Length or this <em>Helicobacter pylori</em>, have a tuft of flagella at their cell pole. In some species, though, the tuft is located elsewhere; for example, a clump of flagella on the concave side of banana-shaped <em>Selenomonas artemidis</em> pushes the cells sideways in a seesawing swimming pattern.</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="motility.html"><a href="motility.html#pseudomonas-flexibilis"><i class="fa fa-check"></i><b>7.6</b> Pseudomonas flexibilis</a><ul>
<li class="chapter" data-level="7.6.1" data-path="motility.html"><a href="motility.html#proteus-mirabilis-proteus_mirabilis_flagellaproteus-mirabilis-adapt-their-motility-machinery-to-their-environment.-in-liquid-the-short-rod-shaped-cells-swim-with-the-help-of-a-handful-of-flagella-distributed-peritrichously-around-their-cell-body.-when-they-encounter-a-solid-surface-the-cells-elongate-and-build-many-more-flagella-as-you-can-see-on-this-cell.-instead-of-swimming-they-now-use-their-flagella-to-propel-themselves-in-groups-across-the-surface-a-motility-mode-known-as-swarming.-this-is-an-example-of-a-differentiated-lifecycle-which-will-come-up-again-in-chapter-8."><i class="fa fa-check"></i><b>7.6.1</b> Proteus mirabilis {#Proteus_mirabilis_flagella}<em>Proteus mirabilis</em> adapt their motility machinery to their environment. In liquid, the short rod-shaped cells swim with the help of a handful of flagella distributed peritrichously around their cell body. When they encounter a solid surface, the cells elongate and build many more flagella, as you can see on this cell. Instead of swimming, they now use their flagella to propel themselves in groups across the surface, a motility mode known as “swarming.” This is an example of a differentiated lifecycle, which will come up again in Chapter 8.</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="motility.html"><a href="motility.html#helicobacter-hepaticus"><i class="fa fa-check"></i><b>7.7</b> Helicobacter hepaticus</a></li>
<li class="chapter" data-level="7.8" data-path="motility.html"><a href="motility.html#borrelia-burgdorferi"><i class="fa fa-check"></i><b>7.8</b> Borrelia burgdorferi</a><ul>
<li class="chapter" data-level="7.8.1" data-path="motility.html"><a href="motility.html#treponema-primitia-treponema_primitiatreponema-primitia-like-this-are-commensal-residents-of-the-termite-gut-helping-break-down-cellulose.-in-addition-to-two-periplasmic-flagella-the-cells-have-arrays-of-bowl--and-hook-like-structures-on-their-surface-the-function-of-which-likely-related-to-motility-remains-mysterious."><i class="fa fa-check"></i><b>7.8.1</b> Treponema primitia {#Treponema_primitia}<em>Treponema primitia</em> like this are commensal residents of the termite gut, helping break down cellulose. In addition to two periplasmic flagella, the cells have arrays of bowl- and hook-like structures on their surface, the function of which, likely related to motility, remains mysterious.</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="motility.html"><a href="motility.html#methanoregula-formicica"><i class="fa fa-check"></i><b>7.9</b> Methanoregula formicica</a></li>
<li class="chapter" data-level="7.10" data-path="motility.html"><a href="motility.html#schematic-archaellum-structure"><i class="fa fa-check"></i><b>7.10</b> 8[Schematic: Archaellum structure]</a></li>
<li class="chapter" data-level="7.11" data-path="motility.html"><a href="motility.html#schematic-archaellar-motor-structure"><i class="fa fa-check"></i><b>7.11</b> 8[Schematic: Archaellar motor structure]</a></li>
<li class="chapter" data-level="7.12" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis"><i class="fa fa-check"></i><b>7.12</b> Thermococcus kodakaraensis</a><ul>
<li class="chapter" data-level="7.12.1" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis-cone_structurethe-archaellar-plate-does-not-come-to-a-point-at-the-tip-but-rather-is-a-conical-frustum-open-at-the-top-resembling-a-lampshade.-in-the-center-of-the-tip-is-a-small-ring-as-you-can-see-in-a-top-view-in-this-lysed-flattened-thermococcus-kodakaraensis-cell.-the-function-of-the-ring-remains-unknown-perhaps-it-helps-nucleate-the-rest-of-the-structure"><i class="fa fa-check"></i><b>7.12.1</b> Thermococcus kodakaraensis {#Cone_structure}The archaellar plate does not come to a point at the tip, but rather is a conical frustum (open at the top), resembling a lampshade. In the center of the tip is a small ring, as you can see in a top view in this lysed, flattened <em>Thermococcus kodakaraensis</em> cell. The function of the ring remains unknown; perhaps it helps nucleate the rest of the structure?</a></li>
<li class="chapter" data-level="7.12.2" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis-organizing_centeras-you-can-see-more-clearly-in-this-partially-lysed-flattened-thermococcus-kodakaraensis-cell-the-conical-plate-in-the-cytoplasm-is-attached-to-more-than-just-the-archaella.-it-is-also-associated-with-chemosensory-arrays-discussed-in-the-next-chapter-and-dna-as-you-can-see-from-the-ribosome-excluding-zone.-this-structure-may-therefore-perform-an-analogous-function-to-bacterial-organizing-proteins-such-as-popz-tethering-cellular-components-into-a-de-facto-pole-for-the-in-this-case-round-cell."><i class="fa fa-check"></i><b>7.12.2</b> Thermococcus kodakaraensis {#Organizing_center}As you can see more clearly in this partially-lysed, flattened <em>Thermococcus kodakaraensis</em> cell, the conical plate in the cytoplasm is attached to more than just the archaella. It is also associated with chemosensory arrays (discussed in the next chapter) and DNA, as you can see from the ribosome-excluding zone. This structure may therefore perform an analogous function to bacterial organizing proteins such as PopZ, tethering cellular components into a <em>de facto</em> pole for the (in this case round) cell.</a></li>
</ul></li>
<li class="chapter" data-level="7.13" data-path="motility.html"><a href="motility.html#myxococcus-xanthus-1"><i class="fa fa-check"></i><b>7.13</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="7.14" data-path="motility.html"><a href="motility.html#schematictype-iv-pilus-structure"><i class="fa fa-check"></i><b>7.14</b> 10[Schematic–Type IV pilus structure]</a></li>
<li class="chapter" data-level="7.15" data-path="motility.html"><a href="motility.html#flavobacterium-johnsoniae"><i class="fa fa-check"></i><b>7.15</b> Flavobacterium johnsoniae</a></li>
<li class="chapter" data-level="7.16" data-path="motility.html"><a href="motility.html#mycoplasma-pneumoniae"><i class="fa fa-check"></i><b>7.16</b> Mycoplasma pneumoniae</a></li>
<li class="chapter" data-level="7.17" data-path="motility.html"><a href="motility.html#schematicterminal-organelle-structure"><i class="fa fa-check"></i><b>7.17</b> 12[ Schematic–Terminal organelle structure]</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="navigation.html"><a href="navigation.html"><i class="fa fa-check"></i><b>8</b> Navigation</a><ul>
<li class="chapter" data-level="8.1" data-path="navigation.html"><a href="navigation.html#schematicchemoreceptor-structure"><i class="fa fa-check"></i><b>8.1</b> 1[Schematic–Chemoreceptor structure]</a></li>
<li class="chapter" data-level="8.2" data-path="navigation.html"><a href="navigation.html#styphimurium"><i class="fa fa-check"></i><b>8.2</b> Styphimurium</a></li>
<li class="chapter" data-level="8.3" data-path="navigation.html"><a href="navigation.html#schematic-chemosensory-array-architecture"><i class="fa fa-check"></i><b>8.3</b> 2[Schematic: Chemosensory array architecture]</a></li>
<li class="chapter" data-level="8.4" data-path="navigation.html"><a href="navigation.html#methanospirillum-hungatei-1"><i class="fa fa-check"></i><b>8.4</b> Methanospirillum hungatei</a></li>
<li class="chapter" data-level="8.5" data-path="navigation.html"><a href="navigation.html#schematicconservation-of-chemosensory-array-architecture"><i class="fa fa-check"></i><b>8.5</b> 3[Schematic–Conservation of chemosensory array architecture]</a></li>
<li class="chapter" data-level="8.6" data-path="navigation.html"><a href="navigation.html#vibrio-cholerae"><i class="fa fa-check"></i><b>8.6</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="8.7" data-path="navigation.html"><a href="navigation.html#aa-length-difference-of-as-little-as-2-nm-is-enough-to-separate-receptors-like-these-in-azospirillum-brasilense-that-sense-oxygen-28-nm-array-and-sources-of-energy-like-malate-30-nm-array.-both-receptors-send-signals-to-the-single-flagellar-motor-promoting-runs-or-direction-switches-to-guide-the-bacterium-through-a-combination-of-chemotaxis-and-aerotaxis-ordered-movement-in-an-oxygen-gradient-to-its-target-plant-roots.-a.-brasilense-fixes-nitrogen-boosting-the-growth-of-plants-it-colonizes."><i class="fa fa-check"></i><b>8.7</b> 4aA length difference of as little as 2 nm is enough to separate receptors, like these in <em>Azospirillum brasilense</em> that sense oxygen (28 nm array) and sources of energy like malate (30 nm array). Both receptors send signals to the single flagellar motor, promoting runs or direction switches to guide the bacterium through a combination of chemotaxis and aerotaxis (ordered movement in an oxygen gradient) to its target: plant roots. <em>A. brasilense</em> fixes nitrogen, boosting the growth of plants it colonizes.</a></li>
<li class="chapter" data-level="8.8" data-path="navigation.html"><a href="navigation.html#methanoregula-formicica-1"><i class="fa fa-check"></i><b>8.8</b> Methanoregula formicica</a></li>
<li class="chapter" data-level="8.9" data-path="navigation.html"><a href="navigation.html#schematiccytoplasmic-chemosensory-array-structure"><i class="fa fa-check"></i><b>8.9</b> 5[Schematic–Cytoplasmic chemosensory array structure]</a></li>
<li class="chapter" data-level="8.10" data-path="navigation.html"><a href="navigation.html#athis-vibrio-cholerae-cell-showcases-a-full-repertoire-of-chemosensory-systems-two-membrane-embedded-arrays-the-shorter-one-signaling-to-the-flagellar-motor-and-the-longer-one-signaling-to-a-still-unknown-target-and-a-cytoplasmic-array-also-with-unknown-function.-in-all-cases-the-basic-architecture-is-identical-hexagonally-arranged-trimers-of-chemoreceptor-dimers."><i class="fa fa-check"></i><b>8.10</b> 5aThis <em>Vibrio cholerae</em> cell showcases a full repertoire of chemosensory systems: two membrane-embedded arrays (the shorter one signaling to the flagellar motor and the longer one signaling to a still-unknown target) and a cytoplasmic array (also with unknown function). In all cases, the basic architecture is identical: hexagonally-arranged trimers of chemoreceptor dimers.</a></li>
<li class="chapter" data-level="8.11" data-path="navigation.html"><a href="navigation.html#magnetospirillum-magneticum"><i class="fa fa-check"></i><b>8.11</b> Magnetospirillum magneticum</a></li>
<li class="chapter" data-level="8.12" data-path="navigation.html"><a href="navigation.html#schematicmamk-structure"><i class="fa fa-check"></i><b>8.12</b> 6[Schematic–MamK structure]</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="lifecycle.html"><a href="lifecycle.html"><i class="fa fa-check"></i><b>9</b> Lifecycle</a><ul>
<li class="chapter" data-level="9.1" data-path="lifecycle.html"><a href="lifecycle.html#helicobacter-pylori"><i class="fa fa-check"></i><b>9.1</b> Helicobacter pylori</a></li>
<li class="chapter" data-level="9.2" data-path="lifecycle.html"><a href="lifecycle.html#thermococcus-kodakaraensis-1"><i class="fa fa-check"></i><b>9.2</b> Thermococcus kodakaraensis</a></li>
<li class="chapter" data-level="9.3" data-path="lifecycle.html"><a href="lifecycle.html#caulobacter-crescentus-7"><i class="fa fa-check"></i><b>9.3</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="9.3.1" data-path="lifecycle.html"><a href="lifecycle.html#prosthecobacter-vanneervenii-holdfastprosthecobacter-vanneervenii-is-a-close-relative-of-caulobacter-crescentus-and-attaches-to-surfaces-by-the-same-mechanism-a-sticky-ball-of-holdfast-at-the-tip-of-its-stalk.-here-you-see-four-cells-whose-stalks-have-attached-to-one-another.-unlike-its-cousin-c.-crescentus-however-p.-vanneervenii-is-non-motile-no-flagellum-and-does-not-have-a-dimorphic-life-cycle.-it-divides-symmetrically-to-produce-two-daughters-each-with-their-own-stalk."><i class="fa fa-check"></i><b>9.3.1</b> Prosthecobacter vanneervenii {#Holdfast}<em>Prosthecobacter vanneervenii</em> is a close relative of Caulobacter crescentus, and attaches to surfaces by the same mechanism: a sticky ball of holdfast at the tip of its stalk. Here you see four cells whose stalks have attached to one another. Unlike its cousin <em>C. crescentus</em>, however, <em>P. vanneervenii</em> is non-motile (no flagellum) and does not have a dimorphic life cycle. It divides symmetrically to produce two daughters, each with their own stalk.</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis"><i class="fa fa-check"></i><b>9.4</b> Bacillus subtilis</a><ul>
<li class="chapter" data-level="9.4.1" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-sporulation_septum_formationfrom-hundreds-of-sporulating-bacillus-subtilis-we-imaged-by-cryoet-this-is-the-only-cell-captured-in-the-process-of-forming-a-septum.-as-you-can-see-it-is-at-a-very-early-stage-in-the-process.-the-fact-that-we-do-not-see-more-intermediates-suggests-that-once-the-septum-starts-to-form-it-quickly-proceeds-to-completion."><i class="fa fa-check"></i><b>9.4.1</b> Bacillus subtilis {#Sporulation_septum_formation}From hundreds of sporulating <em>Bacillus subtilis</em> we imaged by cryoET, this is the only cell captured in the process of forming a septum. As you can see, it is at a very early stage in the process. The fact that we do not see more intermediates suggests that once the septum starts to form, it quickly proceeds to completion.</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-1"><i class="fa fa-check"></i><b>9.5</b> Bacillus subtilis</a><ul>
<li class="chapter" data-level="9.5.1" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-spore_engulfmentthe-process-of-spore-engulfment-by-the-mother-cell-membrane-culminates-in-closure-of-the-membranes-creating-an-outer-spore-membrane-and-an-inner-spore-membrane-on-either-side-of-the-sporulation-septum.-the-final-point-of-separation-is-usually-at-the-center-of-the-cell-pole-but-not-always-as-you-can-see-in-this-bacillus-subtilis-spore-being-engulfed-from-the-side-of-the-mother-cell.-the-layer-to-the-right-of-the-developing-spore-may-be-a-precursor-of-the-coat-morespore-coating."><i class="fa fa-check"></i><b>9.5.1</b> Bacillus subtilis {#Spore_engulfment}The process of spore engulfment by the mother cell membrane culminates in closure of the membranes, creating an outer spore membrane and an inner spore membrane on either side of the sporulation septum. The final point of separation is usually at the center of the cell pole, but not always, as you can see in this <em>Bacillus subtilis</em> spore being engulfed from the side of the mother cell. The layer to the right of the developing spore may be a precursor of the coat [More–Spore coating].</a></li>
<li class="chapter" data-level="9.5.2" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-spore_coatingin-this-later-stage-of-sporulation-you-can-see-that-the-bacillus-subtilis-spore-has-now-separated-completely-from-the-mother-cell-envelope.-it-has-also-moved-away-from-the-cell-pole-this-sometimes-occurs-even-as-the-envelopes-are-still-attached-morespore-engulfment.-the-layer-flanking-the-spore-on-the-right-may-be-the-assembling-spore-coat."><i class="fa fa-check"></i><b>9.5.2</b> Bacillus subtilis {#Spore_coating}In this later stage of sporulation, you can see that the <em>Bacillus subtilis</em> spore has now separated completely from the mother cell envelope. It has also moved away from the cell pole (this sometimes occurs even as the envelopes are still attached [More–Spore engulfment]). The layer flanking the spore on the right may be the assembling spore coat.</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-2"><i class="fa fa-check"></i><b>9.6</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="9.7" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-3"><i class="fa fa-check"></i><b>9.7</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="9.8" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-1"><i class="fa fa-check"></i><b>9.8</b> Acetonema longum</a></li>
<li class="chapter" data-level="9.9" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-2"><i class="fa fa-check"></i><b>9.9</b> Acetonema longum</a><ul>
<li class="chapter" data-level="9.9.1" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-complete_engulfmentthis-acetonema-longum-cell-has-completed-engulfment-creating-a-forespore.-acetonema-longum-forespores-and-mature-spores-typically-contain-several-granules-likely-storing-poly-phosphate.-these-may-help-power-outgrowth-when-the-spore-germinates.-spores-from-other-species-though-like-bacillus-subtilis-do-not-contain-obvious-storage-granules-so-they-may-serve-another-species-specific-role."><i class="fa fa-check"></i><b>9.9.1</b> Acetonema longum {#Complete_engulfment}This <em>Acetonema longum</em> cell has completed engulfment, creating a “forespore.” <em>Acetonema longum</em> forespores (and mature spores) typically contain several granules, likely storing poly-phosphate. These may help power outgrowth when the spore germinates. Spores from other species, though, like <em>Bacillus subtilis</em>, do not contain obvious storage granules so they may serve another, species-specific role.</a></li>
<li class="chapter" data-level="9.9.2" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-mature_sporethis-mature-acetonema-longum-spore-has-been-released-by-lysis-of-its-mother-cell.-note-its-extensive-armor-a-thick-cortex-between-the-membranes-and-a-multilayered-protein-coat."><i class="fa fa-check"></i><b>9.9.2</b> Acetonema longum {#Mature_spore}This mature <em>Acetonema longum</em> spore has been released by lysis of its mother cell. Note its extensive armor: a thick cortex between the membranes, and a multilayered protein coat.</a></li>
<li class="chapter" data-level="9.9.3" data-path="lifecycle.html"><a href="lifecycle.html#sporomusa-acidovorans-sporomusa_acidovorans_sporulationthis-diderm-sporomusa-acidovorans-cell-is-in-the-process-of-forming-an-ovoid-endospore.-note-also-what-may-be-an-early-stage-of-cortex-formation-at-the-end-of-the-spore-as-well-as-novel-structures-at-the-points-of-cell-wall-remodeling.-we-do-not-know-what-they-are-but-they-may-be-involved-in-engulfment."><i class="fa fa-check"></i><b>9.9.3</b> Sporomusa acidovorans {#Sporomusa_acidovorans_sporulation}This diderm <em>Sporomusa acidovorans</em> cell is in the process of forming an ovoid endospore. Note also what may be an early stage of cortex formation at the end of the spore, as well as novel structures at the points of cell wall remodeling. We do not know what they are, but they may be involved in engulfment.</a></li>
</ul></li>
<li class="chapter" data-level="9.10" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-3"><i class="fa fa-check"></i><b>9.10</b> Acetonema longum</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="interaction.html"><a href="interaction.html"><i class="fa fa-check"></i><b>10</b> Interaction</a><ul>
<li class="chapter" data-level="10.1" data-path="interaction.html"><a href="interaction.html#gluconacetobacter-hansenii"><i class="fa fa-check"></i><b>10.1</b> Gluconacetobacter hansenii</a></li>
<li class="chapter" data-level="10.2" data-path="interaction.html"><a href="interaction.html#legionella-pneumophila"><i class="fa fa-check"></i><b>10.2</b> Legionella pneumophila</a></li>
<li class="chapter" data-level="10.3" data-path="interaction.html"><a href="interaction.html#schematiclegionella-t2ss-structure"><i class="fa fa-check"></i><b>10.3</b> 3[Schematic–Legionella T2SS structure]</a></li>
<li class="chapter" data-level="10.4" data-path="interaction.html"><a href="interaction.html#schematiclegionella-t4ss-structure"><i class="fa fa-check"></i><b>10.4</b> 3[Schematic–Legionella T4SS structure]</a></li>
<li class="chapter" data-level="10.5" data-path="interaction.html"><a href="interaction.html#athis-helicobacter-pylori-cell-was-cultured-together-with-human-gastric-epithelial-cells-its-target-for-infection.-it-contains-inactive-type-iv-secretion-systems-as-well-as-several-mysterious-tubes.-the-tubes-are-formed-from-the-outer-membrane-but-we-do-not-know-what-scaffolds-them-they-have-a-constant-diameter-of-37-nm-or-what-makes-the-portals-along-their-length.-the-fact-that-we-sometimes-see-tubes-near-primed-t4sss-and-the-fact-that-the-tubes-are-missing-from-mutant-cells-lacking-t4ss-components-suggests-a-relationship-but-the-details-are-unclear.-much-remains-mysterious-about-this-structure-and-its-potential-role-in-infection."><i class="fa fa-check"></i><b>10.5</b> 3aThis <em>Helicobacter pylori</em> cell was cultured together with human gastric epithelial cells, its target for infection. It contains inactive type IV secretion systems, as well as several mysterious tubes. The tubes are formed from the outer membrane, but we do not know what scaffolds them (they have a constant diameter of 37 nm), or what makes the portals along their length. The fact that we sometimes see tubes near primed T4SSs, and the fact that the tubes are missing from mutant cells lacking T4SS components suggests a relationship, but the details are unclear. Much remains mysterious about this structure and its potential role in infection.</a></li>
<li class="chapter" data-level="10.6" data-path="interaction.html"><a href="interaction.html#pseudomonas-aeruginosa-δexsd"><i class="fa fa-check"></i><b>10.6</b> Pseudomonas aeruginosa ΔexsD</a></li>
<li class="chapter" data-level="10.7" data-path="interaction.html"><a href="interaction.html#a-emd-2667"><i class="fa fa-check"></i><b>10.7</b> 4a EMD-2667</a></li>
<li class="chapter" data-level="10.8" data-path="interaction.html"><a href="interaction.html#b"><i class="fa fa-check"></i><b>10.8</b> 4b</a></li>
<li class="chapter" data-level="10.9" data-path="interaction.html"><a href="interaction.html#escherichia-coli-cdia"><i class="fa fa-check"></i><b>10.9</b> Escherichia coli +CdiA</a></li>
<li class="chapter" data-level="10.10" data-path="interaction.html"><a href="interaction.html#schematiccdi-mechanism"><i class="fa fa-check"></i><b>10.10</b> 5[Schematic–CDI mechanism]</a></li>
<li class="chapter" data-level="10.11" data-path="interaction.html"><a href="interaction.html#myxococcus-xanthus-2"><i class="fa fa-check"></i><b>10.11</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="10.12" data-path="interaction.html"><a href="interaction.html#schematicprimed-t6ss-structure"><i class="fa fa-check"></i><b>10.12</b> 6[Schematic–Primed T6SS structure]</a></li>
<li class="chapter" data-level="10.13" data-path="interaction.html"><a href="interaction.html#vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.13</b> Vibrio cholerae</a><ul>
<li class="chapter" data-level="10.13.1" data-path="interaction.html"><a href="interaction.html#amoebophilus-asiaticus-t6ss_batterysome-cells-assemble-a-few-scattered-type-vi-secretion-systems.-other-species-like-this-amoebophilus-asiaticus-build-ordered-arrays-of-parallel-machines.-these-diderm-bacteria-live-symbiotically-inside-amoebae.-their-t6ss-batteries-help-them-colonize-their-host-perhaps-by-rupturing-the-membrane-of-the-host-phagosome-that-encloses-them-after-they-are-engulfed-releasing-them-into-the-host-cytoplasm.-the-number-of-t6sss-in-each-array-varies-but-can-reach-a-few-dozen."><i class="fa fa-check"></i><b>10.13.1</b> Amoebophilus asiaticus {#T6SS_battery}Some cells assemble a few, scattered type VI secretion systems. Other species, like this <em>Amoebophilus asiaticus</em> build ordered arrays of parallel machines. These diderm bacteria live symbiotically inside amoebae. Their T6SS batteries help them colonize their host, perhaps by rupturing the membrane of the host phagosome that encloses them after they are engulfed, releasing them into the host cytoplasm. The number of T6SSs in each array varies, but can reach a few dozen.</a></li>
<li class="chapter" data-level="10.13.2" data-path="interaction.html"><a href="interaction.html#pseudoalteromonas-luteoviolacea-δt6ssbacteriocin-macspseudoalteromonas-luteoviolacea-produce-massive-arrays-of-macs-contractile-machines-related-to-pyocins.-cells-lyse-to-release-the-arrays-like-the-one-you-see-here-which-can-contain-100-individual-macs-and-extend-more-than-1-µm-across.-the-arrays-are-highly-ordered-with-the-tail-fibers-of-the-macs-networked-into-a-hexagonal-array-that-may-help-synchronize-firing-schematicmac-array-structure.-mac-arrays-serve-an-important-function-for-larvae-of-the-marine-tubeworm-hydroides-elegans-and-likely-the-larvae-of-other-invertebrates-such-as-sea-urchins-and-corals-as-well.-they-signal-to-the-larva-that-it-is-the-right-place-to-settle-down-and-differentiate-into-a-sessile-surface-attached-adult.-the-function-of-mac-arrays-for-the-bacterium-is-less-clear.-it-may-be-that-the-invertebrates-benefit-the-lysed-cells-surviving-relatives-in-the-biofilm.-or-perhaps-the-macs-play-another-completely-unrelated-role."><i class="fa fa-check"></i><b>10.13.2</b> Pseudoalteromonas luteoviolacea ΔT6SS/bacteriocin {#MACs}<em>Pseudoalteromonas luteoviolacea</em> produce massive arrays of MACs, contractile machines related to pyocins. Cells lyse to release the arrays, like the one you see here, which can contain 100 individual MACs and extend more than 1 µm across. The arrays are highly ordered, with the tail fibers of the MACs networked into a hexagonal array that may help synchronize firing [Schematic–MAC array structure]. MAC arrays serve an important function for larvae of the marine tubeworm <em>Hydroides elegans</em>, and likely the larvae of other invertebrates such as sea urchins and corals as well. They signal to the larva that it is the right place to settle down and differentiate into a sessile (surface-attached) adult. The function of MAC arrays for the bacterium is less clear. It may be that the invertebrates benefit the lysed cell’s surviving relatives in the biofilm. Or perhaps the MACs play another, completely unrelated role.</a></li>
</ul></li>
<li class="chapter" data-level="10.14" data-path="interaction.html"><a href="interaction.html#schematicmac-array-structure"><i class="fa fa-check"></i><b>10.14</b> 7[Schematic–MAC array structure]</a></li>
<li class="chapter" data-level="10.15" data-path="interaction.html"><a href="interaction.html#porphyromonas-gingivalis"><i class="fa fa-check"></i><b>10.15</b> Porphyromonas gingivalis</a></li>
<li class="chapter" data-level="10.16" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-vibrio-cholerae"><i class="fa fa-check"></i><b>10.16</b> Bdellovibrio bacteriovorus + Vibrio cholerae</a></li>
<li class="chapter" data-level="10.17" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-escherichia-coli-minicell"><i class="fa fa-check"></i><b>10.17</b> Bdellovibrio bacteriovorus + Escherichia coli minicell</a><ul>
<li class="chapter" data-level="10.17.1" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-escherichia-coli-minicell-bdellovibrio_bacteriovorus_entrywe-still-do-not-know-exactly-how-bdellovibrio-bacteriovorus-enters-its-host-but-the-process-is-extremely-rapid.-in-order-to-catch-a-snapshot-we-mixed-b.-bacteriovorus-with-miniaturized-prey-see-details-on-the-main-page.-as-the-b.-bacteriovorus-cell-cannot-fit-inside-the-prey-it-remains-stuck-half-in-and-half-out.-note-the-tight-collar-of-host-outer-membrane-around-the-invader.-how-this-seal-is-maintained-throughout-invasion-remains-a-mystery."><i class="fa fa-check"></i><b>10.17.1</b> Bdellovibrio bacteriovorus + Escherichia coli minicell {#Bdellovibrio_bacteriovorus_entry}We still do not know exactly how <em>Bdellovibrio bacteriovorus</em> enters its host, but the process is extremely rapid. In order to catch a snapshot, we mixed <em>B. bacteriovorus</em> with miniaturized prey (see details on the main page). As the <em>B. bacteriovorus</em> cell cannot fit inside the prey, it remains stuck half-in and half-out. Note the tight collar of host outer membrane around the invader. How this seal is maintained throughout invasion remains a mystery.</a></li>
</ul></li>
<li class="chapter" data-level="10.18" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.18</b> Bdellovibrio bacteriovorus + Vibrio cholerae</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="viruses.html"><a href="viruses.html"><i class="fa fa-check"></i><b>11</b> Viruses</a><ul>
<li class="chapter" data-level="11.1" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-3"><i class="fa fa-check"></i><b>11.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="11.2" data-path="viruses.html"><a href="viruses.html#bacteroidetes-strain-jt5"><i class="fa fa-check"></i><b>11.2</b> Bacteroidetes strain JT5</a></li>
<li class="chapter" data-level="11.3" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-4"><i class="fa fa-check"></i><b>11.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="11.4" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-2"><i class="fa fa-check"></i><b>11.4</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="11.5" data-path="viruses.html"><a href="viruses.html#escherichia-coli-1"><i class="fa fa-check"></i><b>11.5</b> Escherichia coli</a></li>
<li class="chapter" data-level="11.6" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus-8"><i class="fa fa-check"></i><b>11.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="11.6.1" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus-φcb13_infectionhere-you-see-several-φcb13-phage-that-have-successfully-reached-their-target-receptors-at-the-pole-of-a-caulobacter-crescentus-and-are-in-the-process-of-infecting-the-cell-note-the-mix-of-filled-capsids-that-have-not-yet-released-their-genome-and-empty-capsids-that-have."><i class="fa fa-check"></i><b>11.6.1</b> Caulobacter crescentus {#φCb13_infection}Here you see several φCb13 phage that have successfully reached their target receptors at the pole of a <em>Caulobacter crescentus</em> and are in the process of infecting the cell (note the mix of filled capsids that have not yet released their genome and empty capsids that have).</a></li>
</ul></li>
<li class="chapter" data-level="11.7" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-3"><i class="fa fa-check"></i><b>11.7</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="11.8" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-1"><i class="fa fa-check"></i><b>11.8</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="11.9" data-path="viruses.html"><a href="viruses.html#schematicvap-structure"><i class="fa fa-check"></i><b>11.9</b> 8[Schematic–VAP structure]</a></li>
<li class="chapter" data-level="11.10" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-2"><i class="fa fa-check"></i><b>11.10</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="11.11" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-5"><i class="fa fa-check"></i><b>11.11</b> Shewanella oneidensis</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>12</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.library.caltech.edu" target="blank">Published by Caltech Library</a><li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cell Atlas</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="interaction" class="section level1">
<h1><span class="header-section-number">Chapter 10</span> Interaction</h1>
<p>“We do not have solitary beings. Every creature is, in some sense, connected to and dependent on the rest.” - Lewis Thomas <span class="citation">L. Thomas (<a href="#ref-thomas1974">1974</a>)</span></p>
<p>Biofilm ## Agrobacterium tumefaciens</p>
<p>So far we have largely ignored a major part of a cell’s environment: other organisms. Your cell is far from alone out there. It shares space and resources with cells of the same species (some its own mothers/sisters/daughters), other species, and even other domains. Think about strategies that could help your cell thrive in such a crowded world. For starters, why can’t we all just get along? Cooperation is common in Nature, often in the form of biofilms–communal groups of microorganisms. Biofilms offer advantages to their members, shielding cells on the interior from harsh conditions or antibiotics, and ensuring nutrients produced by cells’ metabolism or released by their death are readily available to others. Biofilms should be familiar to you; think of the scum on your teeth, or the nearest pond. In fact, the earliest physical evidence we have of life on earth is in the form of stromatolites (“layered rocks”)–meter-scale fossilized biofilms of cyanobacteria (you can see still-growing stromatolites in Shark Bay, Australia).</p>
<p>To attach to a biofilm, cells use strategies we have already discussed, including Tad pili and holdfast. <em>Agrobacterium tumefaciens</em> like this one use a holdfast-like polymer called _U_ni_p_olar _p_olysaccharide, or UPP. Some bacteria, including this one, also contribute to the superstructure of the biofilm by secreting cellulose fibers that help form the non-living matrix of the community.</p>
Biofilm (cont’d.)
<div class="figure"><span id="fig:unnamed-chunk-153"></span>
<div id="htmlwidget-8df804fe0b09c9531016" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-8df804fe0b09c9531016">{"x":{"doi":"10.22002/D1.1571","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.1: Agrobacterium tumefaciens Collected by: Elitza Tocheva <a href="https://doi.org/10.22002/D1.1571">10.22002/D1.1571</a>
</p>
</div>
<div id="gluconacetobacter-hansenii" class="section level2">
<h2><span class="header-section-number">10.1</span> Gluconacetobacter hansenii</h2>
<p>Individual cellulose fibers associate into bundles to form a 3D matrix for the cells, as you can see in this thin slice through a <em>Gluconacetobacter hansenii</em> biofilm. Not all biofilms contain cellulose, but all have an extracellular matrix, typically made of proteins secreted by the component cells. These proteins have properties that help them aggregate, like the unipolar polysaccharide you just saw. They may also have other properties useful to the community; for instance, <em>Bacillus subtilis</em> secretes a hydrophobic protein that forms a resistant coat around the biofilm. Matrices also contain lipids and extracellular DNA, and dead cells contribute material to the superstructure. Keep in mind that the lab-grown biofilm you see here consists of a single species, but in Nature a biofilm often contains multiple species.</p>
Type II and Type IV Secretion
<div class="figure"><span id="fig:unnamed-chunk-154"></span>
<div id="htmlwidget-721693ce9ae9f1334642" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-721693ce9ae9f1334642">{"x":{"doi":"10.22002/D1.1572","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.2: Gluconacetobacter hansenii Collected by: William Nicolas <a href="https://doi.org/10.22002/D1.1572">10.22002/D1.1572</a>
</p>
</div>
</div>
<div id="legionella-pneumophila" class="section level2">
<h2><span class="header-section-number">10.2</span> Legionella pneumophila</h2>
<p>How else might your cell deal with the crowd? Well, if you can’t join them, maybe you should try to beat (and eat) them. Bacteria have evolved an impressive arsenal of molecular weaponry. In fact, most of the antibiotics we use were invented by bacteria. In many cases, cells simply release these antibiotics to the environment, either directly or in membrane vesicles, which may travel further. For more specific targeting, though, cells employ a varied array of secretion systems. You already saw some of these nanomachines in Chapter 6: a type III secretion system assembles the flagellum and a type II secretion system makes (and unmakes) type IV pili. Other family members have evolved more militaristic functions.</p>
<p>Type II secretion systems (T2SSs) in <em>Legionella pneumophila</em> like this pump out effector proteins that enable a complex pathogenic lifecycle. <em>L. pneumophila</em> live in several environments: inside amoebae, outside cells (but sometimes with others in biofilms), and inside <em>our</em> cells, where they cause the severe pneumonia known as Legionnaires disease. In each environment, the T2SS pumps out proteins that facilitate growth there. For instance, once we inhale <em>L. pneumophila</em> with water droplets, they are internalized by macrophages. They then use their T2SSs to secrete proteins that prevent their pockets in the cell (called <em>Legionella</em>-containing vacuoles) from being degraded, and suppress our innate immune response. This gives the bacteria time to replicate and establish a persistent colony. The T2SSs also secrete a toxin protein that breaks down lung tissue.</p>
<p>Unlike the secretion systems in Chapter 6, the T2SS does not build an extracellular appendage. Instead, it is thought to work by pumping proteins through the envelope channel using a short piston-like “pseudo-pilus” [Schematic–Legionella T2SS structure], pushing the molecules into the extracellular space. (Keep in mind that for these intracellular pathogens, their extracellular space is the interior of the host cell.)</p>
<p><em>L. pneumophila</em> also use another, unrelated machine, a type IV secretion system (T4SS), to deliver more than 400(!) different kinds of effector proteins into host cells to promote infection. You can see multiple T4SSs primed for action in this cell [Schematic–Legionella T4SS structure]. We do not yet know what they look like during active secretion, so the mechanism remains unclear. In another human pathogen, <em>Helicobacter pylori</em>, inactive T4SSs are sometimes seen next to mysterious extracellular tubes, raising the intriguing possibility that the structures are somehow related [More–Helicobacter pylori tubes].</p>
<p>This <em>L. pneumophila</em> cell is not unusual; it is common for pathogenic bacteria to employ multiple types of secretion systems. The bristling arsenal of bacteria is a fearsome thing.</p>
<div class="figure"><span id="fig:unnamed-chunk-155"></span>
<div id="htmlwidget-50ad76546920eb4559c8" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-50ad76546920eb4559c8">{"x":{"doi":"10.22002/D1.1573","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.3: Legionella pneumophila Collected by: Debnath Ghosal <a href="https://doi.org/10.22002/D1.1573">10.22002/D1.1573</a>
</p>
</div>
</div>
<div id="schematiclegionella-t2ss-structure" class="section level2">
<h2><span class="header-section-number">10.3</span> 3[Schematic–Legionella T2SS structure]</h2>
<p>This average of type II secretion systems from many <em>Legionella pneumophila</em> cells shows the overall structure: a channel through the cell envelope, gated near the outer membrane. The pseudo-pilus can be seen in individual T2SSs, extending from the inner membrane to the bottom of the main channel. It gets washed out in the average, though, indicating that it is relatively flexible (i.e. not always in the same position). We still need to figure out exactly how the system works, including whether the pseudo-pilus does act like a piston, and how the gate opens.</p>
<div class="figure"><span id="fig:unnamed-chunk-156"></span>
<div id="htmlwidget-a1f36ed23715c375dea9" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-a1f36ed23715c375dea9">{"x":{"doi":"10.22002/D1.1579","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.4: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1579">10.22002/D1.1579</a>
</p>
</div>
</div>
<div id="schematiclegionella-t4ss-structure" class="section level2">
<h2><span class="header-section-number">10.4</span> 3[Schematic–Legionella T4SS structure]</h2>
<p>This average of type IV secretion systems from many <em>Legionella pneumophila</em> cells shows the overall structure. Identifying the location of individual components by analyzing mutants led to the model you see here. Each component has a different color, and you can see that many copies of each form rings. We do not yet know how secretion works, but one possibility is that effectors are channeled either straight through the channel from the cytoplasm and/or first into the periplasm and from there into the windowed chamber just inside the outer membrane before finally being escorted out of the cell.</p>
<div class="figure"><span id="fig:unnamed-chunk-157"></span>
<div id="htmlwidget-25c54256b048ab32ab62" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-25c54256b048ab32ab62">{"x":{"doi":"10.22002/D1.1579","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.5: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1579">10.22002/D1.1579</a>
</p>
</div>
</div>
<div id="athis-helicobacter-pylori-cell-was-cultured-together-with-human-gastric-epithelial-cells-its-target-for-infection.-it-contains-inactive-type-iv-secretion-systems-as-well-as-several-mysterious-tubes.-the-tubes-are-formed-from-the-outer-membrane-but-we-do-not-know-what-scaffolds-them-they-have-a-constant-diameter-of-37-nm-or-what-makes-the-portals-along-their-length.-the-fact-that-we-sometimes-see-tubes-near-primed-t4sss-and-the-fact-that-the-tubes-are-missing-from-mutant-cells-lacking-t4ss-components-suggests-a-relationship-but-the-details-are-unclear.-much-remains-mysterious-about-this-structure-and-its-potential-role-in-infection." class="section level2">
<h2><span class="header-section-number">10.5</span> 3aThis <em>Helicobacter pylori</em> cell was cultured together with human gastric epithelial cells, its target for infection. It contains inactive type IV secretion systems, as well as several mysterious tubes. The tubes are formed from the outer membrane, but we do not know what scaffolds them (they have a constant diameter of 37 nm), or what makes the portals along their length. The fact that we sometimes see tubes near primed T4SSs, and the fact that the tubes are missing from mutant cells lacking T4SS components suggests a relationship, but the details are unclear. Much remains mysterious about this structure and its potential role in infection.</h2>
Type III Secretion
<div class="figure"><span id="fig:unnamed-chunk-158"></span>
<div id="htmlwidget-6b914b389c095c2cebb4" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-6b914b389c095c2cebb4">{"x":{"doi":"10.22002/D1.1579","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.6: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1579">10.22002/D1.1579</a>
</p>
</div>
</div>
<div id="pseudomonas-aeruginosa-δexsd" class="section level2">
<h2><span class="header-section-number">10.6</span> Pseudomonas aeruginosa ΔexsD</h2>
<p>What if your cell wanted to go a step further and deliver toxins directly into its target? Perhaps it needs a needle. One of the most common, and best-studied, pathogenic secretion systems is the type III secretion system (T3SS) that assembles a needle-like toxin delivery system called the injectisome. T3SS injectisomes and flagellar motors are closely related, and the basal machines have similar structures [Schematic–Injectisome structure]. Rather than assembling a long helical filament for motility, though, injectisomes form a shorter needle, which you can see on this <em>Pseudomonas aeruginosa</em>. The needle can penetrate a eukaryotic cell, delivering effectors directly to its cytoplasm that block the immune response of the host, and lead to cell lysis.</p>
<p><em>It would be wasteful, or detrimental, for your cell to secrete toxins all the time. Pathogens typically have multi-stage lifecycles, often passing through an intermediate host and/or the environment. Pathogenic machinery like secretion systems are typically only produced when conditions signal that the cell has reached its target host (e.g. increased temperature or salt concentration). Until then, the genes encoding the protein components are turned off. This cell is missing a protein (ExsD) that turns off expression of these injectisome components. As a result, it makes injectisomes even outside the host, enabling us to image their structure.</em></p>
<p>T3SSs are closely related to type II secretion systems (T2SSs) and share a common element: the channel embedded in the outer membrane, formed by many copies of a protein called a secretin [Schematic–Secretin structure]. As you can see, though, the rest of the machine looks very different.</p>
<div class="figure"><span id="fig:unnamed-chunk-159"></span>
<div id="htmlwidget-53617736c974f24cdef6" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-53617736c974f24cdef6">{"x":{"doi":"10.22002/D1.1574","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.7: Pseudomonas aeruginosa Collected by: Steven Wang <a href="https://doi.org/10.22002/D1.1574">10.22002/D1.1574</a>
</p>
</div>
</div>
<div id="a-emd-2667" class="section level2">
<h2><span class="header-section-number">10.7</span> 4a EMD-2667</h2>
<p>[Schematic–Injectisome structure]</p>
<div class="figure"><span id="fig:unnamed-chunk-160"></span>
<div id="htmlwidget-dadfef7ec00654a1597d" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-dadfef7ec00654a1597d">{"x":{"doi":"10.22002/D1.1579","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.8: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1579">10.22002/D1.1579</a>
</p>
</div>
</div>
<div id="b" class="section level2">
<h2><span class="header-section-number">10.8</span> 4b</h2>
<p>[Schematic–Secretin structure]</p>
Type V Secretion
<div class="figure"><span id="fig:unnamed-chunk-161"></span>
<div id="htmlwidget-19ec722df9bb1b141911" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-19ec722df9bb1b141911">{"x":{"doi":"10.22002/D1.1579","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.9: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1579">10.22002/D1.1579</a>
</p>
</div>
</div>
<div id="escherichia-coli-cdia" class="section level2">
<h2><span class="header-section-number">10.9</span> Escherichia coli +CdiA</h2>
<p>Other secretion systems independently evolved similar mechanisms, though using very different structures. Some strains of <em>Escherichia coli</em> have an ability, called “_C_ontact-_D_ependent _I_nhibition” or CDI, to inhibit the growth of neighboring diderm bacterial cells. They do this by coopting a pore in the outer membrane of the target cell to introduce a toxin that halts its growth. To safeguard themselves, they make an antitoxin that binds to the toxin and renders it harmless. Such antidote system are common for microbe-produced toxins.</p>
<p>To deliver the toxin to the target, the cells use type V secretion systems (T5SSs) like the ones on this <em>Escherichia coli</em>. In contrast to the nanomachines you have been seeing, which contain dozens of unique components, this system is elegantly spare: a channel formed by several copies of one protein (CdiB) in the outer membrane secretes another protein, CdiA. Or rather, it secretes half of it. The first half of CdiA forms the stick you see here, with the second half keeping the toxin domain at the end of the protein sequestered in the periplasm. The tip of the stick contains the domain that recognizes and docks to the receptor pore on the target. When it bumps into a neighboring cell and locks on, the second half of CdiA is secreted, whipping the toxin like a tetherball into the other cell [Schematic–CDI mechanism].</p>
<p><em>This cell belongs to a strain that is normally incapable of CDI, but has been genetically engineered to produce the T5SS components. Once again, this lets us image the machines more easily; in this case, the CDI-practicing strain has a thicker layer of extracellular proteins, which obscure the needles in a visual haystack.</em></p>
<div class="figure"><span id="fig:unnamed-chunk-162"></span>
<div id="htmlwidget-e93f62b0922ec3b83ba6" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-e93f62b0922ec3b83ba6">{"x":{"doi":"10.22002/D1.1575","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.10: Escherichia coli Collected by: Poorna Subramanian <a href="https://doi.org/10.22002/D1.1575">10.22002/D1.1575</a>
</p>
</div>
</div>
<div id="schematiccdi-mechanism" class="section level2">
<h2><span class="header-section-number">10.10</span> 5[Schematic–CDI mechanism]</h2>
<p>This animation illustrates how we think the type V secretion system works.</p>
Type VI Secretion
<div class="figure"><span id="fig:unnamed-chunk-163"></span>
<div id="htmlwidget-c44346a78880b553919f" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-c44346a78880b553919f">{"x":{"doi":"10.22002/D1.1579","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.11: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1579">10.22002/D1.1579</a>
</p>
</div>
</div>
<div id="myxococcus-xanthus-2" class="section level2">
<h2><span class="header-section-number">10.11</span> Myxococcus xanthus</h2>
<p>How else could your cell deliver a toxin to a nearby competitor? As usual, Nature has provided another, even more entertaining, option: the poison dart gun. Many diderm bacteria, including <em>Myxococcus xanthus</em> like this, use type VI secretion systems (T6SSs) to launch effector proteins into neighboring cells. The effectors vary–from toxins that cause lysis to factors that promote biofilm formation. The target range of the T6SS is similarly broad, including monoderm and diderm bacteria and eukaryotic cells.</p>
<p>This primed T6SS consists of a hollow outer tube, with a narrower dart inside. The effector protein is loaded at the tip of the dart, where the machine is anchored in the cell envelope [Schematic–Primed T6SS structure]. In a few cells, including this one, we see additional filaments flanking the primed T6SS. They are lacking in other cells of the same species, and their identity and function remain unknown.</p>
<div class="figure"><span id="fig:unnamed-chunk-164"></span>
<div id="htmlwidget-f049f79decbb44daa5e4" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-f049f79decbb44daa5e4">{"x":{"doi":"10.22002/D1.1576","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.12: Myxococcus xanthus Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1576">10.22002/D1.1576</a>
</p>
</div>
</div>
<div id="schematicprimed-t6ss-structure" class="section level2">
<h2><span class="header-section-number">10.12</span> 6[Schematic–Primed T6SS structure]</h2>
Type VI Secretion (cont’d.)
<div class="figure"><span id="fig:unnamed-chunk-165"></span>
<div id="htmlwidget-993da35472e6fa09d01c" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-993da35472e6fa09d01c">{"x":{"doi":"10.22002/D1.1579","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.13: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1579">10.22002/D1.1579</a>
</p>
</div>
</div>
<div id="vibrio-cholerae-1" class="section level2">
<h2><span class="header-section-number">10.13</span> Vibrio cholerae</h2>
<p>When the T6SS fires, the helical outer sheath twists into a shorter, wider form, a conformational change that provides the energy to explosively propel the inner dart, cargo-first, out and into its target. In some species, like this <em>Vibrio cholerae</em>, the outer sheath will then be recycled, broken down into building blocks that can be used to rapidly assemble a new machine. Other species build again from scratch, translating new protein building blocks. Warfare is dynamic, with cells rapidly assembling and firing T6SSs. Cells have even been observed to engage in T6SS duels, with an attacked cell building and firing a T6SS back in the direction from which it was hit. Some species even assemble batteries of parallel T6SSs [More–T6SS battery].</p>
<p>These contractile weapons evolved from the “tail” structure viruses called phage use to inject their genetic material into bacteria and archaea (more on that in the next chapter). A related bacterial weapon, called a pyocin, is even closer to the phage tail, in that it is released to the environment in a primed state, waiting to bump into a target cell to fire. The target is recognized by filamentous proteins on the pyocin called “tail fibers,” the same identification mechanism used by phage. Other members of this evolutionary family are even stranger, like the_M_orphogenesis-_A_ssociated _C_ontractile structures (MACs) released by a marine bacterium which play a key role in the lifecycle of a (eukaryotic) worm [More–MACs] [Schematic-MAC array structure].</p>
<div class="figure"><span id="fig:unnamed-chunk-166"></span>
<div id="htmlwidget-56cc2f60e0582313d542" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-56cc2f60e0582313d542">{"x":{"doi":"10.22002/D1.1577","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.14: Vibrio cholerae Collected by: Martin Pilhofer <a href="https://doi.org/10.22002/D1.1577">10.22002/D1.1577</a>
</p>
</div>
<div id="amoebophilus-asiaticus-t6ss_batterysome-cells-assemble-a-few-scattered-type-vi-secretion-systems.-other-species-like-this-amoebophilus-asiaticus-build-ordered-arrays-of-parallel-machines.-these-diderm-bacteria-live-symbiotically-inside-amoebae.-their-t6ss-batteries-help-them-colonize-their-host-perhaps-by-rupturing-the-membrane-of-the-host-phagosome-that-encloses-them-after-they-are-engulfed-releasing-them-into-the-host-cytoplasm.-the-number-of-t6sss-in-each-array-varies-but-can-reach-a-few-dozen." class="section level3">
<h3><span class="header-section-number">10.13.1</span> Amoebophilus asiaticus {#T6SS_battery}Some cells assemble a few, scattered type VI secretion systems. Other species, like this <em>Amoebophilus asiaticus</em> build ordered arrays of parallel machines. These diderm bacteria live symbiotically inside amoebae. Their T6SS batteries help them colonize their host, perhaps by rupturing the membrane of the host phagosome that encloses them after they are engulfed, releasing them into the host cytoplasm. The number of T6SSs in each array varies, but can reach a few dozen.</h3>
<div class="figure"><span id="fig:unnamed-chunk-167"></span>
<div id="htmlwidget-66dde6b6305c6e4d1363" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-66dde6b6305c6e4d1363">{"x":{"doi":"10.22002/D1.1583","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.15: Amoebophilus asiaticus Collected by: Martin Pilhofer <a href="https://doi.org/10.22002/D1.1583">10.22002/D1.1583</a>
</p>
</div>
</div>
<div id="pseudoalteromonas-luteoviolacea-δt6ssbacteriocin-macspseudoalteromonas-luteoviolacea-produce-massive-arrays-of-macs-contractile-machines-related-to-pyocins.-cells-lyse-to-release-the-arrays-like-the-one-you-see-here-which-can-contain-100-individual-macs-and-extend-more-than-1-µm-across.-the-arrays-are-highly-ordered-with-the-tail-fibers-of-the-macs-networked-into-a-hexagonal-array-that-may-help-synchronize-firing-schematicmac-array-structure.-mac-arrays-serve-an-important-function-for-larvae-of-the-marine-tubeworm-hydroides-elegans-and-likely-the-larvae-of-other-invertebrates-such-as-sea-urchins-and-corals-as-well.-they-signal-to-the-larva-that-it-is-the-right-place-to-settle-down-and-differentiate-into-a-sessile-surface-attached-adult.-the-function-of-mac-arrays-for-the-bacterium-is-less-clear.-it-may-be-that-the-invertebrates-benefit-the-lysed-cells-surviving-relatives-in-the-biofilm.-or-perhaps-the-macs-play-another-completely-unrelated-role." class="section level3">
<h3><span class="header-section-number">10.13.2</span> Pseudoalteromonas luteoviolacea ΔT6SS/bacteriocin {#MACs}<em>Pseudoalteromonas luteoviolacea</em> produce massive arrays of MACs, contractile machines related to pyocins. Cells lyse to release the arrays, like the one you see here, which can contain 100 individual MACs and extend more than 1 µm across. The arrays are highly ordered, with the tail fibers of the MACs networked into a hexagonal array that may help synchronize firing [Schematic–MAC array structure]. MAC arrays serve an important function for larvae of the marine tubeworm <em>Hydroides elegans</em>, and likely the larvae of other invertebrates such as sea urchins and corals as well. They signal to the larva that it is the right place to settle down and differentiate into a sessile (surface-attached) adult. The function of MAC arrays for the bacterium is less clear. It may be that the invertebrates benefit the lysed cell’s surviving relatives in the biofilm. Or perhaps the MACs play another, completely unrelated role.</h3>
<p><em>Note that this MAC array was produced by a cell lacking genes encoding other contractile weaponry (type VI secretion systems and bacteriocins). This was done to make it easier to identify the MACs.</em></p>
<div class="figure"><span id="fig:unnamed-chunk-168"></span>
<div id="htmlwidget-a9bf195b9919c1aa9a2c" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-a9bf195b9919c1aa9a2c">{"x":{"doi":"10.22002/D1.1584","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.16: Pseudoalteromonas luteoviolacea Collected by: Martin Pilhofer <a href="https://doi.org/10.22002/D1.1584">10.22002/D1.1584</a>
</p>
</div>
</div>
</div>
<div id="schematicmac-array-structure" class="section level2">
<h2><span class="header-section-number">10.14</span> 7[Schematic–MAC array structure]</h2>
<p>This digital segmentation of the MAC array you just saw [More–MACs] shows the overall order of the structure. Outer sheaths are in blue, inner rods in green, tail fibers in orange, and networking fibers in white.</p>
Type IX Secretion
<div class="figure"><span id="fig:unnamed-chunk-169"></span>
<div id="htmlwidget-cd33050811c452637959" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-cd33050811c452637959">{"x":{"doi":"10.22002/D1.1579","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.17: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1579">10.22002/D1.1579</a>
</p>
</div>
</div>
<div id="porphyromonas-gingivalis" class="section level2">
<h2><span class="header-section-number">10.15</span> Porphyromonas gingivalis</h2>
<p>Instead of going to all the trouble of launching a weapon, why not just turn yourself into the weapon? <em>Porphyromonas gingivalis</em> like this one use yet another secretion system, the type IX secretion system (T9SS), to secrete proteins called gingipains to the cell surface, where they are anchored to the outer membrane. Gingipains are peptidases, enzymes that chop up other proteins. In their lifestyle as human oral pathogens, the cells use gingipains to block host immune system signals and digest host biomolecules for food, causing the tissue damage of periodontitis.</p>
<p>In Chapter 6 you saw another T9SS with a very different function, in the gliding motility of <em>Flavobacterium johnsoniae</em>. In that case, instead of gingipains, the machine secretes adhesive filaments that are also anchored to the cell surface.</p>
Predation
<div class="figure"><span id="fig:unnamed-chunk-170"></span>
<div id="htmlwidget-92789ec11abeb8ef592a" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-92789ec11abeb8ef592a">{"x":{"doi":"10.22002/D1.1578","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.18: Porphyromonas gingivalis Collected by: Poorna Subramanian <a href="https://doi.org/10.22002/D1.1578">10.22002/D1.1578</a>
</p>
</div>
</div>
<div id="bdellovibrio-bacteriovorus-vibrio-cholerae" class="section level2">
<h2><span class="header-section-number">10.16</span> Bdellovibrio bacteriovorus + Vibrio cholerae</h2>
<p>Some bacteria take the strategy of eating the neighbors to a whole new level. Remember the pack-based hunting of <em>Myxococcus xanthus</em> in Chapter 7 (chasing their prey with type IV pilus (T4P)-based twitching motility)? There are also solitary hunters, like <em>Bdellovibrio bacteriovorus</em>, which has evolved an elaborate vampiric lifestyle centered around its prey. You have already seen examples of <em>B. bacteriovorus</em> cells on the prowl, in what is called “attack phase”. Their bodies are ideally shaped and sized for rapid flagellum-propelled swimming and all superfluous functions like growth and division are halted (remember the highly condensed nucleoid you saw at the end of Chapter 2). When a cell finds a diderm bacterium–<em>Vibrio cholerae</em> in this case–it uses its type IV pili to lock on and pull itself closer, as you can see here.</p>
Predation (cont’d.)
<div class="figure"><span id="fig:unnamed-chunk-171"></span>
<div id="htmlwidget-f337e126f003be1a9802" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-f337e126f003be1a9802">{"x":{"doi":"10.22002/D1.1579","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.19: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1579">10.22002/D1.1579</a>
</p>
</div>
</div>
<div id="bdellovibrio-bacteriovorus-escherichia-coli-minicell" class="section level2">
<h2><span class="header-section-number">10.17</span> Bdellovibrio bacteriovorus + Escherichia coli minicell</h2>
<p>Once a <em>Bdellovibrio bacteriovorus</em> like this one has latched onto its unfortunate host, it opens a portal in the host’s outer membrane, secreting a cocktail of enzymes that alter the host cell wall, causing the cell to round up and enlarge the periplasmic space. The <em>B. bacteriovorus</em> quickly slides into the host periplasm through the portal [More–Bdellovibrio bacteriovorus entry] and seals the door behind it. In the process of preparing for entry, the cell dismantles equipment that is no longer necessary, including its chemosensory array and flagellum. As you can see here, the sheathed flagellum dislocates from the motor and is pulled into the periplasm, where the filament is degraded.</p>
<p><em>Note that the prey here–the small, round cell–comes from a mutant strain of </em>Escherichia coli* that has an altered form of the shape-regulating protein MreB, which results in thinner cells. It is also lacking the Min system that positions the division plane at midcell. As a result, division occurs randomly along the length of the cell. The cells that divide near the pole produce “minicells” like this one. Minicells do not contain a genome and are not viable cells, but they can be useful for experiments. In this case, the reduced thickness of the sample improves the quality of cryoET imaging. It also lets us capture intermediates in the <em>B. bacteriovorus</em> entry process [More–Bdellovibrio bacteriovorus entry].*</p>
<div class="figure"><span id="fig:unnamed-chunk-172"></span>
<div id="htmlwidget-cb86ab83dca43cbe6333" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-cb86ab83dca43cbe6333">{"x":{"doi":"10.22002/D1.1580","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.20: Bdellovibrio bacteriovorus / Escherichia coli Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1580">10.22002/D1.1580</a>
</p>
</div>
<div id="bdellovibrio-bacteriovorus-escherichia-coli-minicell-bdellovibrio_bacteriovorus_entrywe-still-do-not-know-exactly-how-bdellovibrio-bacteriovorus-enters-its-host-but-the-process-is-extremely-rapid.-in-order-to-catch-a-snapshot-we-mixed-b.-bacteriovorus-with-miniaturized-prey-see-details-on-the-main-page.-as-the-b.-bacteriovorus-cell-cannot-fit-inside-the-prey-it-remains-stuck-half-in-and-half-out.-note-the-tight-collar-of-host-outer-membrane-around-the-invader.-how-this-seal-is-maintained-throughout-invasion-remains-a-mystery." class="section level3">
<h3><span class="header-section-number">10.17.1</span> Bdellovibrio bacteriovorus + Escherichia coli minicell {#Bdellovibrio_bacteriovorus_entry}We still do not know exactly how <em>Bdellovibrio bacteriovorus</em> enters its host, but the process is extremely rapid. In order to catch a snapshot, we mixed <em>B. bacteriovorus</em> with miniaturized prey (see details on the main page). As the <em>B. bacteriovorus</em> cell cannot fit inside the prey, it remains stuck half-in and half-out. Note the tight collar of host outer membrane around the invader. How this seal is maintained throughout invasion remains a mystery.</h3>
Predation (cont’d.)
<div class="figure"><span id="fig:unnamed-chunk-173"></span>
<div id="htmlwidget-733b3bf9c3098dba9a80" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-733b3bf9c3098dba9a80">{"x":{"doi":"10.22002/D1.1585","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.21: Bdellovibrio bacteriovorus / Escherichia coli Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1585">10.22002/D1.1585</a>
</p>
</div>
</div>
</div>
<div id="bdellovibrio-bacteriovorus-vibrio-cholerae-1" class="section level2">
<h2><span class="header-section-number">10.18</span> Bdellovibrio bacteriovorus + Vibrio cholerae</h2>
<p>Once inside its host, <em>Bdellovibrio bacteriovorus</em> has the leisure to grow and replicate. From its position in the periplasm, the <em>B. bacteriovorus</em> cell proceeds to digest the contents of the host cytoplasm, which shrinks down to a dense ball of (presumably undigestible) material. This meal provides the fuel necessary for the <em>B. bacteriovorus</em> to grow to several times its original length and undergo a synchronous division to produce several (usually between 2 and 7) daughters. The number of progeny depends on the size of the host cell: two in the case of this relatively small <em>Vibrio cholerae</em>. The progeny, reset to the attack phase (note the condensed nucleoids), finally lyse what is left of the host to escape and swim off in search of their next victims [Schematic–Bdellovibrio bacteriovorus lifecycle].</p>
<p>References</p>
<p>Further Reading</p>
<span class="citation">(<span class="citeproc-not-found" data-reference-id="christie2019"><strong>???</strong></span>)</span> <span class="citation">Flemming et al. (<a href="#ref-flemming2016">2016</a>)</span> <span class="citation">(<span class="citeproc-not-found" data-reference-id="patz2019"><strong>???</strong></span>)</span> <span class="citation">Sockett (<a href="#ref-sockett2009">2009</a>)</span>
<div class="figure"><span id="fig:unnamed-chunk-174"></span>
<div id="htmlwidget-8c67f722713a9d9bceb1" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-8c67f722713a9d9bceb1">{"x":{"doi":"10.22002/D1.1581","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.22: Bdellovibrio bacteriovorus / Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1581">10.22002/D1.1581</a>
</p>
</div>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-flemming2016">
<p>Flemming, H. C., J. Wingender, U. Szewzyk, P. Steinberg, S. A. Rice, and S. Kjelleberg. 2016. “Biofilms: An Emergent Form of Bacterial Life.” <em>Nat Rev Microbiol</em> 14 (9): 563–75. doi:<a href="https://doi.org/10.1038/nrmicro.2016.94">10.1038/nrmicro.2016.94</a>.</p>
</div>
<div id="ref-sockett2009">
<p>Sockett, R. E. 2009. “Predatory Lifestyle of Bdellovibrio Bacteriovorus.” <em>Annu Rev Microbiol</em> 63: 523–39. doi:<a href="https://doi.org/10.1146/annurev.micro.091208.073346">10.1146/annurev.micro.091208.073346</a>.</p>
</div>
<div id="ref-thomas1974">
<p>Thomas, L. 1974. <em>The Lives of a Cell</em>. New York: The Viking Press.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="lifecycle.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="viruses.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": null,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/caltechlibrary/cell-atlas/edit/master/09_Interaction.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["cell-atlas.pdf", "cell-atlas.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
