<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 6 Motility | Atlas of Bacterial and Archaeal Cell Structure</title>
  <meta name="description" content="A visual tour of the microbial cell." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 6 Motility | Atlas of Bacterial and Archaeal Cell Structure" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="img/AtlasLogo.png" />
  <meta property="og:description" content="A visual tour of the microbial cell." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 6 Motility | Atlas of Bacterial and Archaeal Cell Structure" />
  
  <meta name="twitter:description" content="A visual tour of the microbial cell." />
  <meta name="twitter:image" content="img/AtlasLogo.png" />

<meta name="author" content="Catherine M. Oikonomou and Grant J. Jensen" />


<meta name="date" content="2020-08-05" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="division.html"/>
<link rel="next" href="navigation.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/CL-0.1.2/CL.js"></script>
<script src="libs/doivideo-binding-0.0.0.9000/doivideo.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Cell Atlas</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>1</b> Methods</a><ul>
<li class="chapter" data-level="1.1" data-path="methods.html"><a href="methods.html#light-microscopy"><i class="fa fa-check"></i><b>1.1</b> Light Microscopy</a><ul>
<li class="chapter" data-level="1.1.1" data-path="methods.html"><a href="methods.html#Structural_biology_toolkit"><i class="fa fa-check"></i><b>1.1.1</b> Structural biology toolkit</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="methods.html"><a href="methods.html#fluorescence-light-microscopy"><i class="fa fa-check"></i><b>1.2</b> Fluorescence Light Microscopy</a></li>
<li class="chapter" data-level="1.3" data-path="methods.html"><a href="methods.html#scanning-electron-microscopy"><i class="fa fa-check"></i><b>1.3</b> Scanning Electron Microscopy</a><ul>
<li class="chapter" data-level="1.3.1" data-path="methods.html"><a href="methods.html#Electron_microscopy_modes"><i class="fa fa-check"></i><b>1.3.1</b> Electron microscopy modes</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="methods.html"><a href="methods.html#transmission-electron-microscopy"><i class="fa fa-check"></i><b>1.4</b> Transmission Electron Microscopy</a></li>
<li class="chapter" data-level="1.5" data-path="methods.html"><a href="methods.html#cryogenic-electron-microscopy"><i class="fa fa-check"></i><b>1.5</b> Cryogenic Electron Microscopy</a></li>
<li class="chapter" data-level="1.6" data-path="methods.html"><a href="methods.html#electron-cryotomography"><i class="fa fa-check"></i><b>1.6</b> Electron Cryotomography</a><ul>
<li class="chapter" data-level="1.6.1" data-path="methods.html"><a href="methods.html#Missing_wedge"><i class="fa fa-check"></i><b>1.6.1</b> Missing wedge</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="methods.html"><a href="methods.html#sub-tomogram-averaging"><i class="fa fa-check"></i><b>1.7</b> Sub-Tomogram Averaging</a></li>
<li class="chapter" data-level="1.8" data-path="methods.html"><a href="methods.html#single-particle-reconstruction"><i class="fa fa-check"></i><b>1.8</b> Single Particle Reconstruction</a></li>
<li class="chapter" data-level="1.9" data-path="methods.html"><a href="methods.html#x-ray-crystallography"><i class="fa fa-check"></i><b>1.9</b> X-Ray Crystallography</a><ul>
<li class="chapter" data-level="1.9.1" data-path="methods.html"><a href="methods.html#Structural_biology_timeline"><i class="fa fa-check"></i><b>1.9.1</b> Structural biology timeline</a></li>
</ul></li>
<li class="chapter" data-level="1.10" data-path="methods.html"><a href="methods.html#putting-it-all-together"><i class="fa fa-check"></i><b>1.10</b> Putting It All Together</a></li>
<li class="chapter" data-level="1.11" data-path="methods.html"><a href="methods.html#moredifference-mapping"><i class="fa fa-check"></i><b>1.11</b> More–Difference mapping</a></li>
<li class="chapter" data-level="1.12" data-path="methods.html"><a href="methods.html#further-reading"><i class="fa fa-check"></i><b>1.12</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="envelope.html"><a href="envelope.html"><i class="fa fa-check"></i><b>2</b> Envelope</a><ul>
<li class="chapter" data-level="2.1" data-path="envelope.html"><a href="envelope.html#membrane"><i class="fa fa-check"></i><b>2.1</b> Membrane</a><ul>
<li class="chapter" data-level="2.1.1" data-path="envelope.html"><a href="envelope.html#Lipid_bilayer"><i class="fa fa-check"></i><b>2.1.1</b> Lipid bilayer</a></li>
<li class="chapter" data-level="2.1.2" data-path="envelope.html"><a href="envelope.html#PDB:_5T4O_ATP_synthase_and_energy_production"><i class="fa fa-check"></i><b>2.1.2</b> PDB: 5T4O ATP synthase and energy production</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="envelope.html"><a href="envelope.html#cell-wall"><i class="fa fa-check"></i><b>2.2</b> Cell Wall</a></li>
<li class="chapter" data-level="2.3" data-path="envelope.html"><a href="envelope.html#morepeptidoglycan-architecture"><i class="fa fa-check"></i><b>2.3</b> More–Peptidoglycan architecture</a></li>
<li class="chapter" data-level="2.4" data-path="envelope.html"><a href="envelope.html#moremethanobacterium-formicicum"><i class="fa fa-check"></i><b>2.4</b> More–Methanobacterium formicicum</a></li>
<li class="chapter" data-level="2.5" data-path="envelope.html"><a href="envelope.html#outer-membrane"><i class="fa fa-check"></i><b>2.5</b> Outer Membrane</a><ul>
<li class="chapter" data-level="2.5.1" data-path="envelope.html"><a href="envelope.html#pdb-1eq7-brauns-lipoprotein-pdb1eq7brauns_lipoprotein"><i class="fa fa-check"></i><b>2.5.1</b> PDB: 1EQ7 –Braun’s lipoprotein {#PDB:<em>1EQ7</em>–Brauns_lipoprotein}</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="envelope.html"><a href="envelope.html#morediderm-sacculus-architecture"><i class="fa fa-check"></i><b>2.6</b> More–Diderm sacculus architecture</a><ul>
<li class="chapter" data-level="2.6.1" data-path="envelope.html"><a href="envelope.html#Sacculus_remodeling"><i class="fa fa-check"></i><b>2.6.1</b> Sacculus remodeling</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="envelope.html"><a href="envelope.html#vesicles"><i class="fa fa-check"></i><b>2.7</b> Vesicles</a></li>
<li class="chapter" data-level="2.8" data-path="envelope.html"><a href="envelope.html#morepearled-vesicles"><i class="fa fa-check"></i><b>2.8</b> More–Pearled vesicles</a></li>
<li class="chapter" data-level="2.9" data-path="envelope.html"><a href="envelope.html#moretubular-vesicles"><i class="fa fa-check"></i><b>2.9</b> More–Tubular vesicles</a></li>
<li class="chapter" data-level="2.10" data-path="envelope.html"><a href="envelope.html#morecytoplasmic-vesicles"><i class="fa fa-check"></i><b>2.10</b> More–Cytoplasmic vesicles</a></li>
<li class="chapter" data-level="2.11" data-path="envelope.html"><a href="envelope.html#more-cytoplasmic-vesicle-variety"><i class="fa fa-check"></i><b>2.11</b> More: Cytoplasmic vesicle variety</a></li>
<li class="chapter" data-level="2.12" data-path="envelope.html"><a href="envelope.html#morearchaeal-vesicles"><i class="fa fa-check"></i><b>2.12</b> More–Archaeal vesicles</a></li>
<li class="chapter" data-level="2.13" data-path="envelope.html"><a href="envelope.html#variation"><i class="fa fa-check"></i><b>2.13</b> Variation</a></li>
<li class="chapter" data-level="2.14" data-path="envelope.html"><a href="envelope.html#surface-layer"><i class="fa fa-check"></i><b>2.14</b> Surface Layer</a><ul>
<li class="chapter" data-level="2.14.1" data-path="envelope.html"><a href="envelope.html#PDB:_5N97_Surface_layer_architecture"><i class="fa fa-check"></i><b>2.14.1</b> PDB: 5N97 Surface layer architecture</a></li>
</ul></li>
<li class="chapter" data-level="2.15" data-path="envelope.html"><a href="envelope.html#morenanopods"><i class="fa fa-check"></i><b>2.15</b> More–Nanopods</a></li>
<li class="chapter" data-level="2.16" data-path="envelope.html"><a href="envelope.html#surface-layer-variety"><i class="fa fa-check"></i><b>2.16</b> Surface Layer Variety</a></li>
<li class="chapter" data-level="2.17" data-path="envelope.html"><a href="envelope.html#morem.-alcaliphilum"><i class="fa fa-check"></i><b>2.17</b> More–M. alcaliphilum</a></li>
<li class="chapter" data-level="2.18" data-path="envelope.html"><a href="envelope.html#moreclostridium-thermocellum"><i class="fa fa-check"></i><b>2.18</b> More–Clostridium thermocellum</a></li>
<li class="chapter" data-level="2.19" data-path="envelope.html"><a href="envelope.html#moren.-maritimus"><i class="fa fa-check"></i><b>2.19</b> More–N. maritimus</a></li>
<li class="chapter" data-level="2.20" data-path="envelope.html"><a href="envelope.html#moremethanoregula-formicica"><i class="fa fa-check"></i><b>2.20</b> More–Methanoregula formicica</a></li>
<li class="chapter" data-level="2.21" data-path="envelope.html"><a href="envelope.html#sheath"><i class="fa fa-check"></i><b>2.21</b> Sheath</a></li>
<li class="chapter" data-level="2.22" data-path="envelope.html"><a href="envelope.html#dna"><i class="fa fa-check"></i><b>2.22</b> DNA</a></li>
<li class="chapter" data-level="2.23" data-path="envelope.html"><a href="envelope.html#nucleoid"><i class="fa fa-check"></i><b>2.23</b> Nucleoid</a></li>
<li class="chapter" data-level="2.24" data-path="envelope.html"><a href="envelope.html#further-reading-1"><i class="fa fa-check"></i><b>2.24</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="shape.html"><a href="shape.html"><i class="fa fa-check"></i><b>3</b> Shape</a><ul>
<li class="chapter" data-level="3.1" data-path="shape.html"><a href="shape.html#coccoid"><i class="fa fa-check"></i><b>3.1</b> Coccoid</a></li>
<li class="chapter" data-level="3.2" data-path="shape.html"><a href="shape.html#rod"><i class="fa fa-check"></i><b>3.2</b> Rod</a></li>
<li class="chapter" data-level="3.3" data-path="shape.html"><a href="shape.html#morerod-variety"><i class="fa fa-check"></i><b>3.3</b> More–Rod variety</a></li>
<li class="chapter" data-level="3.4" data-path="shape.html"><a href="shape.html#morearchaeal-rods"><i class="fa fa-check"></i><b>3.4</b> More–Archaeal rods</a></li>
<li class="chapter" data-level="3.5" data-path="shape.html"><a href="shape.html#length"><i class="fa fa-check"></i><b>3.5</b> Length</a></li>
<li class="chapter" data-level="3.6" data-path="shape.html"><a href="shape.html#vibrioid"><i class="fa fa-check"></i><b>3.6</b> Vibrioid</a><ul>
<li class="chapter" data-level="3.6.1" data-path="shape.html"><a href="shape.html#PDB:_5U3C_CTP_synthase"><i class="fa fa-check"></i><b>3.6.1</b> PDB: 5U3C CTP synthase</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="shape.html"><a href="shape.html#helical"><i class="fa fa-check"></i><b>3.7</b> Helical</a></li>
<li class="chapter" data-level="3.8" data-path="shape.html"><a href="shape.html#moreborrelia-burgdorferi"><i class="fa fa-check"></i><b>3.8</b> More–Borrelia burgdorferi</a></li>
<li class="chapter" data-level="3.9" data-path="shape.html"><a href="shape.html#prosthecate"><i class="fa fa-check"></i><b>3.9</b> Prosthecate</a><ul>
<li class="chapter" data-level="3.9.1" data-path="shape.html"><a href="shape.html#PDB:_6RIB_Bactofilin"><i class="fa fa-check"></i><b>3.9.1</b> PDB: 6RIB Bactofilin</a></li>
</ul></li>
<li class="chapter" data-level="3.10" data-path="shape.html"><a href="shape.html#morebacterial-microtubules"><i class="fa fa-check"></i><b>3.10</b> More–Bacterial microtubules</a></li>
<li class="chapter" data-level="3.11" data-path="shape.html"><a href="shape.html#square"><i class="fa fa-check"></i><b>3.11</b> Square</a></li>
<li class="chapter" data-level="3.12" data-path="shape.html"><a href="shape.html#moregas-vesicles"><i class="fa fa-check"></i><b>3.12</b> More–Gas vesicles</a></li>
<li class="chapter" data-level="3.13" data-path="shape.html"><a href="shape.html#further-reading-2"><i class="fa fa-check"></i><b>3.13</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="growth.html"><a href="growth.html"><i class="fa fa-check"></i><b>4</b> Growth</a><ul>
<li class="chapter" data-level="4.1" data-path="growth.html"><a href="growth.html#diffusion"><i class="fa fa-check"></i><b>4.1</b> Diffusion</a></li>
<li class="chapter" data-level="4.2" data-path="growth.html"><a href="growth.html#anaerobic-respiration"><i class="fa fa-check"></i><b>4.2</b> Anaerobic Respiration</a></li>
<li class="chapter" data-level="4.3" data-path="growth.html"><a href="growth.html#photosynthesis"><i class="fa fa-check"></i><b>4.3</b> Photosynthesis</a><ul>
<li class="chapter" data-level="4.3.1" data-path="growth.html"><a href="growth.html#pdb-6kgx-photocomplexes-pdb6kgxphotocomplexes"><i class="fa fa-check"></i><b>4.3.1</b> PDB: 6KGX –Photocomplexes {#PDB:<em>6KGX</em>–Photocomplexes}</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="growth.html"><a href="growth.html#moreicm-variety"><i class="fa fa-check"></i><b>4.4</b> More–ICM variety</a></li>
<li class="chapter" data-level="4.5" data-path="growth.html"><a href="growth.html#photosynthesis-contd."><i class="fa fa-check"></i><b>4.5</b> Photosynthesis (cont’d.)</a></li>
<li class="chapter" data-level="4.6" data-path="growth.html"><a href="growth.html#enzyme-regulation"><i class="fa fa-check"></i><b>4.6</b> Enzyme Regulation</a><ul>
<li class="chapter" data-level="4.6.1" data-path="growth.html"><a href="growth.html#pdb-6tqhadhe-spirosome-structure-pdb_6tqhadhe_spirosome_structure"><i class="fa fa-check"></i><b>4.6.1</b> PDB: 6TQH–AdhE spirosome structure {#PDB:_6TQH–AdhE_spirosome_structure} <img src="img/schematics/4_5_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="growth.html"><a href="growth.html#microcompartments"><i class="fa fa-check"></i><b>4.7</b> Microcompartments</a><ul>
<li class="chapter" data-level="4.7.1" data-path="growth.html"><a href="growth.html#pdb-3dktmicrocompartment-shells-pdb_3dktmicrocompartment_shells"><i class="fa fa-check"></i><b>4.7.1</b> PDB: 3DKT–Microcompartment shells {#PDB:_3DKT–Microcompartment_shells} <img src="img/schematics/4_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="growth.html"><a href="growth.html#carboxysomes"><i class="fa fa-check"></i><b>4.8</b> Carboxysomes</a></li>
<li class="chapter" data-level="4.9" data-path="growth.html"><a href="growth.html#morecarboxysome-variety"><i class="fa fa-check"></i><b>4.9</b> More–Carboxysome variety</a></li>
<li class="chapter" data-level="4.10" data-path="growth.html"><a href="growth.html#morelong-carboxysomes"><i class="fa fa-check"></i><b>4.10</b> More–Long carboxysomes</a></li>
<li class="chapter" data-level="4.11" data-path="growth.html"><a href="growth.html#storage"><i class="fa fa-check"></i><b>4.11</b> Storage</a></li>
<li class="chapter" data-level="4.12" data-path="growth.html"><a href="growth.html#morestorage-granule-growth"><i class="fa fa-check"></i><b>4.12</b> More–Storage granule growth</a></li>
<li class="chapter" data-level="4.13" data-path="growth.html"><a href="growth.html#storage-contd."><i class="fa fa-check"></i><b>4.13</b> Storage (cont’d.)</a></li>
<li class="chapter" data-level="4.14" data-path="growth.html"><a href="growth.html#moregranule-containing-carboxysomes"><i class="fa fa-check"></i><b>4.14</b> More–Granule-containing carboxysomes</a></li>
<li class="chapter" data-level="4.15" data-path="growth.html"><a href="growth.html#morecarboxysomes-granule-connections"><i class="fa fa-check"></i><b>4.15</b> More–Carboxysomes-granule connections</a></li>
<li class="chapter" data-level="4.16" data-path="growth.html"><a href="growth.html#archaeal-storage-granules"><i class="fa fa-check"></i><b>4.16</b> Archaeal Storage Granules</a></li>
<li class="chapter" data-level="4.17" data-path="growth.html"><a href="growth.html#morehaloferax-gibbonsii-granules"><i class="fa fa-check"></i><b>4.17</b> More–Haloferax gibbonsii granules</a></li>
<li class="chapter" data-level="4.18" data-path="growth.html"><a href="growth.html#morehalohasta-litchfieldiae-granules"><i class="fa fa-check"></i><b>4.18</b> More–Halohasta litchfieldiae granules</a></li>
<li class="chapter" data-level="4.19" data-path="growth.html"><a href="growth.html#further-reading-3"><i class="fa fa-check"></i><b>4.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="division.html"><a href="division.html"><i class="fa fa-check"></i><b>5</b> Division</a><ul>
<li class="chapter" data-level="5.1" data-path="division.html"><a href="division.html#segregation"><i class="fa fa-check"></i><b>5.1</b> Segregation</a></li>
<li class="chapter" data-level="5.2" data-path="division.html"><a href="division.html#chromosome-segregation"><i class="fa fa-check"></i><b>5.2</b> Chromosome Segregation</a></li>
<li class="chapter" data-level="5.3" data-path="division.html"><a href="division.html#budding"><i class="fa fa-check"></i><b>5.3</b> Budding</a></li>
<li class="chapter" data-level="5.4" data-path="division.html"><a href="division.html#morehyphomonas-stalk"><i class="fa fa-check"></i><b>5.4</b> More–Hyphomonas stalk</a></li>
<li class="chapter" data-level="5.5" data-path="division.html"><a href="division.html#morehyphomonas-lifecycle"><i class="fa fa-check"></i><b>5.5</b> More–Hyphomonas lifecycle</a></li>
<li class="chapter" data-level="5.6" data-path="division.html"><a href="division.html#monoderm-cytokinesis"><i class="fa fa-check"></i><b>5.6</b> Monoderm Cytokinesis</a></li>
<li class="chapter" data-level="5.7" data-path="division.html"><a href="division.html#monoderm-cytokinesis-contd."><i class="fa fa-check"></i><b>5.7</b> Monoderm Cytokinesis (cont’d.)</a></li>
<li class="chapter" data-level="5.8" data-path="division.html"><a href="division.html#diderm-cytokinesis"><i class="fa fa-check"></i><b>5.8</b> Diderm Cytokinesis</a></li>
<li class="chapter" data-level="5.9" data-path="division.html"><a href="division.html#moresphingopyxis-alaskensis-division"><i class="fa fa-check"></i><b>5.9</b> More–Sphingopyxis alaskensis division</a></li>
<li class="chapter" data-level="5.10" data-path="division.html"><a href="division.html#morechloroflexi-division"><i class="fa fa-check"></i><b>5.10</b> More–Chloroflexi division</a></li>
<li class="chapter" data-level="5.11" data-path="division.html"><a href="division.html#diderm-cytokinesis-contd."><i class="fa fa-check"></i><b>5.11</b> Diderm Cytokinesis (cont’d.)</a></li>
<li class="chapter" data-level="5.12" data-path="division.html"><a href="division.html#morecaulobacter-division"><i class="fa fa-check"></i><b>5.12</b> More–Caulobacter division</a></li>
<li class="chapter" data-level="5.13" data-path="division.html"><a href="division.html#budding-contd."><i class="fa fa-check"></i><b>5.13</b> Budding (cont’d.)</a></li>
<li class="chapter" data-level="5.14" data-path="division.html"><a href="division.html#ftsz"><i class="fa fa-check"></i><b>5.14</b> FtsZ</a></li>
<li class="chapter" data-level="5.15" data-path="division.html"><a href="division.html#moreasymmetric-constriction"><i class="fa fa-check"></i><b>5.15</b> More–Asymmetric constriction</a><ul>
<li class="chapter" data-level="5.15.1" data-path="division.html"><a href="division.html#pdb-5mn4-ftsz-structure-pdb5mn4ftsz_structure"><i class="fa fa-check"></i><b>5.15.1</b> PDB: 5MN4 –FtsZ structure {#PDB:<em>5MN4</em>–FtsZ_structure}</a></li>
</ul></li>
<li class="chapter" data-level="5.16" data-path="division.html"><a href="division.html#ftsz-contd."><i class="fa fa-check"></i><b>5.16</b> FtsZ (cont’d.)</a></li>
<li class="chapter" data-level="5.17" data-path="division.html"><a href="division.html#escrt"><i class="fa fa-check"></i><b>5.17</b> ESCRT</a></li>
<li class="chapter" data-level="5.18" data-path="division.html"><a href="division.html#escrt-contd."><i class="fa fa-check"></i><b>5.18</b> ESCRT (cont’d.)</a></li>
<li class="chapter" data-level="5.19" data-path="division.html"><a href="division.html#further-reading-4"><i class="fa fa-check"></i><b>5.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="motility.html"><a href="motility.html"><i class="fa fa-check"></i><b>6</b> Motility</a><ul>
<li class="chapter" data-level="6.1" data-path="motility.html"><a href="motility.html#flagellum"><i class="fa fa-check"></i><b>6.1</b> Flagellum</a><ul>
<li class="chapter" data-level="6.1.1" data-path="motility.html"><a href="motility.html#pdb-5wjtflagellum-structure-pdb_5wjtflagellum_structure"><i class="fa fa-check"></i><b>6.1.1</b> PDB: 5WJT–Flagellum structure {#PDB:_5WJT–Flagellum_structure} <img src="img/schematics/6_1_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="motility.html"><a href="motility.html#flagellar-motor"><i class="fa fa-check"></i><b>6.2</b> Flagellar Motor</a></li>
<li class="chapter" data-level="6.3" data-path="motility.html"><a href="motility.html#moreflagellar-motor-structure"><i class="fa fa-check"></i><b>6.3</b> More–Flagellar motor structure</a><ul>
<li class="chapter" data-level="6.3.1" data-path="motility.html"><a href="motility.html#Flagellar_motor_diversity"><i class="fa fa-check"></i><b>6.3.1</b> Flagellar motor diversity</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="motility.html"><a href="motility.html#flagellar-assembly"><i class="fa fa-check"></i><b>6.4</b> Flagellar Assembly</a></li>
<li class="chapter" data-level="6.5" data-path="motility.html"><a href="motility.html#moreflagellar-motor-disassembly"><i class="fa fa-check"></i><b>6.5</b> More–Flagellar motor disassembly</a></li>
<li class="chapter" data-level="6.6" data-path="motility.html"><a href="motility.html#flagella-patterns"><i class="fa fa-check"></i><b>6.6</b> Flagella Patterns</a></li>
<li class="chapter" data-level="6.7" data-path="motility.html"><a href="motility.html#morelophotrichous-bacteria"><i class="fa fa-check"></i><b>6.7</b> More–Lophotrichous bacteria</a></li>
<li class="chapter" data-level="6.8" data-path="motility.html"><a href="motility.html#flagella-patterns-contd."><i class="fa fa-check"></i><b>6.8</b> Flagella Patterns (cont’d.)</a></li>
<li class="chapter" data-level="6.9" data-path="motility.html"><a href="motility.html#moreproteus-mirabilis-flagella"><i class="fa fa-check"></i><b>6.9</b> More–Proteus mirabilis flagella</a></li>
<li class="chapter" data-level="6.10" data-path="motility.html"><a href="motility.html#flagellar-sheath"><i class="fa fa-check"></i><b>6.10</b> Flagellar Sheath</a></li>
<li class="chapter" data-level="6.11" data-path="motility.html"><a href="motility.html#periplasmic-flagella"><i class="fa fa-check"></i><b>6.11</b> Periplasmic Flagella</a></li>
<li class="chapter" data-level="6.12" data-path="motility.html"><a href="motility.html#moretreponema-primitia"><i class="fa fa-check"></i><b>6.12</b> More–Treponema primitia</a></li>
<li class="chapter" data-level="6.13" data-path="motility.html"><a href="motility.html#archaellum"><i class="fa fa-check"></i><b>6.13</b> Archaellum</a><ul>
<li class="chapter" data-level="6.13.1" data-path="motility.html"><a href="motility.html#pdb-5tfyarchaellum-structure-pdb_5tfyarchaellum_structure"><i class="fa fa-check"></i><b>6.13.1</b> PDB: 5TFY–Archaellum structure {#PDB:_5TFY–Archaellum_structure} <img src="img/schematics/6_8_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="6.14" data-path="motility.html"><a href="motility.html#archaella-patterns"><i class="fa fa-check"></i><b>6.14</b> Archaella Patterns</a></li>
<li class="chapter" data-level="6.15" data-path="motility.html"><a href="motility.html#morecone-structure"><i class="fa fa-check"></i><b>6.15</b> More–Cone structure</a></li>
<li class="chapter" data-level="6.16" data-path="motility.html"><a href="motility.html#moreorganizing-center"><i class="fa fa-check"></i><b>6.16</b> More–Organizing center</a></li>
<li class="chapter" data-level="6.17" data-path="motility.html"><a href="motility.html#type-iv-pilus"><i class="fa fa-check"></i><b>6.17</b> Type IV Pilus</a><ul>
<li class="chapter" data-level="6.17.1" data-path="motility.html"><a href="motility.html#Type_IV_pilus_structure"><i class="fa fa-check"></i><b>6.17.1</b> Type IV pilus structure</a></li>
</ul></li>
<li class="chapter" data-level="6.18" data-path="motility.html"><a href="motility.html#type-ix-secretion-system"><i class="fa fa-check"></i><b>6.18</b> Type IX Secretion System</a></li>
<li class="chapter" data-level="6.19" data-path="motility.html"><a href="motility.html#terminal-organelle"><i class="fa fa-check"></i><b>6.19</b> Terminal Organelle</a></li>
<li class="chapter" data-level="6.20" data-path="motility.html"><a href="motility.html#further-reading-5"><i class="fa fa-check"></i><b>6.20</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="navigation.html"><a href="navigation.html"><i class="fa fa-check"></i><b>7</b> Navigation</a><ul>
<li class="chapter" data-level="7.1" data-path="navigation.html"><a href="navigation.html#chemotaxis"><i class="fa fa-check"></i><b>7.1</b> Chemotaxis</a><ul>
<li class="chapter" data-level="7.1.1" data-path="navigation.html"><a href="navigation.html#pdb-2ch7chemoreceptor-structure-pdb_2ch7chemoreceptor_structure"><i class="fa fa-check"></i><b>7.1.1</b> PDB: 2CH7–Chemoreceptor structure {#PDB:_2CH7–Chemoreceptor_structure} <img src="img/schematics/7_1_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="navigation.html"><a href="navigation.html#chemosensory-array"><i class="fa fa-check"></i><b>7.2</b> Chemosensory Array</a><ul>
<li class="chapter" data-level="7.2.1" data-path="navigation.html"><a href="navigation.html#PDB:_6S1K_Chemosensory_array_architecture"><i class="fa fa-check"></i><b>7.2.1</b> PDB: 6S1K Chemosensory array architecture</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="navigation.html"><a href="navigation.html#chemosensory-array-conservation"><i class="fa fa-check"></i><b>7.3</b> Chemosensory Array Conservation</a></li>
<li class="chapter" data-level="7.4" data-path="navigation.html"><a href="navigation.html#chemoreceptors"><i class="fa fa-check"></i><b>7.4</b> Chemoreceptors</a></li>
<li class="chapter" data-level="7.5" data-path="navigation.html"><a href="navigation.html#moreazospirillum-brasilense-aerotaxis"><i class="fa fa-check"></i><b>7.5</b> More–Azospirillum brasilense aerotaxis</a></li>
<li class="chapter" data-level="7.6" data-path="navigation.html"><a href="navigation.html#cytoplasmic-chemosensory-array"><i class="fa fa-check"></i><b>7.6</b> Cytoplasmic Chemosensory Array</a></li>
<li class="chapter" data-level="7.7" data-path="navigation.html"><a href="navigation.html#morevibrio-cholerae-chemosensory-arrays"><i class="fa fa-check"></i><b>7.7</b> More–Vibrio cholerae chemosensory arrays</a></li>
<li class="chapter" data-level="7.8" data-path="navigation.html"><a href="navigation.html#magnetotaxis"><i class="fa fa-check"></i><b>7.8</b> Magnetotaxis</a><ul>
<li class="chapter" data-level="7.8.1" data-path="navigation.html"><a href="navigation.html#pdb-5jygmamk-structure-pdb_5jygmamk_structure"><i class="fa fa-check"></i><b>7.8.1</b> PDB: 5JYG–MamK structure {#PDB:_5JYG–MamK_structure} <img src="img/schematics/7_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="navigation.html"><a href="navigation.html#further-reading-6"><i class="fa fa-check"></i><b>7.9</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="lifecycle.html"><a href="lifecycle.html"><i class="fa fa-check"></i><b>8</b> Lifecycle</a><ul>
<li class="chapter" data-level="8.1" data-path="lifecycle.html"><a href="lifecycle.html#stationary-phase"><i class="fa fa-check"></i><b>8.1</b> Stationary Phase</a></li>
<li class="chapter" data-level="8.2" data-path="lifecycle.html"><a href="lifecycle.html#genome-protection"><i class="fa fa-check"></i><b>8.2</b> Genome Protection</a></li>
<li class="chapter" data-level="8.3" data-path="lifecycle.html"><a href="lifecycle.html#stationary-phase-contd."><i class="fa fa-check"></i><b>8.3</b> Stationary Phase (cont’d.)</a></li>
<li class="chapter" data-level="8.4" data-path="lifecycle.html"><a href="lifecycle.html#differentiation"><i class="fa fa-check"></i><b>8.4</b> Differentiation</a></li>
<li class="chapter" data-level="8.5" data-path="lifecycle.html"><a href="lifecycle.html#moreholdfast"><i class="fa fa-check"></i><b>8.5</b> More–Holdfast</a></li>
<li class="chapter" data-level="8.6" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-sporulation"><i class="fa fa-check"></i><b>8.6</b> Monoderm Sporulation</a></li>
<li class="chapter" data-level="8.7" data-path="lifecycle.html"><a href="lifecycle.html#moresporulation-septum-formation"><i class="fa fa-check"></i><b>8.7</b> More–Sporulation septum formation</a></li>
<li class="chapter" data-level="8.8" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-sporulation-contd."><i class="fa fa-check"></i><b>8.8</b> Monoderm Sporulation (cont’d.)</a></li>
<li class="chapter" data-level="8.9" data-path="lifecycle.html"><a href="lifecycle.html#morespore-engulfment"><i class="fa fa-check"></i><b>8.9</b> More–Spore engulfment</a></li>
<li class="chapter" data-level="8.10" data-path="lifecycle.html"><a href="lifecycle.html#morespore-coating"><i class="fa fa-check"></i><b>8.10</b> More–Spore coating</a></li>
<li class="chapter" data-level="8.11" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-spores"><i class="fa fa-check"></i><b>8.11</b> Monoderm Spores</a></li>
<li class="chapter" data-level="8.12" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-germination"><i class="fa fa-check"></i><b>8.12</b> Monoderm Germination</a></li>
<li class="chapter" data-level="8.13" data-path="lifecycle.html"><a href="lifecycle.html#diderm-sporulation"><i class="fa fa-check"></i><b>8.13</b> Diderm Sporulation</a></li>
<li class="chapter" data-level="8.14" data-path="lifecycle.html"><a href="lifecycle.html#diderm-sporulation-contd."><i class="fa fa-check"></i><b>8.14</b> Diderm Sporulation (cont’d.)</a></li>
<li class="chapter" data-level="8.15" data-path="lifecycle.html"><a href="lifecycle.html#morecomplete-engulfment"><i class="fa fa-check"></i><b>8.15</b> More–Complete engulfment</a></li>
<li class="chapter" data-level="8.16" data-path="lifecycle.html"><a href="lifecycle.html#moremature-spore"><i class="fa fa-check"></i><b>8.16</b> More–Mature spore</a></li>
<li class="chapter" data-level="8.17" data-path="lifecycle.html"><a href="lifecycle.html#moresporomusa-acidovorans-sporulation"><i class="fa fa-check"></i><b>8.17</b> More–Sporomusa acidovorans sporulation</a></li>
<li class="chapter" data-level="8.18" data-path="lifecycle.html"><a href="lifecycle.html#diderm-germination"><i class="fa fa-check"></i><b>8.18</b> Diderm Germination</a></li>
<li class="chapter" data-level="8.19" data-path="lifecycle.html"><a href="lifecycle.html#further-reading-7"><i class="fa fa-check"></i><b>8.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="interaction.html"><a href="interaction.html"><i class="fa fa-check"></i><b>9</b> Interaction</a><ul>
<li class="chapter" data-level="9.1" data-path="interaction.html"><a href="interaction.html#biofilm"><i class="fa fa-check"></i><b>9.1</b> Biofilm</a></li>
<li class="chapter" data-level="9.2" data-path="interaction.html"><a href="interaction.html#biofilm-contd."><i class="fa fa-check"></i><b>9.2</b> Biofilm (cont’d.)</a></li>
<li class="chapter" data-level="9.3" data-path="interaction.html"><a href="interaction.html#type-ii-and-type-iv-secretion"><i class="fa fa-check"></i><b>9.3</b> Type II and Type IV Secretion</a><ul>
<li class="chapter" data-level="9.3.1" data-path="interaction.html"><a href="interaction.html#emd-20712-emd-20713legionella-t2ss-structure-emd-20712_emd-20713legionella_t2ss_structure"><i class="fa fa-check"></i><b>9.3.1</b> EMD-20712 EMD-20713–Legionella T2SS structure {#EMD-20712_EMD-20713–Legionella_T2SS_structure}</a></li>
<li class="chapter" data-level="9.3.2" data-path="interaction.html"><a href="interaction.html#emd-0566-legionella-t4ss-structure-emd-0566_legionella_t4ss_structure"><i class="fa fa-check"></i><b>9.3.2</b> EMD-0566 –Legionella T4SS structure {#EMD-0566_–Legionella_T4SS_structure}</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="interaction.html"><a href="interaction.html#morehelicobacter-pylori-tubes"><i class="fa fa-check"></i><b>9.4</b> More–Helicobacter pylori tubes</a></li>
<li class="chapter" data-level="9.5" data-path="interaction.html"><a href="interaction.html#type-iii-secretion"><i class="fa fa-check"></i><b>9.5</b> Type III Secretion</a><ul>
<li class="chapter" data-level="9.5.1" data-path="interaction.html"><a href="interaction.html#EMD-20611_Injectisome_structure"><i class="fa fa-check"></i><b>9.5.1</b> EMD-20611 Injectisome structure</a></li>
<li class="chapter" data-level="9.5.2" data-path="interaction.html"><a href="interaction.html#PDB:_5WQ8_Secretin_structure"><i class="fa fa-check"></i><b>9.5.2</b> PDB: 5WQ8 Secretin structure</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="interaction.html"><a href="interaction.html#type-v-secretion"><i class="fa fa-check"></i><b>9.6</b> Type V Secretion</a></li>
<li class="chapter" data-level="9.7" data-path="interaction.html"><a href="interaction.html#type-vi-secretion"><i class="fa fa-check"></i><b>9.7</b> Type VI Secretion</a><ul>
<li class="chapter" data-level="9.7.1" data-path="interaction.html"><a href="interaction.html#pdb-5urwprimed-t6ss-structure-pdb_5urwprimed_t6ss_structure"><i class="fa fa-check"></i><b>9.7.1</b> PDB: 5URW–Primed T6SS structure {#PDB:_5URW–Primed_T6SS_structure} <img src="img/schematics/9_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="9.8" data-path="interaction.html"><a href="interaction.html#type-vi-secretion-contd."><i class="fa fa-check"></i><b>9.8</b> Type VI Secretion (cont’d.)</a><ul>
<li class="chapter" data-level="9.8.1" data-path="interaction.html"><a href="interaction.html#pdb-5urxfired-t6ss-structure-pdb_5urxfired_t6ss_structure"><i class="fa fa-check"></i><b>9.8.1</b> PDB: 5URX–Fired T6SS structure {#PDB:_5URX–Fired_T6SS_structure} <img src="img/schematics/9_7_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="9.9" data-path="interaction.html"><a href="interaction.html#moret6ss-battery"><i class="fa fa-check"></i><b>9.9</b> More–T6SS battery</a></li>
<li class="chapter" data-level="9.10" data-path="interaction.html"><a href="interaction.html#moremacs"><i class="fa fa-check"></i><b>9.10</b> More–MACs</a><ul>
<li class="chapter" data-level="9.10.1" data-path="interaction.html"><a href="interaction.html#MAC_array_structure"><i class="fa fa-check"></i><b>9.10.1</b> MAC array structure</a></li>
</ul></li>
<li class="chapter" data-level="9.11" data-path="interaction.html"><a href="interaction.html#type-ix-secretion"><i class="fa fa-check"></i><b>9.11</b> Type IX Secretion</a></li>
<li class="chapter" data-level="9.12" data-path="interaction.html"><a href="interaction.html#predation"><i class="fa fa-check"></i><b>9.12</b> Predation</a></li>
<li class="chapter" data-level="9.13" data-path="interaction.html"><a href="interaction.html#predation-contd."><i class="fa fa-check"></i><b>9.13</b> Predation (cont’d.)</a></li>
<li class="chapter" data-level="9.14" data-path="interaction.html"><a href="interaction.html#morebdellovibrio-bacteriovorus-entry"><i class="fa fa-check"></i><b>9.14</b> More–Bdellovibrio bacteriovorus entry</a></li>
<li class="chapter" data-level="9.15" data-path="interaction.html"><a href="interaction.html#predation-contd.-1"><i class="fa fa-check"></i><b>9.15</b> Predation (cont’d.)</a></li>
<li class="chapter" data-level="9.16" data-path="interaction.html"><a href="interaction.html#further-reading-8"><i class="fa fa-check"></i><b>9.16</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="viruses.html"><a href="viruses.html"><i class="fa fa-check"></i><b>10</b> Viruses</a><ul>
<li class="chapter" data-level="10.1" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis"><i class="fa fa-check"></i><b>10.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.2" data-path="viruses.html"><a href="viruses.html#bacteroidetes-strain-jt5"><i class="fa fa-check"></i><b>10.2</b> Bacteroidetes strain JT5</a></li>
<li class="chapter" data-level="10.3" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-1"><i class="fa fa-check"></i><b>10.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.4" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae"><i class="fa fa-check"></i><b>10.4</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="10.5" data-path="viruses.html"><a href="viruses.html#escherichia-coli"><i class="fa fa-check"></i><b>10.5</b> Escherichia coli</a></li>
<li class="chapter" data-level="10.6" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus"><i class="fa fa-check"></i><b>10.6</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="10.7" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.7</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="10.8" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus"><i class="fa fa-check"></i><b>10.8</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="10.9" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-1"><i class="fa fa-check"></i><b>10.9</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="10.10" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-2"><i class="fa fa-check"></i><b>10.10</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.11" data-path="viruses.html"><a href="viruses.html#further-reading-9"><i class="fa fa-check"></i><b>10.11</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>11</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.library.caltech.edu" target="blank">Published by Caltech Library</a><li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Atlas of Bacterial and Archaeal Cell Structure</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="motility" class="section level1">
<h1><span class="header-section-number">Chapter 6</span> Motility</h1>
<p>“internal combustion engines do no better.” - Howard Berg <span class="citation">Berg (<a href="#ref-berg1988">1988</a>)</span></p>
<div id="flagellum" class="section level2">
<h2><span class="header-section-number">6.1</span> Flagellum</h2>
<p>Location, location, location. So far, we have focused on how your cell can take the best advantage of its spot in the world. But why not find a better spot? Some cells live stationary lives, attached to a surface or embedded in a biofilm. Many, though, are explorers, using an impressive variety of techniques to move through their environments, finding advantages in places with more food or fewer competitors. In this chapter, we explore how you might make your cell a mobile home.</p>
<p>Most bacteria and archaea live in liquid, so motility means swimming. When you are the size of a cell, though, the backstroke does not get you very far. A rough measure called the Reynolds number describes the relative influence of inertia and viscosity on liquid flow, and this parameter scales with an organism’s size. In the low-Reynolds number world of microbes, inertia is virtually nonexistent. When a rod-shaped bacterium stops swimming, it gets to coast only about the diameter of a hydrogen atom (~1 Å) before stopping. In this molasses-like environment, rotary propellers work much better than paddles.</p>
<p>Nearly all bacteria that swim use the same propeller: a rotary motor embedded in their envelope that spins a long helical fiber called a flagellum. A universal joint called the hook connects the filament to the motor, translating the rotation. Flagella, like the one on this <em>Campylobacter jejuni</em>, are typically many times longer than the cell and take the form of a three-dimensional wave. The filament is highly dynamic, as you would expect, and throughout this book, you will see examples caught in various conformations: straight, curved, or in a typical loose helical waveform like this.</p>
<div class="figure"><span id="fig:unnamed-chunk-99"></span>
<div id="htmlwidget-2dbef06f4f9b077b2a0b" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-2dbef06f4f9b077b2a0b">{"x":{"doi":"10.22002/D1.1525","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.1: Campylobacter jejuni Collected by: Morgan Beeby <a href="https://doi.org/10.22002/D1.1525">10.22002/D1.1525</a>
</p>
</div>
<div id="pdb-5wjtflagellum-structure-pdb_5wjtflagellum_structure" class="section level3">
<h3><span class="header-section-number">6.1.1</span> PDB: 5WJT–Flagellum structure {#PDB:_5WJT–Flagellum_structure} <img src="img/schematics/6_1_1.gif" /><!-- --></h3>
<p>The helical filament of the flagellum is made up of many copies of one or a few flagellin proteins. Each flagellin monomer has a soluble head domain and a hydrophobic alpha-helical tail that bundles together with the tails of other monomers to form a hollow tube. The tube comprises 11 twisting protofilaments, as you can see in this section of a flagellum from <em>Bacillus subtilis</em> <span class="citation">Wang et al. (<a href="#ref-wang2017">2017</a>)</span>.</p>
</div>
</div>
<div id="flagellar-motor" class="section level2">
<h2><span class="header-section-number">6.2</span> Flagellar Motor</h2>
<p>The motor that spins the flagellum is a complicated molecular machine made of many copies of dozens of different proteins, spanning the cell envelope with components in the cytoplasm, the periplasm (in diderms), and outside the cell, as you can see in this <em>Bdellovibrio bacteriovorus</em>. To get a closer look, we can average the individual motors from many cells <a href="motility.html#moreflagellar-motor-structure">More–Flagellar motor structure</a>. Broadly, the motor consists of a stationary part (the “stators”) that anchors the machine in the cell and a rotating part (the “rotor”) that spins the filament. The torque for spinning the flagellum comes from small movements in the stators that kick the rotor in a circle. The energy for these movements comes from the ion potential across the cell membrane that we discussed in Chapter 2; the stators provide a conduit for protons (in most species) or sodium ions (in some marine species) to diffuse down their chemical gradient into the cytoplasm, powering a conformational change in the stators in the process. The energy demands of the machine are high: a single rotation requires about 1,000 protons to flow through the stators, and the motor may spin at more than 6,000 rotations per minute. The fact that cells pay this energetic cost indicates a strong evolutionary selection for motility, or in other words, the powerful advantage your cell can gain by learning to swim.</p>
<p>While the basic plan of the motor is the same in every species that has flagella, there are structural differences that reflect the different environments species encounter [Schematic–Flagellar motor diversity].</p>
<div class="figure"><span id="fig:unnamed-chunk-101"></span>
<div id="htmlwidget-4895c957ec61d9554ca0" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-4895c957ec61d9554ca0">{"x":{"doi":"10.22002/D1.1526","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.2: Bdellovibrio bacteriovorus Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1526">10.22002/D1.1526</a>
</p>
</div>
</div>
<div id="moreflagellar-motor-structure" class="section level2">
<h2><span class="header-section-number">6.3</span> More–Flagellar motor structure</h2>
<p>This is an average of the flagellar motors from more than one thousand <em>Bdellovibrio bacteriovorus</em> cells. Working upward from the base, the major parts of the rotor are the C-ring (for _C_ytoplasmic), the MS-ring (for _M_embrane and _S_upramembrane), the rod, the hook, and finally the filament. They are surrounded by stationary parts: the stator ring (which is dynamic, so we cannot resolve the stators as they cross the inner membrane and connect to the C-ring) and a series of bushings that allow rotation within the cell wall (the _P_eriplasmic or P-ring) and outer membrane (_L_ipopolysaccharide or L-ring). Additional cytoplasmic components form the export apparatus, which is involved in assembly (discussed on the next page).</p>
<div class="figure"><span id="fig:unnamed-chunk-102"></span>
<div id="htmlwidget-0e8fbd67f29b4a4d7dfe" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-0e8fbd67f29b4a4d7dfe">{"x":{"doi":"10.22002/D1.1537","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.3: Bdellovibrio bacteriovorus Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1537">10.22002/D1.1537</a>
</p>
</div>
<div id="Flagellar_motor_diversity" class="section level3">
<h3><span class="header-section-number">6.3.1</span> Flagellar motor diversity</h3>
<p><img src="img/schematics/6_2_1.jpg" /><!-- --></p>
<p>As you can see in these averages of flagellar motors from different species <span class="citation">Murphy, Leadbetter, and Jensen (<a href="#ref-murphy2006">2006</a>)</span> <span class="citation">Chen et al. (<a href="#ref-chen2011">2011</a>)</span> <span class="citation">Zhao, Norris, and Liu (<a href="#ref-zhao2014">2014</a>)</span> <span class="citation">Beeby et al. (<a href="#ref-beeby2016">2016</a>)</span> <span class="citation">Qin et al. (<a href="#ref-qin2017">2017</a>)</span> <span class="citation">Chaban, Coleman, and Beeby (<a href="#ref-chaban2018">2018</a>)</span> <span class="citation">Kaplan et al. (<a href="#ref-kaplan2019">2019</a>)</span> <span class="citation">Ferreira et al. (<a href="#ref-ferreira2019">2019</a>)</span> <span class="citation">Chang et al. (<a href="#ref-chang2019">2019</a>)</span>, bacteria have evolved structural adaptations of their motors to better suit their environments. For instance, if your cell is a pathogen colonizing an animal’s intestinal tract (like <em>Campylobacter jejuni</em>), it will be swimming in more viscous conditions and may therefore have evolved a wider stator ring to generate more torque, along with reinforced anchors in the cell wall and outer membrane to withstand that added torque.</p>
</div>
</div>
<div id="flagellar-assembly" class="section level2">
<h2><span class="header-section-number">6.4</span> Flagellar Assembly</h2>
<p>Operating the flagellar motor is impressive, but so is building it in the first place. Remember that the envelope of bacterial cells is a complicated multilayered barrier. The flagellar motor has about two dozen unique components, each present in many copies, embedded in every layer of the cell envelope. How can your cell get all hundred or more components (tens of thousands if you count the components of the filament) where they need to go? Making the feat even more impressive, the machine builds itself, assembling from the inside out. First the components associated with the inner membrane come together, forming an “export apparatus” which pumps subsequent components across the membrane to assemble in the periplasm and outer membrane (if the cell is a diderm). The energy for this process comes from an ATPase at the base of the machine. You can see a late stage in this assembly process in this <em>Hylemonella gracilis</em> cell. The final piece to be assembled (still missing here) is the flagellar filament, which continues the pattern of outward assembly; flagellin monomers travel through the hollow tube to take their places at the tip. Flagellar motors can also disassemble, for instance when the filament is broken <a href="motility.html#moreflagellar-motor-disassembly">More–Flagellar motor disassembly</a>, and cells may make many new flagella throughout their lifetime.</p>
<p>So the flagellar motor is not just a machine for motility; it is also a machine to secrete molecules outside the cell. Bacteria and archaea contain many such “secretion systems,” each specialized to transport specific macromolecules (e.g. DNA or a protein toxin) across cell envelopes–both their own and sometimes others. Secretion systems are classified by evolutionary relatedness; there are currently ~10 types recognized in bacteria, some of which are also present in archaea. The flagellar motor is an example of a Type III Secretion System. You will see another member of this family in Chapter 9, and examples of many other types in the rest of the book, starting in just a few pages.</p>
<div class="figure"><span id="fig:unnamed-chunk-104"></span>
<div id="htmlwidget-f640222879e12bb9224f" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-f640222879e12bb9224f">{"x":{"doi":"10.22002/D1.1527","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.4: Hylemonella gracilis Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1527">10.22002/D1.1527</a>
</p>
</div>
</div>
<div id="moreflagellar-motor-disassembly" class="section level2">
<h2><span class="header-section-number">6.5</span> More–Flagellar motor disassembly</h2>
<p>When flagella break (or in some species, like <em>Caulobacter crescentus</em>, are ejected so the cell can attach to a surface), the motor is disassembled, usually beginning with the export apparatus. Not everything is dismantled, though; the P- and L-rings remain in place in the cell wall and outer membrane, respectively, as you can see in this <em>Pseudomonas aeruginosa</em> cell. We do not know if the cell has a reason for leaving them there, but one possibility is that they may function as a plug for the outer membrane when the hook is no longer present.</p>
<div class="figure"><span id="fig:unnamed-chunk-105"></span>
<div id="htmlwidget-f845111c6374a0aa8fe8" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-f845111c6374a0aa8fe8">{"x":{"doi":"10.22002/D1.1538","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.5: Pseudomonas aeruginosa Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1538">10.22002/D1.1538</a>
</p>
</div>
</div>
<div id="flagella-patterns" class="section level2">
<h2><span class="header-section-number">6.6</span> Flagella Patterns</h2>
<p>Once assembled, flagella can work in different ways. The motor is bidirectional, and can rotate either clockwise or counterclockwise. Depending on the number and location of flagella on the cell (and the cell’s shape), this can push the cell, pull it, or give rise to even more complicated swimming behavior. Some bacterial species, like the <em>Bdellovibrio bacteriovorus</em> you just saw, are monotrichous (“single haired”), with one flagellum located at one pole to push/pull the cell. Other species, like the <em>Campylobacter jejuni</em> here, have bipolar flagella–one at each end. Still others are lophotrichous (“crest-haired”), with a clump of flagella <a href="motility.html#morelophotrichous-bacteria">More–Lophotrichous bacteria</a>.</p>
<div class="figure"><span id="fig:unnamed-chunk-106"></span>
<div id="htmlwidget-37f0c7b227fc2a4310e5" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-37f0c7b227fc2a4310e5">{"x":{"doi":"10.22002/D1.1528","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.6: Campylobacter jejuni Collected by: Morgan Beeby <a href="https://doi.org/10.22002/D1.1528">10.22002/D1.1528</a>
</p>
</div>
</div>
<div id="morelophotrichous-bacteria" class="section level2">
<h2><span class="header-section-number">6.7</span> More–Lophotrichous bacteria</h2>
<p>Many lophotrichous species, like the <em>Hylemonella gracilis</em> you saw in Chapter 3–Length or this <em>Helicobacter pylori</em>, have a tuft of flagella at their cell pole. In some species, though, the tuft is located elsewhere; for example, a clump of flagella on the concave side of banana-shaped <em>Selenomonas artemidis</em> pushes the cells sideways in a seesawing swimming pattern.</p>
<div class="figure"><span id="fig:unnamed-chunk-107"></span>
<div id="htmlwidget-5e76d4f2b05d4ef017ed" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-5e76d4f2b05d4ef017ed">{"x":{"doi":"10.22002/D1.1539","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.7: Helicobacter pylori Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1539">10.22002/D1.1539</a>
</p>
</div>
</div>
<div id="flagella-patterns-contd." class="section level2">
<h2><span class="header-section-number">6.8</span> Flagella Patterns (cont’d.)</h2>
<p>Still other species are peritrichous (“hair around”), with multiple flagella distributed randomly around the cell, as you can see on this <em>Pseudomonas flexibilis</em>. The number of flagella varies between different species, from relatively few here to considerably more <a href="motility.html#moreproteus-mirabilis-flagella">More–Proteus mirabilis flagella</a>. The well-known model system <em>Escherichia coli</em> is also peritrichously flagellated. In this arrangement, when the flagellar motors are all rotating one direction (counter-clockwise), the flagella form a whip-like bundle that propels the cell to “run” in a straight line. When one or more motors switch to clockwise rotation, the flagella dissociate from the bundle and “tumble” the cell to face a new direction. In the next chapter, you will see how cells use this behavior to seek out favorable spots.</p>
<div class="figure"><span id="fig:unnamed-chunk-108"></span>
<div id="htmlwidget-72ceb9b5c41062eb0bd5" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-72ceb9b5c41062eb0bd5">{"x":{"doi":"10.22002/D1.1529","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.8: Pseudomonas flexibilis Collected by: Morgan Beeby <a href="https://doi.org/10.22002/D1.1529">10.22002/D1.1529</a>
</p>
</div>
</div>
<div id="moreproteus-mirabilis-flagella" class="section level2">
<h2><span class="header-section-number">6.9</span> More–Proteus mirabilis flagella</h2>
<p><em>Proteus mirabilis</em> adapt their motility machinery to their environment. In liquid, the short rod-shaped cells swim with the help of a handful of flagella distributed peritrichously around their cell body. When they encounter a solid surface, the cells elongate and build many more flagella, as you can see on this cell. Instead of swimming, they now use their flagella to propel themselves in groups across the surface, a motility mode known as “swarming.” This is an example of a differentiated lifecycle, which will come up again in Chapter 8.</p>
<div class="figure"><span id="fig:unnamed-chunk-109"></span>
<div id="htmlwidget-bec24cad97195eda0200" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-bec24cad97195eda0200">{"x":{"doi":"10.22002/D1.1540","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.9: Proteus mirabilis Collected by: Qing Yao <a href="https://doi.org/10.22002/D1.1540">10.22002/D1.1540</a>
</p>
</div>
</div>
<div id="flagellar-sheath" class="section level2">
<h2><span class="header-section-number">6.10</span> Flagellar Sheath</h2>
<p>You may have noticed that some of the flagella in this chapter were enclosed within the outer membrane of the cell. We call these “sheathed” flagella. It is a common adaptation in pathogenic species, like this <em>Helicobacter hepaticus</em>. Flagella offer pathogens a great advantage in colonizing their hosts; hosts in turn have learned to use them to identify potential invaders. As a result, the innate immune response of many eukaryotes, from plants to insects to humans, has evolved to recognize the telltale and abundant signal of flagellin proteins in the long filament. If your cell aims to take up residence in such a host, it could therefore benefit from cloaking this strongly antigenic feature.</p>
<div class="figure"><span id="fig:unnamed-chunk-110"></span>
<div id="htmlwidget-18b35d9ec3151f72062e" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-18b35d9ec3151f72062e">{"x":{"doi":"10.22002/D1.1530","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.10: Helicobacter hepaticus Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1530">10.22002/D1.1530</a>
</p>
</div>
</div>
<div id="periplasmic-flagella" class="section level2">
<h2><span class="header-section-number">6.11</span> Periplasmic Flagella</h2>
<p>If your cell <em>is</em> a pathogen, swimming can be very useful, but so can burrowing, for instance between cells in host tissue. To do this, why not turn your cell into a corkscrew with the equipment at hand? Some cells do just this, wrapping their flagellum around their body to back out of a tight spot, or burrow into one. Other, diderm species like the <em>Borrelia burgdorferi</em> here have turned the temporary adaptation into a permanent one: they assemble their flagella <em>inside</em> the cell envelope, with the filaments wrapping around between the cell wall and outer membrane. These “periplasmic” flagella are usually multiple, arising from one or both ends of the cell, and pack together, forming a helical ribbon. The helical ribbon helps give these Spirochetes (“spiral haired”) their characteristic shape; mutants that cannot make flagella are simple rods. Some spirochetes also have additional features that may help them move around in animal hosts <a href="motility.html#moretreponema-primitia">More–Treponema primitia</a></p>
<div class="figure"><span id="fig:unnamed-chunk-111"></span>
<div id="htmlwidget-e66a2078139d9439fc21" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-e66a2078139d9439fc21">{"x":{"doi":"10.22002/D1.1531","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.11: Borrelia burgdorferi Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1531">10.22002/D1.1531</a>
</p>
</div>
</div>
<div id="moretreponema-primitia" class="section level2">
<h2><span class="header-section-number">6.12</span> More–Treponema primitia</h2>
<p><em>Treponema primitia</em> like this are commensal residents of the termite gut, helping break down cellulose. In addition to two periplasmic flagella, the cells have arrays of bowl- and hook-like structures on their surface, the function of which, likely related to motility, remains mysterious.</p>
<div class="figure"><span id="fig:unnamed-chunk-112"></span>
<div id="htmlwidget-a511def144fd4ca88935" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-a511def144fd4ca88935">{"x":{"doi":"10.22002/D1.1531","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.12: Borrelia burgdorferi Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1531">10.22002/D1.1531</a>
</p>
</div>
</div>
<div id="archaellum" class="section level2">
<h2><span class="header-section-number">6.13</span> Archaellum</h2>
<p>Archaea swim, too. And as you might expect, they use similar machinery to do so: an envelope-embedded motor that spins a long extracellular filament. Despite the structural similarity, the machinery evolved independently, another indication of the strong advantage conferred by swimming. To reflect this similar-but-not-the-same character, we call the archaeal analogue of the bacterial flagellum the archaellum. Unlike the flagellum, which is a Type III Secretion System, the archaellum is a Type II Secretion System. As you can see on this <em>Methanoregula formicica</em> cell, archaella are narrower than flagella [Schematic–Archaellum structure]. The motor is also different, and uses the ATPase at the base not just for assembly, but also to power rotation for swimming. Like flagella, archaellar motors can rotate in either direction, resulting in the filaments pushing or pulling the cell.</p>
<div class="figure"><span id="fig:unnamed-chunk-113"></span>
<div id="htmlwidget-59f536494282576332f8" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-59f536494282576332f8">{"x":{"doi":"10.22002/D1.1532","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.13: Methanoregula formicica Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1532">10.22002/D1.1532</a>
</p>
</div>
<div id="pdb-5tfyarchaellum-structure-pdb_5tfyarchaellum_structure" class="section level3">
<h3><span class="header-section-number">6.13.1</span> PDB: 5TFY–Archaellum structure {#PDB:_5TFY–Archaellum_structure} <img src="img/schematics/6_8_1.gif" /><!-- --></h3>
<p>The overall architecture of the helical archaellum is similar to that of the bacterial flagellum, as you can see in this structure from <em>Methanospirillum hungatei</em> <span class="citation">Poweleit et al. (<a href="#ref-poweleit2016">2016</a>)</span>. Each protein subunit is smaller, however, resulting in a narrower filament diameter: ~10 nm, compared to ~24 nm for the flagellum. They are also more tightly packed, so there is no central channel.</p>
</div>
</div>
<div id="archaella-patterns" class="section level2">
<h2><span class="header-section-number">6.14</span> Archaella Patterns</h2>
<p>Similar to flagella in bacteria, different archaeal species employ different numbers and patterns of archaella. Some species have one, others have many, either distributed peritrichously (all around) as in the <em>Methanoregula formicica</em> you just saw, or lophotrichously (clumped) as in this <em>Thermococcus kodakaraensis</em>. In <em>T. kodakaraensis</em> and related species, an additional structure–a large conical plate–is seen in the cytoplasm, perhaps providing leverage for the multiple motors. The plate has a unique structure <a href="motility.html#morecone-structure">More–Cone structure</a> and may act as an organizing center akin to the polar PopZ structure we discussed in the last chapter <a href="motility.html#moreorganizing-center">More–Organizing center</a>. Note the two peaks on this cone; it may be in the process of replicating in preparation for division.</p>
<div class="figure"><span id="fig:unnamed-chunk-115"></span>
<div id="htmlwidget-8e26bdd42feb676f9b72" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-8e26bdd42feb676f9b72">{"x":{"doi":"10.22002/D1.1533","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.14: Thermococcus kodakaraensis Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1533">10.22002/D1.1533</a>
</p>
</div>
</div>
<div id="morecone-structure" class="section level2">
<h2><span class="header-section-number">6.15</span> More–Cone structure</h2>
<p>The archaellar plate does not come to a point at the tip, but rather is a conical frustum (open at the top), resembling a lampshade. In the center of the tip is a small ring, as you can see in a top view in this lysed, flattened <em>Thermococcus kodakaraensis</em> cell. The function of the ring remains unknown; perhaps it helps nucleate the rest of the structure?</p>
<div class="figure"><span id="fig:unnamed-chunk-116"></span>
<div id="htmlwidget-c019e9638090a6cb0497" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-c019e9638090a6cb0497">{"x":{"doi":"10.22002/D1.1542","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.15: Thermococcus kodakaraensis Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1542">10.22002/D1.1542</a>
</p>
</div>
</div>
<div id="moreorganizing-center" class="section level2">
<h2><span class="header-section-number">6.16</span> More–Organizing center</h2>
<p>As you can see more clearly in this partially-lysed, flattened <em>Thermococcus kodakaraensis</em> cell, the conical plate in the cytoplasm is attached to more than just the archaella. It is also associated with chemosensory arrays (discussed in the next chapter) and DNA, as you can see from the ribosome-excluding zone. This structure may therefore perform an analogous function to bacterial organizing proteins such as PopZ, tethering cellular components into a <em>de facto</em> pole for the (in this case round) cell.</p>
<div class="figure"><span id="fig:unnamed-chunk-117"></span>
<div id="htmlwidget-59e15cd3a3ab83fe766c" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-59e15cd3a3ab83fe766c">{"x":{"doi":"10.22002/D1.1543","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.16: Thermococcus kodakaraensis Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1543">10.22002/D1.1543</a>
</p>
</div>
</div>
<div id="type-iv-pilus" class="section level2">
<h2><span class="header-section-number">6.17</span> Type IV Pilus</h2>
<p>If your cell lives on a surface, what is the best way to get around? How about using a grappling hook? Some bacteria, like this <em>Myxococcus xanthus</em>, use a Type II Secretion System related to the archaellar motor to pull themselves around their environment. As you can see, the structure looks familiar: a motor embedded in the envelope with a long extracellular filament. In this case the filament is called a pilus (“hair” in Latin). Bacteria and archaea make many kinds of pili (also generically called fimbriae (“fringe”)) and you will see some of their other functions in later chapters. The <em>M. xanthus</em> pili, classified as Type IV pili, function not as propellers like flagella or archaella, but rather extend, attach to a surface, and then retract to pull the cell toward the attachment point [Schematic–Type IV pilus structure]. The pilus motors are the strongest known in nature, and can retract pili at up to 1 μm/s; the combined action of multiple pili leads to extremely rapid “twitching” motility of the cell over a surface. The motor structure, or basal body, remains intact even when no pilus is assembled. These rod-shaped cells have many basal bodies at both cell poles; to switch direction, the cell simply disassembles the pili on one end and builds new pili from the machines waiting on the other.</p>
<p>In addition to attaching to a surface, the pili can also attach to other <em>M. xanthus</em> cells. This enables the cells to move over surfaces <em>en masse</em>. Combined with their practice of eating other bacteria, this property has led them to be compared to packs of wolves hunting down their prey.</p>
<div class="figure"><span id="fig:unnamed-chunk-118"></span>
<div id="htmlwidget-a5b11493e6865d104a12" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-a5b11493e6865d104a12">{"x":{"doi":"10.22002/D1.1534","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.17: Myxococcus xanthus Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1534">10.22002/D1.1534</a>
</p>
</div>
<div id="Type_IV_pilus_structure" class="section level3">
<h3><span class="header-section-number">6.17.1</span> Type IV pilus structure</h3>
<p><img src="img/schematics/6_10_1.gif" /><!-- --></p>
<p>A series of rings anchor the Type IV Pilus basal body in the cell envelope. This rigid structure provides leverage for an ATPase at the base to rotate an adaptor in the inner membrane. We think that when it spins in one direction, the adaptor scoops pilin monomers diffusing in the inner membrane into the assembling pilus <span class="citation">Chang et al. (<a href="#ref-chang2016">2016</a>)</span>. Once the pilus has reached its target, attachment is sensed by the basal body (we do not yet know how). As a result, the assembly ATPase dissociates and a second, homologous ATPase takes its place. This disassembly ATPase spins the adaptor in the opposite direction, escorting pilin monomers back into the inner membrane, ready to join the next growing pilus.</p>
</div>
</div>
<div id="type-ix-secretion-system" class="section level2">
<h2><span class="header-section-number">6.18</span> Type IX Secretion System</h2>
<p>Other bacteria use different machinery to move over surfaces. <em>Flavobacterium johnsoniae</em> like this cell use a Type IX Secretion System to secrete adhesive filaments. These filaments move on a helical track around the cell, and thereby the cell moves forward on a surface. The power for this gliding movement is thought to come from rotary motors anchored in the cell wall that propel the adhesins along their track in a rack-and-pinion fashion, but we still do not understand the mechanism in detail.</p>
<div class="figure"><span id="fig:unnamed-chunk-120"></span>
<div id="htmlwidget-d80e31d05fc53deec061" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-d80e31d05fc53deec061">{"x":{"doi":"10.22002/D1.1535","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.18: Flavobacterium johnsoniae Collected by: Gregory Henderson <a href="https://doi.org/10.22002/D1.1535">10.22002/D1.1535</a>
</p>
</div>
</div>
<div id="terminal-organelle" class="section level2">
<h2><span class="header-section-number">6.19</span> Terminal Organelle</h2>
<p>Another, familiar, way to get across a surface is to walk. For this to work, though, you need to be able to change the conformation of your body. This is possible if your cell lacks a cell wall or surface layer, like this <em>Mycoplasma pneumoniae</em>. Like the <em>Mycoplasma genitalium</em> you saw in Chapter 2–Membrane, these cells are intracellular pathogens, so they do not need to buttress their membrane against differences in osmolarity. As a result, they are soft and flexible. This may allow them to use a leg-like internal structure called a terminal organelle to crawl, or more elegantly “glide,” across a surface. The exact mechanism is still unclear, but one possibility is that a hinge-like conformational change in the terminal organelle extends and contracts the back of the cell with respect to the front, similar to the movement of an inchworm (but less exaggerated). Combined with adhesion proteins on the cell surface, this might propel the cell forward.</p>
<p>The skeleton-like terminal organelle gives these cells their characteristic flask shape. In combination with their minimal cell envelope, it can also give rise to an unusual method of cell division. In some species (or genetically engineered strains) that lack the division protein FtsZ, <em>Mycoplasma</em> cells still manage to divide. They replicate their terminal organelle normally, as you can see this cell has done, and then the two copies simply walk away from each other, stretching the mother cell between them until the membrane spontaneously separates to produce two daughters.</p>
<p>Keep in mind that these are simply <em>some</em> of the ways we know bacteria and archaea get around, and we continue to discover new ones.</p>
<div class="figure"><span id="fig:unnamed-chunk-121"></span>
<div id="htmlwidget-776022e97dacbed81c3c" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-776022e97dacbed81c3c">{"x":{"doi":"10.22002/D1.1536","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 6.19: Mycoplasma pneumoniae Collected by: Gregory Henderson <a href="https://doi.org/10.22002/D1.1536">10.22002/D1.1536</a>
</p>
</div>
</div>
<div id="further-reading-5" class="section level2">
<h2><span class="header-section-number">6.20</span> Further Reading</h2>
<p><span class="citation">Albers and Jarrell (<a href="#ref-albers2018">2018</a>)</span> <span class="citation">Berg (<a href="#ref-berg2003">2003</a>)</span> <span class="citation">Jarrell and McBride (<a href="#ref-jarrell2008">2008</a>)</span> <span class="citation">Muñoz-Dorado et al. (<a href="#ref-munoz-dorado2016">2016</a>)</span> <span class="citation">Shrivastava and Berg (<a href="#ref-shrivastava2015">2015</a>)</span></p>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-albers2018">
<p>Albers, Sonja-Verena, and Ken F. Jarrell. 2018. “The Archaellum: An Update on the Unique Archaeal Motility Structure.” <em>Trends Microbiol.</em> 26 (4): 351–62. doi:<a href="https://doi.org/10.1016/j.tim.2018.01.004">10.1016/j.tim.2018.01.004</a>.</p>
</div>
<div id="ref-beeby2016">
<p>Beeby, M., D. A. Ribardo, C. A. Brennan, E. G. Ruby, G. J. Jensen, and D. R. Hendrixson. 2016. “Diverse High-Torque Bacterial Flagellar Motors Assemble Wider Stator Rings Using a Conserved Protein Scaffold.” <em>Proc Natl Acad Sci U S A</em> 113 (13): E1917–26. doi:<a href="https://doi.org/10.1073/pnas.1518952113">10.1073/pnas.1518952113</a>.</p>
</div>
<div id="ref-berg1988">
<p>Berg, H. C. 1988. “A Physicist Looks at Bacterial Chemotaxis.” <em>Cold Spring Harb Symp Quant Biol</em> 53 Pt 1: 1–9.</p>
</div>
<div id="ref-berg2003">
<p>Berg, Howard C. 2003. “The Rotary Motor of Bacterial Flagella.” <em>Annu. Rev. Biochem.</em> 72 (1). Annual Reviews: 19–54. doi:<a href="https://doi.org/10.1146/annurev.biochem.72.121801.161737">10.1146/annurev.biochem.72.121801.161737</a>.</p>
</div>
<div id="ref-chaban2018">
<p>Chaban, Bonnie, Izaak Coleman, and Morgan Beeby. 2018. “Evolution of Higher Torque in Campylobacter- Type Bacterial Flagellar Motors.” <em>Scientific Reports</em> 8 (1). Nature Publishing Group: 97. doi:<a href="https://doi.org/10.1038/s41598-017-18115-1">10.1038/s41598-017-18115-1</a>.</p>
</div>
<div id="ref-chang2016">
<p>Chang, Y. W., L. A. Rettberg, A. Treuner-Lange, J. Iwasa, L. Sogaard-Andersen, and G. J. Jensen. 2016. “Architecture of the Type IVa Pilus Machine.” <em>Science</em> 351 (6278): aad2001. doi:<a href="https://doi.org/10.1126/science.aad2001">10.1126/science.aad2001</a>.</p>
</div>
<div id="ref-chang2019">
<p>Chang, Yunjie, Ki Hwan Moon, Xiaowei Zhao, Steven J Norris, MD A Motaleb, and Jun Liu. 2019. “Structural Insights into Flagellar StatorRotor Interactions.” Edited by Edward H Egelman, John Kuriyan, Edward H Egelman, and Masahiro Ito. <em>eLife</em> 8 (July). eLife Sciences Publications, Ltd: e48979. doi:<a href="https://doi.org/10.7554/eLife.48979">10.7554/eLife.48979</a>.</p>
</div>
<div id="ref-chen2011">
<p>Chen, S., M. Beeby, G. E. Murphy, J. R. Leadbetter, D. R. Hendrixson, A. Briegel, Z. Li, et al. 2011. “Structural Diversity of Bacterial Flagellar Motors.” <em>The EMBO Journal</em> 30 (14): 2972–81. doi:<a href="https://doi.org/10.1038/emboj.2011.186">10.1038/emboj.2011.186</a>.</p>
</div>
<div id="ref-ferreira2019">
<p>Ferreira, Josie L., Forson Z. Gao, Florian M. Rossmann, Andrea Nans, Susanne Brenzinger, Rohola Hosseini, Amanda Wilson, et al. 2019. “<span class="math inline">\(\gamma\)</span>-Proteobacteria Eject Their Polar Flagella Under Nutrient Depletion, Retaining Flagellar Motor Relic Structures.” <em>PLOS Biology</em> 17 (3). Public Library of Science: e3000165. doi:<a href="https://doi.org/10.1371/journal.pbio.3000165">10.1371/journal.pbio.3000165</a>.</p>
</div>
<div id="ref-jarrell2008">
<p>Jarrell, K. F., and M. J. McBride. 2008. “The Surprisingly Diverse Ways That Prokaryotes Move.” <em>Nat Rev Microbiol</em> 6 (6): 466–76. doi:<a href="https://doi.org/10.1038/nrmicro1900">10.1038/nrmicro1900</a>.</p>
</div>
<div id="ref-kaplan2019">
<p>Kaplan, Mohammed, Debnath Ghosal, Poorna Subramanian, Catherine M Oikonomou, Andreas Kjaer, Sahand Pirbadian, Davi R Ortega, Ariane Briegel, Mohamed Y El-Naggar, and Grant J Jensen. 2019. “The Presence and Absence of Periplasmic Rings in Bacterial Flagellar Motors Correlates with Stator Type.” Edited by Edward H Egelman, Gisela Storz, Edward H Egelman, Masahiro Ito, and Henning Stahlberg. <em>eLife</em> 8 (January). eLife Sciences Publications, Ltd: e43487. doi:<a href="https://doi.org/10.7554/eLife.43487">10.7554/eLife.43487</a>.</p>
</div>
<div id="ref-munoz-dorado2016">
<p>Muñoz-Dorado, José, Francisco J. Marcos-Torres, Elena García-Bravo, Aurelio Moraleda-Muñoz, and Juana Pérez. 2016. “Myxobacteria: Moving, Killing, Feeding, and Surviving Together.” <em>Front Microbiol</em> 7: 781. doi:<a href="https://doi.org/10.3389/fmicb.2016.00781">10.3389/fmicb.2016.00781</a>.</p>
</div>
<div id="ref-murphy2006">
<p>Murphy, G. E., J. R. Leadbetter, and G. J. Jensen. 2006. “In Situ Structure of the Complete Treponema Primitia Flagellar Motor.” <em>Nature</em> 442 (7106): 1062–4. doi:<a href="https://doi.org/10.1038/nature05015">10.1038/nature05015</a>.</p>
</div>
<div id="ref-poweleit2016">
<p>Poweleit, Nicole, Peng Ge, Hong H. Nguyen, Rachel R. Ogorzalek Loo, Robert P. Gunsalus, and Z. Hong Zhou. 2016. “CryoEM Structure of the Methanospirillum Hungatei Archaellum Reveals Structural Features Distinct from the Bacterial Flagellum and Type IV Pilus.” <em>Nature Microbiology</em> 2 (3). Nature Publishing Group: 1–12. doi:<a href="https://doi.org/10.1038/nmicrobiol.2016.222">10.1038/nmicrobiol.2016.222</a>.</p>
</div>
<div id="ref-qin2017">
<p>Qin, Zhuan, Wei-ting Lin, Shiwei Zhu, Aime T. Franco, and Jun Liu. 2017. “Imaging the Motility and Chemotaxis Machineries in Helicobacter Pylori by Cryo-Electron Tomography.” <em>Journal of Bacteriology</em> 199 (3). American Society for Microbiology Journals. doi:<a href="https://doi.org/10.1128/JB.00695-16">10.1128/JB.00695-16</a>.</p>
</div>
<div id="ref-shrivastava2015">
<p>Shrivastava, Abhishek, and Howard C Berg. 2015. “Towards a Model for Flavobacterium Gliding.” <em>Current Opinion in Microbiology</em>, Growth and development: Eukaryotes and prokaryotes, 28 (December): 93–97. doi:<a href="https://doi.org/10.1016/j.mib.2015.07.018">10.1016/j.mib.2015.07.018</a>.</p>
</div>
<div id="ref-wang2017">
<p>Wang, F., A. M. Burrage, S. Postel, R. E. Clark, A. Orlova, E. J. Sundberg, D. B. Kearns, and E. H. Egelman. 2017. “A Structural Model of Flagellar Filament Switching Across Multiple Bacterial Species.” <em>Nat Commun</em> 8 (1): 960. doi:<a href="https://doi.org/10.1038/s41467-017-01075-5">10.1038/s41467-017-01075-5</a>.</p>
</div>
<div id="ref-zhao2014">
<p>Zhao, X., S. J. Norris, and J. Liu. 2014. “Molecular Architecture of the Bacterial Flagellar Motor in Cells.” <em>Biochemistry</em> 53 (27): 4323–33. doi:<a href="https://doi.org/10.1021/bi500059y">10.1021/bi500059y</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="division.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="navigation.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": null,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/caltechlibrary/cell-atlas/edit/master/06_Motility.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["cell-atlas.pdf", "cell-atlas.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
