<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 Navigation | Cell Atlas</title>
  <meta name="description" content="An atlas produced by the Jensen Lab." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 Navigation | Cell Atlas" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="splash/HeaderImage.jpg" />
  <meta property="og:description" content="An atlas produced by the Jensen Lab." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 Navigation | Cell Atlas" />
  
  <meta name="twitter:description" content="An atlas produced by the Jensen Lab." />
  <meta name="twitter:image" content="splash/HeaderImage.jpg" />

<meta name="author" content="Jensen Lab" />


<meta name="date" content="2020-08-03" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="motility.html"/>
<link rel="next" href="lifecycle.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/CL-0.1.2/CL.js"></script>
<script src="libs/doivideo-binding-0.0.0.9000/doivideo.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Cell Atlas</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>1</b> Methods</a><ul>
<li class="chapter" data-level="1.1" data-path="methods.html"><a href="methods.html#light-microscopy"><i class="fa fa-check"></i><b>1.1</b> Light Microscopy</a><ul>
<li class="chapter" data-level="1.1.1" data-path="methods.html"><a href="methods.html#Structural_biology_toolkit"><i class="fa fa-check"></i><b>1.1.1</b> Structural biology toolkit</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="methods.html"><a href="methods.html#fluorescence-light-microscopy"><i class="fa fa-check"></i><b>1.2</b> Fluorescence Light Microscopy</a></li>
<li class="chapter" data-level="1.3" data-path="methods.html"><a href="methods.html#scanning-electron-microscopy"><i class="fa fa-check"></i><b>1.3</b> Scanning Electron Microscopy</a><ul>
<li class="chapter" data-level="1.3.1" data-path="methods.html"><a href="methods.html#Electron_microscopy_modes"><i class="fa fa-check"></i><b>1.3.1</b> Electron microscopy modes</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="methods.html"><a href="methods.html#transmission-electron-microscopy"><i class="fa fa-check"></i><b>1.4</b> Transmission Electron Microscopy</a></li>
<li class="chapter" data-level="1.5" data-path="methods.html"><a href="methods.html#cryogenic-electron-microscopy"><i class="fa fa-check"></i><b>1.5</b> Cryogenic Electron Microscopy</a></li>
<li class="chapter" data-level="1.6" data-path="methods.html"><a href="methods.html#electron-cryotomography"><i class="fa fa-check"></i><b>1.6</b> Electron Cryotomography</a><ul>
<li class="chapter" data-level="1.6.1" data-path="methods.html"><a href="methods.html#Missing_wedge"><i class="fa fa-check"></i><b>1.6.1</b> Missing wedge</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="methods.html"><a href="methods.html#sub-tomogram-averaging"><i class="fa fa-check"></i><b>1.7</b> Sub-Tomogram Averaging</a></li>
<li class="chapter" data-level="1.8" data-path="methods.html"><a href="methods.html#single-particle-reconstruction"><i class="fa fa-check"></i><b>1.8</b> Single Particle Reconstruction</a></li>
<li class="chapter" data-level="1.9" data-path="methods.html"><a href="methods.html#x-ray-crystallography"><i class="fa fa-check"></i><b>1.9</b> X-Ray Crystallography</a><ul>
<li class="chapter" data-level="1.9.1" data-path="methods.html"><a href="methods.html#Structural_biology_timeline"><i class="fa fa-check"></i><b>1.9.1</b> Structural biology timeline</a></li>
</ul></li>
<li class="chapter" data-level="1.10" data-path="methods.html"><a href="methods.html#putting-it-all-together"><i class="fa fa-check"></i><b>1.10</b> Putting It All Together</a></li>
<li class="chapter" data-level="1.11" data-path="methods.html"><a href="methods.html#moredifference-mapping"><i class="fa fa-check"></i><b>1.11</b> More–Difference mapping</a></li>
<li class="chapter" data-level="1.12" data-path="methods.html"><a href="methods.html#further-reading"><i class="fa fa-check"></i><b>1.12</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="envelope.html"><a href="envelope.html"><i class="fa fa-check"></i><b>2</b> Envelope</a><ul>
<li class="chapter" data-level="2.1" data-path="envelope.html"><a href="envelope.html#membrane"><i class="fa fa-check"></i><b>2.1</b> Membrane</a><ul>
<li class="chapter" data-level="2.1.1" data-path="envelope.html"><a href="envelope.html#Lipid_bilayer"><i class="fa fa-check"></i><b>2.1.1</b> Lipid bilayer</a></li>
<li class="chapter" data-level="2.1.2" data-path="envelope.html"><a href="envelope.html#PDB:_5T4O_ATP_synthase_and_energy_production"><i class="fa fa-check"></i><b>2.1.2</b> PDB: 5T4O ATP synthase and energy production</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="envelope.html"><a href="envelope.html#cell-wall"><i class="fa fa-check"></i><b>2.2</b> Cell Wall</a></li>
<li class="chapter" data-level="2.3" data-path="envelope.html"><a href="envelope.html#morepeptidoglycan-architecture"><i class="fa fa-check"></i><b>2.3</b> More–Peptidoglycan architecture</a></li>
<li class="chapter" data-level="2.4" data-path="envelope.html"><a href="envelope.html#moremethanobacterium-formicicum"><i class="fa fa-check"></i><b>2.4</b> More–Methanobacterium formicicum</a></li>
<li class="chapter" data-level="2.5" data-path="envelope.html"><a href="envelope.html#outer-membrane"><i class="fa fa-check"></i><b>2.5</b> Outer Membrane</a><ul>
<li class="chapter" data-level="2.5.1" data-path="envelope.html"><a href="envelope.html#pdb-1eq7-brauns-lipoprotein-pdb1eq7brauns_lipoprotein"><i class="fa fa-check"></i><b>2.5.1</b> PDB: 1EQ7 –Braun’s lipoprotein {#PDB:<em>1EQ7</em>–Brauns_lipoprotein}</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="envelope.html"><a href="envelope.html#morediderm-sacculus-architecture"><i class="fa fa-check"></i><b>2.6</b> More–Diderm sacculus architecture</a><ul>
<li class="chapter" data-level="2.6.1" data-path="envelope.html"><a href="envelope.html#Sacculus_remodeling"><i class="fa fa-check"></i><b>2.6.1</b> Sacculus remodeling</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="envelope.html"><a href="envelope.html#vesicles"><i class="fa fa-check"></i><b>2.7</b> Vesicles</a></li>
<li class="chapter" data-level="2.8" data-path="envelope.html"><a href="envelope.html#morepearled-vesicles"><i class="fa fa-check"></i><b>2.8</b> More–Pearled vesicles</a></li>
<li class="chapter" data-level="2.9" data-path="envelope.html"><a href="envelope.html#moretubular-vesicles"><i class="fa fa-check"></i><b>2.9</b> More–Tubular vesicles</a></li>
<li class="chapter" data-level="2.10" data-path="envelope.html"><a href="envelope.html#morecytoplasmic-vesicles"><i class="fa fa-check"></i><b>2.10</b> More–Cytoplasmic vesicles</a></li>
<li class="chapter" data-level="2.11" data-path="envelope.html"><a href="envelope.html#more-cytoplasmic-vesicle-variety"><i class="fa fa-check"></i><b>2.11</b> More: Cytoplasmic vesicle variety</a></li>
<li class="chapter" data-level="2.12" data-path="envelope.html"><a href="envelope.html#morearchaeal-vesicles"><i class="fa fa-check"></i><b>2.12</b> More–Archaeal vesicles</a></li>
<li class="chapter" data-level="2.13" data-path="envelope.html"><a href="envelope.html#variation"><i class="fa fa-check"></i><b>2.13</b> Variation</a></li>
<li class="chapter" data-level="2.14" data-path="envelope.html"><a href="envelope.html#surface-layer"><i class="fa fa-check"></i><b>2.14</b> Surface Layer</a><ul>
<li class="chapter" data-level="2.14.1" data-path="envelope.html"><a href="envelope.html#PDB:_5N97_Surface_layer_architecture"><i class="fa fa-check"></i><b>2.14.1</b> PDB: 5N97 Surface layer architecture</a></li>
</ul></li>
<li class="chapter" data-level="2.15" data-path="envelope.html"><a href="envelope.html#morenanopods"><i class="fa fa-check"></i><b>2.15</b> More–Nanopods</a></li>
<li class="chapter" data-level="2.16" data-path="envelope.html"><a href="envelope.html#surface-layer-variety"><i class="fa fa-check"></i><b>2.16</b> Surface Layer Variety</a></li>
<li class="chapter" data-level="2.17" data-path="envelope.html"><a href="envelope.html#morem.-alcaliphilum"><i class="fa fa-check"></i><b>2.17</b> More–M. alcaliphilum</a></li>
<li class="chapter" data-level="2.18" data-path="envelope.html"><a href="envelope.html#moreclostridium-thermocellum"><i class="fa fa-check"></i><b>2.18</b> More–Clostridium thermocellum</a></li>
<li class="chapter" data-level="2.19" data-path="envelope.html"><a href="envelope.html#moren.-maritimus"><i class="fa fa-check"></i><b>2.19</b> More–N. maritimus</a></li>
<li class="chapter" data-level="2.20" data-path="envelope.html"><a href="envelope.html#moremethanoregula-formicica"><i class="fa fa-check"></i><b>2.20</b> More–Methanoregula formicica</a></li>
<li class="chapter" data-level="2.21" data-path="envelope.html"><a href="envelope.html#sheath"><i class="fa fa-check"></i><b>2.21</b> Sheath</a></li>
<li class="chapter" data-level="2.22" data-path="envelope.html"><a href="envelope.html#dna"><i class="fa fa-check"></i><b>2.22</b> DNA</a></li>
<li class="chapter" data-level="2.23" data-path="envelope.html"><a href="envelope.html#nucleoid"><i class="fa fa-check"></i><b>2.23</b> Nucleoid</a></li>
<li class="chapter" data-level="2.24" data-path="envelope.html"><a href="envelope.html#further-reading-1"><i class="fa fa-check"></i><b>2.24</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="shape.html"><a href="shape.html"><i class="fa fa-check"></i><b>3</b> Shape</a><ul>
<li class="chapter" data-level="3.1" data-path="shape.html"><a href="shape.html#coccoid"><i class="fa fa-check"></i><b>3.1</b> Coccoid</a></li>
<li class="chapter" data-level="3.2" data-path="shape.html"><a href="shape.html#rod"><i class="fa fa-check"></i><b>3.2</b> Rod</a></li>
<li class="chapter" data-level="3.3" data-path="shape.html"><a href="shape.html#morerod-variety"><i class="fa fa-check"></i><b>3.3</b> More–Rod variety</a></li>
<li class="chapter" data-level="3.4" data-path="shape.html"><a href="shape.html#morearchaeal-rods"><i class="fa fa-check"></i><b>3.4</b> More–Archaeal rods</a></li>
<li class="chapter" data-level="3.5" data-path="shape.html"><a href="shape.html#length"><i class="fa fa-check"></i><b>3.5</b> Length</a></li>
<li class="chapter" data-level="3.6" data-path="shape.html"><a href="shape.html#vibrioid"><i class="fa fa-check"></i><b>3.6</b> Vibrioid</a><ul>
<li class="chapter" data-level="3.6.1" data-path="shape.html"><a href="shape.html#PDB:_5U3C_CTP_synthase"><i class="fa fa-check"></i><b>3.6.1</b> PDB: 5U3C CTP synthase</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="shape.html"><a href="shape.html#helical"><i class="fa fa-check"></i><b>3.7</b> Helical</a></li>
<li class="chapter" data-level="3.8" data-path="shape.html"><a href="shape.html#moreborrelia-burgdorferi"><i class="fa fa-check"></i><b>3.8</b> More–Borrelia burgdorferi</a></li>
<li class="chapter" data-level="3.9" data-path="shape.html"><a href="shape.html#prosthecate"><i class="fa fa-check"></i><b>3.9</b> Prosthecate</a><ul>
<li class="chapter" data-level="3.9.1" data-path="shape.html"><a href="shape.html#PDB:_6RIB_Bactofilin"><i class="fa fa-check"></i><b>3.9.1</b> PDB: 6RIB Bactofilin</a></li>
</ul></li>
<li class="chapter" data-level="3.10" data-path="shape.html"><a href="shape.html#morebacterial-microtubules"><i class="fa fa-check"></i><b>3.10</b> More–Bacterial microtubules</a></li>
<li class="chapter" data-level="3.11" data-path="shape.html"><a href="shape.html#square"><i class="fa fa-check"></i><b>3.11</b> Square</a></li>
<li class="chapter" data-level="3.12" data-path="shape.html"><a href="shape.html#moregas-vesicles"><i class="fa fa-check"></i><b>3.12</b> More–Gas vesicles</a></li>
<li class="chapter" data-level="3.13" data-path="shape.html"><a href="shape.html#further-reading-2"><i class="fa fa-check"></i><b>3.13</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="growth.html"><a href="growth.html"><i class="fa fa-check"></i><b>4</b> Growth</a><ul>
<li class="chapter" data-level="4.1" data-path="growth.html"><a href="growth.html#diffusion"><i class="fa fa-check"></i><b>4.1</b> Diffusion</a></li>
<li class="chapter" data-level="4.2" data-path="growth.html"><a href="growth.html#anaerobic-respiration"><i class="fa fa-check"></i><b>4.2</b> Anaerobic Respiration</a></li>
<li class="chapter" data-level="4.3" data-path="growth.html"><a href="growth.html#photosynthesis"><i class="fa fa-check"></i><b>4.3</b> Photosynthesis</a><ul>
<li class="chapter" data-level="4.3.1" data-path="growth.html"><a href="growth.html#pdb-6kgx-photocomplexes-pdb6kgxphotocomplexes"><i class="fa fa-check"></i><b>4.3.1</b> PDB: 6KGX –Photocomplexes {#PDB:<em>6KGX</em>–Photocomplexes}</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="growth.html"><a href="growth.html#moreicm-variety"><i class="fa fa-check"></i><b>4.4</b> More–ICM variety</a></li>
<li class="chapter" data-level="4.5" data-path="growth.html"><a href="growth.html#photosynthesis-contd."><i class="fa fa-check"></i><b>4.5</b> Photosynthesis (cont’d.)</a></li>
<li class="chapter" data-level="4.6" data-path="growth.html"><a href="growth.html#enzyme-regulation"><i class="fa fa-check"></i><b>4.6</b> Enzyme Regulation</a><ul>
<li class="chapter" data-level="4.6.1" data-path="growth.html"><a href="growth.html#pdb-6tqhadhe-spirosome-structure-pdb_6tqhadhe_spirosome_structure"><i class="fa fa-check"></i><b>4.6.1</b> PDB: 6TQH–AdhE spirosome structure {#PDB:_6TQH–AdhE_spirosome_structure} <img src="img/schematics/4_5_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="growth.html"><a href="growth.html#microcompartments"><i class="fa fa-check"></i><b>4.7</b> Microcompartments</a><ul>
<li class="chapter" data-level="4.7.1" data-path="growth.html"><a href="growth.html#pdb-3dktmicrocompartment-shells-pdb_3dktmicrocompartment_shells"><i class="fa fa-check"></i><b>4.7.1</b> PDB: 3DKT–Microcompartment shells {#PDB:_3DKT–Microcompartment_shells} <img src="img/schematics/4_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="growth.html"><a href="growth.html#carboxysomes"><i class="fa fa-check"></i><b>4.8</b> Carboxysomes</a></li>
<li class="chapter" data-level="4.9" data-path="growth.html"><a href="growth.html#morecarboxysome-variety"><i class="fa fa-check"></i><b>4.9</b> More–Carboxysome variety</a></li>
<li class="chapter" data-level="4.10" data-path="growth.html"><a href="growth.html#morelong-carboxysomes"><i class="fa fa-check"></i><b>4.10</b> More–Long carboxysomes</a></li>
<li class="chapter" data-level="4.11" data-path="growth.html"><a href="growth.html#storage"><i class="fa fa-check"></i><b>4.11</b> Storage</a></li>
<li class="chapter" data-level="4.12" data-path="growth.html"><a href="growth.html#morestorage-granule-growth"><i class="fa fa-check"></i><b>4.12</b> More–Storage granule growth</a></li>
<li class="chapter" data-level="4.13" data-path="growth.html"><a href="growth.html#storage-contd."><i class="fa fa-check"></i><b>4.13</b> Storage (cont’d.)</a></li>
<li class="chapter" data-level="4.14" data-path="growth.html"><a href="growth.html#moregranule-containing-carboxysomes"><i class="fa fa-check"></i><b>4.14</b> More–Granule-containing carboxysomes</a></li>
<li class="chapter" data-level="4.15" data-path="growth.html"><a href="growth.html#morecarboxysomes-granule-connections"><i class="fa fa-check"></i><b>4.15</b> More–Carboxysomes-granule connections</a></li>
<li class="chapter" data-level="4.16" data-path="growth.html"><a href="growth.html#archaeal-storage-granules"><i class="fa fa-check"></i><b>4.16</b> Archaeal Storage Granules</a></li>
<li class="chapter" data-level="4.17" data-path="growth.html"><a href="growth.html#morehaloferax-gibbonsii-granules"><i class="fa fa-check"></i><b>4.17</b> More–Haloferax gibbonsii granules</a></li>
<li class="chapter" data-level="4.18" data-path="growth.html"><a href="growth.html#morehalohasta-litchfieldiae-granules"><i class="fa fa-check"></i><b>4.18</b> More–Halohasta litchfieldiae granules</a></li>
<li class="chapter" data-level="4.19" data-path="growth.html"><a href="growth.html#further-reading-3"><i class="fa fa-check"></i><b>4.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="division.html"><a href="division.html"><i class="fa fa-check"></i><b>5</b> Division</a><ul>
<li class="chapter" data-level="5.1" data-path="division.html"><a href="division.html#segregation"><i class="fa fa-check"></i><b>5.1</b> Segregation</a></li>
<li class="chapter" data-level="5.2" data-path="division.html"><a href="division.html#chromosome-segregation"><i class="fa fa-check"></i><b>5.2</b> Chromosome Segregation</a></li>
<li class="chapter" data-level="5.3" data-path="division.html"><a href="division.html#budding"><i class="fa fa-check"></i><b>5.3</b> Budding</a></li>
<li class="chapter" data-level="5.4" data-path="division.html"><a href="division.html#morehyphomonas-stalk"><i class="fa fa-check"></i><b>5.4</b> More–Hyphomonas stalk</a></li>
<li class="chapter" data-level="5.5" data-path="division.html"><a href="division.html#morehyphomonas-lifecycle"><i class="fa fa-check"></i><b>5.5</b> More–Hyphomonas lifecycle</a></li>
<li class="chapter" data-level="5.6" data-path="division.html"><a href="division.html#monoderm-cytokinesis"><i class="fa fa-check"></i><b>5.6</b> Monoderm Cytokinesis</a></li>
<li class="chapter" data-level="5.7" data-path="division.html"><a href="division.html#monoderm-cytokinesis-contd."><i class="fa fa-check"></i><b>5.7</b> Monoderm Cytokinesis (cont’d.)</a></li>
<li class="chapter" data-level="5.8" data-path="division.html"><a href="division.html#diderm-cytokinesis"><i class="fa fa-check"></i><b>5.8</b> Diderm Cytokinesis</a></li>
<li class="chapter" data-level="5.9" data-path="division.html"><a href="division.html#moresphingopyxis-alaskensis-division"><i class="fa fa-check"></i><b>5.9</b> More–Sphingopyxis alaskensis division</a></li>
<li class="chapter" data-level="5.10" data-path="division.html"><a href="division.html#morechloroflexi-division"><i class="fa fa-check"></i><b>5.10</b> More–Chloroflexi division</a></li>
<li class="chapter" data-level="5.11" data-path="division.html"><a href="division.html#diderm-cytokinesis-contd."><i class="fa fa-check"></i><b>5.11</b> Diderm Cytokinesis (cont’d.)</a></li>
<li class="chapter" data-level="5.12" data-path="division.html"><a href="division.html#morecaulobacter-division"><i class="fa fa-check"></i><b>5.12</b> More–Caulobacter division</a></li>
<li class="chapter" data-level="5.13" data-path="division.html"><a href="division.html#budding-contd."><i class="fa fa-check"></i><b>5.13</b> Budding (cont’d.)</a></li>
<li class="chapter" data-level="5.14" data-path="division.html"><a href="division.html#ftsz"><i class="fa fa-check"></i><b>5.14</b> FtsZ</a></li>
<li class="chapter" data-level="5.15" data-path="division.html"><a href="division.html#moreasymmetric-constriction"><i class="fa fa-check"></i><b>5.15</b> More–Asymmetric constriction</a><ul>
<li class="chapter" data-level="5.15.1" data-path="division.html"><a href="division.html#pdb-5mn4-ftsz-structure-pdb5mn4ftsz_structure"><i class="fa fa-check"></i><b>5.15.1</b> PDB: 5MN4 –FtsZ structure {#PDB:<em>5MN4</em>–FtsZ_structure}</a></li>
</ul></li>
<li class="chapter" data-level="5.16" data-path="division.html"><a href="division.html#ftsz-contd."><i class="fa fa-check"></i><b>5.16</b> FtsZ (cont’d.)</a></li>
<li class="chapter" data-level="5.17" data-path="division.html"><a href="division.html#escrt"><i class="fa fa-check"></i><b>5.17</b> ESCRT</a></li>
<li class="chapter" data-level="5.18" data-path="division.html"><a href="division.html#escrt-contd."><i class="fa fa-check"></i><b>5.18</b> ESCRT (cont’d.)</a></li>
<li class="chapter" data-level="5.19" data-path="division.html"><a href="division.html#further-reading-4"><i class="fa fa-check"></i><b>5.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="motility.html"><a href="motility.html"><i class="fa fa-check"></i><b>6</b> Motility</a><ul>
<li class="chapter" data-level="6.1" data-path="motility.html"><a href="motility.html#flagellum"><i class="fa fa-check"></i><b>6.1</b> Flagellum</a><ul>
<li class="chapter" data-level="6.1.1" data-path="motility.html"><a href="motility.html#pdb-5wjtflagellum-structure-pdb_5wjtflagellum_structure"><i class="fa fa-check"></i><b>6.1.1</b> PDB: 5WJT–Flagellum structure {#PDB:_5WJT–Flagellum_structure} <img src="img/schematics/6_1_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="motility.html"><a href="motility.html#flagellar-motor"><i class="fa fa-check"></i><b>6.2</b> Flagellar Motor</a></li>
<li class="chapter" data-level="6.3" data-path="motility.html"><a href="motility.html#moreflagellar-motor-structure"><i class="fa fa-check"></i><b>6.3</b> More–Flagellar motor structure</a><ul>
<li class="chapter" data-level="6.3.1" data-path="motility.html"><a href="motility.html#Flagellar_motor_diversity"><i class="fa fa-check"></i><b>6.3.1</b> Flagellar motor diversity</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="motility.html"><a href="motility.html#flagellar-assembly"><i class="fa fa-check"></i><b>6.4</b> Flagellar Assembly</a></li>
<li class="chapter" data-level="6.5" data-path="motility.html"><a href="motility.html#moreflagellar-motor-disassembly"><i class="fa fa-check"></i><b>6.5</b> More–Flagellar motor disassembly</a></li>
<li class="chapter" data-level="6.6" data-path="motility.html"><a href="motility.html#flagella-patterns"><i class="fa fa-check"></i><b>6.6</b> Flagella Patterns</a></li>
<li class="chapter" data-level="6.7" data-path="motility.html"><a href="motility.html#morelophotrichous-bacteria"><i class="fa fa-check"></i><b>6.7</b> More–Lophotrichous bacteria</a></li>
<li class="chapter" data-level="6.8" data-path="motility.html"><a href="motility.html#flagella-patterns-contd."><i class="fa fa-check"></i><b>6.8</b> Flagella Patterns (cont’d.)</a></li>
<li class="chapter" data-level="6.9" data-path="motility.html"><a href="motility.html#moreproteus-mirabilis-flagella"><i class="fa fa-check"></i><b>6.9</b> More–Proteus mirabilis flagella</a></li>
<li class="chapter" data-level="6.10" data-path="motility.html"><a href="motility.html#flagellar-sheath"><i class="fa fa-check"></i><b>6.10</b> Flagellar Sheath</a></li>
<li class="chapter" data-level="6.11" data-path="motility.html"><a href="motility.html#periplasmic-flagella"><i class="fa fa-check"></i><b>6.11</b> Periplasmic Flagella</a></li>
<li class="chapter" data-level="6.12" data-path="motility.html"><a href="motility.html#moretreponema-primitia"><i class="fa fa-check"></i><b>6.12</b> More–Treponema primitia</a></li>
<li class="chapter" data-level="6.13" data-path="motility.html"><a href="motility.html#archaellum"><i class="fa fa-check"></i><b>6.13</b> Archaellum</a><ul>
<li class="chapter" data-level="6.13.1" data-path="motility.html"><a href="motility.html#pdb-5tfyarchaellum-structure-pdb_5tfyarchaellum_structure"><i class="fa fa-check"></i><b>6.13.1</b> PDB: 5TFY–Archaellum structure {#PDB:_5TFY–Archaellum_structure} <img src="img/schematics/6_8_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="6.14" data-path="motility.html"><a href="motility.html#archaella-patterns"><i class="fa fa-check"></i><b>6.14</b> Archaella Patterns</a></li>
<li class="chapter" data-level="6.15" data-path="motility.html"><a href="motility.html#morecone-structure"><i class="fa fa-check"></i><b>6.15</b> More–Cone structure</a></li>
<li class="chapter" data-level="6.16" data-path="motility.html"><a href="motility.html#moreorganizing-center"><i class="fa fa-check"></i><b>6.16</b> More–Organizing center</a></li>
<li class="chapter" data-level="6.17" data-path="motility.html"><a href="motility.html#type-iv-pilus"><i class="fa fa-check"></i><b>6.17</b> Type IV Pilus</a><ul>
<li class="chapter" data-level="6.17.1" data-path="motility.html"><a href="motility.html#Type_IV_pilus_structure"><i class="fa fa-check"></i><b>6.17.1</b> Type IV pilus structure</a></li>
</ul></li>
<li class="chapter" data-level="6.18" data-path="motility.html"><a href="motility.html#type-ix-secretion-system"><i class="fa fa-check"></i><b>6.18</b> Type IX Secretion System</a></li>
<li class="chapter" data-level="6.19" data-path="motility.html"><a href="motility.html#terminal-organelle"><i class="fa fa-check"></i><b>6.19</b> Terminal Organelle</a></li>
<li class="chapter" data-level="6.20" data-path="motility.html"><a href="motility.html#further-reading-5"><i class="fa fa-check"></i><b>6.20</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="navigation.html"><a href="navigation.html"><i class="fa fa-check"></i><b>7</b> Navigation</a><ul>
<li class="chapter" data-level="7.1" data-path="navigation.html"><a href="navigation.html#chemotaxis"><i class="fa fa-check"></i><b>7.1</b> Chemotaxis</a><ul>
<li class="chapter" data-level="7.1.1" data-path="navigation.html"><a href="navigation.html#pdb-2ch7chemoreceptor-structure-pdb_2ch7chemoreceptor_structure"><i class="fa fa-check"></i><b>7.1.1</b> PDB: 2CH7–Chemoreceptor structure {#PDB:_2CH7–Chemoreceptor_structure} <img src="img/schematics/7_1_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="navigation.html"><a href="navigation.html#chemosensory-array"><i class="fa fa-check"></i><b>7.2</b> Chemosensory Array</a><ul>
<li class="chapter" data-level="7.2.1" data-path="navigation.html"><a href="navigation.html#PDB:_6S1K_Chemosensory_array_architecture"><i class="fa fa-check"></i><b>7.2.1</b> PDB: 6S1K Chemosensory array architecture</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="navigation.html"><a href="navigation.html#chemosensory-array-conservation"><i class="fa fa-check"></i><b>7.3</b> Chemosensory Array Conservation</a></li>
<li class="chapter" data-level="7.4" data-path="navigation.html"><a href="navigation.html#chemoreceptors"><i class="fa fa-check"></i><b>7.4</b> Chemoreceptors</a></li>
<li class="chapter" data-level="7.5" data-path="navigation.html"><a href="navigation.html#moreazospirillum-brasilense-aerotaxis"><i class="fa fa-check"></i><b>7.5</b> More–Azospirillum brasilense aerotaxis</a></li>
<li class="chapter" data-level="7.6" data-path="navigation.html"><a href="navigation.html#cytoplasmic-chemosensory-array"><i class="fa fa-check"></i><b>7.6</b> Cytoplasmic Chemosensory Array</a></li>
<li class="chapter" data-level="7.7" data-path="navigation.html"><a href="navigation.html#morevibrio-cholerae-chemosensory-arrays"><i class="fa fa-check"></i><b>7.7</b> More–Vibrio cholerae chemosensory arrays</a></li>
<li class="chapter" data-level="7.8" data-path="navigation.html"><a href="navigation.html#magnetotaxis"><i class="fa fa-check"></i><b>7.8</b> Magnetotaxis</a><ul>
<li class="chapter" data-level="7.8.1" data-path="navigation.html"><a href="navigation.html#pdb-5jygmamk-structure-pdb_5jygmamk_structure"><i class="fa fa-check"></i><b>7.8.1</b> PDB: 5JYG–MamK structure {#PDB:_5JYG–MamK_structure} <img src="img/schematics/7_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="navigation.html"><a href="navigation.html#further-reading-6"><i class="fa fa-check"></i><b>7.9</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="lifecycle.html"><a href="lifecycle.html"><i class="fa fa-check"></i><b>8</b> Lifecycle</a><ul>
<li class="chapter" data-level="8.1" data-path="lifecycle.html"><a href="lifecycle.html#stationary-phase"><i class="fa fa-check"></i><b>8.1</b> Stationary Phase</a></li>
<li class="chapter" data-level="8.2" data-path="lifecycle.html"><a href="lifecycle.html#genome-protection"><i class="fa fa-check"></i><b>8.2</b> Genome Protection</a></li>
<li class="chapter" data-level="8.3" data-path="lifecycle.html"><a href="lifecycle.html#stationary-phase-contd."><i class="fa fa-check"></i><b>8.3</b> Stationary Phase (cont’d.)</a></li>
<li class="chapter" data-level="8.4" data-path="lifecycle.html"><a href="lifecycle.html#differentiation"><i class="fa fa-check"></i><b>8.4</b> Differentiation</a></li>
<li class="chapter" data-level="8.5" data-path="lifecycle.html"><a href="lifecycle.html#moreholdfast"><i class="fa fa-check"></i><b>8.5</b> More–Holdfast</a></li>
<li class="chapter" data-level="8.6" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-sporulation"><i class="fa fa-check"></i><b>8.6</b> Monoderm Sporulation</a></li>
<li class="chapter" data-level="8.7" data-path="lifecycle.html"><a href="lifecycle.html#moresporulation-septum-formation"><i class="fa fa-check"></i><b>8.7</b> More–Sporulation septum formation</a></li>
<li class="chapter" data-level="8.8" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-sporulation-contd."><i class="fa fa-check"></i><b>8.8</b> Monoderm Sporulation (cont’d.)</a></li>
<li class="chapter" data-level="8.9" data-path="lifecycle.html"><a href="lifecycle.html#morespore-engulfment"><i class="fa fa-check"></i><b>8.9</b> More–Spore engulfment</a></li>
<li class="chapter" data-level="8.10" data-path="lifecycle.html"><a href="lifecycle.html#morespore-coating"><i class="fa fa-check"></i><b>8.10</b> More–Spore coating</a></li>
<li class="chapter" data-level="8.11" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-spores"><i class="fa fa-check"></i><b>8.11</b> Monoderm Spores</a></li>
<li class="chapter" data-level="8.12" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-germination"><i class="fa fa-check"></i><b>8.12</b> Monoderm Germination</a></li>
<li class="chapter" data-level="8.13" data-path="lifecycle.html"><a href="lifecycle.html#diderm-sporulation"><i class="fa fa-check"></i><b>8.13</b> Diderm Sporulation</a></li>
<li class="chapter" data-level="8.14" data-path="lifecycle.html"><a href="lifecycle.html#diderm-sporulation-contd."><i class="fa fa-check"></i><b>8.14</b> Diderm Sporulation (cont’d.)</a></li>
<li class="chapter" data-level="8.15" data-path="lifecycle.html"><a href="lifecycle.html#morecomplete-engulfment"><i class="fa fa-check"></i><b>8.15</b> More–Complete engulfment</a></li>
<li class="chapter" data-level="8.16" data-path="lifecycle.html"><a href="lifecycle.html#moremature-spore"><i class="fa fa-check"></i><b>8.16</b> More–Mature spore</a></li>
<li class="chapter" data-level="8.17" data-path="lifecycle.html"><a href="lifecycle.html#moresporomusa-acidovorans-sporulation"><i class="fa fa-check"></i><b>8.17</b> More–Sporomusa acidovorans sporulation</a></li>
<li class="chapter" data-level="8.18" data-path="lifecycle.html"><a href="lifecycle.html#diderm-germination"><i class="fa fa-check"></i><b>8.18</b> Diderm Germination</a></li>
<li class="chapter" data-level="8.19" data-path="lifecycle.html"><a href="lifecycle.html#further-reading-7"><i class="fa fa-check"></i><b>8.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="interaction.html"><a href="interaction.html"><i class="fa fa-check"></i><b>9</b> Interaction</a><ul>
<li class="chapter" data-level="9.1" data-path="interaction.html"><a href="interaction.html#biofilm"><i class="fa fa-check"></i><b>9.1</b> Biofilm</a></li>
<li class="chapter" data-level="9.2" data-path="interaction.html"><a href="interaction.html#biofilm-contd."><i class="fa fa-check"></i><b>9.2</b> Biofilm (cont’d.)</a></li>
<li class="chapter" data-level="9.3" data-path="interaction.html"><a href="interaction.html#type-ii-and-type-iv-secretion"><i class="fa fa-check"></i><b>9.3</b> Type II and Type IV Secretion</a><ul>
<li class="chapter" data-level="9.3.1" data-path="interaction.html"><a href="interaction.html#emd-20712-emd-20713legionella-t2ss-structure-emd-20712_emd-20713legionella_t2ss_structure"><i class="fa fa-check"></i><b>9.3.1</b> EMD-20712 EMD-20713–Legionella T2SS structure {#EMD-20712_EMD-20713–Legionella_T2SS_structure}</a></li>
<li class="chapter" data-level="9.3.2" data-path="interaction.html"><a href="interaction.html#emd-0566-legionella-t4ss-structure-emd-0566_legionella_t4ss_structure"><i class="fa fa-check"></i><b>9.3.2</b> EMD-0566 –Legionella T4SS structure {#EMD-0566_–Legionella_T4SS_structure}</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="interaction.html"><a href="interaction.html#morehelicobacter-pylori-tubes"><i class="fa fa-check"></i><b>9.4</b> More–Helicobacter pylori tubes</a></li>
<li class="chapter" data-level="9.5" data-path="interaction.html"><a href="interaction.html#type-iii-secretion"><i class="fa fa-check"></i><b>9.5</b> Type III Secretion</a><ul>
<li class="chapter" data-level="9.5.1" data-path="interaction.html"><a href="interaction.html#EMD-20611_Injectisome_structure"><i class="fa fa-check"></i><b>9.5.1</b> EMD-20611 Injectisome structure</a></li>
<li class="chapter" data-level="9.5.2" data-path="interaction.html"><a href="interaction.html#PDB:_5WQ8_Secretin_structure"><i class="fa fa-check"></i><b>9.5.2</b> PDB: 5WQ8 Secretin structure</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="interaction.html"><a href="interaction.html#type-v-secretion"><i class="fa fa-check"></i><b>9.6</b> Type V Secretion</a></li>
<li class="chapter" data-level="9.7" data-path="interaction.html"><a href="interaction.html#type-vi-secretion"><i class="fa fa-check"></i><b>9.7</b> Type VI Secretion</a><ul>
<li class="chapter" data-level="9.7.1" data-path="interaction.html"><a href="interaction.html#pdb-5urwprimed-t6ss-structure-pdb_5urwprimed_t6ss_structure"><i class="fa fa-check"></i><b>9.7.1</b> PDB: 5URW–Primed T6SS structure {#PDB:_5URW–Primed_T6SS_structure} <img src="img/schematics/9_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="9.8" data-path="interaction.html"><a href="interaction.html#type-vi-secretion-contd."><i class="fa fa-check"></i><b>9.8</b> Type VI Secretion (cont’d.)</a><ul>
<li class="chapter" data-level="9.8.1" data-path="interaction.html"><a href="interaction.html#pdb-5urxfired-t6ss-structure-pdb_5urxfired_t6ss_structure"><i class="fa fa-check"></i><b>9.8.1</b> PDB: 5URX–Fired T6SS structure {#PDB:_5URX–Fired_T6SS_structure} <img src="img/schematics/9_7_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="9.9" data-path="interaction.html"><a href="interaction.html#moret6ss-battery"><i class="fa fa-check"></i><b>9.9</b> More–T6SS battery</a></li>
<li class="chapter" data-level="9.10" data-path="interaction.html"><a href="interaction.html#moremacs"><i class="fa fa-check"></i><b>9.10</b> More–MACs</a><ul>
<li class="chapter" data-level="9.10.1" data-path="interaction.html"><a href="interaction.html#MAC_array_structure"><i class="fa fa-check"></i><b>9.10.1</b> MAC array structure</a></li>
</ul></li>
<li class="chapter" data-level="9.11" data-path="interaction.html"><a href="interaction.html#type-ix-secretion"><i class="fa fa-check"></i><b>9.11</b> Type IX Secretion</a></li>
<li class="chapter" data-level="9.12" data-path="interaction.html"><a href="interaction.html#predation"><i class="fa fa-check"></i><b>9.12</b> Predation</a></li>
<li class="chapter" data-level="9.13" data-path="interaction.html"><a href="interaction.html#predation-contd."><i class="fa fa-check"></i><b>9.13</b> Predation (cont’d.)</a></li>
<li class="chapter" data-level="9.14" data-path="interaction.html"><a href="interaction.html#morebdellovibrio-bacteriovorus-entry"><i class="fa fa-check"></i><b>9.14</b> More–Bdellovibrio bacteriovorus entry</a></li>
<li class="chapter" data-level="9.15" data-path="interaction.html"><a href="interaction.html#predation-contd.-1"><i class="fa fa-check"></i><b>9.15</b> Predation (cont’d.)</a></li>
<li class="chapter" data-level="9.16" data-path="interaction.html"><a href="interaction.html#further-reading-8"><i class="fa fa-check"></i><b>9.16</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="viruses.html"><a href="viruses.html"><i class="fa fa-check"></i><b>10</b> Viruses</a><ul>
<li class="chapter" data-level="10.1" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis"><i class="fa fa-check"></i><b>10.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.2" data-path="viruses.html"><a href="viruses.html#bacteroidetes-strain-jt5"><i class="fa fa-check"></i><b>10.2</b> Bacteroidetes strain JT5</a></li>
<li class="chapter" data-level="10.3" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-1"><i class="fa fa-check"></i><b>10.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.4" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae"><i class="fa fa-check"></i><b>10.4</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="10.5" data-path="viruses.html"><a href="viruses.html#escherichia-coli"><i class="fa fa-check"></i><b>10.5</b> Escherichia coli</a></li>
<li class="chapter" data-level="10.6" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus"><i class="fa fa-check"></i><b>10.6</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="10.7" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.7</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="10.8" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus"><i class="fa fa-check"></i><b>10.8</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="10.9" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-1"><i class="fa fa-check"></i><b>10.9</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="10.10" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-2"><i class="fa fa-check"></i><b>10.10</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.11" data-path="viruses.html"><a href="viruses.html#further-reading-9"><i class="fa fa-check"></i><b>10.11</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>11</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.library.caltech.edu" target="blank">Published by Caltech Library</a><li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cell Atlas</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="navigation" class="section level1">
<h1><span class="header-section-number">Chapter 7</span> Navigation</h1>
<p>“<em>E. coli</em> forgets where it is going in about 10 seconds.” - Howard Berg <span class="citation">Berg (<a href="#ref-berg1988">1988</a>)</span></p>
<div id="chemotaxis" class="section level2">
<h2><span class="header-section-number">7.1</span> Chemotaxis</h2>
<p>Now that your cell can move, it needs to figure out which way to go. Perhaps it should take a sniff. Chemosensory systems are ancient (they were already present in the common ancestor of bacteria and archaea) and widespread, reflecting their great utility. Chemosensory systems are two-component signaling systems. The first component is a receptor that binds a specific chemical, such as a sugar or amino acid. The binding results in a conformational change that propagates down the long receptor [Schematic–Chemoreceptor structure], turning off a kinase bound at the other end that controls the state of the second component: a response regulator. These response regulator proteins carry the signal elsewhere in the cell by diffusion.</p>
<p>For chemotaxis (“taxis,” or ordered movement, in response to chemicals) the signal is carried to the cell’s motility machinery, usually the flagellar motor, but also type IV pili. Phosphorylated response regulators bind the flagellar motor, switching the direction of rotation. This produces different results depending on the pattern of flagella on the cell. If there is a single flagellum, as on this <em>Shewanella oneidensis</em> cell, it switches the flagellum between pushing and pulling the cell body, reorienting the cell in the process. In peritrichously-flagellated cells like <em>Escherichia coli</em>, it brings the flagella into and out of a bundle. Remember that bundled flagella drive the cell forward in straight “runs” and dissociated flagella “tumble” the cell to try a new direction. Constant feedback from the chemosensory system switches the balance of phosphorylated/unphosphorylated response regulator and therefore keeps the cell heading in the general direction of an attractant chemical cue, or away from a repellant.</p>
<p>Chemosensory systems form arrays containing many copies of the proteins. As in this <em>S. oneidensis</em> cell, arrays are usually located near the flagella they control, with the tips of the chemoreceptors sticking through the membrane into the periplasm or extracellular space where they can detect signals from the environment. At the other end of the array, the associated kinases interact with the response regulators. These kinases, along with an additional structural protein that helps organize the array, form a layer that we see as a dense line in the cytoplasm.</p>
<div class="figure"><span id="fig:unnamed-chunk-122"></span>
<div id="htmlwidget-41e0db12c0be53b3002a" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-41e0db12c0be53b3002a">{"x":{"doi":"10.22002/D1.1545","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 7.1: Shewanella oneidensis Collected by: Mohammed Kaplan <a href="https://doi.org/10.22002/D1.1545">10.22002/D1.1545</a>
</p>
</div>
<div id="pdb-2ch7chemoreceptor-structure-pdb_2ch7chemoreceptor_structure" class="section level3">
<h3><span class="header-section-number">7.1.1</span> PDB: 2CH7–Chemoreceptor structure {#PDB:_2CH7–Chemoreceptor_structure} <img src="img/schematics/7_1_1.gif" /><!-- --></h3>
<p>Chemoreceptors take the form of long rods. A single protein zips back along itself, and then joins together with a second copy, forming a rigid bundle of four intertwining helices (two from each member of the dimer), as you can see in this dimer of receptors from <em>Thermotoga maritima</em> <span class="citation">Park et al. (<a href="#ref-park2006">2006</a>)</span>. Only the cytoplasmic portion is shown here; in the cell, the receptors would also have a membrane-embedded anchor at the top and, beyond that, a small domain in the periplasm to bind the chemical of interest. Once a chemical binds, the signal is transmitted down the length of the receptor to a kinase waiting at the distal tip.</p>
</div>
</div>
<div id="chemosensory-array" class="section level2">
<h2><span class="header-section-number">7.2</span> Chemosensory Array</h2>
<p>Chemosensory arrays are highly ordered, as you can best see from a bird’s-eye view, as in this lysed <em>Salmonella typhimurium</em> cell. Chemoreceptors come together as dimers, which in turn organize into trimers, which are further packed into the extensive hexagonal honeycombed array you see here. The hexagonal arrangement comes from the baseplate, where the kinases and coupling proteins bind into an ordered array [Schematic–Chemosensory array architecture]. In what should be a familiar theme by now, organization provides a great benefit. Bacteria and archaea have a tremendous sense of smell, responding to as few as one or two molecules of an attractant, or many more. In fact, the range of chemical concentrations they can sense extends over 5 orders of magnitude. But how can a single receptor transmit a signal efficiently? Perhaps it should pass on the message to its neighbors. The interlocking network of chemoreceptors enables just this kind of amplification; a target binding to one receptor may translate into activation of 36 adjacent receptors, enormously boosting the gain of the signal.</p>
<div class="figure"><span id="fig:unnamed-chunk-124"></span>
<div id="htmlwidget-62bd0f9ac73e7ff809b9" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-62bd0f9ac73e7ff809b9">{"x":{"doi":"10.22002/D1.1546","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 7.2: Salmonella typhimurium Collected by: Morgan Beeby <a href="https://doi.org/10.22002/D1.1546">10.22002/D1.1546</a>
</p>
</div>
<div id="PDB:_6S1K_Chemosensory_array_architecture" class="section level3">
<h3><span class="header-section-number">7.2.1</span> PDB: 6S1K Chemosensory array architecture</h3>
<p><img src="img/schematics/7_2_1.gif" /><!-- --></p>
<p>Here you see two of the basic units of a chemosensory array from <em>Escherichia coli</em> <span class="citation">Cassidy et al. (<a href="#ref-cassidy2020">2020</a>)</span>. Each unit consists of a trimer of chemoreceptor dimers (in blue), a kinase (in orange), and a coupling protein (in green). In the cell, these units would further associate with four more into a rosette of six units, then with other rosettes, forming the extensive hexagonal array. Also keep in mind that these are just the stable components; additional proteins interact transiently with the receptors and kinases.</p>
</div>
</div>
<div id="chemosensory-array-conservation" class="section level2">
<h2><span class="header-section-number">7.3</span> Chemosensory Array Conservation</h2>
<p>The hexagonal array structure of chemosensory systems is invariant across species, and even across domains. As you can see in this <em>Methanospirillum hungatei</em> cell, the arrays have the same architecture in archaea as in bacteria, with 12 nm center-to-center spacing between the hexagons. This strong conservation indicates the importance of chemotaxis for cells’ fitness, and suggests that evolution had already honed it to a relatively optimal form in the common ancestor of all these cells.</p>
<div class="figure"><span id="fig:unnamed-chunk-126"></span>
<div id="htmlwidget-de03a49f4e384400a26b" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-de03a49f4e384400a26b">{"x":{"doi":"10.22002/D1.1547","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 7.3: Methanospirillum hungatei Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1547">10.22002/D1.1547</a>
</p>
</div>
</div>
<div id="chemoreceptors" class="section level2">
<h2><span class="header-section-number">7.4</span> Chemoreceptors</h2>
<p>While there is only one architectural style for chemosensory systems, there are many different building materials. Each chemoreceptor senses one, or perhaps two related, types of chemicals. This means that if your cell wants to sense multiple chemicals in its environment, it needs a matching assortment of chemoreceptors. This can quickly get complicated. Some signals need to go to the flagella, some to the pili, and others to the transcriptional machinery to turn on or off genes. How can you keep the wires from getting crossed? The simplest approach is just to separate them, and that seems to be exactly what cells do. Chemoreceptors that signal to different systems have different lengths, which sorts them into separate arrays in the cell, as you can see in this <em>Vibrio cholerae</em>. The shorter array signals to the flagellar motor, and the longer array signals to a different target we have not yet identified. Even when multiple chemosensory systems signal to the same target, arrays that respond to different environmental cues are kept separate by length differences in the receptors <a href="navigation.html#moreazospirillum-brasilense-aerotaxis">More–Azospirillum brasilense aerotaxis</a>.</p>
<div class="figure"><span id="fig:unnamed-chunk-127"></span>
<div id="htmlwidget-20dd76c9937f17312d77" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-20dd76c9937f17312d77">{"x":{"doi":"10.22002/D1.1548","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 7.4: Vibrio cholerae Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1548">10.22002/D1.1548</a>
</p>
</div>
</div>
<div id="moreazospirillum-brasilense-aerotaxis" class="section level2">
<h2><span class="header-section-number">7.5</span> More–Azospirillum brasilense aerotaxis</h2>
<p>A length difference of as little as 2 nm is enough to separate receptors, like these in <em>Azospirillum brasilense</em> that sense oxygen (28 nm array) and sources of energy like malate (30 nm array). Both receptors send signals to the single flagellar motor, promoting runs or direction switches to guide the bacterium through a combination of chemotaxis and aerotaxis (ordered movement in an oxygen gradient) to its target: plant roots. <em>A. brasilense</em> fixes nitrogen, boosting the growth of plants it colonizes.</p>
<div class="figure"><span id="fig:unnamed-chunk-128"></span>
<div id="htmlwidget-8d0570f1e6215290f853" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-8d0570f1e6215290f853">{"x":{"doi":"10.22002/D1.1551","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 7.5: Azospirillum brasilense Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1551">10.22002/D1.1551</a>
</p>
</div>
</div>
<div id="cytoplasmic-chemosensory-array" class="section level2">
<h2><span class="header-section-number">7.6</span> Cytoplasmic Chemosensory Array</h2>
<p>In addition to their extensive repertoire of membrane-embedded chemoreceptors, cells may also have another, cytoplasmic, type. Many species of bacteria and archaea, like this <em>Methanoregula formicica</em>, contain cytoplasmic chemosensory arrays. The receptors lack membrane insertion patches. Instead, they interact end-to-end with each other at their chemical-sensing tips, zippering into a double-layered sandwich, with a baseplate of kinases and coupling proteins on either side. In some species, the arrays are curved like this; in others, they are straight <a href="navigation.html#morevibrio-cholerae-chemosensory-arrays">More–Vibrio cholerae chemosensory arrays</a>. What these systems sense remains a mystery; one idea is that they monitor the internal state of the cell (e.g. the levels of various metabolites) to fine-tune signaling responses to meet the current needs of the cell. They are often found along with, and often quite close to, membrane-embedded chemosensory arrays <a href="navigation.html#morevibrio-cholerae-chemosensory-arrays">More–Vibrio cholerae chemosensory arrays</a>.</p>
<div class="figure"><span id="fig:unnamed-chunk-129"></span>
<div id="htmlwidget-ef9acd3a745909e865a9" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-ef9acd3a745909e865a9">{"x":{"doi":"10.22002/D1.1549","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 7.6: Methanoregula formicica Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1549">10.22002/D1.1549</a>
</p>
</div>
</div>
<div id="morevibrio-cholerae-chemosensory-arrays" class="section level2">
<h2><span class="header-section-number">7.7</span> More–Vibrio cholerae chemosensory arrays</h2>
<p>This <em>Vibrio cholerae</em> cell showcases a full repertoire of chemosensory systems: two membrane-embedded arrays (the shorter one signaling to the flagellar motor and the longer one signaling to a still-unknown target) and a cytoplasmic array (also with unknown function). In all cases, the basic architecture is identical: hexagonally-arranged trimers of chemoreceptor dimers.</p>
<div class="figure"><span id="fig:unnamed-chunk-130"></span>
<div id="htmlwidget-74e2a1e6302b4e1033ca" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-74e2a1e6302b4e1033ca">{"x":{"doi":"10.22002/D1.1552","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 7.7: Vibrio cholerae Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1552">10.22002/D1.1552</a>
</p>
</div>
</div>
<div id="magnetotaxis" class="section level2">
<h2><span class="header-section-number">7.8</span> Magnetotaxis</h2>
<p>Chemotaxis is not the only mode of navigation available to your cell. For a clue to others, think about how we orient ourselves in the world. In addition to our nose sniffing out food, we also use our eyes to sense light. Similarly, some photosynthetic bacteria have evolved phototaxis (ordered movement in response to light).</p>
<p>We may also use an external tool to navigate unfamiliar surroundings: a compass. Believe it or not, some bacteria have one, too. Magnetotactic species like this <em>Magnetospirillum magneticum</em> have evolved specialized structures called magnetosomes. They are pockets of inner membrane filled with crystals of a magnetic iron mineral like magnetite. The cell organizes the magnetosomes into a line using filaments of a cytoskeletal protein called MamK [Schematic–MamK structure], which is related to eukaryotic actin. The linear chain of magnetosomes functions like the needle of a compass, aligning the cell in a magnetic field.</p>
<p>Magnetosomes first form with the creation of a pocket of inner membrane, as you can see in this cell. Multiple short chains of magnetosomes may form, which are then organized into a single chain by MamK filaments. As mineralization begins, the pockets enlarge to accommodate the growing crystals, ultimately reaching ~60 nm in diameter. Magnetosome chains are inherited by daughter cells through division. Segregation is relatively easy: the chain spans the division plane so that the inward-growing wedge of cell wall simply splits the chain between magnetosomes, delivering half the chain to each daughter.</p>
<p>Much remains mysterious about these structures. For instance, what forms the membrane pockets? An even bigger question is how cells use their compass. One theory is that magnetosomes guide the aquatic bacteria vertically to the optimal height in the water column for, e.g. oxygen level. However, these species are also found at the magnetic equator, where the Earth’s field is horizontal. Another hypothesis is that magnetic orientation improves the efficiency of navigation by buffering the cell against Brownian motion that would knock it off course.</p>
<div class="figure"><span id="fig:unnamed-chunk-131"></span>
<div id="htmlwidget-16a3e083380e87b9b8eb" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-16a3e083380e87b9b8eb">{"x":{"doi":"10.22002/D1.1550","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 7.8: Magnetospirillum magneticum Collected by: Zhuo Li <a href="https://doi.org/10.22002/D1.1550">10.22002/D1.1550</a>
</p>
</div>
<div id="pdb-5jygmamk-structure-pdb_5jygmamk_structure" class="section level3">
<h3><span class="header-section-number">7.8.1</span> PDB: 5JYG–MamK structure {#PDB:_5JYG–MamK_structure} <img src="img/schematics/7_6_1.gif" /><!-- --></h3>
<p>MamK, like its eukaryotic homologue, actin, polymerizes into double-stranded filaments like this one from <em>Magnetospirillum magneticum</em> <span class="citation">Bergeron et al. (<a href="#ref-bergeron2017">2017</a>)</span>. Unlike eukaryotic actin, though, the two strands are in register, rather than being staggered to pack more tightly.</p>
</div>
</div>
<div id="further-reading-6" class="section level2">
<h2><span class="header-section-number">7.9</span> Further Reading</h2>
<p><span class="citation">Berg (<a href="#ref-berg1988">1988</a>)</span> <span class="citation">Hazelbauer, Falke, and Parkinson (<a href="#ref-hazelbauer2008">2008</a>)</span> <span class="citation">(<span class="citeproc-not-found" data-reference-id="lower2013"><strong>???</strong></span>)</span> <span class="citation">Schuergers et al. (<a href="#ref-schuergers2016">2016</a>)</span></p>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-berg1988">
<p>Berg, H. C. 1988. “A Physicist Looks at Bacterial Chemotaxis.” <em>Cold Spring Harb Symp Quant Biol</em> 53 Pt 1: 1–9.</p>
</div>
<div id="ref-bergeron2017">
<p>Bergeron, J. R., R. Hutto, E. Ozyamak, N. Hom, J. Hansen, O. Draper, M. E. Byrne, S. Keyhani, A. Komeili, and J. M. Kollman. 2017. “Structure of the Magnetosome-Associated Actin-Like MamK Filament at Subnanometer Resolution.” <em>Protein Sci</em> 26 (1): 93–102. doi:<a href="https://doi.org/10.1002/pro.2979">10.1002/pro.2979</a>.</p>
</div>
<div id="ref-cassidy2020">
<p>Cassidy, C. Keith, Benjamin A. Himes, Dapeng Sun, Jun Ma, Gongpu Zhao, John S. Parkinson, Phillip J. Stansfeld, Zaida Luthey-Schulten, and Peijun Zhang. 2020. “Structure and Dynamics of the E. Coli Chemotaxis Core Signaling Complex by Cryo-Electron Tomography and Molecular Simulations.” <em>Communications Biology</em> 3 (1). Nature Publishing Group: 1–10. doi:<a href="https://doi.org/10.1038/s42003-019-0748-0">10.1038/s42003-019-0748-0</a>.</p>
</div>
<div id="ref-hazelbauer2008">
<p>Hazelbauer, Gerald L., Joseph J. Falke, and John S. Parkinson. 2008. “Bacterial Chemoreceptors: High-Performance Signaling in Networked Arrays.” <em>Trends in Biochemical Sciences</em> 33 (1): 9–19. doi:<a href="https://doi.org/10.1016/j.tibs.2007.09.014">10.1016/j.tibs.2007.09.014</a>.</p>
</div>
<div id="ref-park2006">
<p>Park, Sang-Youn, Peter P. Borbat, Gabriela Gonzalez-Bonet, Jaya Bhatnagar, Abiola M. Pollard, Jack H. Freed, Alexandrine M. Bilwes, and Brian R. Crane. 2006. “Reconstruction of the Chemotaxis ReceptorKinase Assembly.” <em>Nature Structural &amp; Molecular Biology</em> 13 (5). Nature Publishing Group: 400–407. doi:<a href="https://doi.org/10.1038/nsmb1085">10.1038/nsmb1085</a>.</p>
</div>
<div id="ref-schuergers2016">
<p>Schuergers, N., T. Lenn, R. Kampmann, M. V. Meissner, T. Esteves, M. Temerinac-Ott, J. G. Korvink, A. R. Lowe, C. W. Mullineaux, and A. Wilde. 2016. “Cyanobacteria Use Micro-Optics to Sense Light Direction.” <em>Elife</em> 5 (February). doi:<a href="https://doi.org/10.7554/eLife.12620">10.7554/eLife.12620</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="motility.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="lifecycle.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": null,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/caltechlibrary/cell-atlas/edit/master/07_Navigation.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["cell-atlas.pdf", "cell-atlas.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
