<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 8 Navigation | Cell Atlas</title>
  <meta name="description" content="An atlas produced by the Jensen Lab." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 8 Navigation | Cell Atlas" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="splash/HeaderImage.jpg" />
  <meta property="og:description" content="An atlas produced by the Jensen Lab." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 8 Navigation | Cell Atlas" />
  
  <meta name="twitter:description" content="An atlas produced by the Jensen Lab." />
  <meta name="twitter:image" content="splash/HeaderImage.jpg" />

<meta name="author" content="Jensen Lab" />


<meta name="date" content="2020-07-29" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="motility.html"/>
<link rel="next" href="lifecycle.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/CL-0.1.2/CL.js"></script>
<script src="libs/doivideo-binding-0.0.0.9000/doivideo.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Cell Atlas</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#section"><i class="fa fa-check"></i><b>1.1</b> 1</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#schematicstructural-biology-toolkit"><i class="fa fa-check"></i><b>2.1</b> 1[Schematic–Structural biology toolkit]</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#escherichia-coli"><i class="fa fa-check"></i><b>2.2</b> Escherichia coli</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#shewanella-oneidensis"><i class="fa fa-check"></i><b>2.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#schematicelectron-microscopy-modes"><i class="fa fa-check"></i><b>2.4</b> 3[Schematic–Electron microscopy modes]</a></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#shewanella-oneidensis-1"><i class="fa fa-check"></i><b>2.5</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="2.6" data-path="methods.html"><a href="methods.html#caulobacter-crescentus"><i class="fa fa-check"></i><b>2.6</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="2.7" data-path="methods.html"><a href="methods.html#caulobacter-crescentus-1"><i class="fa fa-check"></i><b>2.7</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="2.8" data-path="methods.html"><a href="methods.html#schematicmissing-wedge"><i class="fa fa-check"></i><b>2.8</b> 6[Schematic–Missing wedge]</a></li>
<li class="chapter" data-level="2.9" data-path="methods.html"><a href="methods.html#campylobacter-jejuni"><i class="fa fa-check"></i><b>2.9</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.10" data-path="methods.html"><a href="methods.html#campylobacter-jejuni-1"><i class="fa fa-check"></i><b>2.10</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.11" data-path="methods.html"><a href="methods.html#campylobacter-jejuni-2"><i class="fa fa-check"></i><b>2.11</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.12" data-path="methods.html"><a href="methods.html#schematicstructural-biology-timeline"><i class="fa fa-check"></i><b>2.12</b> 9[Schematic–Structural biology timeline]</a></li>
<li class="chapter" data-level="2.13" data-path="methods.html"><a href="methods.html#bdellovibrio-bacteriovorus"><i class="fa fa-check"></i><b>2.13</b> Bdellovibrio bacteriovorus</a><ul>
<li class="chapter" data-level="2.13.1" data-path="methods.html"><a href="methods.html#Difference_mapping"><i class="fa fa-check"></i><b>2.13.1</b> Campylobacter jejuni</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="envelope.html"><a href="envelope.html"><i class="fa fa-check"></i><b>3</b> Envelope</a><ul>
<li class="chapter" data-level="3.1" data-path="envelope.html"><a href="envelope.html#schematic-lipid-bilayer"><i class="fa fa-check"></i><b>3.1</b> 1[Schematic: Lipid bilayer]</a></li>
<li class="chapter" data-level="3.2" data-path="envelope.html"><a href="envelope.html#schematic-atp-synthase-and-energy-production"><i class="fa fa-check"></i><b>3.2</b> 1[Schematic: ATP synthase and energy production]</a></li>
<li class="chapter" data-level="3.3" data-path="envelope.html"><a href="envelope.html#listeria-monocytogenes"><i class="fa fa-check"></i><b>3.3</b> Listeria monocytogenes</a><ul>
<li class="chapter" data-level="3.3.1" data-path="envelope.html"><a href="envelope.html#Peptidoglycan_architecture"><i class="fa fa-check"></i><b>3.3.1</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="3.3.2" data-path="envelope.html"><a href="envelope.html#Methanobacterium_formicicum"><i class="fa fa-check"></i><b>3.3.2</b> Methanobacterium formicicum</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="envelope.html"><a href="envelope.html#cupriavidus-necator"><i class="fa fa-check"></i><b>3.4</b> Cupriavidus necator</a></li>
<li class="chapter" data-level="3.5" data-path="envelope.html"><a href="envelope.html#schematicbrauns-lipoprotein"><i class="fa fa-check"></i><b>3.5</b> 3[Schematic–Braun’s lipoprotein]</a><ul>
<li class="chapter" data-level="3.5.1" data-path="envelope.html"><a href="envelope.html#escherichia-coli-diderm_sacculus_architecturecompare-this-diderm-sacculus-purified-from-escherichia-coli-to-the-monoderm-sacculus-on-the-last-page.-since-this-one-is-thinner-we-can-make-out-more-details-like-the-glycan-strands-running-around-the-circumference-of-the-cell.-the-main-difference-between-the-two-types-of-sacculi-seems-to-be-whether-they-have-largely-a-single-layer-of-peptidoglycan-diderm-or-many-layers-monoderm.-so-even-though-the-two-types-of-wall-look-different-their-architecture-is-fundamentally-the-same.-in-some-circumstances-as-you-will-see-in-chapter-8-cells-can-even-switch-between-the-two-forms."><i class="fa fa-check"></i><b>3.5.1</b> Escherichia coli {#Diderm_sacculus_architecture}Compare this diderm sacculus purified from <em>Escherichia coli</em> to the monoderm sacculus on the last page. Since this one is thinner, we can make out more details, like the glycan strands running around the circumference of the cell. The main difference between the two types of sacculi seems to be whether they have largely a single layer of peptidoglycan (diderm) or many layers (monoderm). So even though the two types of wall look different, their architecture is fundamentally the same. In some circumstances, as you will see in Chapter 8, cells can even switch between the two forms.</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="envelope.html"><a href="envelope.html#schematicsacculus-remodeling"><i class="fa fa-check"></i><b>3.6</b> 3[Schematic–Sacculus remodeling]</a></li>
<li class="chapter" data-level="3.7" data-path="envelope.html"><a href="envelope.html#myxococcus-xanthus"><i class="fa fa-check"></i><b>3.7</b> Myxococcus xanthus</a><ul>
<li class="chapter" data-level="3.7.1" data-path="envelope.html"><a href="envelope.html#borrelia-burgdorferi-pearled_vesiclesdifferent-species-can-produce-outer-membrane-vesicles-that-look-very-different.-the-same-species-can-even-produce-vesicles-that-look-very-different.-sometimes-they-come-off-the-cell-as-a-chain-of-spheres-sometimes-the-spheres-remain-connected-like-a-string-of-pearls-as-in-this-borrelia-burgdorferi-cell.-sometimes-vesicles-form-long-tubes-instead-moretubular-vesicles.-sometimes-the-same-chain-can-be-tubular-in-one-section-usually-at-the-base-connected-to-the-cell-and-a-string-of-spheres-in-another."><i class="fa fa-check"></i><b>3.7.1</b> Borrelia burgdorferi {#Pearled_vesicles}Different species can produce outer membrane vesicles that look very different. The same species can even produce vesicles that look very different. Sometimes they come off the cell as a chain of spheres; sometimes the spheres remain connected, like a string of pearls, as in this <em>Borrelia burgdorferi</em> cell. Sometimes vesicles form long tubes instead [More–Tubular vesicles]. Sometimes the same chain can be tubular in one section (usually at the base, connected to the cell), and a string of spheres in another.</a></li>
<li class="chapter" data-level="3.7.2" data-path="envelope.html"><a href="envelope.html#borrelia-burgdorferi-tubular_vesicleshere-again-from-a-borrelia-burgdorferi-cell-you-can-see-extended-tubular-outer-membrane-vesicles."><i class="fa fa-check"></i><b>3.7.2</b> Borrelia burgdorferi {#Tubular_vesicles}Here, again from a <em>Borrelia burgdorferi</em> cell, you can see extended, tubular outer membrane vesicles.</a></li>
<li class="chapter" data-level="3.7.3" data-path="envelope.html"><a href="envelope.html#myxococcus-xanthus-cytoplasmic_vesiclesnot-all-vesicles-come-from-the-outer-membrane.-the-cytoplasmic-or-inner-membrane-can-also-form-vesicles-that-are-released-into-the-cytoplasm-as-in-this-myxococcus-xanthus-cell-or-into-the-periplasm."><i class="fa fa-check"></i><b>3.7.3</b> Myxococcus xanthus {#Cytoplasmic_vesicles}Not all vesicles come from the outer membrane. The cytoplasmic or inner membrane can also form vesicles that are released into the cytoplasm, as in this <em>Myxococcus xanthus</em> cell, or into the periplasm.</a></li>
<li class="chapter" data-level="3.7.4" data-path="envelope.html"><a href="envelope.html#prosthecobacter-debontii-cytoplasmic_vesicle_varietycytoplasmic-vesicles-exhibit-a-variety-of-sizes-and-shapes.-some-are-nested-with-vesicles-inside-vesicles.-in-this-prosthecobacter-debontii-cell-you-can-see-two-other-morphologies.-one-is-a-large-flattened-horseshoe-shaped-vesicle.-another-is-a-more-typical-spherical-shape-but-is-decorated-with-what-look-like-protein-complexes."><i class="fa fa-check"></i><b>3.7.4</b> Prosthecobacter debontii {#Cytoplasmic_vesicle_variety}Cytoplasmic vesicles exhibit a variety of sizes and shapes. Some are nested, with vesicles inside vesicles. In this <em>Prosthecobacter debontii</em> cell, you can see two other morphologies. One is a large, flattened horseshoe-shaped vesicle. Another is a more typical spherical shape, but is decorated with what look like protein complexes.</a></li>
<li class="chapter" data-level="3.7.5" data-path="envelope.html"><a href="envelope.html#halomicrobium-mukohataei-archaeal_vesiclesjust-like-bacteria-archaea-also-produce-both-internal-and-external-vesicles-as-you-can-see-in-this-halomicrobium-mukohataei-cell."><i class="fa fa-check"></i><b>3.7.5</b> Halomicrobium mukohataei {#Archaeal_vesicles}Just like bacteria, archaea also produce both internal and external vesicles, as you can see in this <em>Halomicrobium mukohataei</em> cell.</a></li>
</ul></li>
<li class="chapter" data-level="3.8" data-path="envelope.html"><a href="envelope.html#mycobacterium-marinum"><i class="fa fa-check"></i><b>3.8</b> Mycobacterium marinum</a></li>
<li class="chapter" data-level="3.9" data-path="envelope.html"><a href="envelope.html#caulobacter-crescentus-2"><i class="fa fa-check"></i><b>3.9</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="3.10" data-path="envelope.html"><a href="envelope.html#schematicsurface-layer-architecture"><i class="fa fa-check"></i><b>3.10</b> 6[Schematic–Surface layer architecture]</a></li>
<li class="chapter" data-level="3.11" data-path="envelope.html"><a href="envelope.html#aarchaea-and-bacteria-with-surface-layers-produce-characteristic-outer-membrane-vesicles-they-bud-off-with-the-surface-layer-attached.-you-will-see-plenty-of-examples-in-the-coming-pages.-delftia-acidovorans-like-this-produce-so-called-nanopods-chains-of-outer-membrane-vesicles-ensheathed-in-surface-layer."><i class="fa fa-check"></i><b>3.11</b> 6aArchaea and bacteria with surface layers produce characteristic outer membrane vesicles: they bud off with the surface layer attached. You will see plenty of examples in the coming pages. <em>Delftia acidovorans</em> like this produce so-called nanopods: chains of outer membrane vesicles ensheathed in surface layer.</a></li>
<li class="chapter" data-level="3.12" data-path="envelope.html"><a href="envelope.html#sulfolobus-solfataricus"><i class="fa fa-check"></i><b>3.12</b> Sulfolobus solfataricus</a><ul>
<li class="chapter" data-level="3.12.1" data-path="envelope.html"><a href="envelope.html#methylomicrobium-alcaliphilum-m._alcaliphilumin-methylomicrobium-alcaliphilum-v-shaped-surface-layer-proteins-come-together-to-form-cups-that-pack-into-a-hexagonal-pattern."><i class="fa fa-check"></i><b>3.12.1</b> Methylomicrobium alcaliphilum {#M._alcaliphilum}In <em>Methylomicrobium alcaliphilum</em>, V-shaped surface layer proteins come together to form cups that pack into a hexagonal pattern.</a></li>
<li class="chapter" data-level="3.12.2" data-path="envelope.html"><a href="envelope.html#clostridium-thermocellum-clostridium_thermocellumnot-only-diderm-bacteria-have-surface-layers-as-you-can-see-on-this-monoderm-clostridium-thermocellum-cell.-the-main-difference-is-that-the-proteins-are-anchored-to-the-cell-wall-rather-than-to-the-outer-membrane-or-associated-lipoproteins."><i class="fa fa-check"></i><b>3.12.2</b> Clostridium thermocellum {#Clostridium_thermocellum}Not only diderm bacteria have surface layers, as you can see on this monoderm <em>Clostridium thermocellum</em> cell. The main difference is that the proteins are anchored to the cell wall, rather than to the outer membrane (or associated lipoproteins).</a></li>
<li class="chapter" data-level="3.12.3" data-path="envelope.html"><a href="envelope.html#nitrosopumilis-maritimus-n._maritimusin-nitrosopumilis-maritimus-surface-layer-proteins-form-hexagonal-rosettes-that-in-turn-pack-into-a-hexagonal-lattice."><i class="fa fa-check"></i><b>3.12.3</b> Nitrosopumilis maritimus {#N._maritimus}In <em>Nitrosopumilis maritimus</em>, surface layer proteins form hexagonal rosettes that in turn pack into a hexagonal lattice.</a></li>
<li class="chapter" data-level="3.12.4" data-path="envelope.html"><a href="envelope.html#methanoregula-formicica-methanoregula_formicicathe-surface-layer-of-methanoregula-formicica-is-a-hexagonal-lattice-of-small-nearly-circular-subunits."><i class="fa fa-check"></i><b>3.12.4</b> Methanoregula formicica {#Methanoregula_formicica}The surface layer of <em>Methanoregula formicica</em> is a hexagonal lattice of small, nearly circular subunits.</a></li>
</ul></li>
<li class="chapter" data-level="3.13" data-path="envelope.html"><a href="envelope.html#methanospirillum-hungatei"><i class="fa fa-check"></i><b>3.13</b> Methanospirillum hungatei</a></li>
<li class="chapter" data-level="3.14" data-path="envelope.html"><a href="envelope.html#haloarcula-argentinensis"><i class="fa fa-check"></i><b>3.14</b> Haloarcula argentinensis</a></li>
<li class="chapter" data-level="3.15" data-path="envelope.html"><a href="envelope.html#bdellovibrio-bacteriovorus-1"><i class="fa fa-check"></i><b>3.15</b> Bdellovibrio bacteriovorus</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="shape.html"><a href="shape.html"><i class="fa fa-check"></i><b>4</b> Shape</a><ul>
<li class="chapter" data-level="4.1" data-path="shape.html"><a href="shape.html#cupriavidus-necator-1"><i class="fa fa-check"></i><b>4.1</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="4.1.1" data-path="shape.html"><a href="shape.html#brucella-abortus-rod_varietynot-all-rod-shaped-cells-are-perfectly-cylindrical.-for-instance-brucella-abortus-like-this-one-are-pear-shaped.-this-is-one-of-the-rod-shaped-bacterial-species-that-do-not-use-mreb."><i class="fa fa-check"></i><b>4.1.1</b> Brucella abortus {#Rod_variety}Not all rod-shaped cells are perfectly cylindrical. For instance, <em>Brucella abortus</em> like this one are pear-shaped. This is one of the rod-shaped bacterial species that do not use MreB.</a></li>
<li class="chapter" data-level="4.1.2" data-path="shape.html"><a href="shape.html#methanoregula-formicica-archaeal_rodssome-archaeal-species-like-this-methanoregula-formicica-are-rod-shaped.-the-rigid-surface-layer-seems-to-help-define-the-shape-but-the-details-are-still-unknown."><i class="fa fa-check"></i><b>4.1.2</b> Methanoregula formicica {#Archaeal_rods}Some archaeal species, like this <em>Methanoregula formicica</em>, are rod-shaped. The rigid surface layer seems to help define the shape, but the details are still unknown.</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="shape.html"><a href="shape.html#hylemonella-gracilis"><i class="fa fa-check"></i><b>4.2</b> Hylemonella gracilis</a></li>
<li class="chapter" data-level="4.3" data-path="shape.html"><a href="shape.html#caulobacter-crescentus-3"><i class="fa fa-check"></i><b>4.3</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="4.4" data-path="shape.html"><a href="shape.html#schematic-ctp-synthase"><i class="fa fa-check"></i><b>4.4</b> 4[Schematic: CTP synthase]</a></li>
<li class="chapter" data-level="4.5" data-path="shape.html"><a href="shape.html#campylobacter-jejuni-3"><i class="fa fa-check"></i><b>4.5</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="4.5.1" data-path="shape.html"><a href="shape.html#borrelia-burgdorferi-borrelia_burgdorferiborrelia-burgdorferi-like-this-cell-use-the-long-filaments-of-their-motility-machinery-flagella-discussed-in-chapter-6-as-a-kind-of-cytoskeleton.-a-bundle-of-flagella-wraps-around-the-cell-in-the-periplasm-between-the-cell-wall-and-the-outer-membrane.-spun-by-motors-at-their-base-more-on-that-in-chapter-6-the-filaments-impart-a-wave-pattern-to-the-growing-sacculus.-without-the-motors-rotation-the-cells-develop-a-rod-shape.-note-that-b.-burgdorferi-are-not-helical-but-rather-adopt-a-two-dimensional-waveform."><i class="fa fa-check"></i><b>4.5.1</b> Borrelia burgdorferi {#Borrelia_burgdorferi}<em>Borrelia burgdorferi</em> like this cell use the long filaments of their motility machinery (flagella, discussed in Chapter 6) as a kind of cytoskeleton. A bundle of flagella wraps around the cell in the periplasm, between the cell wall and the outer membrane. Spun by motors at their base (more on that in Chapter 6), the filaments impart a wave pattern to the growing sacculus. Without the motors’ rotation, the cells develop a rod shape. Note that <em>B. burgdorferi</em> are not helical, but rather adopt a two-dimensional waveform.</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="shape.html"><a href="shape.html#verrucomicrobium-spinosum"><i class="fa fa-check"></i><b>4.6</b> Verrucomicrobium spinosum</a><ul>
<li class="chapter" data-level="4.6.1" data-path="shape.html"><a href="shape.html#Bactofilin"><i class="fa fa-check"></i><b>4.6.1</b> Bactofilin</a></li>
<li class="chapter" data-level="4.6.2" data-path="shape.html"><a href="shape.html#prosthecobacter-vanneervenii-bacterial_microtubulessome-bacterial-species-with-prosthecae-express-structures-similar-to-eukaryotic-microtubules-made-from-two-proteins-called-btuba-and-btubb-to-reflect-their-homology-to-eukaryotic-tubulins.-eukaryotic-microtubules-are-hollow-tubes-formed-by-13-protofilaments-bacterial-microtubules-are-smaller-with-only-5-protofilaments.-cells-commonly-contain-a-bundle-of-microtubules-in-their-prostheca-like-this-prosthecobacter-vanneervenii-cell-which-has-a-bundle-of-four."><i class="fa fa-check"></i><b>4.6.2</b> Prosthecobacter vanneervenii {#Bacterial_microtubules}Some bacterial species with prosthecae express structures similar to eukaryotic microtubules, made from two proteins called BtubA and BtubB to reflect their homology to eukaryotic tubulins. Eukaryotic microtubules are hollow tubes formed by 13 protofilaments; bacterial microtubules are smaller, with only ~5 protofilaments. Cells commonly contain a bundle of microtubules in their prostheca, like this <em>Prosthecobacter vanneervenii</em> cell, which has a bundle of four.</a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="shape.html"><a href="shape.html#haloquadratum-walsbyi"><i class="fa fa-check"></i><b>4.7</b> Haloquadratum walsbyi</a><ul>
<li class="chapter" data-level="4.7.1" data-path="shape.html"><a href="shape.html#halobacterium-salinarum-gas_vesiclessome-species-of-archaea-and-bacteria-use-gas-vesicles-to-control-their-buoyancy.-this-can-allow-them-to-rise-or-fall-in-a-water-column-which-can-be-a-great-advantage.-halobacterium-salinarum-like-this-one-produce-gas-vesicles-in-response-to-cues-from-the-environment-lifting-themselves-out-of-the-sediment-and-into-more-favorable-conditions-of-oxygen-or-sunlight-for-photosynthesis.-this-cell-has-just-started-producing-gas-vesicles-so-they-are-small-and-isolated.-later-the-vesicles-will-elongate-into-cylinders-with-conical-ends-as-you-saw-in-haloquadratum-walsbyi.-each-cell-might-contain-dozens-of-vesicles-and-they-often-cluster-together."><i class="fa fa-check"></i><b>4.7.1</b> Halobacterium salinarum {#Gas_vesicles}Some species of archaea and bacteria use gas vesicles to control their buoyancy. This can allow them to rise or fall in a water column, which can be a great advantage. <em>Halobacterium salinarum</em> like this one produce gas vesicles in response to cues from the environment, lifting themselves out of the sediment and into more favorable conditions of oxygen or sunlight for photosynthesis. This cell has just started producing gas vesicles, so they are small and isolated. Later, the vesicles will elongate into cylinders with conical ends, as you saw in <em>Haloquadratum walsbyi</em>. Each cell might contain dozens of vesicles, and they often cluster together.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="growth.html"><a href="growth.html"><i class="fa fa-check"></i><b>5</b> Growth</a><ul>
<li class="chapter" data-level="5.1" data-path="growth.html"><a href="growth.html#shewanella-oneidensis-2"><i class="fa fa-check"></i><b>5.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="5.2" data-path="growth.html"><a href="growth.html#rhodopseudomonas-palustris"><i class="fa fa-check"></i><b>5.2</b> Rhodopseudomonas palustris</a></li>
<li class="chapter" data-level="5.3" data-path="growth.html"><a href="growth.html#schematic-photocomplexes"><i class="fa fa-check"></i><b>5.3</b> 3[Schematic: Photocomplexes]</a><ul>
<li class="chapter" data-level="5.3.1" data-path="growth.html"><a href="growth.html#methyloprofundus-sedimenti-icm_varietyin-some-species-like-this-methyloprofundus-sedimenti-the-intracytoplasmic-membrane-is-very-extensive-and-appears-to-be-fully-separated-from-the-cell-membrane.-as-you-can-see-it-occupies-much-of-the-cytoplasmic-space."><i class="fa fa-check"></i><b>5.3.1</b> Methyloprofundus sedimenti {#ICM_variety}In some species, like this <em>Methyloprofundus sedimenti</em>, the intracytoplasmic membrane is very extensive and appears to be fully separated from the cell membrane. As you can see, it occupies much of the cytoplasmic space.</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="growth.html"><a href="growth.html#chloroflexi"><i class="fa fa-check"></i><b>5.4</b> Chloroflexi</a></li>
<li class="chapter" data-level="5.5" data-path="growth.html"><a href="growth.html#clostridium-thermocellum"><i class="fa fa-check"></i><b>5.5</b> Clostridium thermocellum</a></li>
<li class="chapter" data-level="5.6" data-path="growth.html"><a href="growth.html#schematicadhe-spirosome-structure"><i class="fa fa-check"></i><b>5.6</b> 5[Schematic–AdhE spirosome structure]</a></li>
<li class="chapter" data-level="5.7" data-path="growth.html"><a href="growth.html#acetonema-longum"><i class="fa fa-check"></i><b>5.7</b> Acetonema longum</a></li>
<li class="chapter" data-level="5.8" data-path="growth.html"><a href="growth.html#schematicmicrocompartment-shells"><i class="fa fa-check"></i><b>5.8</b> 6[Schematic–Microcompartment shells]</a></li>
<li class="chapter" data-level="5.9" data-path="growth.html"><a href="growth.html#thiomonas-intermedia"><i class="fa fa-check"></i><b>5.9</b> Thiomonas intermedia</a><ul>
<li class="chapter" data-level="5.9.1" data-path="growth.html"><a href="growth.html#halothiobacillus-neapolitanus-carboxysome_varietymost-carboxysomes-look-like-the-ones-you-saw-on-the-last-page-but-not-all.-some-are-less-regular-polyhedra-as-in-this-halothiobacillus-neapolitanus-cell.-and-some-are-stranger-still-morelong-carboxysomes."><i class="fa fa-check"></i><b>5.9.1</b> Halothiobacillus neapolitanus {#Carboxysome_variety}Most carboxysomes look like the ones you saw on the last page, but not all. Some are less regular polyhedra, as in this <em>Halothiobacillus neapolitanus</em> cell. And some are stranger still [More–Long carboxysomes].</a></li>
<li class="chapter" data-level="5.9.2" data-path="growth.html"><a href="growth.html#hydrogenovibrio-crunogenus-long_carboxysomesthe-carboxysomes-in-this-hydrogenovibrio-crunogenus-have-grown-abnormally-long-extending-across-the-full-width-of-the-cell.-these-are-unusual-but-less-dramatic-elongations-are-common."><i class="fa fa-check"></i><b>5.9.2</b> Hydrogenovibrio crunogenus {#Long_carboxysomes}The carboxysomes in this <em>Hydrogenovibrio crunogenus</em> have grown abnormally long, extending across the full width of the cell. These are unusual, but less dramatic elongations are common.</a></li>
</ul></li>
<li class="chapter" data-level="5.10" data-path="growth.html"><a href="growth.html#cupriavidus-necator-2"><i class="fa fa-check"></i><b>5.10</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="5.10.1" data-path="growth.html"><a href="growth.html#lysobacter-antibioticus-storage_granule_growthstorage-granules-expand-and-contract-to-accommodate-their-contents.-this-lysobacter-antibioticus-cell-exhibits-a-typical-range-of-sizes-of-poly-phosphate-granules."><i class="fa fa-check"></i><b>5.10.1</b> Lysobacter antibioticus {#Storage_granule_growth}Storage granules expand and contract to accommodate their contents. This <em>Lysobacter antibioticus</em> cell exhibits a typical range of sizes of poly-phosphate granules.</a></li>
</ul></li>
<li class="chapter" data-level="5.11" data-path="growth.html"><a href="growth.html#agrobacterium-tumefaciens"><i class="fa fa-check"></i><b>5.11</b> Agrobacterium tumefaciens</a><ul>
<li class="chapter" data-level="5.11.1" data-path="growth.html"><a href="growth.html#hydrogenovibrio-crunogenus-granule-containing_carboxysomessmall-poly-phosphate-storage-granules-are-sometimes-seen-inside-carboxysomes-as-in-this-hydrogenovibrio-crunogenus-cell.-their-purpose-is-unknown."><i class="fa fa-check"></i><b>5.11.1</b> Hydrogenovibrio crunogenus {#Granule-containing_carboxysomes}Small poly-phosphate storage granules are sometimes seen <em>inside</em> carboxysomes, as in this <em>Hydrogenovibrio crunogenus</em> cell. Their purpose is unknown.</a></li>
<li class="chapter" data-level="5.11.2" data-path="growth.html"><a href="growth.html#halothiobacillus-neapolitanus-carboxysomes-granule_connectionswe-sometimes-see-comb-like-structures-connecting-external-poly-phosphate-storage-granules-with-carboxysomes-as-in-this-halothiobacillus-neapolitanus-cell.-the-nature-of-the-relationship-between-the-structures-and-the-identity-and-function-of-the-combs-remains-a-mystery."><i class="fa fa-check"></i><b>5.11.2</b> Halothiobacillus neapolitanus {#Carboxysomes-granule_connections}We sometimes see comb-like structures connecting external poly-phosphate storage granules with carboxysomes, as in this <em>Halothiobacillus neapolitanus</em> cell. The nature of the relationship between the structures, and the identity and function of the combs, remains a mystery.</a></li>
</ul></li>
<li class="chapter" data-level="5.12" data-path="growth.html"><a href="growth.html#halorubrum-litoreum"><i class="fa fa-check"></i><b>5.12</b> Halorubrum litoreum</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="division.html"><a href="division.html"><i class="fa fa-check"></i><b>6</b> Division</a><ul>
<li class="chapter" data-level="6.1" data-path="division.html"><a href="division.html#caulobacter-crescentus-4"><i class="fa fa-check"></i><b>6.1</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="6.2" data-path="division.html"><a href="division.html#hyphomonas-neptunium"><i class="fa fa-check"></i><b>6.2</b> Hyphomonas neptunium</a><ul>
<li class="chapter" data-level="6.2.1" data-path="division.html"><a href="division.html#hyphomonas-neptunium-hyphomonas_stalkhyphomonas-neptunium-grow-a-single-stalk-from-one-end-of-their-cell-body-similar-to-the-caulobacter-crescentus-you-saw-in-chapter-3.-the-function-of-the-stalk-though-is-different-in-this-budding-bacterium-than-in-c.-crescentus-which-divide-by-more-conventional-fission."><i class="fa fa-check"></i><b>6.2.1</b> Hyphomonas neptunium {#Hyphomonas_stalk}<em>Hyphomonas neptunium</em> grow a single stalk from one end of their cell body, similar to the <em>Caulobacter crescentus</em> you saw in Chapter 3. The function of the stalk, though, is different in this budding bacterium than in <em>C. crescentus</em>, which divide by more conventional fission.</a></li>
<li class="chapter" data-level="6.2.2" data-path="division.html"><a href="division.html#hyphomonas-neptunium-hyphomonas_lifecyclehyphomonas-neptunium-have-evolved-a-program-of-stages-they-pass-through-in-the-course-of-their-life.-when-a-newborn-cell-like-this-one-is-released-it-is-in-the-swarmer-stage-using-its-flagellum-discussed-in-the-next-chapter-to-swim-away-to-a-new-location.-after-a-while-it-settles-down-jettisoning-its-flagellum-and-growing-a-stalk-with-which-to-make-its-own-budthe-next-generation.-once-a-cell-settles-down-into-the-stalked-stage-it-spends-the-rest-of-its-life-sending-off-buds-as-long-as-conditions-are-good.-only-newborn-cells-are-equipped-with-flagella-to-go-adventuring.-we-will-discuss-more-lifecycles-in-chapter-8."><i class="fa fa-check"></i><b>6.2.2</b> Hyphomonas neptunium {#Hyphomonas_lifecycle}<em>Hyphomonas neptunium</em> have evolved a program of stages they pass through in the course of their life. When a newborn cell like this one is released, it is in the “swarmer” stage, using its flagellum (discussed in the next chapter) to swim away to a new location. After a while, it settles down, jettisoning its flagellum and growing a stalk with which to make its own bud–the next generation. Once a cell settles down into the stalked stage, it spends the rest of its life sending off buds as long as conditions are good. Only newborn cells are equipped with flagella to go adventuring. We will discuss more lifecycles in Chapter 8.</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="division.html"><a href="division.html#staphylococcus-aureus"><i class="fa fa-check"></i><b>6.3</b> Staphylococcus aureus</a></li>
<li class="chapter" data-level="6.4" data-path="division.html"><a href="division.html#tetrasphaera-remsis"><i class="fa fa-check"></i><b>6.4</b> Tetrasphaera remsis</a></li>
<li class="chapter" data-level="6.5" data-path="division.html"><a href="division.html#idiomarina-loihiensis"><i class="fa fa-check"></i><b>6.5</b> Idiomarina loihiensis</a><ul>
<li class="chapter" data-level="6.5.1" data-path="division.html"><a href="division.html#sphingopyxis-alaskensis-sphingopyxis_alaskensis_divisionthis-diderm-cell-is-about-halfway-through-cytokinesis.-the-two-daughter-cells-are-still-connected-by-a-wide-bridge-of-cytoplasm-but-the-inner-membrane-and-cell-wall-are-drawing-in-all-around."><i class="fa fa-check"></i><b>6.5.1</b> Sphingopyxis alaskensis {#Sphingopyxis_alaskensis_division}This diderm cell is about halfway through cytokinesis. The two daughter cells are still connected by a wide bridge of cytoplasm, but the inner membrane and cell wall are drawing in all around.</a></li>
<li class="chapter" data-level="6.5.2" data-path="division.html"><a href="division.html#chloroflexi-chloroflexi_divisionthis-diderm-cell-is-in-the-final-stage-of-cytokinesis.-the-cytoplasm-of-the-two-daughter-cells-is-completely-separated-by-inner-membrane-and-cell-wall.-the-outer-membrane-though-has-not-yet-divided."><i class="fa fa-check"></i><b>6.5.2</b> Chloroflexi {#Chloroflexi_division}This diderm cell is in the final stage of cytokinesis. The cytoplasm of the two daughter cells is completely separated by inner membrane and cell wall. The outer membrane, though, has not yet divided.</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="division.html"><a href="division.html#caulobacter-crescentus-5"><i class="fa fa-check"></i><b>6.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="6.6.1" data-path="division.html"><a href="division.html#caulobacter-crescentus-caulobacter_divisionthese-daughter-cells-have-completely-separated-their-mothers-inner-membrane-and-cell-wall-but-not-yet-the-outer-membrane-and-surface-layer."><i class="fa fa-check"></i><b>6.6.1</b> Caulobacter crescentus {#Caulobacter_division}These daughter cells have completely separated their mother’s inner membrane and cell wall, but not yet the outer membrane and surface layer.</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="division.html"><a href="division.html#agrobacterium-tumefaciens-1"><i class="fa fa-check"></i><b>6.7</b> Agrobacterium tumefaciens</a></li>
<li class="chapter" data-level="6.8" data-path="division.html"><a href="division.html#cupriavidus-necator-3"><i class="fa fa-check"></i><b>6.8</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="6.8.1" data-path="division.html"><a href="division.html#cupriavidus-necator-asymmetric_constrictionthis-c.-necator-cell-is-at-a-late-stage-of-cytokinesis.-in-most-cells-constriction-would-be-more-or-less-uniform-around-the-circumference-by-now-but-in-some-cells-including-this-one-the-asymmetry-persists."><i class="fa fa-check"></i><b>6.8.1</b> Cupriavidus necator {#Asymmetric_constriction}This <em>C. necator</em> cell is at a late stage of cytokinesis. In most cells, constriction would be more or less uniform around the circumference by now, but in some cells, including this one, the asymmetry persists.</a></li>
</ul></li>
<li class="chapter" data-level="6.9" data-path="division.html"><a href="division.html#schematicftsz-structure"><i class="fa fa-check"></i><b>6.9</b> 9[Schematic–FtsZ structure]</a></li>
<li class="chapter" data-level="6.10" data-path="division.html"><a href="division.html#caulobacter-crescentus-6"><i class="fa fa-check"></i><b>6.10</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="6.11" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius"><i class="fa fa-check"></i><b>6.11</b> Sulfolobus acidocaldarius</a></li>
<li class="chapter" data-level="6.12" data-path="division.html"><a href="division.html#schematicescrt-structure"><i class="fa fa-check"></i><b>6.12</b> 11[Schematic–ESCRT structure]</a></li>
<li class="chapter" data-level="6.13" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius-1"><i class="fa fa-check"></i><b>6.13</b> Sulfolobus acidocaldarius</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="motility.html"><a href="motility.html"><i class="fa fa-check"></i><b>7</b> Motility</a><ul>
<li class="chapter" data-level="7.1" data-path="motility.html"><a href="motility.html#schematicflagellum-structure"><i class="fa fa-check"></i><b>7.1</b> 1[Schematic–Flagellum structure]</a></li>
<li class="chapter" data-level="7.2" data-path="motility.html"><a href="motility.html#bdellovibrio-bacteriovorus-2"><i class="fa fa-check"></i><b>7.2</b> Bdellovibrio bacteriovorus</a><ul>
<li class="chapter" data-level="7.2.1" data-path="motility.html"><a href="motility.html#bdellovibrio-bacteriovorus-flagellar_motor_structurethis-is-an-average-of-the-flagellar-motors-from-more-than-one-thousand-bdellovibrio-bacteriovorus-cells.-working-upward-from-the-base-the-major-parts-of-the-rotor-are-the-c-ring-for-_c_ytoplasmic-the-ms-ring-for-_m_embrane-and-_s_upramembrane-the-rod-the-hook-and-finally-the-filament.-they-are-surrounded-by-stationary-parts-the-stator-ring-which-is-dynamic-so-we-cannot-resolve-the-stators-as-they-cross-the-inner-membrane-and-connect-to-the-c-ring-and-a-series-of-bushings-that-allow-rotation-within-the-cell-wall-the-_p_eriplasmic-or-p-ring-and-outer-membrane-_l_ipopolysaccharide-or-l-ring.-additional-cytoplasmic-components-form-the-export-apparatus-which-is-involved-in-assembly-discussed-on-the-next-page."><i class="fa fa-check"></i><b>7.2.1</b> Bdellovibrio bacteriovorus {#Flagellar_motor_structure}This is an average of the flagellar motors from more than one thousand <em>Bdellovibrio bacteriovorus</em> cells. Working upward from the base, the major parts of the rotor are the C-ring (for _C_ytoplasmic), the MS-ring (for _M_embrane and _S_upramembrane), the rod, the hook, and finally the filament. They are surrounded by stationary parts: the stator ring (which is dynamic, so we cannot resolve the stators as they cross the inner membrane and connect to the C-ring) and a series of bushings that allow rotation within the cell wall (the _P_eriplasmic or P-ring) and outer membrane (_L_ipopolysaccharide or L-ring). Additional cytoplasmic components form the export apparatus, which is involved in assembly (discussed on the next page).</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="motility.html"><a href="motility.html#schematicflagellar-motor-diversity"><i class="fa fa-check"></i><b>7.3</b> 2[Schematic–Flagellar motor diversity]</a></li>
<li class="chapter" data-level="7.4" data-path="motility.html"><a href="motility.html#hylemonella-gracilis-1"><i class="fa fa-check"></i><b>7.4</b> Hylemonella gracilis</a><ul>
<li class="chapter" data-level="7.4.1" data-path="motility.html"><a href="motility.html#pseudomonas-aeruginosa-flagellar_motor_disassemblywhen-flagella-break-or-in-some-species-like-caulobacter-crescentus-are-ejected-so-the-cell-can-attach-to-a-surface-the-motor-is-disassembled-usually-beginning-with-the-export-apparatus.-not-everything-is-dismantled-though-the-p--and-l-rings-remain-in-place-in-the-cell-wall-and-outer-membrane-respectively-as-you-can-see-in-this-pseudomonas-aeruginosa-cell.-we-do-not-know-if-the-cell-has-a-reason-for-leaving-them-there-but-one-possibility-is-that-they-may-function-as-a-plug-for-the-outer-membrane-when-the-hook-is-no-longer-present."><i class="fa fa-check"></i><b>7.4.1</b> Pseudomonas aeruginosa {#Flagellar_motor_disassembly}When flagella break (or in some species, like <em>Caulobacter crescentus</em>, are ejected so the cell can attach to a surface), the motor is disassembled, usually beginning with the export apparatus. Not everything is dismantled, though; the P- and L-rings remain in place in the cell wall and outer membrane, respectively, as you can see in this <em>Pseudomonas aeruginosa</em> cell. We do not know if the cell has a reason for leaving them there, but one possibility is that they may function as a plug for the outer membrane when the hook is no longer present.</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="motility.html"><a href="motility.html#campylobacter-jejuni-4"><i class="fa fa-check"></i><b>7.5</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="7.5.1" data-path="motility.html"><a href="motility.html#helicobacter-pylori-lophotrichous_bacteriamany-lophotrichous-species-like-the-hylemonella-gracilis-you-saw-in-chapter-3length-or-this-helicobacter-pylori-have-a-tuft-of-flagella-at-their-cell-pole.-in-some-species-though-the-tuft-is-located-elsewhere-for-example-a-clump-of-flagella-on-the-concave-side-of-banana-shaped-selenomonas-artemidis-pushes-the-cells-sideways-in-a-seesawing-swimming-pattern."><i class="fa fa-check"></i><b>7.5.1</b> Helicobacter pylori {#Lophotrichous_bacteria}Many lophotrichous species, like the <em>Hylemonella gracilis</em> you saw in Chapter 3–Length or this <em>Helicobacter pylori</em>, have a tuft of flagella at their cell pole. In some species, though, the tuft is located elsewhere; for example, a clump of flagella on the concave side of banana-shaped <em>Selenomonas artemidis</em> pushes the cells sideways in a seesawing swimming pattern.</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="motility.html"><a href="motility.html#pseudomonas-flexibilis"><i class="fa fa-check"></i><b>7.6</b> Pseudomonas flexibilis</a><ul>
<li class="chapter" data-level="7.6.1" data-path="motility.html"><a href="motility.html#proteus-mirabilis-proteus_mirabilis_flagellaproteus-mirabilis-adapt-their-motility-machinery-to-their-environment.-in-liquid-the-short-rod-shaped-cells-swim-with-the-help-of-a-handful-of-flagella-distributed-peritrichously-around-their-cell-body.-when-they-encounter-a-solid-surface-the-cells-elongate-and-build-many-more-flagella-as-you-can-see-on-this-cell.-instead-of-swimming-they-now-use-their-flagella-to-propel-themselves-in-groups-across-the-surface-a-motility-mode-known-as-swarming.-this-is-an-example-of-a-differentiated-lifecycle-which-will-come-up-again-in-chapter-8."><i class="fa fa-check"></i><b>7.6.1</b> Proteus mirabilis {#Proteus_mirabilis_flagella}<em>Proteus mirabilis</em> adapt their motility machinery to their environment. In liquid, the short rod-shaped cells swim with the help of a handful of flagella distributed peritrichously around their cell body. When they encounter a solid surface, the cells elongate and build many more flagella, as you can see on this cell. Instead of swimming, they now use their flagella to propel themselves in groups across the surface, a motility mode known as “swarming.” This is an example of a differentiated lifecycle, which will come up again in Chapter 8.</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="motility.html"><a href="motility.html#helicobacter-hepaticus"><i class="fa fa-check"></i><b>7.7</b> Helicobacter hepaticus</a></li>
<li class="chapter" data-level="7.8" data-path="motility.html"><a href="motility.html#borrelia-burgdorferi"><i class="fa fa-check"></i><b>7.8</b> Borrelia burgdorferi</a><ul>
<li class="chapter" data-level="7.8.1" data-path="motility.html"><a href="motility.html#treponema-primitia-treponema_primitiatreponema-primitia-like-this-are-commensal-residents-of-the-termite-gut-helping-break-down-cellulose.-in-addition-to-two-periplasmic-flagella-the-cells-have-arrays-of-bowl--and-hook-like-structures-on-their-surface-the-function-of-which-likely-related-to-motility-remains-mysterious."><i class="fa fa-check"></i><b>7.8.1</b> Treponema primitia {#Treponema_primitia}<em>Treponema primitia</em> like this are commensal residents of the termite gut, helping break down cellulose. In addition to two periplasmic flagella, the cells have arrays of bowl- and hook-like structures on their surface, the function of which, likely related to motility, remains mysterious.</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="motility.html"><a href="motility.html#methanoregula-formicica"><i class="fa fa-check"></i><b>7.9</b> Methanoregula formicica</a></li>
<li class="chapter" data-level="7.10" data-path="motility.html"><a href="motility.html#schematic-archaellum-structure"><i class="fa fa-check"></i><b>7.10</b> 8[Schematic: Archaellum structure]</a></li>
<li class="chapter" data-level="7.11" data-path="motility.html"><a href="motility.html#schematic-archaellar-motor-structure"><i class="fa fa-check"></i><b>7.11</b> 8[Schematic: Archaellar motor structure]</a></li>
<li class="chapter" data-level="7.12" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis"><i class="fa fa-check"></i><b>7.12</b> Thermococcus kodakaraensis</a><ul>
<li class="chapter" data-level="7.12.1" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis-cone_structurethe-archaellar-plate-does-not-come-to-a-point-at-the-tip-but-rather-is-a-conical-frustum-open-at-the-top-resembling-a-lampshade.-in-the-center-of-the-tip-is-a-small-ring-as-you-can-see-in-a-top-view-in-this-lysed-flattened-thermococcus-kodakaraensis-cell.-the-function-of-the-ring-remains-unknown-perhaps-it-helps-nucleate-the-rest-of-the-structure"><i class="fa fa-check"></i><b>7.12.1</b> Thermococcus kodakaraensis {#Cone_structure}The archaellar plate does not come to a point at the tip, but rather is a conical frustum (open at the top), resembling a lampshade. In the center of the tip is a small ring, as you can see in a top view in this lysed, flattened <em>Thermococcus kodakaraensis</em> cell. The function of the ring remains unknown; perhaps it helps nucleate the rest of the structure?</a></li>
<li class="chapter" data-level="7.12.2" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis-organizing_centeras-you-can-see-more-clearly-in-this-partially-lysed-flattened-thermococcus-kodakaraensis-cell-the-conical-plate-in-the-cytoplasm-is-attached-to-more-than-just-the-archaella.-it-is-also-associated-with-chemosensory-arrays-discussed-in-the-next-chapter-and-dna-as-you-can-see-from-the-ribosome-excluding-zone.-this-structure-may-therefore-perform-an-analogous-function-to-bacterial-organizing-proteins-such-as-popz-tethering-cellular-components-into-a-de-facto-pole-for-the-in-this-case-round-cell."><i class="fa fa-check"></i><b>7.12.2</b> Thermococcus kodakaraensis {#Organizing_center}As you can see more clearly in this partially-lysed, flattened <em>Thermococcus kodakaraensis</em> cell, the conical plate in the cytoplasm is attached to more than just the archaella. It is also associated with chemosensory arrays (discussed in the next chapter) and DNA, as you can see from the ribosome-excluding zone. This structure may therefore perform an analogous function to bacterial organizing proteins such as PopZ, tethering cellular components into a <em>de facto</em> pole for the (in this case round) cell.</a></li>
</ul></li>
<li class="chapter" data-level="7.13" data-path="motility.html"><a href="motility.html#myxococcus-xanthus-1"><i class="fa fa-check"></i><b>7.13</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="7.14" data-path="motility.html"><a href="motility.html#schematictype-iv-pilus-structure"><i class="fa fa-check"></i><b>7.14</b> 10[Schematic–Type IV pilus structure]</a></li>
<li class="chapter" data-level="7.15" data-path="motility.html"><a href="motility.html#flavobacterium-johnsoniae"><i class="fa fa-check"></i><b>7.15</b> Flavobacterium johnsoniae</a></li>
<li class="chapter" data-level="7.16" data-path="motility.html"><a href="motility.html#mycoplasma-pneumoniae"><i class="fa fa-check"></i><b>7.16</b> Mycoplasma pneumoniae</a></li>
<li class="chapter" data-level="7.17" data-path="motility.html"><a href="motility.html#schematicterminal-organelle-structure"><i class="fa fa-check"></i><b>7.17</b> 12[ Schematic–Terminal organelle structure]</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="navigation.html"><a href="navigation.html"><i class="fa fa-check"></i><b>8</b> Navigation</a><ul>
<li class="chapter" data-level="8.1" data-path="navigation.html"><a href="navigation.html#schematicchemoreceptor-structure"><i class="fa fa-check"></i><b>8.1</b> 1[Schematic–Chemoreceptor structure]</a></li>
<li class="chapter" data-level="8.2" data-path="navigation.html"><a href="navigation.html#styphimurium"><i class="fa fa-check"></i><b>8.2</b> Styphimurium</a></li>
<li class="chapter" data-level="8.3" data-path="navigation.html"><a href="navigation.html#schematic-chemosensory-array-architecture"><i class="fa fa-check"></i><b>8.3</b> 2[Schematic: Chemosensory array architecture]</a></li>
<li class="chapter" data-level="8.4" data-path="navigation.html"><a href="navigation.html#methanospirillum-hungatei-1"><i class="fa fa-check"></i><b>8.4</b> Methanospirillum hungatei</a></li>
<li class="chapter" data-level="8.5" data-path="navigation.html"><a href="navigation.html#schematicconservation-of-chemosensory-array-architecture"><i class="fa fa-check"></i><b>8.5</b> 3[Schematic–Conservation of chemosensory array architecture]</a></li>
<li class="chapter" data-level="8.6" data-path="navigation.html"><a href="navigation.html#vibrio-cholerae"><i class="fa fa-check"></i><b>8.6</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="8.7" data-path="navigation.html"><a href="navigation.html#aa-length-difference-of-as-little-as-2-nm-is-enough-to-separate-receptors-like-these-in-azospirillum-brasilense-that-sense-oxygen-28-nm-array-and-sources-of-energy-like-malate-30-nm-array.-both-receptors-send-signals-to-the-single-flagellar-motor-promoting-runs-or-direction-switches-to-guide-the-bacterium-through-a-combination-of-chemotaxis-and-aerotaxis-ordered-movement-in-an-oxygen-gradient-to-its-target-plant-roots.-a.-brasilense-fixes-nitrogen-boosting-the-growth-of-plants-it-colonizes."><i class="fa fa-check"></i><b>8.7</b> 4aA length difference of as little as 2 nm is enough to separate receptors, like these in <em>Azospirillum brasilense</em> that sense oxygen (28 nm array) and sources of energy like malate (30 nm array). Both receptors send signals to the single flagellar motor, promoting runs or direction switches to guide the bacterium through a combination of chemotaxis and aerotaxis (ordered movement in an oxygen gradient) to its target: plant roots. <em>A. brasilense</em> fixes nitrogen, boosting the growth of plants it colonizes.</a></li>
<li class="chapter" data-level="8.8" data-path="navigation.html"><a href="navigation.html#methanoregula-formicica-1"><i class="fa fa-check"></i><b>8.8</b> Methanoregula formicica</a></li>
<li class="chapter" data-level="8.9" data-path="navigation.html"><a href="navigation.html#schematiccytoplasmic-chemosensory-array-structure"><i class="fa fa-check"></i><b>8.9</b> 5[Schematic–Cytoplasmic chemosensory array structure]</a></li>
<li class="chapter" data-level="8.10" data-path="navigation.html"><a href="navigation.html#athis-vibrio-cholerae-cell-showcases-a-full-repertoire-of-chemosensory-systems-two-membrane-embedded-arrays-the-shorter-one-signaling-to-the-flagellar-motor-and-the-longer-one-signaling-to-a-still-unknown-target-and-a-cytoplasmic-array-also-with-unknown-function.-in-all-cases-the-basic-architecture-is-identical-hexagonally-arranged-trimers-of-chemoreceptor-dimers."><i class="fa fa-check"></i><b>8.10</b> 5aThis <em>Vibrio cholerae</em> cell showcases a full repertoire of chemosensory systems: two membrane-embedded arrays (the shorter one signaling to the flagellar motor and the longer one signaling to a still-unknown target) and a cytoplasmic array (also with unknown function). In all cases, the basic architecture is identical: hexagonally-arranged trimers of chemoreceptor dimers.</a></li>
<li class="chapter" data-level="8.11" data-path="navigation.html"><a href="navigation.html#magnetospirillum-magneticum"><i class="fa fa-check"></i><b>8.11</b> Magnetospirillum magneticum</a></li>
<li class="chapter" data-level="8.12" data-path="navigation.html"><a href="navigation.html#schematicmamk-structure"><i class="fa fa-check"></i><b>8.12</b> 6[Schematic–MamK structure]</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="lifecycle.html"><a href="lifecycle.html"><i class="fa fa-check"></i><b>9</b> Lifecycle</a><ul>
<li class="chapter" data-level="9.1" data-path="lifecycle.html"><a href="lifecycle.html#helicobacter-pylori"><i class="fa fa-check"></i><b>9.1</b> Helicobacter pylori</a></li>
<li class="chapter" data-level="9.2" data-path="lifecycle.html"><a href="lifecycle.html#thermococcus-kodakaraensis-1"><i class="fa fa-check"></i><b>9.2</b> Thermococcus kodakaraensis</a></li>
<li class="chapter" data-level="9.3" data-path="lifecycle.html"><a href="lifecycle.html#caulobacter-crescentus-7"><i class="fa fa-check"></i><b>9.3</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="9.3.1" data-path="lifecycle.html"><a href="lifecycle.html#prosthecobacter-vanneervenii-holdfastprosthecobacter-vanneervenii-is-a-close-relative-of-caulobacter-crescentus-and-attaches-to-surfaces-by-the-same-mechanism-a-sticky-ball-of-holdfast-at-the-tip-of-its-stalk.-here-you-see-four-cells-whose-stalks-have-attached-to-one-another.-unlike-its-cousin-c.-crescentus-however-p.-vanneervenii-is-non-motile-no-flagellum-and-does-not-have-a-dimorphic-life-cycle.-it-divides-symmetrically-to-produce-two-daughters-each-with-their-own-stalk."><i class="fa fa-check"></i><b>9.3.1</b> Prosthecobacter vanneervenii {#Holdfast}<em>Prosthecobacter vanneervenii</em> is a close relative of Caulobacter crescentus, and attaches to surfaces by the same mechanism: a sticky ball of holdfast at the tip of its stalk. Here you see four cells whose stalks have attached to one another. Unlike its cousin <em>C. crescentus</em>, however, <em>P. vanneervenii</em> is non-motile (no flagellum) and does not have a dimorphic life cycle. It divides symmetrically to produce two daughters, each with their own stalk.</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis"><i class="fa fa-check"></i><b>9.4</b> Bacillus subtilis</a><ul>
<li class="chapter" data-level="9.4.1" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-sporulation_septum_formationfrom-hundreds-of-sporulating-bacillus-subtilis-we-imaged-by-cryoet-this-is-the-only-cell-captured-in-the-process-of-forming-a-septum.-as-you-can-see-it-is-at-a-very-early-stage-in-the-process.-the-fact-that-we-do-not-see-more-intermediates-suggests-that-once-the-septum-starts-to-form-it-quickly-proceeds-to-completion."><i class="fa fa-check"></i><b>9.4.1</b> Bacillus subtilis {#Sporulation_septum_formation}From hundreds of sporulating <em>Bacillus subtilis</em> we imaged by cryoET, this is the only cell captured in the process of forming a septum. As you can see, it is at a very early stage in the process. The fact that we do not see more intermediates suggests that once the septum starts to form, it quickly proceeds to completion.</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-1"><i class="fa fa-check"></i><b>9.5</b> Bacillus subtilis</a><ul>
<li class="chapter" data-level="9.5.1" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-spore_engulfmentthe-process-of-spore-engulfment-by-the-mother-cell-membrane-culminates-in-closure-of-the-membranes-creating-an-outer-spore-membrane-and-an-inner-spore-membrane-on-either-side-of-the-sporulation-septum.-the-final-point-of-separation-is-usually-at-the-center-of-the-cell-pole-but-not-always-as-you-can-see-in-this-bacillus-subtilis-spore-being-engulfed-from-the-side-of-the-mother-cell.-the-layer-to-the-right-of-the-developing-spore-may-be-a-precursor-of-the-coat-morespore-coating."><i class="fa fa-check"></i><b>9.5.1</b> Bacillus subtilis {#Spore_engulfment}The process of spore engulfment by the mother cell membrane culminates in closure of the membranes, creating an outer spore membrane and an inner spore membrane on either side of the sporulation septum. The final point of separation is usually at the center of the cell pole, but not always, as you can see in this <em>Bacillus subtilis</em> spore being engulfed from the side of the mother cell. The layer to the right of the developing spore may be a precursor of the coat [More–Spore coating].</a></li>
<li class="chapter" data-level="9.5.2" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-spore_coatingin-this-later-stage-of-sporulation-you-can-see-that-the-bacillus-subtilis-spore-has-now-separated-completely-from-the-mother-cell-envelope.-it-has-also-moved-away-from-the-cell-pole-this-sometimes-occurs-even-as-the-envelopes-are-still-attached-morespore-engulfment.-the-layer-flanking-the-spore-on-the-right-may-be-the-assembling-spore-coat."><i class="fa fa-check"></i><b>9.5.2</b> Bacillus subtilis {#Spore_coating}In this later stage of sporulation, you can see that the <em>Bacillus subtilis</em> spore has now separated completely from the mother cell envelope. It has also moved away from the cell pole (this sometimes occurs even as the envelopes are still attached [More–Spore engulfment]). The layer flanking the spore on the right may be the assembling spore coat.</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-2"><i class="fa fa-check"></i><b>9.6</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="9.7" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-3"><i class="fa fa-check"></i><b>9.7</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="9.8" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-1"><i class="fa fa-check"></i><b>9.8</b> Acetonema longum</a></li>
<li class="chapter" data-level="9.9" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-2"><i class="fa fa-check"></i><b>9.9</b> Acetonema longum</a><ul>
<li class="chapter" data-level="9.9.1" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-complete_engulfmentthis-acetonema-longum-cell-has-completed-engulfment-creating-a-forespore.-acetonema-longum-forespores-and-mature-spores-typically-contain-several-granules-likely-storing-poly-phosphate.-these-may-help-power-outgrowth-when-the-spore-germinates.-spores-from-other-species-though-like-bacillus-subtilis-do-not-contain-obvious-storage-granules-so-they-may-serve-another-species-specific-role."><i class="fa fa-check"></i><b>9.9.1</b> Acetonema longum {#Complete_engulfment}This <em>Acetonema longum</em> cell has completed engulfment, creating a “forespore.” <em>Acetonema longum</em> forespores (and mature spores) typically contain several granules, likely storing poly-phosphate. These may help power outgrowth when the spore germinates. Spores from other species, though, like <em>Bacillus subtilis</em>, do not contain obvious storage granules so they may serve another, species-specific role.</a></li>
<li class="chapter" data-level="9.9.2" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-mature_sporethis-mature-acetonema-longum-spore-has-been-released-by-lysis-of-its-mother-cell.-note-its-extensive-armor-a-thick-cortex-between-the-membranes-and-a-multilayered-protein-coat."><i class="fa fa-check"></i><b>9.9.2</b> Acetonema longum {#Mature_spore}This mature <em>Acetonema longum</em> spore has been released by lysis of its mother cell. Note its extensive armor: a thick cortex between the membranes, and a multilayered protein coat.</a></li>
<li class="chapter" data-level="9.9.3" data-path="lifecycle.html"><a href="lifecycle.html#sporomusa-acidovorans-sporomusa_acidovorans_sporulationthis-diderm-sporomusa-acidovorans-cell-is-in-the-process-of-forming-an-ovoid-endospore.-note-also-what-may-be-an-early-stage-of-cortex-formation-at-the-end-of-the-spore-as-well-as-novel-structures-at-the-points-of-cell-wall-remodeling.-we-do-not-know-what-they-are-but-they-may-be-involved-in-engulfment."><i class="fa fa-check"></i><b>9.9.3</b> Sporomusa acidovorans {#Sporomusa_acidovorans_sporulation}This diderm <em>Sporomusa acidovorans</em> cell is in the process of forming an ovoid endospore. Note also what may be an early stage of cortex formation at the end of the spore, as well as novel structures at the points of cell wall remodeling. We do not know what they are, but they may be involved in engulfment.</a></li>
</ul></li>
<li class="chapter" data-level="9.10" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-3"><i class="fa fa-check"></i><b>9.10</b> Acetonema longum</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="interaction.html"><a href="interaction.html"><i class="fa fa-check"></i><b>10</b> Interaction</a><ul>
<li class="chapter" data-level="10.1" data-path="interaction.html"><a href="interaction.html#gluconacetobacter-hansenii"><i class="fa fa-check"></i><b>10.1</b> Gluconacetobacter hansenii</a></li>
<li class="chapter" data-level="10.2" data-path="interaction.html"><a href="interaction.html#legionella-pneumophila"><i class="fa fa-check"></i><b>10.2</b> Legionella pneumophila</a></li>
<li class="chapter" data-level="10.3" data-path="interaction.html"><a href="interaction.html#schematiclegionella-t2ss-structure"><i class="fa fa-check"></i><b>10.3</b> 3[Schematic–Legionella T2SS structure]</a></li>
<li class="chapter" data-level="10.4" data-path="interaction.html"><a href="interaction.html#schematiclegionella-t4ss-structure"><i class="fa fa-check"></i><b>10.4</b> 3[Schematic–Legionella T4SS structure]</a></li>
<li class="chapter" data-level="10.5" data-path="interaction.html"><a href="interaction.html#athis-helicobacter-pylori-cell-was-cultured-together-with-human-gastric-epithelial-cells-its-target-for-infection.-it-contains-inactive-type-iv-secretion-systems-as-well-as-several-mysterious-tubes.-the-tubes-are-formed-from-the-outer-membrane-but-we-do-not-know-what-scaffolds-them-they-have-a-constant-diameter-of-37-nm-or-what-makes-the-portals-along-their-length.-the-fact-that-we-sometimes-see-tubes-near-primed-t4sss-and-the-fact-that-the-tubes-are-missing-from-mutant-cells-lacking-t4ss-components-suggests-a-relationship-but-the-details-are-unclear.-much-remains-mysterious-about-this-structure-and-its-potential-role-in-infection."><i class="fa fa-check"></i><b>10.5</b> 3aThis <em>Helicobacter pylori</em> cell was cultured together with human gastric epithelial cells, its target for infection. It contains inactive type IV secretion systems, as well as several mysterious tubes. The tubes are formed from the outer membrane, but we do not know what scaffolds them (they have a constant diameter of 37 nm), or what makes the portals along their length. The fact that we sometimes see tubes near primed T4SSs, and the fact that the tubes are missing from mutant cells lacking T4SS components suggests a relationship, but the details are unclear. Much remains mysterious about this structure and its potential role in infection.</a></li>
<li class="chapter" data-level="10.6" data-path="interaction.html"><a href="interaction.html#pseudomonas-aeruginosa-δexsd"><i class="fa fa-check"></i><b>10.6</b> Pseudomonas aeruginosa ΔexsD</a></li>
<li class="chapter" data-level="10.7" data-path="interaction.html"><a href="interaction.html#a-emd-2667"><i class="fa fa-check"></i><b>10.7</b> 4a EMD-2667</a></li>
<li class="chapter" data-level="10.8" data-path="interaction.html"><a href="interaction.html#b"><i class="fa fa-check"></i><b>10.8</b> 4b</a></li>
<li class="chapter" data-level="10.9" data-path="interaction.html"><a href="interaction.html#escherichia-coli-cdia"><i class="fa fa-check"></i><b>10.9</b> Escherichia coli +CdiA</a></li>
<li class="chapter" data-level="10.10" data-path="interaction.html"><a href="interaction.html#schematiccdi-mechanism"><i class="fa fa-check"></i><b>10.10</b> 5[Schematic–CDI mechanism]</a></li>
<li class="chapter" data-level="10.11" data-path="interaction.html"><a href="interaction.html#myxococcus-xanthus-2"><i class="fa fa-check"></i><b>10.11</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="10.12" data-path="interaction.html"><a href="interaction.html#schematicprimed-t6ss-structure"><i class="fa fa-check"></i><b>10.12</b> 6[Schematic–Primed T6SS structure]</a></li>
<li class="chapter" data-level="10.13" data-path="interaction.html"><a href="interaction.html#vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.13</b> Vibrio cholerae</a><ul>
<li class="chapter" data-level="10.13.1" data-path="interaction.html"><a href="interaction.html#amoebophilus-asiaticus-t6ss_batterysome-cells-assemble-a-few-scattered-type-vi-secretion-systems.-other-species-like-this-amoebophilus-asiaticus-build-ordered-arrays-of-parallel-machines.-these-diderm-bacteria-live-symbiotically-inside-amoebae.-their-t6ss-batteries-help-them-colonize-their-host-perhaps-by-rupturing-the-membrane-of-the-host-phagosome-that-encloses-them-after-they-are-engulfed-releasing-them-into-the-host-cytoplasm.-the-number-of-t6sss-in-each-array-varies-but-can-reach-a-few-dozen."><i class="fa fa-check"></i><b>10.13.1</b> Amoebophilus asiaticus {#T6SS_battery}Some cells assemble a few, scattered type VI secretion systems. Other species, like this <em>Amoebophilus asiaticus</em> build ordered arrays of parallel machines. These diderm bacteria live symbiotically inside amoebae. Their T6SS batteries help them colonize their host, perhaps by rupturing the membrane of the host phagosome that encloses them after they are engulfed, releasing them into the host cytoplasm. The number of T6SSs in each array varies, but can reach a few dozen.</a></li>
<li class="chapter" data-level="10.13.2" data-path="interaction.html"><a href="interaction.html#pseudoalteromonas-luteoviolacea-δt6ssbacteriocin-macspseudoalteromonas-luteoviolacea-produce-massive-arrays-of-macs-contractile-machines-related-to-pyocins.-cells-lyse-to-release-the-arrays-like-the-one-you-see-here-which-can-contain-100-individual-macs-and-extend-more-than-1-µm-across.-the-arrays-are-highly-ordered-with-the-tail-fibers-of-the-macs-networked-into-a-hexagonal-array-that-may-help-synchronize-firing-schematicmac-array-structure.-mac-arrays-serve-an-important-function-for-larvae-of-the-marine-tubeworm-hydroides-elegans-and-likely-the-larvae-of-other-invertebrates-such-as-sea-urchins-and-corals-as-well.-they-signal-to-the-larva-that-it-is-the-right-place-to-settle-down-and-differentiate-into-a-sessile-surface-attached-adult.-the-function-of-mac-arrays-for-the-bacterium-is-less-clear.-it-may-be-that-the-invertebrates-benefit-the-lysed-cells-surviving-relatives-in-the-biofilm.-or-perhaps-the-macs-play-another-completely-unrelated-role."><i class="fa fa-check"></i><b>10.13.2</b> Pseudoalteromonas luteoviolacea ΔT6SS/bacteriocin {#MACs}<em>Pseudoalteromonas luteoviolacea</em> produce massive arrays of MACs, contractile machines related to pyocins. Cells lyse to release the arrays, like the one you see here, which can contain 100 individual MACs and extend more than 1 µm across. The arrays are highly ordered, with the tail fibers of the MACs networked into a hexagonal array that may help synchronize firing [Schematic–MAC array structure]. MAC arrays serve an important function for larvae of the marine tubeworm <em>Hydroides elegans</em>, and likely the larvae of other invertebrates such as sea urchins and corals as well. They signal to the larva that it is the right place to settle down and differentiate into a sessile (surface-attached) adult. The function of MAC arrays for the bacterium is less clear. It may be that the invertebrates benefit the lysed cell’s surviving relatives in the biofilm. Or perhaps the MACs play another, completely unrelated role.</a></li>
</ul></li>
<li class="chapter" data-level="10.14" data-path="interaction.html"><a href="interaction.html#schematicmac-array-structure"><i class="fa fa-check"></i><b>10.14</b> 7[Schematic–MAC array structure]</a></li>
<li class="chapter" data-level="10.15" data-path="interaction.html"><a href="interaction.html#porphyromonas-gingivalis"><i class="fa fa-check"></i><b>10.15</b> Porphyromonas gingivalis</a></li>
<li class="chapter" data-level="10.16" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-vibrio-cholerae"><i class="fa fa-check"></i><b>10.16</b> Bdellovibrio bacteriovorus + Vibrio cholerae</a></li>
<li class="chapter" data-level="10.17" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-escherichia-coli-minicell"><i class="fa fa-check"></i><b>10.17</b> Bdellovibrio bacteriovorus + Escherichia coli minicell</a><ul>
<li class="chapter" data-level="10.17.1" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-escherichia-coli-minicell-bdellovibrio_bacteriovorus_entrywe-still-do-not-know-exactly-how-bdellovibrio-bacteriovorus-enters-its-host-but-the-process-is-extremely-rapid.-in-order-to-catch-a-snapshot-we-mixed-b.-bacteriovorus-with-miniaturized-prey-see-details-on-the-main-page.-as-the-b.-bacteriovorus-cell-cannot-fit-inside-the-prey-it-remains-stuck-half-in-and-half-out.-note-the-tight-collar-of-host-outer-membrane-around-the-invader.-how-this-seal-is-maintained-throughout-invasion-remains-a-mystery."><i class="fa fa-check"></i><b>10.17.1</b> Bdellovibrio bacteriovorus + Escherichia coli minicell {#Bdellovibrio_bacteriovorus_entry}We still do not know exactly how <em>Bdellovibrio bacteriovorus</em> enters its host, but the process is extremely rapid. In order to catch a snapshot, we mixed <em>B. bacteriovorus</em> with miniaturized prey (see details on the main page). As the <em>B. bacteriovorus</em> cell cannot fit inside the prey, it remains stuck half-in and half-out. Note the tight collar of host outer membrane around the invader. How this seal is maintained throughout invasion remains a mystery.</a></li>
</ul></li>
<li class="chapter" data-level="10.18" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.18</b> Bdellovibrio bacteriovorus + Vibrio cholerae</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="viruses.html"><a href="viruses.html"><i class="fa fa-check"></i><b>11</b> Viruses</a><ul>
<li class="chapter" data-level="11.1" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-3"><i class="fa fa-check"></i><b>11.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="11.2" data-path="viruses.html"><a href="viruses.html#bacteroidetes-strain-jt5"><i class="fa fa-check"></i><b>11.2</b> Bacteroidetes strain JT5</a></li>
<li class="chapter" data-level="11.3" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-4"><i class="fa fa-check"></i><b>11.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="11.4" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-2"><i class="fa fa-check"></i><b>11.4</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="11.5" data-path="viruses.html"><a href="viruses.html#escherichia-coli-1"><i class="fa fa-check"></i><b>11.5</b> Escherichia coli</a></li>
<li class="chapter" data-level="11.6" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus-8"><i class="fa fa-check"></i><b>11.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="11.6.1" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus-φcb13_infectionhere-you-see-several-φcb13-phage-that-have-successfully-reached-their-target-receptors-at-the-pole-of-a-caulobacter-crescentus-and-are-in-the-process-of-infecting-the-cell-note-the-mix-of-filled-capsids-that-have-not-yet-released-their-genome-and-empty-capsids-that-have."><i class="fa fa-check"></i><b>11.6.1</b> Caulobacter crescentus {#φCb13_infection}Here you see several φCb13 phage that have successfully reached their target receptors at the pole of a <em>Caulobacter crescentus</em> and are in the process of infecting the cell (note the mix of filled capsids that have not yet released their genome and empty capsids that have).</a></li>
</ul></li>
<li class="chapter" data-level="11.7" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-3"><i class="fa fa-check"></i><b>11.7</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="11.8" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-1"><i class="fa fa-check"></i><b>11.8</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="11.9" data-path="viruses.html"><a href="viruses.html#schematicvap-structure"><i class="fa fa-check"></i><b>11.9</b> 8[Schematic–VAP structure]</a></li>
<li class="chapter" data-level="11.10" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-2"><i class="fa fa-check"></i><b>11.10</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="11.11" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-5"><i class="fa fa-check"></i><b>11.11</b> Shewanella oneidensis</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>12</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.library.caltech.edu" target="blank">Published by Caltech Library</a><li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cell Atlas</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="navigation" class="section level1">
<h1><span class="header-section-number">Chapter 8</span> Navigation</h1>
<p>“<em>E. coli</em> forgets where it is going in about 10 seconds.” - Howard Berg <span class="citation">H. C. Berg (<a href="#ref-berg1988">1988</a>)</span></p>
<p>Chemotaxis ## Shewanella oneidensis</p>
<p>Now that your cell can move, it needs to figure out which way to go. Perhaps it should take a sniff. Chemosensory systems are ancient (they were already present in the common ancestor of bacteria and archaea) and widespread, reflecting their great utility. Chemosensory systems are two-component signaling systems. The first component is a receptor that binds a specific chemical, such as a sugar or amino acid. The binding results in a conformational change that propagates down the long receptor [Schematic–Chemoreceptor structure], turning off a kinase bound at the other end that controls the state of the second component: a response regulator. These response regulator proteins carry the signal elsewhere in the cell by diffusion.</p>
<p>For chemotaxis (“taxis,” or ordered movement, in response to chemicals) the signal is carried to the cell’s motility machinery, usually the flagellar motor, but also type IV pili. Phosphorylated response regulators bind the flagellar motor, switching the direction of rotation. This produces different results depending on the pattern of flagella on the cell. If there is a single flagellum, as on this <em>Shewanella oneidensis</em> cell, it switches the flagellum between pushing and pulling the cell body, reorienting the cell in the process. In peritrichously-flagellated cells like <em>Escherichia coli</em>, it brings the flagella into and out of a bundle. Remember that bundled flagella drive the cell forward in straight “runs” and dissociated flagella “tumble” the cell to try a new direction. Constant feedback from the chemosensory system switches the balance of phosphorylated/unphosphorylated response regulator and therefore keeps the cell heading in the general direction of an attractant chemical cue, or away from a repellant.</p>
<p>Chemosensory systems form arrays containing many copies of the proteins. As in this <em>S. oneidensis</em> cell, arrays are usually located near the flagella they control, with the tips of the chemoreceptors sticking through the membrane into the periplasm or extracellular space where they can detect signals from the environment. At the other end of the array, the associated kinases interact with the response regulators. These kinases, along with an additional structural protein that helps organize the array, form a layer that we see as a dense line in the cytoplasm.</p>
<div class="figure"><span id="fig:unnamed-chunk-122"></span>
<div id="htmlwidget-d5615589edfe7ece00f2" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-d5615589edfe7ece00f2">{"x":{"doi":"10.22002/D1.1545","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.1: Shewanella oneidensis Collected by: Mohammed Kaplan <a href="https://doi.org/10.22002/D1.1545">10.22002/D1.1545</a>
</p>
</div>
<div id="schematicchemoreceptor-structure" class="section level2">
<h2><span class="header-section-number">8.1</span> 1[Schematic–Chemoreceptor structure]</h2>
Chemosensory Array
<div class="figure"><span id="fig:unnamed-chunk-123"></span>
<div id="htmlwidget-66fa007c70147313c9e3" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-66fa007c70147313c9e3">{"x":{"doi":"10.22002/D1.1545","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.2: Shewanella oneidensis Collected by: Mohammed Kaplan <a href="https://doi.org/10.22002/D1.1545">10.22002/D1.1545</a>
</p>
</div>
</div>
<div id="styphimurium" class="section level2">
<h2><span class="header-section-number">8.2</span> Styphimurium</h2>
<p>Chemosensory arrays are highly ordered, as you can best see from a bird’s-eye view, as in this lysed <em>Salmonella typhimurium</em> cell. Chemoreceptors come together as dimers, which in turn organize into trimers, which are further packed into the extensive hexagonal honeycombed array you see here. The hexagonal arrangement comes from the baseplate, where the kinases and coupling proteins bind into an ordered array [Schematic–Chemosensory array architecture]. In what should be a familiar theme by now, organization provides a great benefit. Bacteria and archaea have a tremendous sense of smell, responding to as few as one or two molecules of an attractant, or many more. In fact, the range of chemical concentrations they can sense extends over 5 orders of magnitude. But how can a single receptor transmit a signal efficiently? Perhaps it should pass on the message to its neighbors. The interlocking network of chemoreceptors enables just this kind of amplification; a target binding to one receptor may translate into activation of 36 adjacent receptors, enormously boosting the gain of the signal.</p>
<div class="figure"><span id="fig:unnamed-chunk-124"></span>
<div id="htmlwidget-0adcffc137d24beba977" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-0adcffc137d24beba977">{"x":{"doi":"10.22002/D1.1546","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.3: Salmonella typhimurium Collected by: Morgan Beeby <a href="https://doi.org/10.22002/D1.1546">10.22002/D1.1546</a>
</p>
</div>
</div>
<div id="schematic-chemosensory-array-architecture" class="section level2">
<h2><span class="header-section-number">8.3</span> 2[Schematic: Chemosensory array architecture]</h2>
Chemosensory Array Conservation
<div class="figure"><span id="fig:unnamed-chunk-125"></span>
<div id="htmlwidget-fd00e9f7168b6eeafb59" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-fd00e9f7168b6eeafb59">{"x":{"doi":"10.22002/D1.1546","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.4: Salmonella typhimurium Collected by: Morgan Beeby <a href="https://doi.org/10.22002/D1.1546">10.22002/D1.1546</a>
</p>
</div>
</div>
<div id="methanospirillum-hungatei-1" class="section level2">
<h2><span class="header-section-number">8.4</span> Methanospirillum hungatei</h2>
<p>The hexagonal array structure of chemosensory systems is invariant across species [Schematic–Chemosensory array conservation], and even across domains. As you can see in this <em>Methanospirillum hungatei</em> cell, the arrays have the same architecture in archaea as in bacteria, with 12 nm center-to-center spacing between the hexagons. This strong conservation indicates the importance of chemotaxis for cells’ fitness, and suggests that evolution had already honed it to a relatively optimal form in the common ancestor of all these cells.</p>
<div class="figure"><span id="fig:unnamed-chunk-126"></span>
<div id="htmlwidget-a667620bf5ed22e087ab" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-a667620bf5ed22e087ab">{"x":{"doi":"10.22002/D1.1547","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.5: Methanospirillum hungatei Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1547">10.22002/D1.1547</a>
</p>
</div>
</div>
<div id="schematicconservation-of-chemosensory-array-architecture" class="section level2">
<h2><span class="header-section-number">8.5</span> 3[Schematic–Conservation of chemosensory array architecture]</h2>
Chemoreceptors
<div class="figure"><span id="fig:unnamed-chunk-127"></span>
<div id="htmlwidget-6d310d241d8a3f51c93b" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-6d310d241d8a3f51c93b">{"x":{"doi":"10.22002/D1.1547","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.6: Methanospirillum hungatei Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1547">10.22002/D1.1547</a>
</p>
</div>
</div>
<div id="vibrio-cholerae" class="section level2">
<h2><span class="header-section-number">8.6</span> Vibrio cholerae</h2>
<p>While there is only one architectural style for chemosensory systems, there are many different building materials. Each chemoreceptor senses one, or perhaps two related, types of chemicals. This means that if your cell wants to sense multiple chemicals in its environment, it needs a matching assortment of chemoreceptors. This can quickly get complicated. Some signals need to go to the flagella, some to the pili, and others to the transcriptional machinery to turn on or off genes. How can you keep the wires from getting crossed? The simplest approach is just to separate them, and that seems to be exactly what cells do. Chemoreceptors that signal to different systems have different lengths, which sorts them into separate arrays in the cell, as you can see in this <em>Vibrio cholerae</em>. The shorter array signals to the flagellar motor, and the longer array signals to a different target we have not yet identified. Even when multiple chemosensory systems signal to the same target, arrays that respond to different environmental cues are kept separate by length differences in the receptors [More–Azospirillum brasilense aerotaxis].</p>
<div class="figure"><span id="fig:unnamed-chunk-128"></span>
<div id="htmlwidget-e5e9df8841ed25e05649" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-e5e9df8841ed25e05649">{"x":{"doi":"10.22002/D1.1548","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.7: Vibrio cholerae Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1548">10.22002/D1.1548</a>
</p>
</div>
</div>
<div id="aa-length-difference-of-as-little-as-2-nm-is-enough-to-separate-receptors-like-these-in-azospirillum-brasilense-that-sense-oxygen-28-nm-array-and-sources-of-energy-like-malate-30-nm-array.-both-receptors-send-signals-to-the-single-flagellar-motor-promoting-runs-or-direction-switches-to-guide-the-bacterium-through-a-combination-of-chemotaxis-and-aerotaxis-ordered-movement-in-an-oxygen-gradient-to-its-target-plant-roots.-a.-brasilense-fixes-nitrogen-boosting-the-growth-of-plants-it-colonizes." class="section level2">
<h2><span class="header-section-number">8.7</span> 4aA length difference of as little as 2 nm is enough to separate receptors, like these in <em>Azospirillum brasilense</em> that sense oxygen (28 nm array) and sources of energy like malate (30 nm array). Both receptors send signals to the single flagellar motor, promoting runs or direction switches to guide the bacterium through a combination of chemotaxis and aerotaxis (ordered movement in an oxygen gradient) to its target: plant roots. <em>A. brasilense</em> fixes nitrogen, boosting the growth of plants it colonizes.</h2>
Cytoplasmic Chemosensory Array
<div class="figure"><span id="fig:unnamed-chunk-129"></span>
<div id="htmlwidget-6a01630ed6a352814ecf" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-6a01630ed6a352814ecf">{"x":{"doi":"10.22002/D1.1548","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.8: Vibrio cholerae Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1548">10.22002/D1.1548</a>
</p>
</div>
</div>
<div id="methanoregula-formicica-1" class="section level2">
<h2><span class="header-section-number">8.8</span> Methanoregula formicica</h2>
<p>In addition to their extensive repertoire of membrane-embedded chemoreceptors, cells may also have another, cytoplasmic, type. Many species of bacteria and archaea, like this <em>Methanoregula formicica</em>, contain cytoplasmic chemosensory arrays. The receptors lack membrane insertion patches. Instead, they interact end-to-end with each other at their chemical-sensing tips, zippering into a double-layered sandwich, with a baseplate of kinases and coupling proteins on either side [Schematic–Cytoplasmic chemosensory array structure]. In some species, the arrays are curved like this; in others, they are straight [More–Vibrio cholerae chemosensory arrays]. What these systems sense remains a mystery; one idea is that they monitor the internal state of the cell (e.g. the levels of various metabolites) to fine-tune signaling responses to meet the current needs of the cell. They are often found along with, and often quite close to, membrane-embedded chemosensory arrays [More–Vibrio cholerae chemosensory arrays].</p>
<div class="figure"><span id="fig:unnamed-chunk-130"></span>
<div id="htmlwidget-887f706d834a1e7ff5e5" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-887f706d834a1e7ff5e5">{"x":{"doi":"10.22002/D1.1549","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.9: Methanoregula formicica Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1549">10.22002/D1.1549</a>
</p>
</div>
</div>
<div id="schematiccytoplasmic-chemosensory-array-structure" class="section level2">
<h2><span class="header-section-number">8.9</span> 5[Schematic–Cytoplasmic chemosensory array structure]</h2>
<div class="figure"><span id="fig:unnamed-chunk-131"></span>
<div id="htmlwidget-bee85b0f0a789fb28ce1" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-bee85b0f0a789fb28ce1">{"x":{"doi":"10.22002/D1.1549","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.10: Methanoregula formicica Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1549">10.22002/D1.1549</a>
</p>
</div>
</div>
<div id="athis-vibrio-cholerae-cell-showcases-a-full-repertoire-of-chemosensory-systems-two-membrane-embedded-arrays-the-shorter-one-signaling-to-the-flagellar-motor-and-the-longer-one-signaling-to-a-still-unknown-target-and-a-cytoplasmic-array-also-with-unknown-function.-in-all-cases-the-basic-architecture-is-identical-hexagonally-arranged-trimers-of-chemoreceptor-dimers." class="section level2">
<h2><span class="header-section-number">8.10</span> 5aThis <em>Vibrio cholerae</em> cell showcases a full repertoire of chemosensory systems: two membrane-embedded arrays (the shorter one signaling to the flagellar motor and the longer one signaling to a still-unknown target) and a cytoplasmic array (also with unknown function). In all cases, the basic architecture is identical: hexagonally-arranged trimers of chemoreceptor dimers.</h2>
Magnetotaxis
<div class="figure"><span id="fig:unnamed-chunk-132"></span>
<div id="htmlwidget-85a4013cbde1112f626c" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-85a4013cbde1112f626c">{"x":{"doi":"10.22002/D1.1549","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.11: Methanoregula formicica Collected by: Ariane Briegel <a href="https://doi.org/10.22002/D1.1549">10.22002/D1.1549</a>
</p>
</div>
</div>
<div id="magnetospirillum-magneticum" class="section level2">
<h2><span class="header-section-number">8.11</span> Magnetospirillum magneticum</h2>
<p>Chemotaxis is not the only mode of navigation available to your cell. For a clue to others, think about how we orient ourselves in the world. In addition to our nose sniffing out food, we also use our eyes to sense light. Similarly, some photosynthetic bacteria have evolved phototaxis (ordered movement in response to light).</p>
<p>We may also use an external tool to navigate unfamiliar surroundings: a compass. Believe it or not, some bacteria have one, too. Magnetotactic species like this <em>Magnetospirillum magneticum</em> have evolved specialized structures called magnetosomes. They are pockets of inner membrane filled with crystals of a magnetic iron mineral like magnetite. The cell organizes the magnetosomes into a line using filaments of a cytoskeletal protein called MamK [Schematic–MamK structure], which is related to eukaryotic actin. The linear chain of magnetosomes functions like the needle of a compass, aligning the cell in a magnetic field.</p>
<p>Magnetosomes first form with the creation of a pocket of inner membrane, as you can see in this cell. Multiple short chains of magnetosomes may form, which are then organized into a single chain by MamK filaments. As mineralization begins, the pockets enlarge to accommodate the growing crystals, ultimately reaching ~60 nm in diameter. Magnetosome chains are inherited by daughter cells through division. Segregation is relatively easy: the chain spans the division plane so that the inward-growing wedge of cell wall simply splits the chain between magnetosomes, delivering half the chain to each daughter.</p>
<p>Much remains mysterious about these structures. For instance, what forms the membrane pockets? An even bigger question is how cells use their compass. One theory is that magnetosomes guide the aquatic bacteria vertically to the optimal height in the water column for, e.g. oxygen level. However, these species are also found at the magnetic equator, where the Earth’s field is horizontal. Another hypothesis is that magnetic orientation improves the efficiency of navigation by buffering the cell against Brownian motion that would knock it off course.</p>
<div class="figure"><span id="fig:unnamed-chunk-133"></span>
<div id="htmlwidget-7e35a55c5648f22115da" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-7e35a55c5648f22115da">{"x":{"doi":"10.22002/D1.1550","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.12: Magnetospirillum magneticum Collected by: Zhuo Li <a href="https://doi.org/10.22002/D1.1550">10.22002/D1.1550</a>
</p>
</div>
</div>
<div id="schematicmamk-structure" class="section level2">
<h2><span class="header-section-number">8.12</span> 6[Schematic–MamK structure]</h2>
<p>References</p>
<p>Further Reading</p>
<span class="citation">H. C. Berg (<a href="#ref-berg1988">1988</a>)</span> <span class="citation">Hazelbauer, Falke, and Parkinson (<a href="#ref-hazelbauer2008">2008</a>)</span> <span class="citation">(<span class="citeproc-not-found" data-reference-id="lower2013"><strong>???</strong></span>)</span> <span class="citation">Schuergers et al. (<a href="#ref-schuergers2016">2016</a>)</span>
<div class="figure"><span id="fig:unnamed-chunk-134"></span>
<div id="htmlwidget-ae21aacc736645e5963e" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-ae21aacc736645e5963e">{"x":{"doi":"10.22002/D1.1550","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 8.13: Magnetospirillum magneticum Collected by: Zhuo Li <a href="https://doi.org/10.22002/D1.1550">10.22002/D1.1550</a>
</p>
</div>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-berg1988">
<p>Berg, H. C. 1988. “A Physicist Looks at Bacterial Chemotaxis.” <em>Cold Spring Harb Symp Quant Biol</em> 53 Pt 1: 1–9.</p>
</div>
<div id="ref-hazelbauer2008">
<p>Hazelbauer, Gerald L., Joseph J. Falke, and John S. Parkinson. 2008. “Bacterial Chemoreceptors: High-Performance Signaling in Networked Arrays.” <em>Trends in Biochemical Sciences</em> 33 (1): 9–19. doi:<a href="https://doi.org/10.1016/j.tibs.2007.09.014">10.1016/j.tibs.2007.09.014</a>.</p>
</div>
<div id="ref-schuergers2016">
<p>Schuergers, N., T. Lenn, R. Kampmann, M. V. Meissner, T. Esteves, M. Temerinac-Ott, J. G. Korvink, A. R. Lowe, C. W. Mullineaux, and A. Wilde. 2016. “Cyanobacteria Use Micro-Optics to Sense Light Direction.” <em>Elife</em> 5 (February). doi:<a href="https://doi.org/10.7554/eLife.12620">10.7554/eLife.12620</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="motility.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="lifecycle.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": null,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/caltechlibrary/cell-atlas/edit/master/07_Navigation.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["cell-atlas.pdf", "cell-atlas.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
