<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 10 Viruses | Atlas of Bacterial and Archaeal Cell Structure</title>
  <meta name="description" content="A visual tour of the microbial cell." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 10 Viruses | Atlas of Bacterial and Archaeal Cell Structure" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="img/AtlasLogo.png" />
  <meta property="og:description" content="A visual tour of the microbial cell." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 10 Viruses | Atlas of Bacterial and Archaeal Cell Structure" />
  
  <meta name="twitter:description" content="A visual tour of the microbial cell." />
  <meta name="twitter:image" content="img/AtlasLogo.png" />

<meta name="author" content="Catherine M. Oikonomou and Grant J. Jensen" />


<meta name="date" content="2020-08-05" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="interaction.html"/>
<link rel="next" href="references.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/CL-0.1.2/CL.js"></script>
<script src="libs/doivideo-binding-0.0.0.9000/doivideo.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Cell Atlas</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>1</b> Methods</a><ul>
<li class="chapter" data-level="1.1" data-path="methods.html"><a href="methods.html#light-microscopy"><i class="fa fa-check"></i><b>1.1</b> Light Microscopy</a><ul>
<li class="chapter" data-level="1.1.1" data-path="methods.html"><a href="methods.html#Structural_biology_toolkit"><i class="fa fa-check"></i><b>1.1.1</b> Structural biology toolkit</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="methods.html"><a href="methods.html#fluorescence-light-microscopy"><i class="fa fa-check"></i><b>1.2</b> Fluorescence Light Microscopy</a></li>
<li class="chapter" data-level="1.3" data-path="methods.html"><a href="methods.html#scanning-electron-microscopy"><i class="fa fa-check"></i><b>1.3</b> Scanning Electron Microscopy</a><ul>
<li class="chapter" data-level="1.3.1" data-path="methods.html"><a href="methods.html#Electron_microscopy_modes"><i class="fa fa-check"></i><b>1.3.1</b> Electron microscopy modes</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="methods.html"><a href="methods.html#transmission-electron-microscopy"><i class="fa fa-check"></i><b>1.4</b> Transmission Electron Microscopy</a></li>
<li class="chapter" data-level="1.5" data-path="methods.html"><a href="methods.html#cryogenic-electron-microscopy"><i class="fa fa-check"></i><b>1.5</b> Cryogenic Electron Microscopy</a></li>
<li class="chapter" data-level="1.6" data-path="methods.html"><a href="methods.html#electron-cryotomography"><i class="fa fa-check"></i><b>1.6</b> Electron Cryotomography</a><ul>
<li class="chapter" data-level="1.6.1" data-path="methods.html"><a href="methods.html#Missing_wedge"><i class="fa fa-check"></i><b>1.6.1</b> Missing wedge</a></li>
</ul></li>
<li class="chapter" data-level="1.7" data-path="methods.html"><a href="methods.html#sub-tomogram-averaging"><i class="fa fa-check"></i><b>1.7</b> Sub-Tomogram Averaging</a></li>
<li class="chapter" data-level="1.8" data-path="methods.html"><a href="methods.html#single-particle-reconstruction"><i class="fa fa-check"></i><b>1.8</b> Single Particle Reconstruction</a></li>
<li class="chapter" data-level="1.9" data-path="methods.html"><a href="methods.html#x-ray-crystallography"><i class="fa fa-check"></i><b>1.9</b> X-Ray Crystallography</a><ul>
<li class="chapter" data-level="1.9.1" data-path="methods.html"><a href="methods.html#Structural_biology_timeline"><i class="fa fa-check"></i><b>1.9.1</b> Structural biology timeline</a></li>
</ul></li>
<li class="chapter" data-level="1.10" data-path="methods.html"><a href="methods.html#putting-it-all-together"><i class="fa fa-check"></i><b>1.10</b> Putting It All Together</a></li>
<li class="chapter" data-level="1.11" data-path="methods.html"><a href="methods.html#moredifference-mapping"><i class="fa fa-check"></i><b>1.11</b> More–Difference mapping</a></li>
<li class="chapter" data-level="1.12" data-path="methods.html"><a href="methods.html#further-reading"><i class="fa fa-check"></i><b>1.12</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="envelope.html"><a href="envelope.html"><i class="fa fa-check"></i><b>2</b> Envelope</a><ul>
<li class="chapter" data-level="2.1" data-path="envelope.html"><a href="envelope.html#membrane"><i class="fa fa-check"></i><b>2.1</b> Membrane</a><ul>
<li class="chapter" data-level="2.1.1" data-path="envelope.html"><a href="envelope.html#Lipid_bilayer"><i class="fa fa-check"></i><b>2.1.1</b> Lipid bilayer</a></li>
<li class="chapter" data-level="2.1.2" data-path="envelope.html"><a href="envelope.html#PDB:_5T4O_ATP_synthase_and_energy_production"><i class="fa fa-check"></i><b>2.1.2</b> PDB: 5T4O ATP synthase and energy production</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="envelope.html"><a href="envelope.html#cell-wall"><i class="fa fa-check"></i><b>2.2</b> Cell Wall</a></li>
<li class="chapter" data-level="2.3" data-path="envelope.html"><a href="envelope.html#morepeptidoglycan-architecture"><i class="fa fa-check"></i><b>2.3</b> More–Peptidoglycan architecture</a></li>
<li class="chapter" data-level="2.4" data-path="envelope.html"><a href="envelope.html#moremethanobacterium-formicicum"><i class="fa fa-check"></i><b>2.4</b> More–Methanobacterium formicicum</a></li>
<li class="chapter" data-level="2.5" data-path="envelope.html"><a href="envelope.html#outer-membrane"><i class="fa fa-check"></i><b>2.5</b> Outer Membrane</a><ul>
<li class="chapter" data-level="2.5.1" data-path="envelope.html"><a href="envelope.html#pdb-1eq7-brauns-lipoprotein-pdb1eq7brauns_lipoprotein"><i class="fa fa-check"></i><b>2.5.1</b> PDB: 1EQ7 –Braun’s lipoprotein {#PDB:<em>1EQ7</em>–Brauns_lipoprotein}</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="envelope.html"><a href="envelope.html#morediderm-sacculus-architecture"><i class="fa fa-check"></i><b>2.6</b> More–Diderm sacculus architecture</a><ul>
<li class="chapter" data-level="2.6.1" data-path="envelope.html"><a href="envelope.html#Sacculus_remodeling"><i class="fa fa-check"></i><b>2.6.1</b> Sacculus remodeling</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="envelope.html"><a href="envelope.html#vesicles"><i class="fa fa-check"></i><b>2.7</b> Vesicles</a></li>
<li class="chapter" data-level="2.8" data-path="envelope.html"><a href="envelope.html#morepearled-vesicles"><i class="fa fa-check"></i><b>2.8</b> More–Pearled vesicles</a></li>
<li class="chapter" data-level="2.9" data-path="envelope.html"><a href="envelope.html#moretubular-vesicles"><i class="fa fa-check"></i><b>2.9</b> More–Tubular vesicles</a></li>
<li class="chapter" data-level="2.10" data-path="envelope.html"><a href="envelope.html#morecytoplasmic-vesicles"><i class="fa fa-check"></i><b>2.10</b> More–Cytoplasmic vesicles</a></li>
<li class="chapter" data-level="2.11" data-path="envelope.html"><a href="envelope.html#more-cytoplasmic-vesicle-variety"><i class="fa fa-check"></i><b>2.11</b> More: Cytoplasmic vesicle variety</a></li>
<li class="chapter" data-level="2.12" data-path="envelope.html"><a href="envelope.html#morearchaeal-vesicles"><i class="fa fa-check"></i><b>2.12</b> More–Archaeal vesicles</a></li>
<li class="chapter" data-level="2.13" data-path="envelope.html"><a href="envelope.html#variation"><i class="fa fa-check"></i><b>2.13</b> Variation</a></li>
<li class="chapter" data-level="2.14" data-path="envelope.html"><a href="envelope.html#surface-layer"><i class="fa fa-check"></i><b>2.14</b> Surface Layer</a><ul>
<li class="chapter" data-level="2.14.1" data-path="envelope.html"><a href="envelope.html#PDB:_5N97_Surface_layer_architecture"><i class="fa fa-check"></i><b>2.14.1</b> PDB: 5N97 Surface layer architecture</a></li>
</ul></li>
<li class="chapter" data-level="2.15" data-path="envelope.html"><a href="envelope.html#morenanopods"><i class="fa fa-check"></i><b>2.15</b> More–Nanopods</a></li>
<li class="chapter" data-level="2.16" data-path="envelope.html"><a href="envelope.html#surface-layer-variety"><i class="fa fa-check"></i><b>2.16</b> Surface Layer Variety</a></li>
<li class="chapter" data-level="2.17" data-path="envelope.html"><a href="envelope.html#morem.-alcaliphilum"><i class="fa fa-check"></i><b>2.17</b> More–M. alcaliphilum</a></li>
<li class="chapter" data-level="2.18" data-path="envelope.html"><a href="envelope.html#moreclostridium-thermocellum"><i class="fa fa-check"></i><b>2.18</b> More–Clostridium thermocellum</a></li>
<li class="chapter" data-level="2.19" data-path="envelope.html"><a href="envelope.html#moren.-maritimus"><i class="fa fa-check"></i><b>2.19</b> More–N. maritimus</a></li>
<li class="chapter" data-level="2.20" data-path="envelope.html"><a href="envelope.html#moremethanoregula-formicica"><i class="fa fa-check"></i><b>2.20</b> More–Methanoregula formicica</a></li>
<li class="chapter" data-level="2.21" data-path="envelope.html"><a href="envelope.html#sheath"><i class="fa fa-check"></i><b>2.21</b> Sheath</a></li>
<li class="chapter" data-level="2.22" data-path="envelope.html"><a href="envelope.html#dna"><i class="fa fa-check"></i><b>2.22</b> DNA</a></li>
<li class="chapter" data-level="2.23" data-path="envelope.html"><a href="envelope.html#nucleoid"><i class="fa fa-check"></i><b>2.23</b> Nucleoid</a></li>
<li class="chapter" data-level="2.24" data-path="envelope.html"><a href="envelope.html#further-reading-1"><i class="fa fa-check"></i><b>2.24</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="shape.html"><a href="shape.html"><i class="fa fa-check"></i><b>3</b> Shape</a><ul>
<li class="chapter" data-level="3.1" data-path="shape.html"><a href="shape.html#coccoid"><i class="fa fa-check"></i><b>3.1</b> Coccoid</a></li>
<li class="chapter" data-level="3.2" data-path="shape.html"><a href="shape.html#rod"><i class="fa fa-check"></i><b>3.2</b> Rod</a></li>
<li class="chapter" data-level="3.3" data-path="shape.html"><a href="shape.html#morerod-variety"><i class="fa fa-check"></i><b>3.3</b> More–Rod variety</a></li>
<li class="chapter" data-level="3.4" data-path="shape.html"><a href="shape.html#morearchaeal-rods"><i class="fa fa-check"></i><b>3.4</b> More–Archaeal rods</a></li>
<li class="chapter" data-level="3.5" data-path="shape.html"><a href="shape.html#length"><i class="fa fa-check"></i><b>3.5</b> Length</a></li>
<li class="chapter" data-level="3.6" data-path="shape.html"><a href="shape.html#vibrioid"><i class="fa fa-check"></i><b>3.6</b> Vibrioid</a><ul>
<li class="chapter" data-level="3.6.1" data-path="shape.html"><a href="shape.html#PDB:_5U3C_CTP_synthase"><i class="fa fa-check"></i><b>3.6.1</b> PDB: 5U3C CTP synthase</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="shape.html"><a href="shape.html#helical"><i class="fa fa-check"></i><b>3.7</b> Helical</a></li>
<li class="chapter" data-level="3.8" data-path="shape.html"><a href="shape.html#moreborrelia-burgdorferi"><i class="fa fa-check"></i><b>3.8</b> More–Borrelia burgdorferi</a></li>
<li class="chapter" data-level="3.9" data-path="shape.html"><a href="shape.html#prosthecate"><i class="fa fa-check"></i><b>3.9</b> Prosthecate</a><ul>
<li class="chapter" data-level="3.9.1" data-path="shape.html"><a href="shape.html#PDB:_6RIB_Bactofilin"><i class="fa fa-check"></i><b>3.9.1</b> PDB: 6RIB Bactofilin</a></li>
</ul></li>
<li class="chapter" data-level="3.10" data-path="shape.html"><a href="shape.html#morebacterial-microtubules"><i class="fa fa-check"></i><b>3.10</b> More–Bacterial microtubules</a></li>
<li class="chapter" data-level="3.11" data-path="shape.html"><a href="shape.html#square"><i class="fa fa-check"></i><b>3.11</b> Square</a></li>
<li class="chapter" data-level="3.12" data-path="shape.html"><a href="shape.html#moregas-vesicles"><i class="fa fa-check"></i><b>3.12</b> More–Gas vesicles</a></li>
<li class="chapter" data-level="3.13" data-path="shape.html"><a href="shape.html#further-reading-2"><i class="fa fa-check"></i><b>3.13</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="growth.html"><a href="growth.html"><i class="fa fa-check"></i><b>4</b> Growth</a><ul>
<li class="chapter" data-level="4.1" data-path="growth.html"><a href="growth.html#diffusion"><i class="fa fa-check"></i><b>4.1</b> Diffusion</a></li>
<li class="chapter" data-level="4.2" data-path="growth.html"><a href="growth.html#anaerobic-respiration"><i class="fa fa-check"></i><b>4.2</b> Anaerobic Respiration</a></li>
<li class="chapter" data-level="4.3" data-path="growth.html"><a href="growth.html#photosynthesis"><i class="fa fa-check"></i><b>4.3</b> Photosynthesis</a><ul>
<li class="chapter" data-level="4.3.1" data-path="growth.html"><a href="growth.html#pdb-6kgx-photocomplexes-pdb6kgxphotocomplexes"><i class="fa fa-check"></i><b>4.3.1</b> PDB: 6KGX –Photocomplexes {#PDB:<em>6KGX</em>–Photocomplexes}</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="growth.html"><a href="growth.html#moreicm-variety"><i class="fa fa-check"></i><b>4.4</b> More–ICM variety</a></li>
<li class="chapter" data-level="4.5" data-path="growth.html"><a href="growth.html#photosynthesis-contd."><i class="fa fa-check"></i><b>4.5</b> Photosynthesis (cont’d.)</a></li>
<li class="chapter" data-level="4.6" data-path="growth.html"><a href="growth.html#enzyme-regulation"><i class="fa fa-check"></i><b>4.6</b> Enzyme Regulation</a><ul>
<li class="chapter" data-level="4.6.1" data-path="growth.html"><a href="growth.html#pdb-6tqhadhe-spirosome-structure-pdb_6tqhadhe_spirosome_structure"><i class="fa fa-check"></i><b>4.6.1</b> PDB: 6TQH–AdhE spirosome structure {#PDB:_6TQH–AdhE_spirosome_structure} <img src="img/schematics/4_5_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="growth.html"><a href="growth.html#microcompartments"><i class="fa fa-check"></i><b>4.7</b> Microcompartments</a><ul>
<li class="chapter" data-level="4.7.1" data-path="growth.html"><a href="growth.html#pdb-3dktmicrocompartment-shells-pdb_3dktmicrocompartment_shells"><i class="fa fa-check"></i><b>4.7.1</b> PDB: 3DKT–Microcompartment shells {#PDB:_3DKT–Microcompartment_shells} <img src="img/schematics/4_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="4.8" data-path="growth.html"><a href="growth.html#carboxysomes"><i class="fa fa-check"></i><b>4.8</b> Carboxysomes</a></li>
<li class="chapter" data-level="4.9" data-path="growth.html"><a href="growth.html#morecarboxysome-variety"><i class="fa fa-check"></i><b>4.9</b> More–Carboxysome variety</a></li>
<li class="chapter" data-level="4.10" data-path="growth.html"><a href="growth.html#morelong-carboxysomes"><i class="fa fa-check"></i><b>4.10</b> More–Long carboxysomes</a></li>
<li class="chapter" data-level="4.11" data-path="growth.html"><a href="growth.html#storage"><i class="fa fa-check"></i><b>4.11</b> Storage</a></li>
<li class="chapter" data-level="4.12" data-path="growth.html"><a href="growth.html#morestorage-granule-growth"><i class="fa fa-check"></i><b>4.12</b> More–Storage granule growth</a></li>
<li class="chapter" data-level="4.13" data-path="growth.html"><a href="growth.html#storage-contd."><i class="fa fa-check"></i><b>4.13</b> Storage (cont’d.)</a></li>
<li class="chapter" data-level="4.14" data-path="growth.html"><a href="growth.html#moregranule-containing-carboxysomes"><i class="fa fa-check"></i><b>4.14</b> More–Granule-containing carboxysomes</a></li>
<li class="chapter" data-level="4.15" data-path="growth.html"><a href="growth.html#morecarboxysomes-granule-connections"><i class="fa fa-check"></i><b>4.15</b> More–Carboxysomes-granule connections</a></li>
<li class="chapter" data-level="4.16" data-path="growth.html"><a href="growth.html#archaeal-storage-granules"><i class="fa fa-check"></i><b>4.16</b> Archaeal Storage Granules</a></li>
<li class="chapter" data-level="4.17" data-path="growth.html"><a href="growth.html#morehaloferax-gibbonsii-granules"><i class="fa fa-check"></i><b>4.17</b> More–Haloferax gibbonsii granules</a></li>
<li class="chapter" data-level="4.18" data-path="growth.html"><a href="growth.html#morehalohasta-litchfieldiae-granules"><i class="fa fa-check"></i><b>4.18</b> More–Halohasta litchfieldiae granules</a></li>
<li class="chapter" data-level="4.19" data-path="growth.html"><a href="growth.html#further-reading-3"><i class="fa fa-check"></i><b>4.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="division.html"><a href="division.html"><i class="fa fa-check"></i><b>5</b> Division</a><ul>
<li class="chapter" data-level="5.1" data-path="division.html"><a href="division.html#segregation"><i class="fa fa-check"></i><b>5.1</b> Segregation</a></li>
<li class="chapter" data-level="5.2" data-path="division.html"><a href="division.html#chromosome-segregation"><i class="fa fa-check"></i><b>5.2</b> Chromosome Segregation</a></li>
<li class="chapter" data-level="5.3" data-path="division.html"><a href="division.html#budding"><i class="fa fa-check"></i><b>5.3</b> Budding</a></li>
<li class="chapter" data-level="5.4" data-path="division.html"><a href="division.html#morehyphomonas-stalk"><i class="fa fa-check"></i><b>5.4</b> More–Hyphomonas stalk</a></li>
<li class="chapter" data-level="5.5" data-path="division.html"><a href="division.html#morehyphomonas-lifecycle"><i class="fa fa-check"></i><b>5.5</b> More–Hyphomonas lifecycle</a></li>
<li class="chapter" data-level="5.6" data-path="division.html"><a href="division.html#monoderm-cytokinesis"><i class="fa fa-check"></i><b>5.6</b> Monoderm Cytokinesis</a></li>
<li class="chapter" data-level="5.7" data-path="division.html"><a href="division.html#monoderm-cytokinesis-contd."><i class="fa fa-check"></i><b>5.7</b> Monoderm Cytokinesis (cont’d.)</a></li>
<li class="chapter" data-level="5.8" data-path="division.html"><a href="division.html#diderm-cytokinesis"><i class="fa fa-check"></i><b>5.8</b> Diderm Cytokinesis</a></li>
<li class="chapter" data-level="5.9" data-path="division.html"><a href="division.html#moresphingopyxis-alaskensis-division"><i class="fa fa-check"></i><b>5.9</b> More–Sphingopyxis alaskensis division</a></li>
<li class="chapter" data-level="5.10" data-path="division.html"><a href="division.html#morechloroflexi-division"><i class="fa fa-check"></i><b>5.10</b> More–Chloroflexi division</a></li>
<li class="chapter" data-level="5.11" data-path="division.html"><a href="division.html#diderm-cytokinesis-contd."><i class="fa fa-check"></i><b>5.11</b> Diderm Cytokinesis (cont’d.)</a></li>
<li class="chapter" data-level="5.12" data-path="division.html"><a href="division.html#morecaulobacter-division"><i class="fa fa-check"></i><b>5.12</b> More–Caulobacter division</a></li>
<li class="chapter" data-level="5.13" data-path="division.html"><a href="division.html#budding-contd."><i class="fa fa-check"></i><b>5.13</b> Budding (cont’d.)</a></li>
<li class="chapter" data-level="5.14" data-path="division.html"><a href="division.html#ftsz"><i class="fa fa-check"></i><b>5.14</b> FtsZ</a></li>
<li class="chapter" data-level="5.15" data-path="division.html"><a href="division.html#moreasymmetric-constriction"><i class="fa fa-check"></i><b>5.15</b> More–Asymmetric constriction</a><ul>
<li class="chapter" data-level="5.15.1" data-path="division.html"><a href="division.html#pdb-5mn4-ftsz-structure-pdb5mn4ftsz_structure"><i class="fa fa-check"></i><b>5.15.1</b> PDB: 5MN4 –FtsZ structure {#PDB:<em>5MN4</em>–FtsZ_structure}</a></li>
</ul></li>
<li class="chapter" data-level="5.16" data-path="division.html"><a href="division.html#ftsz-contd."><i class="fa fa-check"></i><b>5.16</b> FtsZ (cont’d.)</a></li>
<li class="chapter" data-level="5.17" data-path="division.html"><a href="division.html#escrt"><i class="fa fa-check"></i><b>5.17</b> ESCRT</a></li>
<li class="chapter" data-level="5.18" data-path="division.html"><a href="division.html#escrt-contd."><i class="fa fa-check"></i><b>5.18</b> ESCRT (cont’d.)</a></li>
<li class="chapter" data-level="5.19" data-path="division.html"><a href="division.html#further-reading-4"><i class="fa fa-check"></i><b>5.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="motility.html"><a href="motility.html"><i class="fa fa-check"></i><b>6</b> Motility</a><ul>
<li class="chapter" data-level="6.1" data-path="motility.html"><a href="motility.html#flagellum"><i class="fa fa-check"></i><b>6.1</b> Flagellum</a><ul>
<li class="chapter" data-level="6.1.1" data-path="motility.html"><a href="motility.html#pdb-5wjtflagellum-structure-pdb_5wjtflagellum_structure"><i class="fa fa-check"></i><b>6.1.1</b> PDB: 5WJT–Flagellum structure {#PDB:_5WJT–Flagellum_structure} <img src="img/schematics/6_1_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="6.2" data-path="motility.html"><a href="motility.html#flagellar-motor"><i class="fa fa-check"></i><b>6.2</b> Flagellar Motor</a></li>
<li class="chapter" data-level="6.3" data-path="motility.html"><a href="motility.html#moreflagellar-motor-structure"><i class="fa fa-check"></i><b>6.3</b> More–Flagellar motor structure</a><ul>
<li class="chapter" data-level="6.3.1" data-path="motility.html"><a href="motility.html#Flagellar_motor_diversity"><i class="fa fa-check"></i><b>6.3.1</b> Flagellar motor diversity</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="motility.html"><a href="motility.html#flagellar-assembly"><i class="fa fa-check"></i><b>6.4</b> Flagellar Assembly</a></li>
<li class="chapter" data-level="6.5" data-path="motility.html"><a href="motility.html#moreflagellar-motor-disassembly"><i class="fa fa-check"></i><b>6.5</b> More–Flagellar motor disassembly</a></li>
<li class="chapter" data-level="6.6" data-path="motility.html"><a href="motility.html#flagella-patterns"><i class="fa fa-check"></i><b>6.6</b> Flagella Patterns</a></li>
<li class="chapter" data-level="6.7" data-path="motility.html"><a href="motility.html#morelophotrichous-bacteria"><i class="fa fa-check"></i><b>6.7</b> More–Lophotrichous bacteria</a></li>
<li class="chapter" data-level="6.8" data-path="motility.html"><a href="motility.html#flagella-patterns-contd."><i class="fa fa-check"></i><b>6.8</b> Flagella Patterns (cont’d.)</a></li>
<li class="chapter" data-level="6.9" data-path="motility.html"><a href="motility.html#moreproteus-mirabilis-flagella"><i class="fa fa-check"></i><b>6.9</b> More–Proteus mirabilis flagella</a></li>
<li class="chapter" data-level="6.10" data-path="motility.html"><a href="motility.html#flagellar-sheath"><i class="fa fa-check"></i><b>6.10</b> Flagellar Sheath</a></li>
<li class="chapter" data-level="6.11" data-path="motility.html"><a href="motility.html#periplasmic-flagella"><i class="fa fa-check"></i><b>6.11</b> Periplasmic Flagella</a></li>
<li class="chapter" data-level="6.12" data-path="motility.html"><a href="motility.html#moretreponema-primitia"><i class="fa fa-check"></i><b>6.12</b> More–Treponema primitia</a></li>
<li class="chapter" data-level="6.13" data-path="motility.html"><a href="motility.html#archaellum"><i class="fa fa-check"></i><b>6.13</b> Archaellum</a><ul>
<li class="chapter" data-level="6.13.1" data-path="motility.html"><a href="motility.html#pdb-5tfyarchaellum-structure-pdb_5tfyarchaellum_structure"><i class="fa fa-check"></i><b>6.13.1</b> PDB: 5TFY–Archaellum structure {#PDB:_5TFY–Archaellum_structure} <img src="img/schematics/6_8_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="6.14" data-path="motility.html"><a href="motility.html#archaella-patterns"><i class="fa fa-check"></i><b>6.14</b> Archaella Patterns</a></li>
<li class="chapter" data-level="6.15" data-path="motility.html"><a href="motility.html#morecone-structure"><i class="fa fa-check"></i><b>6.15</b> More–Cone structure</a></li>
<li class="chapter" data-level="6.16" data-path="motility.html"><a href="motility.html#moreorganizing-center"><i class="fa fa-check"></i><b>6.16</b> More–Organizing center</a></li>
<li class="chapter" data-level="6.17" data-path="motility.html"><a href="motility.html#type-iv-pilus"><i class="fa fa-check"></i><b>6.17</b> Type IV Pilus</a><ul>
<li class="chapter" data-level="6.17.1" data-path="motility.html"><a href="motility.html#Type_IV_pilus_structure"><i class="fa fa-check"></i><b>6.17.1</b> Type IV pilus structure</a></li>
</ul></li>
<li class="chapter" data-level="6.18" data-path="motility.html"><a href="motility.html#type-ix-secretion-system"><i class="fa fa-check"></i><b>6.18</b> Type IX Secretion System</a></li>
<li class="chapter" data-level="6.19" data-path="motility.html"><a href="motility.html#terminal-organelle"><i class="fa fa-check"></i><b>6.19</b> Terminal Organelle</a></li>
<li class="chapter" data-level="6.20" data-path="motility.html"><a href="motility.html#further-reading-5"><i class="fa fa-check"></i><b>6.20</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="navigation.html"><a href="navigation.html"><i class="fa fa-check"></i><b>7</b> Navigation</a><ul>
<li class="chapter" data-level="7.1" data-path="navigation.html"><a href="navigation.html#chemotaxis"><i class="fa fa-check"></i><b>7.1</b> Chemotaxis</a><ul>
<li class="chapter" data-level="7.1.1" data-path="navigation.html"><a href="navigation.html#pdb-2ch7chemoreceptor-structure-pdb_2ch7chemoreceptor_structure"><i class="fa fa-check"></i><b>7.1.1</b> PDB: 2CH7–Chemoreceptor structure {#PDB:_2CH7–Chemoreceptor_structure} <img src="img/schematics/7_1_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="navigation.html"><a href="navigation.html#chemosensory-array"><i class="fa fa-check"></i><b>7.2</b> Chemosensory Array</a><ul>
<li class="chapter" data-level="7.2.1" data-path="navigation.html"><a href="navigation.html#PDB:_6S1K_Chemosensory_array_architecture"><i class="fa fa-check"></i><b>7.2.1</b> PDB: 6S1K Chemosensory array architecture</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="navigation.html"><a href="navigation.html#chemosensory-array-conservation"><i class="fa fa-check"></i><b>7.3</b> Chemosensory Array Conservation</a></li>
<li class="chapter" data-level="7.4" data-path="navigation.html"><a href="navigation.html#chemoreceptors"><i class="fa fa-check"></i><b>7.4</b> Chemoreceptors</a></li>
<li class="chapter" data-level="7.5" data-path="navigation.html"><a href="navigation.html#moreazospirillum-brasilense-aerotaxis"><i class="fa fa-check"></i><b>7.5</b> More–Azospirillum brasilense aerotaxis</a></li>
<li class="chapter" data-level="7.6" data-path="navigation.html"><a href="navigation.html#cytoplasmic-chemosensory-array"><i class="fa fa-check"></i><b>7.6</b> Cytoplasmic Chemosensory Array</a></li>
<li class="chapter" data-level="7.7" data-path="navigation.html"><a href="navigation.html#morevibrio-cholerae-chemosensory-arrays"><i class="fa fa-check"></i><b>7.7</b> More–Vibrio cholerae chemosensory arrays</a></li>
<li class="chapter" data-level="7.8" data-path="navigation.html"><a href="navigation.html#magnetotaxis"><i class="fa fa-check"></i><b>7.8</b> Magnetotaxis</a><ul>
<li class="chapter" data-level="7.8.1" data-path="navigation.html"><a href="navigation.html#pdb-5jygmamk-structure-pdb_5jygmamk_structure"><i class="fa fa-check"></i><b>7.8.1</b> PDB: 5JYG–MamK structure {#PDB:_5JYG–MamK_structure} <img src="img/schematics/7_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="navigation.html"><a href="navigation.html#further-reading-6"><i class="fa fa-check"></i><b>7.9</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="lifecycle.html"><a href="lifecycle.html"><i class="fa fa-check"></i><b>8</b> Lifecycle</a><ul>
<li class="chapter" data-level="8.1" data-path="lifecycle.html"><a href="lifecycle.html#stationary-phase"><i class="fa fa-check"></i><b>8.1</b> Stationary Phase</a></li>
<li class="chapter" data-level="8.2" data-path="lifecycle.html"><a href="lifecycle.html#genome-protection"><i class="fa fa-check"></i><b>8.2</b> Genome Protection</a></li>
<li class="chapter" data-level="8.3" data-path="lifecycle.html"><a href="lifecycle.html#stationary-phase-contd."><i class="fa fa-check"></i><b>8.3</b> Stationary Phase (cont’d.)</a></li>
<li class="chapter" data-level="8.4" data-path="lifecycle.html"><a href="lifecycle.html#differentiation"><i class="fa fa-check"></i><b>8.4</b> Differentiation</a></li>
<li class="chapter" data-level="8.5" data-path="lifecycle.html"><a href="lifecycle.html#moreholdfast"><i class="fa fa-check"></i><b>8.5</b> More–Holdfast</a></li>
<li class="chapter" data-level="8.6" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-sporulation"><i class="fa fa-check"></i><b>8.6</b> Monoderm Sporulation</a></li>
<li class="chapter" data-level="8.7" data-path="lifecycle.html"><a href="lifecycle.html#moresporulation-septum-formation"><i class="fa fa-check"></i><b>8.7</b> More–Sporulation septum formation</a></li>
<li class="chapter" data-level="8.8" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-sporulation-contd."><i class="fa fa-check"></i><b>8.8</b> Monoderm Sporulation (cont’d.)</a></li>
<li class="chapter" data-level="8.9" data-path="lifecycle.html"><a href="lifecycle.html#morespore-engulfment"><i class="fa fa-check"></i><b>8.9</b> More–Spore engulfment</a></li>
<li class="chapter" data-level="8.10" data-path="lifecycle.html"><a href="lifecycle.html#morespore-coating"><i class="fa fa-check"></i><b>8.10</b> More–Spore coating</a></li>
<li class="chapter" data-level="8.11" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-spores"><i class="fa fa-check"></i><b>8.11</b> Monoderm Spores</a></li>
<li class="chapter" data-level="8.12" data-path="lifecycle.html"><a href="lifecycle.html#monoderm-germination"><i class="fa fa-check"></i><b>8.12</b> Monoderm Germination</a></li>
<li class="chapter" data-level="8.13" data-path="lifecycle.html"><a href="lifecycle.html#diderm-sporulation"><i class="fa fa-check"></i><b>8.13</b> Diderm Sporulation</a></li>
<li class="chapter" data-level="8.14" data-path="lifecycle.html"><a href="lifecycle.html#diderm-sporulation-contd."><i class="fa fa-check"></i><b>8.14</b> Diderm Sporulation (cont’d.)</a></li>
<li class="chapter" data-level="8.15" data-path="lifecycle.html"><a href="lifecycle.html#morecomplete-engulfment"><i class="fa fa-check"></i><b>8.15</b> More–Complete engulfment</a></li>
<li class="chapter" data-level="8.16" data-path="lifecycle.html"><a href="lifecycle.html#moremature-spore"><i class="fa fa-check"></i><b>8.16</b> More–Mature spore</a></li>
<li class="chapter" data-level="8.17" data-path="lifecycle.html"><a href="lifecycle.html#moresporomusa-acidovorans-sporulation"><i class="fa fa-check"></i><b>8.17</b> More–Sporomusa acidovorans sporulation</a></li>
<li class="chapter" data-level="8.18" data-path="lifecycle.html"><a href="lifecycle.html#diderm-germination"><i class="fa fa-check"></i><b>8.18</b> Diderm Germination</a></li>
<li class="chapter" data-level="8.19" data-path="lifecycle.html"><a href="lifecycle.html#further-reading-7"><i class="fa fa-check"></i><b>8.19</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="interaction.html"><a href="interaction.html"><i class="fa fa-check"></i><b>9</b> Interaction</a><ul>
<li class="chapter" data-level="9.1" data-path="interaction.html"><a href="interaction.html#biofilm"><i class="fa fa-check"></i><b>9.1</b> Biofilm</a></li>
<li class="chapter" data-level="9.2" data-path="interaction.html"><a href="interaction.html#biofilm-contd."><i class="fa fa-check"></i><b>9.2</b> Biofilm (cont’d.)</a></li>
<li class="chapter" data-level="9.3" data-path="interaction.html"><a href="interaction.html#type-ii-and-type-iv-secretion"><i class="fa fa-check"></i><b>9.3</b> Type II and Type IV Secretion</a><ul>
<li class="chapter" data-level="9.3.1" data-path="interaction.html"><a href="interaction.html#emd-20712-emd-20713legionella-t2ss-structure-emd-20712_emd-20713legionella_t2ss_structure"><i class="fa fa-check"></i><b>9.3.1</b> EMD-20712 EMD-20713–Legionella T2SS structure {#EMD-20712_EMD-20713–Legionella_T2SS_structure}</a></li>
<li class="chapter" data-level="9.3.2" data-path="interaction.html"><a href="interaction.html#emd-0566-legionella-t4ss-structure-emd-0566_legionella_t4ss_structure"><i class="fa fa-check"></i><b>9.3.2</b> EMD-0566 –Legionella T4SS structure {#EMD-0566_–Legionella_T4SS_structure}</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="interaction.html"><a href="interaction.html#morehelicobacter-pylori-tubes"><i class="fa fa-check"></i><b>9.4</b> More–Helicobacter pylori tubes</a></li>
<li class="chapter" data-level="9.5" data-path="interaction.html"><a href="interaction.html#type-iii-secretion"><i class="fa fa-check"></i><b>9.5</b> Type III Secretion</a><ul>
<li class="chapter" data-level="9.5.1" data-path="interaction.html"><a href="interaction.html#EMD-20611_Injectisome_structure"><i class="fa fa-check"></i><b>9.5.1</b> EMD-20611 Injectisome structure</a></li>
<li class="chapter" data-level="9.5.2" data-path="interaction.html"><a href="interaction.html#PDB:_5WQ8_Secretin_structure"><i class="fa fa-check"></i><b>9.5.2</b> PDB: 5WQ8 Secretin structure</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="interaction.html"><a href="interaction.html#type-v-secretion"><i class="fa fa-check"></i><b>9.6</b> Type V Secretion</a></li>
<li class="chapter" data-level="9.7" data-path="interaction.html"><a href="interaction.html#type-vi-secretion"><i class="fa fa-check"></i><b>9.7</b> Type VI Secretion</a><ul>
<li class="chapter" data-level="9.7.1" data-path="interaction.html"><a href="interaction.html#pdb-5urwprimed-t6ss-structure-pdb_5urwprimed_t6ss_structure"><i class="fa fa-check"></i><b>9.7.1</b> PDB: 5URW–Primed T6SS structure {#PDB:_5URW–Primed_T6SS_structure} <img src="img/schematics/9_6_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="9.8" data-path="interaction.html"><a href="interaction.html#type-vi-secretion-contd."><i class="fa fa-check"></i><b>9.8</b> Type VI Secretion (cont’d.)</a><ul>
<li class="chapter" data-level="9.8.1" data-path="interaction.html"><a href="interaction.html#pdb-5urxfired-t6ss-structure-pdb_5urxfired_t6ss_structure"><i class="fa fa-check"></i><b>9.8.1</b> PDB: 5URX–Fired T6SS structure {#PDB:_5URX–Fired_T6SS_structure} <img src="img/schematics/9_7_1.gif" /><!-- --></a></li>
</ul></li>
<li class="chapter" data-level="9.9" data-path="interaction.html"><a href="interaction.html#moret6ss-battery"><i class="fa fa-check"></i><b>9.9</b> More–T6SS battery</a></li>
<li class="chapter" data-level="9.10" data-path="interaction.html"><a href="interaction.html#moremacs"><i class="fa fa-check"></i><b>9.10</b> More–MACs</a><ul>
<li class="chapter" data-level="9.10.1" data-path="interaction.html"><a href="interaction.html#MAC_array_structure"><i class="fa fa-check"></i><b>9.10.1</b> MAC array structure</a></li>
</ul></li>
<li class="chapter" data-level="9.11" data-path="interaction.html"><a href="interaction.html#type-ix-secretion"><i class="fa fa-check"></i><b>9.11</b> Type IX Secretion</a></li>
<li class="chapter" data-level="9.12" data-path="interaction.html"><a href="interaction.html#predation"><i class="fa fa-check"></i><b>9.12</b> Predation</a></li>
<li class="chapter" data-level="9.13" data-path="interaction.html"><a href="interaction.html#predation-contd."><i class="fa fa-check"></i><b>9.13</b> Predation (cont’d.)</a></li>
<li class="chapter" data-level="9.14" data-path="interaction.html"><a href="interaction.html#morebdellovibrio-bacteriovorus-entry"><i class="fa fa-check"></i><b>9.14</b> More–Bdellovibrio bacteriovorus entry</a></li>
<li class="chapter" data-level="9.15" data-path="interaction.html"><a href="interaction.html#predation-contd.-1"><i class="fa fa-check"></i><b>9.15</b> Predation (cont’d.)</a></li>
<li class="chapter" data-level="9.16" data-path="interaction.html"><a href="interaction.html#further-reading-8"><i class="fa fa-check"></i><b>9.16</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="viruses.html"><a href="viruses.html"><i class="fa fa-check"></i><b>10</b> Viruses</a><ul>
<li class="chapter" data-level="10.1" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis"><i class="fa fa-check"></i><b>10.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.2" data-path="viruses.html"><a href="viruses.html#bacteroidetes-strain-jt5"><i class="fa fa-check"></i><b>10.2</b> Bacteroidetes strain JT5</a></li>
<li class="chapter" data-level="10.3" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-1"><i class="fa fa-check"></i><b>10.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.4" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae"><i class="fa fa-check"></i><b>10.4</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="10.5" data-path="viruses.html"><a href="viruses.html#escherichia-coli"><i class="fa fa-check"></i><b>10.5</b> Escherichia coli</a></li>
<li class="chapter" data-level="10.6" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus"><i class="fa fa-check"></i><b>10.6</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="10.7" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.7</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="10.8" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus"><i class="fa fa-check"></i><b>10.8</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="10.9" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-1"><i class="fa fa-check"></i><b>10.9</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="10.10" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-2"><i class="fa fa-check"></i><b>10.10</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="10.11" data-path="viruses.html"><a href="viruses.html#further-reading-9"><i class="fa fa-check"></i><b>10.11</b> Further Reading</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>11</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.library.caltech.edu" target="blank">Published by Caltech Library</a><li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Atlas of Bacterial and Archaeal Cell Structure</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="viruses" class="section level1">
<h1><span class="header-section-number">Chapter 10</span> Viruses</h1>
<p>“Phages are the winners in the game of life.” - Forest Rohwer <span class="citation">Rohwer et al. (<a href="#ref-rohwer2014">2014</a>)</span></p>
<div id="shewanella-oneidensis" class="section level2">
<h2><span class="header-section-number">10.1</span> Shewanella oneidensis</h2>
<p>Life is a battlefield for your cell, and other cells are not the only enemy. Attack can come from another front: viruses called phage (short for bacteriophage, or “bacterium eaters” because they were first identified infecting and killing bacteria). Like all viruses, phage use cells as vehicles for their replication. And, just like eukaryotic viruses, there is an army of different phage that prey on bacteria and archaea. No species is known to be immune.</p>
<p>The basic strategy of a phage is to inject its genome into a host cell, use the cell’s machinery to replicate the genome and manufacture the proteins it encodes, then use these materials to build the next generation of phage. Since the phage is dependent on the cell’s machinery for replication, it does not fulfill the criteria for a living organism. (Despite this distinction, we still talk about the viral “lifecycle.”) By cleverly outsourcing, phage can be highly streamlined structurally, little more than a genome in an envelope en route to its next destination. All phage have a genome (either double or single-stranded DNA or RNA), tightly packed inside a protective envelope of either protein (forming a “capsid”) or lipid, or both. The genome-containing shell is called the “head” of the phage. Many phage also have a tail, like this myophage attacking <em>Shewanella oneidensis</em>. Fibers at the tip of the tail recognize features on their target host and allow the phage to dock.</p>
<div class="figure"><span id="fig:unnamed-chunk-173"></span>
<div id="htmlwidget-40d49f675fb1615e9d04" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-40d49f675fb1615e9d04">{"x":{"doi":"10.22002/D1.1586","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.1: Shewanella oneidensis Collected by: Poorna Subramanian <a href="https://doi.org/10.22002/D1.1586">10.22002/D1.1586</a>
</p>
</div>
</div>
<div id="bacteroidetes-strain-jt5" class="section level2">
<h2><span class="header-section-number">10.2</span> Bacteroidetes strain JT5</h2>
<p>Once docked, the phage delivers its cargo. The tails of some, but not all, phage are contractile, punching a channel through the cell’s envelope to inject the genome, emptying the capsid as you see in this myophage infecting a <em>Bacteroidetes</em> cell. This syringe-like mechanism should be familiar from the type VI secretion system and related weapons discussed in the last chapter. Evolution coopted phage tails for those structures, likely taking advantage of viral genes integrated into the genome of a bacterial host (more on that in a few pages).</p>
<div class="figure"><span id="fig:unnamed-chunk-174"></span>
<div id="htmlwidget-ac6d293978e461f56d83" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-ac6d293978e461f56d83">{"x":{"doi":"10.22002/D1.1587","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.2: Bacteroidetes JT5 Collected by: Elitza Tocheva <a href="https://doi.org/10.22002/D1.1587">10.22002/D1.1587</a>
</p>
</div>
</div>
<div id="shewanella-oneidensis-1" class="section level2">
<h2><span class="header-section-number">10.3</span> Shewanella oneidensis</h2>
<p>Some phage have tails that are not contractile, like this siphophage attacking a <em>Shewanella oneidensis</em> cell. The long, flexible tail still serves the essential function of crossing the barrier of the cell’s envelope, providing a conduit for the genome to enter the cell.</p>
<p>Note that we already saw a different type of phage attacking <em>S. oneidensis</em>. While most phage are adapted to infect one or a few closely-related species of bacteria or archaea, there are often multiple different types of phage adapted to attack each species. The diversity of phage is staggering; one type lacks a single gene with detectable homology to any other in any known virus or cell.</p>
<div class="figure"><span id="fig:unnamed-chunk-175"></span>
<div id="htmlwidget-5c69688987505d66a561" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-5c69688987505d66a561">{"x":{"doi":"10.22002/D1.1588","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.3: Shewanella oneidensis Collected by: Poorna Subramanian <a href="https://doi.org/10.22002/D1.1588">10.22002/D1.1588</a>
</p>
</div>
</div>
<div id="vibrio-cholerae" class="section level2">
<h2><span class="header-section-number">10.4</span> Vibrio cholerae</h2>
<p>Not all phage tails are so long. Podophage (which this unidentified attacker of a <em>Vibrio cholerae</em> may be) have short tails that extend after docking to cross the host envelope. Other phage lack tails entirely.</p>
<p>Similarly, heads can look very different. The unique rocket-ship shape of this phage’s capsid highlights the diversity of phage morphology. Some, mostly infecting archaea in extreme environments, have heads shaped like bottles or lemons. Others take the form of long, thin filaments.</p>
<div class="figure"><span id="fig:unnamed-chunk-176"></span>
<div id="htmlwidget-5c3a6dd3d9faa66cc79e" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-5c3a6dd3d9faa66cc79e">{"x":{"doi":"10.22002/D1.1589","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.4: Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1589">10.22002/D1.1589</a>
</p>
</div>
</div>
<div id="escherichia-coli" class="section level2">
<h2><span class="header-section-number">10.5</span> Escherichia coli</h2>
<p>Unlike your cell, phage cannot direct their movement. They simply ride the currents of Brownian motion, waiting to bump into a target receptor they can latch onto. Their abundance helps; in the ocean, there are about ten times as many phage as bacteria. Still, it might be a long wait, so evolution has fashioned some mechanisms to speed things up. One strategy is to target a receptor not on the cell body itself, but on a long, more easily-encountered cell appendage.</p>
<p>Some species of bacteria, including this <em>Escherichia coli</em> use “competence pili” assembled by a type IV secretion system to take up DNA from other cells. This “horizontal gene transfer” (in contrast to “vertical” heredity from mother to daughter) lets asexually-reproducing cells mix and match genes to potential advantage. It also blurs the line between species (more about that in the <em>Phylogenetic Explorer</em>). And in this case, it offers an advantage to these MS2 leviphage, which have evolved to recognize the pilus and inject their own RNA into the nucleic acid uptake channel.</p>
<div class="figure"><span id="fig:unnamed-chunk-177"></span>
<div id="htmlwidget-f25ddabca00afd966c5b" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-f25ddabca00afd966c5b">{"x":{"doi":"10.22002/D1.1590","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.5: Escherichia coli Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1590">10.22002/D1.1590</a>
</p>
</div>
</div>
<div id="caulobacter-crescentus" class="section level2">
<h2><span class="header-section-number">10.6</span> Caulobacter crescentus</h2>
<p>Other phage use bacterial appendages in an even cleverer way. These φCb13 myophage take advantage of the cell’s motility to hitch a ride to the cell. They have a filament on their head that wraps into the helical groove of the <em>Caulobacter crescentus</em> flagellum. Remember that flagella can spin either clockwise or counter-clockwise. If this single polar flagellum spins clockwise, pushing the cell body, the attached phage would unscrew like a nut off the end of the flagellum. But if the flagellum spins counter-clockwise, the nut will instead spin down to the cell body, where its tail fibers will find their receptors and its tail will inject the genome [More–φCb13 infection].</p>
<p>10_6a_Caulobacter crescentus More–φCb13 infection</p>
<p>Here you see several φCb13 phage that have successfully reached their target receptors at the pole of a <em>Caulobacter crescentus</em> and are in the process of infecting the cell (note the mix of filled capsids that have not yet released their genome and empty capsids that have).</p>
<div class="figure"><span id="fig:unnamed-chunk-178"></span>
<div id="htmlwidget-51459fb655d607b3430b" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-51459fb655d607b3430b">{"x":{"doi":"10.22002/D1.1591","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.6: Caulobacter crescentus Collected by: Elizabeth Wright <a href="https://doi.org/10.22002/D1.1591">10.22002/D1.1591</a>
</p>
</div>
</div>
<div id="vibrio-cholerae-1" class="section level2">
<h2><span class="header-section-number">10.7</span> Vibrio cholerae</h2>
<p>Once inside a cell, phage adopt one of two strategies. The first is to go through a straightforward round of replication, using their host’s machinery. The second is to stick around a while. To do that, the phage inserts its genome into that of its host, creating an addition called a “prophage” that will be propagated indefinitely through cell replication. At some point in the future, usually in response to a stress that threatens the host, the prophage uses special DNA sequences at its ends akin to an ejection seat to pop its genes back out of the host genome, and carry out a normal viral replication and release cycle. Prophage are not simply passengers, though. Some confer a beneficial function on the host; a filamentous inophage carries the disease-causing “cholera toxin” in the <em>Vibrio cholerae</em> genome. Prophage also offer a source of genetic diversity to their host. Mutations sometimes alter the ejection sequences, rendering a prophage a permanent part of the genome, or genetic recombination may shuffle some or part of the prophage into a different part of the genome. Further evolution may then put these pieces to new use. This is likely how bacteria got the contractile weapons you saw in the last chapter, repurposing genes encoding phage tail components.</p>
<p>Whether the phage replicates immediately or waits a while as a prophage first, the cycle of replication and release is rapid. The virus hijacks the cell’s protein-making machinery to churn out packaging proteins like capsid, and its replication machinery to churn out copies of the genome. Genomes are quickly (and very densely) packed into new heads, tails attached if necessary, and the mature phage released. Not all phage lyse their hosts; some filamentous phage (which are commonly beneficial) use secretion machinery to exit without harm. Most, though, use proteins that self-assemble into ports in the inner membrane, allowing another phage-encoded protein to access and chew up the cell wall, causing the cell to lyse. Here you can see a snapshot of this process in a <em>Vibrio cholerae</em> cell infected by a phage that is not its virulence-enhancing friend. The phage is replicating, and has just lysed the cell to make its escape.</p>
<div class="figure"><span id="fig:unnamed-chunk-179"></span>
<div id="htmlwidget-1cb390038a080938126c" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-1cb390038a080938126c">{"x":{"doi":"10.22002/D1.1592","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.7: Vibrio cholerae Collected by: Poorna Subramanian <a href="https://doi.org/10.22002/D1.1592">10.22002/D1.1592</a>
</p>
</div>
</div>
<div id="sulfolobus-solfataricus" class="section level2">
<h2><span class="header-section-number">10.8</span> Sulfolobus solfataricus</h2>
<p>Some archaeal phage make an even more dramatic exit, through _v_irus-_a_ssociated _p_yramids, or VAPs. You can see these escape hatches in this <em>Sulfolobus solfataricus</em> cell infected with *_S_ulfolobus* _T_urreted _I_cosahedral _V_irus (STIV) [Schematic–STIV structure]. The VAPs are made from a single phage-encoded protein that self-assembles into a seven-sided pyramid in the membrane. Note how the VAPs poke through and disrupt the cell’s surface layer. Note also the many, many copies the phage has made. We sometimes see storage granules inside VAPs, likely simply pushed there by excluding forces from the densely-packed nucleoid and assembling viral capsids in the center of the cell.</p>
<p>10_8_1 Schematic: PDB: 3J31–STIV structure As their name implies, STIV have icosahedral capsids with protruding turrets. The hexameric repeating unit in the shell is formed by three copies of a double-lobed protein, as you can see in this structure <span class="citation">Veesler et al. (<a href="#ref-veesler2013">2013</a>)</span>.</p>
<div class="figure"><span id="fig:unnamed-chunk-180"></span>
<div id="htmlwidget-9f53907194a044b7755e" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-9f53907194a044b7755e">{"x":{"doi":"10.22002/D1.1593","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.8: Sulfolobus solfataricus Collected by: Lu Gan <a href="https://doi.org/10.22002/D1.1593">10.22002/D1.1593</a>
</p>
</div>
</div>
<div id="sulfolobus-solfataricus-1" class="section level2">
<h2><span class="header-section-number">10.9</span> Sulfolobus solfataricus</h2>
<p>When the STIV phage is ready to make its great escape, a still-unknown signal triggers the pyramids to open along their seams, peeling back to create 100 nm-wide holes for the phage, and all the other contents of the cell, to pour out, as you can see in this lysed <em>Sulfolobus solfataricus</em> cell infected with the same virus.</p>
<div class="figure"><span id="fig:unnamed-chunk-181"></span>
<div id="htmlwidget-294d1983d4064c85b4ef" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-294d1983d4064c85b4ef">{"x":{"doi":"10.22002/D1.1594","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.9: Sulfolobus solfataricus Collected by: Lu Gan <a href="https://doi.org/10.22002/D1.1594">10.22002/D1.1594</a>
</p>
</div>
</div>
<div id="shewanella-oneidensis-2" class="section level2">
<h2><span class="header-section-number">10.10</span> Shewanella oneidensis</h2>
<p>So what can your cell do to protect itself from these attackers? The most well-known defense is the CRISPR-Cas system now widely used to edit genomes across the tree of life. Its original function, though, is as an immune system similar to our own. Bacterial and archaeal cells “record” snippets of the genes of infecting phage in their own genome, enabling the cell to rapidly identify and chop up matching genes in future infections, much as our immune system stores the memory of antigen fragments for rapid neutralization in the future.</p>
<p>Defense can also be structural. Surface layers may protect cells from some phage. And then there is always distraction. One function of extracellular vesicles may be to provide decoys for phage to attack, as you can see with this myophage attacking a <em>Shewanella oneidensis</em> outer membrane vesicle. The war between cells and viruses is an ancient one, and continually raging. Despite your cell’s best efforts, it is very likely that it will end its life in a burst of phage. It is far from alone in this fate, though. In the ocean off Florida, it has been estimated that every day phage kill off half the bacterial population. But each lysed cell provides a feast for a starving neighbor, and DNA for more evolutionary tinkering, continuing to shape the weird, wonderful microbial world.</p>
<div class="figure"><span id="fig:unnamed-chunk-182"></span>
<div id="htmlwidget-2ed43c89811ede4949da" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-2ed43c89811ede4949da">{"x":{"doi":"10.22002/D1.1595","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 10.10: Shewanella oneidensis Collected by: Lauren Ann Metskas <a href="https://doi.org/10.22002/D1.1595">10.22002/D1.1595</a>
</p>
</div>
</div>
<div id="further-reading-9" class="section level2">
<h2><span class="header-section-number">10.11</span> Further Reading</h2>
<p><span class="citation">Keen (<a href="#ref-keen2015">2015</a>)</span> <span class="citation">Prangishvili et al. (<a href="#ref-prangishvili2017">2017</a>)</span> <span class="citation">Rohwer et al. (<a href="#ref-rohwer2014">2014</a>)</span></p>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-keen2015">
<p>Keen, E. C. 2015. “A Century of Phage Research: Bacteriophages and the Shaping of Modern Biology.” <em>Bioessays</em> 37 (1): 6–9. doi:<a href="https://doi.org/10.1002/bies.201400152">10.1002/bies.201400152</a>.</p>
</div>
<div id="ref-prangishvili2017">
<p>Prangishvili, D., D. H. Bamford, P. Forterre, J. Iranzo, E. V. Koonin, and M. Krupovic. 2017. “The Enigmatic Archaeal Virosphere.” <em>Nat Rev Microbiol</em> 15 (12): 724–39. doi:<a href="https://doi.org/10.1038/nrmicro.2017.125">10.1038/nrmicro.2017.125</a>.</p>
</div>
<div id="ref-rohwer2014">
<p>Rohwer, F., M. Youle, H. Maughan, and N. Hisakawa. 2014. <em>Life in Our Phage World</em>. San Diego, CA: Wholon.</p>
</div>
<div id="ref-veesler2013">
<p>Veesler, David, Thiam-Seng Ng, Anoop K. Sendamarai, Brian J. Eilers, C. Martin Lawrence, Shee-Mei Lok, Mark J. Young, John E. Johnson, and Chi-yu Fu. 2013. “Atomic Structure of the 75 MDa Extremophile Sulfolobus Turreted Icosahedral Virus Determined by CryoEM and X-Ray Crystallography.” <em>PNAS</em> 110 (14). National Academy of Sciences: 5504–9. doi:<a href="https://doi.org/10.1073/pnas.1300601110">10.1073/pnas.1300601110</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="interaction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": null,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/caltechlibrary/cell-atlas/edit/master/10_Viruses.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["cell-atlas.pdf", "cell-atlas.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
