<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 11 Viruses | Cell Atlas</title>
  <meta name="description" content="An atlas produced by the Jensen Lab." />
  <meta name="generator" content="bookdown 0.17 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 11 Viruses | Cell Atlas" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="splash/HeaderImage.jpg" />
  <meta property="og:description" content="An atlas produced by the Jensen Lab." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 11 Viruses | Cell Atlas" />
  
  <meta name="twitter:description" content="An atlas produced by the Jensen Lab." />
  <meta name="twitter:image" content="splash/HeaderImage.jpg" />

<meta name="author" content="Jensen Lab" />


<meta name="date" content="2020-07-29" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="interaction.html"/>
<link rel="next" href="references.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.1/htmlwidgets.js"></script>
<script src="libs/CL-0.1.2/CL.js"></script>
<script src="libs/doivideo-binding-0.0.0.9000/doivideo.js"></script>



<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Cell Atlas</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a><ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgments"><i class="fa fa-check"></i>Acknowledgments</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#section"><i class="fa fa-check"></i><b>1.1</b> 1</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="methods.html"><a href="methods.html"><i class="fa fa-check"></i><b>2</b> Methods</a><ul>
<li class="chapter" data-level="2.1" data-path="methods.html"><a href="methods.html#schematicstructural-biology-toolkit"><i class="fa fa-check"></i><b>2.1</b> 1[Schematic–Structural biology toolkit]</a></li>
<li class="chapter" data-level="2.2" data-path="methods.html"><a href="methods.html#escherichia-coli"><i class="fa fa-check"></i><b>2.2</b> Escherichia coli</a></li>
<li class="chapter" data-level="2.3" data-path="methods.html"><a href="methods.html#shewanella-oneidensis"><i class="fa fa-check"></i><b>2.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="2.4" data-path="methods.html"><a href="methods.html#schematicelectron-microscopy-modes"><i class="fa fa-check"></i><b>2.4</b> 3[Schematic–Electron microscopy modes]</a></li>
<li class="chapter" data-level="2.5" data-path="methods.html"><a href="methods.html#shewanella-oneidensis-1"><i class="fa fa-check"></i><b>2.5</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="2.6" data-path="methods.html"><a href="methods.html#caulobacter-crescentus"><i class="fa fa-check"></i><b>2.6</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="2.7" data-path="methods.html"><a href="methods.html#caulobacter-crescentus-1"><i class="fa fa-check"></i><b>2.7</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="2.8" data-path="methods.html"><a href="methods.html#schematicmissing-wedge"><i class="fa fa-check"></i><b>2.8</b> 6[Schematic–Missing wedge]</a></li>
<li class="chapter" data-level="2.9" data-path="methods.html"><a href="methods.html#campylobacter-jejuni"><i class="fa fa-check"></i><b>2.9</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.10" data-path="methods.html"><a href="methods.html#campylobacter-jejuni-1"><i class="fa fa-check"></i><b>2.10</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.11" data-path="methods.html"><a href="methods.html#campylobacter-jejuni-2"><i class="fa fa-check"></i><b>2.11</b> Campylobacter jejuni</a></li>
<li class="chapter" data-level="2.12" data-path="methods.html"><a href="methods.html#schematicstructural-biology-timeline"><i class="fa fa-check"></i><b>2.12</b> 9[Schematic–Structural biology timeline]</a></li>
<li class="chapter" data-level="2.13" data-path="methods.html"><a href="methods.html#bdellovibrio-bacteriovorus"><i class="fa fa-check"></i><b>2.13</b> Bdellovibrio bacteriovorus</a><ul>
<li class="chapter" data-level="2.13.1" data-path="methods.html"><a href="methods.html#Difference_mapping"><i class="fa fa-check"></i><b>2.13.1</b> Campylobacter jejuni</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="envelope.html"><a href="envelope.html"><i class="fa fa-check"></i><b>3</b> Envelope</a><ul>
<li class="chapter" data-level="3.1" data-path="envelope.html"><a href="envelope.html#schematic-lipid-bilayer"><i class="fa fa-check"></i><b>3.1</b> 1[Schematic: Lipid bilayer]</a></li>
<li class="chapter" data-level="3.2" data-path="envelope.html"><a href="envelope.html#schematic-atp-synthase-and-energy-production"><i class="fa fa-check"></i><b>3.2</b> 1[Schematic: ATP synthase and energy production]</a></li>
<li class="chapter" data-level="3.3" data-path="envelope.html"><a href="envelope.html#listeria-monocytogenes"><i class="fa fa-check"></i><b>3.3</b> Listeria monocytogenes</a><ul>
<li class="chapter" data-level="3.3.1" data-path="envelope.html"><a href="envelope.html#Peptidoglycan_architecture"><i class="fa fa-check"></i><b>3.3.1</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="3.3.2" data-path="envelope.html"><a href="envelope.html#Methanobacterium_formicicum"><i class="fa fa-check"></i><b>3.3.2</b> Methanobacterium formicicum</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="envelope.html"><a href="envelope.html#cupriavidus-necator"><i class="fa fa-check"></i><b>3.4</b> Cupriavidus necator</a></li>
<li class="chapter" data-level="3.5" data-path="envelope.html"><a href="envelope.html#schematicbrauns-lipoprotein"><i class="fa fa-check"></i><b>3.5</b> 3[Schematic–Braun’s lipoprotein]</a><ul>
<li class="chapter" data-level="3.5.1" data-path="envelope.html"><a href="envelope.html#escherichia-coli-diderm_sacculus_architecturecompare-this-diderm-sacculus-purified-from-escherichia-coli-to-the-monoderm-sacculus-on-the-last-page.-since-this-one-is-thinner-we-can-make-out-more-details-like-the-glycan-strands-running-around-the-circumference-of-the-cell.-the-main-difference-between-the-two-types-of-sacculi-seems-to-be-whether-they-have-largely-a-single-layer-of-peptidoglycan-diderm-or-many-layers-monoderm.-so-even-though-the-two-types-of-wall-look-different-their-architecture-is-fundamentally-the-same.-in-some-circumstances-as-you-will-see-in-chapter-8-cells-can-even-switch-between-the-two-forms."><i class="fa fa-check"></i><b>3.5.1</b> Escherichia coli {#Diderm_sacculus_architecture}Compare this diderm sacculus purified from <em>Escherichia coli</em> to the monoderm sacculus on the last page. Since this one is thinner, we can make out more details, like the glycan strands running around the circumference of the cell. The main difference between the two types of sacculi seems to be whether they have largely a single layer of peptidoglycan (diderm) or many layers (monoderm). So even though the two types of wall look different, their architecture is fundamentally the same. In some circumstances, as you will see in Chapter 8, cells can even switch between the two forms.</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="envelope.html"><a href="envelope.html#schematicsacculus-remodeling"><i class="fa fa-check"></i><b>3.6</b> 3[Schematic–Sacculus remodeling]</a></li>
<li class="chapter" data-level="3.7" data-path="envelope.html"><a href="envelope.html#myxococcus-xanthus"><i class="fa fa-check"></i><b>3.7</b> Myxococcus xanthus</a><ul>
<li class="chapter" data-level="3.7.1" data-path="envelope.html"><a href="envelope.html#borrelia-burgdorferi-pearled_vesiclesdifferent-species-can-produce-outer-membrane-vesicles-that-look-very-different.-the-same-species-can-even-produce-vesicles-that-look-very-different.-sometimes-they-come-off-the-cell-as-a-chain-of-spheres-sometimes-the-spheres-remain-connected-like-a-string-of-pearls-as-in-this-borrelia-burgdorferi-cell.-sometimes-vesicles-form-long-tubes-instead-moretubular-vesicles.-sometimes-the-same-chain-can-be-tubular-in-one-section-usually-at-the-base-connected-to-the-cell-and-a-string-of-spheres-in-another."><i class="fa fa-check"></i><b>3.7.1</b> Borrelia burgdorferi {#Pearled_vesicles}Different species can produce outer membrane vesicles that look very different. The same species can even produce vesicles that look very different. Sometimes they come off the cell as a chain of spheres; sometimes the spheres remain connected, like a string of pearls, as in this <em>Borrelia burgdorferi</em> cell. Sometimes vesicles form long tubes instead [More–Tubular vesicles]. Sometimes the same chain can be tubular in one section (usually at the base, connected to the cell), and a string of spheres in another.</a></li>
<li class="chapter" data-level="3.7.2" data-path="envelope.html"><a href="envelope.html#borrelia-burgdorferi-tubular_vesicleshere-again-from-a-borrelia-burgdorferi-cell-you-can-see-extended-tubular-outer-membrane-vesicles."><i class="fa fa-check"></i><b>3.7.2</b> Borrelia burgdorferi {#Tubular_vesicles}Here, again from a <em>Borrelia burgdorferi</em> cell, you can see extended, tubular outer membrane vesicles.</a></li>
<li class="chapter" data-level="3.7.3" data-path="envelope.html"><a href="envelope.html#myxococcus-xanthus-cytoplasmic_vesiclesnot-all-vesicles-come-from-the-outer-membrane.-the-cytoplasmic-or-inner-membrane-can-also-form-vesicles-that-are-released-into-the-cytoplasm-as-in-this-myxococcus-xanthus-cell-or-into-the-periplasm."><i class="fa fa-check"></i><b>3.7.3</b> Myxococcus xanthus {#Cytoplasmic_vesicles}Not all vesicles come from the outer membrane. The cytoplasmic or inner membrane can also form vesicles that are released into the cytoplasm, as in this <em>Myxococcus xanthus</em> cell, or into the periplasm.</a></li>
<li class="chapter" data-level="3.7.4" data-path="envelope.html"><a href="envelope.html#prosthecobacter-debontii-cytoplasmic_vesicle_varietycytoplasmic-vesicles-exhibit-a-variety-of-sizes-and-shapes.-some-are-nested-with-vesicles-inside-vesicles.-in-this-prosthecobacter-debontii-cell-you-can-see-two-other-morphologies.-one-is-a-large-flattened-horseshoe-shaped-vesicle.-another-is-a-more-typical-spherical-shape-but-is-decorated-with-what-look-like-protein-complexes."><i class="fa fa-check"></i><b>3.7.4</b> Prosthecobacter debontii {#Cytoplasmic_vesicle_variety}Cytoplasmic vesicles exhibit a variety of sizes and shapes. Some are nested, with vesicles inside vesicles. In this <em>Prosthecobacter debontii</em> cell, you can see two other morphologies. One is a large, flattened horseshoe-shaped vesicle. Another is a more typical spherical shape, but is decorated with what look like protein complexes.</a></li>
<li class="chapter" data-level="3.7.5" data-path="envelope.html"><a href="envelope.html#halomicrobium-mukohataei-archaeal_vesiclesjust-like-bacteria-archaea-also-produce-both-internal-and-external-vesicles-as-you-can-see-in-this-halomicrobium-mukohataei-cell."><i class="fa fa-check"></i><b>3.7.5</b> Halomicrobium mukohataei {#Archaeal_vesicles}Just like bacteria, archaea also produce both internal and external vesicles, as you can see in this <em>Halomicrobium mukohataei</em> cell.</a></li>
</ul></li>
<li class="chapter" data-level="3.8" data-path="envelope.html"><a href="envelope.html#mycobacterium-marinum"><i class="fa fa-check"></i><b>3.8</b> Mycobacterium marinum</a></li>
<li class="chapter" data-level="3.9" data-path="envelope.html"><a href="envelope.html#caulobacter-crescentus-2"><i class="fa fa-check"></i><b>3.9</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="3.10" data-path="envelope.html"><a href="envelope.html#schematicsurface-layer-architecture"><i class="fa fa-check"></i><b>3.10</b> 6[Schematic–Surface layer architecture]</a></li>
<li class="chapter" data-level="3.11" data-path="envelope.html"><a href="envelope.html#aarchaea-and-bacteria-with-surface-layers-produce-characteristic-outer-membrane-vesicles-they-bud-off-with-the-surface-layer-attached.-you-will-see-plenty-of-examples-in-the-coming-pages.-delftia-acidovorans-like-this-produce-so-called-nanopods-chains-of-outer-membrane-vesicles-ensheathed-in-surface-layer."><i class="fa fa-check"></i><b>3.11</b> 6aArchaea and bacteria with surface layers produce characteristic outer membrane vesicles: they bud off with the surface layer attached. You will see plenty of examples in the coming pages. <em>Delftia acidovorans</em> like this produce so-called nanopods: chains of outer membrane vesicles ensheathed in surface layer.</a></li>
<li class="chapter" data-level="3.12" data-path="envelope.html"><a href="envelope.html#sulfolobus-solfataricus"><i class="fa fa-check"></i><b>3.12</b> Sulfolobus solfataricus</a><ul>
<li class="chapter" data-level="3.12.1" data-path="envelope.html"><a href="envelope.html#methylomicrobium-alcaliphilum-m._alcaliphilumin-methylomicrobium-alcaliphilum-v-shaped-surface-layer-proteins-come-together-to-form-cups-that-pack-into-a-hexagonal-pattern."><i class="fa fa-check"></i><b>3.12.1</b> Methylomicrobium alcaliphilum {#M._alcaliphilum}In <em>Methylomicrobium alcaliphilum</em>, V-shaped surface layer proteins come together to form cups that pack into a hexagonal pattern.</a></li>
<li class="chapter" data-level="3.12.2" data-path="envelope.html"><a href="envelope.html#clostridium-thermocellum-clostridium_thermocellumnot-only-diderm-bacteria-have-surface-layers-as-you-can-see-on-this-monoderm-clostridium-thermocellum-cell.-the-main-difference-is-that-the-proteins-are-anchored-to-the-cell-wall-rather-than-to-the-outer-membrane-or-associated-lipoproteins."><i class="fa fa-check"></i><b>3.12.2</b> Clostridium thermocellum {#Clostridium_thermocellum}Not only diderm bacteria have surface layers, as you can see on this monoderm <em>Clostridium thermocellum</em> cell. The main difference is that the proteins are anchored to the cell wall, rather than to the outer membrane (or associated lipoproteins).</a></li>
<li class="chapter" data-level="3.12.3" data-path="envelope.html"><a href="envelope.html#nitrosopumilis-maritimus-n._maritimusin-nitrosopumilis-maritimus-surface-layer-proteins-form-hexagonal-rosettes-that-in-turn-pack-into-a-hexagonal-lattice."><i class="fa fa-check"></i><b>3.12.3</b> Nitrosopumilis maritimus {#N._maritimus}In <em>Nitrosopumilis maritimus</em>, surface layer proteins form hexagonal rosettes that in turn pack into a hexagonal lattice.</a></li>
<li class="chapter" data-level="3.12.4" data-path="envelope.html"><a href="envelope.html#methanoregula-formicica-methanoregula_formicicathe-surface-layer-of-methanoregula-formicica-is-a-hexagonal-lattice-of-small-nearly-circular-subunits."><i class="fa fa-check"></i><b>3.12.4</b> Methanoregula formicica {#Methanoregula_formicica}The surface layer of <em>Methanoregula formicica</em> is a hexagonal lattice of small, nearly circular subunits.</a></li>
</ul></li>
<li class="chapter" data-level="3.13" data-path="envelope.html"><a href="envelope.html#methanospirillum-hungatei"><i class="fa fa-check"></i><b>3.13</b> Methanospirillum hungatei</a></li>
<li class="chapter" data-level="3.14" data-path="envelope.html"><a href="envelope.html#haloarcula-argentinensis"><i class="fa fa-check"></i><b>3.14</b> Haloarcula argentinensis</a></li>
<li class="chapter" data-level="3.15" data-path="envelope.html"><a href="envelope.html#bdellovibrio-bacteriovorus-1"><i class="fa fa-check"></i><b>3.15</b> Bdellovibrio bacteriovorus</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="shape.html"><a href="shape.html"><i class="fa fa-check"></i><b>4</b> Shape</a><ul>
<li class="chapter" data-level="4.1" data-path="shape.html"><a href="shape.html#cupriavidus-necator-1"><i class="fa fa-check"></i><b>4.1</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="4.1.1" data-path="shape.html"><a href="shape.html#brucella-abortus-rod_varietynot-all-rod-shaped-cells-are-perfectly-cylindrical.-for-instance-brucella-abortus-like-this-one-are-pear-shaped.-this-is-one-of-the-rod-shaped-bacterial-species-that-do-not-use-mreb."><i class="fa fa-check"></i><b>4.1.1</b> Brucella abortus {#Rod_variety}Not all rod-shaped cells are perfectly cylindrical. For instance, <em>Brucella abortus</em> like this one are pear-shaped. This is one of the rod-shaped bacterial species that do not use MreB.</a></li>
<li class="chapter" data-level="4.1.2" data-path="shape.html"><a href="shape.html#methanoregula-formicica-archaeal_rodssome-archaeal-species-like-this-methanoregula-formicica-are-rod-shaped.-the-rigid-surface-layer-seems-to-help-define-the-shape-but-the-details-are-still-unknown."><i class="fa fa-check"></i><b>4.1.2</b> Methanoregula formicica {#Archaeal_rods}Some archaeal species, like this <em>Methanoregula formicica</em>, are rod-shaped. The rigid surface layer seems to help define the shape, but the details are still unknown.</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="shape.html"><a href="shape.html#hylemonella-gracilis"><i class="fa fa-check"></i><b>4.2</b> Hylemonella gracilis</a></li>
<li class="chapter" data-level="4.3" data-path="shape.html"><a href="shape.html#caulobacter-crescentus-3"><i class="fa fa-check"></i><b>4.3</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="4.4" data-path="shape.html"><a href="shape.html#schematic-ctp-synthase"><i class="fa fa-check"></i><b>4.4</b> 4[Schematic: CTP synthase]</a></li>
<li class="chapter" data-level="4.5" data-path="shape.html"><a href="shape.html#campylobacter-jejuni-3"><i class="fa fa-check"></i><b>4.5</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="4.5.1" data-path="shape.html"><a href="shape.html#borrelia-burgdorferi-borrelia_burgdorferiborrelia-burgdorferi-like-this-cell-use-the-long-filaments-of-their-motility-machinery-flagella-discussed-in-chapter-6-as-a-kind-of-cytoskeleton.-a-bundle-of-flagella-wraps-around-the-cell-in-the-periplasm-between-the-cell-wall-and-the-outer-membrane.-spun-by-motors-at-their-base-more-on-that-in-chapter-6-the-filaments-impart-a-wave-pattern-to-the-growing-sacculus.-without-the-motors-rotation-the-cells-develop-a-rod-shape.-note-that-b.-burgdorferi-are-not-helical-but-rather-adopt-a-two-dimensional-waveform."><i class="fa fa-check"></i><b>4.5.1</b> Borrelia burgdorferi {#Borrelia_burgdorferi}<em>Borrelia burgdorferi</em> like this cell use the long filaments of their motility machinery (flagella, discussed in Chapter 6) as a kind of cytoskeleton. A bundle of flagella wraps around the cell in the periplasm, between the cell wall and the outer membrane. Spun by motors at their base (more on that in Chapter 6), the filaments impart a wave pattern to the growing sacculus. Without the motors’ rotation, the cells develop a rod shape. Note that <em>B. burgdorferi</em> are not helical, but rather adopt a two-dimensional waveform.</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="shape.html"><a href="shape.html#verrucomicrobium-spinosum"><i class="fa fa-check"></i><b>4.6</b> Verrucomicrobium spinosum</a><ul>
<li class="chapter" data-level="4.6.1" data-path="shape.html"><a href="shape.html#Bactofilin"><i class="fa fa-check"></i><b>4.6.1</b> Bactofilin</a></li>
<li class="chapter" data-level="4.6.2" data-path="shape.html"><a href="shape.html#prosthecobacter-vanneervenii-bacterial_microtubulessome-bacterial-species-with-prosthecae-express-structures-similar-to-eukaryotic-microtubules-made-from-two-proteins-called-btuba-and-btubb-to-reflect-their-homology-to-eukaryotic-tubulins.-eukaryotic-microtubules-are-hollow-tubes-formed-by-13-protofilaments-bacterial-microtubules-are-smaller-with-only-5-protofilaments.-cells-commonly-contain-a-bundle-of-microtubules-in-their-prostheca-like-this-prosthecobacter-vanneervenii-cell-which-has-a-bundle-of-four."><i class="fa fa-check"></i><b>4.6.2</b> Prosthecobacter vanneervenii {#Bacterial_microtubules}Some bacterial species with prosthecae express structures similar to eukaryotic microtubules, made from two proteins called BtubA and BtubB to reflect their homology to eukaryotic tubulins. Eukaryotic microtubules are hollow tubes formed by 13 protofilaments; bacterial microtubules are smaller, with only ~5 protofilaments. Cells commonly contain a bundle of microtubules in their prostheca, like this <em>Prosthecobacter vanneervenii</em> cell, which has a bundle of four.</a></li>
</ul></li>
<li class="chapter" data-level="4.7" data-path="shape.html"><a href="shape.html#haloquadratum-walsbyi"><i class="fa fa-check"></i><b>4.7</b> Haloquadratum walsbyi</a><ul>
<li class="chapter" data-level="4.7.1" data-path="shape.html"><a href="shape.html#halobacterium-salinarum-gas_vesiclessome-species-of-archaea-and-bacteria-use-gas-vesicles-to-control-their-buoyancy.-this-can-allow-them-to-rise-or-fall-in-a-water-column-which-can-be-a-great-advantage.-halobacterium-salinarum-like-this-one-produce-gas-vesicles-in-response-to-cues-from-the-environment-lifting-themselves-out-of-the-sediment-and-into-more-favorable-conditions-of-oxygen-or-sunlight-for-photosynthesis.-this-cell-has-just-started-producing-gas-vesicles-so-they-are-small-and-isolated.-later-the-vesicles-will-elongate-into-cylinders-with-conical-ends-as-you-saw-in-haloquadratum-walsbyi.-each-cell-might-contain-dozens-of-vesicles-and-they-often-cluster-together."><i class="fa fa-check"></i><b>4.7.1</b> Halobacterium salinarum {#Gas_vesicles}Some species of archaea and bacteria use gas vesicles to control their buoyancy. This can allow them to rise or fall in a water column, which can be a great advantage. <em>Halobacterium salinarum</em> like this one produce gas vesicles in response to cues from the environment, lifting themselves out of the sediment and into more favorable conditions of oxygen or sunlight for photosynthesis. This cell has just started producing gas vesicles, so they are small and isolated. Later, the vesicles will elongate into cylinders with conical ends, as you saw in <em>Haloquadratum walsbyi</em>. Each cell might contain dozens of vesicles, and they often cluster together.</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="growth.html"><a href="growth.html"><i class="fa fa-check"></i><b>5</b> Growth</a><ul>
<li class="chapter" data-level="5.1" data-path="growth.html"><a href="growth.html#shewanella-oneidensis-2"><i class="fa fa-check"></i><b>5.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="5.2" data-path="growth.html"><a href="growth.html#rhodopseudomonas-palustris"><i class="fa fa-check"></i><b>5.2</b> Rhodopseudomonas palustris</a></li>
<li class="chapter" data-level="5.3" data-path="growth.html"><a href="growth.html#schematic-photocomplexes"><i class="fa fa-check"></i><b>5.3</b> 3[Schematic: Photocomplexes]</a><ul>
<li class="chapter" data-level="5.3.1" data-path="growth.html"><a href="growth.html#methyloprofundus-sedimenti-icm_varietyin-some-species-like-this-methyloprofundus-sedimenti-the-intracytoplasmic-membrane-is-very-extensive-and-appears-to-be-fully-separated-from-the-cell-membrane.-as-you-can-see-it-occupies-much-of-the-cytoplasmic-space."><i class="fa fa-check"></i><b>5.3.1</b> Methyloprofundus sedimenti {#ICM_variety}In some species, like this <em>Methyloprofundus sedimenti</em>, the intracytoplasmic membrane is very extensive and appears to be fully separated from the cell membrane. As you can see, it occupies much of the cytoplasmic space.</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="growth.html"><a href="growth.html#chloroflexi"><i class="fa fa-check"></i><b>5.4</b> Chloroflexi</a></li>
<li class="chapter" data-level="5.5" data-path="growth.html"><a href="growth.html#clostridium-thermocellum"><i class="fa fa-check"></i><b>5.5</b> Clostridium thermocellum</a></li>
<li class="chapter" data-level="5.6" data-path="growth.html"><a href="growth.html#schematicadhe-spirosome-structure"><i class="fa fa-check"></i><b>5.6</b> 5[Schematic–AdhE spirosome structure]</a></li>
<li class="chapter" data-level="5.7" data-path="growth.html"><a href="growth.html#acetonema-longum"><i class="fa fa-check"></i><b>5.7</b> Acetonema longum</a></li>
<li class="chapter" data-level="5.8" data-path="growth.html"><a href="growth.html#schematicmicrocompartment-shells"><i class="fa fa-check"></i><b>5.8</b> 6[Schematic–Microcompartment shells]</a></li>
<li class="chapter" data-level="5.9" data-path="growth.html"><a href="growth.html#thiomonas-intermedia"><i class="fa fa-check"></i><b>5.9</b> Thiomonas intermedia</a><ul>
<li class="chapter" data-level="5.9.1" data-path="growth.html"><a href="growth.html#halothiobacillus-neapolitanus-carboxysome_varietymost-carboxysomes-look-like-the-ones-you-saw-on-the-last-page-but-not-all.-some-are-less-regular-polyhedra-as-in-this-halothiobacillus-neapolitanus-cell.-and-some-are-stranger-still-morelong-carboxysomes."><i class="fa fa-check"></i><b>5.9.1</b> Halothiobacillus neapolitanus {#Carboxysome_variety}Most carboxysomes look like the ones you saw on the last page, but not all. Some are less regular polyhedra, as in this <em>Halothiobacillus neapolitanus</em> cell. And some are stranger still [More–Long carboxysomes].</a></li>
<li class="chapter" data-level="5.9.2" data-path="growth.html"><a href="growth.html#hydrogenovibrio-crunogenus-long_carboxysomesthe-carboxysomes-in-this-hydrogenovibrio-crunogenus-have-grown-abnormally-long-extending-across-the-full-width-of-the-cell.-these-are-unusual-but-less-dramatic-elongations-are-common."><i class="fa fa-check"></i><b>5.9.2</b> Hydrogenovibrio crunogenus {#Long_carboxysomes}The carboxysomes in this <em>Hydrogenovibrio crunogenus</em> have grown abnormally long, extending across the full width of the cell. These are unusual, but less dramatic elongations are common.</a></li>
</ul></li>
<li class="chapter" data-level="5.10" data-path="growth.html"><a href="growth.html#cupriavidus-necator-2"><i class="fa fa-check"></i><b>5.10</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="5.10.1" data-path="growth.html"><a href="growth.html#lysobacter-antibioticus-storage_granule_growthstorage-granules-expand-and-contract-to-accommodate-their-contents.-this-lysobacter-antibioticus-cell-exhibits-a-typical-range-of-sizes-of-poly-phosphate-granules."><i class="fa fa-check"></i><b>5.10.1</b> Lysobacter antibioticus {#Storage_granule_growth}Storage granules expand and contract to accommodate their contents. This <em>Lysobacter antibioticus</em> cell exhibits a typical range of sizes of poly-phosphate granules.</a></li>
</ul></li>
<li class="chapter" data-level="5.11" data-path="growth.html"><a href="growth.html#agrobacterium-tumefaciens"><i class="fa fa-check"></i><b>5.11</b> Agrobacterium tumefaciens</a><ul>
<li class="chapter" data-level="5.11.1" data-path="growth.html"><a href="growth.html#hydrogenovibrio-crunogenus-granule-containing_carboxysomessmall-poly-phosphate-storage-granules-are-sometimes-seen-inside-carboxysomes-as-in-this-hydrogenovibrio-crunogenus-cell.-their-purpose-is-unknown."><i class="fa fa-check"></i><b>5.11.1</b> Hydrogenovibrio crunogenus {#Granule-containing_carboxysomes}Small poly-phosphate storage granules are sometimes seen <em>inside</em> carboxysomes, as in this <em>Hydrogenovibrio crunogenus</em> cell. Their purpose is unknown.</a></li>
<li class="chapter" data-level="5.11.2" data-path="growth.html"><a href="growth.html#halothiobacillus-neapolitanus-carboxysomes-granule_connectionswe-sometimes-see-comb-like-structures-connecting-external-poly-phosphate-storage-granules-with-carboxysomes-as-in-this-halothiobacillus-neapolitanus-cell.-the-nature-of-the-relationship-between-the-structures-and-the-identity-and-function-of-the-combs-remains-a-mystery."><i class="fa fa-check"></i><b>5.11.2</b> Halothiobacillus neapolitanus {#Carboxysomes-granule_connections}We sometimes see comb-like structures connecting external poly-phosphate storage granules with carboxysomes, as in this <em>Halothiobacillus neapolitanus</em> cell. The nature of the relationship between the structures, and the identity and function of the combs, remains a mystery.</a></li>
</ul></li>
<li class="chapter" data-level="5.12" data-path="growth.html"><a href="growth.html#halorubrum-litoreum"><i class="fa fa-check"></i><b>5.12</b> Halorubrum litoreum</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="division.html"><a href="division.html"><i class="fa fa-check"></i><b>6</b> Division</a><ul>
<li class="chapter" data-level="6.1" data-path="division.html"><a href="division.html#caulobacter-crescentus-4"><i class="fa fa-check"></i><b>6.1</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="6.2" data-path="division.html"><a href="division.html#hyphomonas-neptunium"><i class="fa fa-check"></i><b>6.2</b> Hyphomonas neptunium</a><ul>
<li class="chapter" data-level="6.2.1" data-path="division.html"><a href="division.html#hyphomonas-neptunium-hyphomonas_stalkhyphomonas-neptunium-grow-a-single-stalk-from-one-end-of-their-cell-body-similar-to-the-caulobacter-crescentus-you-saw-in-chapter-3.-the-function-of-the-stalk-though-is-different-in-this-budding-bacterium-than-in-c.-crescentus-which-divide-by-more-conventional-fission."><i class="fa fa-check"></i><b>6.2.1</b> Hyphomonas neptunium {#Hyphomonas_stalk}<em>Hyphomonas neptunium</em> grow a single stalk from one end of their cell body, similar to the <em>Caulobacter crescentus</em> you saw in Chapter 3. The function of the stalk, though, is different in this budding bacterium than in <em>C. crescentus</em>, which divide by more conventional fission.</a></li>
<li class="chapter" data-level="6.2.2" data-path="division.html"><a href="division.html#hyphomonas-neptunium-hyphomonas_lifecyclehyphomonas-neptunium-have-evolved-a-program-of-stages-they-pass-through-in-the-course-of-their-life.-when-a-newborn-cell-like-this-one-is-released-it-is-in-the-swarmer-stage-using-its-flagellum-discussed-in-the-next-chapter-to-swim-away-to-a-new-location.-after-a-while-it-settles-down-jettisoning-its-flagellum-and-growing-a-stalk-with-which-to-make-its-own-budthe-next-generation.-once-a-cell-settles-down-into-the-stalked-stage-it-spends-the-rest-of-its-life-sending-off-buds-as-long-as-conditions-are-good.-only-newborn-cells-are-equipped-with-flagella-to-go-adventuring.-we-will-discuss-more-lifecycles-in-chapter-8."><i class="fa fa-check"></i><b>6.2.2</b> Hyphomonas neptunium {#Hyphomonas_lifecycle}<em>Hyphomonas neptunium</em> have evolved a program of stages they pass through in the course of their life. When a newborn cell like this one is released, it is in the “swarmer” stage, using its flagellum (discussed in the next chapter) to swim away to a new location. After a while, it settles down, jettisoning its flagellum and growing a stalk with which to make its own bud–the next generation. Once a cell settles down into the stalked stage, it spends the rest of its life sending off buds as long as conditions are good. Only newborn cells are equipped with flagella to go adventuring. We will discuss more lifecycles in Chapter 8.</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="division.html"><a href="division.html#staphylococcus-aureus"><i class="fa fa-check"></i><b>6.3</b> Staphylococcus aureus</a></li>
<li class="chapter" data-level="6.4" data-path="division.html"><a href="division.html#tetrasphaera-remsis"><i class="fa fa-check"></i><b>6.4</b> Tetrasphaera remsis</a></li>
<li class="chapter" data-level="6.5" data-path="division.html"><a href="division.html#idiomarina-loihiensis"><i class="fa fa-check"></i><b>6.5</b> Idiomarina loihiensis</a><ul>
<li class="chapter" data-level="6.5.1" data-path="division.html"><a href="division.html#sphingopyxis-alaskensis-sphingopyxis_alaskensis_divisionthis-diderm-cell-is-about-halfway-through-cytokinesis.-the-two-daughter-cells-are-still-connected-by-a-wide-bridge-of-cytoplasm-but-the-inner-membrane-and-cell-wall-are-drawing-in-all-around."><i class="fa fa-check"></i><b>6.5.1</b> Sphingopyxis alaskensis {#Sphingopyxis_alaskensis_division}This diderm cell is about halfway through cytokinesis. The two daughter cells are still connected by a wide bridge of cytoplasm, but the inner membrane and cell wall are drawing in all around.</a></li>
<li class="chapter" data-level="6.5.2" data-path="division.html"><a href="division.html#chloroflexi-chloroflexi_divisionthis-diderm-cell-is-in-the-final-stage-of-cytokinesis.-the-cytoplasm-of-the-two-daughter-cells-is-completely-separated-by-inner-membrane-and-cell-wall.-the-outer-membrane-though-has-not-yet-divided."><i class="fa fa-check"></i><b>6.5.2</b> Chloroflexi {#Chloroflexi_division}This diderm cell is in the final stage of cytokinesis. The cytoplasm of the two daughter cells is completely separated by inner membrane and cell wall. The outer membrane, though, has not yet divided.</a></li>
</ul></li>
<li class="chapter" data-level="6.6" data-path="division.html"><a href="division.html#caulobacter-crescentus-5"><i class="fa fa-check"></i><b>6.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="6.6.1" data-path="division.html"><a href="division.html#caulobacter-crescentus-caulobacter_divisionthese-daughter-cells-have-completely-separated-their-mothers-inner-membrane-and-cell-wall-but-not-yet-the-outer-membrane-and-surface-layer."><i class="fa fa-check"></i><b>6.6.1</b> Caulobacter crescentus {#Caulobacter_division}These daughter cells have completely separated their mother’s inner membrane and cell wall, but not yet the outer membrane and surface layer.</a></li>
</ul></li>
<li class="chapter" data-level="6.7" data-path="division.html"><a href="division.html#agrobacterium-tumefaciens-1"><i class="fa fa-check"></i><b>6.7</b> Agrobacterium tumefaciens</a></li>
<li class="chapter" data-level="6.8" data-path="division.html"><a href="division.html#cupriavidus-necator-3"><i class="fa fa-check"></i><b>6.8</b> Cupriavidus necator</a><ul>
<li class="chapter" data-level="6.8.1" data-path="division.html"><a href="division.html#cupriavidus-necator-asymmetric_constrictionthis-c.-necator-cell-is-at-a-late-stage-of-cytokinesis.-in-most-cells-constriction-would-be-more-or-less-uniform-around-the-circumference-by-now-but-in-some-cells-including-this-one-the-asymmetry-persists."><i class="fa fa-check"></i><b>6.8.1</b> Cupriavidus necator {#Asymmetric_constriction}This <em>C. necator</em> cell is at a late stage of cytokinesis. In most cells, constriction would be more or less uniform around the circumference by now, but in some cells, including this one, the asymmetry persists.</a></li>
</ul></li>
<li class="chapter" data-level="6.9" data-path="division.html"><a href="division.html#schematicftsz-structure"><i class="fa fa-check"></i><b>6.9</b> 9[Schematic–FtsZ structure]</a></li>
<li class="chapter" data-level="6.10" data-path="division.html"><a href="division.html#caulobacter-crescentus-6"><i class="fa fa-check"></i><b>6.10</b> Caulobacter crescentus</a></li>
<li class="chapter" data-level="6.11" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius"><i class="fa fa-check"></i><b>6.11</b> Sulfolobus acidocaldarius</a></li>
<li class="chapter" data-level="6.12" data-path="division.html"><a href="division.html#schematicescrt-structure"><i class="fa fa-check"></i><b>6.12</b> 11[Schematic–ESCRT structure]</a></li>
<li class="chapter" data-level="6.13" data-path="division.html"><a href="division.html#sulfolobus-acidocaldarius-1"><i class="fa fa-check"></i><b>6.13</b> Sulfolobus acidocaldarius</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="motility.html"><a href="motility.html"><i class="fa fa-check"></i><b>7</b> Motility</a><ul>
<li class="chapter" data-level="7.1" data-path="motility.html"><a href="motility.html#schematicflagellum-structure"><i class="fa fa-check"></i><b>7.1</b> 1[Schematic–Flagellum structure]</a></li>
<li class="chapter" data-level="7.2" data-path="motility.html"><a href="motility.html#bdellovibrio-bacteriovorus-2"><i class="fa fa-check"></i><b>7.2</b> Bdellovibrio bacteriovorus</a><ul>
<li class="chapter" data-level="7.2.1" data-path="motility.html"><a href="motility.html#bdellovibrio-bacteriovorus-flagellar_motor_structurethis-is-an-average-of-the-flagellar-motors-from-more-than-one-thousand-bdellovibrio-bacteriovorus-cells.-working-upward-from-the-base-the-major-parts-of-the-rotor-are-the-c-ring-for-_c_ytoplasmic-the-ms-ring-for-_m_embrane-and-_s_upramembrane-the-rod-the-hook-and-finally-the-filament.-they-are-surrounded-by-stationary-parts-the-stator-ring-which-is-dynamic-so-we-cannot-resolve-the-stators-as-they-cross-the-inner-membrane-and-connect-to-the-c-ring-and-a-series-of-bushings-that-allow-rotation-within-the-cell-wall-the-_p_eriplasmic-or-p-ring-and-outer-membrane-_l_ipopolysaccharide-or-l-ring.-additional-cytoplasmic-components-form-the-export-apparatus-which-is-involved-in-assembly-discussed-on-the-next-page."><i class="fa fa-check"></i><b>7.2.1</b> Bdellovibrio bacteriovorus {#Flagellar_motor_structure}This is an average of the flagellar motors from more than one thousand <em>Bdellovibrio bacteriovorus</em> cells. Working upward from the base, the major parts of the rotor are the C-ring (for _C_ytoplasmic), the MS-ring (for _M_embrane and _S_upramembrane), the rod, the hook, and finally the filament. They are surrounded by stationary parts: the stator ring (which is dynamic, so we cannot resolve the stators as they cross the inner membrane and connect to the C-ring) and a series of bushings that allow rotation within the cell wall (the _P_eriplasmic or P-ring) and outer membrane (_L_ipopolysaccharide or L-ring). Additional cytoplasmic components form the export apparatus, which is involved in assembly (discussed on the next page).</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="motility.html"><a href="motility.html#schematicflagellar-motor-diversity"><i class="fa fa-check"></i><b>7.3</b> 2[Schematic–Flagellar motor diversity]</a></li>
<li class="chapter" data-level="7.4" data-path="motility.html"><a href="motility.html#hylemonella-gracilis-1"><i class="fa fa-check"></i><b>7.4</b> Hylemonella gracilis</a><ul>
<li class="chapter" data-level="7.4.1" data-path="motility.html"><a href="motility.html#pseudomonas-aeruginosa-flagellar_motor_disassemblywhen-flagella-break-or-in-some-species-like-caulobacter-crescentus-are-ejected-so-the-cell-can-attach-to-a-surface-the-motor-is-disassembled-usually-beginning-with-the-export-apparatus.-not-everything-is-dismantled-though-the-p--and-l-rings-remain-in-place-in-the-cell-wall-and-outer-membrane-respectively-as-you-can-see-in-this-pseudomonas-aeruginosa-cell.-we-do-not-know-if-the-cell-has-a-reason-for-leaving-them-there-but-one-possibility-is-that-they-may-function-as-a-plug-for-the-outer-membrane-when-the-hook-is-no-longer-present."><i class="fa fa-check"></i><b>7.4.1</b> Pseudomonas aeruginosa {#Flagellar_motor_disassembly}When flagella break (or in some species, like <em>Caulobacter crescentus</em>, are ejected so the cell can attach to a surface), the motor is disassembled, usually beginning with the export apparatus. Not everything is dismantled, though; the P- and L-rings remain in place in the cell wall and outer membrane, respectively, as you can see in this <em>Pseudomonas aeruginosa</em> cell. We do not know if the cell has a reason for leaving them there, but one possibility is that they may function as a plug for the outer membrane when the hook is no longer present.</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="motility.html"><a href="motility.html#campylobacter-jejuni-4"><i class="fa fa-check"></i><b>7.5</b> Campylobacter jejuni</a><ul>
<li class="chapter" data-level="7.5.1" data-path="motility.html"><a href="motility.html#helicobacter-pylori-lophotrichous_bacteriamany-lophotrichous-species-like-the-hylemonella-gracilis-you-saw-in-chapter-3length-or-this-helicobacter-pylori-have-a-tuft-of-flagella-at-their-cell-pole.-in-some-species-though-the-tuft-is-located-elsewhere-for-example-a-clump-of-flagella-on-the-concave-side-of-banana-shaped-selenomonas-artemidis-pushes-the-cells-sideways-in-a-seesawing-swimming-pattern."><i class="fa fa-check"></i><b>7.5.1</b> Helicobacter pylori {#Lophotrichous_bacteria}Many lophotrichous species, like the <em>Hylemonella gracilis</em> you saw in Chapter 3–Length or this <em>Helicobacter pylori</em>, have a tuft of flagella at their cell pole. In some species, though, the tuft is located elsewhere; for example, a clump of flagella on the concave side of banana-shaped <em>Selenomonas artemidis</em> pushes the cells sideways in a seesawing swimming pattern.</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="motility.html"><a href="motility.html#pseudomonas-flexibilis"><i class="fa fa-check"></i><b>7.6</b> Pseudomonas flexibilis</a><ul>
<li class="chapter" data-level="7.6.1" data-path="motility.html"><a href="motility.html#proteus-mirabilis-proteus_mirabilis_flagellaproteus-mirabilis-adapt-their-motility-machinery-to-their-environment.-in-liquid-the-short-rod-shaped-cells-swim-with-the-help-of-a-handful-of-flagella-distributed-peritrichously-around-their-cell-body.-when-they-encounter-a-solid-surface-the-cells-elongate-and-build-many-more-flagella-as-you-can-see-on-this-cell.-instead-of-swimming-they-now-use-their-flagella-to-propel-themselves-in-groups-across-the-surface-a-motility-mode-known-as-swarming.-this-is-an-example-of-a-differentiated-lifecycle-which-will-come-up-again-in-chapter-8."><i class="fa fa-check"></i><b>7.6.1</b> Proteus mirabilis {#Proteus_mirabilis_flagella}<em>Proteus mirabilis</em> adapt their motility machinery to their environment. In liquid, the short rod-shaped cells swim with the help of a handful of flagella distributed peritrichously around their cell body. When they encounter a solid surface, the cells elongate and build many more flagella, as you can see on this cell. Instead of swimming, they now use their flagella to propel themselves in groups across the surface, a motility mode known as “swarming.” This is an example of a differentiated lifecycle, which will come up again in Chapter 8.</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="motility.html"><a href="motility.html#helicobacter-hepaticus"><i class="fa fa-check"></i><b>7.7</b> Helicobacter hepaticus</a></li>
<li class="chapter" data-level="7.8" data-path="motility.html"><a href="motility.html#borrelia-burgdorferi"><i class="fa fa-check"></i><b>7.8</b> Borrelia burgdorferi</a><ul>
<li class="chapter" data-level="7.8.1" data-path="motility.html"><a href="motility.html#treponema-primitia-treponema_primitiatreponema-primitia-like-this-are-commensal-residents-of-the-termite-gut-helping-break-down-cellulose.-in-addition-to-two-periplasmic-flagella-the-cells-have-arrays-of-bowl--and-hook-like-structures-on-their-surface-the-function-of-which-likely-related-to-motility-remains-mysterious."><i class="fa fa-check"></i><b>7.8.1</b> Treponema primitia {#Treponema_primitia}<em>Treponema primitia</em> like this are commensal residents of the termite gut, helping break down cellulose. In addition to two periplasmic flagella, the cells have arrays of bowl- and hook-like structures on their surface, the function of which, likely related to motility, remains mysterious.</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="motility.html"><a href="motility.html#methanoregula-formicica"><i class="fa fa-check"></i><b>7.9</b> Methanoregula formicica</a></li>
<li class="chapter" data-level="7.10" data-path="motility.html"><a href="motility.html#schematic-archaellum-structure"><i class="fa fa-check"></i><b>7.10</b> 8[Schematic: Archaellum structure]</a></li>
<li class="chapter" data-level="7.11" data-path="motility.html"><a href="motility.html#schematic-archaellar-motor-structure"><i class="fa fa-check"></i><b>7.11</b> 8[Schematic: Archaellar motor structure]</a></li>
<li class="chapter" data-level="7.12" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis"><i class="fa fa-check"></i><b>7.12</b> Thermococcus kodakaraensis</a><ul>
<li class="chapter" data-level="7.12.1" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis-cone_structurethe-archaellar-plate-does-not-come-to-a-point-at-the-tip-but-rather-is-a-conical-frustum-open-at-the-top-resembling-a-lampshade.-in-the-center-of-the-tip-is-a-small-ring-as-you-can-see-in-a-top-view-in-this-lysed-flattened-thermococcus-kodakaraensis-cell.-the-function-of-the-ring-remains-unknown-perhaps-it-helps-nucleate-the-rest-of-the-structure"><i class="fa fa-check"></i><b>7.12.1</b> Thermococcus kodakaraensis {#Cone_structure}The archaellar plate does not come to a point at the tip, but rather is a conical frustum (open at the top), resembling a lampshade. In the center of the tip is a small ring, as you can see in a top view in this lysed, flattened <em>Thermococcus kodakaraensis</em> cell. The function of the ring remains unknown; perhaps it helps nucleate the rest of the structure?</a></li>
<li class="chapter" data-level="7.12.2" data-path="motility.html"><a href="motility.html#thermococcus-kodakaraensis-organizing_centeras-you-can-see-more-clearly-in-this-partially-lysed-flattened-thermococcus-kodakaraensis-cell-the-conical-plate-in-the-cytoplasm-is-attached-to-more-than-just-the-archaella.-it-is-also-associated-with-chemosensory-arrays-discussed-in-the-next-chapter-and-dna-as-you-can-see-from-the-ribosome-excluding-zone.-this-structure-may-therefore-perform-an-analogous-function-to-bacterial-organizing-proteins-such-as-popz-tethering-cellular-components-into-a-de-facto-pole-for-the-in-this-case-round-cell."><i class="fa fa-check"></i><b>7.12.2</b> Thermococcus kodakaraensis {#Organizing_center}As you can see more clearly in this partially-lysed, flattened <em>Thermococcus kodakaraensis</em> cell, the conical plate in the cytoplasm is attached to more than just the archaella. It is also associated with chemosensory arrays (discussed in the next chapter) and DNA, as you can see from the ribosome-excluding zone. This structure may therefore perform an analogous function to bacterial organizing proteins such as PopZ, tethering cellular components into a <em>de facto</em> pole for the (in this case round) cell.</a></li>
</ul></li>
<li class="chapter" data-level="7.13" data-path="motility.html"><a href="motility.html#myxococcus-xanthus-1"><i class="fa fa-check"></i><b>7.13</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="7.14" data-path="motility.html"><a href="motility.html#schematictype-iv-pilus-structure"><i class="fa fa-check"></i><b>7.14</b> 10[Schematic–Type IV pilus structure]</a></li>
<li class="chapter" data-level="7.15" data-path="motility.html"><a href="motility.html#flavobacterium-johnsoniae"><i class="fa fa-check"></i><b>7.15</b> Flavobacterium johnsoniae</a></li>
<li class="chapter" data-level="7.16" data-path="motility.html"><a href="motility.html#mycoplasma-pneumoniae"><i class="fa fa-check"></i><b>7.16</b> Mycoplasma pneumoniae</a></li>
<li class="chapter" data-level="7.17" data-path="motility.html"><a href="motility.html#schematicterminal-organelle-structure"><i class="fa fa-check"></i><b>7.17</b> 12[ Schematic–Terminal organelle structure]</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="navigation.html"><a href="navigation.html"><i class="fa fa-check"></i><b>8</b> Navigation</a><ul>
<li class="chapter" data-level="8.1" data-path="navigation.html"><a href="navigation.html#schematicchemoreceptor-structure"><i class="fa fa-check"></i><b>8.1</b> 1[Schematic–Chemoreceptor structure]</a></li>
<li class="chapter" data-level="8.2" data-path="navigation.html"><a href="navigation.html#styphimurium"><i class="fa fa-check"></i><b>8.2</b> Styphimurium</a></li>
<li class="chapter" data-level="8.3" data-path="navigation.html"><a href="navigation.html#schematic-chemosensory-array-architecture"><i class="fa fa-check"></i><b>8.3</b> 2[Schematic: Chemosensory array architecture]</a></li>
<li class="chapter" data-level="8.4" data-path="navigation.html"><a href="navigation.html#methanospirillum-hungatei-1"><i class="fa fa-check"></i><b>8.4</b> Methanospirillum hungatei</a></li>
<li class="chapter" data-level="8.5" data-path="navigation.html"><a href="navigation.html#schematicconservation-of-chemosensory-array-architecture"><i class="fa fa-check"></i><b>8.5</b> 3[Schematic–Conservation of chemosensory array architecture]</a></li>
<li class="chapter" data-level="8.6" data-path="navigation.html"><a href="navigation.html#vibrio-cholerae"><i class="fa fa-check"></i><b>8.6</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="8.7" data-path="navigation.html"><a href="navigation.html#aa-length-difference-of-as-little-as-2-nm-is-enough-to-separate-receptors-like-these-in-azospirillum-brasilense-that-sense-oxygen-28-nm-array-and-sources-of-energy-like-malate-30-nm-array.-both-receptors-send-signals-to-the-single-flagellar-motor-promoting-runs-or-direction-switches-to-guide-the-bacterium-through-a-combination-of-chemotaxis-and-aerotaxis-ordered-movement-in-an-oxygen-gradient-to-its-target-plant-roots.-a.-brasilense-fixes-nitrogen-boosting-the-growth-of-plants-it-colonizes."><i class="fa fa-check"></i><b>8.7</b> 4aA length difference of as little as 2 nm is enough to separate receptors, like these in <em>Azospirillum brasilense</em> that sense oxygen (28 nm array) and sources of energy like malate (30 nm array). Both receptors send signals to the single flagellar motor, promoting runs or direction switches to guide the bacterium through a combination of chemotaxis and aerotaxis (ordered movement in an oxygen gradient) to its target: plant roots. <em>A. brasilense</em> fixes nitrogen, boosting the growth of plants it colonizes.</a></li>
<li class="chapter" data-level="8.8" data-path="navigation.html"><a href="navigation.html#methanoregula-formicica-1"><i class="fa fa-check"></i><b>8.8</b> Methanoregula formicica</a></li>
<li class="chapter" data-level="8.9" data-path="navigation.html"><a href="navigation.html#schematiccytoplasmic-chemosensory-array-structure"><i class="fa fa-check"></i><b>8.9</b> 5[Schematic–Cytoplasmic chemosensory array structure]</a></li>
<li class="chapter" data-level="8.10" data-path="navigation.html"><a href="navigation.html#athis-vibrio-cholerae-cell-showcases-a-full-repertoire-of-chemosensory-systems-two-membrane-embedded-arrays-the-shorter-one-signaling-to-the-flagellar-motor-and-the-longer-one-signaling-to-a-still-unknown-target-and-a-cytoplasmic-array-also-with-unknown-function.-in-all-cases-the-basic-architecture-is-identical-hexagonally-arranged-trimers-of-chemoreceptor-dimers."><i class="fa fa-check"></i><b>8.10</b> 5aThis <em>Vibrio cholerae</em> cell showcases a full repertoire of chemosensory systems: two membrane-embedded arrays (the shorter one signaling to the flagellar motor and the longer one signaling to a still-unknown target) and a cytoplasmic array (also with unknown function). In all cases, the basic architecture is identical: hexagonally-arranged trimers of chemoreceptor dimers.</a></li>
<li class="chapter" data-level="8.11" data-path="navigation.html"><a href="navigation.html#magnetospirillum-magneticum"><i class="fa fa-check"></i><b>8.11</b> Magnetospirillum magneticum</a></li>
<li class="chapter" data-level="8.12" data-path="navigation.html"><a href="navigation.html#schematicmamk-structure"><i class="fa fa-check"></i><b>8.12</b> 6[Schematic–MamK structure]</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="lifecycle.html"><a href="lifecycle.html"><i class="fa fa-check"></i><b>9</b> Lifecycle</a><ul>
<li class="chapter" data-level="9.1" data-path="lifecycle.html"><a href="lifecycle.html#helicobacter-pylori"><i class="fa fa-check"></i><b>9.1</b> Helicobacter pylori</a></li>
<li class="chapter" data-level="9.2" data-path="lifecycle.html"><a href="lifecycle.html#thermococcus-kodakaraensis-1"><i class="fa fa-check"></i><b>9.2</b> Thermococcus kodakaraensis</a></li>
<li class="chapter" data-level="9.3" data-path="lifecycle.html"><a href="lifecycle.html#caulobacter-crescentus-7"><i class="fa fa-check"></i><b>9.3</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="9.3.1" data-path="lifecycle.html"><a href="lifecycle.html#prosthecobacter-vanneervenii-holdfastprosthecobacter-vanneervenii-is-a-close-relative-of-caulobacter-crescentus-and-attaches-to-surfaces-by-the-same-mechanism-a-sticky-ball-of-holdfast-at-the-tip-of-its-stalk.-here-you-see-four-cells-whose-stalks-have-attached-to-one-another.-unlike-its-cousin-c.-crescentus-however-p.-vanneervenii-is-non-motile-no-flagellum-and-does-not-have-a-dimorphic-life-cycle.-it-divides-symmetrically-to-produce-two-daughters-each-with-their-own-stalk."><i class="fa fa-check"></i><b>9.3.1</b> Prosthecobacter vanneervenii {#Holdfast}<em>Prosthecobacter vanneervenii</em> is a close relative of Caulobacter crescentus, and attaches to surfaces by the same mechanism: a sticky ball of holdfast at the tip of its stalk. Here you see four cells whose stalks have attached to one another. Unlike its cousin <em>C. crescentus</em>, however, <em>P. vanneervenii</em> is non-motile (no flagellum) and does not have a dimorphic life cycle. It divides symmetrically to produce two daughters, each with their own stalk.</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis"><i class="fa fa-check"></i><b>9.4</b> Bacillus subtilis</a><ul>
<li class="chapter" data-level="9.4.1" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-sporulation_septum_formationfrom-hundreds-of-sporulating-bacillus-subtilis-we-imaged-by-cryoet-this-is-the-only-cell-captured-in-the-process-of-forming-a-septum.-as-you-can-see-it-is-at-a-very-early-stage-in-the-process.-the-fact-that-we-do-not-see-more-intermediates-suggests-that-once-the-septum-starts-to-form-it-quickly-proceeds-to-completion."><i class="fa fa-check"></i><b>9.4.1</b> Bacillus subtilis {#Sporulation_septum_formation}From hundreds of sporulating <em>Bacillus subtilis</em> we imaged by cryoET, this is the only cell captured in the process of forming a septum. As you can see, it is at a very early stage in the process. The fact that we do not see more intermediates suggests that once the septum starts to form, it quickly proceeds to completion.</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-1"><i class="fa fa-check"></i><b>9.5</b> Bacillus subtilis</a><ul>
<li class="chapter" data-level="9.5.1" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-spore_engulfmentthe-process-of-spore-engulfment-by-the-mother-cell-membrane-culminates-in-closure-of-the-membranes-creating-an-outer-spore-membrane-and-an-inner-spore-membrane-on-either-side-of-the-sporulation-septum.-the-final-point-of-separation-is-usually-at-the-center-of-the-cell-pole-but-not-always-as-you-can-see-in-this-bacillus-subtilis-spore-being-engulfed-from-the-side-of-the-mother-cell.-the-layer-to-the-right-of-the-developing-spore-may-be-a-precursor-of-the-coat-morespore-coating."><i class="fa fa-check"></i><b>9.5.1</b> Bacillus subtilis {#Spore_engulfment}The process of spore engulfment by the mother cell membrane culminates in closure of the membranes, creating an outer spore membrane and an inner spore membrane on either side of the sporulation septum. The final point of separation is usually at the center of the cell pole, but not always, as you can see in this <em>Bacillus subtilis</em> spore being engulfed from the side of the mother cell. The layer to the right of the developing spore may be a precursor of the coat [More–Spore coating].</a></li>
<li class="chapter" data-level="9.5.2" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-spore_coatingin-this-later-stage-of-sporulation-you-can-see-that-the-bacillus-subtilis-spore-has-now-separated-completely-from-the-mother-cell-envelope.-it-has-also-moved-away-from-the-cell-pole-this-sometimes-occurs-even-as-the-envelopes-are-still-attached-morespore-engulfment.-the-layer-flanking-the-spore-on-the-right-may-be-the-assembling-spore-coat."><i class="fa fa-check"></i><b>9.5.2</b> Bacillus subtilis {#Spore_coating}In this later stage of sporulation, you can see that the <em>Bacillus subtilis</em> spore has now separated completely from the mother cell envelope. It has also moved away from the cell pole (this sometimes occurs even as the envelopes are still attached [More–Spore engulfment]). The layer flanking the spore on the right may be the assembling spore coat.</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-2"><i class="fa fa-check"></i><b>9.6</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="9.7" data-path="lifecycle.html"><a href="lifecycle.html#bacillus-subtilis-3"><i class="fa fa-check"></i><b>9.7</b> Bacillus subtilis</a></li>
<li class="chapter" data-level="9.8" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-1"><i class="fa fa-check"></i><b>9.8</b> Acetonema longum</a></li>
<li class="chapter" data-level="9.9" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-2"><i class="fa fa-check"></i><b>9.9</b> Acetonema longum</a><ul>
<li class="chapter" data-level="9.9.1" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-complete_engulfmentthis-acetonema-longum-cell-has-completed-engulfment-creating-a-forespore.-acetonema-longum-forespores-and-mature-spores-typically-contain-several-granules-likely-storing-poly-phosphate.-these-may-help-power-outgrowth-when-the-spore-germinates.-spores-from-other-species-though-like-bacillus-subtilis-do-not-contain-obvious-storage-granules-so-they-may-serve-another-species-specific-role."><i class="fa fa-check"></i><b>9.9.1</b> Acetonema longum {#Complete_engulfment}This <em>Acetonema longum</em> cell has completed engulfment, creating a “forespore.” <em>Acetonema longum</em> forespores (and mature spores) typically contain several granules, likely storing poly-phosphate. These may help power outgrowth when the spore germinates. Spores from other species, though, like <em>Bacillus subtilis</em>, do not contain obvious storage granules so they may serve another, species-specific role.</a></li>
<li class="chapter" data-level="9.9.2" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-mature_sporethis-mature-acetonema-longum-spore-has-been-released-by-lysis-of-its-mother-cell.-note-its-extensive-armor-a-thick-cortex-between-the-membranes-and-a-multilayered-protein-coat."><i class="fa fa-check"></i><b>9.9.2</b> Acetonema longum {#Mature_spore}This mature <em>Acetonema longum</em> spore has been released by lysis of its mother cell. Note its extensive armor: a thick cortex between the membranes, and a multilayered protein coat.</a></li>
<li class="chapter" data-level="9.9.3" data-path="lifecycle.html"><a href="lifecycle.html#sporomusa-acidovorans-sporomusa_acidovorans_sporulationthis-diderm-sporomusa-acidovorans-cell-is-in-the-process-of-forming-an-ovoid-endospore.-note-also-what-may-be-an-early-stage-of-cortex-formation-at-the-end-of-the-spore-as-well-as-novel-structures-at-the-points-of-cell-wall-remodeling.-we-do-not-know-what-they-are-but-they-may-be-involved-in-engulfment."><i class="fa fa-check"></i><b>9.9.3</b> Sporomusa acidovorans {#Sporomusa_acidovorans_sporulation}This diderm <em>Sporomusa acidovorans</em> cell is in the process of forming an ovoid endospore. Note also what may be an early stage of cortex formation at the end of the spore, as well as novel structures at the points of cell wall remodeling. We do not know what they are, but they may be involved in engulfment.</a></li>
</ul></li>
<li class="chapter" data-level="9.10" data-path="lifecycle.html"><a href="lifecycle.html#acetonema-longum-3"><i class="fa fa-check"></i><b>9.10</b> Acetonema longum</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="interaction.html"><a href="interaction.html"><i class="fa fa-check"></i><b>10</b> Interaction</a><ul>
<li class="chapter" data-level="10.1" data-path="interaction.html"><a href="interaction.html#gluconacetobacter-hansenii"><i class="fa fa-check"></i><b>10.1</b> Gluconacetobacter hansenii</a></li>
<li class="chapter" data-level="10.2" data-path="interaction.html"><a href="interaction.html#legionella-pneumophila"><i class="fa fa-check"></i><b>10.2</b> Legionella pneumophila</a></li>
<li class="chapter" data-level="10.3" data-path="interaction.html"><a href="interaction.html#schematiclegionella-t2ss-structure"><i class="fa fa-check"></i><b>10.3</b> 3[Schematic–Legionella T2SS structure]</a></li>
<li class="chapter" data-level="10.4" data-path="interaction.html"><a href="interaction.html#schematiclegionella-t4ss-structure"><i class="fa fa-check"></i><b>10.4</b> 3[Schematic–Legionella T4SS structure]</a></li>
<li class="chapter" data-level="10.5" data-path="interaction.html"><a href="interaction.html#athis-helicobacter-pylori-cell-was-cultured-together-with-human-gastric-epithelial-cells-its-target-for-infection.-it-contains-inactive-type-iv-secretion-systems-as-well-as-several-mysterious-tubes.-the-tubes-are-formed-from-the-outer-membrane-but-we-do-not-know-what-scaffolds-them-they-have-a-constant-diameter-of-37-nm-or-what-makes-the-portals-along-their-length.-the-fact-that-we-sometimes-see-tubes-near-primed-t4sss-and-the-fact-that-the-tubes-are-missing-from-mutant-cells-lacking-t4ss-components-suggests-a-relationship-but-the-details-are-unclear.-much-remains-mysterious-about-this-structure-and-its-potential-role-in-infection."><i class="fa fa-check"></i><b>10.5</b> 3aThis <em>Helicobacter pylori</em> cell was cultured together with human gastric epithelial cells, its target for infection. It contains inactive type IV secretion systems, as well as several mysterious tubes. The tubes are formed from the outer membrane, but we do not know what scaffolds them (they have a constant diameter of 37 nm), or what makes the portals along their length. The fact that we sometimes see tubes near primed T4SSs, and the fact that the tubes are missing from mutant cells lacking T4SS components suggests a relationship, but the details are unclear. Much remains mysterious about this structure and its potential role in infection.</a></li>
<li class="chapter" data-level="10.6" data-path="interaction.html"><a href="interaction.html#pseudomonas-aeruginosa-δexsd"><i class="fa fa-check"></i><b>10.6</b> Pseudomonas aeruginosa ΔexsD</a></li>
<li class="chapter" data-level="10.7" data-path="interaction.html"><a href="interaction.html#a-emd-2667"><i class="fa fa-check"></i><b>10.7</b> 4a EMD-2667</a></li>
<li class="chapter" data-level="10.8" data-path="interaction.html"><a href="interaction.html#b"><i class="fa fa-check"></i><b>10.8</b> 4b</a></li>
<li class="chapter" data-level="10.9" data-path="interaction.html"><a href="interaction.html#escherichia-coli-cdia"><i class="fa fa-check"></i><b>10.9</b> Escherichia coli +CdiA</a></li>
<li class="chapter" data-level="10.10" data-path="interaction.html"><a href="interaction.html#schematiccdi-mechanism"><i class="fa fa-check"></i><b>10.10</b> 5[Schematic–CDI mechanism]</a></li>
<li class="chapter" data-level="10.11" data-path="interaction.html"><a href="interaction.html#myxococcus-xanthus-2"><i class="fa fa-check"></i><b>10.11</b> Myxococcus xanthus</a></li>
<li class="chapter" data-level="10.12" data-path="interaction.html"><a href="interaction.html#schematicprimed-t6ss-structure"><i class="fa fa-check"></i><b>10.12</b> 6[Schematic–Primed T6SS structure]</a></li>
<li class="chapter" data-level="10.13" data-path="interaction.html"><a href="interaction.html#vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.13</b> Vibrio cholerae</a><ul>
<li class="chapter" data-level="10.13.1" data-path="interaction.html"><a href="interaction.html#amoebophilus-asiaticus-t6ss_batterysome-cells-assemble-a-few-scattered-type-vi-secretion-systems.-other-species-like-this-amoebophilus-asiaticus-build-ordered-arrays-of-parallel-machines.-these-diderm-bacteria-live-symbiotically-inside-amoebae.-their-t6ss-batteries-help-them-colonize-their-host-perhaps-by-rupturing-the-membrane-of-the-host-phagosome-that-encloses-them-after-they-are-engulfed-releasing-them-into-the-host-cytoplasm.-the-number-of-t6sss-in-each-array-varies-but-can-reach-a-few-dozen."><i class="fa fa-check"></i><b>10.13.1</b> Amoebophilus asiaticus {#T6SS_battery}Some cells assemble a few, scattered type VI secretion systems. Other species, like this <em>Amoebophilus asiaticus</em> build ordered arrays of parallel machines. These diderm bacteria live symbiotically inside amoebae. Their T6SS batteries help them colonize their host, perhaps by rupturing the membrane of the host phagosome that encloses them after they are engulfed, releasing them into the host cytoplasm. The number of T6SSs in each array varies, but can reach a few dozen.</a></li>
<li class="chapter" data-level="10.13.2" data-path="interaction.html"><a href="interaction.html#pseudoalteromonas-luteoviolacea-δt6ssbacteriocin-macspseudoalteromonas-luteoviolacea-produce-massive-arrays-of-macs-contractile-machines-related-to-pyocins.-cells-lyse-to-release-the-arrays-like-the-one-you-see-here-which-can-contain-100-individual-macs-and-extend-more-than-1-µm-across.-the-arrays-are-highly-ordered-with-the-tail-fibers-of-the-macs-networked-into-a-hexagonal-array-that-may-help-synchronize-firing-schematicmac-array-structure.-mac-arrays-serve-an-important-function-for-larvae-of-the-marine-tubeworm-hydroides-elegans-and-likely-the-larvae-of-other-invertebrates-such-as-sea-urchins-and-corals-as-well.-they-signal-to-the-larva-that-it-is-the-right-place-to-settle-down-and-differentiate-into-a-sessile-surface-attached-adult.-the-function-of-mac-arrays-for-the-bacterium-is-less-clear.-it-may-be-that-the-invertebrates-benefit-the-lysed-cells-surviving-relatives-in-the-biofilm.-or-perhaps-the-macs-play-another-completely-unrelated-role."><i class="fa fa-check"></i><b>10.13.2</b> Pseudoalteromonas luteoviolacea ΔT6SS/bacteriocin {#MACs}<em>Pseudoalteromonas luteoviolacea</em> produce massive arrays of MACs, contractile machines related to pyocins. Cells lyse to release the arrays, like the one you see here, which can contain 100 individual MACs and extend more than 1 µm across. The arrays are highly ordered, with the tail fibers of the MACs networked into a hexagonal array that may help synchronize firing [Schematic–MAC array structure]. MAC arrays serve an important function for larvae of the marine tubeworm <em>Hydroides elegans</em>, and likely the larvae of other invertebrates such as sea urchins and corals as well. They signal to the larva that it is the right place to settle down and differentiate into a sessile (surface-attached) adult. The function of MAC arrays for the bacterium is less clear. It may be that the invertebrates benefit the lysed cell’s surviving relatives in the biofilm. Or perhaps the MACs play another, completely unrelated role.</a></li>
</ul></li>
<li class="chapter" data-level="10.14" data-path="interaction.html"><a href="interaction.html#schematicmac-array-structure"><i class="fa fa-check"></i><b>10.14</b> 7[Schematic–MAC array structure]</a></li>
<li class="chapter" data-level="10.15" data-path="interaction.html"><a href="interaction.html#porphyromonas-gingivalis"><i class="fa fa-check"></i><b>10.15</b> Porphyromonas gingivalis</a></li>
<li class="chapter" data-level="10.16" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-vibrio-cholerae"><i class="fa fa-check"></i><b>10.16</b> Bdellovibrio bacteriovorus + Vibrio cholerae</a></li>
<li class="chapter" data-level="10.17" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-escherichia-coli-minicell"><i class="fa fa-check"></i><b>10.17</b> Bdellovibrio bacteriovorus + Escherichia coli minicell</a><ul>
<li class="chapter" data-level="10.17.1" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-escherichia-coli-minicell-bdellovibrio_bacteriovorus_entrywe-still-do-not-know-exactly-how-bdellovibrio-bacteriovorus-enters-its-host-but-the-process-is-extremely-rapid.-in-order-to-catch-a-snapshot-we-mixed-b.-bacteriovorus-with-miniaturized-prey-see-details-on-the-main-page.-as-the-b.-bacteriovorus-cell-cannot-fit-inside-the-prey-it-remains-stuck-half-in-and-half-out.-note-the-tight-collar-of-host-outer-membrane-around-the-invader.-how-this-seal-is-maintained-throughout-invasion-remains-a-mystery."><i class="fa fa-check"></i><b>10.17.1</b> Bdellovibrio bacteriovorus + Escherichia coli minicell {#Bdellovibrio_bacteriovorus_entry}We still do not know exactly how <em>Bdellovibrio bacteriovorus</em> enters its host, but the process is extremely rapid. In order to catch a snapshot, we mixed <em>B. bacteriovorus</em> with miniaturized prey (see details on the main page). As the <em>B. bacteriovorus</em> cell cannot fit inside the prey, it remains stuck half-in and half-out. Note the tight collar of host outer membrane around the invader. How this seal is maintained throughout invasion remains a mystery.</a></li>
</ul></li>
<li class="chapter" data-level="10.18" data-path="interaction.html"><a href="interaction.html#bdellovibrio-bacteriovorus-vibrio-cholerae-1"><i class="fa fa-check"></i><b>10.18</b> Bdellovibrio bacteriovorus + Vibrio cholerae</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="viruses.html"><a href="viruses.html"><i class="fa fa-check"></i><b>11</b> Viruses</a><ul>
<li class="chapter" data-level="11.1" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-3"><i class="fa fa-check"></i><b>11.1</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="11.2" data-path="viruses.html"><a href="viruses.html#bacteroidetes-strain-jt5"><i class="fa fa-check"></i><b>11.2</b> Bacteroidetes strain JT5</a></li>
<li class="chapter" data-level="11.3" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-4"><i class="fa fa-check"></i><b>11.3</b> Shewanella oneidensis</a></li>
<li class="chapter" data-level="11.4" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-2"><i class="fa fa-check"></i><b>11.4</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="11.5" data-path="viruses.html"><a href="viruses.html#escherichia-coli-1"><i class="fa fa-check"></i><b>11.5</b> Escherichia coli</a></li>
<li class="chapter" data-level="11.6" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus-8"><i class="fa fa-check"></i><b>11.6</b> Caulobacter crescentus</a><ul>
<li class="chapter" data-level="11.6.1" data-path="viruses.html"><a href="viruses.html#caulobacter-crescentus-φcb13_infectionhere-you-see-several-φcb13-phage-that-have-successfully-reached-their-target-receptors-at-the-pole-of-a-caulobacter-crescentus-and-are-in-the-process-of-infecting-the-cell-note-the-mix-of-filled-capsids-that-have-not-yet-released-their-genome-and-empty-capsids-that-have."><i class="fa fa-check"></i><b>11.6.1</b> Caulobacter crescentus {#φCb13_infection}Here you see several φCb13 phage that have successfully reached their target receptors at the pole of a <em>Caulobacter crescentus</em> and are in the process of infecting the cell (note the mix of filled capsids that have not yet released their genome and empty capsids that have).</a></li>
</ul></li>
<li class="chapter" data-level="11.7" data-path="viruses.html"><a href="viruses.html#vibrio-cholerae-3"><i class="fa fa-check"></i><b>11.7</b> Vibrio cholerae</a></li>
<li class="chapter" data-level="11.8" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-1"><i class="fa fa-check"></i><b>11.8</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="11.9" data-path="viruses.html"><a href="viruses.html#schematicvap-structure"><i class="fa fa-check"></i><b>11.9</b> 8[Schematic–VAP structure]</a></li>
<li class="chapter" data-level="11.10" data-path="viruses.html"><a href="viruses.html#sulfolobus-solfataricus-2"><i class="fa fa-check"></i><b>11.10</b> Sulfolobus solfataricus</a></li>
<li class="chapter" data-level="11.11" data-path="viruses.html"><a href="viruses.html#shewanella-oneidensis-5"><i class="fa fa-check"></i><b>11.11</b> Shewanella oneidensis</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>12</b> References</a></li>
<li class="divider"></li>
<li><a href="https://www.library.caltech.edu" target="blank">Published by Caltech Library</a><li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Cell Atlas</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="viruses" class="section level1">
<h1><span class="header-section-number">Chapter 11</span> Viruses</h1>
<p>“Phages are the winners in the game of life.” - Forest Rohwer <span class="citation">Rohwer et al. (<a href="#ref-rohwer2014">2014</a>)</span></p>
<div id="shewanella-oneidensis-3" class="section level2">
<h2><span class="header-section-number">11.1</span> Shewanella oneidensis</h2>
<p>Life is a battlefield for your cell, and other cells are not the only enemy. Attack can come from another front: viruses called phage (short for bacteriophage, or “bacterium eaters” because they were first identified infecting and killing bacteria). Like all viruses, phage use cells as vehicles for their replication. And, just like eukaryotic viruses, there is an army of different phage that prey on bacteria and archaea. No species is known to be immune.</p>
<p>The basic strategy of a phage is to inject its genome into a host cell, use the cell’s machinery to replicate the genome and manufacture the proteins it encodes, then use these materials to build the next generation of phage. Since the phage is dependent on the cell’s machinery for replication, it does not fulfill the criteria for a living organism. (Despite this distinction, we still talk about the viral “lifecycle.”) By cleverly outsourcing, phage can be highly streamlined structurally, little more than a genome in an envelope en route to its next destination. All phage have a genome (either double or single-stranded DNA or RNA), tightly packed inside a protective envelope of either protein (forming a “capsid”) or lipid, or both. The genome-containing shell is called the “head” of the phage. Many phage also have a tail, like this myophage attacking <em>Shewanella oneidensis</em>. Fibers at the tip of the tail recognize features on their target host and allow the phage to dock.</p>
<div class="figure"><span id="fig:unnamed-chunk-175"></span>
<div id="htmlwidget-5379ca98bd07ca6f4df1" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-5379ca98bd07ca6f4df1">{"x":{"doi":"10.22002/D1.1586","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.1: Shewanella oneidensis Collected by: Poorna Subramanian <a href="https://doi.org/10.22002/D1.1586">10.22002/D1.1586</a>
</p>
</div>
</div>
<div id="bacteroidetes-strain-jt5" class="section level2">
<h2><span class="header-section-number">11.2</span> Bacteroidetes strain JT5</h2>
<p>Once docked, the phage delivers its cargo. The tails of some, but not all, phage are contractile, punching a channel through the cell’s envelope to inject the genome, emptying the capsid as you see in this myophage infecting a <em>Bacteroidetes</em> cell. This syringe-like mechanism should be familiar from the type VI secretion system and related weapons discussed in the last chapter. Evolution coopted phage tails for those structures, likely taking advantage of viral genes integrated into the genome of a bacterial host (more on that in a few pages).</p>
<div class="figure"><span id="fig:unnamed-chunk-176"></span>
<div id="htmlwidget-c8a32585721e2741f758" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-c8a32585721e2741f758">{"x":{"doi":"10.22002/D1.1587","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.2: Bacteroidetes JT5 Collected by: Elitza Tocheva <a href="https://doi.org/10.22002/D1.1587">10.22002/D1.1587</a>
</p>
</div>
</div>
<div id="shewanella-oneidensis-4" class="section level2">
<h2><span class="header-section-number">11.3</span> Shewanella oneidensis</h2>
<p>Some phage have tails that are not contractile, like this siphophage attacking a <em>Shewanella oneidensis</em> cell. The long, flexible tail still serves the essential function of crossing the barrier of the cell’s envelope, providing a conduit for the genome to enter the cell.</p>
<p>Note that we already saw a different type of phage attacking <em>S. oneidensis</em>. While most phage are adapted to infect one or a few closely-related species of bacteria or archaea, there are often multiple different types of phage adapted to attack each species. The diversity of phage is staggering; one type lacks a single gene with detectable homology to any other in any known virus or cell.</p>
<div class="figure"><span id="fig:unnamed-chunk-177"></span>
<div id="htmlwidget-a12e9ab5781f95b1bc11" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-a12e9ab5781f95b1bc11">{"x":{"doi":"10.22002/D1.1588","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.3: Shewanella oneidensis Collected by: Poorna Subramanian <a href="https://doi.org/10.22002/D1.1588">10.22002/D1.1588</a>
</p>
</div>
</div>
<div id="vibrio-cholerae-2" class="section level2">
<h2><span class="header-section-number">11.4</span> Vibrio cholerae</h2>
<p>Not all phage tails are so long. Podophage (which this unidentified attacker of a <em>Vibrio cholerae</em> may be) have short tails that extend after docking to cross the host envelope. Other phage lack tails entirely.</p>
<p>Similarly, heads can look very different. The unique rocket-ship shape of this phage’s capsid highlights the diversity of phage morphology. Some, mostly infecting archaea in extreme environments, have heads shaped like bottles or lemons. Others take the form of long, thin filaments.</p>
<div class="figure"><span id="fig:unnamed-chunk-178"></span>
<div id="htmlwidget-86233ec07ef78967f704" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-86233ec07ef78967f704">{"x":{"doi":"10.22002/D1.1589","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.4: Vibrio cholerae Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1589">10.22002/D1.1589</a>
</p>
</div>
</div>
<div id="escherichia-coli-1" class="section level2">
<h2><span class="header-section-number">11.5</span> Escherichia coli</h2>
<p>Unlike your cell, phage cannot direct their movement. They simply ride the currents of Brownian motion, waiting to bump into a target receptor they can latch onto. Their abundance helps; in the ocean, there are about ten times as many phage as bacteria. Still, it might be a long wait, so evolution has fashioned some mechanisms to speed things up. One strategy is to target a receptor not on the cell body itself, but on a long, more easily-encountered cell appendage.</p>
<p>Some species of bacteria, including this <em>Escherichia coli</em> use “competence pili” assembled by a type IV secretion system to take up DNA from other cells. This “horizontal gene transfer” (in contrast to “vertical” heredity from mother to daughter) lets asexually-reproducing cells mix and match genes to potential advantage. It also blurs the line between species (more about that in the <em>Phylogenetic Explorer</em>). And in this case, it offers an advantage to these MS2 leviphage, which have evolved to recognize the pilus and inject their own RNA into the nucleic acid uptake channel.</p>
<div class="figure"><span id="fig:unnamed-chunk-179"></span>
<div id="htmlwidget-384bb28654c32b70214c" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-384bb28654c32b70214c">{"x":{"doi":"10.22002/D1.1590","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.5: Escherichia coli Collected by: Yi-Wei Chang <a href="https://doi.org/10.22002/D1.1590">10.22002/D1.1590</a>
</p>
</div>
</div>
<div id="caulobacter-crescentus-8" class="section level2">
<h2><span class="header-section-number">11.6</span> Caulobacter crescentus</h2>
<p>Other phage use bacterial appendages in an even cleverer way. These φCb13 myophage take advantage of the cell’s motility to hitch a ride to the cell. They have a filament on their head that wraps into the helical groove of the <em>Caulobacter crescentus</em> flagellum. Remember that flagella can spin either clockwise or counter-clockwise. If this single polar flagellum spins clockwise, pushing the cell body, the attached phage would unscrew like a nut off the end of the flagellum. But if the flagellum spins counter-clockwise, the nut will instead spin down to the cell body, where its tail fibers will find their receptors and its tail will inject the genome [More–φCb13 infection].</p>
<div class="figure"><span id="fig:unnamed-chunk-180"></span>
<div id="htmlwidget-059f3f4dbfee15543515" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-059f3f4dbfee15543515">{"x":{"doi":"10.22002/D1.1591","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.6: Caulobacter crescentus Collected by: Elizabeth Wright <a href="https://doi.org/10.22002/D1.1591">10.22002/D1.1591</a>
</p>
</div>
<div id="caulobacter-crescentus-φcb13_infectionhere-you-see-several-φcb13-phage-that-have-successfully-reached-their-target-receptors-at-the-pole-of-a-caulobacter-crescentus-and-are-in-the-process-of-infecting-the-cell-note-the-mix-of-filled-capsids-that-have-not-yet-released-their-genome-and-empty-capsids-that-have." class="section level3">
<h3><span class="header-section-number">11.6.1</span> Caulobacter crescentus {#φCb13_infection}Here you see several φCb13 phage that have successfully reached their target receptors at the pole of a <em>Caulobacter crescentus</em> and are in the process of infecting the cell (note the mix of filled capsids that have not yet released their genome and empty capsids that have).</h3>
<div class="figure"><span id="fig:unnamed-chunk-181"></span>
<div id="htmlwidget-5af81bcfaab7c57abe3c" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-5af81bcfaab7c57abe3c">{"x":{"doi":"10.22002/D1.1596","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.7: Caulobacter crescentus Collected by: Elizabeth Wright <a href="https://doi.org/10.22002/D1.1596">10.22002/D1.1596</a>
</p>
</div>
</div>
</div>
<div id="vibrio-cholerae-3" class="section level2">
<h2><span class="header-section-number">11.7</span> Vibrio cholerae</h2>
<p>Once inside a cell, phage adopt one of two strategies. The first is to go through a straightforward round of replication, using their host’s machinery. The second is to stick around a while. To do that, the phage inserts its genome into that of its host, creating an addition called a “prophage” that will be propagated indefinitely through cell replication. At some point in the future, usually in response to a stress that threatens the host, the prophage uses special DNA sequences at its ends akin to an ejection seat to pop its genes back out of the host genome, and carry out a normal viral replication and release cycle. Prophage are not simply passengers, though. Some confer a beneficial function on the host; a filamentous inophage carries the disease-causing “cholera toxin” in the <em>Vibrio cholerae</em> genome. Prophage also offer a source of genetic diversity to their host. Mutations sometimes alter the ejection sequences, rendering a prophage a permanent part of the genome, or genetic recombination may shuffle some or part of the prophage into a different part of the genome. Further evolution may then put these pieces to new use. This is likely how bacteria got the contractile weapons you saw in the last chapter, repurposing genes encoding phage tail components.</p>
<p>Whether the phage replicates immediately or waits a while as a prophage first, the cycle of replication and release is rapid. The virus hijacks the cell’s protein-making machinery to churn out packaging proteins like capsid, and its replication machinery to churn out copies of the genome. Genomes are quickly (and very densely) packed into new heads, tails attached if necessary, and the mature phage released. Not all phage lyse their hosts; some filamentous phage (which are commonly beneficial) use secretion machinery to exit without harm. Most, though, use proteins that self-assemble into ports in the inner membrane, allowing another phage-encoded protein to access and chew up the cell wall, causing the cell to lyse. Here you can see a snapshot of this process in a <em>Vibrio cholerae</em> cell infected by a phage that is not its virulence-enhancing friend. The phage is replicating, and has just lysed the cell to make its escape.</p>
<div class="figure"><span id="fig:unnamed-chunk-182"></span>
<div id="htmlwidget-fb2538a73ffadf511ee1" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-fb2538a73ffadf511ee1">{"x":{"doi":"10.22002/D1.1592","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.8: Vibrio cholerae Collected by: Poorna Subramanian <a href="https://doi.org/10.22002/D1.1592">10.22002/D1.1592</a>
</p>
</div>
</div>
<div id="sulfolobus-solfataricus-1" class="section level2">
<h2><span class="header-section-number">11.8</span> Sulfolobus solfataricus</h2>
<p>Some archaeal phage make an even more dramatic exit, through _v_irus-_a_ssociated _p_yramids, or VAPs. You can see these escape hatches in this <em>Sulfolobus solfataricus</em> cell infected with *_S_ulfolobus* _T_urreted _I_cosahedral _V_irus (STIV). The VAPs are made from a single phage-encoded protein that self-assembles into a seven-sided pyramid in the inner membrane [Schematic–VAP structure]. Note how the VAPs poke through and disrupt the cell’s surface layer. Note also the many, many copies the phage has made. We sometimes see storage granules inside VAPs, likely simply pushed there by excluding forces from the densely-packed nucleoid and assembling viral capsids in the center of the cell.</p>
<div class="figure"><span id="fig:unnamed-chunk-183"></span>
<div id="htmlwidget-a1b879416dac2a79c78b" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-a1b879416dac2a79c78b">{"x":{"doi":"10.22002/D1.1593","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.9: Sulfolobus solfataricus Collected by: Lu Gan <a href="https://doi.org/10.22002/D1.1593">10.22002/D1.1593</a>
</p>
</div>
</div>
<div id="schematicvap-structure" class="section level2">
<h2><span class="header-section-number">11.9</span> 8[Schematic–VAP structure]</h2>
<div class="figure"><span id="fig:unnamed-chunk-184"></span>
<div id="htmlwidget-645c0c5dca3da8cb8e2b" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-645c0c5dca3da8cb8e2b">{"x":{"doi":"10.22002/D1.1594","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.10: Sulfolobus solfataricus Collected by: Lu Gan <a href="https://doi.org/10.22002/D1.1594">10.22002/D1.1594</a>
</p>
</div>
</div>
<div id="sulfolobus-solfataricus-2" class="section level2">
<h2><span class="header-section-number">11.10</span> Sulfolobus solfataricus</h2>
<p>When the STIV phage is ready to make its great escape, a still-unknown signal triggers the pyramids to open along their seams, peeling back to create 100 nm-wide holes for the phage, and all the other contents of the cell, to pour out, as you can see in this lysed <em>Sulfolobus solfataricus</em> cell infected with the same virus.</p>
<div class="figure"><span id="fig:unnamed-chunk-185"></span>
<div id="htmlwidget-eac938c8d375cea2d57a" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-eac938c8d375cea2d57a">{"x":{"doi":"10.22002/D1.1594","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.11: Sulfolobus solfataricus Collected by: Lu Gan <a href="https://doi.org/10.22002/D1.1594">10.22002/D1.1594</a>
</p>
</div>
</div>
<div id="shewanella-oneidensis-5" class="section level2">
<h2><span class="header-section-number">11.11</span> Shewanella oneidensis</h2>
<p>So what can your cell do to protect itself from these attackers? The most well-known defense is the CRISPR-Cas system now widely used to edit genomes across the tree of life. Its original function, though, is as an immune system similar to our own. Bacterial and archaeal cells “record” snippets of the genes of infecting phage in their own genome, enabling the cell to rapidly identify and chop up matching genes in future infections, much as our immune system stores the memory of antigen fragments for rapid neutralization in the future.</p>
<p>Defense can also be structural. Surface layers may protect cells from some phage. And then there is always distraction. One function of extracellular vesicles may be to provide decoys for phage to attack, as you can see with this myophage attacking a <em>Shewanella oneidensis</em> outer membrane vesicle. The war between cells and viruses is an ancient one, and continually raging. Despite your cell’s best efforts, it is very likely that it will end its life in a burst of phage. It is far from alone in this fate, though. In the ocean off Florida, it has been estimated that every day phage kill off half the bacterial population. But each lysed cell provides a feast for a starving neighbor, and DNA for more evolutionary tinkering, continuing to shape the weird, wonderful microbial world.</p>
<p>References</p>
<p>Further Reading</p>
<p><span class="citation">Keen (<a href="#ref-keen2015">2015</a>)</span> <span class="citation">Prangishvili et al. (<a href="#ref-prangishvili2017">2017</a>)</span> <span class="citation">Rohwer et al. (<a href="#ref-rohwer2014">2014</a>)</span></p>
<div class="figure"><span id="fig:unnamed-chunk-186"></span>
<div id="htmlwidget-171af3ceb57cb2e89d36" style="width:672px;height:480px;" class="doivideo html-widget"></div>
<script type="application/json" data-for="htmlwidget-171af3ceb57cb2e89d36">{"x":{"doi":"10.22002/D1.1595","index":0},"evals":[],"jsHooks":[]}</script>
<p class="caption">
Figure 11.12: Shewanella oneidensis Collected by: Lauren Ann Metskas <a href="https://doi.org/10.22002/D1.1595">10.22002/D1.1595</a>
</p>
</div>

</div>
</div>
<h3> References</h3>
<div id="refs" class="references">
<div id="ref-keen2015">
<p>Keen, E. C. 2015. “A Century of Phage Research: Bacteriophages and the Shaping of Modern Biology.” <em>Bioessays</em> 37 (1): 6–9. doi:<a href="https://doi.org/10.1002/bies.201400152">10.1002/bies.201400152</a>.</p>
</div>
<div id="ref-prangishvili2017">
<p>Prangishvili, D., D. H. Bamford, P. Forterre, J. Iranzo, E. V. Koonin, and M. Krupovic. 2017. “The Enigmatic Archaeal Virosphere.” <em>Nat Rev Microbiol</em> 15 (12): 724–39. doi:<a href="https://doi.org/10.1038/nrmicro.2017.125">10.1038/nrmicro.2017.125</a>.</p>
</div>
<div id="ref-rohwer2014">
<p>Rohwer, F., M. Youle, H. Maughan, and N. Hisakawa. 2014. <em>Life in Our Phage World</em>. San Diego, CA: Wholon.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="interaction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": null,
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/caltechlibrary/cell-atlas/edit/master/10_Viruses.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["cell-atlas.pdf", "cell-atlas.epub"],
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
